{"nbformat":4,"nbformat_minor":0,"metadata":{"colab":{"name":"Balrog_0.2.1.ipynb","provenance":[],"collapsed_sections":[],"toc_visible":true,"machine_shape":"hm","mount_file_id":"1rAQd-UsKhliEHw_tDMGv2a_UMS4iPA3V","authorship_tag":"ABX9TyOY6ldtg5zPu8nKdZdYiJF0"},"kernelspec":{"name":"python3","display_name":"Python 3"},"accelerator":"GPU","widgets":{"application/vnd.jupyter.widget-state+json":{"da4cdb519f6b4666beb6c961d72ec00f":{"model_module":"@jupyter-widgets/controls","model_name":"HBoxModel","state":{"_view_name":"HBoxView","_dom_classes":[],"_model_name":"HBoxModel","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.5.0","box_style":"","layout":"IPY_MODEL_d3bde4a8d3a6425d9e6be647d3da3508","_model_module":"@jupyter-widgets/controls","children":["IPY_MODEL_1eeed60e23394683b08c69c3b49320e3","IPY_MODEL_871c9d93cd164b469d0db083d564170f"]}},"d3bde4a8d3a6425d9e6be647d3da3508":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"1eeed60e23394683b08c69c3b49320e3":{"model_module":"@jupyter-widgets/controls","model_name":"FloatProgressModel","state":{"_view_name":"ProgressView","style":"IPY_MODEL_ae270eec15ce4cf8a732b5f01ab50f32","_dom_classes":[],"description":"100%","_model_name":"FloatProgressModel","bar_style":"success","max":25,"_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":25,"_view_count":null,"_view_module_version":"1.5.0","orientation":"horizontal","min":0,"description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_c9f42d5e44eb4e2685074de46c1ec7a3"}},"871c9d93cd164b469d0db083d564170f":{"model_module":"@jupyter-widgets/controls","model_name":"HTMLModel","state":{"_view_name":"HTMLView","style":"IPY_MODEL_422c6dc5dfe34eb883589d2c95372375","_dom_classes":[],"description":"","_model_name":"HTMLModel","placeholder":"​","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":" 25/25 [02:33&lt;00:00,  6.12s/ batch]","_view_count":null,"_view_module_version":"1.5.0","description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_06bcbdfbe6aa41388e032ae794ca3612"}},"ae270eec15ce4cf8a732b5f01ab50f32":{"model_module":"@jupyter-widgets/controls","model_name":"ProgressStyleModel","state":{"_view_name":"StyleView","_model_name":"ProgressStyleModel","description_width":"initial","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","bar_color":null,"_model_module":"@jupyter-widgets/controls"}},"c9f42d5e44eb4e2685074de46c1ec7a3":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"422c6dc5dfe34eb883589d2c95372375":{"model_module":"@jupyter-widgets/controls","model_name":"DescriptionStyleModel","state":{"_view_name":"StyleView","_model_name":"DescriptionStyleModel","description_width":"","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","_model_module":"@jupyter-widgets/controls"}},"06bcbdfbe6aa41388e032ae794ca3612":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"b14bd3ff1365460fa6b77069a7d29d9d":{"model_module":"@jupyter-widgets/controls","model_name":"HBoxModel","state":{"_view_name":"HBoxView","_dom_classes":[],"_model_name":"HBoxModel","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.5.0","box_style":"","layout":"IPY_MODEL_02d760d5d0a54aeb83b7102378b827d2","_model_module":"@jupyter-widgets/controls","children":["IPY_MODEL_df26c1fa04a54d52bdaf203aacb334f0","IPY_MODEL_1bc273565e2a4be3bc2f79f89a7360cf"]}},"02d760d5d0a54aeb83b7102378b827d2":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"df26c1fa04a54d52bdaf203aacb334f0":{"model_module":"@jupyter-widgets/controls","model_name":"FloatProgressModel","state":{"_view_name":"ProgressView","style":"IPY_MODEL_694bd9677f894f9a84c1688590b51ece","_dom_classes":[],"description":"100%","_model_name":"FloatProgressModel","bar_style":"success","max":8,"_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":8,"_view_count":null,"_view_module_version":"1.5.0","orientation":"horizontal","min":0,"description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_f165dc550e144003a55a0face7905d25"}},"1bc273565e2a4be3bc2f79f89a7360cf":{"model_module":"@jupyter-widgets/controls","model_name":"HTMLModel","state":{"_view_name":"HTMLView","style":"IPY_MODEL_09cd78096e4547dc830d8c7690dfcd99","_dom_classes":[],"description":"","_model_name":"HTMLModel","placeholder":"​","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":" 8/8 [02:21&lt;00:00, 17.74s/ batch]","_view_count":null,"_view_module_version":"1.5.0","description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_7a5a5b5dda594b30b2496c868abce4d8"}},"694bd9677f894f9a84c1688590b51ece":{"model_module":"@jupyter-widgets/controls","model_name":"ProgressStyleModel","state":{"_view_name":"StyleView","_model_name":"ProgressStyleModel","description_width":"initial","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","bar_color":null,"_model_module":"@jupyter-widgets/controls"}},"f165dc550e144003a55a0face7905d25":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"09cd78096e4547dc830d8c7690dfcd99":{"model_module":"@jupyter-widgets/controls","model_name":"DescriptionStyleModel","state":{"_view_name":"StyleView","_model_name":"DescriptionStyleModel","description_width":"","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","_model_module":"@jupyter-widgets/controls"}},"7a5a5b5dda594b30b2496c868abce4d8":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"7097bf4a7d214150b67139f710b78906":{"model_module":"@jupyter-widgets/controls","model_name":"HBoxModel","state":{"_view_name":"HBoxView","_dom_classes":[],"_model_name":"HBoxModel","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.5.0","box_style":"","layout":"IPY_MODEL_eddfa44895784f9db52b5a07d6ccc706","_model_module":"@jupyter-widgets/controls","children":["IPY_MODEL_12d5adcc1c2145779e457e098978e657","IPY_MODEL_d5bade233f784a6eb8224a92585eecab"]}},"eddfa44895784f9db52b5a07d6ccc706":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"12d5adcc1c2145779e457e098978e657":{"model_module":"@jupyter-widgets/controls","model_name":"FloatProgressModel","state":{"_view_name":"ProgressView","style":"IPY_MODEL_ed2f96be5384494ca6bc8db664fe2860","_dom_classes":[],"description":"100%","_model_name":"FloatProgressModel","bar_style":"success","max":72662,"_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":72662,"_view_count":null,"_view_module_version":"1.5.0","orientation":"horizontal","min":0,"description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_70bc560c0ae14a068ce5dfab409b33fa"}},"d5bade233f784a6eb8224a92585eecab":{"model_module":"@jupyter-widgets/controls","model_name":"HTMLModel","state":{"_view_name":"HTMLView","style":"IPY_MODEL_3a36a242d82b478f8635e6e95dfff7e2","_dom_classes":[],"description":"","_model_name":"HTMLModel","placeholder":"​","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":" 72662/72662 [02:20&lt;00:00, 515.54 ORF/s]","_view_count":null,"_view_module_version":"1.5.0","description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_13a7e3fed79b4d7690afd032f6bc140b"}},"ed2f96be5384494ca6bc8db664fe2860":{"model_module":"@jupyter-widgets/controls","model_name":"ProgressStyleModel","state":{"_view_name":"StyleView","_model_name":"ProgressStyleModel","description_width":"initial","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","bar_color":null,"_model_module":"@jupyter-widgets/controls"}},"70bc560c0ae14a068ce5dfab409b33fa":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"3a36a242d82b478f8635e6e95dfff7e2":{"model_module":"@jupyter-widgets/controls","model_name":"DescriptionStyleModel","state":{"_view_name":"StyleView","_model_name":"DescriptionStyleModel","description_width":"","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","_model_module":"@jupyter-widgets/controls"}},"13a7e3fed79b4d7690afd032f6bc140b":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"6952fed2527b44f5b8c3c4baf6e42323":{"model_module":"@jupyter-widgets/controls","model_name":"HBoxModel","state":{"_view_name":"HBoxView","_dom_classes":[],"_model_name":"HBoxModel","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.5.0","box_style":"","layout":"IPY_MODEL_94be386a3dd247ce94854c1885d2af87","_model_module":"@jupyter-widgets/controls","children":["IPY_MODEL_64dd6ee8f97d4987b9dccf3f22af88fd","IPY_MODEL_763a27ed77b44335b435fb379e2042c0"]}},"94be386a3dd247ce94854c1885d2af87":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"64dd6ee8f97d4987b9dccf3f22af88fd":{"model_module":"@jupyter-widgets/controls","model_name":"FloatProgressModel","state":{"_view_name":"ProgressView","style":"IPY_MODEL_417744e421a94605a8bee944c1277ff7","_dom_classes":[],"description":"100%","_model_name":"FloatProgressModel","bar_style":"success","max":42,"_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":42,"_view_count":null,"_view_module_version":"1.5.0","orientation":"horizontal","min":0,"description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_0da25a719e6f4cafb69b91cf96bf9066"}},"763a27ed77b44335b435fb379e2042c0":{"model_module":"@jupyter-widgets/controls","model_name":"HTMLModel","state":{"_view_name":"HTMLView","style":"IPY_MODEL_11d348f3d6064674a9917625d1a05f18","_dom_classes":[],"description":"","_model_name":"HTMLModel","placeholder":"​","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":" 42/42 [00:23&lt;00:00,  1.80 batch/s]","_view_count":null,"_view_module_version":"1.5.0","description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_580e8c67600745eb9dff2fa9e91e7875"}},"417744e421a94605a8bee944c1277ff7":{"model_module":"@jupyter-widgets/controls","model_name":"ProgressStyleModel","state":{"_view_name":"StyleView","_model_name":"ProgressStyleModel","description_width":"initial","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","bar_color":null,"_model_module":"@jupyter-widgets/controls"}},"0da25a719e6f4cafb69b91cf96bf9066":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"11d348f3d6064674a9917625d1a05f18":{"model_module":"@jupyter-widgets/controls","model_name":"DescriptionStyleModel","state":{"_view_name":"StyleView","_model_name":"DescriptionStyleModel","description_width":"","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","_model_module":"@jupyter-widgets/controls"}},"580e8c67600745eb9dff2fa9e91e7875":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"8e1e5a8fb3d340be9050c46787dfa567":{"model_module":"@jupyter-widgets/controls","model_name":"HBoxModel","state":{"_view_name":"HBoxView","_dom_classes":[],"_model_name":"HBoxModel","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.5.0","box_style":"","layout":"IPY_MODEL_d76cef09dc574192b61c69e1b7dcb280","_model_module":"@jupyter-widgets/controls","children":["IPY_MODEL_9f829c0e9d9f4fa4a1cbad1d70ee651b","IPY_MODEL_e1b66fbd726d438a921a38a72cdb8517"]}},"d76cef09dc574192b61c69e1b7dcb280":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"9f829c0e9d9f4fa4a1cbad1d70ee651b":{"model_module":"@jupyter-widgets/controls","model_name":"FloatProgressModel","state":{"_view_name":"ProgressView","style":"IPY_MODEL_93f5a7d134584a3c9790a3f1fa48ddac","_dom_classes":[],"description":"100%","_model_name":"FloatProgressModel","bar_style":"success","max":13,"_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":13,"_view_count":null,"_view_module_version":"1.5.0","orientation":"horizontal","min":0,"description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_26bdfeed0cf44f9697c23f8489a1d552"}},"e1b66fbd726d438a921a38a72cdb8517":{"model_module":"@jupyter-widgets/controls","model_name":"HTMLModel","state":{"_view_name":"HTMLView","style":"IPY_MODEL_24aa927e762c4dcfb02b3923efa39fc3","_dom_classes":[],"description":"","_model_name":"HTMLModel","placeholder":"​","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":" 13/13 [00:05&lt;00:00,  2.28 batch/s]","_view_count":null,"_view_module_version":"1.5.0","description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_9acbc9592970499cbe566bad1eceb409"}},"93f5a7d134584a3c9790a3f1fa48ddac":{"model_module":"@jupyter-widgets/controls","model_name":"ProgressStyleModel","state":{"_view_name":"StyleView","_model_name":"ProgressStyleModel","description_width":"initial","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","bar_color":null,"_model_module":"@jupyter-widgets/controls"}},"26bdfeed0cf44f9697c23f8489a1d552":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"24aa927e762c4dcfb02b3923efa39fc3":{"model_module":"@jupyter-widgets/controls","model_name":"DescriptionStyleModel","state":{"_view_name":"StyleView","_model_name":"DescriptionStyleModel","description_width":"","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","_model_module":"@jupyter-widgets/controls"}},"9acbc9592970499cbe566bad1eceb409":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"e4f66d6bc4eb4de9b13c1ca3cba2dfd1":{"model_module":"@jupyter-widgets/controls","model_name":"HBoxModel","state":{"_view_name":"HBoxView","_dom_classes":[],"_model_name":"HBoxModel","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.5.0","box_style":"","layout":"IPY_MODEL_f5638b740dda4dcf9e02dcb3db3f850a","_model_module":"@jupyter-widgets/controls","children":["IPY_MODEL_286f32fabedb41c4ae250736ed4b2687","IPY_MODEL_6c862d4d685640a6bbf756a45b6ca358"]}},"f5638b740dda4dcf9e02dcb3db3f850a":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"286f32fabedb41c4ae250736ed4b2687":{"model_module":"@jupyter-widgets/controls","model_name":"FloatProgressModel","state":{"_view_name":"ProgressView","style":"IPY_MODEL_b0b1b6aceac647e7bc87c1e6b7ab963e","_dom_classes":[],"description":"100%","_model_name":"FloatProgressModel","bar_style":"success","max":123251,"_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":123251,"_view_count":null,"_view_module_version":"1.5.0","orientation":"horizontal","min":0,"description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_1966050d50df4e5fa61c344bcbedcf72"}},"6c862d4d685640a6bbf756a45b6ca358":{"model_module":"@jupyter-widgets/controls","model_name":"HTMLModel","state":{"_view_name":"HTMLView","style":"IPY_MODEL_8c84f0524ea54746af648e61f4de1357","_dom_classes":[],"description":"","_model_name":"HTMLModel","placeholder":"​","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":" 123251/123251 [03:45&lt;00:00, 547.54 ORF/s]","_view_count":null,"_view_module_version":"1.5.0","description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_cf24edd863e54251aaafecba928a1af1"}},"b0b1b6aceac647e7bc87c1e6b7ab963e":{"model_module":"@jupyter-widgets/controls","model_name":"ProgressStyleModel","state":{"_view_name":"StyleView","_model_name":"ProgressStyleModel","description_width":"initial","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","bar_color":null,"_model_module":"@jupyter-widgets/controls"}},"1966050d50df4e5fa61c344bcbedcf72":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"8c84f0524ea54746af648e61f4de1357":{"model_module":"@jupyter-widgets/controls","model_name":"DescriptionStyleModel","state":{"_view_name":"StyleView","_model_name":"DescriptionStyleModel","description_width":"","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","_model_module":"@jupyter-widgets/controls"}},"cf24edd863e54251aaafecba928a1af1":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"514c76651fe8496093dce5ec30a93235":{"model_module":"@jupyter-widgets/controls","model_name":"HBoxModel","state":{"_view_name":"HBoxView","_dom_classes":[],"_model_name":"HBoxModel","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.5.0","box_style":"","layout":"IPY_MODEL_be91e05dd6f1462499455e735520c6c4","_model_module":"@jupyter-widgets/controls","children":["IPY_MODEL_e693bb84b9364e8788d4967558ae4aee","IPY_MODEL_0cdf6419ae774738aa29042c7524b8ab"]}},"be91e05dd6f1462499455e735520c6c4":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"e693bb84b9364e8788d4967558ae4aee":{"model_module":"@jupyter-widgets/controls","model_name":"FloatProgressModel","state":{"_view_name":"ProgressView","style":"IPY_MODEL_16945ee114f144e69b7e487de75f3ebe","_dom_classes":[],"description":"100%","_model_name":"FloatProgressModel","bar_style":"success","max":112,"_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":112,"_view_count":null,"_view_module_version":"1.5.0","orientation":"horizontal","min":0,"description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_d8ea1410a9a14dc8b32c48aed0e7f8a9"}},"0cdf6419ae774738aa29042c7524b8ab":{"model_module":"@jupyter-widgets/controls","model_name":"HTMLModel","state":{"_view_name":"HTMLView","style":"IPY_MODEL_3dfa0d577cde4648915080b283556bb9","_dom_classes":[],"description":"","_model_name":"HTMLModel","placeholder":"​","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":" 112/112 [00:42&lt;00:00,  2.65 batch/s]","_view_count":null,"_view_module_version":"1.5.0","description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_2c9d4b616c694dc5b9501e739ffd5ce2"}},"16945ee114f144e69b7e487de75f3ebe":{"model_module":"@jupyter-widgets/controls","model_name":"ProgressStyleModel","state":{"_view_name":"StyleView","_model_name":"ProgressStyleModel","description_width":"initial","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","bar_color":null,"_model_module":"@jupyter-widgets/controls"}},"d8ea1410a9a14dc8b32c48aed0e7f8a9":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"3dfa0d577cde4648915080b283556bb9":{"model_module":"@jupyter-widgets/controls","model_name":"DescriptionStyleModel","state":{"_view_name":"StyleView","_model_name":"DescriptionStyleModel","description_width":"","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","_model_module":"@jupyter-widgets/controls"}},"2c9d4b616c694dc5b9501e739ffd5ce2":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"8b2846b7778741789dca148ee24644c8":{"model_module":"@jupyter-widgets/controls","model_name":"HBoxModel","state":{"_view_name":"HBoxView","_dom_classes":[],"_model_name":"HBoxModel","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.5.0","box_style":"","layout":"IPY_MODEL_ecd26d2ebf0143099c6cd7d898782314","_model_module":"@jupyter-widgets/controls","children":["IPY_MODEL_1fa2112045a34649af22014f4573a8b1","IPY_MODEL_b285e2674307413a8fa3b2ce129fa8a7"]}},"ecd26d2ebf0143099c6cd7d898782314":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"1fa2112045a34649af22014f4573a8b1":{"model_module":"@jupyter-widgets/controls","model_name":"FloatProgressModel","state":{"_view_name":"ProgressView","style":"IPY_MODEL_151bf004a7be41e797af74f9af0467da","_dom_classes":[],"description":"100%","_model_name":"FloatProgressModel","bar_style":"success","max":34,"_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":34,"_view_count":null,"_view_module_version":"1.5.0","orientation":"horizontal","min":0,"description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_c28cbdf0377b4a26ae445a27da23257d"}},"b285e2674307413a8fa3b2ce129fa8a7":{"model_module":"@jupyter-widgets/controls","model_name":"HTMLModel","state":{"_view_name":"HTMLView","style":"IPY_MODEL_49276ce427b54502985e7d75ec3dc68a","_dom_classes":[],"description":"","_model_name":"HTMLModel","placeholder":"​","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":" 34/34 [00:41&lt;00:00,  1.23s/ batch]","_view_count":null,"_view_module_version":"1.5.0","description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_223927e5925f49bbaf4756efe366f7be"}},"151bf004a7be41e797af74f9af0467da":{"model_module":"@jupyter-widgets/controls","model_name":"ProgressStyleModel","state":{"_view_name":"StyleView","_model_name":"ProgressStyleModel","description_width":"initial","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","bar_color":null,"_model_module":"@jupyter-widgets/controls"}},"c28cbdf0377b4a26ae445a27da23257d":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"49276ce427b54502985e7d75ec3dc68a":{"model_module":"@jupyter-widgets/controls","model_name":"DescriptionStyleModel","state":{"_view_name":"StyleView","_model_name":"DescriptionStyleModel","description_width":"","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","_model_module":"@jupyter-widgets/controls"}},"223927e5925f49bbaf4756efe366f7be":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"178edc8405c24d979b6b7ee906db4adc":{"model_module":"@jupyter-widgets/controls","model_name":"HBoxModel","state":{"_view_name":"HBoxView","_dom_classes":[],"_model_name":"HBoxModel","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.5.0","box_style":"","layout":"IPY_MODEL_871e4c30234a4682a78d86ca317ac461","_model_module":"@jupyter-widgets/controls","children":["IPY_MODEL_844615901a194af385b5d73066ebdcb1","IPY_MODEL_e0ceb7c6b0174789b44e0a222ab99bbf"]}},"871e4c30234a4682a78d86ca317ac461":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"844615901a194af385b5d73066ebdcb1":{"model_module":"@jupyter-widgets/controls","model_name":"FloatProgressModel","state":{"_view_name":"ProgressView","style":"IPY_MODEL_7fc7382b989e40efbb208cb8a297bf70","_dom_classes":[],"description":"100%","_model_name":"FloatProgressModel","bar_style":"success","max":302978,"_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":302978,"_view_count":null,"_view_module_version":"1.5.0","orientation":"horizontal","min":0,"description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_de39049e0a54431aaafebd99c11ce997"}},"e0ceb7c6b0174789b44e0a222ab99bbf":{"model_module":"@jupyter-widgets/controls","model_name":"HTMLModel","state":{"_view_name":"HTMLView","style":"IPY_MODEL_7213abe1316d4a3f906a67e419a23f78","_dom_classes":[],"description":"","_model_name":"HTMLModel","placeholder":"​","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":" 302978/302978 [10:39&lt;00:00, 473.56 ORF/s]","_view_count":null,"_view_module_version":"1.5.0","description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_c58b0550035441cc895db9a7cf7fe695"}},"7fc7382b989e40efbb208cb8a297bf70":{"model_module":"@jupyter-widgets/controls","model_name":"ProgressStyleModel","state":{"_view_name":"StyleView","_model_name":"ProgressStyleModel","description_width":"initial","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","bar_color":null,"_model_module":"@jupyter-widgets/controls"}},"de39049e0a54431aaafebd99c11ce997":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"7213abe1316d4a3f906a67e419a23f78":{"model_module":"@jupyter-widgets/controls","model_name":"DescriptionStyleModel","state":{"_view_name":"StyleView","_model_name":"DescriptionStyleModel","description_width":"","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","_model_module":"@jupyter-widgets/controls"}},"c58b0550035441cc895db9a7cf7fe695":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"33254081567441e8a0a886e191604f93":{"model_module":"@jupyter-widgets/controls","model_name":"HBoxModel","state":{"_view_name":"HBoxView","_dom_classes":[],"_model_name":"HBoxModel","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.5.0","box_style":"","layout":"IPY_MODEL_f4093d6e83e748ab89188792c21962ec","_model_module":"@jupyter-widgets/controls","children":["IPY_MODEL_8cc2e136ae6646ad8311bf483198980d","IPY_MODEL_db26a6b65a404aad82e46704f5e4a31b"]}},"f4093d6e83e748ab89188792c21962ec":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"8cc2e136ae6646ad8311bf483198980d":{"model_module":"@jupyter-widgets/controls","model_name":"FloatProgressModel","state":{"_view_name":"ProgressView","style":"IPY_MODEL_d034968b22d34616b192c856190aad76","_dom_classes":[],"description":"100%","_model_name":"FloatProgressModel","bar_style":"success","max":3700,"_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":3700,"_view_count":null,"_view_module_version":"1.5.0","orientation":"horizontal","min":0,"description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_8d7de09dd85b4a5b8fc9405356c92b05"}},"db26a6b65a404aad82e46704f5e4a31b":{"model_module":"@jupyter-widgets/controls","model_name":"HTMLModel","state":{"_view_name":"HTMLView","style":"IPY_MODEL_86e5c4bf8e7b4d37a1a9d547886b6bd1","_dom_classes":[],"description":"","_model_name":"HTMLModel","placeholder":"​","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":" 3700/3700 [04:15&lt;00:00, 14.48 ORF/s]","_view_count":null,"_view_module_version":"1.5.0","description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_0781a951ae7b4af3a8523c36081a3799"}},"d034968b22d34616b192c856190aad76":{"model_module":"@jupyter-widgets/controls","model_name":"ProgressStyleModel","state":{"_view_name":"StyleView","_model_name":"ProgressStyleModel","description_width":"initial","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","bar_color":null,"_model_module":"@jupyter-widgets/controls"}},"8d7de09dd85b4a5b8fc9405356c92b05":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"86e5c4bf8e7b4d37a1a9d547886b6bd1":{"model_module":"@jupyter-widgets/controls","model_name":"DescriptionStyleModel","state":{"_view_name":"StyleView","_model_name":"DescriptionStyleModel","description_width":"","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","_model_module":"@jupyter-widgets/controls"}},"0781a951ae7b4af3a8523c36081a3799":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"59be591aa4e04fbf9cebacfd3c697dd4":{"model_module":"@jupyter-widgets/controls","model_name":"HBoxModel","state":{"_view_name":"HBoxView","_dom_classes":[],"_model_name":"HBoxModel","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.5.0","box_style":"","layout":"IPY_MODEL_14db8feca9bd406da5801430c1bab5f3","_model_module":"@jupyter-widgets/controls","children":["IPY_MODEL_be631a5be4d34a42bb7e0a7a91d6c5df","IPY_MODEL_e793877ba7bc46edb916982a8ba2e793"]}},"14db8feca9bd406da5801430c1bab5f3":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"be631a5be4d34a42bb7e0a7a91d6c5df":{"model_module":"@jupyter-widgets/controls","model_name":"FloatProgressModel","state":{"_view_name":"ProgressView","style":"IPY_MODEL_d60ceb70cf1c40d0b373189de129837e","_dom_classes":[],"description":"100%","_model_name":"FloatProgressModel","bar_style":"success","max":24280,"_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":24280,"_view_count":null,"_view_module_version":"1.5.0","orientation":"horizontal","min":0,"description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_465e693afc954ad3b13456efbed1cfa0"}},"e793877ba7bc46edb916982a8ba2e793":{"model_module":"@jupyter-widgets/controls","model_name":"HTMLModel","state":{"_view_name":"HTMLView","style":"IPY_MODEL_0f3043541faf4a0a9d9fce4064c74195","_dom_classes":[],"description":"","_model_name":"HTMLModel","placeholder":"​","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":" 24280/24280 [03:13&lt;00:00, 125.25 ORF/s]","_view_count":null,"_view_module_version":"1.5.0","description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_da3ff2c9e9ef47f3b9a07d35610e4598"}},"d60ceb70cf1c40d0b373189de129837e":{"model_module":"@jupyter-widgets/controls","model_name":"ProgressStyleModel","state":{"_view_name":"StyleView","_model_name":"ProgressStyleModel","description_width":"initial","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","bar_color":null,"_model_module":"@jupyter-widgets/controls"}},"465e693afc954ad3b13456efbed1cfa0":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"0f3043541faf4a0a9d9fce4064c74195":{"model_module":"@jupyter-widgets/controls","model_name":"DescriptionStyleModel","state":{"_view_name":"StyleView","_model_name":"DescriptionStyleModel","description_width":"","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","_model_module":"@jupyter-widgets/controls"}},"da3ff2c9e9ef47f3b9a07d35610e4598":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"14da851f7f264917b338e5c2ea0cd230":{"model_module":"@jupyter-widgets/controls","model_name":"HBoxModel","state":{"_view_name":"HBoxView","_dom_classes":[],"_model_name":"HBoxModel","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.5.0","box_style":"","layout":"IPY_MODEL_b6fc5e16cc094cd0aae84aef905d5778","_model_module":"@jupyter-widgets/controls","children":["IPY_MODEL_03aabf73446b48979ec56ecd9e783418","IPY_MODEL_68c1745c0bc149a786d44c1f4cbb9dfb"]}},"b6fc5e16cc094cd0aae84aef905d5778":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"03aabf73446b48979ec56ecd9e783418":{"model_module":"@jupyter-widgets/controls","model_name":"FloatProgressModel","state":{"_view_name":"ProgressView","style":"IPY_MODEL_735179f8ad51422c9bc08184300bbc42","_dom_classes":[],"description":"100%","_model_name":"FloatProgressModel","bar_style":"success","max":2086,"_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":2086,"_view_count":null,"_view_module_version":"1.5.0","orientation":"horizontal","min":0,"description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_47c22f8b5624436db06b1ab5733dbf83"}},"68c1745c0bc149a786d44c1f4cbb9dfb":{"model_module":"@jupyter-widgets/controls","model_name":"HTMLModel","state":{"_view_name":"HTMLView","style":"IPY_MODEL_49401a7216db453b8d6399e2ac7c7674","_dom_classes":[],"description":"","_model_name":"HTMLModel","placeholder":"​","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":" 2086/2086 [01:04&lt;00:00, 32.17 ORF/s]","_view_count":null,"_view_module_version":"1.5.0","description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_34a06dd9e1e546d8af70a81c34b5e020"}},"735179f8ad51422c9bc08184300bbc42":{"model_module":"@jupyter-widgets/controls","model_name":"ProgressStyleModel","state":{"_view_name":"StyleView","_model_name":"ProgressStyleModel","description_width":"initial","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","bar_color":null,"_model_module":"@jupyter-widgets/controls"}},"47c22f8b5624436db06b1ab5733dbf83":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"49401a7216db453b8d6399e2ac7c7674":{"model_module":"@jupyter-widgets/controls","model_name":"DescriptionStyleModel","state":{"_view_name":"StyleView","_model_name":"DescriptionStyleModel","description_width":"","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","_model_module":"@jupyter-widgets/controls"}},"34a06dd9e1e546d8af70a81c34b5e020":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"ad34d5fafbea45598758d5a9658ebdc6":{"model_module":"@jupyter-widgets/controls","model_name":"HBoxModel","state":{"_view_name":"HBoxView","_dom_classes":[],"_model_name":"HBoxModel","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.5.0","box_style":"","layout":"IPY_MODEL_e170eee78e0d417a941bf672fcebaf29","_model_module":"@jupyter-widgets/controls","children":["IPY_MODEL_dddebf28504c40c3ae83fe583999f35a","IPY_MODEL_473a227e09cb429f8d78593801d1bd42"]}},"e170eee78e0d417a941bf672fcebaf29":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"dddebf28504c40c3ae83fe583999f35a":{"model_module":"@jupyter-widgets/controls","model_name":"FloatProgressModel","state":{"_view_name":"ProgressView","style":"IPY_MODEL_13f8cfed826c4e509198cfb1dd8ef99c","_dom_classes":[],"description":"100%","_model_name":"FloatProgressModel","bar_style":"success","max":44,"_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":44,"_view_count":null,"_view_module_version":"1.5.0","orientation":"horizontal","min":0,"description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_3373940272584820a6fa53d80bcb438e"}},"473a227e09cb429f8d78593801d1bd42":{"model_module":"@jupyter-widgets/controls","model_name":"HTMLModel","state":{"_view_name":"HTMLView","style":"IPY_MODEL_f44b9edb35fc4b68966dbfd5b837aabe","_dom_classes":[],"description":"","_model_name":"HTMLModel","placeholder":"​","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":" 44/44 [00:16&lt;00:00,  2.62 batch/s]","_view_count":null,"_view_module_version":"1.5.0","description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_d6fa4a8231d045eca7a79c4c55ffa562"}},"13f8cfed826c4e509198cfb1dd8ef99c":{"model_module":"@jupyter-widgets/controls","model_name":"ProgressStyleModel","state":{"_view_name":"StyleView","_model_name":"ProgressStyleModel","description_width":"initial","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","bar_color":null,"_model_module":"@jupyter-widgets/controls"}},"3373940272584820a6fa53d80bcb438e":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"f44b9edb35fc4b68966dbfd5b837aabe":{"model_module":"@jupyter-widgets/controls","model_name":"DescriptionStyleModel","state":{"_view_name":"StyleView","_model_name":"DescriptionStyleModel","description_width":"","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","_model_module":"@jupyter-widgets/controls"}},"d6fa4a8231d045eca7a79c4c55ffa562":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"b5536e67d2a641debb7a27a6e3c1764a":{"model_module":"@jupyter-widgets/controls","model_name":"HBoxModel","state":{"_view_name":"HBoxView","_dom_classes":[],"_model_name":"HBoxModel","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.5.0","box_style":"","layout":"IPY_MODEL_826781541a1c4e748a6622077319997d","_model_module":"@jupyter-widgets/controls","children":["IPY_MODEL_8a9f1f9b6af6481da0711661a50d909a","IPY_MODEL_6dbdc11ebe0b486a97bd0d93274f9285"]}},"826781541a1c4e748a6622077319997d":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"8a9f1f9b6af6481da0711661a50d909a":{"model_module":"@jupyter-widgets/controls","model_name":"FloatProgressModel","state":{"_view_name":"ProgressView","style":"IPY_MODEL_36ddebfa70a645eea6b3dc3cb5f1856e","_dom_classes":[],"description":"100%","_model_name":"FloatProgressModel","bar_style":"success","max":14,"_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":14,"_view_count":null,"_view_module_version":"1.5.0","orientation":"horizontal","min":0,"description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_52cc71359d4646aa80ab561f5575dae7"}},"6dbdc11ebe0b486a97bd0d93274f9285":{"model_module":"@jupyter-widgets/controls","model_name":"HTMLModel","state":{"_view_name":"HTMLView","style":"IPY_MODEL_9f93b9e5b5dd438e8ff9111e5fb8d122","_dom_classes":[],"description":"","_model_name":"HTMLModel","placeholder":"​","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":" 14/14 [00:11&lt;00:00,  1.19 batch/s]","_view_count":null,"_view_module_version":"1.5.0","description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_3c1be3caa9264c75bc94cbccd04d313c"}},"36ddebfa70a645eea6b3dc3cb5f1856e":{"model_module":"@jupyter-widgets/controls","model_name":"ProgressStyleModel","state":{"_view_name":"StyleView","_model_name":"ProgressStyleModel","description_width":"initial","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","bar_color":null,"_model_module":"@jupyter-widgets/controls"}},"52cc71359d4646aa80ab561f5575dae7":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"9f93b9e5b5dd438e8ff9111e5fb8d122":{"model_module":"@jupyter-widgets/controls","model_name":"DescriptionStyleModel","state":{"_view_name":"StyleView","_model_name":"DescriptionStyleModel","description_width":"","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","_model_module":"@jupyter-widgets/controls"}},"3c1be3caa9264c75bc94cbccd04d313c":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"e0dc1c3d48334cbfa8d6730667496ee5":{"model_module":"@jupyter-widgets/controls","model_name":"HBoxModel","state":{"_view_name":"HBoxView","_dom_classes":[],"_model_name":"HBoxModel","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.5.0","box_style":"","layout":"IPY_MODEL_1fa4b1e2304547718fae119a8741ef48","_model_module":"@jupyter-widgets/controls","children":["IPY_MODEL_3711b6d1875041f8b1cad4332f23a898","IPY_MODEL_c69c8dff969f496d9fb48c41b6454f10"]}},"1fa4b1e2304547718fae119a8741ef48":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"3711b6d1875041f8b1cad4332f23a898":{"model_module":"@jupyter-widgets/controls","model_name":"FloatProgressModel","state":{"_view_name":"ProgressView","style":"IPY_MODEL_5ee79f0c4c534706a60541394aaa3669","_dom_classes":[],"description":"100%","_model_name":"FloatProgressModel","bar_style":"success","max":130767,"_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":130767,"_view_count":null,"_view_module_version":"1.5.0","orientation":"horizontal","min":0,"description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_009f57620ac342b99695ecaef8e98c3b"}},"c69c8dff969f496d9fb48c41b6454f10":{"model_module":"@jupyter-widgets/controls","model_name":"HTMLModel","state":{"_view_name":"HTMLView","style":"IPY_MODEL_6b89f86113e443b79bc5e4bcf9dd516b","_dom_classes":[],"description":"","_model_name":"HTMLModel","placeholder":"​","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":" 130767/130767 [03:27&lt;00:00, 630.05 ORF/s]","_view_count":null,"_view_module_version":"1.5.0","description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_345c8d6271b541a4a396796839a77b2b"}},"5ee79f0c4c534706a60541394aaa3669":{"model_module":"@jupyter-widgets/controls","model_name":"ProgressStyleModel","state":{"_view_name":"StyleView","_model_name":"ProgressStyleModel","description_width":"initial","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","bar_color":null,"_model_module":"@jupyter-widgets/controls"}},"009f57620ac342b99695ecaef8e98c3b":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"6b89f86113e443b79bc5e4bcf9dd516b":{"model_module":"@jupyter-widgets/controls","model_name":"DescriptionStyleModel","state":{"_view_name":"StyleView","_model_name":"DescriptionStyleModel","description_width":"","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","_model_module":"@jupyter-widgets/controls"}},"345c8d6271b541a4a396796839a77b2b":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"61764a2ad59c421c821478b2ac42f19b":{"model_module":"@jupyter-widgets/controls","model_name":"HBoxModel","state":{"_view_name":"HBoxView","_dom_classes":[],"_model_name":"HBoxModel","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.5.0","box_style":"","layout":"IPY_MODEL_f74da91a0eb7493d8d3f7d55e40b3197","_model_module":"@jupyter-widgets/controls","children":["IPY_MODEL_143a0c57fffa47dc8ceaed5663f900f3","IPY_MODEL_db3bee86c726446f843b21c83ea605c5"]}},"f74da91a0eb7493d8d3f7d55e40b3197":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"143a0c57fffa47dc8ceaed5663f900f3":{"model_module":"@jupyter-widgets/controls","model_name":"FloatProgressModel","state":{"_view_name":"ProgressView","style":"IPY_MODEL_9a33ea3293b245f7b92f02c7efbb1e0c","_dom_classes":[],"description":"100%","_model_name":"FloatProgressModel","bar_style":"success","max":83,"_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":83,"_view_count":null,"_view_module_version":"1.5.0","orientation":"horizontal","min":0,"description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_2546b341fb194ad59b9511b6703aa8bd"}},"db3bee86c726446f843b21c83ea605c5":{"model_module":"@jupyter-widgets/controls","model_name":"HTMLModel","state":{"_view_name":"HTMLView","style":"IPY_MODEL_846d2025b14c49168bdeb659a2e5deb0","_dom_classes":[],"description":"","_model_name":"HTMLModel","placeholder":"​","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":" 83/83 [00:42&lt;00:00,  1.94 batch/s]","_view_count":null,"_view_module_version":"1.5.0","description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_57211ef600dd4fe58bb88febd2bf4c81"}},"9a33ea3293b245f7b92f02c7efbb1e0c":{"model_module":"@jupyter-widgets/controls","model_name":"ProgressStyleModel","state":{"_view_name":"StyleView","_model_name":"ProgressStyleModel","description_width":"initial","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","bar_color":null,"_model_module":"@jupyter-widgets/controls"}},"2546b341fb194ad59b9511b6703aa8bd":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"846d2025b14c49168bdeb659a2e5deb0":{"model_module":"@jupyter-widgets/controls","model_name":"DescriptionStyleModel","state":{"_view_name":"StyleView","_model_name":"DescriptionStyleModel","description_width":"","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","_model_module":"@jupyter-widgets/controls"}},"57211ef600dd4fe58bb88febd2bf4c81":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"53e48adaac844b999781e04c08ecbda6":{"model_module":"@jupyter-widgets/controls","model_name":"HBoxModel","state":{"_view_name":"HBoxView","_dom_classes":[],"_model_name":"HBoxModel","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.5.0","box_style":"","layout":"IPY_MODEL_04028c28791948adab3e05eafc37e7a0","_model_module":"@jupyter-widgets/controls","children":["IPY_MODEL_ef40aad5fcca4a368f5691154bed6efe","IPY_MODEL_c506d78f8fa64f85ab0b8a8997a85eea"]}},"04028c28791948adab3e05eafc37e7a0":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"ef40aad5fcca4a368f5691154bed6efe":{"model_module":"@jupyter-widgets/controls","model_name":"FloatProgressModel","state":{"_view_name":"ProgressView","style":"IPY_MODEL_586cf9202a6b441689238627057f8111","_dom_classes":[],"description":"100%","_model_name":"FloatProgressModel","bar_style":"success","max":25,"_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":25,"_view_count":null,"_view_module_version":"1.5.0","orientation":"horizontal","min":0,"description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_e04d66c213204801a6410ed9f0735239"}},"c506d78f8fa64f85ab0b8a8997a85eea":{"model_module":"@jupyter-widgets/controls","model_name":"HTMLModel","state":{"_view_name":"HTMLView","style":"IPY_MODEL_720e61f2312e472c8a4e12dd9f92d673","_dom_classes":[],"description":"","_model_name":"HTMLModel","placeholder":"​","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":" 25/25 [00:34&lt;00:00,  1.39s/ batch]","_view_count":null,"_view_module_version":"1.5.0","description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_3c9beedee6a847a881d82739809a0114"}},"586cf9202a6b441689238627057f8111":{"model_module":"@jupyter-widgets/controls","model_name":"ProgressStyleModel","state":{"_view_name":"StyleView","_model_name":"ProgressStyleModel","description_width":"initial","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","bar_color":null,"_model_module":"@jupyter-widgets/controls"}},"e04d66c213204801a6410ed9f0735239":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"720e61f2312e472c8a4e12dd9f92d673":{"model_module":"@jupyter-widgets/controls","model_name":"DescriptionStyleModel","state":{"_view_name":"StyleView","_model_name":"DescriptionStyleModel","description_width":"","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","_model_module":"@jupyter-widgets/controls"}},"3c9beedee6a847a881d82739809a0114":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"ae9787fe5de34d368d18fd66b8140f23":{"model_module":"@jupyter-widgets/controls","model_name":"HBoxModel","state":{"_view_name":"HBoxView","_dom_classes":[],"_model_name":"HBoxModel","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.5.0","box_style":"","layout":"IPY_MODEL_e73c2f0f43254958948701e1e5fe7b99","_model_module":"@jupyter-widgets/controls","children":["IPY_MODEL_0859fa2ab7764509a3d9ece60ec379bf","IPY_MODEL_6c6dfd704c50401f81fd6bab3fd8e7a8"]}},"e73c2f0f43254958948701e1e5fe7b99":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"0859fa2ab7764509a3d9ece60ec379bf":{"model_module":"@jupyter-widgets/controls","model_name":"FloatProgressModel","state":{"_view_name":"ProgressView","style":"IPY_MODEL_ddbc4e0b90d446878a00296527552d9e","_dom_classes":[],"description":"100%","_model_name":"FloatProgressModel","bar_style":"success","max":248007,"_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":248007,"_view_count":null,"_view_module_version":"1.5.0","orientation":"horizontal","min":0,"description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_d7ad6ff53d794f97ba18155038042615"}},"6c6dfd704c50401f81fd6bab3fd8e7a8":{"model_module":"@jupyter-widgets/controls","model_name":"HTMLModel","state":{"_view_name":"HTMLView","style":"IPY_MODEL_0c2c8d73c95649eca9bc5afd97ae9ddc","_dom_classes":[],"description":"","_model_name":"HTMLModel","placeholder":"​","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":" 248007/248007 [05:32&lt;00:00, 746.78 ORF/s]","_view_count":null,"_view_module_version":"1.5.0","description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_60fc5356b70f4f4ea0c736bc47ba5e82"}},"ddbc4e0b90d446878a00296527552d9e":{"model_module":"@jupyter-widgets/controls","model_name":"ProgressStyleModel","state":{"_view_name":"StyleView","_model_name":"ProgressStyleModel","description_width":"initial","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","bar_color":null,"_model_module":"@jupyter-widgets/controls"}},"d7ad6ff53d794f97ba18155038042615":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"0c2c8d73c95649eca9bc5afd97ae9ddc":{"model_module":"@jupyter-widgets/controls","model_name":"DescriptionStyleModel","state":{"_view_name":"StyleView","_model_name":"DescriptionStyleModel","description_width":"","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","_model_module":"@jupyter-widgets/controls"}},"60fc5356b70f4f4ea0c736bc47ba5e82":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"e00e98f3550144f3aad0b161323907a0":{"model_module":"@jupyter-widgets/controls","model_name":"HBoxModel","state":{"_view_name":"HBoxView","_dom_classes":[],"_model_name":"HBoxModel","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.5.0","box_style":"","layout":"IPY_MODEL_aead515250204e488a73d572829be9c9","_model_module":"@jupyter-widgets/controls","children":["IPY_MODEL_efb7aad94c8c4e72994e64dea67b1271","IPY_MODEL_be5b416e49474b588fb0000989c5b6b1"]}},"aead515250204e488a73d572829be9c9":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"efb7aad94c8c4e72994e64dea67b1271":{"model_module":"@jupyter-widgets/controls","model_name":"FloatProgressModel","state":{"_view_name":"ProgressView","style":"IPY_MODEL_44f811a17e554f90804ac204c883ffd0","_dom_classes":[],"description":"100%","_model_name":"FloatProgressModel","bar_style":"success","max":70,"_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":70,"_view_count":null,"_view_module_version":"1.5.0","orientation":"horizontal","min":0,"description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_9bbfed2ab6044d8ba87176f29dd8a479"}},"be5b416e49474b588fb0000989c5b6b1":{"model_module":"@jupyter-widgets/controls","model_name":"HTMLModel","state":{"_view_name":"HTMLView","style":"IPY_MODEL_4bb795f3f2824e2e9929f9815787582b","_dom_classes":[],"description":"","_model_name":"HTMLModel","placeholder":"​","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":" 70/70 [00:25&lt;00:00,  2.74 batch/s]","_view_count":null,"_view_module_version":"1.5.0","description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_4117d870d64747eb97bf52456e6eea0b"}},"44f811a17e554f90804ac204c883ffd0":{"model_module":"@jupyter-widgets/controls","model_name":"ProgressStyleModel","state":{"_view_name":"StyleView","_model_name":"ProgressStyleModel","description_width":"initial","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","bar_color":null,"_model_module":"@jupyter-widgets/controls"}},"9bbfed2ab6044d8ba87176f29dd8a479":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"4bb795f3f2824e2e9929f9815787582b":{"model_module":"@jupyter-widgets/controls","model_name":"DescriptionStyleModel","state":{"_view_name":"StyleView","_model_name":"DescriptionStyleModel","description_width":"","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","_model_module":"@jupyter-widgets/controls"}},"4117d870d64747eb97bf52456e6eea0b":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"0991db3a6cf442acb396adda7cf1d9e7":{"model_module":"@jupyter-widgets/controls","model_name":"HBoxModel","state":{"_view_name":"HBoxView","_dom_classes":[],"_model_name":"HBoxModel","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.5.0","box_style":"","layout":"IPY_MODEL_57ea2a54938842f28e3967ab19338165","_model_module":"@jupyter-widgets/controls","children":["IPY_MODEL_c436830fcffb468192cbd05f3ff6f521","IPY_MODEL_f1457eb6091b4a0bad1a2bdd6c2eb021"]}},"57ea2a54938842f28e3967ab19338165":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"c436830fcffb468192cbd05f3ff6f521":{"model_module":"@jupyter-widgets/controls","model_name":"FloatProgressModel","state":{"_view_name":"ProgressView","style":"IPY_MODEL_6b5964e1fc6243d0a503405bd5fb3e39","_dom_classes":[],"description":"100%","_model_name":"FloatProgressModel","bar_style":"success","max":21,"_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":21,"_view_count":null,"_view_module_version":"1.5.0","orientation":"horizontal","min":0,"description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_90496715e51d4bfab1137065681c9921"}},"f1457eb6091b4a0bad1a2bdd6c2eb021":{"model_module":"@jupyter-widgets/controls","model_name":"HTMLModel","state":{"_view_name":"HTMLView","style":"IPY_MODEL_be6ce7baf4f8499c891790a45bd2918d","_dom_classes":[],"description":"","_model_name":"HTMLModel","placeholder":"​","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":" 21/21 [00:46&lt;00:00,  2.24s/ batch]","_view_count":null,"_view_module_version":"1.5.0","description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_f8f4d7f2ce724bcb8c1138553fd4a5d6"}},"6b5964e1fc6243d0a503405bd5fb3e39":{"model_module":"@jupyter-widgets/controls","model_name":"ProgressStyleModel","state":{"_view_name":"StyleView","_model_name":"ProgressStyleModel","description_width":"initial","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","bar_color":null,"_model_module":"@jupyter-widgets/controls"}},"90496715e51d4bfab1137065681c9921":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"be6ce7baf4f8499c891790a45bd2918d":{"model_module":"@jupyter-widgets/controls","model_name":"DescriptionStyleModel","state":{"_view_name":"StyleView","_model_name":"DescriptionStyleModel","description_width":"","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","_model_module":"@jupyter-widgets/controls"}},"f8f4d7f2ce724bcb8c1138553fd4a5d6":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"0eda738208ca4abfad9bdf7c7242e420":{"model_module":"@jupyter-widgets/controls","model_name":"HBoxModel","state":{"_view_name":"HBoxView","_dom_classes":[],"_model_name":"HBoxModel","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.5.0","box_style":"","layout":"IPY_MODEL_faaf732ea014444480ac0e63b331ea97","_model_module":"@jupyter-widgets/controls","children":["IPY_MODEL_3701d94822064f259058c32875c12816","IPY_MODEL_798749bc7ce44a8ebcd7dbaaeba91301"]}},"faaf732ea014444480ac0e63b331ea97":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"3701d94822064f259058c32875c12816":{"model_module":"@jupyter-widgets/controls","model_name":"FloatProgressModel","state":{"_view_name":"ProgressView","style":"IPY_MODEL_38197428cace427eb877d9df2dc8ba61","_dom_classes":[],"description":"100%","_model_name":"FloatProgressModel","bar_style":"success","max":209623,"_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":209623,"_view_count":null,"_view_module_version":"1.5.0","orientation":"horizontal","min":0,"description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_9ae8f26e24ff44e5bef5facc2665c41c"}},"798749bc7ce44a8ebcd7dbaaeba91301":{"model_module":"@jupyter-widgets/controls","model_name":"HTMLModel","state":{"_view_name":"HTMLView","style":"IPY_MODEL_57722d71cc7d4d7f98a56df39e445cd6","_dom_classes":[],"description":"","_model_name":"HTMLModel","placeholder":"​","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":" 209623/209623 [05:22&lt;00:00, 649.21 ORF/s]","_view_count":null,"_view_module_version":"1.5.0","description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_518576a310a044b8bcdf66902eca9d02"}},"38197428cace427eb877d9df2dc8ba61":{"model_module":"@jupyter-widgets/controls","model_name":"ProgressStyleModel","state":{"_view_name":"StyleView","_model_name":"ProgressStyleModel","description_width":"initial","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","bar_color":null,"_model_module":"@jupyter-widgets/controls"}},"9ae8f26e24ff44e5bef5facc2665c41c":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"57722d71cc7d4d7f98a56df39e445cd6":{"model_module":"@jupyter-widgets/controls","model_name":"DescriptionStyleModel","state":{"_view_name":"StyleView","_model_name":"DescriptionStyleModel","description_width":"","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","_model_module":"@jupyter-widgets/controls"}},"518576a310a044b8bcdf66902eca9d02":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"bb83e6df8656425fb855240b40d559d9":{"model_module":"@jupyter-widgets/controls","model_name":"HBoxModel","state":{"_view_name":"HBoxView","_dom_classes":[],"_model_name":"HBoxModel","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.5.0","box_style":"","layout":"IPY_MODEL_7c3da321f94045e38be71b4053f3a8bc","_model_module":"@jupyter-widgets/controls","children":["IPY_MODEL_17d7d204fcfc45aca93c8c911b73d505","IPY_MODEL_3fb6b79fd3ad4b57bc691bafc008660f"]}},"7c3da321f94045e38be71b4053f3a8bc":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"17d7d204fcfc45aca93c8c911b73d505":{"model_module":"@jupyter-widgets/controls","model_name":"FloatProgressModel","state":{"_view_name":"ProgressView","style":"IPY_MODEL_3b4201977cf9450b8a0e9fd71df61067","_dom_classes":[],"description":"100%","_model_name":"FloatProgressModel","bar_style":"success","max":43,"_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":43,"_view_count":null,"_view_module_version":"1.5.0","orientation":"horizontal","min":0,"description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_71f7100aab72407487996a211ca1e832"}},"3fb6b79fd3ad4b57bc691bafc008660f":{"model_module":"@jupyter-widgets/controls","model_name":"HTMLModel","state":{"_view_name":"HTMLView","style":"IPY_MODEL_971259dd396140b6b20974bae82b3280","_dom_classes":[],"description":"","_model_name":"HTMLModel","placeholder":"​","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":" 43/43 [00:16&lt;00:00,  2.57 batch/s]","_view_count":null,"_view_module_version":"1.5.0","description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_1621fa3675fb410a9ba8a0a64da88cbb"}},"3b4201977cf9450b8a0e9fd71df61067":{"model_module":"@jupyter-widgets/controls","model_name":"ProgressStyleModel","state":{"_view_name":"StyleView","_model_name":"ProgressStyleModel","description_width":"initial","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","bar_color":null,"_model_module":"@jupyter-widgets/controls"}},"71f7100aab72407487996a211ca1e832":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"971259dd396140b6b20974bae82b3280":{"model_module":"@jupyter-widgets/controls","model_name":"DescriptionStyleModel","state":{"_view_name":"StyleView","_model_name":"DescriptionStyleModel","description_width":"","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","_model_module":"@jupyter-widgets/controls"}},"1621fa3675fb410a9ba8a0a64da88cbb":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"49abb9be43d348dfa5df3012b39a6853":{"model_module":"@jupyter-widgets/controls","model_name":"HBoxModel","state":{"_view_name":"HBoxView","_dom_classes":[],"_model_name":"HBoxModel","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.5.0","box_style":"","layout":"IPY_MODEL_a4762e7caa214b9f90f9bce9a00979bc","_model_module":"@jupyter-widgets/controls","children":["IPY_MODEL_32935676e90647da9115a2d7c45e3768","IPY_MODEL_ab0f7f85a2ee435caf9b649e4e89961c"]}},"a4762e7caa214b9f90f9bce9a00979bc":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"32935676e90647da9115a2d7c45e3768":{"model_module":"@jupyter-widgets/controls","model_name":"FloatProgressModel","state":{"_view_name":"ProgressView","style":"IPY_MODEL_1ea88fbb7ce14e339dac6c3f122ba3c7","_dom_classes":[],"description":"100%","_model_name":"FloatProgressModel","bar_style":"success","max":13,"_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":13,"_view_count":null,"_view_module_version":"1.5.0","orientation":"horizontal","min":0,"description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_3adbfc7d554b4c1e87c48bb46132f0d2"}},"ab0f7f85a2ee435caf9b649e4e89961c":{"model_module":"@jupyter-widgets/controls","model_name":"HTMLModel","state":{"_view_name":"HTMLView","style":"IPY_MODEL_1bcb40e361c547ea86ada2d07f66ea4e","_dom_classes":[],"description":"","_model_name":"HTMLModel","placeholder":"​","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":" 13/13 [00:16&lt;00:00,  1.27s/ batch]","_view_count":null,"_view_module_version":"1.5.0","description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_30bb982cd72443bda3938937b365f2af"}},"1ea88fbb7ce14e339dac6c3f122ba3c7":{"model_module":"@jupyter-widgets/controls","model_name":"ProgressStyleModel","state":{"_view_name":"StyleView","_model_name":"ProgressStyleModel","description_width":"initial","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","bar_color":null,"_model_module":"@jupyter-widgets/controls"}},"3adbfc7d554b4c1e87c48bb46132f0d2":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"1bcb40e361c547ea86ada2d07f66ea4e":{"model_module":"@jupyter-widgets/controls","model_name":"DescriptionStyleModel","state":{"_view_name":"StyleView","_model_name":"DescriptionStyleModel","description_width":"","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","_model_module":"@jupyter-widgets/controls"}},"30bb982cd72443bda3938937b365f2af":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"b602f01384d74070bcc76e312e63d85a":{"model_module":"@jupyter-widgets/controls","model_name":"HBoxModel","state":{"_view_name":"HBoxView","_dom_classes":[],"_model_name":"HBoxModel","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.5.0","box_style":"","layout":"IPY_MODEL_20d6f036dcba4c6b85a91e9c73a40cca","_model_module":"@jupyter-widgets/controls","children":["IPY_MODEL_ce96b3d84c504515ab0cd66ae4eeeded","IPY_MODEL_6cb36105169748c39492c3569b419b58"]}},"20d6f036dcba4c6b85a91e9c73a40cca":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"ce96b3d84c504515ab0cd66ae4eeeded":{"model_module":"@jupyter-widgets/controls","model_name":"FloatProgressModel","state":{"_view_name":"ProgressView","style":"IPY_MODEL_77cbda44d5f14b8db3c2984c54d58c21","_dom_classes":[],"description":"100%","_model_name":"FloatProgressModel","bar_style":"success","max":126841,"_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":126841,"_view_count":null,"_view_module_version":"1.5.0","orientation":"horizontal","min":0,"description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_db412f14b0a54e0989719091b78423b1"}},"6cb36105169748c39492c3569b419b58":{"model_module":"@jupyter-widgets/controls","model_name":"HTMLModel","state":{"_view_name":"HTMLView","style":"IPY_MODEL_934c551acca5419d9d7a294b20d127e4","_dom_classes":[],"description":"","_model_name":"HTMLModel","placeholder":"​","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":" 126841/126841 [04:13&lt;00:00, 500.81 ORF/s]","_view_count":null,"_view_module_version":"1.5.0","description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_6f06b70d679546c98ec007d6eaef1d08"}},"77cbda44d5f14b8db3c2984c54d58c21":{"model_module":"@jupyter-widgets/controls","model_name":"ProgressStyleModel","state":{"_view_name":"StyleView","_model_name":"ProgressStyleModel","description_width":"initial","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","bar_color":null,"_model_module":"@jupyter-widgets/controls"}},"db412f14b0a54e0989719091b78423b1":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"934c551acca5419d9d7a294b20d127e4":{"model_module":"@jupyter-widgets/controls","model_name":"DescriptionStyleModel","state":{"_view_name":"StyleView","_model_name":"DescriptionStyleModel","description_width":"","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","_model_module":"@jupyter-widgets/controls"}},"6f06b70d679546c98ec007d6eaef1d08":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"6827cf9798a849939a10a17e4306c4da":{"model_module":"@jupyter-widgets/controls","model_name":"HBoxModel","state":{"_view_name":"HBoxView","_dom_classes":[],"_model_name":"HBoxModel","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.5.0","box_style":"","layout":"IPY_MODEL_196f216e86264341a8d37028edcffedc","_model_module":"@jupyter-widgets/controls","children":["IPY_MODEL_c97d25a21aca45409d9133b4a10e1bfb","IPY_MODEL_ba378aa1216147d494df154efa51f62c"]}},"196f216e86264341a8d37028edcffedc":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"c97d25a21aca45409d9133b4a10e1bfb":{"model_module":"@jupyter-widgets/controls","model_name":"FloatProgressModel","state":{"_view_name":"ProgressView","style":"IPY_MODEL_25546c74d08d4c4591bf310057d34942","_dom_classes":[],"description":"100%","_model_name":"FloatProgressModel","bar_style":"success","max":28,"_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":28,"_view_count":null,"_view_module_version":"1.5.0","orientation":"horizontal","min":0,"description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_0054d1d788bf4c8ba4b2472992c89e06"}},"ba378aa1216147d494df154efa51f62c":{"model_module":"@jupyter-widgets/controls","model_name":"HTMLModel","state":{"_view_name":"HTMLView","style":"IPY_MODEL_810c5792fea545a2a40b8cc51a6d50ac","_dom_classes":[],"description":"","_model_name":"HTMLModel","placeholder":"​","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":" 28/28 [00:10&lt;00:00,  2.71 batch/s]","_view_count":null,"_view_module_version":"1.5.0","description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_d9aeead6a612458b9b2bb4b0b0b74459"}},"25546c74d08d4c4591bf310057d34942":{"model_module":"@jupyter-widgets/controls","model_name":"ProgressStyleModel","state":{"_view_name":"StyleView","_model_name":"ProgressStyleModel","description_width":"initial","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","bar_color":null,"_model_module":"@jupyter-widgets/controls"}},"0054d1d788bf4c8ba4b2472992c89e06":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"810c5792fea545a2a40b8cc51a6d50ac":{"model_module":"@jupyter-widgets/controls","model_name":"DescriptionStyleModel","state":{"_view_name":"StyleView","_model_name":"DescriptionStyleModel","description_width":"","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","_model_module":"@jupyter-widgets/controls"}},"d9aeead6a612458b9b2bb4b0b0b74459":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"305a0d07e2f841759d1a07379bc042f1":{"model_module":"@jupyter-widgets/controls","model_name":"HBoxModel","state":{"_view_name":"HBoxView","_dom_classes":[],"_model_name":"HBoxModel","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.5.0","box_style":"","layout":"IPY_MODEL_b79cc6c5b54a429caaecce637f16ea47","_model_module":"@jupyter-widgets/controls","children":["IPY_MODEL_feb4c24378264ff5a56329d4b45d3a11","IPY_MODEL_07e26f7f828746ec977aa18ce1c5b8c7"]}},"b79cc6c5b54a429caaecce637f16ea47":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"feb4c24378264ff5a56329d4b45d3a11":{"model_module":"@jupyter-widgets/controls","model_name":"FloatProgressModel","state":{"_view_name":"ProgressView","style":"IPY_MODEL_2a2a7b0b3d914ff6a488ad1070c7dfdd","_dom_classes":[],"description":"100%","_model_name":"FloatProgressModel","bar_style":"success","max":9,"_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":9,"_view_count":null,"_view_module_version":"1.5.0","orientation":"horizontal","min":0,"description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_d65a74383a3b4cf8b44ae1ae22e660b4"}},"07e26f7f828746ec977aa18ce1c5b8c7":{"model_module":"@jupyter-widgets/controls","model_name":"HTMLModel","state":{"_view_name":"HTMLView","style":"IPY_MODEL_dcd15c93ed2748ffaf0aa17526cfd714","_dom_classes":[],"description":"","_model_name":"HTMLModel","placeholder":"​","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":" 9/9 [02:54&lt;00:00, 19.44s/ batch]","_view_count":null,"_view_module_version":"1.5.0","description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_2424ac20648343a1b2603b716ba4fd87"}},"2a2a7b0b3d914ff6a488ad1070c7dfdd":{"model_module":"@jupyter-widgets/controls","model_name":"ProgressStyleModel","state":{"_view_name":"StyleView","_model_name":"ProgressStyleModel","description_width":"initial","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","bar_color":null,"_model_module":"@jupyter-widgets/controls"}},"d65a74383a3b4cf8b44ae1ae22e660b4":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"dcd15c93ed2748ffaf0aa17526cfd714":{"model_module":"@jupyter-widgets/controls","model_name":"DescriptionStyleModel","state":{"_view_name":"StyleView","_model_name":"DescriptionStyleModel","description_width":"","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","_model_module":"@jupyter-widgets/controls"}},"2424ac20648343a1b2603b716ba4fd87":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"c4b1be860dec4cad817d25d78dd3a568":{"model_module":"@jupyter-widgets/controls","model_name":"HBoxModel","state":{"_view_name":"HBoxView","_dom_classes":[],"_model_name":"HBoxModel","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.5.0","box_style":"","layout":"IPY_MODEL_312ca1028195486dbdbda00b0458b56a","_model_module":"@jupyter-widgets/controls","children":["IPY_MODEL_7861960dc4eb4a879db6165f430a3485","IPY_MODEL_59e9ab03f2db4616acf544062462bed2"]}},"312ca1028195486dbdbda00b0458b56a":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"7861960dc4eb4a879db6165f430a3485":{"model_module":"@jupyter-widgets/controls","model_name":"FloatProgressModel","state":{"_view_name":"ProgressView","style":"IPY_MODEL_f53c53e5c7714b27b8bfe928f6b0d081","_dom_classes":[],"description":"100%","_model_name":"FloatProgressModel","bar_style":"success","max":82782,"_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":82782,"_view_count":null,"_view_module_version":"1.5.0","orientation":"horizontal","min":0,"description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_7dc674bbe7a04fa59e4d9e9b268d32e6"}},"59e9ab03f2db4616acf544062462bed2":{"model_module":"@jupyter-widgets/controls","model_name":"HTMLModel","state":{"_view_name":"HTMLView","style":"IPY_MODEL_ffe86fb3119f446b908eae4fd4e218a7","_dom_classes":[],"description":"","_model_name":"HTMLModel","placeholder":"​","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":" 82782/82782 [02:52&lt;00:00, 481.06 ORF/s]","_view_count":null,"_view_module_version":"1.5.0","description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_99300600de8847ec93939b8694fd3975"}},"f53c53e5c7714b27b8bfe928f6b0d081":{"model_module":"@jupyter-widgets/controls","model_name":"ProgressStyleModel","state":{"_view_name":"StyleView","_model_name":"ProgressStyleModel","description_width":"initial","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","bar_color":null,"_model_module":"@jupyter-widgets/controls"}},"7dc674bbe7a04fa59e4d9e9b268d32e6":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"ffe86fb3119f446b908eae4fd4e218a7":{"model_module":"@jupyter-widgets/controls","model_name":"DescriptionStyleModel","state":{"_view_name":"StyleView","_model_name":"DescriptionStyleModel","description_width":"","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","_model_module":"@jupyter-widgets/controls"}},"99300600de8847ec93939b8694fd3975":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"b170446e153a470a8c0a960dda54c53c":{"model_module":"@jupyter-widgets/controls","model_name":"HBoxModel","state":{"_view_name":"HBoxView","_dom_classes":[],"_model_name":"HBoxModel","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.5.0","box_style":"","layout":"IPY_MODEL_2b7eb64842e14001a070c580fb7e9458","_model_module":"@jupyter-widgets/controls","children":["IPY_MODEL_2d518733b3d04cc4801eb6afdfb5d8e3","IPY_MODEL_9791287b659842e1bc884b9bce35c991"]}},"2b7eb64842e14001a070c580fb7e9458":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"2d518733b3d04cc4801eb6afdfb5d8e3":{"model_module":"@jupyter-widgets/controls","model_name":"FloatProgressModel","state":{"_view_name":"ProgressView","style":"IPY_MODEL_04491a5e7bfc4cedacbb65aad16c63ad","_dom_classes":[],"description":"100%","_model_name":"FloatProgressModel","bar_style":"success","max":68,"_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":68,"_view_count":null,"_view_module_version":"1.5.0","orientation":"horizontal","min":0,"description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_21ba9c9ab96847709cbbab9a54b79bf8"}},"9791287b659842e1bc884b9bce35c991":{"model_module":"@jupyter-widgets/controls","model_name":"HTMLModel","state":{"_view_name":"HTMLView","style":"IPY_MODEL_8a0c306ab9a64524a0464af1a3b9ea2a","_dom_classes":[],"description":"","_model_name":"HTMLModel","placeholder":"​","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":" 68/68 [00:29&lt;00:00,  2.31 batch/s]","_view_count":null,"_view_module_version":"1.5.0","description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_03d8ee229ea146099e70a2eed310e181"}},"04491a5e7bfc4cedacbb65aad16c63ad":{"model_module":"@jupyter-widgets/controls","model_name":"ProgressStyleModel","state":{"_view_name":"StyleView","_model_name":"ProgressStyleModel","description_width":"initial","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","bar_color":null,"_model_module":"@jupyter-widgets/controls"}},"21ba9c9ab96847709cbbab9a54b79bf8":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"8a0c306ab9a64524a0464af1a3b9ea2a":{"model_module":"@jupyter-widgets/controls","model_name":"DescriptionStyleModel","state":{"_view_name":"StyleView","_model_name":"DescriptionStyleModel","description_width":"","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","_model_module":"@jupyter-widgets/controls"}},"03d8ee229ea146099e70a2eed310e181":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"16e15acd67614143aea32b54880875e6":{"model_module":"@jupyter-widgets/controls","model_name":"HBoxModel","state":{"_view_name":"HBoxView","_dom_classes":[],"_model_name":"HBoxModel","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.5.0","box_style":"","layout":"IPY_MODEL_e001aee9a6b944b9896142536b2c9045","_model_module":"@jupyter-widgets/controls","children":["IPY_MODEL_d0ea5af060db4251b587a0b80539283a","IPY_MODEL_8d43d817e9ef40e69929dfef73c2a679"]}},"e001aee9a6b944b9896142536b2c9045":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"d0ea5af060db4251b587a0b80539283a":{"model_module":"@jupyter-widgets/controls","model_name":"FloatProgressModel","state":{"_view_name":"ProgressView","style":"IPY_MODEL_207b4f24800344e88f7ec941f9ced720","_dom_classes":[],"description":"100%","_model_name":"FloatProgressModel","bar_style":"success","max":21,"_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":21,"_view_count":null,"_view_module_version":"1.5.0","orientation":"horizontal","min":0,"description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_e6918311940f4da4be42ff5d1cdb5adb"}},"8d43d817e9ef40e69929dfef73c2a679":{"model_module":"@jupyter-widgets/controls","model_name":"HTMLModel","state":{"_view_name":"HTMLView","style":"IPY_MODEL_24a97b8f15fa4d15a1908f7d0a491a2d","_dom_classes":[],"description":"","_model_name":"HTMLModel","placeholder":"​","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":" 21/21 [00:33&lt;00:00,  1.61s/ batch]","_view_count":null,"_view_module_version":"1.5.0","description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_f87952a5ad654568872da47aa3d4afee"}},"207b4f24800344e88f7ec941f9ced720":{"model_module":"@jupyter-widgets/controls","model_name":"ProgressStyleModel","state":{"_view_name":"StyleView","_model_name":"ProgressStyleModel","description_width":"initial","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","bar_color":null,"_model_module":"@jupyter-widgets/controls"}},"e6918311940f4da4be42ff5d1cdb5adb":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"24a97b8f15fa4d15a1908f7d0a491a2d":{"model_module":"@jupyter-widgets/controls","model_name":"DescriptionStyleModel","state":{"_view_name":"StyleView","_model_name":"DescriptionStyleModel","description_width":"","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","_model_module":"@jupyter-widgets/controls"}},"f87952a5ad654568872da47aa3d4afee":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"5d55188da98243b88ea68e297494abe9":{"model_module":"@jupyter-widgets/controls","model_name":"HBoxModel","state":{"_view_name":"HBoxView","_dom_classes":[],"_model_name":"HBoxModel","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.5.0","box_style":"","layout":"IPY_MODEL_a109cc3660ea49d38cb25b7bc861ee7c","_model_module":"@jupyter-widgets/controls","children":["IPY_MODEL_81d1d4abe6a54cdf8582f4e9c6b4827a","IPY_MODEL_33fab451902b40e7a873946ef717d3ae"]}},"a109cc3660ea49d38cb25b7bc861ee7c":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"81d1d4abe6a54cdf8582f4e9c6b4827a":{"model_module":"@jupyter-widgets/controls","model_name":"FloatProgressModel","state":{"_view_name":"ProgressView","style":"IPY_MODEL_60831f6755664505b45bdd10e25ea0bf","_dom_classes":[],"description":"100%","_model_name":"FloatProgressModel","bar_style":"success","max":201861,"_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":201861,"_view_count":null,"_view_module_version":"1.5.0","orientation":"horizontal","min":0,"description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_28aae1edabf241989237e5a446210f42"}},"33fab451902b40e7a873946ef717d3ae":{"model_module":"@jupyter-widgets/controls","model_name":"HTMLModel","state":{"_view_name":"HTMLView","style":"IPY_MODEL_b9cc2a33e1384ab8905d645886cca071","_dom_classes":[],"description":"","_model_name":"HTMLModel","placeholder":"​","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":" 201861/201861 [05:16&lt;00:00, 637.07 ORF/s]","_view_count":null,"_view_module_version":"1.5.0","description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_8e04b6c2928841d7927f28f2b023234e"}},"60831f6755664505b45bdd10e25ea0bf":{"model_module":"@jupyter-widgets/controls","model_name":"ProgressStyleModel","state":{"_view_name":"StyleView","_model_name":"ProgressStyleModel","description_width":"initial","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","bar_color":null,"_model_module":"@jupyter-widgets/controls"}},"28aae1edabf241989237e5a446210f42":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"b9cc2a33e1384ab8905d645886cca071":{"model_module":"@jupyter-widgets/controls","model_name":"DescriptionStyleModel","state":{"_view_name":"StyleView","_model_name":"DescriptionStyleModel","description_width":"","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","_model_module":"@jupyter-widgets/controls"}},"8e04b6c2928841d7927f28f2b023234e":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"6ea4c17aa5464fd89f91f787d0a6ad32":{"model_module":"@jupyter-widgets/controls","model_name":"HBoxModel","state":{"_view_name":"HBoxView","_dom_classes":[],"_model_name":"HBoxModel","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.5.0","box_style":"","layout":"IPY_MODEL_19a49f4420944212824fd1a7541ecb15","_model_module":"@jupyter-widgets/controls","children":["IPY_MODEL_d91c36dbcfb14d19ba85928c926e7549","IPY_MODEL_3b841a0e673c4e36b477b948af9e5805"]}},"19a49f4420944212824fd1a7541ecb15":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"d91c36dbcfb14d19ba85928c926e7549":{"model_module":"@jupyter-widgets/controls","model_name":"FloatProgressModel","state":{"_view_name":"ProgressView","style":"IPY_MODEL_bf145ca2eb264395a3fc1fd61ff7b227","_dom_classes":[],"description":"100%","_model_name":"FloatProgressModel","bar_style":"success","max":32,"_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":32,"_view_count":null,"_view_module_version":"1.5.0","orientation":"horizontal","min":0,"description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_77ccc65f928f4cef91d2ab1c6fbd6d15"}},"3b841a0e673c4e36b477b948af9e5805":{"model_module":"@jupyter-widgets/controls","model_name":"HTMLModel","state":{"_view_name":"HTMLView","style":"IPY_MODEL_e4135426d6384b4899006261dd491e64","_dom_classes":[],"description":"","_model_name":"HTMLModel","placeholder":"​","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":" 32/32 [00:23&lt;00:00,  1.35 batch/s]","_view_count":null,"_view_module_version":"1.5.0","description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_7a6bdaad713d485cbce87f7756a42e54"}},"bf145ca2eb264395a3fc1fd61ff7b227":{"model_module":"@jupyter-widgets/controls","model_name":"ProgressStyleModel","state":{"_view_name":"StyleView","_model_name":"ProgressStyleModel","description_width":"initial","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","bar_color":null,"_model_module":"@jupyter-widgets/controls"}},"77ccc65f928f4cef91d2ab1c6fbd6d15":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"e4135426d6384b4899006261dd491e64":{"model_module":"@jupyter-widgets/controls","model_name":"DescriptionStyleModel","state":{"_view_name":"StyleView","_model_name":"DescriptionStyleModel","description_width":"","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","_model_module":"@jupyter-widgets/controls"}},"7a6bdaad713d485cbce87f7756a42e54":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"e9cd6a10e5f54cbf97a4580e36e04237":{"model_module":"@jupyter-widgets/controls","model_name":"HBoxModel","state":{"_view_name":"HBoxView","_dom_classes":[],"_model_name":"HBoxModel","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.5.0","box_style":"","layout":"IPY_MODEL_2f0bfa9660fc4b6ea6199e2c7603bdb1","_model_module":"@jupyter-widgets/controls","children":["IPY_MODEL_173caee44a874609a057f9d92d495ca8","IPY_MODEL_a8d999d17d2448ac89fbb1c46d7e95dc"]}},"2f0bfa9660fc4b6ea6199e2c7603bdb1":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"173caee44a874609a057f9d92d495ca8":{"model_module":"@jupyter-widgets/controls","model_name":"FloatProgressModel","state":{"_view_name":"ProgressView","style":"IPY_MODEL_bb5f69c24ed64f70a63f4b1cfa78f8fb","_dom_classes":[],"description":"100%","_model_name":"FloatProgressModel","bar_style":"success","max":10,"_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":10,"_view_count":null,"_view_module_version":"1.5.0","orientation":"horizontal","min":0,"description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_82780bfee37f42be8089b00f95825768"}},"a8d999d17d2448ac89fbb1c46d7e95dc":{"model_module":"@jupyter-widgets/controls","model_name":"HTMLModel","state":{"_view_name":"HTMLView","style":"IPY_MODEL_9dad734bb0b04cb2a6c53b64880963d4","_dom_classes":[],"description":"","_model_name":"HTMLModel","placeholder":"​","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":" 10/10 [00:08&lt;00:00,  1.12 batch/s]","_view_count":null,"_view_module_version":"1.5.0","description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_3b44a02b78434826a1bfc8671a96eed7"}},"bb5f69c24ed64f70a63f4b1cfa78f8fb":{"model_module":"@jupyter-widgets/controls","model_name":"ProgressStyleModel","state":{"_view_name":"StyleView","_model_name":"ProgressStyleModel","description_width":"initial","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","bar_color":null,"_model_module":"@jupyter-widgets/controls"}},"82780bfee37f42be8089b00f95825768":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"9dad734bb0b04cb2a6c53b64880963d4":{"model_module":"@jupyter-widgets/controls","model_name":"DescriptionStyleModel","state":{"_view_name":"StyleView","_model_name":"DescriptionStyleModel","description_width":"","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","_model_module":"@jupyter-widgets/controls"}},"3b44a02b78434826a1bfc8671a96eed7":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"ba22ecbbf35d4d1fb453c08123191325":{"model_module":"@jupyter-widgets/controls","model_name":"HBoxModel","state":{"_view_name":"HBoxView","_dom_classes":[],"_model_name":"HBoxModel","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.5.0","box_style":"","layout":"IPY_MODEL_0ff140ade11c4ccf98f7d169e97152df","_model_module":"@jupyter-widgets/controls","children":["IPY_MODEL_7462302f6fca40df975d4041d78a4897","IPY_MODEL_9b2ac9f12d2648d0bc84ff8e6002ef60"]}},"0ff140ade11c4ccf98f7d169e97152df":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"7462302f6fca40df975d4041d78a4897":{"model_module":"@jupyter-widgets/controls","model_name":"FloatProgressModel","state":{"_view_name":"ProgressView","style":"IPY_MODEL_e2585d01d62343dd8bf3937a36a739b1","_dom_classes":[],"description":"100%","_model_name":"FloatProgressModel","bar_style":"success","max":95634,"_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":95634,"_view_count":null,"_view_module_version":"1.5.0","orientation":"horizontal","min":0,"description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_58ba64dcc187478b9ad01d4e4e53716e"}},"9b2ac9f12d2648d0bc84ff8e6002ef60":{"model_module":"@jupyter-widgets/controls","model_name":"HTMLModel","state":{"_view_name":"HTMLView","style":"IPY_MODEL_d3495b04062741a2828127e7f79593b0","_dom_classes":[],"description":"","_model_name":"HTMLModel","placeholder":"​","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":" 95634/95634 [03:02&lt;00:00, 523.03 ORF/s]","_view_count":null,"_view_module_version":"1.5.0","description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_4bd32ce4ec07480da6f50f816d3704e9"}},"e2585d01d62343dd8bf3937a36a739b1":{"model_module":"@jupyter-widgets/controls","model_name":"ProgressStyleModel","state":{"_view_name":"StyleView","_model_name":"ProgressStyleModel","description_width":"initial","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","bar_color":null,"_model_module":"@jupyter-widgets/controls"}},"58ba64dcc187478b9ad01d4e4e53716e":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"d3495b04062741a2828127e7f79593b0":{"model_module":"@jupyter-widgets/controls","model_name":"DescriptionStyleModel","state":{"_view_name":"StyleView","_model_name":"DescriptionStyleModel","description_width":"","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","_model_module":"@jupyter-widgets/controls"}},"4bd32ce4ec07480da6f50f816d3704e9":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"7ff6a0f775b04308bdd6d3b4fce51764":{"model_module":"@jupyter-widgets/controls","model_name":"HBoxModel","state":{"_view_name":"HBoxView","_dom_classes":[],"_model_name":"HBoxModel","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.5.0","box_style":"","layout":"IPY_MODEL_f8573da2f9854215990f605c78a80cc8","_model_module":"@jupyter-widgets/controls","children":["IPY_MODEL_eb3e905d04fc445b94bb1a3c473b2d4b","IPY_MODEL_c66ac24011da43b7828f54430f7b0667"]}},"f8573da2f9854215990f605c78a80cc8":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"eb3e905d04fc445b94bb1a3c473b2d4b":{"model_module":"@jupyter-widgets/controls","model_name":"FloatProgressModel","state":{"_view_name":"ProgressView","style":"IPY_MODEL_94a3c09e73a34e029eb3c29884cb0f9c","_dom_classes":[],"description":"100%","_model_name":"FloatProgressModel","bar_style":"success","max":17,"_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":17,"_view_count":null,"_view_module_version":"1.5.0","orientation":"horizontal","min":0,"description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_ac9b03a4d22045e7be34b8116930865d"}},"c66ac24011da43b7828f54430f7b0667":{"model_module":"@jupyter-widgets/controls","model_name":"HTMLModel","state":{"_view_name":"HTMLView","style":"IPY_MODEL_e1f58a4904a446e2a450c4df6144c5e0","_dom_classes":[],"description":"","_model_name":"HTMLModel","placeholder":"​","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":" 17/17 [02:53&lt;00:00, 10.20s/ batch]","_view_count":null,"_view_module_version":"1.5.0","description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_9e4c6c6c34604458825c5da3530a2bf3"}},"94a3c09e73a34e029eb3c29884cb0f9c":{"model_module":"@jupyter-widgets/controls","model_name":"ProgressStyleModel","state":{"_view_name":"StyleView","_model_name":"ProgressStyleModel","description_width":"initial","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","bar_color":null,"_model_module":"@jupyter-widgets/controls"}},"ac9b03a4d22045e7be34b8116930865d":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"e1f58a4904a446e2a450c4df6144c5e0":{"model_module":"@jupyter-widgets/controls","model_name":"DescriptionStyleModel","state":{"_view_name":"StyleView","_model_name":"DescriptionStyleModel","description_width":"","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","_model_module":"@jupyter-widgets/controls"}},"9e4c6c6c34604458825c5da3530a2bf3":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"ba1e0ca3afb34fa6a1f6a024745b5dc0":{"model_module":"@jupyter-widgets/controls","model_name":"HBoxModel","state":{"_view_name":"HBoxView","_dom_classes":[],"_model_name":"HBoxModel","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.5.0","box_style":"","layout":"IPY_MODEL_0809fa4a2888423a9ef97eeb5e4f1a60","_model_module":"@jupyter-widgets/controls","children":["IPY_MODEL_04326aed601f416194e1a70a598abc18","IPY_MODEL_37f3b11384f14f8c9b94d587d7dc6fa2"]}},"0809fa4a2888423a9ef97eeb5e4f1a60":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"04326aed601f416194e1a70a598abc18":{"model_module":"@jupyter-widgets/controls","model_name":"FloatProgressModel","state":{"_view_name":"ProgressView","style":"IPY_MODEL_5b768691e00d48e98d3f8899a4437420","_dom_classes":[],"description":"100%","_model_name":"FloatProgressModel","bar_style":"success","max":5,"_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":5,"_view_count":null,"_view_module_version":"1.5.0","orientation":"horizontal","min":0,"description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_ba2d0be33ded47e6a457993fa9def5e8"}},"37f3b11384f14f8c9b94d587d7dc6fa2":{"model_module":"@jupyter-widgets/controls","model_name":"HTMLModel","state":{"_view_name":"HTMLView","style":"IPY_MODEL_166e9d3138b444a39dcb42efe259ac6a","_dom_classes":[],"description":"","_model_name":"HTMLModel","placeholder":"​","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":" 5/5 [02:46&lt;00:00, 33.30s/ batch]","_view_count":null,"_view_module_version":"1.5.0","description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_8dbb0e6073244354b86507f3e33a113b"}},"5b768691e00d48e98d3f8899a4437420":{"model_module":"@jupyter-widgets/controls","model_name":"ProgressStyleModel","state":{"_view_name":"StyleView","_model_name":"ProgressStyleModel","description_width":"initial","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","bar_color":null,"_model_module":"@jupyter-widgets/controls"}},"ba2d0be33ded47e6a457993fa9def5e8":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"166e9d3138b444a39dcb42efe259ac6a":{"model_module":"@jupyter-widgets/controls","model_name":"DescriptionStyleModel","state":{"_view_name":"StyleView","_model_name":"DescriptionStyleModel","description_width":"","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","_model_module":"@jupyter-widgets/controls"}},"8dbb0e6073244354b86507f3e33a113b":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"3dc805dc47c643e19f9b9d222f7a7f0d":{"model_module":"@jupyter-widgets/controls","model_name":"HBoxModel","state":{"_view_name":"HBoxView","_dom_classes":[],"_model_name":"HBoxModel","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.5.0","box_style":"","layout":"IPY_MODEL_25a763a216a74be0bb84d1848e419d90","_model_module":"@jupyter-widgets/controls","children":["IPY_MODEL_db1fbe17dc5b4c58af627474ea9aea3f","IPY_MODEL_cb791b818af64c4fabb19552650f6295"]}},"25a763a216a74be0bb84d1848e419d90":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"db1fbe17dc5b4c58af627474ea9aea3f":{"model_module":"@jupyter-widgets/controls","model_name":"FloatProgressModel","state":{"_view_name":"ProgressView","style":"IPY_MODEL_98b7e3c73f824bf6a1c295a56de7d682","_dom_classes":[],"description":"100%","_model_name":"FloatProgressModel","bar_style":"success","max":49809,"_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":49809,"_view_count":null,"_view_module_version":"1.5.0","orientation":"horizontal","min":0,"description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_835621e2ba374aa78f44d6c508ed296b"}},"cb791b818af64c4fabb19552650f6295":{"model_module":"@jupyter-widgets/controls","model_name":"HTMLModel","state":{"_view_name":"HTMLView","style":"IPY_MODEL_f2938a90717e4ed28c7f8584ecb76fe1","_dom_classes":[],"description":"","_model_name":"HTMLModel","placeholder":"​","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":" 49809/49809 [02:43&lt;00:00, 304.40 ORF/s]","_view_count":null,"_view_module_version":"1.5.0","description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_9df573f0a34b4654840d5370af5e9212"}},"98b7e3c73f824bf6a1c295a56de7d682":{"model_module":"@jupyter-widgets/controls","model_name":"ProgressStyleModel","state":{"_view_name":"StyleView","_model_name":"ProgressStyleModel","description_width":"initial","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","bar_color":null,"_model_module":"@jupyter-widgets/controls"}},"835621e2ba374aa78f44d6c508ed296b":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"f2938a90717e4ed28c7f8584ecb76fe1":{"model_module":"@jupyter-widgets/controls","model_name":"DescriptionStyleModel","state":{"_view_name":"StyleView","_model_name":"DescriptionStyleModel","description_width":"","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","_model_module":"@jupyter-widgets/controls"}},"9df573f0a34b4654840d5370af5e9212":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"3811d8b5a11c46adb189f822412a4872":{"model_module":"@jupyter-widgets/controls","model_name":"HBoxModel","state":{"_view_name":"HBoxView","_dom_classes":[],"_model_name":"HBoxModel","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.5.0","box_style":"","layout":"IPY_MODEL_eb378cc881ec486eb992948e1bdbf0d7","_model_module":"@jupyter-widgets/controls","children":["IPY_MODEL_71902de491834536b1194dc46765de57","IPY_MODEL_090ae5a595f44b59a8f248d1f740cac9"]}},"eb378cc881ec486eb992948e1bdbf0d7":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"71902de491834536b1194dc46765de57":{"model_module":"@jupyter-widgets/controls","model_name":"FloatProgressModel","state":{"_view_name":"ProgressView","style":"IPY_MODEL_c09b2e98aa744e3da6e5110eb1252e0e","_dom_classes":[],"description":"100%","_model_name":"FloatProgressModel","bar_style":"success","max":170,"_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":170,"_view_count":null,"_view_module_version":"1.5.0","orientation":"horizontal","min":0,"description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_60e4b4feec38486f96eb075e89dc8d9a"}},"090ae5a595f44b59a8f248d1f740cac9":{"model_module":"@jupyter-widgets/controls","model_name":"HTMLModel","state":{"_view_name":"HTMLView","style":"IPY_MODEL_cd12f8b58c19464a8d05993e261f0b15","_dom_classes":[],"description":"","_model_name":"HTMLModel","placeholder":"​","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":" 170/170 [01:07&lt;00:00,  2.50 batch/s]","_view_count":null,"_view_module_version":"1.5.0","description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_9e2876efafd84af49fd70741906ed9bf"}},"c09b2e98aa744e3da6e5110eb1252e0e":{"model_module":"@jupyter-widgets/controls","model_name":"ProgressStyleModel","state":{"_view_name":"StyleView","_model_name":"ProgressStyleModel","description_width":"initial","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","bar_color":null,"_model_module":"@jupyter-widgets/controls"}},"60e4b4feec38486f96eb075e89dc8d9a":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"cd12f8b58c19464a8d05993e261f0b15":{"model_module":"@jupyter-widgets/controls","model_name":"DescriptionStyleModel","state":{"_view_name":"StyleView","_model_name":"DescriptionStyleModel","description_width":"","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","_model_module":"@jupyter-widgets/controls"}},"9e2876efafd84af49fd70741906ed9bf":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"a148a06e19654464a603b25d760833c0":{"model_module":"@jupyter-widgets/controls","model_name":"HBoxModel","state":{"_view_name":"HBoxView","_dom_classes":[],"_model_name":"HBoxModel","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.5.0","box_style":"","layout":"IPY_MODEL_985e182f4450419796be72256cc70190","_model_module":"@jupyter-widgets/controls","children":["IPY_MODEL_08435c07626d410ebe6757dd24703b24","IPY_MODEL_d0928e23276d45bab3fe0d7f20d7b4ab"]}},"985e182f4450419796be72256cc70190":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"08435c07626d410ebe6757dd24703b24":{"model_module":"@jupyter-widgets/controls","model_name":"FloatProgressModel","state":{"_view_name":"ProgressView","style":"IPY_MODEL_ec234bec2534408f80889d820a54f6c6","_dom_classes":[],"description":"100%","_model_name":"FloatProgressModel","bar_style":"success","max":51,"_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":51,"_view_count":null,"_view_module_version":"1.5.0","orientation":"horizontal","min":0,"description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_0d4e32eee09246cf99fbd31f28fba214"}},"d0928e23276d45bab3fe0d7f20d7b4ab":{"model_module":"@jupyter-widgets/controls","model_name":"HTMLModel","state":{"_view_name":"HTMLView","style":"IPY_MODEL_69f8dd33a14a4b50ac7994cc6b16b411","_dom_classes":[],"description":"","_model_name":"HTMLModel","placeholder":"​","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":" 51/51 [00:13&lt;00:00,  3.92 batch/s]","_view_count":null,"_view_module_version":"1.5.0","description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_773c40274e2b45428a7bbfb19b6ee690"}},"ec234bec2534408f80889d820a54f6c6":{"model_module":"@jupyter-widgets/controls","model_name":"ProgressStyleModel","state":{"_view_name":"StyleView","_model_name":"ProgressStyleModel","description_width":"initial","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","bar_color":null,"_model_module":"@jupyter-widgets/controls"}},"0d4e32eee09246cf99fbd31f28fba214":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"69f8dd33a14a4b50ac7994cc6b16b411":{"model_module":"@jupyter-widgets/controls","model_name":"DescriptionStyleModel","state":{"_view_name":"StyleView","_model_name":"DescriptionStyleModel","description_width":"","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","_model_module":"@jupyter-widgets/controls"}},"773c40274e2b45428a7bbfb19b6ee690":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"c0e4c752431f4e3593dbd77e001eee62":{"model_module":"@jupyter-widgets/controls","model_name":"HBoxModel","state":{"_view_name":"HBoxView","_dom_classes":[],"_model_name":"HBoxModel","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.5.0","box_style":"","layout":"IPY_MODEL_935b9acc374c4d548508f885fc6227a9","_model_module":"@jupyter-widgets/controls","children":["IPY_MODEL_42d4676308894d70bbb8082274130b2c","IPY_MODEL_72f0941e1abd4b89a689866e5275a315"]}},"935b9acc374c4d548508f885fc6227a9":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"42d4676308894d70bbb8082274130b2c":{"model_module":"@jupyter-widgets/controls","model_name":"FloatProgressModel","state":{"_view_name":"ProgressView","style":"IPY_MODEL_f391a572d84f462ebff52c15711484d3","_dom_classes":[],"description":"100%","_model_name":"FloatProgressModel","bar_style":"success","max":508244,"_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":508244,"_view_count":null,"_view_module_version":"1.5.0","orientation":"horizontal","min":0,"description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_b40a1a6e80eb4265a0690eb0cab6947a"}},"72f0941e1abd4b89a689866e5275a315":{"model_module":"@jupyter-widgets/controls","model_name":"HTMLModel","state":{"_view_name":"HTMLView","style":"IPY_MODEL_d69b78fc39e94155a2d869144a6f3784","_dom_classes":[],"description":"","_model_name":"HTMLModel","placeholder":"​","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":" 508244/508244 [09:19&lt;00:00, 908.05 ORF/s]","_view_count":null,"_view_module_version":"1.5.0","description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_ea1fdc9c158b4d898f91a664ac01b6df"}},"f391a572d84f462ebff52c15711484d3":{"model_module":"@jupyter-widgets/controls","model_name":"ProgressStyleModel","state":{"_view_name":"StyleView","_model_name":"ProgressStyleModel","description_width":"initial","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","bar_color":null,"_model_module":"@jupyter-widgets/controls"}},"b40a1a6e80eb4265a0690eb0cab6947a":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"d69b78fc39e94155a2d869144a6f3784":{"model_module":"@jupyter-widgets/controls","model_name":"DescriptionStyleModel","state":{"_view_name":"StyleView","_model_name":"DescriptionStyleModel","description_width":"","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","_model_module":"@jupyter-widgets/controls"}},"ea1fdc9c158b4d898f91a664ac01b6df":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"7477b8c7e1f6469aa9c868870bff7fe1":{"model_module":"@jupyter-widgets/controls","model_name":"HBoxModel","state":{"_view_name":"HBoxView","_dom_classes":[],"_model_name":"HBoxModel","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.5.0","box_style":"","layout":"IPY_MODEL_a542c353bbb940608438d4d33a10b17a","_model_module":"@jupyter-widgets/controls","children":["IPY_MODEL_e2417219645941a5b0d0507b9e396fea","IPY_MODEL_7d96c23469984d5b8eced8a71aa7dfaa"]}},"a542c353bbb940608438d4d33a10b17a":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"e2417219645941a5b0d0507b9e396fea":{"model_module":"@jupyter-widgets/controls","model_name":"FloatProgressModel","state":{"_view_name":"ProgressView","style":"IPY_MODEL_800b1c365e5a42d7bdaf1adb82ffc032","_dom_classes":[],"description":"100%","_model_name":"FloatProgressModel","bar_style":"success","max":67,"_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":67,"_view_count":null,"_view_module_version":"1.5.0","orientation":"horizontal","min":0,"description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_22a020a53ff94b79a02762b2dfcf1b95"}},"7d96c23469984d5b8eced8a71aa7dfaa":{"model_module":"@jupyter-widgets/controls","model_name":"HTMLModel","state":{"_view_name":"HTMLView","style":"IPY_MODEL_0859cda235fc423b94d19b86aa47ad66","_dom_classes":[],"description":"","_model_name":"HTMLModel","placeholder":"​","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":" 67/67 [00:27&lt;00:00,  2.44 batch/s]","_view_count":null,"_view_module_version":"1.5.0","description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_c1b844a0f1164d2cb362b2dcee6b8760"}},"800b1c365e5a42d7bdaf1adb82ffc032":{"model_module":"@jupyter-widgets/controls","model_name":"ProgressStyleModel","state":{"_view_name":"StyleView","_model_name":"ProgressStyleModel","description_width":"initial","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","bar_color":null,"_model_module":"@jupyter-widgets/controls"}},"22a020a53ff94b79a02762b2dfcf1b95":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"0859cda235fc423b94d19b86aa47ad66":{"model_module":"@jupyter-widgets/controls","model_name":"DescriptionStyleModel","state":{"_view_name":"StyleView","_model_name":"DescriptionStyleModel","description_width":"","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","_model_module":"@jupyter-widgets/controls"}},"c1b844a0f1164d2cb362b2dcee6b8760":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"8b88b8c9bcce4d47b17a69226d365f4f":{"model_module":"@jupyter-widgets/controls","model_name":"HBoxModel","state":{"_view_name":"HBoxView","_dom_classes":[],"_model_name":"HBoxModel","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.5.0","box_style":"","layout":"IPY_MODEL_0da41d632d034324bd5c8855d9bd009b","_model_module":"@jupyter-widgets/controls","children":["IPY_MODEL_5ada2a253f89490d90a088dab65c945a","IPY_MODEL_109e2e4f30824e3b99cffe87ec96aa2a"]}},"0da41d632d034324bd5c8855d9bd009b":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"5ada2a253f89490d90a088dab65c945a":{"model_module":"@jupyter-widgets/controls","model_name":"FloatProgressModel","state":{"_view_name":"ProgressView","style":"IPY_MODEL_bd54d22adf6845b69a303eb2ed5127db","_dom_classes":[],"description":"100%","_model_name":"FloatProgressModel","bar_style":"success","max":20,"_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":20,"_view_count":null,"_view_module_version":"1.5.0","orientation":"horizontal","min":0,"description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_fa3b1b586e9d49f0b8f8bbe5587ce126"}},"109e2e4f30824e3b99cffe87ec96aa2a":{"model_module":"@jupyter-widgets/controls","model_name":"HTMLModel","state":{"_view_name":"HTMLView","style":"IPY_MODEL_574ff11707764819a8c71505b8e1741d","_dom_classes":[],"description":"","_model_name":"HTMLModel","placeholder":"​","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":" 20/20 [06:27&lt;00:00, 19.40s/ batch]","_view_count":null,"_view_module_version":"1.5.0","description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_90c3c041eb124813ab188c739e666a01"}},"bd54d22adf6845b69a303eb2ed5127db":{"model_module":"@jupyter-widgets/controls","model_name":"ProgressStyleModel","state":{"_view_name":"StyleView","_model_name":"ProgressStyleModel","description_width":"initial","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","bar_color":null,"_model_module":"@jupyter-widgets/controls"}},"fa3b1b586e9d49f0b8f8bbe5587ce126":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"574ff11707764819a8c71505b8e1741d":{"model_module":"@jupyter-widgets/controls","model_name":"DescriptionStyleModel","state":{"_view_name":"StyleView","_model_name":"DescriptionStyleModel","description_width":"","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","_model_module":"@jupyter-widgets/controls"}},"90c3c041eb124813ab188c739e666a01":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"87030ead123f4fe699f7235fe8b401d2":{"model_module":"@jupyter-widgets/controls","model_name":"HBoxModel","state":{"_view_name":"HBoxView","_dom_classes":[],"_model_name":"HBoxModel","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.5.0","box_style":"","layout":"IPY_MODEL_9b132133815b4b67b4d79fbcfd8dd9bb","_model_module":"@jupyter-widgets/controls","children":["IPY_MODEL_09dbf9bbfd1d4c7f9db74fc1916c69b0","IPY_MODEL_73c781146ced4a84ae4bfd2357e56520"]}},"9b132133815b4b67b4d79fbcfd8dd9bb":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"09dbf9bbfd1d4c7f9db74fc1916c69b0":{"model_module":"@jupyter-widgets/controls","model_name":"FloatProgressModel","state":{"_view_name":"ProgressView","style":"IPY_MODEL_9d65b49789eb4a24be05ff84a84f7d6b","_dom_classes":[],"description":"100%","_model_name":"FloatProgressModel","bar_style":"success","max":199251,"_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":199251,"_view_count":null,"_view_module_version":"1.5.0","orientation":"horizontal","min":0,"description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_c4489fd85dd34a85b138a2069c0cf8ed"}},"73c781146ced4a84ae4bfd2357e56520":{"model_module":"@jupyter-widgets/controls","model_name":"HTMLModel","state":{"_view_name":"HTMLView","style":"IPY_MODEL_7a418d3674374e81a199ecced5b48392","_dom_classes":[],"description":"","_model_name":"HTMLModel","placeholder":"​","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":" 199251/199251 [06:16&lt;00:00, 529.58 ORF/s]","_view_count":null,"_view_module_version":"1.5.0","description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_90b8ef8a81e74a58bdbd780bf9e52527"}},"9d65b49789eb4a24be05ff84a84f7d6b":{"model_module":"@jupyter-widgets/controls","model_name":"ProgressStyleModel","state":{"_view_name":"StyleView","_model_name":"ProgressStyleModel","description_width":"initial","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","bar_color":null,"_model_module":"@jupyter-widgets/controls"}},"c4489fd85dd34a85b138a2069c0cf8ed":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"7a418d3674374e81a199ecced5b48392":{"model_module":"@jupyter-widgets/controls","model_name":"DescriptionStyleModel","state":{"_view_name":"StyleView","_model_name":"DescriptionStyleModel","description_width":"","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","_model_module":"@jupyter-widgets/controls"}},"90b8ef8a81e74a58bdbd780bf9e52527":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"7c6d6006f1684eb783858af378569668":{"model_module":"@jupyter-widgets/controls","model_name":"HBoxModel","state":{"_view_name":"HBoxView","_dom_classes":[],"_model_name":"HBoxModel","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.5.0","box_style":"","layout":"IPY_MODEL_6ebaf995fc2e49f8ba1e26ae02dd1167","_model_module":"@jupyter-widgets/controls","children":["IPY_MODEL_31fffdc377b346af985938644bfedde5","IPY_MODEL_cf761e67322e4d7e80057614e890636b"]}},"6ebaf995fc2e49f8ba1e26ae02dd1167":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"31fffdc377b346af985938644bfedde5":{"model_module":"@jupyter-widgets/controls","model_name":"FloatProgressModel","state":{"_view_name":"ProgressView","style":"IPY_MODEL_3dda42eb6bf441c4b959e4334aa69b12","_dom_classes":[],"description":"100%","_model_name":"FloatProgressModel","bar_style":"success","max":42,"_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":42,"_view_count":null,"_view_module_version":"1.5.0","orientation":"horizontal","min":0,"description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_edcb35a54ed846659358dd753f334051"}},"cf761e67322e4d7e80057614e890636b":{"model_module":"@jupyter-widgets/controls","model_name":"HTMLModel","state":{"_view_name":"HTMLView","style":"IPY_MODEL_32d226d94dd545c7a4c7a2f09a30bbf1","_dom_classes":[],"description":"","_model_name":"HTMLModel","placeholder":"​","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":" 42/42 [00:43&lt;00:00,  1.03s/ batch]","_view_count":null,"_view_module_version":"1.5.0","description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_174870024aab4d20b082acaf165d331d"}},"3dda42eb6bf441c4b959e4334aa69b12":{"model_module":"@jupyter-widgets/controls","model_name":"ProgressStyleModel","state":{"_view_name":"StyleView","_model_name":"ProgressStyleModel","description_width":"initial","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","bar_color":null,"_model_module":"@jupyter-widgets/controls"}},"edcb35a54ed846659358dd753f334051":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"32d226d94dd545c7a4c7a2f09a30bbf1":{"model_module":"@jupyter-widgets/controls","model_name":"DescriptionStyleModel","state":{"_view_name":"StyleView","_model_name":"DescriptionStyleModel","description_width":"","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","_model_module":"@jupyter-widgets/controls"}},"174870024aab4d20b082acaf165d331d":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"3876f9ce8b284399b2b9649839e1fe02":{"model_module":"@jupyter-widgets/controls","model_name":"HBoxModel","state":{"_view_name":"HBoxView","_dom_classes":[],"_model_name":"HBoxModel","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.5.0","box_style":"","layout":"IPY_MODEL_9d575a923fad4f7aa7b7268a74503196","_model_module":"@jupyter-widgets/controls","children":["IPY_MODEL_cf4d06d76e0e40bd87f7e1f3abb05e69","IPY_MODEL_deea867310294a838b7d76c4dc876e30"]}},"9d575a923fad4f7aa7b7268a74503196":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"cf4d06d76e0e40bd87f7e1f3abb05e69":{"model_module":"@jupyter-widgets/controls","model_name":"FloatProgressModel","state":{"_view_name":"ProgressView","style":"IPY_MODEL_6e0458ad042b46b1a2a345f79c884429","_dom_classes":[],"description":"100%","_model_name":"FloatProgressModel","bar_style":"success","max":13,"_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":13,"_view_count":null,"_view_module_version":"1.5.0","orientation":"horizontal","min":0,"description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_52f23e66b7d8473f943e2bd145c908ff"}},"deea867310294a838b7d76c4dc876e30":{"model_module":"@jupyter-widgets/controls","model_name":"HTMLModel","state":{"_view_name":"HTMLView","style":"IPY_MODEL_441ee3967d624d0cbcc4239c1751237c","_dom_classes":[],"description":"","_model_name":"HTMLModel","placeholder":"​","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":" 13/13 [00:23&lt;00:00,  1.84s/ batch]","_view_count":null,"_view_module_version":"1.5.0","description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_df12288a746548e4a304a726c63bbba5"}},"6e0458ad042b46b1a2a345f79c884429":{"model_module":"@jupyter-widgets/controls","model_name":"ProgressStyleModel","state":{"_view_name":"StyleView","_model_name":"ProgressStyleModel","description_width":"initial","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","bar_color":null,"_model_module":"@jupyter-widgets/controls"}},"52f23e66b7d8473f943e2bd145c908ff":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"441ee3967d624d0cbcc4239c1751237c":{"model_module":"@jupyter-widgets/controls","model_name":"DescriptionStyleModel","state":{"_view_name":"StyleView","_model_name":"DescriptionStyleModel","description_width":"","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","_model_module":"@jupyter-widgets/controls"}},"df12288a746548e4a304a726c63bbba5":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"1b46dca39e954201b0852959a1c4c10c":{"model_module":"@jupyter-widgets/controls","model_name":"HBoxModel","state":{"_view_name":"HBoxView","_dom_classes":[],"_model_name":"HBoxModel","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.5.0","box_style":"","layout":"IPY_MODEL_b7ab6dcf8c2b45f489cd3627493db567","_model_module":"@jupyter-widgets/controls","children":["IPY_MODEL_c0f00aab3fc2495389b9664975acb8d0","IPY_MODEL_fcb8e1368fc2457c9d395fc3aa498402"]}},"b7ab6dcf8c2b45f489cd3627493db567":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"c0f00aab3fc2495389b9664975acb8d0":{"model_module":"@jupyter-widgets/controls","model_name":"FloatProgressModel","state":{"_view_name":"ProgressView","style":"IPY_MODEL_076e78d7968f45c9a2d9e07c0d637e9e","_dom_classes":[],"description":"100%","_model_name":"FloatProgressModel","bar_style":"success","max":106600,"_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":106600,"_view_count":null,"_view_module_version":"1.5.0","orientation":"horizontal","min":0,"description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_d3739899900349749c6b1f73c27e17d9"}},"fcb8e1368fc2457c9d395fc3aa498402":{"model_module":"@jupyter-widgets/controls","model_name":"HTMLModel","state":{"_view_name":"HTMLView","style":"IPY_MODEL_cb82757552d041458a50de12d82a1e18","_dom_classes":[],"description":"","_model_name":"HTMLModel","placeholder":"​","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":" 106600/106600 [08:08&lt;00:00, 218.07 ORF/s]","_view_count":null,"_view_module_version":"1.5.0","description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_17957ce99fee47bd8a8849c79e08e304"}},"076e78d7968f45c9a2d9e07c0d637e9e":{"model_module":"@jupyter-widgets/controls","model_name":"ProgressStyleModel","state":{"_view_name":"StyleView","_model_name":"ProgressStyleModel","description_width":"initial","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","bar_color":null,"_model_module":"@jupyter-widgets/controls"}},"d3739899900349749c6b1f73c27e17d9":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"cb82757552d041458a50de12d82a1e18":{"model_module":"@jupyter-widgets/controls","model_name":"DescriptionStyleModel","state":{"_view_name":"StyleView","_model_name":"DescriptionStyleModel","description_width":"","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","_model_module":"@jupyter-widgets/controls"}},"17957ce99fee47bd8a8849c79e08e304":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"fe61a9a41ba347b9ba49a5e7d5d0f5b8":{"model_module":"@jupyter-widgets/controls","model_name":"HBoxModel","state":{"_view_name":"HBoxView","_dom_classes":[],"_model_name":"HBoxModel","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.5.0","box_style":"","layout":"IPY_MODEL_47a6367ec63940d4a1be218c64f3b6d8","_model_module":"@jupyter-widgets/controls","children":["IPY_MODEL_fe4df2995a624f5cbbc21ac2e8867671","IPY_MODEL_6a7aeffe10f849cd8428da1c73e5bc5a"]}},"47a6367ec63940d4a1be218c64f3b6d8":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"fe4df2995a624f5cbbc21ac2e8867671":{"model_module":"@jupyter-widgets/controls","model_name":"FloatProgressModel","state":{"_view_name":"ProgressView","style":"IPY_MODEL_d35562b393334dea96aef8ca76263b7d","_dom_classes":[],"description":"100%","_model_name":"FloatProgressModel","bar_style":"success","max":8425,"_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":8425,"_view_count":null,"_view_module_version":"1.5.0","orientation":"horizontal","min":0,"description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_c0a4b3b957a3427da3ac9ffaba4e5a5c"}},"6a7aeffe10f849cd8428da1c73e5bc5a":{"model_module":"@jupyter-widgets/controls","model_name":"HTMLModel","state":{"_view_name":"HTMLView","style":"IPY_MODEL_2d16a44a6c6f403a81bce7986b14896b","_dom_classes":[],"description":"","_model_name":"HTMLModel","placeholder":"​","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":" 8425/8425 [04:37&lt;00:00, 30.39 ORF/s]","_view_count":null,"_view_module_version":"1.5.0","description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_5cd25146117c4a1b8feb92bd5589af02"}},"d35562b393334dea96aef8ca76263b7d":{"model_module":"@jupyter-widgets/controls","model_name":"ProgressStyleModel","state":{"_view_name":"StyleView","_model_name":"ProgressStyleModel","description_width":"initial","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","bar_color":null,"_model_module":"@jupyter-widgets/controls"}},"c0a4b3b957a3427da3ac9ffaba4e5a5c":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"2d16a44a6c6f403a81bce7986b14896b":{"model_module":"@jupyter-widgets/controls","model_name":"DescriptionStyleModel","state":{"_view_name":"StyleView","_model_name":"DescriptionStyleModel","description_width":"","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","_model_module":"@jupyter-widgets/controls"}},"5cd25146117c4a1b8feb92bd5589af02":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"498794a26d4349288751a0bd1224b452":{"model_module":"@jupyter-widgets/controls","model_name":"HBoxModel","state":{"_view_name":"HBoxView","_dom_classes":[],"_model_name":"HBoxModel","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.5.0","box_style":"","layout":"IPY_MODEL_6b9a5ab9b3e448ff815500fda597b3fc","_model_module":"@jupyter-widgets/controls","children":["IPY_MODEL_730f13e59f744db099078bd58d6ccd0a","IPY_MODEL_24c218447be94737b439e9df45b48038"]}},"6b9a5ab9b3e448ff815500fda597b3fc":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"730f13e59f744db099078bd58d6ccd0a":{"model_module":"@jupyter-widgets/controls","model_name":"FloatProgressModel","state":{"_view_name":"ProgressView","style":"IPY_MODEL_5e396a7b01e54ff6a2f8a1cad7e43dbb","_dom_classes":[],"description":"100%","_model_name":"FloatProgressModel","bar_style":"success","max":5811,"_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":5811,"_view_count":null,"_view_module_version":"1.5.0","orientation":"horizontal","min":0,"description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_e074c54b514b409cbe623e98251bd14c"}},"24c218447be94737b439e9df45b48038":{"model_module":"@jupyter-widgets/controls","model_name":"HTMLModel","state":{"_view_name":"HTMLView","style":"IPY_MODEL_200d2b3abb96408b9fb8924e36e840fa","_dom_classes":[],"description":"","_model_name":"HTMLModel","placeholder":"​","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":" 5811/5811 [03:13&lt;00:00, 29.97 ORF/s]","_view_count":null,"_view_module_version":"1.5.0","description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_f3d33a1bc60044d1b51aaa7da072714b"}},"5e396a7b01e54ff6a2f8a1cad7e43dbb":{"model_module":"@jupyter-widgets/controls","model_name":"ProgressStyleModel","state":{"_view_name":"StyleView","_model_name":"ProgressStyleModel","description_width":"initial","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","bar_color":null,"_model_module":"@jupyter-widgets/controls"}},"e074c54b514b409cbe623e98251bd14c":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"200d2b3abb96408b9fb8924e36e840fa":{"model_module":"@jupyter-widgets/controls","model_name":"DescriptionStyleModel","state":{"_view_name":"StyleView","_model_name":"DescriptionStyleModel","description_width":"","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","_model_module":"@jupyter-widgets/controls"}},"f3d33a1bc60044d1b51aaa7da072714b":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"e60d3a18c6ae4a948beb4c89e82f1fa5":{"model_module":"@jupyter-widgets/controls","model_name":"HBoxModel","state":{"_view_name":"HBoxView","_dom_classes":[],"_model_name":"HBoxModel","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.5.0","box_style":"","layout":"IPY_MODEL_aec35d01a9b6422b92c82cfac0ae1cf5","_model_module":"@jupyter-widgets/controls","children":["IPY_MODEL_1cc2d8033406477e91ab45cb89f8a741","IPY_MODEL_69f3067b87c24f8a8fab2d3b37f7dc40"]}},"aec35d01a9b6422b92c82cfac0ae1cf5":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"1cc2d8033406477e91ab45cb89f8a741":{"model_module":"@jupyter-widgets/controls","model_name":"FloatProgressModel","state":{"_view_name":"ProgressView","style":"IPY_MODEL_c43b0ce8e6a74e6eb2260fa694750c84","_dom_classes":[],"description":"100%","_model_name":"FloatProgressModel","bar_style":"success","max":2159,"_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":2159,"_view_count":null,"_view_module_version":"1.5.0","orientation":"horizontal","min":0,"description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_e30f0a7a645e4392b4c27281edc1dd0a"}},"69f3067b87c24f8a8fab2d3b37f7dc40":{"model_module":"@jupyter-widgets/controls","model_name":"HTMLModel","state":{"_view_name":"HTMLView","style":"IPY_MODEL_f448bc3649e04ad4bc52762141f08033","_dom_classes":[],"description":"","_model_name":"HTMLModel","placeholder":"​","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":" 2159/2159 [02:00&lt;00:00, 17.95 ORF/s]","_view_count":null,"_view_module_version":"1.5.0","description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_861fbe6e0b614a969c14a49b2c0a2178"}},"c43b0ce8e6a74e6eb2260fa694750c84":{"model_module":"@jupyter-widgets/controls","model_name":"ProgressStyleModel","state":{"_view_name":"StyleView","_model_name":"ProgressStyleModel","description_width":"initial","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","bar_color":null,"_model_module":"@jupyter-widgets/controls"}},"e30f0a7a645e4392b4c27281edc1dd0a":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"f448bc3649e04ad4bc52762141f08033":{"model_module":"@jupyter-widgets/controls","model_name":"DescriptionStyleModel","state":{"_view_name":"StyleView","_model_name":"DescriptionStyleModel","description_width":"","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","_model_module":"@jupyter-widgets/controls"}},"861fbe6e0b614a969c14a49b2c0a2178":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"42c7bcfeeba04f34910e1b23a09b1ff5":{"model_module":"@jupyter-widgets/controls","model_name":"HBoxModel","state":{"_view_name":"HBoxView","_dom_classes":[],"_model_name":"HBoxModel","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.5.0","box_style":"","layout":"IPY_MODEL_e50ab5036adc4ab8b6785158417b63e7","_model_module":"@jupyter-widgets/controls","children":["IPY_MODEL_8bbc74bdec5b40ed96f468cefdb41be7","IPY_MODEL_64b359513a8840e79a964a10725daf4c"]}},"e50ab5036adc4ab8b6785158417b63e7":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"8bbc74bdec5b40ed96f468cefdb41be7":{"model_module":"@jupyter-widgets/controls","model_name":"FloatProgressModel","state":{"_view_name":"ProgressView","style":"IPY_MODEL_809cbe2e605240509274bc37cd6d1db1","_dom_classes":[],"description":"100%","_model_name":"FloatProgressModel","bar_style":"success","max":45,"_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":45,"_view_count":null,"_view_module_version":"1.5.0","orientation":"horizontal","min":0,"description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_b10f9726bac5425bae383b90098fbbd5"}},"64b359513a8840e79a964a10725daf4c":{"model_module":"@jupyter-widgets/controls","model_name":"HTMLModel","state":{"_view_name":"HTMLView","style":"IPY_MODEL_f25104f7dcea4031a7c3541d21dd6d99","_dom_classes":[],"description":"","_model_name":"HTMLModel","placeholder":"​","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":" 45/45 [00:32&lt;00:00,  1.40 batch/s]","_view_count":null,"_view_module_version":"1.5.0","description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_3040fc326fb14b15a9df1f73ac1df872"}},"809cbe2e605240509274bc37cd6d1db1":{"model_module":"@jupyter-widgets/controls","model_name":"ProgressStyleModel","state":{"_view_name":"StyleView","_model_name":"ProgressStyleModel","description_width":"initial","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","bar_color":null,"_model_module":"@jupyter-widgets/controls"}},"b10f9726bac5425bae383b90098fbbd5":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"f25104f7dcea4031a7c3541d21dd6d99":{"model_module":"@jupyter-widgets/controls","model_name":"DescriptionStyleModel","state":{"_view_name":"StyleView","_model_name":"DescriptionStyleModel","description_width":"","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","_model_module":"@jupyter-widgets/controls"}},"3040fc326fb14b15a9df1f73ac1df872":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"b56a4f4aea4e4125b30b44c47fa59bcd":{"model_module":"@jupyter-widgets/controls","model_name":"HBoxModel","state":{"_view_name":"HBoxView","_dom_classes":[],"_model_name":"HBoxModel","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.5.0","box_style":"","layout":"IPY_MODEL_5b6d5116f19545ac84f11cc5cd95b1ff","_model_module":"@jupyter-widgets/controls","children":["IPY_MODEL_5436d2df1e0f46afb492a03f849eb52c","IPY_MODEL_de6a5d30e10841fe862c5117515a615e"]}},"5b6d5116f19545ac84f11cc5cd95b1ff":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"5436d2df1e0f46afb492a03f849eb52c":{"model_module":"@jupyter-widgets/controls","model_name":"FloatProgressModel","state":{"_view_name":"ProgressView","style":"IPY_MODEL_3ef1000b6c74494b8c9671b6d4240387","_dom_classes":[],"description":"100%","_model_name":"FloatProgressModel","bar_style":"success","max":14,"_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":14,"_view_count":null,"_view_module_version":"1.5.0","orientation":"horizontal","min":0,"description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_1a0f644abb4b422fa4b689baa147a4b4"}},"de6a5d30e10841fe862c5117515a615e":{"model_module":"@jupyter-widgets/controls","model_name":"HTMLModel","state":{"_view_name":"HTMLView","style":"IPY_MODEL_3afed851917b4e67822ed136d6d8f0d9","_dom_classes":[],"description":"","_model_name":"HTMLModel","placeholder":"​","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":" 14/14 [00:12&lt;00:00,  1.14 batch/s]","_view_count":null,"_view_module_version":"1.5.0","description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_d93243760bfc432bbd1c1952de2177c0"}},"3ef1000b6c74494b8c9671b6d4240387":{"model_module":"@jupyter-widgets/controls","model_name":"ProgressStyleModel","state":{"_view_name":"StyleView","_model_name":"ProgressStyleModel","description_width":"initial","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","bar_color":null,"_model_module":"@jupyter-widgets/controls"}},"1a0f644abb4b422fa4b689baa147a4b4":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"3afed851917b4e67822ed136d6d8f0d9":{"model_module":"@jupyter-widgets/controls","model_name":"DescriptionStyleModel","state":{"_view_name":"StyleView","_model_name":"DescriptionStyleModel","description_width":"","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","_model_module":"@jupyter-widgets/controls"}},"d93243760bfc432bbd1c1952de2177c0":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"c1005321f2dd4114bc203299b14edd55":{"model_module":"@jupyter-widgets/controls","model_name":"HBoxModel","state":{"_view_name":"HBoxView","_dom_classes":[],"_model_name":"HBoxModel","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.5.0","box_style":"","layout":"IPY_MODEL_50f33aaa6c6948738d50bd913ff5e2ad","_model_module":"@jupyter-widgets/controls","children":["IPY_MODEL_860c4d8da93e45b392ab36172abaacfe","IPY_MODEL_0e65ba43e6a04225b9725e9a2cedf272"]}},"50f33aaa6c6948738d50bd913ff5e2ad":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"860c4d8da93e45b392ab36172abaacfe":{"model_module":"@jupyter-widgets/controls","model_name":"FloatProgressModel","state":{"_view_name":"ProgressView","style":"IPY_MODEL_b3e148d0e2ca4e77bb8773fd15c88747","_dom_classes":[],"description":"100%","_model_name":"FloatProgressModel","bar_style":"success","max":134931,"_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":134931,"_view_count":null,"_view_module_version":"1.5.0","orientation":"horizontal","min":0,"description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_69238751767f4ed19b19b26f9e1a61bf"}},"0e65ba43e6a04225b9725e9a2cedf272":{"model_module":"@jupyter-widgets/controls","model_name":"HTMLModel","state":{"_view_name":"HTMLView","style":"IPY_MODEL_e77af656852f4f60aadb9631678ae138","_dom_classes":[],"description":"","_model_name":"HTMLModel","placeholder":"​","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":" 134931/134931 [03:44&lt;00:00, 602.14 ORF/s]","_view_count":null,"_view_module_version":"1.5.0","description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_26d6d73041ab449fa335576dbf933cf8"}},"b3e148d0e2ca4e77bb8773fd15c88747":{"model_module":"@jupyter-widgets/controls","model_name":"ProgressStyleModel","state":{"_view_name":"StyleView","_model_name":"ProgressStyleModel","description_width":"initial","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","bar_color":null,"_model_module":"@jupyter-widgets/controls"}},"69238751767f4ed19b19b26f9e1a61bf":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"e77af656852f4f60aadb9631678ae138":{"model_module":"@jupyter-widgets/controls","model_name":"DescriptionStyleModel","state":{"_view_name":"StyleView","_model_name":"DescriptionStyleModel","description_width":"","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","_model_module":"@jupyter-widgets/controls"}},"26d6d73041ab449fa335576dbf933cf8":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"7f4a050d4b8742daac16ef0ad9f09807":{"model_module":"@jupyter-widgets/controls","model_name":"HBoxModel","state":{"_view_name":"HBoxView","_dom_classes":[],"_model_name":"HBoxModel","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.5.0","box_style":"","layout":"IPY_MODEL_30f67e017b6740b0b4a04e7fde06eae3","_model_module":"@jupyter-widgets/controls","children":["IPY_MODEL_af4641e70222411b97c81d0e7c88a302","IPY_MODEL_e8fafa172d8f4751bfa18334db466c26"]}},"30f67e017b6740b0b4a04e7fde06eae3":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"af4641e70222411b97c81d0e7c88a302":{"model_module":"@jupyter-widgets/controls","model_name":"FloatProgressModel","state":{"_view_name":"ProgressView","style":"IPY_MODEL_adceb29c304746f898e708d042b85575","_dom_classes":[],"description":"100%","_model_name":"FloatProgressModel","bar_style":"success","max":70,"_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":70,"_view_count":null,"_view_module_version":"1.5.0","orientation":"horizontal","min":0,"description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_83037ce554e14c8c977458e62c60e481"}},"e8fafa172d8f4751bfa18334db466c26":{"model_module":"@jupyter-widgets/controls","model_name":"HTMLModel","state":{"_view_name":"HTMLView","style":"IPY_MODEL_4fa630c008b745f99468331276cc83d5","_dom_classes":[],"description":"","_model_name":"HTMLModel","placeholder":"​","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":" 70/70 [00:28&lt;00:00,  2.50 batch/s]","_view_count":null,"_view_module_version":"1.5.0","description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_1e22872207b847f38ee7a715601142e2"}},"adceb29c304746f898e708d042b85575":{"model_module":"@jupyter-widgets/controls","model_name":"ProgressStyleModel","state":{"_view_name":"StyleView","_model_name":"ProgressStyleModel","description_width":"initial","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","bar_color":null,"_model_module":"@jupyter-widgets/controls"}},"83037ce554e14c8c977458e62c60e481":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"4fa630c008b745f99468331276cc83d5":{"model_module":"@jupyter-widgets/controls","model_name":"DescriptionStyleModel","state":{"_view_name":"StyleView","_model_name":"DescriptionStyleModel","description_width":"","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","_model_module":"@jupyter-widgets/controls"}},"1e22872207b847f38ee7a715601142e2":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"be0fe09d5db044368c3590f366fa4df6":{"model_module":"@jupyter-widgets/controls","model_name":"HBoxModel","state":{"_view_name":"HBoxView","_dom_classes":[],"_model_name":"HBoxModel","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.5.0","box_style":"","layout":"IPY_MODEL_a2ef668488b0424d90700855d3e302c5","_model_module":"@jupyter-widgets/controls","children":["IPY_MODEL_fdb7d11f3302431fa960798371ead030","IPY_MODEL_196e78d502ac450fac3d8a85c31f7a82"]}},"a2ef668488b0424d90700855d3e302c5":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"fdb7d11f3302431fa960798371ead030":{"model_module":"@jupyter-widgets/controls","model_name":"FloatProgressModel","state":{"_view_name":"ProgressView","style":"IPY_MODEL_7691217ae4a74f1daaee9f7088e0cfc1","_dom_classes":[],"description":"100%","_model_name":"FloatProgressModel","bar_style":"success","max":21,"_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":21,"_view_count":null,"_view_module_version":"1.5.0","orientation":"horizontal","min":0,"description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_0dbb447f0ab1452e83cb6ea2394c1147"}},"196e78d502ac450fac3d8a85c31f7a82":{"model_module":"@jupyter-widgets/controls","model_name":"HTMLModel","state":{"_view_name":"HTMLView","style":"IPY_MODEL_cd2b543b23b34f09913684c273be3639","_dom_classes":[],"description":"","_model_name":"HTMLModel","placeholder":"​","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":" 21/21 [00:21&lt;00:00,  1.00s/ batch]","_view_count":null,"_view_module_version":"1.5.0","description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_68590eae93d64d77bd5343884ba2027e"}},"7691217ae4a74f1daaee9f7088e0cfc1":{"model_module":"@jupyter-widgets/controls","model_name":"ProgressStyleModel","state":{"_view_name":"StyleView","_model_name":"ProgressStyleModel","description_width":"initial","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","bar_color":null,"_model_module":"@jupyter-widgets/controls"}},"0dbb447f0ab1452e83cb6ea2394c1147":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"cd2b543b23b34f09913684c273be3639":{"model_module":"@jupyter-widgets/controls","model_name":"DescriptionStyleModel","state":{"_view_name":"StyleView","_model_name":"DescriptionStyleModel","description_width":"","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","_model_module":"@jupyter-widgets/controls"}},"68590eae93d64d77bd5343884ba2027e":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"f8c70f32e2b84044b4b72d6ded8cb5f1":{"model_module":"@jupyter-widgets/controls","model_name":"HBoxModel","state":{"_view_name":"HBoxView","_dom_classes":[],"_model_name":"HBoxModel","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.5.0","box_style":"","layout":"IPY_MODEL_02992fa870a044d883987b60db7ee137","_model_module":"@jupyter-widgets/controls","children":["IPY_MODEL_e0d5c1df49cc42e78e56de9b6c6f18d2","IPY_MODEL_5e9e8daed9ae45cf91427e6676f9ba26"]}},"02992fa870a044d883987b60db7ee137":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"e0d5c1df49cc42e78e56de9b6c6f18d2":{"model_module":"@jupyter-widgets/controls","model_name":"FloatProgressModel","state":{"_view_name":"ProgressView","style":"IPY_MODEL_14f45256198e431c898f32434cf9c4c0","_dom_classes":[],"description":"100%","_model_name":"FloatProgressModel","bar_style":"success","max":209617,"_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":209617,"_view_count":null,"_view_module_version":"1.5.0","orientation":"horizontal","min":0,"description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_cd3036815f95411c901b277abf26066d"}},"5e9e8daed9ae45cf91427e6676f9ba26":{"model_module":"@jupyter-widgets/controls","model_name":"HTMLModel","state":{"_view_name":"HTMLView","style":"IPY_MODEL_49bed964498847378407c1006436cc84","_dom_classes":[],"description":"","_model_name":"HTMLModel","placeholder":"​","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":" 209617/209617 [04:19&lt;00:00, 807.74 ORF/s]","_view_count":null,"_view_module_version":"1.5.0","description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_8fdc280797b34e20b5c3b7b2691ddbae"}},"14f45256198e431c898f32434cf9c4c0":{"model_module":"@jupyter-widgets/controls","model_name":"ProgressStyleModel","state":{"_view_name":"StyleView","_model_name":"ProgressStyleModel","description_width":"initial","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","bar_color":null,"_model_module":"@jupyter-widgets/controls"}},"cd3036815f95411c901b277abf26066d":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"49bed964498847378407c1006436cc84":{"model_module":"@jupyter-widgets/controls","model_name":"DescriptionStyleModel","state":{"_view_name":"StyleView","_model_name":"DescriptionStyleModel","description_width":"","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","_model_module":"@jupyter-widgets/controls"}},"8fdc280797b34e20b5c3b7b2691ddbae":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"b6b4a12026d443bb93e1e812242210b3":{"model_module":"@jupyter-widgets/controls","model_name":"HBoxModel","state":{"_view_name":"HBoxView","_dom_classes":[],"_model_name":"HBoxModel","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.5.0","box_style":"","layout":"IPY_MODEL_47490d21cc584e6980c2b3ccc7f31252","_model_module":"@jupyter-widgets/controls","children":["IPY_MODEL_8ccf8f4fc5fd472bb5598be9abada985","IPY_MODEL_de59ede900fb4d2c8dea6f16ac58c4c9"]}},"47490d21cc584e6980c2b3ccc7f31252":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"8ccf8f4fc5fd472bb5598be9abada985":{"model_module":"@jupyter-widgets/controls","model_name":"FloatProgressModel","state":{"_view_name":"ProgressView","style":"IPY_MODEL_cbf16b4b8612418da193c248945b217c","_dom_classes":[],"description":"100%","_model_name":"FloatProgressModel","bar_style":"success","max":21,"_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":21,"_view_count":null,"_view_module_version":"1.5.0","orientation":"horizontal","min":0,"description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_36703bc738254e798ff58fd950a8286a"}},"de59ede900fb4d2c8dea6f16ac58c4c9":{"model_module":"@jupyter-widgets/controls","model_name":"HTMLModel","state":{"_view_name":"HTMLView","style":"IPY_MODEL_24227e837ea44c719b93feb8179037a7","_dom_classes":[],"description":"","_model_name":"HTMLModel","placeholder":"​","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":" 21/21 [02:54&lt;00:00,  8.32s/ batch]","_view_count":null,"_view_module_version":"1.5.0","description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_1b0551d95bf444ddb121e410559508d7"}},"cbf16b4b8612418da193c248945b217c":{"model_module":"@jupyter-widgets/controls","model_name":"ProgressStyleModel","state":{"_view_name":"StyleView","_model_name":"ProgressStyleModel","description_width":"initial","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","bar_color":null,"_model_module":"@jupyter-widgets/controls"}},"36703bc738254e798ff58fd950a8286a":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"24227e837ea44c719b93feb8179037a7":{"model_module":"@jupyter-widgets/controls","model_name":"DescriptionStyleModel","state":{"_view_name":"StyleView","_model_name":"DescriptionStyleModel","description_width":"","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","_model_module":"@jupyter-widgets/controls"}},"1b0551d95bf444ddb121e410559508d7":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"9e44a9be7c92461885888134efc64624":{"model_module":"@jupyter-widgets/controls","model_name":"HBoxModel","state":{"_view_name":"HBoxView","_dom_classes":[],"_model_name":"HBoxModel","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.5.0","box_style":"","layout":"IPY_MODEL_f2227ad0fa484f16abfbbc0c22f98dd6","_model_module":"@jupyter-widgets/controls","children":["IPY_MODEL_430e7e54da6e4e4eaf40895c577795a8","IPY_MODEL_cf8df510db74462bb06bc0e7be25daa2"]}},"f2227ad0fa484f16abfbbc0c22f98dd6":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"430e7e54da6e4e4eaf40895c577795a8":{"model_module":"@jupyter-widgets/controls","model_name":"FloatProgressModel","state":{"_view_name":"ProgressView","style":"IPY_MODEL_35f6390b61194edcab859a7467b0a890","_dom_classes":[],"description":"100%","_model_name":"FloatProgressModel","bar_style":"success","max":7,"_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":7,"_view_count":null,"_view_module_version":"1.5.0","orientation":"horizontal","min":0,"description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_bc68ce9abb5e421ba7ab60a9bb11af75"}},"cf8df510db74462bb06bc0e7be25daa2":{"model_module":"@jupyter-widgets/controls","model_name":"HTMLModel","state":{"_view_name":"HTMLView","style":"IPY_MODEL_620fa4ea256c42a1bc94cc2211960318","_dom_classes":[],"description":"","_model_name":"HTMLModel","placeholder":"​","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":" 7/7 [00:05&lt;00:00,  1.39 batch/s]","_view_count":null,"_view_module_version":"1.5.0","description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_5e95d076c35d4269b8ef962b21dc0c6d"}},"35f6390b61194edcab859a7467b0a890":{"model_module":"@jupyter-widgets/controls","model_name":"ProgressStyleModel","state":{"_view_name":"StyleView","_model_name":"ProgressStyleModel","description_width":"initial","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","bar_color":null,"_model_module":"@jupyter-widgets/controls"}},"bc68ce9abb5e421ba7ab60a9bb11af75":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"620fa4ea256c42a1bc94cc2211960318":{"model_module":"@jupyter-widgets/controls","model_name":"DescriptionStyleModel","state":{"_view_name":"StyleView","_model_name":"DescriptionStyleModel","description_width":"","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","_model_module":"@jupyter-widgets/controls"}},"5e95d076c35d4269b8ef962b21dc0c6d":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"2683050fae9240ad9710728db708861f":{"model_module":"@jupyter-widgets/controls","model_name":"HBoxModel","state":{"_view_name":"HBoxView","_dom_classes":[],"_model_name":"HBoxModel","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.5.0","box_style":"","layout":"IPY_MODEL_ec0531af1bc84131b61cc218eef02daf","_model_module":"@jupyter-widgets/controls","children":["IPY_MODEL_f4e89d51c9224936a5d839f36f2fd501","IPY_MODEL_40f1225169e24f4d8cec1f80fa696c39"]}},"ec0531af1bc84131b61cc218eef02daf":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"f4e89d51c9224936a5d839f36f2fd501":{"model_module":"@jupyter-widgets/controls","model_name":"FloatProgressModel","state":{"_view_name":"ProgressView","style":"IPY_MODEL_23c63c0618e843d2a3a4a44d7374bec7","_dom_classes":[],"description":"100%","_model_name":"FloatProgressModel","bar_style":"success","max":61429,"_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":61429,"_view_count":null,"_view_module_version":"1.5.0","orientation":"horizontal","min":0,"description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_59ea683f41694029b46b4f07872e533a"}},"40f1225169e24f4d8cec1f80fa696c39":{"model_module":"@jupyter-widgets/controls","model_name":"HTMLModel","state":{"_view_name":"HTMLView","style":"IPY_MODEL_dea5761846e2455596352aed5fba9602","_dom_classes":[],"description":"","_model_name":"HTMLModel","placeholder":"​","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":" 61429/61429 [02:43&lt;00:00, 376.70 ORF/s]","_view_count":null,"_view_module_version":"1.5.0","description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_c51932cdcc2f4f61b5879fbdf0e0ae4c"}},"23c63c0618e843d2a3a4a44d7374bec7":{"model_module":"@jupyter-widgets/controls","model_name":"ProgressStyleModel","state":{"_view_name":"StyleView","_model_name":"ProgressStyleModel","description_width":"initial","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","bar_color":null,"_model_module":"@jupyter-widgets/controls"}},"59ea683f41694029b46b4f07872e533a":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"dea5761846e2455596352aed5fba9602":{"model_module":"@jupyter-widgets/controls","model_name":"DescriptionStyleModel","state":{"_view_name":"StyleView","_model_name":"DescriptionStyleModel","description_width":"","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","_model_module":"@jupyter-widgets/controls"}},"c51932cdcc2f4f61b5879fbdf0e0ae4c":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"b2d6d7bd6bf648d6a740aa7a70402d1b":{"model_module":"@jupyter-widgets/controls","model_name":"HBoxModel","state":{"_view_name":"HBoxView","_dom_classes":[],"_model_name":"HBoxModel","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.5.0","box_style":"","layout":"IPY_MODEL_8f6dcbd988c249e5bf35d9757111d585","_model_module":"@jupyter-widgets/controls","children":["IPY_MODEL_dedec14dab614486a6feafecd990ec84","IPY_MODEL_86e34b070a4d42d7adf6762759f4407d"]}},"8f6dcbd988c249e5bf35d9757111d585":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"dedec14dab614486a6feafecd990ec84":{"model_module":"@jupyter-widgets/controls","model_name":"FloatProgressModel","state":{"_view_name":"ProgressView","style":"IPY_MODEL_c95e70d65dcc42cba9248e4334d1e176","_dom_classes":[],"description":"100%","_model_name":"FloatProgressModel","bar_style":"success","max":54,"_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":54,"_view_count":null,"_view_module_version":"1.5.0","orientation":"horizontal","min":0,"description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_01434d140eca410e8745e1215bf1fefb"}},"86e34b070a4d42d7adf6762759f4407d":{"model_module":"@jupyter-widgets/controls","model_name":"HTMLModel","state":{"_view_name":"HTMLView","style":"IPY_MODEL_48e82748c8bb43ba9ef99d0f707000d2","_dom_classes":[],"description":"","_model_name":"HTMLModel","placeholder":"​","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":" 54/54 [00:31&lt;00:00,  1.73 batch/s]","_view_count":null,"_view_module_version":"1.5.0","description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_25e94db2d34346eba1e0bc42a321ce23"}},"c95e70d65dcc42cba9248e4334d1e176":{"model_module":"@jupyter-widgets/controls","model_name":"ProgressStyleModel","state":{"_view_name":"StyleView","_model_name":"ProgressStyleModel","description_width":"initial","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","bar_color":null,"_model_module":"@jupyter-widgets/controls"}},"01434d140eca410e8745e1215bf1fefb":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"48e82748c8bb43ba9ef99d0f707000d2":{"model_module":"@jupyter-widgets/controls","model_name":"DescriptionStyleModel","state":{"_view_name":"StyleView","_model_name":"DescriptionStyleModel","description_width":"","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","_model_module":"@jupyter-widgets/controls"}},"25e94db2d34346eba1e0bc42a321ce23":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"b1a23333630047b0bffb03ba74441786":{"model_module":"@jupyter-widgets/controls","model_name":"HBoxModel","state":{"_view_name":"HBoxView","_dom_classes":[],"_model_name":"HBoxModel","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.5.0","box_style":"","layout":"IPY_MODEL_920ad935251c4972828fc6c3f043f132","_model_module":"@jupyter-widgets/controls","children":["IPY_MODEL_4f282684235d4a02bcc9ff54d1e34d60","IPY_MODEL_1a6c836b951642d59adaea033c94ea14"]}},"920ad935251c4972828fc6c3f043f132":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"4f282684235d4a02bcc9ff54d1e34d60":{"model_module":"@jupyter-widgets/controls","model_name":"FloatProgressModel","state":{"_view_name":"ProgressView","style":"IPY_MODEL_2ac9adc70f0b49c88a3b445e67aa312e","_dom_classes":[],"description":"100%","_model_name":"FloatProgressModel","bar_style":"success","max":17,"_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":17,"_view_count":null,"_view_module_version":"1.5.0","orientation":"horizontal","min":0,"description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_d01fa867e813486aa062608cd9adcd7a"}},"1a6c836b951642d59adaea033c94ea14":{"model_module":"@jupyter-widgets/controls","model_name":"HTMLModel","state":{"_view_name":"HTMLView","style":"IPY_MODEL_62579df95b834ece89a1b1d2996b117f","_dom_classes":[],"description":"","_model_name":"HTMLModel","placeholder":"​","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":" 17/17 [00:07&lt;00:00,  2.39 batch/s]","_view_count":null,"_view_module_version":"1.5.0","description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_41cd45b5d661474b9a4b8be7b0fa7339"}},"2ac9adc70f0b49c88a3b445e67aa312e":{"model_module":"@jupyter-widgets/controls","model_name":"ProgressStyleModel","state":{"_view_name":"StyleView","_model_name":"ProgressStyleModel","description_width":"initial","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","bar_color":null,"_model_module":"@jupyter-widgets/controls"}},"d01fa867e813486aa062608cd9adcd7a":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"62579df95b834ece89a1b1d2996b117f":{"model_module":"@jupyter-widgets/controls","model_name":"DescriptionStyleModel","state":{"_view_name":"StyleView","_model_name":"DescriptionStyleModel","description_width":"","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","_model_module":"@jupyter-widgets/controls"}},"41cd45b5d661474b9a4b8be7b0fa7339":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"1a840f291ae745a19988416f7cc68b12":{"model_module":"@jupyter-widgets/controls","model_name":"HBoxModel","state":{"_view_name":"HBoxView","_dom_classes":[],"_model_name":"HBoxModel","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.5.0","box_style":"","layout":"IPY_MODEL_35a0d255c6664a2fb5ba937c4ba4782d","_model_module":"@jupyter-widgets/controls","children":["IPY_MODEL_eb336aba67d147c8a660ecfcfef01f21","IPY_MODEL_5c42b55981a348e3b90bee4ad176e767"]}},"35a0d255c6664a2fb5ba937c4ba4782d":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"eb336aba67d147c8a660ecfcfef01f21":{"model_module":"@jupyter-widgets/controls","model_name":"FloatProgressModel","state":{"_view_name":"ProgressView","style":"IPY_MODEL_a1110b083757480f9af5c9b41caf07db","_dom_classes":[],"description":"100%","_model_name":"FloatProgressModel","bar_style":"success","max":161843,"_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":161843,"_view_count":null,"_view_module_version":"1.5.0","orientation":"horizontal","min":0,"description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_20ec467f52b6418bba0272f54a956d14"}},"5c42b55981a348e3b90bee4ad176e767":{"model_module":"@jupyter-widgets/controls","model_name":"HTMLModel","state":{"_view_name":"HTMLView","style":"IPY_MODEL_e5ddc57632c94731bceabd16b8ba2db2","_dom_classes":[],"description":"","_model_name":"HTMLModel","placeholder":"​","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":" 161843/161843 [04:23&lt;00:00, 614.70 ORF/s]","_view_count":null,"_view_module_version":"1.5.0","description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_cf6509d24a61446caf95ae6704d91d45"}},"a1110b083757480f9af5c9b41caf07db":{"model_module":"@jupyter-widgets/controls","model_name":"ProgressStyleModel","state":{"_view_name":"StyleView","_model_name":"ProgressStyleModel","description_width":"initial","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","bar_color":null,"_model_module":"@jupyter-widgets/controls"}},"20ec467f52b6418bba0272f54a956d14":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"e5ddc57632c94731bceabd16b8ba2db2":{"model_module":"@jupyter-widgets/controls","model_name":"DescriptionStyleModel","state":{"_view_name":"StyleView","_model_name":"DescriptionStyleModel","description_width":"","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","_model_module":"@jupyter-widgets/controls"}},"cf6509d24a61446caf95ae6704d91d45":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"6aefc36b55a6463792803e9f96b30729":{"model_module":"@jupyter-widgets/controls","model_name":"HBoxModel","state":{"_view_name":"HBoxView","_dom_classes":[],"_model_name":"HBoxModel","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.5.0","box_style":"","layout":"IPY_MODEL_5b96c43215f145a582e79644743a06f0","_model_module":"@jupyter-widgets/controls","children":["IPY_MODEL_179d24996e9d4a79815d0f72bef79e2d","IPY_MODEL_6a3b3a05b4434b1b99c8dbee8d2e0a17"]}},"5b96c43215f145a582e79644743a06f0":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"179d24996e9d4a79815d0f72bef79e2d":{"model_module":"@jupyter-widgets/controls","model_name":"FloatProgressModel","state":{"_view_name":"ProgressView","style":"IPY_MODEL_1dceb63e845a4332b080ffee7246b8d5","_dom_classes":[],"description":"100%","_model_name":"FloatProgressModel","bar_style":"success","max":14,"_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":14,"_view_count":null,"_view_module_version":"1.5.0","orientation":"horizontal","min":0,"description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_dee7b2fa51ab4bdeb9f7f8421a1fddbe"}},"6a3b3a05b4434b1b99c8dbee8d2e0a17":{"model_module":"@jupyter-widgets/controls","model_name":"HTMLModel","state":{"_view_name":"HTMLView","style":"IPY_MODEL_69c0856a314b422f9cb2b9a70887199d","_dom_classes":[],"description":"","_model_name":"HTMLModel","placeholder":"​","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":" 14/14 [02:35&lt;00:00, 11.13s/ batch]","_view_count":null,"_view_module_version":"1.5.0","description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_ffba12776cef44129c9936ff23c7e97d"}},"1dceb63e845a4332b080ffee7246b8d5":{"model_module":"@jupyter-widgets/controls","model_name":"ProgressStyleModel","state":{"_view_name":"StyleView","_model_name":"ProgressStyleModel","description_width":"initial","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","bar_color":null,"_model_module":"@jupyter-widgets/controls"}},"dee7b2fa51ab4bdeb9f7f8421a1fddbe":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"69c0856a314b422f9cb2b9a70887199d":{"model_module":"@jupyter-widgets/controls","model_name":"DescriptionStyleModel","state":{"_view_name":"StyleView","_model_name":"DescriptionStyleModel","description_width":"","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","_model_module":"@jupyter-widgets/controls"}},"ffba12776cef44129c9936ff23c7e97d":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"b6115ec16a4e446f873fb9861cb2759e":{"model_module":"@jupyter-widgets/controls","model_name":"HBoxModel","state":{"_view_name":"HBoxView","_dom_classes":[],"_model_name":"HBoxModel","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.5.0","box_style":"","layout":"IPY_MODEL_d213c1b771354e36b6f2e3e4ca593384","_model_module":"@jupyter-widgets/controls","children":["IPY_MODEL_f9a4fd81ab944a47affba0901dbaee39","IPY_MODEL_8ecd206a1e164eb0b3832c307f4e0eca"]}},"d213c1b771354e36b6f2e3e4ca593384":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"f9a4fd81ab944a47affba0901dbaee39":{"model_module":"@jupyter-widgets/controls","model_name":"FloatProgressModel","state":{"_view_name":"ProgressView","style":"IPY_MODEL_17ed9cc6b1504c23a637b32d340ece0c","_dom_classes":[],"description":"100%","_model_name":"FloatProgressModel","bar_style":"success","max":5,"_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":5,"_view_count":null,"_view_module_version":"1.5.0","orientation":"horizontal","min":0,"description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_4f624b707cc7475faaead91b508215ef"}},"8ecd206a1e164eb0b3832c307f4e0eca":{"model_module":"@jupyter-widgets/controls","model_name":"HTMLModel","state":{"_view_name":"HTMLView","style":"IPY_MODEL_fedcc602da9e4d33bef4c7d4efe2cdfd","_dom_classes":[],"description":"","_model_name":"HTMLModel","placeholder":"​","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":" 5/5 [02:28&lt;00:00, 29.78s/ batch]","_view_count":null,"_view_module_version":"1.5.0","description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_6db1af3e9b734055a1c105086c8a2553"}},"17ed9cc6b1504c23a637b32d340ece0c":{"model_module":"@jupyter-widgets/controls","model_name":"ProgressStyleModel","state":{"_view_name":"StyleView","_model_name":"ProgressStyleModel","description_width":"initial","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","bar_color":null,"_model_module":"@jupyter-widgets/controls"}},"4f624b707cc7475faaead91b508215ef":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"fedcc602da9e4d33bef4c7d4efe2cdfd":{"model_module":"@jupyter-widgets/controls","model_name":"DescriptionStyleModel","state":{"_view_name":"StyleView","_model_name":"DescriptionStyleModel","description_width":"","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","_model_module":"@jupyter-widgets/controls"}},"6db1af3e9b734055a1c105086c8a2553":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"3e59f5aad0ba4248a195040d52f58961":{"model_module":"@jupyter-widgets/controls","model_name":"HBoxModel","state":{"_view_name":"HBoxView","_dom_classes":[],"_model_name":"HBoxModel","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.5.0","box_style":"","layout":"IPY_MODEL_6b82460eb62e48199bc7acc134b6e478","_model_module":"@jupyter-widgets/controls","children":["IPY_MODEL_9b100766f0b44471a5415e57c6625385","IPY_MODEL_c4307e1712e14e179c5360c7813bc031"]}},"6b82460eb62e48199bc7acc134b6e478":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"9b100766f0b44471a5415e57c6625385":{"model_module":"@jupyter-widgets/controls","model_name":"FloatProgressModel","state":{"_view_name":"ProgressView","style":"IPY_MODEL_2551ae79e3354d27a055f4bcea6ade0e","_dom_classes":[],"description":"100%","_model_name":"FloatProgressModel","bar_style":"success","max":41193,"_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":41193,"_view_count":null,"_view_module_version":"1.5.0","orientation":"horizontal","min":0,"description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_9ef7da027d174b8191eece7d8640b4bd"}},"c4307e1712e14e179c5360c7813bc031":{"model_module":"@jupyter-widgets/controls","model_name":"HTMLModel","state":{"_view_name":"HTMLView","style":"IPY_MODEL_c5796767d0ed4bad8d2206958405b886","_dom_classes":[],"description":"","_model_name":"HTMLModel","placeholder":"​","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":" 41193/41193 [02:26&lt;00:00, 281.87 ORF/s]","_view_count":null,"_view_module_version":"1.5.0","description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_ba34135ae45a4c3e9ac905a628295ddd"}},"2551ae79e3354d27a055f4bcea6ade0e":{"model_module":"@jupyter-widgets/controls","model_name":"ProgressStyleModel","state":{"_view_name":"StyleView","_model_name":"ProgressStyleModel","description_width":"initial","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","bar_color":null,"_model_module":"@jupyter-widgets/controls"}},"9ef7da027d174b8191eece7d8640b4bd":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"c5796767d0ed4bad8d2206958405b886":{"model_module":"@jupyter-widgets/controls","model_name":"DescriptionStyleModel","state":{"_view_name":"StyleView","_model_name":"DescriptionStyleModel","description_width":"","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","_model_module":"@jupyter-widgets/controls"}},"ba34135ae45a4c3e9ac905a628295ddd":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"3023cac4c45946a290dc588f01dc67dc":{"model_module":"@jupyter-widgets/controls","model_name":"HBoxModel","state":{"_view_name":"HBoxView","_dom_classes":[],"_model_name":"HBoxModel","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.5.0","box_style":"","layout":"IPY_MODEL_db154e83b72942e3b3b9c6fc95d19317","_model_module":"@jupyter-widgets/controls","children":["IPY_MODEL_4b820f05742a48529dc96b749095dcae","IPY_MODEL_b83e89e7625a40e48b295cff2a8ccfdf"]}},"db154e83b72942e3b3b9c6fc95d19317":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"4b820f05742a48529dc96b749095dcae":{"model_module":"@jupyter-widgets/controls","model_name":"FloatProgressModel","state":{"_view_name":"ProgressView","style":"IPY_MODEL_83ed1572e0cd452bad9c09d513a60a4b","_dom_classes":[],"description":"100%","_model_name":"FloatProgressModel","bar_style":"success","max":80,"_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":80,"_view_count":null,"_view_module_version":"1.5.0","orientation":"horizontal","min":0,"description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_8e9958779d5b4ec2b68b3392187d4cec"}},"b83e89e7625a40e48b295cff2a8ccfdf":{"model_module":"@jupyter-widgets/controls","model_name":"HTMLModel","state":{"_view_name":"HTMLView","style":"IPY_MODEL_a6e3ffdc094b4ebab75e2d15af58900c","_dom_classes":[],"description":"","_model_name":"HTMLModel","placeholder":"​","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":" 80/80 [00:29&lt;00:00,  2.69 batch/s]","_view_count":null,"_view_module_version":"1.5.0","description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_5132f80498ba445fa9dca4ea68a837a9"}},"83ed1572e0cd452bad9c09d513a60a4b":{"model_module":"@jupyter-widgets/controls","model_name":"ProgressStyleModel","state":{"_view_name":"StyleView","_model_name":"ProgressStyleModel","description_width":"initial","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","bar_color":null,"_model_module":"@jupyter-widgets/controls"}},"8e9958779d5b4ec2b68b3392187d4cec":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"a6e3ffdc094b4ebab75e2d15af58900c":{"model_module":"@jupyter-widgets/controls","model_name":"DescriptionStyleModel","state":{"_view_name":"StyleView","_model_name":"DescriptionStyleModel","description_width":"","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","_model_module":"@jupyter-widgets/controls"}},"5132f80498ba445fa9dca4ea68a837a9":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"3aae4efe6f1a43ba9f273945b0ab88ce":{"model_module":"@jupyter-widgets/controls","model_name":"HBoxModel","state":{"_view_name":"HBoxView","_dom_classes":[],"_model_name":"HBoxModel","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.5.0","box_style":"","layout":"IPY_MODEL_850a77392d76484da6ca8caab08a91b5","_model_module":"@jupyter-widgets/controls","children":["IPY_MODEL_f322e6bbe8644e9893db095ef3909a05","IPY_MODEL_d4025c085c7c4ee38dae66327a1bb28c"]}},"850a77392d76484da6ca8caab08a91b5":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"f322e6bbe8644e9893db095ef3909a05":{"model_module":"@jupyter-widgets/controls","model_name":"FloatProgressModel","state":{"_view_name":"ProgressView","style":"IPY_MODEL_52eeee61c14f401db42f6bc246a2b4f1","_dom_classes":[],"description":"100%","_model_name":"FloatProgressModel","bar_style":"success","max":24,"_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":24,"_view_count":null,"_view_module_version":"1.5.0","orientation":"horizontal","min":0,"description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_ff5ba691d930485abfb9961320fd3456"}},"d4025c085c7c4ee38dae66327a1bb28c":{"model_module":"@jupyter-widgets/controls","model_name":"HTMLModel","state":{"_view_name":"HTMLView","style":"IPY_MODEL_70ec28133b6c4e91abc7186826598fee","_dom_classes":[],"description":"","_model_name":"HTMLModel","placeholder":"​","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":" 24/24 [00:11&lt;00:00,  2.05 batch/s]","_view_count":null,"_view_module_version":"1.5.0","description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_5f1467ec1f5e440294a49f58c639a07b"}},"52eeee61c14f401db42f6bc246a2b4f1":{"model_module":"@jupyter-widgets/controls","model_name":"ProgressStyleModel","state":{"_view_name":"StyleView","_model_name":"ProgressStyleModel","description_width":"initial","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","bar_color":null,"_model_module":"@jupyter-widgets/controls"}},"ff5ba691d930485abfb9961320fd3456":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"70ec28133b6c4e91abc7186826598fee":{"model_module":"@jupyter-widgets/controls","model_name":"DescriptionStyleModel","state":{"_view_name":"StyleView","_model_name":"DescriptionStyleModel","description_width":"","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","_model_module":"@jupyter-widgets/controls"}},"5f1467ec1f5e440294a49f58c639a07b":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"1c3ef650221348099251f50de17d63d2":{"model_module":"@jupyter-widgets/controls","model_name":"HBoxModel","state":{"_view_name":"HBoxView","_dom_classes":[],"_model_name":"HBoxModel","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.5.0","box_style":"","layout":"IPY_MODEL_7013ac84768741228feec42b38732970","_model_module":"@jupyter-widgets/controls","children":["IPY_MODEL_5c810254686244c994d95b5111e25b38","IPY_MODEL_31ff1d5edd974f5ea1f854b88a9332eb"]}},"7013ac84768741228feec42b38732970":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"5c810254686244c994d95b5111e25b38":{"model_module":"@jupyter-widgets/controls","model_name":"FloatProgressModel","state":{"_view_name":"ProgressView","style":"IPY_MODEL_866a9f3ed335486bac2bbfbbaedfecba","_dom_classes":[],"description":"100%","_model_name":"FloatProgressModel","bar_style":"success","max":238297,"_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":238297,"_view_count":null,"_view_module_version":"1.5.0","orientation":"horizontal","min":0,"description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_c6f3a5c6b313451995e3ea41b8bf9296"}},"31ff1d5edd974f5ea1f854b88a9332eb":{"model_module":"@jupyter-widgets/controls","model_name":"HTMLModel","state":{"_view_name":"HTMLView","style":"IPY_MODEL_f67fa5fc5bd347e394c2010687a7724e","_dom_classes":[],"description":"","_model_name":"HTMLModel","placeholder":"​","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":" 238297/238297 [05:12&lt;00:00, 761.97 ORF/s]","_view_count":null,"_view_module_version":"1.5.0","description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_957c77360e0948038ff45e34fb230df0"}},"866a9f3ed335486bac2bbfbbaedfecba":{"model_module":"@jupyter-widgets/controls","model_name":"ProgressStyleModel","state":{"_view_name":"StyleView","_model_name":"ProgressStyleModel","description_width":"initial","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","bar_color":null,"_model_module":"@jupyter-widgets/controls"}},"c6f3a5c6b313451995e3ea41b8bf9296":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"f67fa5fc5bd347e394c2010687a7724e":{"model_module":"@jupyter-widgets/controls","model_name":"DescriptionStyleModel","state":{"_view_name":"StyleView","_model_name":"DescriptionStyleModel","description_width":"","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","_model_module":"@jupyter-widgets/controls"}},"957c77360e0948038ff45e34fb230df0":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"2fced687a73844b290a7cc87b85a8ace":{"model_module":"@jupyter-widgets/controls","model_name":"HBoxModel","state":{"_view_name":"HBoxView","_dom_classes":[],"_model_name":"HBoxModel","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.5.0","box_style":"","layout":"IPY_MODEL_58b15c95a28a46ca9ba8a16236839ff6","_model_module":"@jupyter-widgets/controls","children":["IPY_MODEL_8091d47ab1ef4610ae49a0414531b9cf","IPY_MODEL_fafcccf5e0774bd0a478ce5b5a323351"]}},"58b15c95a28a46ca9ba8a16236839ff6":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"8091d47ab1ef4610ae49a0414531b9cf":{"model_module":"@jupyter-widgets/controls","model_name":"FloatProgressModel","state":{"_view_name":"ProgressView","style":"IPY_MODEL_558228d59740442e832d40534ea823d3","_dom_classes":[],"description":"100%","_model_name":"FloatProgressModel","bar_style":"success","max":20,"_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":20,"_view_count":null,"_view_module_version":"1.5.0","orientation":"horizontal","min":0,"description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_0dabca880b60497980042d730896f8e9"}},"fafcccf5e0774bd0a478ce5b5a323351":{"model_module":"@jupyter-widgets/controls","model_name":"HTMLModel","state":{"_view_name":"HTMLView","style":"IPY_MODEL_28424d5d85384c9e86c3a9634965916f","_dom_classes":[],"description":"","_model_name":"HTMLModel","placeholder":"​","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":" 20/20 [00:06&lt;00:00,  3.07 batch/s]","_view_count":null,"_view_module_version":"1.5.0","description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_99210e7680a044468ba6677849f4c753"}},"558228d59740442e832d40534ea823d3":{"model_module":"@jupyter-widgets/controls","model_name":"ProgressStyleModel","state":{"_view_name":"StyleView","_model_name":"ProgressStyleModel","description_width":"initial","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","bar_color":null,"_model_module":"@jupyter-widgets/controls"}},"0dabca880b60497980042d730896f8e9":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"28424d5d85384c9e86c3a9634965916f":{"model_module":"@jupyter-widgets/controls","model_name":"DescriptionStyleModel","state":{"_view_name":"StyleView","_model_name":"DescriptionStyleModel","description_width":"","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","_model_module":"@jupyter-widgets/controls"}},"99210e7680a044468ba6677849f4c753":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"82b536966beb414699990fb2ff46d4c6":{"model_module":"@jupyter-widgets/controls","model_name":"HBoxModel","state":{"_view_name":"HBoxView","_dom_classes":[],"_model_name":"HBoxModel","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.5.0","box_style":"","layout":"IPY_MODEL_83976e90324d4c6b90173634662ce966","_model_module":"@jupyter-widgets/controls","children":["IPY_MODEL_71053655895648488ae1d0d4248c9ff5","IPY_MODEL_2ee02607aaff4a7eaaa92a6fdcde436d"]}},"83976e90324d4c6b90173634662ce966":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"71053655895648488ae1d0d4248c9ff5":{"model_module":"@jupyter-widgets/controls","model_name":"FloatProgressModel","state":{"_view_name":"ProgressView","style":"IPY_MODEL_3fa63f1c6138400ba6ad6c709ee37f79","_dom_classes":[],"description":"100%","_model_name":"FloatProgressModel","bar_style":"success","max":6,"_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":6,"_view_count":null,"_view_module_version":"1.5.0","orientation":"horizontal","min":0,"description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_9cef327d2b2c4a51a898a5d154b91556"}},"2ee02607aaff4a7eaaa92a6fdcde436d":{"model_module":"@jupyter-widgets/controls","model_name":"HTMLModel","state":{"_view_name":"HTMLView","style":"IPY_MODEL_c720054a5b014b14a97fa007defb4ad2","_dom_classes":[],"description":"","_model_name":"HTMLModel","placeholder":"​","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":" 6/6 [03:40&lt;00:00, 36.74s/ batch]","_view_count":null,"_view_module_version":"1.5.0","description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_742d81128cf646bc99d99c4f943f92ea"}},"3fa63f1c6138400ba6ad6c709ee37f79":{"model_module":"@jupyter-widgets/controls","model_name":"ProgressStyleModel","state":{"_view_name":"StyleView","_model_name":"ProgressStyleModel","description_width":"initial","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","bar_color":null,"_model_module":"@jupyter-widgets/controls"}},"9cef327d2b2c4a51a898a5d154b91556":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"c720054a5b014b14a97fa007defb4ad2":{"model_module":"@jupyter-widgets/controls","model_name":"DescriptionStyleModel","state":{"_view_name":"StyleView","_model_name":"DescriptionStyleModel","description_width":"","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","_model_module":"@jupyter-widgets/controls"}},"742d81128cf646bc99d99c4f943f92ea":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"ef69d21fcd854d6bb5cb864741b0f7af":{"model_module":"@jupyter-widgets/controls","model_name":"HBoxModel","state":{"_view_name":"HBoxView","_dom_classes":[],"_model_name":"HBoxModel","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.5.0","box_style":"","layout":"IPY_MODEL_c128377ac5a044edaacc98abe66c6ca0","_model_module":"@jupyter-widgets/controls","children":["IPY_MODEL_933c33a4499646778a679d3d125e4aa0","IPY_MODEL_445d985206484b2887f4f275c0dfee0f"]}},"c128377ac5a044edaacc98abe66c6ca0":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"933c33a4499646778a679d3d125e4aa0":{"model_module":"@jupyter-widgets/controls","model_name":"FloatProgressModel","state":{"_view_name":"ProgressView","style":"IPY_MODEL_958337241be84d56848c10d7d4f06233","_dom_classes":[],"description":"100%","_model_name":"FloatProgressModel","bar_style":"success","max":56484,"_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":56484,"_view_count":null,"_view_module_version":"1.5.0","orientation":"horizontal","min":0,"description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_b243e09747034abb9de8f998b9cfbaed"}},"445d985206484b2887f4f275c0dfee0f":{"model_module":"@jupyter-widgets/controls","model_name":"HTMLModel","state":{"_view_name":"HTMLView","style":"IPY_MODEL_79b53f43d4194fcc90350d3dd79fd7bf","_dom_classes":[],"description":"","_model_name":"HTMLModel","placeholder":"​","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":" 56484/56484 [03:36&lt;00:00, 260.97 ORF/s]","_view_count":null,"_view_module_version":"1.5.0","description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_686e063ff3d5422d8a4a3f08553e9b4e"}},"958337241be84d56848c10d7d4f06233":{"model_module":"@jupyter-widgets/controls","model_name":"ProgressStyleModel","state":{"_view_name":"StyleView","_model_name":"ProgressStyleModel","description_width":"initial","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","bar_color":null,"_model_module":"@jupyter-widgets/controls"}},"b243e09747034abb9de8f998b9cfbaed":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"79b53f43d4194fcc90350d3dd79fd7bf":{"model_module":"@jupyter-widgets/controls","model_name":"DescriptionStyleModel","state":{"_view_name":"StyleView","_model_name":"DescriptionStyleModel","description_width":"","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","_model_module":"@jupyter-widgets/controls"}},"686e063ff3d5422d8a4a3f08553e9b4e":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"f13b81222bfa4e74a80096f3e476fe79":{"model_module":"@jupyter-widgets/controls","model_name":"HBoxModel","state":{"_view_name":"HBoxView","_dom_classes":[],"_model_name":"HBoxModel","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.5.0","box_style":"","layout":"IPY_MODEL_32bd9ec427ff438484902ba44ccafe00","_model_module":"@jupyter-widgets/controls","children":["IPY_MODEL_7454a6c7309a4b6ca2e559101b6853de","IPY_MODEL_13dcdd9a6b254306b4647a78d65b4ca2"]}},"32bd9ec427ff438484902ba44ccafe00":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"7454a6c7309a4b6ca2e559101b6853de":{"model_module":"@jupyter-widgets/controls","model_name":"FloatProgressModel","state":{"_view_name":"ProgressView","style":"IPY_MODEL_36096e01d4b54ce38b6ffd257ae8bc6a","_dom_classes":[],"description":"100%","_model_name":"FloatProgressModel","bar_style":"success","max":2629,"_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":2629,"_view_count":null,"_view_module_version":"1.5.0","orientation":"horizontal","min":0,"description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_8c2818f43bfb4891a76e63d587159d93"}},"13dcdd9a6b254306b4647a78d65b4ca2":{"model_module":"@jupyter-widgets/controls","model_name":"HTMLModel","state":{"_view_name":"HTMLView","style":"IPY_MODEL_0ca96e00995a49448da38378fb21593f","_dom_classes":[],"description":"","_model_name":"HTMLModel","placeholder":"​","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":" 2629/2629 [01:12&lt;00:00, 36.24 ORF/s]","_view_count":null,"_view_module_version":"1.5.0","description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_77d9aa6c03b94b09ba45bf29e416a603"}},"36096e01d4b54ce38b6ffd257ae8bc6a":{"model_module":"@jupyter-widgets/controls","model_name":"ProgressStyleModel","state":{"_view_name":"StyleView","_model_name":"ProgressStyleModel","description_width":"initial","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","bar_color":null,"_model_module":"@jupyter-widgets/controls"}},"8c2818f43bfb4891a76e63d587159d93":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"0ca96e00995a49448da38378fb21593f":{"model_module":"@jupyter-widgets/controls","model_name":"DescriptionStyleModel","state":{"_view_name":"StyleView","_model_name":"DescriptionStyleModel","description_width":"","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","_model_module":"@jupyter-widgets/controls"}},"77d9aa6c03b94b09ba45bf29e416a603":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"10ee481c890b4dc4af099c17b074adaf":{"model_module":"@jupyter-widgets/controls","model_name":"HBoxModel","state":{"_view_name":"HBoxView","_dom_classes":[],"_model_name":"HBoxModel","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.5.0","box_style":"","layout":"IPY_MODEL_428e704de79a48cca7f9bb2a00dc2cf1","_model_module":"@jupyter-widgets/controls","children":["IPY_MODEL_517f7ca421404b14902f86e9835abe01","IPY_MODEL_66c657f697174c9c8ff983ecb70635d0"]}},"428e704de79a48cca7f9bb2a00dc2cf1":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"517f7ca421404b14902f86e9835abe01":{"model_module":"@jupyter-widgets/controls","model_name":"FloatProgressModel","state":{"_view_name":"ProgressView","style":"IPY_MODEL_c1c548a8273748ddabb2e21f20ad6c8a","_dom_classes":[],"description":"100%","_model_name":"FloatProgressModel","bar_style":"success","max":54,"_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":54,"_view_count":null,"_view_module_version":"1.5.0","orientation":"horizontal","min":0,"description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_a86103bb4b22439d8be1f8f6f0e538a4"}},"66c657f697174c9c8ff983ecb70635d0":{"model_module":"@jupyter-widgets/controls","model_name":"HTMLModel","state":{"_view_name":"HTMLView","style":"IPY_MODEL_2a56b80c189e4f11a418a9e7eedfa70b","_dom_classes":[],"description":"","_model_name":"HTMLModel","placeholder":"​","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":" 54/54 [00:31&lt;00:00,  1.70 batch/s]","_view_count":null,"_view_module_version":"1.5.0","description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_a441c365d1a949b1ba6ba3de1ebf705e"}},"c1c548a8273748ddabb2e21f20ad6c8a":{"model_module":"@jupyter-widgets/controls","model_name":"ProgressStyleModel","state":{"_view_name":"StyleView","_model_name":"ProgressStyleModel","description_width":"initial","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","bar_color":null,"_model_module":"@jupyter-widgets/controls"}},"a86103bb4b22439d8be1f8f6f0e538a4":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"2a56b80c189e4f11a418a9e7eedfa70b":{"model_module":"@jupyter-widgets/controls","model_name":"DescriptionStyleModel","state":{"_view_name":"StyleView","_model_name":"DescriptionStyleModel","description_width":"","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","_model_module":"@jupyter-widgets/controls"}},"a441c365d1a949b1ba6ba3de1ebf705e":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"a525832e1581473b9c905d43ac755765":{"model_module":"@jupyter-widgets/controls","model_name":"HBoxModel","state":{"_view_name":"HBoxView","_dom_classes":[],"_model_name":"HBoxModel","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.5.0","box_style":"","layout":"IPY_MODEL_33a3df8f23e74de688385ea6966d60a1","_model_module":"@jupyter-widgets/controls","children":["IPY_MODEL_75b3dfbfb19542ffb68441020cf857c7","IPY_MODEL_74745996703b475f836f23acd2cad6b9"]}},"33a3df8f23e74de688385ea6966d60a1":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"75b3dfbfb19542ffb68441020cf857c7":{"model_module":"@jupyter-widgets/controls","model_name":"FloatProgressModel","state":{"_view_name":"ProgressView","style":"IPY_MODEL_197d5ae29b8342a6857b1c9adcf19b1b","_dom_classes":[],"description":"100%","_model_name":"FloatProgressModel","bar_style":"success","max":16,"_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":16,"_view_count":null,"_view_module_version":"1.5.0","orientation":"horizontal","min":0,"description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_179d4f3613874cf9b3e8c13a1682109b"}},"74745996703b475f836f23acd2cad6b9":{"model_module":"@jupyter-widgets/controls","model_name":"HTMLModel","state":{"_view_name":"HTMLView","style":"IPY_MODEL_93943b016b354e5581cd4fd27331c588","_dom_classes":[],"description":"","_model_name":"HTMLModel","placeholder":"​","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":" 16/16 [00:07&lt;00:00,  2.19 batch/s]","_view_count":null,"_view_module_version":"1.5.0","description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_956d3adb191a4e4cb454e555076596e6"}},"197d5ae29b8342a6857b1c9adcf19b1b":{"model_module":"@jupyter-widgets/controls","model_name":"ProgressStyleModel","state":{"_view_name":"StyleView","_model_name":"ProgressStyleModel","description_width":"initial","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","bar_color":null,"_model_module":"@jupyter-widgets/controls"}},"179d4f3613874cf9b3e8c13a1682109b":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"93943b016b354e5581cd4fd27331c588":{"model_module":"@jupyter-widgets/controls","model_name":"DescriptionStyleModel","state":{"_view_name":"StyleView","_model_name":"DescriptionStyleModel","description_width":"","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","_model_module":"@jupyter-widgets/controls"}},"956d3adb191a4e4cb454e555076596e6":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"a52dda18eb02436aa05d93ee6cd2d293":{"model_module":"@jupyter-widgets/controls","model_name":"HBoxModel","state":{"_view_name":"HBoxView","_dom_classes":[],"_model_name":"HBoxModel","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.5.0","box_style":"","layout":"IPY_MODEL_ea4c7af5d8044a55824e277c1ae3b443","_model_module":"@jupyter-widgets/controls","children":["IPY_MODEL_08d519ba69e24bb6b9b4e71fd2ce25ca","IPY_MODEL_8f144fe3697b4db4b2064cb72da0962f"]}},"ea4c7af5d8044a55824e277c1ae3b443":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"08d519ba69e24bb6b9b4e71fd2ce25ca":{"model_module":"@jupyter-widgets/controls","model_name":"FloatProgressModel","state":{"_view_name":"ProgressView","style":"IPY_MODEL_fa2cb35f22404da5969d3a0dcd743a19","_dom_classes":[],"description":"100%","_model_name":"FloatProgressModel","bar_style":"success","max":159003,"_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":159003,"_view_count":null,"_view_module_version":"1.5.0","orientation":"horizontal","min":0,"description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_ed4699ec4a314447bf0a43c133462074"}},"8f144fe3697b4db4b2064cb72da0962f":{"model_module":"@jupyter-widgets/controls","model_name":"HTMLModel","state":{"_view_name":"HTMLView","style":"IPY_MODEL_a67056da37f74dceb4cf71e18ea862c3","_dom_classes":[],"description":"","_model_name":"HTMLModel","placeholder":"​","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":" 159003/159003 [03:37&lt;00:00, 730.07 ORF/s]","_view_count":null,"_view_module_version":"1.5.0","description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_e9be16f790634d6f9d16923902bca068"}},"fa2cb35f22404da5969d3a0dcd743a19":{"model_module":"@jupyter-widgets/controls","model_name":"ProgressStyleModel","state":{"_view_name":"StyleView","_model_name":"ProgressStyleModel","description_width":"initial","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","bar_color":null,"_model_module":"@jupyter-widgets/controls"}},"ed4699ec4a314447bf0a43c133462074":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"a67056da37f74dceb4cf71e18ea862c3":{"model_module":"@jupyter-widgets/controls","model_name":"DescriptionStyleModel","state":{"_view_name":"StyleView","_model_name":"DescriptionStyleModel","description_width":"","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","_model_module":"@jupyter-widgets/controls"}},"e9be16f790634d6f9d16923902bca068":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"d1d7312e6a8748c882ad3d46a918e629":{"model_module":"@jupyter-widgets/controls","model_name":"HBoxModel","state":{"_view_name":"HBoxView","_dom_classes":[],"_model_name":"HBoxModel","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.5.0","box_style":"","layout":"IPY_MODEL_aa34ebdec80c419ebf6222109f326b9a","_model_module":"@jupyter-widgets/controls","children":["IPY_MODEL_1ee5f73417bc4c0b9aa35c19fd757c89","IPY_MODEL_4fabe138882e4528a44eafb3d76cb6f6"]}},"aa34ebdec80c419ebf6222109f326b9a":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"1ee5f73417bc4c0b9aa35c19fd757c89":{"model_module":"@jupyter-widgets/controls","model_name":"FloatProgressModel","state":{"_view_name":"ProgressView","style":"IPY_MODEL_b6e9ee7cc4504326a3cb01f817e62119","_dom_classes":[],"description":"100%","_model_name":"FloatProgressModel","bar_style":"success","max":61,"_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":61,"_view_count":null,"_view_module_version":"1.5.0","orientation":"horizontal","min":0,"description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_12a02fba26da4df79049c025f99cf501"}},"4fabe138882e4528a44eafb3d76cb6f6":{"model_module":"@jupyter-widgets/controls","model_name":"HTMLModel","state":{"_view_name":"HTMLView","style":"IPY_MODEL_af8ebcecd3dc44e4b9650cf11f20ce8c","_dom_classes":[],"description":"","_model_name":"HTMLModel","placeholder":"​","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":" 61/61 [00:25&lt;00:00,  2.36 batch/s]","_view_count":null,"_view_module_version":"1.5.0","description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_b92a80cdfc4d4dc7a93ca6bc7109110c"}},"b6e9ee7cc4504326a3cb01f817e62119":{"model_module":"@jupyter-widgets/controls","model_name":"ProgressStyleModel","state":{"_view_name":"StyleView","_model_name":"ProgressStyleModel","description_width":"initial","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","bar_color":null,"_model_module":"@jupyter-widgets/controls"}},"12a02fba26da4df79049c025f99cf501":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"af8ebcecd3dc44e4b9650cf11f20ce8c":{"model_module":"@jupyter-widgets/controls","model_name":"DescriptionStyleModel","state":{"_view_name":"StyleView","_model_name":"DescriptionStyleModel","description_width":"","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","_model_module":"@jupyter-widgets/controls"}},"b92a80cdfc4d4dc7a93ca6bc7109110c":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"6a8fc129d6344bab9c879b3e1cb9b199":{"model_module":"@jupyter-widgets/controls","model_name":"HBoxModel","state":{"_view_name":"HBoxView","_dom_classes":[],"_model_name":"HBoxModel","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.5.0","box_style":"","layout":"IPY_MODEL_93142b9ba4b84a2990f1fd12a0d34dac","_model_module":"@jupyter-widgets/controls","children":["IPY_MODEL_d22c872269d041cdad7a6e0d9a63f5e5","IPY_MODEL_4cfd8a8b7ef64d06887ca1200ccd7c45"]}},"93142b9ba4b84a2990f1fd12a0d34dac":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"d22c872269d041cdad7a6e0d9a63f5e5":{"model_module":"@jupyter-widgets/controls","model_name":"FloatProgressModel","state":{"_view_name":"ProgressView","style":"IPY_MODEL_580e38767dd645ac8d4edff8dfbd1416","_dom_classes":[],"description":"100%","_model_name":"FloatProgressModel","bar_style":"success","max":19,"_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":19,"_view_count":null,"_view_module_version":"1.5.0","orientation":"horizontal","min":0,"description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_73fa5b856bde4235b032dda0b8c12722"}},"4cfd8a8b7ef64d06887ca1200ccd7c45":{"model_module":"@jupyter-widgets/controls","model_name":"HTMLModel","state":{"_view_name":"HTMLView","style":"IPY_MODEL_a143cbd2434a4e0985284f6eff645952","_dom_classes":[],"description":"","_model_name":"HTMLModel","placeholder":"​","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":" 19/19 [00:22&lt;00:00,  1.21s/ batch]","_view_count":null,"_view_module_version":"1.5.0","description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_bf4a5449b28d4598a4562a9621b68bb8"}},"580e38767dd645ac8d4edff8dfbd1416":{"model_module":"@jupyter-widgets/controls","model_name":"ProgressStyleModel","state":{"_view_name":"StyleView","_model_name":"ProgressStyleModel","description_width":"initial","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","bar_color":null,"_model_module":"@jupyter-widgets/controls"}},"73fa5b856bde4235b032dda0b8c12722":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"a143cbd2434a4e0985284f6eff645952":{"model_module":"@jupyter-widgets/controls","model_name":"DescriptionStyleModel","state":{"_view_name":"StyleView","_model_name":"DescriptionStyleModel","description_width":"","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","_model_module":"@jupyter-widgets/controls"}},"bf4a5449b28d4598a4562a9621b68bb8":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"11c0335cfc7c4a249343656f5f9b9d39":{"model_module":"@jupyter-widgets/controls","model_name":"HBoxModel","state":{"_view_name":"HBoxView","_dom_classes":[],"_model_name":"HBoxModel","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.5.0","box_style":"","layout":"IPY_MODEL_424a266b77b14c519c2dcf07723818e3","_model_module":"@jupyter-widgets/controls","children":["IPY_MODEL_6cd10c7bfc0a470f884dc3e17389314f","IPY_MODEL_baeb0f5ab7ce40abb8a174370c372610"]}},"424a266b77b14c519c2dcf07723818e3":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"6cd10c7bfc0a470f884dc3e17389314f":{"model_module":"@jupyter-widgets/controls","model_name":"FloatProgressModel","state":{"_view_name":"ProgressView","style":"IPY_MODEL_11794d1cd9dd45eca634d549e70f7c91","_dom_classes":[],"description":"100%","_model_name":"FloatProgressModel","bar_style":"success","max":182262,"_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":182262,"_view_count":null,"_view_module_version":"1.5.0","orientation":"horizontal","min":0,"description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_b3f20d08053647baaf6fccde5f3c313b"}},"baeb0f5ab7ce40abb8a174370c372610":{"model_module":"@jupyter-widgets/controls","model_name":"HTMLModel","state":{"_view_name":"HTMLView","style":"IPY_MODEL_19698a9a0a0b4c7d8fc65327a291a05a","_dom_classes":[],"description":"","_model_name":"HTMLModel","placeholder":"​","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":" 182262/182262 [03:57&lt;00:00, 766.26 ORF/s]","_view_count":null,"_view_module_version":"1.5.0","description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_4dd9ada0368f408a895a06460ea44ae9"}},"11794d1cd9dd45eca634d549e70f7c91":{"model_module":"@jupyter-widgets/controls","model_name":"ProgressStyleModel","state":{"_view_name":"StyleView","_model_name":"ProgressStyleModel","description_width":"initial","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","bar_color":null,"_model_module":"@jupyter-widgets/controls"}},"b3f20d08053647baaf6fccde5f3c313b":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"19698a9a0a0b4c7d8fc65327a291a05a":{"model_module":"@jupyter-widgets/controls","model_name":"DescriptionStyleModel","state":{"_view_name":"StyleView","_model_name":"DescriptionStyleModel","description_width":"","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","_model_module":"@jupyter-widgets/controls"}},"4dd9ada0368f408a895a06460ea44ae9":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"6b905dab19c24646ba210604f80e34e7":{"model_module":"@jupyter-widgets/controls","model_name":"HBoxModel","state":{"_view_name":"HBoxView","_dom_classes":[],"_model_name":"HBoxModel","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.5.0","box_style":"","layout":"IPY_MODEL_940b395e1b3c455d86468cc0712fee0f","_model_module":"@jupyter-widgets/controls","children":["IPY_MODEL_9a24d32bb6c44325a37d12f238357a8b","IPY_MODEL_ba5d8ada68ef44ce9e4c5c6512a7e293"]}},"940b395e1b3c455d86468cc0712fee0f":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"9a24d32bb6c44325a37d12f238357a8b":{"model_module":"@jupyter-widgets/controls","model_name":"FloatProgressModel","state":{"_view_name":"ProgressView","style":"IPY_MODEL_8cda377991624527abcff22ef6024408","_dom_classes":[],"description":"100%","_model_name":"FloatProgressModel","bar_style":"success","max":32,"_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":32,"_view_count":null,"_view_module_version":"1.5.0","orientation":"horizontal","min":0,"description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_b3be1d29e8eb4607b98f03d5eb760732"}},"ba5d8ada68ef44ce9e4c5c6512a7e293":{"model_module":"@jupyter-widgets/controls","model_name":"HTMLModel","state":{"_view_name":"HTMLView","style":"IPY_MODEL_beb10e6ad42a4565a3049c98b085b74e","_dom_classes":[],"description":"","_model_name":"HTMLModel","placeholder":"​","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":" 32/32 [00:14&lt;00:00,  2.28 batch/s]","_view_count":null,"_view_module_version":"1.5.0","description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_f94bb66208a24e5eb030859a16484167"}},"8cda377991624527abcff22ef6024408":{"model_module":"@jupyter-widgets/controls","model_name":"ProgressStyleModel","state":{"_view_name":"StyleView","_model_name":"ProgressStyleModel","description_width":"initial","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","bar_color":null,"_model_module":"@jupyter-widgets/controls"}},"b3be1d29e8eb4607b98f03d5eb760732":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"beb10e6ad42a4565a3049c98b085b74e":{"model_module":"@jupyter-widgets/controls","model_name":"DescriptionStyleModel","state":{"_view_name":"StyleView","_model_name":"DescriptionStyleModel","description_width":"","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","_model_module":"@jupyter-widgets/controls"}},"f94bb66208a24e5eb030859a16484167":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"3b97a4d2484f4e75849d7bb799ce2367":{"model_module":"@jupyter-widgets/controls","model_name":"HBoxModel","state":{"_view_name":"HBoxView","_dom_classes":[],"_model_name":"HBoxModel","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.5.0","box_style":"","layout":"IPY_MODEL_b43ad6d87e7645d6b33f4496117e517c","_model_module":"@jupyter-widgets/controls","children":["IPY_MODEL_af4c9e2e37d34dd2a43dc040c219869f","IPY_MODEL_9efc006ccd4149e79f90709e0f688615"]}},"b43ad6d87e7645d6b33f4496117e517c":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"af4c9e2e37d34dd2a43dc040c219869f":{"model_module":"@jupyter-widgets/controls","model_name":"FloatProgressModel","state":{"_view_name":"ProgressView","style":"IPY_MODEL_650433116a7045f5bd7641ebbbf86f21","_dom_classes":[],"description":"100%","_model_name":"FloatProgressModel","bar_style":"success","max":10,"_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":10,"_view_count":null,"_view_module_version":"1.5.0","orientation":"horizontal","min":0,"description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_397326b34bcd4567a5f414b687e91a7f"}},"9efc006ccd4149e79f90709e0f688615":{"model_module":"@jupyter-widgets/controls","model_name":"HTMLModel","state":{"_view_name":"HTMLView","style":"IPY_MODEL_72c905c36cba46779435a4d0f6d42615","_dom_classes":[],"description":"","_model_name":"HTMLModel","placeholder":"​","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":" 10/10 [03:22&lt;00:00, 20.29s/ batch]","_view_count":null,"_view_module_version":"1.5.0","description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_f20d31635dd44c398ed2438a4a29a560"}},"650433116a7045f5bd7641ebbbf86f21":{"model_module":"@jupyter-widgets/controls","model_name":"ProgressStyleModel","state":{"_view_name":"StyleView","_model_name":"ProgressStyleModel","description_width":"initial","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","bar_color":null,"_model_module":"@jupyter-widgets/controls"}},"397326b34bcd4567a5f414b687e91a7f":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"72c905c36cba46779435a4d0f6d42615":{"model_module":"@jupyter-widgets/controls","model_name":"DescriptionStyleModel","state":{"_view_name":"StyleView","_model_name":"DescriptionStyleModel","description_width":"","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","_model_module":"@jupyter-widgets/controls"}},"f20d31635dd44c398ed2438a4a29a560":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"1c669e42110245e8a4da4bd82320bb12":{"model_module":"@jupyter-widgets/controls","model_name":"HBoxModel","state":{"_view_name":"HBoxView","_dom_classes":[],"_model_name":"HBoxModel","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.5.0","box_style":"","layout":"IPY_MODEL_893a3df7523a447c90666d96aba494d5","_model_module":"@jupyter-widgets/controls","children":["IPY_MODEL_236b6ed87dab4c178a2357c8cb2964da","IPY_MODEL_d0dad0ee4ba74b7d8aa7693c6a2bc2d5"]}},"893a3df7523a447c90666d96aba494d5":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"236b6ed87dab4c178a2357c8cb2964da":{"model_module":"@jupyter-widgets/controls","model_name":"FloatProgressModel","state":{"_view_name":"ProgressView","style":"IPY_MODEL_c47faa5bb4494928b08819f95222502e","_dom_classes":[],"description":"100%","_model_name":"FloatProgressModel","bar_style":"success","max":94855,"_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":94855,"_view_count":null,"_view_module_version":"1.5.0","orientation":"horizontal","min":0,"description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_bb39b5902e3b4f1399fa60cdec70d0fc"}},"d0dad0ee4ba74b7d8aa7693c6a2bc2d5":{"model_module":"@jupyter-widgets/controls","model_name":"HTMLModel","state":{"_view_name":"HTMLView","style":"IPY_MODEL_a3d4ca7173894f02990dc0384392999e","_dom_classes":[],"description":"","_model_name":"HTMLModel","placeholder":"​","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":" 94855/94855 [03:19&lt;00:00, 476.05 ORF/s]","_view_count":null,"_view_module_version":"1.5.0","description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_1891f81a709b45f99b118eb7f0574173"}},"c47faa5bb4494928b08819f95222502e":{"model_module":"@jupyter-widgets/controls","model_name":"ProgressStyleModel","state":{"_view_name":"StyleView","_model_name":"ProgressStyleModel","description_width":"initial","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","bar_color":null,"_model_module":"@jupyter-widgets/controls"}},"bb39b5902e3b4f1399fa60cdec70d0fc":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"a3d4ca7173894f02990dc0384392999e":{"model_module":"@jupyter-widgets/controls","model_name":"DescriptionStyleModel","state":{"_view_name":"StyleView","_model_name":"DescriptionStyleModel","description_width":"","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","_model_module":"@jupyter-widgets/controls"}},"1891f81a709b45f99b118eb7f0574173":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"96df2e0cbe3c4bf690bde1aaab48fb07":{"model_module":"@jupyter-widgets/controls","model_name":"HBoxModel","state":{"_view_name":"HBoxView","_dom_classes":[],"_model_name":"HBoxModel","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.5.0","box_style":"","layout":"IPY_MODEL_ddb83383522d4559b1d387209752f87d","_model_module":"@jupyter-widgets/controls","children":["IPY_MODEL_f318a67b29454ff282b36f5efd54db7f","IPY_MODEL_44afc1c59215410d9882f9d5e8e9d4c6"]}},"ddb83383522d4559b1d387209752f87d":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"f318a67b29454ff282b36f5efd54db7f":{"model_module":"@jupyter-widgets/controls","model_name":"FloatProgressModel","state":{"_view_name":"ProgressView","style":"IPY_MODEL_8bd025074c464265bd5070266910a3cb","_dom_classes":[],"description":"100%","_model_name":"FloatProgressModel","bar_style":"success","max":32,"_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":32,"_view_count":null,"_view_module_version":"1.5.0","orientation":"horizontal","min":0,"description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_1ac345d33e894f92a16716ff002ba8d2"}},"44afc1c59215410d9882f9d5e8e9d4c6":{"model_module":"@jupyter-widgets/controls","model_name":"HTMLModel","state":{"_view_name":"HTMLView","style":"IPY_MODEL_488a41ef48af40949c616ae2ecf0e036","_dom_classes":[],"description":"","_model_name":"HTMLModel","placeholder":"​","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":" 32/32 [00:10&lt;00:00,  3.04 batch/s]","_view_count":null,"_view_module_version":"1.5.0","description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_002ad5b03c214dd4bdf1b8ee41c310a2"}},"8bd025074c464265bd5070266910a3cb":{"model_module":"@jupyter-widgets/controls","model_name":"ProgressStyleModel","state":{"_view_name":"StyleView","_model_name":"ProgressStyleModel","description_width":"initial","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","bar_color":null,"_model_module":"@jupyter-widgets/controls"}},"1ac345d33e894f92a16716ff002ba8d2":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"488a41ef48af40949c616ae2ecf0e036":{"model_module":"@jupyter-widgets/controls","model_name":"DescriptionStyleModel","state":{"_view_name":"StyleView","_model_name":"DescriptionStyleModel","description_width":"","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","_model_module":"@jupyter-widgets/controls"}},"002ad5b03c214dd4bdf1b8ee41c310a2":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"5e5b167cdae742b383abd1d5ce530e8e":{"model_module":"@jupyter-widgets/controls","model_name":"HBoxModel","state":{"_view_name":"HBoxView","_dom_classes":[],"_model_name":"HBoxModel","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.5.0","box_style":"","layout":"IPY_MODEL_1332211c11624343967723356f6b965a","_model_module":"@jupyter-widgets/controls","children":["IPY_MODEL_9df362dd299e48e98e1497600186e8cf","IPY_MODEL_f632f47971b8429c800e8ac5522111d3"]}},"1332211c11624343967723356f6b965a":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"9df362dd299e48e98e1497600186e8cf":{"model_module":"@jupyter-widgets/controls","model_name":"FloatProgressModel","state":{"_view_name":"ProgressView","style":"IPY_MODEL_104fa651b180484aa4472d46e746ab9c","_dom_classes":[],"description":"100%","_model_name":"FloatProgressModel","bar_style":"success","max":10,"_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":10,"_view_count":null,"_view_module_version":"1.5.0","orientation":"horizontal","min":0,"description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_d2ca87870527474abdeeb22cee1bb0b7"}},"f632f47971b8429c800e8ac5522111d3":{"model_module":"@jupyter-widgets/controls","model_name":"HTMLModel","state":{"_view_name":"HTMLView","style":"IPY_MODEL_d1b6b419f9cd47e5b44b6d99e03157bd","_dom_classes":[],"description":"","_model_name":"HTMLModel","placeholder":"​","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":" 10/10 [00:16&lt;00:00,  1.66s/ batch]","_view_count":null,"_view_module_version":"1.5.0","description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_a9235e1158b3454f9411d31ebeaa2089"}},"104fa651b180484aa4472d46e746ab9c":{"model_module":"@jupyter-widgets/controls","model_name":"ProgressStyleModel","state":{"_view_name":"StyleView","_model_name":"ProgressStyleModel","description_width":"initial","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","bar_color":null,"_model_module":"@jupyter-widgets/controls"}},"d2ca87870527474abdeeb22cee1bb0b7":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"d1b6b419f9cd47e5b44b6d99e03157bd":{"model_module":"@jupyter-widgets/controls","model_name":"DescriptionStyleModel","state":{"_view_name":"StyleView","_model_name":"DescriptionStyleModel","description_width":"","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","_model_module":"@jupyter-widgets/controls"}},"a9235e1158b3454f9411d31ebeaa2089":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"2f8725285e434f4e973d69cd223bdd71":{"model_module":"@jupyter-widgets/controls","model_name":"HBoxModel","state":{"_view_name":"HBoxView","_dom_classes":[],"_model_name":"HBoxModel","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.5.0","box_style":"","layout":"IPY_MODEL_2dffb23ec2b142ac8647038f0d758735","_model_module":"@jupyter-widgets/controls","children":["IPY_MODEL_75a6882f705148dd9191bad00f42a448","IPY_MODEL_9ad0f8861b4c465581ce77d08d8b063d"]}},"2dffb23ec2b142ac8647038f0d758735":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"75a6882f705148dd9191bad00f42a448":{"model_module":"@jupyter-widgets/controls","model_name":"FloatProgressModel","state":{"_view_name":"ProgressView","style":"IPY_MODEL_7d2c8ac9c8844a04a78af3fb2e8bd9a0","_dom_classes":[],"description":"100%","_model_name":"FloatProgressModel","bar_style":"success","max":95683,"_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":95683,"_view_count":null,"_view_module_version":"1.5.0","orientation":"horizontal","min":0,"description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_b88fc27e2ca146dd8a62950b76534f6f"}},"9ad0f8861b4c465581ce77d08d8b063d":{"model_module":"@jupyter-widgets/controls","model_name":"HTMLModel","state":{"_view_name":"HTMLView","style":"IPY_MODEL_510a16ca46b24217bda728505a599110","_dom_classes":[],"description":"","_model_name":"HTMLModel","placeholder":"​","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":" 95683/95683 [03:24&lt;00:00, 467.62 ORF/s]","_view_count":null,"_view_module_version":"1.5.0","description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_6243922fdfd948b1b7f8bb0fab1fd1a3"}},"7d2c8ac9c8844a04a78af3fb2e8bd9a0":{"model_module":"@jupyter-widgets/controls","model_name":"ProgressStyleModel","state":{"_view_name":"StyleView","_model_name":"ProgressStyleModel","description_width":"initial","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","bar_color":null,"_model_module":"@jupyter-widgets/controls"}},"b88fc27e2ca146dd8a62950b76534f6f":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"510a16ca46b24217bda728505a599110":{"model_module":"@jupyter-widgets/controls","model_name":"DescriptionStyleModel","state":{"_view_name":"StyleView","_model_name":"DescriptionStyleModel","description_width":"","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","_model_module":"@jupyter-widgets/controls"}},"6243922fdfd948b1b7f8bb0fab1fd1a3":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"048e9cad5a1c4d058a0fc845706c88bc":{"model_module":"@jupyter-widgets/controls","model_name":"HBoxModel","state":{"_view_name":"HBoxView","_dom_classes":[],"_model_name":"HBoxModel","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.5.0","box_style":"","layout":"IPY_MODEL_dfc6ec5277c2439b8c5636be64da33a7","_model_module":"@jupyter-widgets/controls","children":["IPY_MODEL_399f3d667b0141128c3659ca3baa580b","IPY_MODEL_5b6c3143ef2a44fb8dc7d9e2aa35992a"]}},"dfc6ec5277c2439b8c5636be64da33a7":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"399f3d667b0141128c3659ca3baa580b":{"model_module":"@jupyter-widgets/controls","model_name":"FloatProgressModel","state":{"_view_name":"ProgressView","style":"IPY_MODEL_cb97d558d70745d492951221527a3ca9","_dom_classes":[],"description":"100%","_model_name":"FloatProgressModel","bar_style":"success","max":19,"_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":19,"_view_count":null,"_view_module_version":"1.5.0","orientation":"horizontal","min":0,"description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_4d2b66a2c8a343219bf46485b54bb4cd"}},"5b6c3143ef2a44fb8dc7d9e2aa35992a":{"model_module":"@jupyter-widgets/controls","model_name":"HTMLModel","state":{"_view_name":"HTMLView","style":"IPY_MODEL_34a1630a35cc4842b7ab260ce900b84e","_dom_classes":[],"description":"","_model_name":"HTMLModel","placeholder":"​","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":" 19/19 [00:07&lt;00:00,  2.53 batch/s]","_view_count":null,"_view_module_version":"1.5.0","description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_7ba8ff0a8bc0459aa669343962370265"}},"cb97d558d70745d492951221527a3ca9":{"model_module":"@jupyter-widgets/controls","model_name":"ProgressStyleModel","state":{"_view_name":"StyleView","_model_name":"ProgressStyleModel","description_width":"initial","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","bar_color":null,"_model_module":"@jupyter-widgets/controls"}},"4d2b66a2c8a343219bf46485b54bb4cd":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"34a1630a35cc4842b7ab260ce900b84e":{"model_module":"@jupyter-widgets/controls","model_name":"DescriptionStyleModel","state":{"_view_name":"StyleView","_model_name":"DescriptionStyleModel","description_width":"","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","_model_module":"@jupyter-widgets/controls"}},"7ba8ff0a8bc0459aa669343962370265":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"b4e9dd7b7a2441cbbb462522ae5cb527":{"model_module":"@jupyter-widgets/controls","model_name":"HBoxModel","state":{"_view_name":"HBoxView","_dom_classes":[],"_model_name":"HBoxModel","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.5.0","box_style":"","layout":"IPY_MODEL_b3001c008fbb4853b80acbd044285923","_model_module":"@jupyter-widgets/controls","children":["IPY_MODEL_f4e00cb2d3fc46cc8f12787c1df6a2fe","IPY_MODEL_57f51cb6a2b9457e99ed00414b1ba65c"]}},"b3001c008fbb4853b80acbd044285923":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"f4e00cb2d3fc46cc8f12787c1df6a2fe":{"model_module":"@jupyter-widgets/controls","model_name":"FloatProgressModel","state":{"_view_name":"ProgressView","style":"IPY_MODEL_214c14929d1e47cda3e273f2e4e7bccb","_dom_classes":[],"description":"100%","_model_name":"FloatProgressModel","bar_style":"success","max":6,"_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":6,"_view_count":null,"_view_module_version":"1.5.0","orientation":"horizontal","min":0,"description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_8de9c07b6e274f6285de21d825d13e7b"}},"57f51cb6a2b9457e99ed00414b1ba65c":{"model_module":"@jupyter-widgets/controls","model_name":"HTMLModel","state":{"_view_name":"HTMLView","style":"IPY_MODEL_fd791f38eb6d41e1b3ebca8c8300a694","_dom_classes":[],"description":"","_model_name":"HTMLModel","placeholder":"​","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":" 6/6 [00:08&lt;00:00,  1.33s/ batch]","_view_count":null,"_view_module_version":"1.5.0","description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_5d0cef0c621047ee9fe785628a313a93"}},"214c14929d1e47cda3e273f2e4e7bccb":{"model_module":"@jupyter-widgets/controls","model_name":"ProgressStyleModel","state":{"_view_name":"StyleView","_model_name":"ProgressStyleModel","description_width":"initial","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","bar_color":null,"_model_module":"@jupyter-widgets/controls"}},"8de9c07b6e274f6285de21d825d13e7b":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"fd791f38eb6d41e1b3ebca8c8300a694":{"model_module":"@jupyter-widgets/controls","model_name":"DescriptionStyleModel","state":{"_view_name":"StyleView","_model_name":"DescriptionStyleModel","description_width":"","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","_model_module":"@jupyter-widgets/controls"}},"5d0cef0c621047ee9fe785628a313a93":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"13b85acb0a834129afec188f7e5f5609":{"model_module":"@jupyter-widgets/controls","model_name":"HBoxModel","state":{"_view_name":"HBoxView","_dom_classes":[],"_model_name":"HBoxModel","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.5.0","box_style":"","layout":"IPY_MODEL_1f6d0ec7f8cb45e88c3d18b94a6ddbac","_model_module":"@jupyter-widgets/controls","children":["IPY_MODEL_ddc83a472df847bdb342ee9d0d312183","IPY_MODEL_e51252e7acf642eaacc0d9d3557ea6e3"]}},"1f6d0ec7f8cb45e88c3d18b94a6ddbac":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"ddc83a472df847bdb342ee9d0d312183":{"model_module":"@jupyter-widgets/controls","model_name":"FloatProgressModel","state":{"_view_name":"ProgressView","style":"IPY_MODEL_c2253cbc1cea457eaaf5d18f55e0dd8a","_dom_classes":[],"description":"100%","_model_name":"FloatProgressModel","bar_style":"success","max":54299,"_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":54299,"_view_count":null,"_view_module_version":"1.5.0","orientation":"horizontal","min":0,"description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_90631b1c41c14295a83d76d5d2708bfd"}},"e51252e7acf642eaacc0d9d3557ea6e3":{"model_module":"@jupyter-widgets/controls","model_name":"HTMLModel","state":{"_view_name":"HTMLView","style":"IPY_MODEL_c910c56e752e4eb0be3a429c5c464ec1","_dom_classes":[],"description":"","_model_name":"HTMLModel","placeholder":"​","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":" 54299/54299 [02:46&lt;00:00, 326.13 ORF/s]","_view_count":null,"_view_module_version":"1.5.0","description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_8d6ee044481642d2b7efe618854add92"}},"c2253cbc1cea457eaaf5d18f55e0dd8a":{"model_module":"@jupyter-widgets/controls","model_name":"ProgressStyleModel","state":{"_view_name":"StyleView","_model_name":"ProgressStyleModel","description_width":"initial","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","bar_color":null,"_model_module":"@jupyter-widgets/controls"}},"90631b1c41c14295a83d76d5d2708bfd":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"c910c56e752e4eb0be3a429c5c464ec1":{"model_module":"@jupyter-widgets/controls","model_name":"DescriptionStyleModel","state":{"_view_name":"StyleView","_model_name":"DescriptionStyleModel","description_width":"","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","_model_module":"@jupyter-widgets/controls"}},"8d6ee044481642d2b7efe618854add92":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"66720ecd5d564308ae9537abf796386c":{"model_module":"@jupyter-widgets/controls","model_name":"HBoxModel","state":{"_view_name":"HBoxView","_dom_classes":[],"_model_name":"HBoxModel","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.5.0","box_style":"","layout":"IPY_MODEL_db02fa05c81740a0ba2a4691e673d35a","_model_module":"@jupyter-widgets/controls","children":["IPY_MODEL_98d1de15e751488eb9d4c2669cf45a29","IPY_MODEL_3ef003d9a62645a78a11765575503bb8"]}},"db02fa05c81740a0ba2a4691e673d35a":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"98d1de15e751488eb9d4c2669cf45a29":{"model_module":"@jupyter-widgets/controls","model_name":"FloatProgressModel","state":{"_view_name":"ProgressView","style":"IPY_MODEL_827390fc79594891a9755e1acb202408","_dom_classes":[],"description":"100%","_model_name":"FloatProgressModel","bar_style":"success","max":44,"_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":44,"_view_count":null,"_view_module_version":"1.5.0","orientation":"horizontal","min":0,"description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_983f09f3272343109457407f0b337969"}},"3ef003d9a62645a78a11765575503bb8":{"model_module":"@jupyter-widgets/controls","model_name":"HTMLModel","state":{"_view_name":"HTMLView","style":"IPY_MODEL_4f7ae95210ae4cdbb56277dc5af53c3f","_dom_classes":[],"description":"","_model_name":"HTMLModel","placeholder":"​","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":" 44/44 [00:21&lt;00:00,  2.07 batch/s]","_view_count":null,"_view_module_version":"1.5.0","description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_4dfa210f96b44b5bab0964123b1c3cc9"}},"827390fc79594891a9755e1acb202408":{"model_module":"@jupyter-widgets/controls","model_name":"ProgressStyleModel","state":{"_view_name":"StyleView","_model_name":"ProgressStyleModel","description_width":"initial","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","bar_color":null,"_model_module":"@jupyter-widgets/controls"}},"983f09f3272343109457407f0b337969":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"4f7ae95210ae4cdbb56277dc5af53c3f":{"model_module":"@jupyter-widgets/controls","model_name":"DescriptionStyleModel","state":{"_view_name":"StyleView","_model_name":"DescriptionStyleModel","description_width":"","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","_model_module":"@jupyter-widgets/controls"}},"4dfa210f96b44b5bab0964123b1c3cc9":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"aafcacd6301b45168b390b80669a4b20":{"model_module":"@jupyter-widgets/controls","model_name":"HBoxModel","state":{"_view_name":"HBoxView","_dom_classes":[],"_model_name":"HBoxModel","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.5.0","box_style":"","layout":"IPY_MODEL_f41f7c65f517462d8ae2cdc4cc95f4fe","_model_module":"@jupyter-widgets/controls","children":["IPY_MODEL_5e4f38941d9f408cab263ffa93802b35","IPY_MODEL_954f34de9cb2492b9ba36849a99de8ba"]}},"f41f7c65f517462d8ae2cdc4cc95f4fe":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"5e4f38941d9f408cab263ffa93802b35":{"model_module":"@jupyter-widgets/controls","model_name":"FloatProgressModel","state":{"_view_name":"ProgressView","style":"IPY_MODEL_d1d1aab522ee43c38bfb9e99fc78272b","_dom_classes":[],"description":"100%","_model_name":"FloatProgressModel","bar_style":"success","max":14,"_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":14,"_view_count":null,"_view_module_version":"1.5.0","orientation":"horizontal","min":0,"description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_baeaceb412cc4714b4f2c4f39be96cd2"}},"954f34de9cb2492b9ba36849a99de8ba":{"model_module":"@jupyter-widgets/controls","model_name":"HTMLModel","state":{"_view_name":"HTMLView","style":"IPY_MODEL_3a79f4c22a2a4f2f843b85afc4c9d89f","_dom_classes":[],"description":"","_model_name":"HTMLModel","placeholder":"​","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":" 14/14 [00:03&lt;00:00,  4.45 batch/s]","_view_count":null,"_view_module_version":"1.5.0","description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_c37956e75e4f42159f486411760aadbe"}},"d1d1aab522ee43c38bfb9e99fc78272b":{"model_module":"@jupyter-widgets/controls","model_name":"ProgressStyleModel","state":{"_view_name":"StyleView","_model_name":"ProgressStyleModel","description_width":"initial","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","bar_color":null,"_model_module":"@jupyter-widgets/controls"}},"baeaceb412cc4714b4f2c4f39be96cd2":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"3a79f4c22a2a4f2f843b85afc4c9d89f":{"model_module":"@jupyter-widgets/controls","model_name":"DescriptionStyleModel","state":{"_view_name":"StyleView","_model_name":"DescriptionStyleModel","description_width":"","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","_model_module":"@jupyter-widgets/controls"}},"c37956e75e4f42159f486411760aadbe":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"1dc41c6bde60429c867d1728ac1d9bcd":{"model_module":"@jupyter-widgets/controls","model_name":"HBoxModel","state":{"_view_name":"HBoxView","_dom_classes":[],"_model_name":"HBoxModel","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.5.0","box_style":"","layout":"IPY_MODEL_df1936c315e441cdb7d8a243b419f752","_model_module":"@jupyter-widgets/controls","children":["IPY_MODEL_db121121bd384d789e3b776b1b50c6ce","IPY_MODEL_f9ce533748d24f229f83f08a684c1285"]}},"df1936c315e441cdb7d8a243b419f752":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"db121121bd384d789e3b776b1b50c6ce":{"model_module":"@jupyter-widgets/controls","model_name":"FloatProgressModel","state":{"_view_name":"ProgressView","style":"IPY_MODEL_a7415db33f1143dbb8de2484652f1d39","_dom_classes":[],"description":"100%","_model_name":"FloatProgressModel","bar_style":"success","max":130339,"_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":130339,"_view_count":null,"_view_module_version":"1.5.0","orientation":"horizontal","min":0,"description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_4d09103f77824c229241793dd018123c"}},"f9ce533748d24f229f83f08a684c1285":{"model_module":"@jupyter-widgets/controls","model_name":"HTMLModel","state":{"_view_name":"HTMLView","style":"IPY_MODEL_1948317f52fb40ec8edd7e64c1a62a31","_dom_classes":[],"description":"","_model_name":"HTMLModel","placeholder":"​","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":" 130339/130339 [03:19&lt;00:00, 653.43 ORF/s]","_view_count":null,"_view_module_version":"1.5.0","description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_32e71ccb416642278fdfd0b38700621c"}},"a7415db33f1143dbb8de2484652f1d39":{"model_module":"@jupyter-widgets/controls","model_name":"ProgressStyleModel","state":{"_view_name":"StyleView","_model_name":"ProgressStyleModel","description_width":"initial","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","bar_color":null,"_model_module":"@jupyter-widgets/controls"}},"4d09103f77824c229241793dd018123c":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"1948317f52fb40ec8edd7e64c1a62a31":{"model_module":"@jupyter-widgets/controls","model_name":"DescriptionStyleModel","state":{"_view_name":"StyleView","_model_name":"DescriptionStyleModel","description_width":"","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","_model_module":"@jupyter-widgets/controls"}},"32e71ccb416642278fdfd0b38700621c":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"05347276863b4446bfe50360ce120ec3":{"model_module":"@jupyter-widgets/controls","model_name":"HBoxModel","state":{"_view_name":"HBoxView","_dom_classes":[],"_model_name":"HBoxModel","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.5.0","box_style":"","layout":"IPY_MODEL_ab0ba327302244f88acbcfa71ef0fa09","_model_module":"@jupyter-widgets/controls","children":["IPY_MODEL_f24d6a20e6fd4c2c88e8f6975c7b572c","IPY_MODEL_23cf0d36b47142228868df11fa4537f7"]}},"ab0ba327302244f88acbcfa71ef0fa09":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"f24d6a20e6fd4c2c88e8f6975c7b572c":{"model_module":"@jupyter-widgets/controls","model_name":"FloatProgressModel","state":{"_view_name":"ProgressView","style":"IPY_MODEL_0b0829b2ce154060a8dca996697e4905","_dom_classes":[],"description":"100%","_model_name":"FloatProgressModel","bar_style":"success","max":36,"_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":36,"_view_count":null,"_view_module_version":"1.5.0","orientation":"horizontal","min":0,"description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_09a41721dc934dbdb0a274ed10ad5393"}},"23cf0d36b47142228868df11fa4537f7":{"model_module":"@jupyter-widgets/controls","model_name":"HTMLModel","state":{"_view_name":"HTMLView","style":"IPY_MODEL_5504bb9c952d4d9f9158b185c5b0b126","_dom_classes":[],"description":"","_model_name":"HTMLModel","placeholder":"​","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":" 36/36 [00:13&lt;00:00,  2.58 batch/s]","_view_count":null,"_view_module_version":"1.5.0","description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_fce26a6f90c6425ca90ae7882f138630"}},"0b0829b2ce154060a8dca996697e4905":{"model_module":"@jupyter-widgets/controls","model_name":"ProgressStyleModel","state":{"_view_name":"StyleView","_model_name":"ProgressStyleModel","description_width":"initial","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","bar_color":null,"_model_module":"@jupyter-widgets/controls"}},"09a41721dc934dbdb0a274ed10ad5393":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"5504bb9c952d4d9f9158b185c5b0b126":{"model_module":"@jupyter-widgets/controls","model_name":"DescriptionStyleModel","state":{"_view_name":"StyleView","_model_name":"DescriptionStyleModel","description_width":"","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","_model_module":"@jupyter-widgets/controls"}},"fce26a6f90c6425ca90ae7882f138630":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"4db70922ee6a461cbe2760331f1de5ad":{"model_module":"@jupyter-widgets/controls","model_name":"HBoxModel","state":{"_view_name":"HBoxView","_dom_classes":[],"_model_name":"HBoxModel","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.5.0","box_style":"","layout":"IPY_MODEL_131ea6bf5244452cb80141133eba99fc","_model_module":"@jupyter-widgets/controls","children":["IPY_MODEL_ad1952ba8cb64e8a92c805214d9a7568","IPY_MODEL_a4bd0137a58846b2a69729ac0f6e0a7d"]}},"131ea6bf5244452cb80141133eba99fc":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"ad1952ba8cb64e8a92c805214d9a7568":{"model_module":"@jupyter-widgets/controls","model_name":"FloatProgressModel","state":{"_view_name":"ProgressView","style":"IPY_MODEL_324cbc39e0634811ad4c0b254ddf1f82","_dom_classes":[],"description":"100%","_model_name":"FloatProgressModel","bar_style":"success","max":11,"_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":11,"_view_count":null,"_view_module_version":"1.5.0","orientation":"horizontal","min":0,"description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_2eb389abad7f4e44b0609b409242853e"}},"a4bd0137a58846b2a69729ac0f6e0a7d":{"model_module":"@jupyter-widgets/controls","model_name":"HTMLModel","state":{"_view_name":"HTMLView","style":"IPY_MODEL_4bc7bb9972ff4da1992df065182d1d4b","_dom_classes":[],"description":"","_model_name":"HTMLModel","placeholder":"​","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":" 11/11 [00:14&lt;00:00,  1.31s/ batch]","_view_count":null,"_view_module_version":"1.5.0","description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_8b4c54d448e94fb9ab8ae91cac0573b6"}},"324cbc39e0634811ad4c0b254ddf1f82":{"model_module":"@jupyter-widgets/controls","model_name":"ProgressStyleModel","state":{"_view_name":"StyleView","_model_name":"ProgressStyleModel","description_width":"initial","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","bar_color":null,"_model_module":"@jupyter-widgets/controls"}},"2eb389abad7f4e44b0609b409242853e":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"4bc7bb9972ff4da1992df065182d1d4b":{"model_module":"@jupyter-widgets/controls","model_name":"DescriptionStyleModel","state":{"_view_name":"StyleView","_model_name":"DescriptionStyleModel","description_width":"","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","_model_module":"@jupyter-widgets/controls"}},"8b4c54d448e94fb9ab8ae91cac0573b6":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"669d6a3b074545029153f230f2ce4be2":{"model_module":"@jupyter-widgets/controls","model_name":"HBoxModel","state":{"_view_name":"HBoxView","_dom_classes":[],"_model_name":"HBoxModel","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.5.0","box_style":"","layout":"IPY_MODEL_68e60a11dec4432a9e43b65cf4a2aa60","_model_module":"@jupyter-widgets/controls","children":["IPY_MODEL_102d99916d8e40d994398d4e7985f828","IPY_MODEL_a0fbe76d1a2644a9b80daa71388bfa0e"]}},"68e60a11dec4432a9e43b65cf4a2aa60":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"102d99916d8e40d994398d4e7985f828":{"model_module":"@jupyter-widgets/controls","model_name":"FloatProgressModel","state":{"_view_name":"ProgressView","style":"IPY_MODEL_d0ee01fbf61841418da5050390bd145c","_dom_classes":[],"description":"100%","_model_name":"FloatProgressModel","bar_style":"success","max":105276,"_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":105276,"_view_count":null,"_view_module_version":"1.5.0","orientation":"horizontal","min":0,"description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_2bfe731a952b4925914055ac59f9d3b7"}},"a0fbe76d1a2644a9b80daa71388bfa0e":{"model_module":"@jupyter-widgets/controls","model_name":"HTMLModel","state":{"_view_name":"HTMLView","style":"IPY_MODEL_0ea887311c1843fab1cb9f07a07269c1","_dom_classes":[],"description":"","_model_name":"HTMLModel","placeholder":"​","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":" 105276/105276 [05:32&lt;00:00, 316.86 ORF/s]","_view_count":null,"_view_module_version":"1.5.0","description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_eebcb6a650644ab8a1a67632ce2b37a3"}},"d0ee01fbf61841418da5050390bd145c":{"model_module":"@jupyter-widgets/controls","model_name":"ProgressStyleModel","state":{"_view_name":"StyleView","_model_name":"ProgressStyleModel","description_width":"initial","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","bar_color":null,"_model_module":"@jupyter-widgets/controls"}},"2bfe731a952b4925914055ac59f9d3b7":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"0ea887311c1843fab1cb9f07a07269c1":{"model_module":"@jupyter-widgets/controls","model_name":"DescriptionStyleModel","state":{"_view_name":"StyleView","_model_name":"DescriptionStyleModel","description_width":"","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","_model_module":"@jupyter-widgets/controls"}},"eebcb6a650644ab8a1a67632ce2b37a3":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"d5be9d7e4fda47d9b59031db23e486dd":{"model_module":"@jupyter-widgets/controls","model_name":"HBoxModel","state":{"_view_name":"HBoxView","_dom_classes":[],"_model_name":"HBoxModel","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.5.0","box_style":"","layout":"IPY_MODEL_146f30b1b0d940f2b6f40e943fa66a08","_model_module":"@jupyter-widgets/controls","children":["IPY_MODEL_47be945ac3f74aec8f8e52b8288c4764","IPY_MODEL_ded5bf10b9c741c9a97c3ed92397e3f8"]}},"146f30b1b0d940f2b6f40e943fa66a08":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"47be945ac3f74aec8f8e52b8288c4764":{"model_module":"@jupyter-widgets/controls","model_name":"FloatProgressModel","state":{"_view_name":"ProgressView","style":"IPY_MODEL_13b0529806224dadb3003bc66393a8a0","_dom_classes":[],"description":"100%","_model_name":"FloatProgressModel","bar_style":"success","max":462,"_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":462,"_view_count":null,"_view_module_version":"1.5.0","orientation":"horizontal","min":0,"description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_3e95208a7ecc477ba83a8ca71b980c63"}},"ded5bf10b9c741c9a97c3ed92397e3f8":{"model_module":"@jupyter-widgets/controls","model_name":"HTMLModel","state":{"_view_name":"HTMLView","style":"IPY_MODEL_70336258d400491186277a7496f0993b","_dom_classes":[],"description":"","_model_name":"HTMLModel","placeholder":"​","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":" 462/462 [02:04&lt;00:00,  3.72 ORF/s]","_view_count":null,"_view_module_version":"1.5.0","description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_6b0ced3ec2494fd6beb7747c88d0b705"}},"13b0529806224dadb3003bc66393a8a0":{"model_module":"@jupyter-widgets/controls","model_name":"ProgressStyleModel","state":{"_view_name":"StyleView","_model_name":"ProgressStyleModel","description_width":"initial","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","bar_color":null,"_model_module":"@jupyter-widgets/controls"}},"3e95208a7ecc477ba83a8ca71b980c63":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"70336258d400491186277a7496f0993b":{"model_module":"@jupyter-widgets/controls","model_name":"DescriptionStyleModel","state":{"_view_name":"StyleView","_model_name":"DescriptionStyleModel","description_width":"","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","_model_module":"@jupyter-widgets/controls"}},"6b0ced3ec2494fd6beb7747c88d0b705":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"6289935e919349d9b32246b8693b0705":{"model_module":"@jupyter-widgets/controls","model_name":"HBoxModel","state":{"_view_name":"HBoxView","_dom_classes":[],"_model_name":"HBoxModel","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.5.0","box_style":"","layout":"IPY_MODEL_419ec030973f493d83723cf372b39c19","_model_module":"@jupyter-widgets/controls","children":["IPY_MODEL_774187c41ef14bbaa3e9d3c780a6d1db","IPY_MODEL_a6e38d9a2a584c5d8f79e3e9ac4be562"]}},"419ec030973f493d83723cf372b39c19":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"774187c41ef14bbaa3e9d3c780a6d1db":{"model_module":"@jupyter-widgets/controls","model_name":"FloatProgressModel","state":{"_view_name":"ProgressView","style":"IPY_MODEL_a2f8e7d3b1c24e6eb653f2627910d8a8","_dom_classes":[],"description":"100%","_model_name":"FloatProgressModel","bar_style":"success","max":241,"_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":241,"_view_count":null,"_view_module_version":"1.5.0","orientation":"horizontal","min":0,"description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_49d0b240625341dd80562ea3f6b97704"}},"a6e38d9a2a584c5d8f79e3e9ac4be562":{"model_module":"@jupyter-widgets/controls","model_name":"HTMLModel","state":{"_view_name":"HTMLView","style":"IPY_MODEL_6c14ff7c6ba64ef698a014b7928a8a3c","_dom_classes":[],"description":"","_model_name":"HTMLModel","placeholder":"​","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":" 241/241 [01:04&lt;00:00,  3.72 ORF/s]","_view_count":null,"_view_module_version":"1.5.0","description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_6967231bf8d44373b154621889a863a4"}},"a2f8e7d3b1c24e6eb653f2627910d8a8":{"model_module":"@jupyter-widgets/controls","model_name":"ProgressStyleModel","state":{"_view_name":"StyleView","_model_name":"ProgressStyleModel","description_width":"initial","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","bar_color":null,"_model_module":"@jupyter-widgets/controls"}},"49d0b240625341dd80562ea3f6b97704":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"6c14ff7c6ba64ef698a014b7928a8a3c":{"model_module":"@jupyter-widgets/controls","model_name":"DescriptionStyleModel","state":{"_view_name":"StyleView","_model_name":"DescriptionStyleModel","description_width":"","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","_model_module":"@jupyter-widgets/controls"}},"6967231bf8d44373b154621889a863a4":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"e20337dbc1894121baf459979ae468ce":{"model_module":"@jupyter-widgets/controls","model_name":"HBoxModel","state":{"_view_name":"HBoxView","_dom_classes":[],"_model_name":"HBoxModel","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.5.0","box_style":"","layout":"IPY_MODEL_73919c47423f4a15800c42d85c4ae7b6","_model_module":"@jupyter-widgets/controls","children":["IPY_MODEL_50248c51c8db4c3f897d00ceea70da84","IPY_MODEL_e795012b794f4caaba8fa8ddf235e734"]}},"73919c47423f4a15800c42d85c4ae7b6":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"50248c51c8db4c3f897d00ceea70da84":{"model_module":"@jupyter-widgets/controls","model_name":"FloatProgressModel","state":{"_view_name":"ProgressView","style":"IPY_MODEL_463874cb89b84b5cb9ab4b908b639b2d","_dom_classes":[],"description":"100%","_model_name":"FloatProgressModel","bar_style":"success","max":66,"_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":66,"_view_count":null,"_view_module_version":"1.5.0","orientation":"horizontal","min":0,"description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_00351e06106c487595cfe1cdddb3f039"}},"e795012b794f4caaba8fa8ddf235e734":{"model_module":"@jupyter-widgets/controls","model_name":"HTMLModel","state":{"_view_name":"HTMLView","style":"IPY_MODEL_5bb46eb9613d4adeab2820efac86e92e","_dom_classes":[],"description":"","_model_name":"HTMLModel","placeholder":"​","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":" 66/66 [00:26&lt;00:00,  2.46 batch/s]","_view_count":null,"_view_module_version":"1.5.0","description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_3ce6fac5e20a4ec6b84a1c93ac1b11cc"}},"463874cb89b84b5cb9ab4b908b639b2d":{"model_module":"@jupyter-widgets/controls","model_name":"ProgressStyleModel","state":{"_view_name":"StyleView","_model_name":"ProgressStyleModel","description_width":"initial","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","bar_color":null,"_model_module":"@jupyter-widgets/controls"}},"00351e06106c487595cfe1cdddb3f039":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"5bb46eb9613d4adeab2820efac86e92e":{"model_module":"@jupyter-widgets/controls","model_name":"DescriptionStyleModel","state":{"_view_name":"StyleView","_model_name":"DescriptionStyleModel","description_width":"","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","_model_module":"@jupyter-widgets/controls"}},"3ce6fac5e20a4ec6b84a1c93ac1b11cc":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"eb7d892bcef947a0aeeb415db7478c83":{"model_module":"@jupyter-widgets/controls","model_name":"HBoxModel","state":{"_view_name":"HBoxView","_dom_classes":[],"_model_name":"HBoxModel","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.5.0","box_style":"","layout":"IPY_MODEL_368f59574a6c4e43affe904c0ed6f9c4","_model_module":"@jupyter-widgets/controls","children":["IPY_MODEL_250fe10b54b64564a0b1102b7fee3893","IPY_MODEL_3c89569b92b5428fadfddeb59267c3fd"]}},"368f59574a6c4e43affe904c0ed6f9c4":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"250fe10b54b64564a0b1102b7fee3893":{"model_module":"@jupyter-widgets/controls","model_name":"FloatProgressModel","state":{"_view_name":"ProgressView","style":"IPY_MODEL_ad29f941aa384e75898803957718d437","_dom_classes":[],"description":"100%","_model_name":"FloatProgressModel","bar_style":"success","max":20,"_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":20,"_view_count":null,"_view_module_version":"1.5.0","orientation":"horizontal","min":0,"description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_06281a8432b24a82ba70b7a5ece82937"}},"3c89569b92b5428fadfddeb59267c3fd":{"model_module":"@jupyter-widgets/controls","model_name":"HTMLModel","state":{"_view_name":"HTMLView","style":"IPY_MODEL_f7ab0ee90c8c4bfab8a00c6dee340244","_dom_classes":[],"description":"","_model_name":"HTMLModel","placeholder":"​","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":" 20/20 [00:21&lt;00:00,  1.10s/ batch]","_view_count":null,"_view_module_version":"1.5.0","description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_be5fd4b11a914b67b558df5c0d67db6d"}},"ad29f941aa384e75898803957718d437":{"model_module":"@jupyter-widgets/controls","model_name":"ProgressStyleModel","state":{"_view_name":"StyleView","_model_name":"ProgressStyleModel","description_width":"initial","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","bar_color":null,"_model_module":"@jupyter-widgets/controls"}},"06281a8432b24a82ba70b7a5ece82937":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"f7ab0ee90c8c4bfab8a00c6dee340244":{"model_module":"@jupyter-widgets/controls","model_name":"DescriptionStyleModel","state":{"_view_name":"StyleView","_model_name":"DescriptionStyleModel","description_width":"","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","_model_module":"@jupyter-widgets/controls"}},"be5fd4b11a914b67b558df5c0d67db6d":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"24c0025637e240a889d0edbb570a461c":{"model_module":"@jupyter-widgets/controls","model_name":"HBoxModel","state":{"_view_name":"HBoxView","_dom_classes":[],"_model_name":"HBoxModel","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.5.0","box_style":"","layout":"IPY_MODEL_1e3a008aa07f473a89edff3a8ff3fbe5","_model_module":"@jupyter-widgets/controls","children":["IPY_MODEL_686f9ec36dd64fc8b319ac3660d42893","IPY_MODEL_6b7cd42d00194ad2ac0db151f1c78fa8"]}},"1e3a008aa07f473a89edff3a8ff3fbe5":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"686f9ec36dd64fc8b319ac3660d42893":{"model_module":"@jupyter-widgets/controls","model_name":"FloatProgressModel","state":{"_view_name":"ProgressView","style":"IPY_MODEL_425f3b50fc9844bb87855d18d1adee7e","_dom_classes":[],"description":"100%","_model_name":"FloatProgressModel","bar_style":"success","max":195625,"_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":195625,"_view_count":null,"_view_module_version":"1.5.0","orientation":"horizontal","min":0,"description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_18f792efaca746b68b8fb2c016f98394"}},"6b7cd42d00194ad2ac0db151f1c78fa8":{"model_module":"@jupyter-widgets/controls","model_name":"HTMLModel","state":{"_view_name":"HTMLView","style":"IPY_MODEL_e0dbcbd4341340168c26e5d0f2cb7bf9","_dom_classes":[],"description":"","_model_name":"HTMLModel","placeholder":"​","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":" 195625/195625 [04:19&lt;00:00, 754.64 ORF/s]","_view_count":null,"_view_module_version":"1.5.0","description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_8ce1c7b2a0f044dda5fa631e79808997"}},"425f3b50fc9844bb87855d18d1adee7e":{"model_module":"@jupyter-widgets/controls","model_name":"ProgressStyleModel","state":{"_view_name":"StyleView","_model_name":"ProgressStyleModel","description_width":"initial","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","bar_color":null,"_model_module":"@jupyter-widgets/controls"}},"18f792efaca746b68b8fb2c016f98394":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"e0dbcbd4341340168c26e5d0f2cb7bf9":{"model_module":"@jupyter-widgets/controls","model_name":"DescriptionStyleModel","state":{"_view_name":"StyleView","_model_name":"DescriptionStyleModel","description_width":"","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","_model_module":"@jupyter-widgets/controls"}},"8ce1c7b2a0f044dda5fa631e79808997":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"1be10f9fd0ef47a98590193f26431ac6":{"model_module":"@jupyter-widgets/controls","model_name":"HBoxModel","state":{"_view_name":"HBoxView","_dom_classes":[],"_model_name":"HBoxModel","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.5.0","box_style":"","layout":"IPY_MODEL_105eb8d04a8e488facae3a9e8b1b4246","_model_module":"@jupyter-widgets/controls","children":["IPY_MODEL_716c55d8ab5a49d08541a09bae84231d","IPY_MODEL_6026e44ce2d1424e909a34b7903d24f6"]}},"105eb8d04a8e488facae3a9e8b1b4246":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"716c55d8ab5a49d08541a09bae84231d":{"model_module":"@jupyter-widgets/controls","model_name":"FloatProgressModel","state":{"_view_name":"ProgressView","style":"IPY_MODEL_3772c3a5f9cc478fbae54859dcb2989f","_dom_classes":[],"description":"100%","_model_name":"FloatProgressModel","bar_style":"success","max":17,"_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":17,"_view_count":null,"_view_module_version":"1.5.0","orientation":"horizontal","min":0,"description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_7e47648bca8d4ab08e5350f9e24e8f77"}},"6026e44ce2d1424e909a34b7903d24f6":{"model_module":"@jupyter-widgets/controls","model_name":"HTMLModel","state":{"_view_name":"HTMLView","style":"IPY_MODEL_8fea29557e88469199c134be217af84a","_dom_classes":[],"description":"","_model_name":"HTMLModel","placeholder":"​","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":" 17/17 [00:18&lt;00:00,  1.07s/ batch]","_view_count":null,"_view_module_version":"1.5.0","description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_a2f1370bcd3c4ab3b40f062478fe38f7"}},"3772c3a5f9cc478fbae54859dcb2989f":{"model_module":"@jupyter-widgets/controls","model_name":"ProgressStyleModel","state":{"_view_name":"StyleView","_model_name":"ProgressStyleModel","description_width":"initial","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","bar_color":null,"_model_module":"@jupyter-widgets/controls"}},"7e47648bca8d4ab08e5350f9e24e8f77":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"8fea29557e88469199c134be217af84a":{"model_module":"@jupyter-widgets/controls","model_name":"DescriptionStyleModel","state":{"_view_name":"StyleView","_model_name":"DescriptionStyleModel","description_width":"","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","_model_module":"@jupyter-widgets/controls"}},"a2f1370bcd3c4ab3b40f062478fe38f7":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"fb61f186dda04a64a658bea713b75d80":{"model_module":"@jupyter-widgets/controls","model_name":"HBoxModel","state":{"_view_name":"HBoxView","_dom_classes":[],"_model_name":"HBoxModel","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.5.0","box_style":"","layout":"IPY_MODEL_bb20f5da50704ea6a63871e41f8a0dc6","_model_module":"@jupyter-widgets/controls","children":["IPY_MODEL_bdd159e261764407896cd04ee7c1f704","IPY_MODEL_a6f38f576b9a4fed8b05a9d855ef2ec5"]}},"bb20f5da50704ea6a63871e41f8a0dc6":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"bdd159e261764407896cd04ee7c1f704":{"model_module":"@jupyter-widgets/controls","model_name":"FloatProgressModel","state":{"_view_name":"ProgressView","style":"IPY_MODEL_08373bedb2504b47a1f3f82790e36680","_dom_classes":[],"description":"100%","_model_name":"FloatProgressModel","bar_style":"success","max":5,"_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":5,"_view_count":null,"_view_module_version":"1.5.0","orientation":"horizontal","min":0,"description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_fe72085ce55d4abc913ef23994de1db1"}},"a6f38f576b9a4fed8b05a9d855ef2ec5":{"model_module":"@jupyter-widgets/controls","model_name":"HTMLModel","state":{"_view_name":"HTMLView","style":"IPY_MODEL_f60d4c6403094e1fa258c61e9e5e4f87","_dom_classes":[],"description":"","_model_name":"HTMLModel","placeholder":"​","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":" 5/5 [00:11&lt;00:00,  2.32s/ batch]","_view_count":null,"_view_module_version":"1.5.0","description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_7fc2c07bba314176a0380da6c5431df8"}},"08373bedb2504b47a1f3f82790e36680":{"model_module":"@jupyter-widgets/controls","model_name":"ProgressStyleModel","state":{"_view_name":"StyleView","_model_name":"ProgressStyleModel","description_width":"initial","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","bar_color":null,"_model_module":"@jupyter-widgets/controls"}},"fe72085ce55d4abc913ef23994de1db1":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"f60d4c6403094e1fa258c61e9e5e4f87":{"model_module":"@jupyter-widgets/controls","model_name":"DescriptionStyleModel","state":{"_view_name":"StyleView","_model_name":"DescriptionStyleModel","description_width":"","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","_model_module":"@jupyter-widgets/controls"}},"7fc2c07bba314176a0380da6c5431df8":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"ab6e48c0a5c84a4d9ca9bdc68d258b26":{"model_module":"@jupyter-widgets/controls","model_name":"HBoxModel","state":{"_view_name":"HBoxView","_dom_classes":[],"_model_name":"HBoxModel","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.5.0","box_style":"","layout":"IPY_MODEL_3b7e948712ad4973a92914a2b38f770f","_model_module":"@jupyter-widgets/controls","children":["IPY_MODEL_22b3ebccf90849f4a8eebb18b5885d0b","IPY_MODEL_463af667128b495bad961a9716989ce8"]}},"3b7e948712ad4973a92914a2b38f770f":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"22b3ebccf90849f4a8eebb18b5885d0b":{"model_module":"@jupyter-widgets/controls","model_name":"FloatProgressModel","state":{"_view_name":"ProgressView","style":"IPY_MODEL_0728ae28907f4d378b4c85a285544d61","_dom_classes":[],"description":"100%","_model_name":"FloatProgressModel","bar_style":"success","max":48762,"_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":48762,"_view_count":null,"_view_module_version":"1.5.0","orientation":"horizontal","min":0,"description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_c99da85bb91044bc8381d62c599d2246"}},"463af667128b495bad961a9716989ce8":{"model_module":"@jupyter-widgets/controls","model_name":"HTMLModel","state":{"_view_name":"HTMLView","style":"IPY_MODEL_5fa8d526c5694781b6a234503a8f129a","_dom_classes":[],"description":"","_model_name":"HTMLModel","placeholder":"​","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":" 48762/48762 [03:29&lt;00:00, 232.72 ORF/s]","_view_count":null,"_view_module_version":"1.5.0","description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_6b460597a6f94b2697ff705692347803"}},"0728ae28907f4d378b4c85a285544d61":{"model_module":"@jupyter-widgets/controls","model_name":"ProgressStyleModel","state":{"_view_name":"StyleView","_model_name":"ProgressStyleModel","description_width":"initial","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","bar_color":null,"_model_module":"@jupyter-widgets/controls"}},"c99da85bb91044bc8381d62c599d2246":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"5fa8d526c5694781b6a234503a8f129a":{"model_module":"@jupyter-widgets/controls","model_name":"DescriptionStyleModel","state":{"_view_name":"StyleView","_model_name":"DescriptionStyleModel","description_width":"","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","_model_module":"@jupyter-widgets/controls"}},"6b460597a6f94b2697ff705692347803":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"e50537a8c3bc44abb9b5a523f75c918a":{"model_module":"@jupyter-widgets/controls","model_name":"HBoxModel","state":{"_view_name":"HBoxView","_dom_classes":[],"_model_name":"HBoxModel","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.5.0","box_style":"","layout":"IPY_MODEL_1a04d76b4e484fc29a61bda8163f9bcd","_model_module":"@jupyter-widgets/controls","children":["IPY_MODEL_98009d431f4644649c69618a0c4e7d09","IPY_MODEL_2fca7af9bfcd44abbbfea2948e9c8408"]}},"1a04d76b4e484fc29a61bda8163f9bcd":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"98009d431f4644649c69618a0c4e7d09":{"model_module":"@jupyter-widgets/controls","model_name":"FloatProgressModel","state":{"_view_name":"ProgressView","style":"IPY_MODEL_75b429ffd09a4046b453bf052c413a93","_dom_classes":[],"description":"100%","_model_name":"FloatProgressModel","bar_style":"success","max":1077,"_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":1077,"_view_count":null,"_view_module_version":"1.5.0","orientation":"horizontal","min":0,"description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_0ec4ec65622b4cebb7496ba5bf216399"}},"2fca7af9bfcd44abbbfea2948e9c8408":{"model_module":"@jupyter-widgets/controls","model_name":"HTMLModel","state":{"_view_name":"HTMLView","style":"IPY_MODEL_950d7e161779443d9b52b671aa52f94a","_dom_classes":[],"description":"","_model_name":"HTMLModel","placeholder":"​","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":" 1077/1077 [01:09&lt;00:00, 15.46 ORF/s]","_view_count":null,"_view_module_version":"1.5.0","description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_daadc2234e954511a2432af34f2382a1"}},"75b429ffd09a4046b453bf052c413a93":{"model_module":"@jupyter-widgets/controls","model_name":"ProgressStyleModel","state":{"_view_name":"StyleView","_model_name":"ProgressStyleModel","description_width":"initial","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","bar_color":null,"_model_module":"@jupyter-widgets/controls"}},"0ec4ec65622b4cebb7496ba5bf216399":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"950d7e161779443d9b52b671aa52f94a":{"model_module":"@jupyter-widgets/controls","model_name":"DescriptionStyleModel","state":{"_view_name":"StyleView","_model_name":"DescriptionStyleModel","description_width":"","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","_model_module":"@jupyter-widgets/controls"}},"daadc2234e954511a2432af34f2382a1":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"722d734d9e97467c8ad0f1576da77e5b":{"model_module":"@jupyter-widgets/controls","model_name":"HBoxModel","state":{"_view_name":"HBoxView","_dom_classes":[],"_model_name":"HBoxModel","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.5.0","box_style":"","layout":"IPY_MODEL_037da24f8fc64f058b581366aed9be25","_model_module":"@jupyter-widgets/controls","children":["IPY_MODEL_cf676574faaf43309fa3c2ea16ccebce","IPY_MODEL_822437c544eb49aaaf870bb94bc74225"]}},"037da24f8fc64f058b581366aed9be25":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"cf676574faaf43309fa3c2ea16ccebce":{"model_module":"@jupyter-widgets/controls","model_name":"FloatProgressModel","state":{"_view_name":"ProgressView","style":"IPY_MODEL_392721d48b2244f6875bdbfa671ae194","_dom_classes":[],"description":"100%","_model_name":"FloatProgressModel","bar_style":"success","max":81,"_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":81,"_view_count":null,"_view_module_version":"1.5.0","orientation":"horizontal","min":0,"description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_92d9cae39b0549d8b1f6e69bdb6bd23d"}},"822437c544eb49aaaf870bb94bc74225":{"model_module":"@jupyter-widgets/controls","model_name":"HTMLModel","state":{"_view_name":"HTMLView","style":"IPY_MODEL_84543aeff71f4800ad353b4d23e63597","_dom_classes":[],"description":"","_model_name":"HTMLModel","placeholder":"​","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":" 81/81 [00:27&lt;00:00,  2.95 batch/s]","_view_count":null,"_view_module_version":"1.5.0","description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_2c64bdfb46814d449097ddd7fbffb8bd"}},"392721d48b2244f6875bdbfa671ae194":{"model_module":"@jupyter-widgets/controls","model_name":"ProgressStyleModel","state":{"_view_name":"StyleView","_model_name":"ProgressStyleModel","description_width":"initial","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","bar_color":null,"_model_module":"@jupyter-widgets/controls"}},"92d9cae39b0549d8b1f6e69bdb6bd23d":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"84543aeff71f4800ad353b4d23e63597":{"model_module":"@jupyter-widgets/controls","model_name":"DescriptionStyleModel","state":{"_view_name":"StyleView","_model_name":"DescriptionStyleModel","description_width":"","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","_model_module":"@jupyter-widgets/controls"}},"2c64bdfb46814d449097ddd7fbffb8bd":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"e18dcbc034b0447fa67a49cabfdd6dbf":{"model_module":"@jupyter-widgets/controls","model_name":"HBoxModel","state":{"_view_name":"HBoxView","_dom_classes":[],"_model_name":"HBoxModel","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.5.0","box_style":"","layout":"IPY_MODEL_39ceb44191b64aa19c5653b14ee86328","_model_module":"@jupyter-widgets/controls","children":["IPY_MODEL_1b9fdd84364b4542b0c44ec7e65e9c0c","IPY_MODEL_3fce6e9fe82748a093248fce6ca667f7"]}},"39ceb44191b64aa19c5653b14ee86328":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"1b9fdd84364b4542b0c44ec7e65e9c0c":{"model_module":"@jupyter-widgets/controls","model_name":"FloatProgressModel","state":{"_view_name":"ProgressView","style":"IPY_MODEL_e35978b1e58e42e9887b6db1d87245aa","_dom_classes":[],"description":"100%","_model_name":"FloatProgressModel","bar_style":"success","max":25,"_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":25,"_view_count":null,"_view_module_version":"1.5.0","orientation":"horizontal","min":0,"description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_25a8246a2b5e4ff19a9afcd413175479"}},"3fce6e9fe82748a093248fce6ca667f7":{"model_module":"@jupyter-widgets/controls","model_name":"HTMLModel","state":{"_view_name":"HTMLView","style":"IPY_MODEL_5a87e31f444b4705a7780539b7f0f32a","_dom_classes":[],"description":"","_model_name":"HTMLModel","placeholder":"​","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":" 25/25 [00:29&lt;00:00,  1.19s/ batch]","_view_count":null,"_view_module_version":"1.5.0","description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_192659ffefa9444d8c1d91302b8cdf8c"}},"e35978b1e58e42e9887b6db1d87245aa":{"model_module":"@jupyter-widgets/controls","model_name":"ProgressStyleModel","state":{"_view_name":"StyleView","_model_name":"ProgressStyleModel","description_width":"initial","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","bar_color":null,"_model_module":"@jupyter-widgets/controls"}},"25a8246a2b5e4ff19a9afcd413175479":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"5a87e31f444b4705a7780539b7f0f32a":{"model_module":"@jupyter-widgets/controls","model_name":"DescriptionStyleModel","state":{"_view_name":"StyleView","_model_name":"DescriptionStyleModel","description_width":"","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","_model_module":"@jupyter-widgets/controls"}},"192659ffefa9444d8c1d91302b8cdf8c":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"4d7ca9761f994e1a92d8e6e58670b6a3":{"model_module":"@jupyter-widgets/controls","model_name":"HBoxModel","state":{"_view_name":"HBoxView","_dom_classes":[],"_model_name":"HBoxModel","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.5.0","box_style":"","layout":"IPY_MODEL_bf32d62a15ab4261aaed43a876d79a94","_model_module":"@jupyter-widgets/controls","children":["IPY_MODEL_00eee8ab8103449aab0bcfa589786592","IPY_MODEL_07aa810e50fa47d0bbd6c625e325635c"]}},"bf32d62a15ab4261aaed43a876d79a94":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"00eee8ab8103449aab0bcfa589786592":{"model_module":"@jupyter-widgets/controls","model_name":"FloatProgressModel","state":{"_view_name":"ProgressView","style":"IPY_MODEL_6fd705ce60264da098e9f8251488fc69","_dom_classes":[],"description":"100%","_model_name":"FloatProgressModel","bar_style":"success","max":242033,"_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":242033,"_view_count":null,"_view_module_version":"1.5.0","orientation":"horizontal","min":0,"description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_eeac415a06524045a5cfdf548a959217"}},"07aa810e50fa47d0bbd6c625e325635c":{"model_module":"@jupyter-widgets/controls","model_name":"HTMLModel","state":{"_view_name":"HTMLView","style":"IPY_MODEL_4ece9edc9dd6484eadb29dd06f384cb8","_dom_classes":[],"description":"","_model_name":"HTMLModel","placeholder":"​","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":" 242033/242033 [00:46&lt;00:00, 5251.13 ORF/s]","_view_count":null,"_view_module_version":"1.5.0","description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_3856284bc140489e8cc079b2fcb212ef"}},"6fd705ce60264da098e9f8251488fc69":{"model_module":"@jupyter-widgets/controls","model_name":"ProgressStyleModel","state":{"_view_name":"StyleView","_model_name":"ProgressStyleModel","description_width":"initial","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","bar_color":null,"_model_module":"@jupyter-widgets/controls"}},"eeac415a06524045a5cfdf548a959217":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"4ece9edc9dd6484eadb29dd06f384cb8":{"model_module":"@jupyter-widgets/controls","model_name":"DescriptionStyleModel","state":{"_view_name":"StyleView","_model_name":"DescriptionStyleModel","description_width":"","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","_model_module":"@jupyter-widgets/controls"}},"3856284bc140489e8cc079b2fcb212ef":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}}}}},"cells":[{"cell_type":"markdown","metadata":{"id":"6SYiIHRhZM-m","colab_type":"text"},"source":["# Google Colab Intro\n","#### This notebook downloads data and performs computations. These computations are made significantly faster with the use of a GPU, provided free by Google as part of Colab.\n","\n","#### Press the play button on the left side of each cell to run it. Alternatively, hold shift or ctrl and press enter to run cells.\n","#### Double click the top of a cell to inspect the code inside and change things. Double click the right side of the cell to hide the code.\n","#### Have fun!\n","\n","#### (this project is intended as a proof of concept for a different approach to finding genes, not a drop in replacement for a well-optimized gene finder. A faster C++ version will probably be released at some point if it seems like that would be useful)"]},{"cell_type":"markdown","metadata":{"id":"aEszqvakZN9C","colab_type":"text"},"source":["# Setup"]},{"cell_type":"code","metadata":{"id":"VMJQa6JAXDf_","colab_type":"code","cellView":"form","colab":{"base_uri":"https://localhost:8080/","height":204},"executionInfo":{"status":"ok","timestamp":1593437406122,"user_tz":240,"elapsed":7248,"user":{"displayName":"Markus Sommer","photoUrl":"https://lh3.googleusercontent.com/a-/AOh14GjXujbpCzD6oVvcRIT5EP-oi3Bif3Cfcu7npJUnce8=s64","userId":"08083804319459268544"}},"outputId":"e5036867-1b4c-4ac2-d413-586e88a13014"},"source":["# @title Install dependencies\n","!pip install biopython\n","!pip install torch\n","print(\"\\nDone\")"],"execution_count":1,"outputs":[{"output_type":"stream","text":["Collecting biopython\n","\u001b[?25l  Downloading https://files.pythonhosted.org/packages/a8/66/134dbd5f885fc71493c61b6cf04c9ea08082da28da5ed07709b02857cbd0/biopython-1.77-cp36-cp36m-manylinux1_x86_64.whl (2.3MB)\n","\u001b[K     |████████████████████████████████| 2.3MB 4.6MB/s \n","\u001b[?25hRequirement already satisfied: numpy in /usr/local/lib/python3.6/dist-packages (from biopython) (1.18.5)\n","Installing collected packages: biopython\n","Successfully installed biopython-1.77\n","Requirement already satisfied: torch in /usr/local/lib/python3.6/dist-packages (1.5.1+cu101)\n","Requirement already satisfied: future in /usr/local/lib/python3.6/dist-packages (from torch) (0.16.0)\n","Requirement already satisfied: numpy in /usr/local/lib/python3.6/dist-packages (from torch) (1.18.5)\n","\n","Done\n"],"name":"stdout"}]},{"cell_type":"code","metadata":{"id":"pe1PHlcjeHS8","colab_type":"code","cellView":"form","colab":{"base_uri":"https://localhost:8080/","height":34},"executionInfo":{"status":"ok","timestamp":1593437409594,"user_tz":240,"elapsed":10706,"user":{"displayName":"Markus Sommer","photoUrl":"https://lh3.googleusercontent.com/a-/AOh14GjXujbpCzD6oVvcRIT5EP-oi3Bif3Cfcu7npJUnce8=s64","userId":"08083804319459268544"}},"outputId":"494986df-b1be-4e38-dc98-97086d058909"},"source":["# @title Import python packages\n","import os\n","import sys\n","import gzip\n","import copy\n","import time\n","import pandas\n","import pickle\n","import numpy as np\n","from tqdm.auto import tqdm\n","from Bio import SeqIO\n","from scipy.special import expit\n","from scipy.special import logit\n","import networkx as nx\n","from multiprocessing import Pool\n","\n","import torch\n","import torch.nn as nn\n","import torch.nn.functional as F\n","from torch.nn.utils import weight_norm\n","\n","print(\"Done\")"],"execution_count":2,"outputs":[{"output_type":"stream","text":["Done\n"],"name":"stdout"}]},{"cell_type":"code","metadata":{"id":"usWmfqJNhM2X","colab_type":"code","cellView":"both","colab":{"base_uri":"https://localhost:8080/","height":34},"executionInfo":{"status":"ok","timestamp":1593437410394,"user_tz":240,"elapsed":789,"user":{"displayName":"Markus Sommer","photoUrl":"https://lh3.googleusercontent.com/a-/AOh14GjXujbpCzD6oVvcRIT5EP-oi3Bif3Cfcu7npJUnce8=s64","userId":"08083804319459268544"}},"outputId":"9a978067-7551-41c4-b040-df73a791ae12"},"source":["# @title Set parameters (double click here to change defaults)\n","\"\"\" Print what the program is doing.\"\"\"\n","verbose = True\n","\n","\"\"\" Maximum ORF (open reading frame) overlap length in nucleotides.\"\"\"\n","max_gene_overlap = 60\n","\n","\"\"\" Minimum ORF length in nucleotides.\"\"\"\n","min_orf_length = 60\n","\n","\"\"\" Use mmseqs2 and a gene score cutoff to remove most false positive predictions.\"\"\"\n","filter_gene_predictions = True\n","\n","\"\"\" Use kmer prefilter to increase gene sensitivity. \n","May not play nice with high GC genomes.\"\"\"\n","filter_kmer = True\n","\n","\"\"\" Nucleotide to amino acid translation table. 11 for most bacteria/archaea.\n","4 for Mycoplasma/Spiroplasma. Can be 1 to 25.\"\"\"\n","translation_table = 11\n","# translation_table = 4\n","\n","\"\"\" Maximum number of forward connections in the directed acyclic graph used to\n","find a set of coherent genes in each genome.\n","Higher values will slow execution time but may slightly increase performace.\n","Recommended range 30 to 100.\"\"\"\n","max_forward_connections = 50\n","\n","\"\"\" Batch size for the temporal convolutional network used to score genes.\n","Small batches and big batches slow down the model. Very big batches may crash the \n","GPU. \"\"\"\n","gene_batch_size = 3000\n","TIS_batch_size = 10000\n","\n","\"\"\" Where the pre-trained gene model should be saved.\"\"\"\n","model_dir = \"/home\"\n","\n","\n","\"\"\" All following are internal parameters. Change at your own risk.\"\"\"\n","weight_gene_prob = 0.9746869839852076 \n","weight_TIS_prob = 0.25380288790532707 \n","score_threshold = 0.47256101519707244\n","weight_ATG = 0.84249804151264 \n","weight_GTG = 0.7083689705744909\n","weight_TTG = 0.7512400826652517 \n","unidirectional_penalty_per_base = 3.895921717182765 # 3' 5' overlap\n","convergent_penalty_per_base = 4.603432608883688 # 3' 3' overlap\n","divergent_penalty_per_base = 3.3830814940689975 # 5' 5' overlap\n","\n","k_seengene = 10\n","multimer_threshold = 2\n","\n","\n","# weight_gene_prob = 0.9746869839852076 \n","# weight_TIS_prob = 0.25380288790532707 \n","# score_threshold = 0.47256101519707244\n","# weight_ATG = 0.84\n","# weight_GTG = 0.14\n","# weight_TTG = 0.03 \n","# unidirectional_penalty_per_base = 3 # 3' 5' overlap\n","# convergent_penalty_per_base = 3 # 3' 3' overlap\n","# divergent_penalty_per_base = 5 # 5' 5' overlap\n","\n","# weight_gene_prob = 0.9578349986086553 \n","# weight_TIS_prob = 0.5144895885507309 \n","# score_threshold = 0.18612804507042569\n","# weight_ATG = 0.6958167545680435 \n","# weight_GTG = 0.5112666591536028\n","# weight_TTG = 0.5372931869382979 \n","# unidirectional_penalty_per_base = 2.7082978223323337 # 3' 5' overlap\n","# convergent_penalty_per_base = 9.28139793016752 # 3' 3' overlap\n","# divergent_penalty_per_base = 0.7174449102593538 # 5' 5' overlap\n","\n","print(\"Done\")"],"execution_count":3,"outputs":[{"output_type":"stream","text":["Done\n"],"name":"stdout"}]},{"cell_type":"code","metadata":{"id":"idOvBwhe0NtP","colab_type":"code","cellView":"form","colab":{"base_uri":"https://localhost:8080/","height":102},"executionInfo":{"status":"ok","timestamp":1593437433805,"user_tz":240,"elapsed":19911,"user":{"displayName":"Markus Sommer","photoUrl":"https://lh3.googleusercontent.com/a-/AOh14GjXujbpCzD6oVvcRIT5EP-oi3Bif3Cfcu7npJUnce8=s64","userId":"08083804319459268544"}},"outputId":"222f6b11-80ff-42c1-a646-085492598426"},"source":["# @title Load pre-trained gene and translation initiation site models\n","\n","\"\"\" if you're interested in the inner workings of the \n","temporal convolutional network, see hubconf.py in the Github repo below.\"\"\"\n","\n","repo = \"salzberg-lab/Balrog\"\n","\n","torch.hub.set_dir(model_dir)\n","if torch.cuda.device_count() > 0:\n","    print(\"GPU detected...\")\n","    model = torch.hub.load(repo, \"geneTCN\", force_reload=True).cuda()\n","    model_tis = torch.hub.load(repo, \"tisTCN\", force_reload=False).cuda()\n","    time.sleep(0.5)\n","    print(\"\\nDone\")\n","else:\n","    print(\"No GPU detected, using CPU...\")\n","    model = torch.hub.load(repo, \"geneTCN\", force_reload=True)\n","    model_tis = torch.hub.load(repo, \"tisTCN\", force_reload=False)\n","    time.sleep(0.5)\n","    print(\"\\nDone\")"],"execution_count":4,"outputs":[{"output_type":"stream","text":["GPU detected...\n"],"name":"stdout"},{"output_type":"stream","text":["Downloading: \"https://github.com/salzberg-lab/Balrog/archive/master.zip\" to /home/master.zip\n","Using cache found in /home/salzberg-lab_Balrog_master\n"],"name":"stderr"},{"output_type":"stream","text":["\n","Done\n"],"name":"stdout"}]},{"cell_type":"code","metadata":{"id":"wQMPFNbdk8vv","colab_type":"code","cellView":"form","colab":{"base_uri":"https://localhost:8080/","height":85},"executionInfo":{"status":"ok","timestamp":1593437443347,"user_tz":240,"elapsed":27775,"user":{"displayName":"Markus Sommer","photoUrl":"https://lh3.googleusercontent.com/a-/AOh14GjXujbpCzD6oVvcRIT5EP-oi3Bif3Cfcu7npJUnce8=s64","userId":"08083804319459268544"}},"outputId":"cfac4e1a-1f27-42ea-e952-a6a493a87255"},"source":["# @title Prepare kmer-based prefilter\n","if filter_kmer:\n","    # decompress kmer filter\n","    seengene_dir = \"/content/kmerfilter/\"\n","    !mkdir {seengene_dir}\n","    %cd {seengene_dir}\n","    !tar -xvzf /home/salzberg-lab_Balrog_master/kmer_filter/genexa_10mer_thresh2_minusARF_all.tar.gz\n","    genexa_kmer_path = os.path.join(seengene_dir, \"10mer_thresh2_minusARF_all.pkl\")\n","\n","    # load kmer filter\n","    with open(genexa_kmer_path, \"rb\") as f:\n","        aa_kmer_set = pickle.load(f)\n","\n","print(\"\\nDone\")"],"execution_count":5,"outputs":[{"output_type":"stream","text":["/content/kmerfilter\n","10mer_thresh2_minusARF_all.pkl\n","\n","Done\n"],"name":"stdout"}]},{"cell_type":"code","metadata":{"id":"p3jzwdcGp7XN","colab_type":"code","cellView":"form","colab":{"base_uri":"https://localhost:8080/","height":1000},"executionInfo":{"status":"ok","timestamp":1593437536838,"user_tz":240,"elapsed":113821,"user":{"displayName":"Markus Sommer","photoUrl":"https://lh3.googleusercontent.com/a-/AOh14GjXujbpCzD6oVvcRIT5EP-oi3Bif3Cfcu7npJUnce8=s64","userId":"08083804319459268544"}},"outputId":"c5a95eab-e424-4339-bfbf-7a5bff8f30ea"},"source":["# @title Prepare mmseqs2 (will be skipped if filter_gene_predictions option is set to False)\n","\n","if filter_gene_predictions:\n","    ![ $(uname -m) = \"x86_64\" ] && echo \"64bit: Yes\" || echo \"64bit: No\"\n","    !grep -q sse4_1 /proc/cpuinfo && echo \"SSE4.1: Yes\" || echo \"SSE4.1: No\"\n","    !grep -q avx2 /proc/cpuinfo && echo \"AVX2: Yes\" || echo \"AVX2: No\"\n","\n","    # install\n","    !mkdir /content/mmseqs2\n","    %cd /content/mmseqs2\n","    !wget https://mmseqs.com/latest/mmseqs-linux-avx2.tar.gz\n","    !tar xvzf mmseqs-linux-avx2.tar.gz\n","\n","    # decompress fasta\n","    !mkdir /content/mmseqs2/genexa\n","    %cd /content/mmseqs2/genexa\n","    !!tar -xvzf /home/salzberg-lab_Balrog_master/protein_filter/genexa_genes.tar.gz\n","    genexa_fasta_path = \"/content/mmseqs2/genexa/genexa_genes.fasta\"\n","\n","    # create DB\n","    genexa_DB_path = \"/content/mmseqs2/genexa/genexaDB\"\n","    !/content/mmseqs2/mmseqs/bin/mmseqs createdb {genexa_fasta_path} {genexa_DB_path}\n","\n","    # build mmseqs index\n","    !mkdir /content/mmseqs2/tmp\n","    !/content/mmseqs2/mmseqs/bin/mmseqs createindex {genexa_DB_path} /content/mmseqs2/tmp\n","\n","    # download swissprot DB\n","    !mkdir /content/mmseqs2/swissprot\n","    !/content/mmseqs2/mmseqs/bin/mmseqs databases UniProtKB/Swiss-Prot /content/mmseqs2/swissprot/swissprotDB /content/mmseqs2/tmp\n","    swissprot_DB_path = \"/content/mmseqs2/swissprot/swissprotDB\"\n","    \n","print(\"\\nDone\")"],"execution_count":6,"outputs":[{"output_type":"stream","text":["64bit: Yes\n","SSE4.1: Yes\n","AVX2: Yes\n","/content/mmseqs2\n","--2020-06-29 13:30:43--  https://mmseqs.com/latest/mmseqs-linux-avx2.tar.gz\n","Resolving mmseqs.com (mmseqs.com)... 141.5.100.26\n","Connecting to mmseqs.com (mmseqs.com)|141.5.100.26|:443... connected.\n","HTTP request sent, awaiting response... 200 OK\n","Length: 28568695 (27M) [application/octet-stream]\n","Saving to: ‘mmseqs-linux-avx2.tar.gz’\n","\n","mmseqs-linux-avx2.t 100%[===================>]  27.25M  12.1MB/s    in 2.2s    \n","\n","2020-06-29 13:30:46 (12.1 MB/s) - ‘mmseqs-linux-avx2.tar.gz’ saved [28568695/28568695]\n","\n","mmseqs/\n","mmseqs/README.md\n","mmseqs/examples/\n","mmseqs/examples/QUERY.fasta\n","mmseqs/examples/DB.fasta\n","mmseqs/util/\n","mmseqs/util/bash-completion.sh\n","mmseqs/bin/\n","mmseqs/bin/mmseqs\n","mmseqs/userguide.pdf\n","mmseqs/matrices/\n","mmseqs/matrices/blosum85.out\n","mmseqs/matrices/blosum40.out\n","mmseqs/matrices/PAM50.out\n","mmseqs/matrices/PAM90.out\n","mmseqs/matrices/PAM120.out\n","mmseqs/matrices/blosum100.out\n","mmseqs/matrices/VTML160.out\n","mmseqs/matrices/PAM160.out\n","mmseqs/matrices/VTML20.out\n","mmseqs/matrices/PAM40.out\n","mmseqs/matrices/VTML40.out\n","mmseqs/matrices/blosum45.out\n","mmseqs/matrices/PAM20.out\n","mmseqs/matrices/blosum95.out\n","mmseqs/matrices/PAM30.out\n","mmseqs/matrices/PAM70.out\n","mmseqs/matrices/nucleotide.out\n","mmseqs/matrices/PAM100.out\n","mmseqs/matrices/blosum90.out\n","mmseqs/matrices/blosum62.out\n","mmseqs/matrices/PAM170.out\n","mmseqs/matrices/VTML80.out\n","mmseqs/matrices/PAM130.out\n","mmseqs/matrices/PAM140.out\n","mmseqs/matrices/blosum70.out\n","mmseqs/matrices/PAM150.out\n","mmseqs/matrices/PAM60.out\n","mmseqs/matrices/PAM10.out\n","mmseqs/matrices/PAM190.out\n","mmseqs/matrices/VTML120.out\n","mmseqs/matrices/PAM180.out\n","mmseqs/matrices/blosum50.out\n","mmseqs/matrices/blosum60.out\n","mmseqs/matrices/blosum55.out\n","mmseqs/matrices/PAM80.out\n","mmseqs/matrices/PAM110.out\n","mmseqs/matrices/blosum35.out\n","mmseqs/matrices/blosum30.out\n","mmseqs/matrices/blosum65.out\n","mmseqs/matrices/blosum80.out\n","mmseqs/matrices/blosum75.out\n","mmseqs/matrices/VTML10.out\n","mmseqs/LICENCE.md\n","/content/mmseqs2/genexa\n","createdb /content/mmseqs2/genexa/genexa_genes.fasta /content/mmseqs2/genexa/genexaDB \n","\n","MMseqs Version:       \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Database type         \t0\n","Shuffle input database\ttrue\n","Createdb mode         \t0\n","Write lookup file     \t1\n","Offset of numeric ids \t0\n","Compressed            \t0\n","Verbosity             \t3\n","\n","Converting sequences\n","[468338] 0s 697ms\n","Time for merging to genexaDB_h: 0h 0m 0s 132ms\n","Time for merging to genexaDB: 0h 0m 0s 342ms\n","Database type: Aminoacid\n","Time for processing: 0h 0m 2s 134ms\n","createindex /content/mmseqs2/genexa/genexaDB /content/mmseqs2/tmp \n","\n","MMseqs Version:          \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Seed substitution matrix \tnucl:nucleotide.out,aa:VTML80.out\n","k-mer length             \t0\n","Alphabet size            \tnucl:5,aa:21\n","Compositional bias       \t1\n","Max sequence length      \t65535\n","Max results per query    \t300\n","Mask residues            \t1\n","Mask lower case residues \t0\n","Spaced k-mers            \t1\n","Spaced k-mer pattern     \t\n","Sensitivity              \t7.5\n","k-score                  \t0\n","Check compatible         \t0\n","Search type              \t0\n","Split database           \t0\n","Split memory limit       \t0\n","Verbosity                \t3\n","Threads                  \t4\n","Min codons in orf        \t30\n","Max codons in length     \t32734\n","Max orf gaps             \t2147483647\n","Contig start mode        \t2\n","Contig end mode          \t2\n","Orf start mode           \t1\n","Forward frames           \t1,2,3\n","Reverse frames           \t1,2,3\n","Translation table        \t1\n","Translate orf            \t0\n","Use all table starts     \tfalse\n","Offset of numeric ids    \t0\n","Create lookup            \t0\n","Compressed               \t0\n","Add orf stop             \tfalse\n","Overlap between sequences\t0\n","Sequence split mode      \t1\n","Strand selection         \t1\n","Remove temporary files   \tfalse\n","\n","createindex /content/mmseqs2/genexa/genexaDB /content/mmseqs2/tmp \n","\n","MMseqs Version:          \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Seed substitution matrix \tnucl:nucleotide.out,aa:VTML80.out\n","k-mer length             \t0\n","Alphabet size            \tnucl:5,aa:21\n","Compositional bias       \t1\n","Max sequence length      \t65535\n","Max results per query    \t300\n","Mask residues            \t1\n","Mask lower case residues \t0\n","Spaced k-mers            \t1\n","Spaced k-mer pattern     \t\n","Sensitivity              \t7.5\n","k-score                  \t0\n","Check compatible         \t0\n","Search type              \t0\n","Split database           \t0\n","Split memory limit       \t0\n","Verbosity                \t3\n","Threads                  \t4\n","Min codons in orf        \t30\n","Max codons in length     \t32734\n","Max orf gaps             \t2147483647\n","Contig start mode        \t2\n","Contig end mode          \t2\n","Orf start mode           \t1\n","Forward frames           \t1,2,3\n","Reverse frames           \t1,2,3\n","Translation table        \t1\n","Translate orf            \t0\n","Use all table starts     \tfalse\n","Offset of numeric ids    \t0\n","Create lookup            \t0\n","Compressed               \t0\n","Add orf stop             \tfalse\n","Overlap between sequences\t0\n","Sequence split mode      \t1\n","Strand selection         \t1\n","Remove temporary files   \tfalse\n","\n","indexdb /content/mmseqs2/genexa/genexaDB /content/mmseqs2/genexa/genexaDB --seed-sub-mat nucl:nucleotide.out,aa:VTML80.out -k 0 --alph-size nucl:5,aa:21 --comp-bias-corr 1 --max-seq-len 65535 --max-seqs 300 --mask 1 --mask-lower-case 0 --spaced-kmer-mode 1 -s 7.5 --k-score 0 --check-compatible 0 --search-type 0 --split 0 --split-memory-limit 0 -v 3 --threads 4 \n","\n","Estimated memory consumption: 2G\n","Write VERSION (0)\n","Write META (1)\n","Write SCOREMATRIX3MER (4)\n","Write SCOREMATRIX2MER (3)\n","Write SCOREMATRIXNAME (2)\n","Write SPACEDPATTERN (23)\n","Write DBR1INDEX (5)\n","Write DBR1DATA (6)\n","Write HDR1INDEX (18)\n","Write HDR1DATA (19)\n","Write GENERATOR (22)\n","Index table: counting k-mers\n","[=================================================================] 100.00% 468.35K 19s 634ms\n","Index table: Masked residues: 931982\n","Index table: fill\n","[=================================================================] 100.00% 468.35K 14s 197ms\n","Index statistics\n","Entries:          155110361\n","DB size:          1375 MB\n","Avg k-mer size:   2.423599\n","Top 10 k-mers\n","    TSGGGV\t2599\n","    RAARQG\t2269\n","    TSGGGI\t1980\n","    AVQQSL\t1945\n","    RLTKGS\t1733\n","    TTGGGV\t1443\n","    LAAAQQ\t896\n","    IARQGS\t831\n","    ALREVV\t810\n","    TSGGGT\t772\n","Write ENTRIES (9)\n","Write ENTRIESOFFSETS (10)\n","Write SEQINDEXDATASIZE (15)\n","Write SEQINDEXSEQOFFSET (16)\n","Write SEQINDEXDATA (14)\n","Write ENTRIESNUM (12)\n","Write SEQCOUNT (13)\n","Time for merging to genexaDB.idx: 0h 0m 0s 0ms\n","Time for processing: 0h 0m 42s 204ms\n","databases UniProtKB/Swiss-Prot /content/mmseqs2/swissprot/swissprotDB /content/mmseqs2/tmp \n","\n","MMseqs Version:              \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Force restart with latest tmp\tfalse\n","Remove temporary files       \tfalse\n","Compressed                   \t0\n","Threads                      \t4\n","Verbosity                    \t3\n","\n","  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current\n","                                 Dload  Upload   Total   Spent    Left  Speed\n","100   151  100   151    0     0    130      0  0:00:01  0:00:01 --:--:--   130\n","  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current\n","                                 Dload  Upload   Total   Spent    Left  Speed\n","100 85.2M  100 85.2M    0     0  8827k      0  0:00:09  0:00:09 --:--:-- 17.6M\n","createdb /content/mmseqs2/tmp/6904747282255197622/uniprot_sprot.fasta.gz /content/mmseqs2/swissprot/swissprotDB --compressed 0 -v 3 \n","\n","Converting sequences\n","[562672] 2s 287ms\n","Time for merging to swissprotDB_h: 0h 0m 0s 245ms\n","Time for merging to swissprotDB: 0h 0m 0s 436ms\n","Database type: Aminoacid\n","Time for processing: 0h 0m 3s 659ms\n","prefixid /content/mmseqs2/swissprot/swissprotDB_h /content/mmseqs2/tmp/6904747282255197622/header_pref.tsv --tsv --threads 4 -v 3 \n","\n","[=================================================================] 100.00% 562.76K 0s 587ms\n","Time for merging to header_pref.tsv: 0h 0m 0s 236ms\n","Time for processing: 0h 0m 0s 917ms\n","\u001b[33mTmp /content/mmseqs2/tmp/6904747282255197622/taxonomy folder does not exist or is not a directory.\n","\u001b[39mCreate dir /content/mmseqs2/tmp/6904747282255197622/taxonomy\n","createtaxdb /content/mmseqs2/swissprot/swissprotDB /content/mmseqs2/tmp/6904747282255197622/taxonomy --threads 4 -v 3 \n","\n","Download taxdump.tar.gz\n","  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current\n","                                 Dload  Upload   Total   Spent    Left  Speed\n","100 50.0M  100 50.0M    0     0  23.6M      0  0:00:02  0:00:02 --:--:-- 23.6M\n","Database created\n","\n","Done\n"],"name":"stdout"}]},{"cell_type":"markdown","metadata":{"id":"1Ea6jrhpfF4y","colab_type":"text"},"source":["# Gene Prediction"]},{"cell_type":"code","metadata":{"id":"uh274cXnWlva","colab_type":"code","cellView":"form","colab":{"resources":{"http://localhost:8080/nbextensions/google.colab/files.js":{"data":"Ly8gQ29weXJpZ2h0IDIwMTcgR29vZ2xlIExMQwovLwovLyBMaWNlbnNlZCB1bmRlciB0aGUgQXBhY2hlIExpY2Vuc2UsIFZlcnNpb24gMi4wICh0aGUgIkxpY2Vuc2UiKTsKLy8geW91IG1heSBub3QgdXNlIHRoaXMgZmlsZSBleGNlcHQgaW4gY29tcGxpYW5jZSB3aXRoIHRoZSBMaWNlbnNlLgovLyBZb3UgbWF5IG9idGFpbiBhIGNvcHkgb2YgdGhlIExpY2Vuc2UgYXQKLy8KLy8gICAgICBodHRwOi8vd3d3LmFwYWNoZS5vcmcvbGljZW5zZXMvTElDRU5TRS0yLjAKLy8KLy8gVW5sZXNzIHJlcXVpcmVkIGJ5IGFwcGxpY2FibGUgbGF3IG9yIGFncmVlZCB0byBpbiB3cml0aW5nLCBzb2Z0d2FyZQovLyBkaXN0cmlidXRlZCB1bmRlciB0aGUgTGljZW5zZSBpcyBkaXN0cmlidXRlZCBvbiBhbiAiQVMgSVMiIEJBU0lTLAovLyBXSVRIT1VUIFdBUlJBTlRJRVMgT1IgQ09ORElUSU9OUyBPRiBBTlkgS0lORCwgZWl0aGVyIGV4cHJlc3Mgb3IgaW1wbGllZC4KLy8gU2VlIHRoZSBMaWNlbnNlIGZvciB0aGUgc3BlY2lmaWMgbGFuZ3VhZ2UgZ292ZXJuaW5nIHBlcm1pc3Npb25zIGFuZAovLyBsaW1pdGF0aW9ucyB1bmRlciB0aGUgTGljZW5zZS4KCi8qKgogKiBAZmlsZW92ZXJ2aWV3IEhlbHBlcnMgZm9yIGdvb2dsZS5jb2xhYiBQeXRob24gbW9kdWxlLgogKi8KKGZ1bmN0aW9uKHNjb3BlKSB7CmZ1bmN0aW9uIHNwYW4odGV4dCwgc3R5bGVBdHRyaWJ1dGVzID0ge30pIHsKICBjb25zdCBlbGVtZW50ID0gZG9jdW1lbnQuY3JlYXRlRWxlbWVudCgnc3BhbicpOwogIGVsZW1lbnQudGV4dENvbnRlbnQgPSB0ZXh0OwogIGZvciAoY29uc3Qga2V5IG9mIE9iamVjdC5rZXlzKHN0eWxlQXR0cmlidXRlcykpIHsKICAgIGVsZW1lbnQuc3R5bGVba2V5XSA9IHN0eWxlQXR0cmlidXRlc1trZXldOwogIH0KICByZXR1cm4gZWxlbWVudDsKfQoKLy8gTWF4IG51bWJlciBvZiBieXRlcyB3aGljaCB3aWxsIGJlIHVwbG9hZGVkIGF0IGEgdGltZS4KY29uc3QgTUFYX1BBWUxPQURfU0laRSA9IDEwMCAqIDEwMjQ7CgpmdW5jdGlvbiBfdXBsb2FkRmlsZXMoaW5wdXRJZCwgb3V0cHV0SWQpIHsKICBjb25zdCBzdGVwcyA9IHVwbG9hZEZpbGVzU3RlcChpbnB1dElkLCBvdXRwdXRJZCk7CiAgY29uc3Qgb3V0cHV0RWxlbWVudCA9IGRvY3VtZW50LmdldEVsZW1lbnRCeUlkKG91dHB1dElkKTsKICAvLyBDYWNoZSBzdGVwcyBvbiB0aGUgb3V0cHV0RWxlbWVudCB0byBtYWtlIGl0IGF2YWlsYWJsZSBmb3IgdGhlIG5leHQgY2FsbAogIC8vIHRvIHVwbG9hZEZpbGVzQ29udGludWUgZnJvbSBQeXRob24uCiAgb3V0cHV0RWxlbWVudC5zdGVwcyA9IHN0ZXBzOwoKICByZXR1cm4gX3VwbG9hZEZpbGVzQ29udGludWUob3V0cHV0SWQpOwp9CgovLyBUaGlzIGlzIHJvdWdobHkgYW4gYXN5bmMgZ2VuZXJhdG9yIChub3Qgc3VwcG9ydGVkIGluIHRoZSBicm93c2VyIHlldCksCi8vIHdoZXJlIHRoZXJlIGFyZSBtdWx0aXBsZSBhc3luY2hyb25vdXMgc3RlcHMgYW5kIHRoZSBQeXRob24gc2lkZSBpcyBnb2luZwovLyB0byBwb2xsIGZvciBjb21wbGV0aW9uIG9mIGVhY2ggc3RlcC4KLy8gVGhpcyB1c2VzIGEgUHJvbWlzZSB0byBibG9jayB0aGUgcHl0aG9uIHNpZGUgb24gY29tcGxldGlvbiBvZiBlYWNoIHN0ZXAsCi8vIHRoZW4gcGFzc2VzIHRoZSByZXN1bHQgb2YgdGhlIHByZXZpb3VzIHN0ZXAgYXMgdGhlIGlucHV0IHRvIHRoZSBuZXh0IHN0ZXAuCmZ1bmN0aW9uIF91cGxvYWRGaWxlc0NvbnRpbnVlKG91dHB1dElkKSB7CiAgY29uc3Qgb3V0cHV0RWxlbWVudCA9IGRvY3VtZW50LmdldEVsZW1lbnRCeUlkKG91dHB1dElkKTsKICBjb25zdCBzdGVwcyA9IG91dHB1dEVsZW1lbnQuc3RlcHM7CgogIGNvbnN0IG5leHQgPSBzdGVwcy5uZXh0KG91dHB1dEVsZW1lbnQubGFzdFByb21pc2VWYWx1ZSk7CiAgcmV0dXJuIFByb21pc2UucmVzb2x2ZShuZXh0LnZhbHVlLnByb21pc2UpLnRoZW4oKHZhbHVlKSA9PiB7CiAgICAvLyBDYWNoZSB0aGUgbGFzdCBwcm9taXNlIHZhbHVlIHRvIG1ha2UgaXQgYXZhaWxhYmxlIHRvIHRoZSBuZXh0CiAgICAvLyBzdGVwIG9mIHRoZSBnZW5lcmF0b3IuCiAgICBvdXRwdXRFbGVtZW50Lmxhc3RQcm9taXNlVmFsdWUgPSB2YWx1ZTsKICAgIHJldHVybiBuZXh0LnZhbHVlLnJlc3BvbnNlOwogIH0pOwp9CgovKioKICogR2VuZXJhdG9yIGZ1bmN0aW9uIHdoaWNoIGlzIGNhbGxlZCBiZXR3ZWVuIGVhY2ggYXN5bmMgc3RlcCBvZiB0aGUgdXBsb2FkCiAqIHByb2Nlc3MuCiAqIEBwYXJhbSB7c3RyaW5nfSBpbnB1dElkIEVsZW1lbnQgSUQgb2YgdGhlIGlucHV0IGZpbGUgcGlja2VyIGVsZW1lbnQuCiAqIEBwYXJhbSB7c3RyaW5nfSBvdXRwdXRJZCBFbGVtZW50IElEIG9mIHRoZSBvdXRwdXQgZGlzcGxheS4KICogQHJldHVybiB7IUl0ZXJhYmxlPCFPYmplY3Q+fSBJdGVyYWJsZSBvZiBuZXh0IHN0ZXBzLgogKi8KZnVuY3Rpb24qIHVwbG9hZEZpbGVzU3RlcChpbnB1dElkLCBvdXRwdXRJZCkgewogIGNvbnN0IGlucHV0RWxlbWVudCA9IGRvY3VtZW50LmdldEVsZW1lbnRCeUlkKGlucHV0SWQpOwogIGlucHV0RWxlbWVudC5kaXNhYmxlZCA9IGZhbHNlOwoKICBjb25zdCBvdXRwdXRFbGVtZW50ID0gZG9jdW1lbnQuZ2V0RWxlbWVudEJ5SWQob3V0cHV0SWQpOwogIG91dHB1dEVsZW1lbnQuaW5uZXJIVE1MID0gJyc7CgogIGNvbnN0IHBpY2tlZFByb21pc2UgPSBuZXcgUHJvbWlzZSgocmVzb2x2ZSkgPT4gewogICAgaW5wdXRFbGVtZW50LmFkZEV2ZW50TGlzdGVuZXIoJ2NoYW5nZScsIChlKSA9PiB7CiAgICAgIHJlc29sdmUoZS50YXJnZXQuZmlsZXMpOwogICAgfSk7CiAgfSk7CgogIGNvbnN0IGNhbmNlbCA9IGRvY3VtZW50LmNyZWF0ZUVsZW1lbnQoJ2J1dHRvbicpOwogIGlucHV0RWxlbWVudC5wYXJlbnRFbGVtZW50LmFwcGVuZENoaWxkKGNhbmNlbCk7CiAgY2FuY2VsLnRleHRDb250ZW50ID0gJ0NhbmNlbCB1cGxvYWQnOwogIGNvbnN0IGNhbmNlbFByb21pc2UgPSBuZXcgUHJvbWlzZSgocmVzb2x2ZSkgPT4gewogICAgY2FuY2VsLm9uY2xpY2sgPSAoKSA9PiB7CiAgICAgIHJlc29sdmUobnVsbCk7CiAgICB9OwogIH0pOwoKICAvLyBXYWl0IGZvciB0aGUgdXNlciB0byBwaWNrIHRoZSBmaWxlcy4KICBjb25zdCBmaWxlcyA9IHlpZWxkIHsKICAgIHByb21pc2U6IFByb21pc2UucmFjZShbcGlja2VkUHJvbWlzZSwgY2FuY2VsUHJvbWlzZV0pLAogICAgcmVzcG9uc2U6IHsKICAgICAgYWN0aW9uOiAnc3RhcnRpbmcnLAogICAgfQogIH07CgogIGNhbmNlbC5yZW1vdmUoKTsKCiAgLy8gRGlzYWJsZSB0aGUgaW5wdXQgZWxlbWVudCBzaW5jZSBmdXJ0aGVyIHBpY2tzIGFyZSBub3QgYWxsb3dlZC4KICBpbnB1dEVsZW1lbnQuZGlzYWJsZWQgPSB0cnVlOwoKICBpZiAoIWZpbGVzKSB7CiAgICByZXR1cm4gewogICAgICByZXNwb25zZTogewogICAgICAgIGFjdGlvbjogJ2NvbXBsZXRlJywKICAgICAgfQogICAgfTsKICB9CgogIGZvciAoY29uc3QgZmlsZSBvZiBmaWxlcykgewogICAgY29uc3QgbGkgPSBkb2N1bWVudC5jcmVhdGVFbGVtZW50KCdsaScpOwogICAgbGkuYXBwZW5kKHNwYW4oZmlsZS5uYW1lLCB7Zm9udFdlaWdodDogJ2JvbGQnfSkpOwogICAgbGkuYXBwZW5kKHNwYW4oCiAgICAgICAgYCgke2ZpbGUudHlwZSB8fCAnbi9hJ30pIC0gJHtmaWxlLnNpemV9IGJ5dGVzLCBgICsKICAgICAgICBgbGFzdCBtb2RpZmllZDogJHsKICAgICAgICAgICAgZmlsZS5sYXN0TW9kaWZpZWREYXRlID8gZmlsZS5sYXN0TW9kaWZpZWREYXRlLnRvTG9jYWxlRGF0ZVN0cmluZygpIDoKICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgJ24vYSd9IC0gYCkpOwogICAgY29uc3QgcGVyY2VudCA9IHNwYW4oJzAlIGRvbmUnKTsKICAgIGxpLmFwcGVuZENoaWxkKHBlcmNlbnQpOwoKICAgIG91dHB1dEVsZW1lbnQuYXBwZW5kQ2hpbGQobGkpOwoKICAgIGNvbnN0IGZpbGVEYXRhUHJvbWlzZSA9IG5ldyBQcm9taXNlKChyZXNvbHZlKSA9PiB7CiAgICAgIGNvbnN0IHJlYWRlciA9IG5ldyBGaWxlUmVhZGVyKCk7CiAgICAgIHJlYWRlci5vbmxvYWQgPSAoZSkgPT4gewogICAgICAgIHJlc29sdmUoZS50YXJnZXQucmVzdWx0KTsKICAgICAgfTsKICAgICAgcmVhZGVyLnJlYWRBc0FycmF5QnVmZmVyKGZpbGUpOwogICAgfSk7CiAgICAvLyBXYWl0IGZvciB0aGUgZGF0YSB0byBiZSByZWFkeS4KICAgIGxldCBmaWxlRGF0YSA9IHlpZWxkIHsKICAgICAgcHJvbWlzZTogZmlsZURhdGFQcm9taXNlLAogICAgICByZXNwb25zZTogewogICAgICAgIGFjdGlvbjogJ2NvbnRpbnVlJywKICAgICAgfQogICAgfTsKCiAgICAvLyBVc2UgYSBjaHVua2VkIHNlbmRpbmcgdG8gYXZvaWQgbWVzc2FnZSBzaXplIGxpbWl0cy4gU2VlIGIvNjIxMTU2NjAuCiAgICBsZXQgcG9zaXRpb24gPSAwOwogICAgd2hpbGUgKHBvc2l0aW9uIDwgZmlsZURhdGEuYnl0ZUxlbmd0aCkgewogICAgICBjb25zdCBsZW5ndGggPSBNYXRoLm1pbihmaWxlRGF0YS5ieXRlTGVuZ3RoIC0gcG9zaXRpb24sIE1BWF9QQVlMT0FEX1NJWkUpOwogICAgICBjb25zdCBjaHVuayA9IG5ldyBVaW50OEFycmF5KGZpbGVEYXRhLCBwb3NpdGlvbiwgbGVuZ3RoKTsKICAgICAgcG9zaXRpb24gKz0gbGVuZ3RoOwoKICAgICAgY29uc3QgYmFzZTY0ID0gYnRvYShTdHJpbmcuZnJvbUNoYXJDb2RlLmFwcGx5KG51bGwsIGNodW5rKSk7CiAgICAgIHlpZWxkIHsKICAgICAgICByZXNwb25zZTogewogICAgICAgICAgYWN0aW9uOiAnYXBwZW5kJywKICAgICAgICAgIGZpbGU6IGZpbGUubmFtZSwKICAgICAgICAgIGRhdGE6IGJhc2U2NCwKICAgICAgICB9LAogICAgICB9OwogICAgICBwZXJjZW50LnRleHRDb250ZW50ID0KICAgICAgICAgIGAke01hdGgucm91bmQoKHBvc2l0aW9uIC8gZmlsZURhdGEuYnl0ZUxlbmd0aCkgKiAxMDApfSUgZG9uZWA7CiAgICB9CiAgfQoKICAvLyBBbGwgZG9uZS4KICB5aWVsZCB7CiAgICByZXNwb25zZTogewogICAgICBhY3Rpb246ICdjb21wbGV0ZScsCiAgICB9CiAgfTsKfQoKc2NvcGUuZ29vZ2xlID0gc2NvcGUuZ29vZ2xlIHx8IHt9OwpzY29wZS5nb29nbGUuY29sYWIgPSBzY29wZS5nb29nbGUuY29sYWIgfHwge307CnNjb3BlLmdvb2dsZS5jb2xhYi5fZmlsZXMgPSB7CiAgX3VwbG9hZEZpbGVzLAogIF91cGxvYWRGaWxlc0NvbnRpbnVlLAp9Owp9KShzZWxmKTsK","ok":true,"headers":[["content-type","application/javascript"]],"status":200,"status_text":""}},"base_uri":"https://localhost:8080/","height":1000},"executionInfo":{"status":"ok","timestamp":1593443138507,"user_tz":240,"elapsed":134146,"user":{"displayName":"Markus Sommer","photoUrl":"https://lh3.googleusercontent.com/a-/AOh14GjXujbpCzD6oVvcRIT5EP-oi3Bif3Cfcu7npJUnce8=s64","userId":"08083804319459268544"}},"outputId":"0fcb30c4-5b83-47a6-a0a4-eb2d17897233"},"source":["# @title Upload bacterial genomes as FASTA or gzipped FASTA.\n","from google.colab import files\n","genome_dict = files.upload()\n","\n","print(\"\\nDone\")"],"execution_count":16,"outputs":[{"output_type":"display_data","data":{"text/html":["\n","     <input type=\"file\" id=\"files-04077a72-b3be-4320-b5b8-93894cf6573d\" name=\"files[]\" multiple disabled\n","        style=\"border:none\" />\n","     <output id=\"result-04077a72-b3be-4320-b5b8-93894cf6573d\">\n","      Upload widget is only available when the cell has been executed in the\n","      current browser session. Please rerun this cell to enable.\n","      </output>\n","      <script src=\"/nbextensions/google.colab/files.js\"></script> "],"text/plain":["<IPython.core.display.HTML object>"]},"metadata":{"tags":[]}},{"output_type":"stream","text":["Saving GCF_900465355.1_Ran1-1_genomic.fna.gz to GCF_900465355.1_Ran1-1_genomic.fna.gz\n","Saving GCF_003047205.1_ASM304720v1_genomic.fna.gz to GCF_003047205.1_ASM304720v1_genomic.fna.gz\n","Saving GCF_002142475.1_ASM214247v1_genomic.fna.gz to GCF_002142475.1_ASM214247v1_genomic.fna (1).gz\n","Saving GCF_002113825.1_ASM211382v1_genomic.fna.gz to GCF_002113825.1_ASM211382v1_genomic.fna.gz\n","Saving GCF_002009335.2_ASM200933v2_genomic.fna.gz to GCF_002009335.2_ASM200933v2_genomic.fna.gz\n","Saving GCF_002005165.1_ASM200516v1_genomic.fna.gz to GCF_002005165.1_ASM200516v1_genomic.fna.gz\n","Saving GCF_001729945.1_ASM172994v1_genomic.fna.gz to GCF_001729945.1_ASM172994v1_genomic.fna.gz\n","Saving GCF_001547735.1_ASM154773v1_genomic.fna.gz to GCF_001547735.1_ASM154773v1_genomic.fna.gz\n","Saving GCF_001263175.1_ASM126317v1_genomic.fna.gz to GCF_001263175.1_ASM126317v1_genomic.fna.gz\n","Saving GCF_001027025.1_ASM102702v1_genomic.fna.gz to GCF_001027025.1_ASM102702v1_genomic.fna.gz\n","Saving GCF_000814265.1_ASM81426v1_genomic.fna.gz to GCF_000814265.1_ASM81426v1_genomic.fna.gz\n","Saving GCF_000737325.1_ASM73732v2_genomic.fna.gz to GCF_000737325.1_ASM73732v2_genomic.fna.gz\n","Saving GCF_000725345.1_ASM72534v1_genomic.fna.gz to GCF_000725345.1_ASM72534v1_genomic.fna.gz\n","Saving GCF_000661895.1_ASM66189v1_genomic.fna.gz to GCF_000661895.1_ASM66189v1_genomic.fna.gz\n","Saving GCF_000525675.1_ASM52567v1_genomic.fna.gz to GCF_000525675.1_ASM52567v1_genomic.fna.gz\n","Saving GCF_000219875.1_ASM21987v1_genomic.fna.gz to GCF_000219875.1_ASM21987v1_genomic.fna.gz\n","Saving GCF_000212395.1_ASM21239v1_genomic.fna.gz to GCF_000212395.1_ASM21239v1_genomic.fna.gz\n","Saving GCF_000195295.1_ASM19529v1_genomic.fna.gz to GCF_000195295.1_ASM19529v1_genomic.fna.gz\n","Saving GCF_000191045.1_ASM19104v1_genomic.fna.gz to GCF_000191045.1_ASM19104v1_genomic.fna.gz\n","Saving GCF_000189295.2_ASM18929v2_genomic.fna.gz to GCF_000189295.2_ASM18929v2_genomic.fna.gz\n","Saving GCF_000185805.1_ASM18580v1_genomic.fna.gz to GCF_000185805.1_ASM18580v1_genomic.fna.gz\n","Saving GCF_000184705.1_ASM18470v1_genomic.fna.gz to GCF_000184705.1_ASM18470v1_genomic.fna.gz\n","Saving GCF_000177635.2_ASM17763v2_genomic.fna.gz to GCF_000177635.2_ASM17763v2_genomic.fna.gz\n","Saving GCF_000147695.2_ASM14769v3_genomic.fna.gz to GCF_000147695.2_ASM14769v3_genomic.fna.gz\n","Saving GCF_000144645.1_ASM14464v1_genomic.fna.gz to GCF_000144645.1_ASM14464v1_genomic.fna.gz\n","Saving GCF_000020965.1_ASM2096v1_genomic.fna.gz to GCF_000020965.1_ASM2096v1_genomic.fna.gz\n","Saving GCF_000020505.1_ASM2050v1_genomic.fna.gz to GCF_000020505.1_ASM2050v1_genomic.fna.gz\n","Saving GCF_000020005.1_ASM2000v1_genomic.fna.gz to GCF_000020005.1_ASM2000v1_genomic.fna.gz\n","Saving GCF_000009905.1_ASM990v1_genomic.fna.gz to GCF_000009905.1_ASM990v1_genomic.fna.gz\n","Saving GCF_000008625.1_ASM862v1_genomic.fna.gz to GCF_000008625.1_ASM862v1_genomic.fna.gz\n","Saving GCF_000005845.2_ASM584v2_genomic.fna.gz to GCF_000005845.2_ASM584v2_genomic.fna.gz\n","\n","Done\n"],"name":"stdout"}]},{"cell_type":"code","metadata":{"id":"NC4llyrGWIuK","colab_type":"code","cellView":"form","colab":{"base_uri":"https://localhost:8080/","height":1000,"referenced_widgets":["da4cdb519f6b4666beb6c961d72ec00f","d3bde4a8d3a6425d9e6be647d3da3508","1eeed60e23394683b08c69c3b49320e3","871c9d93cd164b469d0db083d564170f","ae270eec15ce4cf8a732b5f01ab50f32","c9f42d5e44eb4e2685074de46c1ec7a3","422c6dc5dfe34eb883589d2c95372375","06bcbdfbe6aa41388e032ae794ca3612","b14bd3ff1365460fa6b77069a7d29d9d","02d760d5d0a54aeb83b7102378b827d2","df26c1fa04a54d52bdaf203aacb334f0","1bc273565e2a4be3bc2f79f89a7360cf","694bd9677f894f9a84c1688590b51ece","f165dc550e144003a55a0face7905d25","09cd78096e4547dc830d8c7690dfcd99","7a5a5b5dda594b30b2496c868abce4d8","7097bf4a7d214150b67139f710b78906","eddfa44895784f9db52b5a07d6ccc706","12d5adcc1c2145779e457e098978e657","d5bade233f784a6eb8224a92585eecab","ed2f96be5384494ca6bc8db664fe2860","70bc560c0ae14a068ce5dfab409b33fa","3a36a242d82b478f8635e6e95dfff7e2","13a7e3fed79b4d7690afd032f6bc140b","6952fed2527b44f5b8c3c4baf6e42323","94be386a3dd247ce94854c1885d2af87","64dd6ee8f97d4987b9dccf3f22af88fd","763a27ed77b44335b435fb379e2042c0","417744e421a94605a8bee944c1277ff7","0da25a719e6f4cafb69b91cf96bf9066","11d348f3d6064674a9917625d1a05f18","580e8c67600745eb9dff2fa9e91e7875","8e1e5a8fb3d340be9050c46787dfa567","d76cef09dc574192b61c69e1b7dcb280","9f829c0e9d9f4fa4a1cbad1d70ee651b","e1b66fbd726d438a921a38a72cdb8517","93f5a7d134584a3c9790a3f1fa48ddac","26bdfeed0cf44f9697c23f8489a1d552","24aa927e762c4dcfb02b3923efa39fc3","9acbc9592970499cbe566bad1eceb409","e4f66d6bc4eb4de9b13c1ca3cba2dfd1","f5638b740dda4dcf9e02dcb3db3f850a","286f32fabedb41c4ae250736ed4b2687","6c862d4d685640a6bbf756a45b6ca358","b0b1b6aceac647e7bc87c1e6b7ab963e","1966050d50df4e5fa61c344bcbedcf72","8c84f0524ea54746af648e61f4de1357","cf24edd863e54251aaafecba928a1af1","514c76651fe8496093dce5ec30a93235","be91e05dd6f1462499455e735520c6c4","e693bb84b9364e8788d4967558ae4aee","0cdf6419ae774738aa29042c7524b8ab","16945ee114f144e69b7e487de75f3ebe","d8ea1410a9a14dc8b32c48aed0e7f8a9","3dfa0d577cde4648915080b283556bb9","2c9d4b616c694dc5b9501e739ffd5ce2","8b2846b7778741789dca148ee24644c8","ecd26d2ebf0143099c6cd7d898782314","1fa2112045a34649af22014f4573a8b1","b285e2674307413a8fa3b2ce129fa8a7","151bf004a7be41e797af74f9af0467da","c28cbdf0377b4a26ae445a27da23257d","49276ce427b54502985e7d75ec3dc68a","223927e5925f49bbaf4756efe366f7be","178edc8405c24d979b6b7ee906db4adc","871e4c30234a4682a78d86ca317ac461","844615901a194af385b5d73066ebdcb1","e0ceb7c6b0174789b44e0a222ab99bbf","7fc7382b989e40efbb208cb8a297bf70","de39049e0a54431aaafebd99c11ce997","7213abe1316d4a3f906a67e419a23f78","c58b0550035441cc895db9a7cf7fe695","33254081567441e8a0a886e191604f93","f4093d6e83e748ab89188792c21962ec","8cc2e136ae6646ad8311bf483198980d","db26a6b65a404aad82e46704f5e4a31b","d034968b22d34616b192c856190aad76","8d7de09dd85b4a5b8fc9405356c92b05","86e5c4bf8e7b4d37a1a9d547886b6bd1","0781a951ae7b4af3a8523c36081a3799","59be591aa4e04fbf9cebacfd3c697dd4","14db8feca9bd406da5801430c1bab5f3","be631a5be4d34a42bb7e0a7a91d6c5df","e793877ba7bc46edb916982a8ba2e793","d60ceb70cf1c40d0b373189de129837e","465e693afc954ad3b13456efbed1cfa0","0f3043541faf4a0a9d9fce4064c74195","da3ff2c9e9ef47f3b9a07d35610e4598","14da851f7f264917b338e5c2ea0cd230","b6fc5e16cc094cd0aae84aef905d5778","03aabf73446b48979ec56ecd9e783418","68c1745c0bc149a786d44c1f4cbb9dfb","735179f8ad51422c9bc08184300bbc42","47c22f8b5624436db06b1ab5733dbf83","49401a7216db453b8d6399e2ac7c7674","34a06dd9e1e546d8af70a81c34b5e020","ad34d5fafbea45598758d5a9658ebdc6","e170eee78e0d417a941bf672fcebaf29","dddebf28504c40c3ae83fe583999f35a","473a227e09cb429f8d78593801d1bd42","13f8cfed826c4e509198cfb1dd8ef99c","3373940272584820a6fa53d80bcb438e","f44b9edb35fc4b68966dbfd5b837aabe","d6fa4a8231d045eca7a79c4c55ffa562","b5536e67d2a641debb7a27a6e3c1764a","826781541a1c4e748a6622077319997d","8a9f1f9b6af6481da0711661a50d909a","6dbdc11ebe0b486a97bd0d93274f9285","36ddebfa70a645eea6b3dc3cb5f1856e","52cc71359d4646aa80ab561f5575dae7","9f93b9e5b5dd438e8ff9111e5fb8d122","3c1be3caa9264c75bc94cbccd04d313c","e0dc1c3d48334cbfa8d6730667496ee5","1fa4b1e2304547718fae119a8741ef48","3711b6d1875041f8b1cad4332f23a898","c69c8dff969f496d9fb48c41b6454f10","5ee79f0c4c534706a60541394aaa3669","009f57620ac342b99695ecaef8e98c3b","6b89f86113e443b79bc5e4bcf9dd516b","345c8d6271b541a4a396796839a77b2b","61764a2ad59c421c821478b2ac42f19b","f74da91a0eb7493d8d3f7d55e40b3197","143a0c57fffa47dc8ceaed5663f900f3","db3bee86c726446f843b21c83ea605c5","9a33ea3293b245f7b92f02c7efbb1e0c","2546b341fb194ad59b9511b6703aa8bd","846d2025b14c49168bdeb659a2e5deb0","57211ef600dd4fe58bb88febd2bf4c81","53e48adaac844b999781e04c08ecbda6","04028c28791948adab3e05eafc37e7a0","ef40aad5fcca4a368f5691154bed6efe","c506d78f8fa64f85ab0b8a8997a85eea","586cf9202a6b441689238627057f8111","e04d66c213204801a6410ed9f0735239","720e61f2312e472c8a4e12dd9f92d673","3c9beedee6a847a881d82739809a0114","ae9787fe5de34d368d18fd66b8140f23","e73c2f0f43254958948701e1e5fe7b99","0859fa2ab7764509a3d9ece60ec379bf","6c6dfd704c50401f81fd6bab3fd8e7a8","ddbc4e0b90d446878a00296527552d9e","d7ad6ff53d794f97ba18155038042615","0c2c8d73c95649eca9bc5afd97ae9ddc","60fc5356b70f4f4ea0c736bc47ba5e82","e00e98f3550144f3aad0b161323907a0","aead515250204e488a73d572829be9c9","efb7aad94c8c4e72994e64dea67b1271","be5b416e49474b588fb0000989c5b6b1","44f811a17e554f90804ac204c883ffd0","9bbfed2ab6044d8ba87176f29dd8a479","4bb795f3f2824e2e9929f9815787582b","4117d870d64747eb97bf52456e6eea0b","0991db3a6cf442acb396adda7cf1d9e7","57ea2a54938842f28e3967ab19338165","c436830fcffb468192cbd05f3ff6f521","f1457eb6091b4a0bad1a2bdd6c2eb021","6b5964e1fc6243d0a503405bd5fb3e39","90496715e51d4bfab1137065681c9921","be6ce7baf4f8499c891790a45bd2918d","f8f4d7f2ce724bcb8c1138553fd4a5d6","0eda738208ca4abfad9bdf7c7242e420","faaf732ea014444480ac0e63b331ea97","3701d94822064f259058c32875c12816","798749bc7ce44a8ebcd7dbaaeba91301","38197428cace427eb877d9df2dc8ba61","9ae8f26e24ff44e5bef5facc2665c41c","57722d71cc7d4d7f98a56df39e445cd6","518576a310a044b8bcdf66902eca9d02","bb83e6df8656425fb855240b40d559d9","7c3da321f94045e38be71b4053f3a8bc","17d7d204fcfc45aca93c8c911b73d505","3fb6b79fd3ad4b57bc691bafc008660f","3b4201977cf9450b8a0e9fd71df61067","71f7100aab72407487996a211ca1e832","971259dd396140b6b20974bae82b3280","1621fa3675fb410a9ba8a0a64da88cbb","49abb9be43d348dfa5df3012b39a6853","a4762e7caa214b9f90f9bce9a00979bc","32935676e90647da9115a2d7c45e3768","ab0f7f85a2ee435caf9b649e4e89961c","1ea88fbb7ce14e339dac6c3f122ba3c7","3adbfc7d554b4c1e87c48bb46132f0d2","1bcb40e361c547ea86ada2d07f66ea4e","30bb982cd72443bda3938937b365f2af","b602f01384d74070bcc76e312e63d85a","20d6f036dcba4c6b85a91e9c73a40cca","ce96b3d84c504515ab0cd66ae4eeeded","6cb36105169748c39492c3569b419b58","77cbda44d5f14b8db3c2984c54d58c21","db412f14b0a54e0989719091b78423b1","934c551acca5419d9d7a294b20d127e4","6f06b70d679546c98ec007d6eaef1d08","6827cf9798a849939a10a17e4306c4da","196f216e86264341a8d37028edcffedc","c97d25a21aca45409d9133b4a10e1bfb","ba378aa1216147d494df154efa51f62c","25546c74d08d4c4591bf310057d34942","0054d1d788bf4c8ba4b2472992c89e06","810c5792fea545a2a40b8cc51a6d50ac","d9aeead6a612458b9b2bb4b0b0b74459","305a0d07e2f841759d1a07379bc042f1","b79cc6c5b54a429caaecce637f16ea47","feb4c24378264ff5a56329d4b45d3a11","07e26f7f828746ec977aa18ce1c5b8c7","2a2a7b0b3d914ff6a488ad1070c7dfdd","d65a74383a3b4cf8b44ae1ae22e660b4","dcd15c93ed2748ffaf0aa17526cfd714","2424ac20648343a1b2603b716ba4fd87","c4b1be860dec4cad817d25d78dd3a568","312ca1028195486dbdbda00b0458b56a","7861960dc4eb4a879db6165f430a3485","59e9ab03f2db4616acf544062462bed2","f53c53e5c7714b27b8bfe928f6b0d081","7dc674bbe7a04fa59e4d9e9b268d32e6","ffe86fb3119f446b908eae4fd4e218a7","99300600de8847ec93939b8694fd3975","b170446e153a470a8c0a960dda54c53c","2b7eb64842e14001a070c580fb7e9458","2d518733b3d04cc4801eb6afdfb5d8e3","9791287b659842e1bc884b9bce35c991","04491a5e7bfc4cedacbb65aad16c63ad","21ba9c9ab96847709cbbab9a54b79bf8","8a0c306ab9a64524a0464af1a3b9ea2a","03d8ee229ea146099e70a2eed310e181","16e15acd67614143aea32b54880875e6","e001aee9a6b944b9896142536b2c9045","d0ea5af060db4251b587a0b80539283a","8d43d817e9ef40e69929dfef73c2a679","207b4f24800344e88f7ec941f9ced720","e6918311940f4da4be42ff5d1cdb5adb","24a97b8f15fa4d15a1908f7d0a491a2d","f87952a5ad654568872da47aa3d4afee","5d55188da98243b88ea68e297494abe9","a109cc3660ea49d38cb25b7bc861ee7c","81d1d4abe6a54cdf8582f4e9c6b4827a","33fab451902b40e7a873946ef717d3ae","60831f6755664505b45bdd10e25ea0bf","28aae1edabf241989237e5a446210f42","b9cc2a33e1384ab8905d645886cca071","8e04b6c2928841d7927f28f2b023234e","6ea4c17aa5464fd89f91f787d0a6ad32","19a49f4420944212824fd1a7541ecb15","d91c36dbcfb14d19ba85928c926e7549","3b841a0e673c4e36b477b948af9e5805","bf145ca2eb264395a3fc1fd61ff7b227","77ccc65f928f4cef91d2ab1c6fbd6d15","e4135426d6384b4899006261dd491e64","7a6bdaad713d485cbce87f7756a42e54","e9cd6a10e5f54cbf97a4580e36e04237","2f0bfa9660fc4b6ea6199e2c7603bdb1","173caee44a874609a057f9d92d495ca8","a8d999d17d2448ac89fbb1c46d7e95dc","bb5f69c24ed64f70a63f4b1cfa78f8fb","82780bfee37f42be8089b00f95825768","9dad734bb0b04cb2a6c53b64880963d4","3b44a02b78434826a1bfc8671a96eed7","ba22ecbbf35d4d1fb453c08123191325","0ff140ade11c4ccf98f7d169e97152df","7462302f6fca40df975d4041d78a4897","9b2ac9f12d2648d0bc84ff8e6002ef60","e2585d01d62343dd8bf3937a36a739b1","58ba64dcc187478b9ad01d4e4e53716e","d3495b04062741a2828127e7f79593b0","4bd32ce4ec07480da6f50f816d3704e9","7ff6a0f775b04308bdd6d3b4fce51764","f8573da2f9854215990f605c78a80cc8","eb3e905d04fc445b94bb1a3c473b2d4b","c66ac24011da43b7828f54430f7b0667","94a3c09e73a34e029eb3c29884cb0f9c","ac9b03a4d22045e7be34b8116930865d","e1f58a4904a446e2a450c4df6144c5e0","9e4c6c6c34604458825c5da3530a2bf3","ba1e0ca3afb34fa6a1f6a024745b5dc0","0809fa4a2888423a9ef97eeb5e4f1a60","04326aed601f416194e1a70a598abc18","37f3b11384f14f8c9b94d587d7dc6fa2","5b768691e00d48e98d3f8899a4437420","ba2d0be33ded47e6a457993fa9def5e8","166e9d3138b444a39dcb42efe259ac6a","8dbb0e6073244354b86507f3e33a113b","3dc805dc47c643e19f9b9d222f7a7f0d","25a763a216a74be0bb84d1848e419d90","db1fbe17dc5b4c58af627474ea9aea3f","cb791b818af64c4fabb19552650f6295","98b7e3c73f824bf6a1c295a56de7d682","835621e2ba374aa78f44d6c508ed296b","f2938a90717e4ed28c7f8584ecb76fe1","9df573f0a34b4654840d5370af5e9212","3811d8b5a11c46adb189f822412a4872","eb378cc881ec486eb992948e1bdbf0d7","71902de491834536b1194dc46765de57","090ae5a595f44b59a8f248d1f740cac9","c09b2e98aa744e3da6e5110eb1252e0e","60e4b4feec38486f96eb075e89dc8d9a","cd12f8b58c19464a8d05993e261f0b15","9e2876efafd84af49fd70741906ed9bf","a148a06e19654464a603b25d760833c0","985e182f4450419796be72256cc70190","08435c07626d410ebe6757dd24703b24","d0928e23276d45bab3fe0d7f20d7b4ab","ec234bec2534408f80889d820a54f6c6","0d4e32eee09246cf99fbd31f28fba214","69f8dd33a14a4b50ac7994cc6b16b411","773c40274e2b45428a7bbfb19b6ee690","c0e4c752431f4e3593dbd77e001eee62","935b9acc374c4d548508f885fc6227a9","42d4676308894d70bbb8082274130b2c","72f0941e1abd4b89a689866e5275a315","f391a572d84f462ebff52c15711484d3","b40a1a6e80eb4265a0690eb0cab6947a","d69b78fc39e94155a2d869144a6f3784","ea1fdc9c158b4d898f91a664ac01b6df","7477b8c7e1f6469aa9c868870bff7fe1","a542c353bbb940608438d4d33a10b17a","e2417219645941a5b0d0507b9e396fea","7d96c23469984d5b8eced8a71aa7dfaa","800b1c365e5a42d7bdaf1adb82ffc032","22a020a53ff94b79a02762b2dfcf1b95","0859cda235fc423b94d19b86aa47ad66","c1b844a0f1164d2cb362b2dcee6b8760","8b88b8c9bcce4d47b17a69226d365f4f","0da41d632d034324bd5c8855d9bd009b","5ada2a253f89490d90a088dab65c945a","109e2e4f30824e3b99cffe87ec96aa2a","bd54d22adf6845b69a303eb2ed5127db","fa3b1b586e9d49f0b8f8bbe5587ce126","574ff11707764819a8c71505b8e1741d","90c3c041eb124813ab188c739e666a01","87030ead123f4fe699f7235fe8b401d2","9b132133815b4b67b4d79fbcfd8dd9bb","09dbf9bbfd1d4c7f9db74fc1916c69b0","73c781146ced4a84ae4bfd2357e56520","9d65b49789eb4a24be05ff84a84f7d6b","c4489fd85dd34a85b138a2069c0cf8ed","7a418d3674374e81a199ecced5b48392","90b8ef8a81e74a58bdbd780bf9e52527","7c6d6006f1684eb783858af378569668","6ebaf995fc2e49f8ba1e26ae02dd1167","31fffdc377b346af985938644bfedde5","cf761e67322e4d7e80057614e890636b","3dda42eb6bf441c4b959e4334aa69b12","edcb35a54ed846659358dd753f334051","32d226d94dd545c7a4c7a2f09a30bbf1","174870024aab4d20b082acaf165d331d","3876f9ce8b284399b2b9649839e1fe02","9d575a923fad4f7aa7b7268a74503196","cf4d06d76e0e40bd87f7e1f3abb05e69","deea867310294a838b7d76c4dc876e30","6e0458ad042b46b1a2a345f79c884429","52f23e66b7d8473f943e2bd145c908ff","441ee3967d624d0cbcc4239c1751237c","df12288a746548e4a304a726c63bbba5","1b46dca39e954201b0852959a1c4c10c","b7ab6dcf8c2b45f489cd3627493db567","c0f00aab3fc2495389b9664975acb8d0","fcb8e1368fc2457c9d395fc3aa498402","076e78d7968f45c9a2d9e07c0d637e9e","d3739899900349749c6b1f73c27e17d9","cb82757552d041458a50de12d82a1e18","17957ce99fee47bd8a8849c79e08e304","fe61a9a41ba347b9ba49a5e7d5d0f5b8","47a6367ec63940d4a1be218c64f3b6d8","fe4df2995a624f5cbbc21ac2e8867671","6a7aeffe10f849cd8428da1c73e5bc5a","d35562b393334dea96aef8ca76263b7d","c0a4b3b957a3427da3ac9ffaba4e5a5c","2d16a44a6c6f403a81bce7986b14896b","5cd25146117c4a1b8feb92bd5589af02","498794a26d4349288751a0bd1224b452","6b9a5ab9b3e448ff815500fda597b3fc","730f13e59f744db099078bd58d6ccd0a","24c218447be94737b439e9df45b48038","5e396a7b01e54ff6a2f8a1cad7e43dbb","e074c54b514b409cbe623e98251bd14c","200d2b3abb96408b9fb8924e36e840fa","f3d33a1bc60044d1b51aaa7da072714b","e60d3a18c6ae4a948beb4c89e82f1fa5","aec35d01a9b6422b92c82cfac0ae1cf5","1cc2d8033406477e91ab45cb89f8a741","69f3067b87c24f8a8fab2d3b37f7dc40","c43b0ce8e6a74e6eb2260fa694750c84","e30f0a7a645e4392b4c27281edc1dd0a","f448bc3649e04ad4bc52762141f08033","861fbe6e0b614a969c14a49b2c0a2178","42c7bcfeeba04f34910e1b23a09b1ff5","e50ab5036adc4ab8b6785158417b63e7","8bbc74bdec5b40ed96f468cefdb41be7","64b359513a8840e79a964a10725daf4c","809cbe2e605240509274bc37cd6d1db1","b10f9726bac5425bae383b90098fbbd5","f25104f7dcea4031a7c3541d21dd6d99","3040fc326fb14b15a9df1f73ac1df872","b56a4f4aea4e4125b30b44c47fa59bcd","5b6d5116f19545ac84f11cc5cd95b1ff","5436d2df1e0f46afb492a03f849eb52c","de6a5d30e10841fe862c5117515a615e","3ef1000b6c74494b8c9671b6d4240387","1a0f644abb4b422fa4b689baa147a4b4","3afed851917b4e67822ed136d6d8f0d9","d93243760bfc432bbd1c1952de2177c0","c1005321f2dd4114bc203299b14edd55","50f33aaa6c6948738d50bd913ff5e2ad","860c4d8da93e45b392ab36172abaacfe","0e65ba43e6a04225b9725e9a2cedf272","b3e148d0e2ca4e77bb8773fd15c88747","69238751767f4ed19b19b26f9e1a61bf","e77af656852f4f60aadb9631678ae138","26d6d73041ab449fa335576dbf933cf8","7f4a050d4b8742daac16ef0ad9f09807","30f67e017b6740b0b4a04e7fde06eae3","af4641e70222411b97c81d0e7c88a302","e8fafa172d8f4751bfa18334db466c26","adceb29c304746f898e708d042b85575","83037ce554e14c8c977458e62c60e481","4fa630c008b745f99468331276cc83d5","1e22872207b847f38ee7a715601142e2","be0fe09d5db044368c3590f366fa4df6","a2ef668488b0424d90700855d3e302c5","fdb7d11f3302431fa960798371ead030","196e78d502ac450fac3d8a85c31f7a82","7691217ae4a74f1daaee9f7088e0cfc1","0dbb447f0ab1452e83cb6ea2394c1147","cd2b543b23b34f09913684c273be3639","68590eae93d64d77bd5343884ba2027e","f8c70f32e2b84044b4b72d6ded8cb5f1","02992fa870a044d883987b60db7ee137","e0d5c1df49cc42e78e56de9b6c6f18d2","5e9e8daed9ae45cf91427e6676f9ba26","14f45256198e431c898f32434cf9c4c0","cd3036815f95411c901b277abf26066d","49bed964498847378407c1006436cc84","8fdc280797b34e20b5c3b7b2691ddbae","b6b4a12026d443bb93e1e812242210b3","47490d21cc584e6980c2b3ccc7f31252","8ccf8f4fc5fd472bb5598be9abada985","de59ede900fb4d2c8dea6f16ac58c4c9","cbf16b4b8612418da193c248945b217c","36703bc738254e798ff58fd950a8286a","24227e837ea44c719b93feb8179037a7","1b0551d95bf444ddb121e410559508d7","9e44a9be7c92461885888134efc64624","f2227ad0fa484f16abfbbc0c22f98dd6","430e7e54da6e4e4eaf40895c577795a8","cf8df510db74462bb06bc0e7be25daa2","35f6390b61194edcab859a7467b0a890","bc68ce9abb5e421ba7ab60a9bb11af75","620fa4ea256c42a1bc94cc2211960318","5e95d076c35d4269b8ef962b21dc0c6d","2683050fae9240ad9710728db708861f","ec0531af1bc84131b61cc218eef02daf","f4e89d51c9224936a5d839f36f2fd501","40f1225169e24f4d8cec1f80fa696c39","23c63c0618e843d2a3a4a44d7374bec7","59ea683f41694029b46b4f07872e533a","dea5761846e2455596352aed5fba9602","c51932cdcc2f4f61b5879fbdf0e0ae4c","b2d6d7bd6bf648d6a740aa7a70402d1b","8f6dcbd988c249e5bf35d9757111d585","dedec14dab614486a6feafecd990ec84","86e34b070a4d42d7adf6762759f4407d","c95e70d65dcc42cba9248e4334d1e176","01434d140eca410e8745e1215bf1fefb","48e82748c8bb43ba9ef99d0f707000d2","25e94db2d34346eba1e0bc42a321ce23","b1a23333630047b0bffb03ba74441786","920ad935251c4972828fc6c3f043f132","4f282684235d4a02bcc9ff54d1e34d60","1a6c836b951642d59adaea033c94ea14","2ac9adc70f0b49c88a3b445e67aa312e","d01fa867e813486aa062608cd9adcd7a","62579df95b834ece89a1b1d2996b117f","41cd45b5d661474b9a4b8be7b0fa7339","1a840f291ae745a19988416f7cc68b12","35a0d255c6664a2fb5ba937c4ba4782d","eb336aba67d147c8a660ecfcfef01f21","5c42b55981a348e3b90bee4ad176e767","a1110b083757480f9af5c9b41caf07db","20ec467f52b6418bba0272f54a956d14","e5ddc57632c94731bceabd16b8ba2db2","cf6509d24a61446caf95ae6704d91d45","6aefc36b55a6463792803e9f96b30729","5b96c43215f145a582e79644743a06f0","179d24996e9d4a79815d0f72bef79e2d","6a3b3a05b4434b1b99c8dbee8d2e0a17","1dceb63e845a4332b080ffee7246b8d5","dee7b2fa51ab4bdeb9f7f8421a1fddbe","69c0856a314b422f9cb2b9a70887199d","ffba12776cef44129c9936ff23c7e97d","b6115ec16a4e446f873fb9861cb2759e","d213c1b771354e36b6f2e3e4ca593384","f9a4fd81ab944a47affba0901dbaee39","8ecd206a1e164eb0b3832c307f4e0eca","17ed9cc6b1504c23a637b32d340ece0c","4f624b707cc7475faaead91b508215ef","fedcc602da9e4d33bef4c7d4efe2cdfd","6db1af3e9b734055a1c105086c8a2553","3e59f5aad0ba4248a195040d52f58961","6b82460eb62e48199bc7acc134b6e478","9b100766f0b44471a5415e57c6625385","c4307e1712e14e179c5360c7813bc031","2551ae79e3354d27a055f4bcea6ade0e","9ef7da027d174b8191eece7d8640b4bd","c5796767d0ed4bad8d2206958405b886","ba34135ae45a4c3e9ac905a628295ddd","3023cac4c45946a290dc588f01dc67dc","db154e83b72942e3b3b9c6fc95d19317","4b820f05742a48529dc96b749095dcae","b83e89e7625a40e48b295cff2a8ccfdf","83ed1572e0cd452bad9c09d513a60a4b","8e9958779d5b4ec2b68b3392187d4cec","a6e3ffdc094b4ebab75e2d15af58900c","5132f80498ba445fa9dca4ea68a837a9","3aae4efe6f1a43ba9f273945b0ab88ce","850a77392d76484da6ca8caab08a91b5","f322e6bbe8644e9893db095ef3909a05","d4025c085c7c4ee38dae66327a1bb28c","52eeee61c14f401db42f6bc246a2b4f1","ff5ba691d930485abfb9961320fd3456","70ec28133b6c4e91abc7186826598fee","5f1467ec1f5e440294a49f58c639a07b","1c3ef650221348099251f50de17d63d2","7013ac84768741228feec42b38732970","5c810254686244c994d95b5111e25b38","31ff1d5edd974f5ea1f854b88a9332eb","866a9f3ed335486bac2bbfbbaedfecba","c6f3a5c6b313451995e3ea41b8bf9296","f67fa5fc5bd347e394c2010687a7724e","957c77360e0948038ff45e34fb230df0","2fced687a73844b290a7cc87b85a8ace","58b15c95a28a46ca9ba8a16236839ff6","8091d47ab1ef4610ae49a0414531b9cf","fafcccf5e0774bd0a478ce5b5a323351","558228d59740442e832d40534ea823d3","0dabca880b60497980042d730896f8e9","28424d5d85384c9e86c3a9634965916f","99210e7680a044468ba6677849f4c753","82b536966beb414699990fb2ff46d4c6","83976e90324d4c6b90173634662ce966","71053655895648488ae1d0d4248c9ff5","2ee02607aaff4a7eaaa92a6fdcde436d","3fa63f1c6138400ba6ad6c709ee37f79","9cef327d2b2c4a51a898a5d154b91556","c720054a5b014b14a97fa007defb4ad2","742d81128cf646bc99d99c4f943f92ea","ef69d21fcd854d6bb5cb864741b0f7af","c128377ac5a044edaacc98abe66c6ca0","933c33a4499646778a679d3d125e4aa0","445d985206484b2887f4f275c0dfee0f","958337241be84d56848c10d7d4f06233","b243e09747034abb9de8f998b9cfbaed","79b53f43d4194fcc90350d3dd79fd7bf","686e063ff3d5422d8a4a3f08553e9b4e","f13b81222bfa4e74a80096f3e476fe79","32bd9ec427ff438484902ba44ccafe00","7454a6c7309a4b6ca2e559101b6853de","13dcdd9a6b254306b4647a78d65b4ca2","36096e01d4b54ce38b6ffd257ae8bc6a","8c2818f43bfb4891a76e63d587159d93","0ca96e00995a49448da38378fb21593f","77d9aa6c03b94b09ba45bf29e416a603","10ee481c890b4dc4af099c17b074adaf","428e704de79a48cca7f9bb2a00dc2cf1","517f7ca421404b14902f86e9835abe01","66c657f697174c9c8ff983ecb70635d0","c1c548a8273748ddabb2e21f20ad6c8a","a86103bb4b22439d8be1f8f6f0e538a4","2a56b80c189e4f11a418a9e7eedfa70b","a441c365d1a949b1ba6ba3de1ebf705e","a525832e1581473b9c905d43ac755765","33a3df8f23e74de688385ea6966d60a1","75b3dfbfb19542ffb68441020cf857c7","74745996703b475f836f23acd2cad6b9","197d5ae29b8342a6857b1c9adcf19b1b","179d4f3613874cf9b3e8c13a1682109b","93943b016b354e5581cd4fd27331c588","956d3adb191a4e4cb454e555076596e6","a52dda18eb02436aa05d93ee6cd2d293","ea4c7af5d8044a55824e277c1ae3b443","08d519ba69e24bb6b9b4e71fd2ce25ca","8f144fe3697b4db4b2064cb72da0962f","fa2cb35f22404da5969d3a0dcd743a19","ed4699ec4a314447bf0a43c133462074","a67056da37f74dceb4cf71e18ea862c3","e9be16f790634d6f9d16923902bca068","d1d7312e6a8748c882ad3d46a918e629","aa34ebdec80c419ebf6222109f326b9a","1ee5f73417bc4c0b9aa35c19fd757c89","4fabe138882e4528a44eafb3d76cb6f6","b6e9ee7cc4504326a3cb01f817e62119","12a02fba26da4df79049c025f99cf501","af8ebcecd3dc44e4b9650cf11f20ce8c","b92a80cdfc4d4dc7a93ca6bc7109110c","6a8fc129d6344bab9c879b3e1cb9b199","93142b9ba4b84a2990f1fd12a0d34dac","d22c872269d041cdad7a6e0d9a63f5e5","4cfd8a8b7ef64d06887ca1200ccd7c45","580e38767dd645ac8d4edff8dfbd1416","73fa5b856bde4235b032dda0b8c12722","a143cbd2434a4e0985284f6eff645952","bf4a5449b28d4598a4562a9621b68bb8","11c0335cfc7c4a249343656f5f9b9d39","424a266b77b14c519c2dcf07723818e3","6cd10c7bfc0a470f884dc3e17389314f","baeb0f5ab7ce40abb8a174370c372610","11794d1cd9dd45eca634d549e70f7c91","b3f20d08053647baaf6fccde5f3c313b","19698a9a0a0b4c7d8fc65327a291a05a","4dd9ada0368f408a895a06460ea44ae9","6b905dab19c24646ba210604f80e34e7","940b395e1b3c455d86468cc0712fee0f","9a24d32bb6c44325a37d12f238357a8b","ba5d8ada68ef44ce9e4c5c6512a7e293","8cda377991624527abcff22ef6024408","b3be1d29e8eb4607b98f03d5eb760732","beb10e6ad42a4565a3049c98b085b74e","f94bb66208a24e5eb030859a16484167","3b97a4d2484f4e75849d7bb799ce2367","b43ad6d87e7645d6b33f4496117e517c","af4c9e2e37d34dd2a43dc040c219869f","9efc006ccd4149e79f90709e0f688615","650433116a7045f5bd7641ebbbf86f21","397326b34bcd4567a5f414b687e91a7f","72c905c36cba46779435a4d0f6d42615","f20d31635dd44c398ed2438a4a29a560","1c669e42110245e8a4da4bd82320bb12","893a3df7523a447c90666d96aba494d5","236b6ed87dab4c178a2357c8cb2964da","d0dad0ee4ba74b7d8aa7693c6a2bc2d5","c47faa5bb4494928b08819f95222502e","bb39b5902e3b4f1399fa60cdec70d0fc","a3d4ca7173894f02990dc0384392999e","1891f81a709b45f99b118eb7f0574173","96df2e0cbe3c4bf690bde1aaab48fb07","ddb83383522d4559b1d387209752f87d","f318a67b29454ff282b36f5efd54db7f","44afc1c59215410d9882f9d5e8e9d4c6","8bd025074c464265bd5070266910a3cb","1ac345d33e894f92a16716ff002ba8d2","488a41ef48af40949c616ae2ecf0e036","002ad5b03c214dd4bdf1b8ee41c310a2","5e5b167cdae742b383abd1d5ce530e8e","1332211c11624343967723356f6b965a","9df362dd299e48e98e1497600186e8cf","f632f47971b8429c800e8ac5522111d3","104fa651b180484aa4472d46e746ab9c","d2ca87870527474abdeeb22cee1bb0b7","d1b6b419f9cd47e5b44b6d99e03157bd","a9235e1158b3454f9411d31ebeaa2089","2f8725285e434f4e973d69cd223bdd71","2dffb23ec2b142ac8647038f0d758735","75a6882f705148dd9191bad00f42a448","9ad0f8861b4c465581ce77d08d8b063d","7d2c8ac9c8844a04a78af3fb2e8bd9a0","b88fc27e2ca146dd8a62950b76534f6f","510a16ca46b24217bda728505a599110","6243922fdfd948b1b7f8bb0fab1fd1a3","048e9cad5a1c4d058a0fc845706c88bc","dfc6ec5277c2439b8c5636be64da33a7","399f3d667b0141128c3659ca3baa580b","5b6c3143ef2a44fb8dc7d9e2aa35992a","cb97d558d70745d492951221527a3ca9","4d2b66a2c8a343219bf46485b54bb4cd","34a1630a35cc4842b7ab260ce900b84e","7ba8ff0a8bc0459aa669343962370265","b4e9dd7b7a2441cbbb462522ae5cb527","b3001c008fbb4853b80acbd044285923","f4e00cb2d3fc46cc8f12787c1df6a2fe","57f51cb6a2b9457e99ed00414b1ba65c","214c14929d1e47cda3e273f2e4e7bccb","8de9c07b6e274f6285de21d825d13e7b","fd791f38eb6d41e1b3ebca8c8300a694","5d0cef0c621047ee9fe785628a313a93","13b85acb0a834129afec188f7e5f5609","1f6d0ec7f8cb45e88c3d18b94a6ddbac","ddc83a472df847bdb342ee9d0d312183","e51252e7acf642eaacc0d9d3557ea6e3","c2253cbc1cea457eaaf5d18f55e0dd8a","90631b1c41c14295a83d76d5d2708bfd","c910c56e752e4eb0be3a429c5c464ec1","8d6ee044481642d2b7efe618854add92","66720ecd5d564308ae9537abf796386c","db02fa05c81740a0ba2a4691e673d35a","98d1de15e751488eb9d4c2669cf45a29","3ef003d9a62645a78a11765575503bb8","827390fc79594891a9755e1acb202408","983f09f3272343109457407f0b337969","4f7ae95210ae4cdbb56277dc5af53c3f","4dfa210f96b44b5bab0964123b1c3cc9","aafcacd6301b45168b390b80669a4b20","f41f7c65f517462d8ae2cdc4cc95f4fe","5e4f38941d9f408cab263ffa93802b35","954f34de9cb2492b9ba36849a99de8ba","d1d1aab522ee43c38bfb9e99fc78272b","baeaceb412cc4714b4f2c4f39be96cd2","3a79f4c22a2a4f2f843b85afc4c9d89f","c37956e75e4f42159f486411760aadbe","1dc41c6bde60429c867d1728ac1d9bcd","df1936c315e441cdb7d8a243b419f752","db121121bd384d789e3b776b1b50c6ce","f9ce533748d24f229f83f08a684c1285","a7415db33f1143dbb8de2484652f1d39","4d09103f77824c229241793dd018123c","1948317f52fb40ec8edd7e64c1a62a31","32e71ccb416642278fdfd0b38700621c","05347276863b4446bfe50360ce120ec3","ab0ba327302244f88acbcfa71ef0fa09","f24d6a20e6fd4c2c88e8f6975c7b572c","23cf0d36b47142228868df11fa4537f7","0b0829b2ce154060a8dca996697e4905","09a41721dc934dbdb0a274ed10ad5393","5504bb9c952d4d9f9158b185c5b0b126","fce26a6f90c6425ca90ae7882f138630","4db70922ee6a461cbe2760331f1de5ad","131ea6bf5244452cb80141133eba99fc","ad1952ba8cb64e8a92c805214d9a7568","a4bd0137a58846b2a69729ac0f6e0a7d","324cbc39e0634811ad4c0b254ddf1f82","2eb389abad7f4e44b0609b409242853e","4bc7bb9972ff4da1992df065182d1d4b","8b4c54d448e94fb9ab8ae91cac0573b6","669d6a3b074545029153f230f2ce4be2","68e60a11dec4432a9e43b65cf4a2aa60","102d99916d8e40d994398d4e7985f828","a0fbe76d1a2644a9b80daa71388bfa0e","d0ee01fbf61841418da5050390bd145c","2bfe731a952b4925914055ac59f9d3b7","0ea887311c1843fab1cb9f07a07269c1","eebcb6a650644ab8a1a67632ce2b37a3","d5be9d7e4fda47d9b59031db23e486dd","146f30b1b0d940f2b6f40e943fa66a08","47be945ac3f74aec8f8e52b8288c4764","ded5bf10b9c741c9a97c3ed92397e3f8","13b0529806224dadb3003bc66393a8a0","3e95208a7ecc477ba83a8ca71b980c63","70336258d400491186277a7496f0993b","6b0ced3ec2494fd6beb7747c88d0b705","6289935e919349d9b32246b8693b0705","419ec030973f493d83723cf372b39c19","774187c41ef14bbaa3e9d3c780a6d1db","a6e38d9a2a584c5d8f79e3e9ac4be562","a2f8e7d3b1c24e6eb653f2627910d8a8","49d0b240625341dd80562ea3f6b97704","6c14ff7c6ba64ef698a014b7928a8a3c","6967231bf8d44373b154621889a863a4","e20337dbc1894121baf459979ae468ce","73919c47423f4a15800c42d85c4ae7b6","50248c51c8db4c3f897d00ceea70da84","e795012b794f4caaba8fa8ddf235e734","463874cb89b84b5cb9ab4b908b639b2d","00351e06106c487595cfe1cdddb3f039","5bb46eb9613d4adeab2820efac86e92e","3ce6fac5e20a4ec6b84a1c93ac1b11cc","eb7d892bcef947a0aeeb415db7478c83","368f59574a6c4e43affe904c0ed6f9c4","250fe10b54b64564a0b1102b7fee3893","3c89569b92b5428fadfddeb59267c3fd","ad29f941aa384e75898803957718d437","06281a8432b24a82ba70b7a5ece82937","f7ab0ee90c8c4bfab8a00c6dee340244","be5fd4b11a914b67b558df5c0d67db6d","24c0025637e240a889d0edbb570a461c","1e3a008aa07f473a89edff3a8ff3fbe5","686f9ec36dd64fc8b319ac3660d42893","6b7cd42d00194ad2ac0db151f1c78fa8","425f3b50fc9844bb87855d18d1adee7e","18f792efaca746b68b8fb2c016f98394","e0dbcbd4341340168c26e5d0f2cb7bf9","8ce1c7b2a0f044dda5fa631e79808997","1be10f9fd0ef47a98590193f26431ac6","105eb8d04a8e488facae3a9e8b1b4246","716c55d8ab5a49d08541a09bae84231d","6026e44ce2d1424e909a34b7903d24f6","3772c3a5f9cc478fbae54859dcb2989f","7e47648bca8d4ab08e5350f9e24e8f77","8fea29557e88469199c134be217af84a","a2f1370bcd3c4ab3b40f062478fe38f7","fb61f186dda04a64a658bea713b75d80","bb20f5da50704ea6a63871e41f8a0dc6","bdd159e261764407896cd04ee7c1f704","a6f38f576b9a4fed8b05a9d855ef2ec5","08373bedb2504b47a1f3f82790e36680","fe72085ce55d4abc913ef23994de1db1","f60d4c6403094e1fa258c61e9e5e4f87","7fc2c07bba314176a0380da6c5431df8","ab6e48c0a5c84a4d9ca9bdc68d258b26","3b7e948712ad4973a92914a2b38f770f","22b3ebccf90849f4a8eebb18b5885d0b","463af667128b495bad961a9716989ce8","0728ae28907f4d378b4c85a285544d61","c99da85bb91044bc8381d62c599d2246","5fa8d526c5694781b6a234503a8f129a","6b460597a6f94b2697ff705692347803","e50537a8c3bc44abb9b5a523f75c918a","1a04d76b4e484fc29a61bda8163f9bcd","98009d431f4644649c69618a0c4e7d09","2fca7af9bfcd44abbbfea2948e9c8408","75b429ffd09a4046b453bf052c413a93","0ec4ec65622b4cebb7496ba5bf216399","950d7e161779443d9b52b671aa52f94a","daadc2234e954511a2432af34f2382a1","722d734d9e97467c8ad0f1576da77e5b","037da24f8fc64f058b581366aed9be25","cf676574faaf43309fa3c2ea16ccebce","822437c544eb49aaaf870bb94bc74225","392721d48b2244f6875bdbfa671ae194","92d9cae39b0549d8b1f6e69bdb6bd23d","84543aeff71f4800ad353b4d23e63597","2c64bdfb46814d449097ddd7fbffb8bd","e18dcbc034b0447fa67a49cabfdd6dbf","39ceb44191b64aa19c5653b14ee86328","1b9fdd84364b4542b0c44ec7e65e9c0c","3fce6e9fe82748a093248fce6ca667f7","e35978b1e58e42e9887b6db1d87245aa","25a8246a2b5e4ff19a9afcd413175479","5a87e31f444b4705a7780539b7f0f32a","192659ffefa9444d8c1d91302b8cdf8c","4d7ca9761f994e1a92d8e6e58670b6a3","bf32d62a15ab4261aaed43a876d79a94","00eee8ab8103449aab0bcfa589786592","07aa810e50fa47d0bbd6c625e325635c","6fd705ce60264da098e9f8251488fc69","eeac415a06524045a5cfdf548a959217","4ece9edc9dd6484eadb29dd06f384cb8","3856284bc140489e8cc079b2fcb212ef"]},"executionInfo":{"status":"ok","timestamp":1593453433273,"user_tz":240,"elapsed":10401744,"user":{"displayName":"Markus Sommer","photoUrl":"https://lh3.googleusercontent.com/a-/AOh14GjXujbpCzD6oVvcRIT5EP-oi3Bif3Cfcu7npJUnce8=s64","userId":"08083804319459268544"}},"outputId":"c853528e-fa40-4f05-d133-7d8eb7125940"},"source":["# @title Find genes\n","\n","# This code is a work in progress. \n","# Much of it was written quickly and can be (significantly) improved.\n","# It is much slower than it needs to be and will probably get rewritten at some point.\n","\n","\n","def tokenize_aa_seq(aa_seq):\n","    \"\"\" Convert amino acid letters to integers.\"\"\"\n","    table = {\"L\": 1,\n","             \"V\": 2,\n","             \"I\": 3,\n","             \"M\": 4,\n","             \"C\": 5,\n","             \"A\": 6,\n","             \"G\": 7,\n","             \"S\": 8,\n","             \"T\": 9,\n","             \"P\": 10,\n","             \"F\": 11,\n","             \"Y\": 12,\n","             \"W\": 13,\n","             \"E\": 14,\n","             \"D\": 15,\n","             \"N\": 16,\n","             \"Q\": 17,\n","             \"K\": 18,\n","             \"R\": 19,\n","             \"H\": 20,\n","             \"*\": 0,\n","             \"X\": 0}\n","    tokenized = torch.tensor([table[aa] for aa in aa_seq])\n","    return tokenized\n","\n","\n","def get_start_codon(seq, orfcoords, strand):\n","    if strand == 1:\n","        # forward strand\n","        startcoord = orfcoords[0]\n","        return seq[startcoord-3:startcoord]\n","    else:\n","        # reverse strand\n","        startcoord = orfcoords[1]\n","        return seq[startcoord:startcoord+3].reverse_complement()\n","\n","\n","def find_ORFs(nuc_seq, minimum_length):\n","    \"\"\"find positions of all open reading frames in given reading frame\"\"\"\n","    if translation_table == 11:\n","        starts = set([\"ATG\", \"GTG\", \"TTG\"])\n","        stops = set([\"TAA\", \"TAG\", \"TGA\"])\n","    elif translation_table == 4:\n","        starts = set([\"ATG\", \"GTG\", \"TTG\"])\n","        stops = set([\"TAA\", \"TAG\"])\n","    else:\n","        print(\"Translation table \", translation_table, \" not implemented. Please open a GitHub issue if this is a problem.\")\n","        sys.exit()\n","\n","    ORF_startstop = []\n","    temp_starts = []\n","    l = len(nuc_seq)\n","    for i in range(0, l, 3): \n","        if i==0 or nuc_seq[i:i+3] in starts:\n","            temp_starts.append(i)\n","            continue\n","        if ((nuc_seq[i:i+3] in stops) or (i+3==l)) and len(temp_starts) != 0:\n","            for start in temp_starts:\n","                if (i-start >= minimum_length):\n","                    ORF_startstop.append((start, i))\n","            temp_starts = []\n","    return ORF_startstop\n","\n","\n","def get_ORF_info(seq_list):\n","    ORF_seq = []\n","    ORF_coord = []\n","    ORF_nucseq = []\n","    for i, seq in enumerate(seq_list[:]):\n","        # frame 0: starts at 0\n","        # frame 1: starts at 1\n","        # frame 2: starts at 2\n","        # frame r0: ends at 0, MAY NOT START AT THE LAST COORD DUE TO MULTIPLE OF 3 DIFFERENCES\n","        # frame r1: ends at 1\n","        # frame r2: ends at 2\n","\n","        seqstr = str(seq)\n","        seq_c = seq.complement()\n","        seqstr_c = str(seq_c)\n","        l = len(seqstr)\n","        frame_0_end = (l-0)-(l-0)%3+0\n","        frame_1_end = (l-1)-(l-1)%3+1\n","        frame_2_end = (l-2)-(l-2)%3+2\n","\n","        frame_0 = find_ORFs(seqstr[0:frame_0_end], min_orf_length)\n","        frame_1 = find_ORFs(seqstr[1:frame_1_end], min_orf_length)\n","        frame_2 = find_ORFs(seqstr[2:frame_2_end], min_orf_length)\n","\n","        frame_r0 = find_ORFs(seqstr_c[0:frame_0_end][::-1], min_orf_length)\n","        frame_r1 = find_ORFs(seqstr_c[1:frame_1_end][::-1], min_orf_length)\n","        frame_r2 = find_ORFs(seqstr_c[2:frame_2_end][::-1], min_orf_length)\n","\n","        # standardize coords\n","        ORF_0f_standard_nuccoord = [(x[0]+3, x[1]) for x in frame_0]\n","        ORF_1f_standard_nuccoord = [(x[0]+4, x[1]+1) for x in frame_1]\n","        ORF_2f_standard_nuccoord = [(x[0]+5, x[1]+2) for x in frame_2]\n","\n","        ORF_0r_standard_nuccoord = [(frame_0_end-x[1], frame_0_end-x[0]-3) for x in frame_r0]\n","        ORF_1r_standard_nuccoord = [(frame_1_end-x[1], frame_1_end-x[0]-3) for x in frame_r1]\n","        ORF_2r_standard_nuccoord = [(frame_2_end-x[1], frame_2_end-x[0]-3) for x in frame_r2]\n","\n","        # translate once per frame, then slice\n","        aa_0 = str(seq[0:frame_0_end].translate(table=translation_table, to_stop=False))\n","        aa_1 = str(seq[1:frame_1_end].translate(table=translation_table, to_stop=False))\n","        aa_2 = str(seq[2:frame_2_end].translate(table=translation_table, to_stop=False))\n","        aa_r0 = str(seq_c[0:frame_0_end][::-1].translate(table=translation_table, to_stop=False))\n","        aa_r1 = str(seq_c[1:frame_1_end][::-1].translate(table=translation_table, to_stop=False))\n","        aa_r2 = str(seq_c[2:frame_2_end][::-1].translate(table=translation_table, to_stop=False))\n","\n","        ORF_0f_aa = [aa_0[slice(*tuple(int(idx/3) for idx in x))][::-1] for x in frame_0] # reversed because model is trained with first amino acid directly upstream of stop codon\n","        ORF_1f_aa = [aa_1[slice(*tuple(int(idx/3) for idx in x))][::-1] for x in frame_1] \n","        ORF_2f_aa = [aa_2[slice(*tuple(int(idx/3) for idx in x))][::-1] for x in frame_2]\n","        ORF_0r_aa = [aa_r0[slice(*tuple(int(idx/3) for idx in x))][::-1] for x in frame_r0]\n","        ORF_1r_aa = [aa_r1[slice(*tuple(int(idx/3) for idx in x))][::-1] for x in frame_r1]\n","        ORF_2r_aa = [aa_r2[slice(*tuple(int(idx/3) for idx in x))][::-1] for x in frame_r2]\n","\n","        ORF_seq.append([ORF_0f_aa, ORF_1f_aa, ORF_2f_aa, \n","                        ORF_0r_aa, ORF_1r_aa, ORF_2r_aa])\n","        ORF_coord.append([ORF_0f_standard_nuccoord, ORF_1f_standard_nuccoord, ORF_2f_standard_nuccoord, \n","                          ORF_0r_standard_nuccoord, ORF_1r_standard_nuccoord, ORF_2r_standard_nuccoord])\n","        \n","        ORF_nucseq.append([str(seq[0:frame_0_end]), # all 5' to 3'\n","                           str(seq[1:frame_1_end]),\n","                           str(seq[2:frame_2_end]),\n","                           str(seq_c[0:frame_0_end][::-1]),\n","                           str(seq_c[1:frame_1_end][::-1]),\n","                           str(seq_c[2:frame_2_end][::-1])])\n","    return ORF_seq, ORF_nucseq, ORF_coord\n","\n","\n","def analyze_overlap(coords0, coords1, strand0, strand1,\n","                    unidirectional_penalty_per_base,\n","                    convergent_penalty_per_base,\n","                    divergent_penalty_per_base):\n","    overlap = coords0[1] - coords1[0] # TODO account for fully overlapped gene\n","\n","    if overlap <= 0:\n","        compatible, penalty = True, 0\n","        return compatible, penalty\n","    \n","    if overlap > max_gene_overlap:\n","        compatible, penalty = False, 0\n","        return compatible, penalty\n","\n","    # get prime locations\n","    if strand0 == 1:\n","        threeprime0 = coords0[1]\n","        fiveprime0 = coords0[0]\n","    else:\n","        threeprime0 = coords0[0]\n","        fiveprime0 = coords0[1]\n","    if strand1 == 1:\n","        threeprime1 = coords1[1]\n","        fiveprime1 = coords1[0]\n","    else:\n","        threeprime1 = coords1[0]\n","        fiveprime1 = coords1[1]\n","    \n","    # exclude ORFs in same frame sharing same stop codon\n","    if strand0 == strand1 and threeprime0 == threeprime1:\n","        compatible, penalty = False, 0\n","        return compatible, penalty\n","\n","    # unidirectional overlap\n","    if (threeprime0 < fiveprime0) == (threeprime1 < fiveprime1):\n","        compatible, penalty = True, overlap * unidirectional_penalty_per_base\n","        return compatible, penalty\n","\n","    # convergent overlap\n","    if (fiveprime0 < threeprime1 <= threeprime0) or (fiveprime1 < threeprime0 <= threeprime1):\n","        compatible, penalty = True, overlap * convergent_penalty_per_base\n","        return compatible, penalty\n","    \n","    # divergent overlap\n","    if (threeprime0 < fiveprime1 <= fiveprime0) or (threeprime1 < fiveprime0 <= fiveprime1):\n","        compatible, penalty = True, overlap * divergent_penalty_per_base\n","        return compatible, penalty\n","\n","    return True, 0 # edge case of exactly 1 ORF\n","\n","\n","def predict(X):\n","    model.eval()\n","    with torch.no_grad():\n","        if torch.cuda.device_count() > 0:\n","            X_enc = F.one_hot(X, 21).permute(0,2,1).float().cuda()\n","        else:\n","            X_enc = F.one_hot(X, 21).permute(0,2,1).float()\n","        probs = expit(model(X_enc).cpu())\n","    return probs\n","\n","def predict_tis(X):\n","    model_tis.eval()\n","    with torch.no_grad():\n","        if torch.cuda.device_count() > 0:\n","            X_enc = F.one_hot(X, 4).permute(0,2,1).float().cuda()\n","        else:\n","            X_enc = F.one_hot(X, 4).permute(0,2,1).float()\n","        probs = expit(model_tis(X_enc).cpu())\n","    return probs\n","\n","nuc_encode = {\"A\":0,\n","              \"T\":1,\n","              \"G\":2,\n","              \"C\":3,\n","              \"N\":0,\n","              \"M\":0,\n","              \"R\":0,\n","              \"Y\":0,\n","              \"W\":0,\n","              \"K\":0}\n","              \n","start_enc = {\"ATG\":0,\n","             \"GTG\":1,\n","             \"TTG\":2}\n","\n","def tensor_to_seq(tensor):\n","    table = {0: \"X\",\n","             1: \"L\",\n","             2: \"V\",\n","             3: \"I\",\n","             4: \"M\",\n","             5: \"C\",\n","             6: \"A\",\n","             7: \"G\",\n","             8: \"S\",\n","             9: \"T\",\n","             10: \"P\",\n","             11: \"F\",\n","             12: \"Y\",\n","             13: \"W\",\n","             14: \"E\",\n","             15: \"D\",\n","             16: \"N\",\n","             17: \"Q\",\n","             18: \"K\",\n","             19: \"R\",\n","             20: \"H\"}\n","    return \"\".join([table[x] for x in tensor])\n","\n","def kmerfilter(seq):\n","    kmerset = kmerize(seq, k_seengene)\n","    s = [x in aa_kmer_set for x in kmerset]\n","    seen = np.sum(s) >= multimer_threshold\n","    return seen\n","\n","def kmerize(seq, k):\n","    kmerset = set()\n","    for i in range(len(seq) - k + 1):\n","        kmer = tuple(seq[i: i + k].tolist())\n","        kmerset.add(kmer)\n","    return kmerset\n","\n","\n","\n","# find genes for each uploaded genome\n","GCF_list = []\n","contig_name_list = []\n","contig_length_list = []\n","contig_seq_list = []\n","contig_gene_coord_list = []\n","contig_gene_strand_list = []\n","\n","for genome_name in genome_dict.keys():\n","    if verbose:\n","        print(\"Reading fasta file\", str(genome_name) + \"...\\n\")\n","\n","    # read genome sequence\n","    seq_list = []\n","    contig_name_sublist = []\n","    contig_length_sublist = []\n","    if os.path.splitext(genome_name)[1].lower() == \".gz\":\n","        with gzip.open(genome_name, \"rt\") as f:\n","            for record in SeqIO.parse(f, \"fasta\"):\n","                seq_list.append(record.seq)\n","                contig_name_sublist.append(record.id)\n","                contig_length_sublist.append(len(record.seq))\n","    else:\n","        with open(genome_name, \"rt\") as f:\n","            for record in SeqIO.parse(f, \"fasta\"):\n","                seq_list.append(record.seq)\n","                contig_name_sublist.append(record.id)\n","                contig_length_sublist.append(len(record.seq))\n","    contig_name_list.append(contig_name_sublist)\n","    contig_length_list.append(contig_length_sublist)\n","    contig_seq_list.append(seq_list)\n","\n","    # get sequences and coordinates of ORFs\n","    if verbose:\n","        print(\"Finding and translating open reading frames...\\n\")\n","\n","    ORF_seq_list, ORF_nucseq_list, ORF_coord_list = get_ORF_info(seq_list)\n","\n","    # combine ORFs to submit to GPU in batches\n","    ORF_seq_combine = []\n","    for i, contig in enumerate(ORF_seq_list):\n","        for j, frame in enumerate(contig):\n","            for k, coord in enumerate(frame):\n","                ORF_seq_combine.append(coord)\n","\n","    # encode amino acids as integers\n","    if verbose:\n","        print(\"Encoding amino acids...\\n\")\n","    ORF_seq_enc = [tokenize_aa_seq(x) for x in ORF_seq_combine]\n","\n","    # seengene check\n","    if filter_kmer:\n","        if verbose:\n","            print(\"Applying kmer prefilter...\\n\")\n","        seengene = []\n","        for s in ORF_seq_enc:\n","            kmerset = kmerize(s, k_seengene)\n","            s = [x in aa_kmer_set for x in kmerset]\n","            seen = np.sum(s) >= multimer_threshold\n","\n","            seengene.append(seen)\n","        # with Pool(4) as P:\n","        #     seengene = P.map(kmerfilter, ORF_seq_enc)\n","\n","    # score\n","    if verbose:\n","        print(\"Scoring ORFs with temporal convolutional network...\\n\")\n","\n","    # sort by length to minimize impact of batch padding \n","    ORF_lengths = np.asarray([len(x) for x in ORF_seq_enc])\n","    length_idx = np.argsort(ORF_lengths)\n","    ORF_seq_sorted = [ORF_seq_enc[i] for i in length_idx]\n","\n","    # pad to allow creation of batch matrix\n","    prob_list = []\n","    for i in tqdm(range(0, len(ORF_seq_sorted), gene_batch_size), unit=\" batch\"):\n","        batch = ORF_seq_sorted[i:i+gene_batch_size]\n","        seq_lengths = torch.LongTensor(list(map(len, batch)))\n","        seq_tensor = torch.zeros((len(batch), seq_lengths.max())).long()\n","\n","        for idx, (seq, seqlen) in enumerate(zip(batch, seq_lengths)):\n","            seq_tensor[idx, :seqlen] = torch.LongTensor(seq)\n","\n","        pred_all = predict(seq_tensor)\n","\n","        pred = []\n","        for j, length in enumerate(seq_lengths):\n","            subseq = pred_all[j, 0, 0:int(length)]\n","            predprob = float(expit(torch.mean(logit(subseq))))\n","            pred.append(predprob)\n","        \n","        prob_list.extend(pred)\n","    prob_arr = np.asarray(prob_list, dtype=float)\n","\n","    # unsort\n","    unsort_idx = np.argsort(length_idx)\n","    ORF_prob = prob_arr[unsort_idx]\n","\n","    # recombine ORFs\n","    idx = 0\n","    ORF_gene_score = copy.deepcopy(ORF_coord_list) # fill coord matrix with scores\n","    for i, contig in enumerate(ORF_gene_score):\n","        for j, frame in enumerate(contig):\n","            for k, coord in enumerate(frame):\n","                ORF_gene_score[i][j][k] = float(ORF_prob[idx])\n","                idx += 1\n","\n","    # create strand information\n","    ORF_strand_flat = []\n","    for i, seq in enumerate(ORF_seq_list):\n","        if not any(seq):\n","            ORF_strand_flat.append([])\n","            continue\n","        n_forward = len(seq[0]) + len(seq[1]) + len(seq[2])\n","        n_reverse = len(seq[3]) + len(seq[4]) + len(seq[5])\n","        ORF_allframe_strand = [1]*n_forward + [-1]*n_reverse\n","        ORF_strand_flat.append(ORF_allframe_strand)\n","\n","    # flatten coords within contigs\n","    ORF_coord_flat = [[item for sublist in x for item in sublist] for x in ORF_coord_list]\n","\n","    # get ORF lengths\n","    ORF_length_flat = [[coords[1]-coords[0] for coords in x] for x in ORF_coord_flat]\n","    \n","    if verbose:\n","        print(\"Scoring translation initiation sites...\\n\")\n","\n","    # extract nucleotide sequence surrounding potential start codons\n","    ORF_TIS_seq = copy.deepcopy(ORF_coord_list)\n","    ORF_start_codon = copy.deepcopy(ORF_coord_list)\n","\n","    for i, contig in enumerate(ORF_TIS_seq):\n","        n = 0 # count to index into flat structure # TODO make sure this works as expected\n","\n","        nucseq = ORF_nucseq_list[i][0] # easier to use coords relative to single nucseq\n","        nucseq_c = ORF_nucseq_list[i][3][::-1]\n","        contig_nuclength = len(nucseq)\n","\n","\n","        for j, frame in enumerate(contig):\n","            for k, temp in enumerate(frame):\n","                if any(temp):\n","                    coords = ORF_coord_list[i][j][k]\n","                    strand = ORF_strand_flat[i][n]\n","                    n += 1\n","                    if strand == 1:\n","                        fiveprime = coords[0]\n","                        if fiveprime >= 16 + 3: # NOTE 16 HARD CODED HERE\n","                            downstream = nucseq[fiveprime: fiveprime + 16]\n","                            upstream = nucseq[fiveprime - 16 - 3: fiveprime - 3]\n","                            start_codon = start_enc[nucseq[fiveprime-3: fiveprime]]\n","                            TIS_seq = torch.tensor([nuc_encode[c] for c in (upstream + downstream)[::-1]], dtype=int) # model scores 3' to 5' direction\n","                        else:\n","                            TIS_seq = -1 # deal with gene fragments later\n","                            start_codon = 2\n","\n","                        ORF_TIS_seq[i][j][k] = TIS_seq\n","                        ORF_start_codon[i][j][k] = start_codon\n","                        \n","                    else: # reverse strand\n","                        fiveprime = coords[1]\n","                        if contig_nuclength - fiveprime + 3 >= 16 + 3: # NOTE 16 HARD CODED HERE\n","                            downstream = nucseq_c[fiveprime - 16: fiveprime][::-1]\n","                            upstream = nucseq_c[fiveprime + 3: fiveprime + 3 + 16][::-1]\n","                            start_codon = start_enc[nucseq_c[fiveprime: fiveprime + 3][::-1]]\n","                            TIS_seq = torch.tensor([nuc_encode[c] for c in (upstream + downstream)[::-1]], dtype=int) # model scores 3' to 5' direction\n","                        else:\n","                            TIS_seq = -1 # deal with gene fragments later\n","                            start_codon = 2\n","                            \n","                        ORF_TIS_seq[i][j][k] = TIS_seq\n","                        ORF_start_codon[i][j][k] = start_codon\n","\n","    # flatten TIS for batching\n","    ORF_TIS_prob = copy.deepcopy(ORF_TIS_seq)\n","\n","    ORF_TIS_seq_flat = []\n","    ORF_TIS_seq_idx = []\n","    for i, contig in enumerate(ORF_TIS_seq):\n","        for j, frame in enumerate(contig):\n","            for k, seq in enumerate(frame):\n","                if type(seq) == int: # fragment\n","                    ORF_TIS_prob[i][j][k] = 0.5 # HOW BEST TO DEAL WITH FRAGMENT TIS?\n","                elif len(seq) != 32:\n","                    ORF_TIS_prob[i][j][k] = 0.5 \n","                else:\n","                    ORF_TIS_seq_flat.append(seq)\n","                    ORF_TIS_seq_idx.append((i, j, k))\n","\n","    # # score TIS\n","    # ORF_TIS_seq_all = torch.stack(ORF_TIS_seq_flat)\n","    # y_pred_TIS = predict_tis(ORF_TIS_seq_all)\n","\n","    # batch score TIS\n","    TIS_prob_list = []\n","    for i in tqdm(range(0, len(ORF_TIS_seq_flat), TIS_batch_size), unit=\" batch\"):\n","        batch = ORF_TIS_seq_flat[i:i+TIS_batch_size]\n","        foo = torch.stack(batch)\n","        pred = predict_tis(foo)\n","\n","        TIS_prob_list.extend(pred)\n","    y_pred_TIS = np.asarray(TIS_prob_list, dtype=float)\n","\n","    # reindex batched scores\n","    for i, prob in enumerate(y_pred_TIS):\n","        idx = ORF_TIS_seq_idx[i]\n","        ORF_TIS_prob[idx[0]][idx[1]][idx[2]] = float(prob)\n","\n","    # combine all info into single score for each ORF\n","    if filter_kmer:\n","        ORF_score_flat = []\n","        for i, contig in enumerate(ORF_gene_score):\n","            if not any(contig):\n","                ORF_score_flat.append([])\n","                continue\n","            temp = []\n","            seengene_idx = 0\n","            for j, frame in enumerate(contig):\n","                for k, geneprob in enumerate(frame):\n","                    length = ORF_coord_list[i][j][k][1] - ORF_coord_list[i][j][k][0] + 1 \n","                    TIS_prob = ORF_TIS_prob[i][j][k]\n","                    start_codon = ORF_start_codon[i][j][k]\n","                    ATG = start_codon == 0\n","                    GTG = start_codon == 1\n","                    TTG = start_codon == 2\n","\n","                    combprob =   geneprob * weight_gene_prob \\\n","                            + TIS_prob * weight_TIS_prob \\\n","                            + ATG * weight_ATG \\\n","                            + GTG * weight_TTG \\\n","                            + TTG * weight_GTG\n","                    maxprob = weight_gene_prob + weight_TIS_prob + max(weight_ATG, weight_TTG, weight_GTG)\n","                    probthresh = score_threshold * maxprob\n","                    score = (combprob - probthresh) * length  + 1e6*seengene[seengene_idx]\n","                    seengene_idx += 1\n","\n","                    temp.append(score)\n","            ORF_score_flat.append(temp)\n","\n","    else:\n","        ORF_score_flat = []\n","        for i, contig in enumerate(ORF_gene_score):\n","            if not any(contig):\n","                ORF_score_flat.append([])\n","                continue\n","            temp = []\n","            for j, frame in enumerate(contig):\n","                for k, geneprob in enumerate(frame):\n","                    length = ORF_coord_list[i][j][k][1] - ORF_coord_list[i][j][k][0] + 1 \n","                    TIS_prob = ORF_TIS_prob[i][j][k]\n","                    start_codon = ORF_start_codon[i][j][k]\n","                    ATG = start_codon == 0\n","                    GTG = start_codon == 1\n","                    TTG = start_codon == 2\n","\n","                    combprob =   geneprob * weight_gene_prob \\\n","                            + TIS_prob * weight_TIS_prob \\\n","                            + ATG * weight_ATG \\\n","                            + GTG * weight_TTG \\\n","                            + TTG * weight_GTG\n","                    maxprob = weight_gene_prob + weight_TIS_prob + max(weight_ATG, weight_TTG, weight_GTG)\n","                    probthresh = score_threshold * maxprob\n","                    score = (combprob - probthresh) * length\n","\n","                    temp.append(score)\n","            ORF_score_flat.append(temp)\n","\n","    # DAGs to maximize geneiness on each contig\n","    contig_gene_coord = []\n","    contig_gene_strand = []\n","\n","    for i, coords in enumerate(ORF_coord_flat):\n","        if verbose:\n","            print(\"Creating weighted directed acyclic graph of all ORFs and subORFs on contig \" + str(i) + \"...\\n\")\n","\n","        # sort coords, lengths, strands, and scores\n","        startpos = np.array([x[0] for x in coords])\n","        sortidx = list(np.argsort(startpos))\n","\n","        coords_sorted = [coords[j] for j in sortidx]\n","\n","        lengths = ORF_length_flat[i]\n","        lengths_sorted = [lengths[j] for j in sortidx]\n","\n","        scores = ORF_score_flat[i]\n","        scores_sorted = [scores[j] for j in sortidx]\n","\n","        strands = ORF_strand_flat[i]\n","        strands_sorted = [strands[j] for j in sortidx]\n","\n","        # create DAG\n","        G = nx.DiGraph()\n","        \n","        # add null starting node\n","        G.add_node(0, coords=(-1, -1))\n","        n_connections = 0\n","        idx_offset = 1\n","        while n_connections < max_forward_connections:\n","            k = idx_offset\n","            idx_offset += 1\n","            if k > len(scores_sorted)-1: # dont try to add edge past last ORF\n","                n_connections += 1\n","                continue\n","            G.add_node(k, coords = coords_sorted[k-1])\n","            edge_weight = scores_sorted[k-1]\n","            G.add_edge(0, k, weight = edge_weight) # weight corresponds to score of gene at tip of arrow\n","            n_connections += 1\n","\n","        # add edges between compatible ORFs\n","        for j in tqdm(range(1, len(scores_sorted)-1), unit=\" ORF\"):\n","            n_connections = 0\n","            idx_offset = 1\n","\n","            G.add_node(j, coords=coords_sorted[j-1])\n","\n","            while n_connections < max_forward_connections:\n","                k = j + idx_offset\n","                idx_offset += 1\n","\n","                if k > len(scores_sorted)-1: # dont try to add edge past end of contigs\n","                    n_connections += 1\n","                    continue \n","\n","                coords0 = coords_sorted[j-1]\n","                coords1 = coords_sorted[k-1]\n","\n","                strand0 = strands_sorted[j-1]\n","                strand1 = strands_sorted[k-1]\n","\n","                compat, penalty = analyze_overlap(coords0, coords1, \n","                                                  strand0, strand1,\n","                                                  unidirectional_penalty_per_base,\n","                                                  convergent_penalty_per_base,\n","                                                  divergent_penalty_per_base)\n","\n","                if compat:\n","                    score = scores_sorted[k-1] - penalty\n","                    G.add_node(k, coords=coords_sorted[k-1])\n","                    G.add_edge(j, k, weight=score)\n","                    n_connections += 1\n","\n","        # solve for geneiest path through contig\n","        if verbose:\n","            print(\"Maximizing geneiness...\")\n","\n","        max_ORF_PATH_withnull = nx.algorithms.dag.dag_longest_path(G)\n","        max_ORF_PATH = [x-1 for x in max_ORF_PATH_withnull[1:]]\n","\n","        graph_score = [scores_sorted[j] for j in max_ORF_PATH]\n","        graph_score_total = np.sum(graph_score)\n","\n","        gene_predict_coords = [coords_sorted[j] for j in max_ORF_PATH]\n","        gene_predict_strand = [strands_sorted[j] for j in max_ORF_PATH]\n","\n","        # mmseqs filter\n","        if filter_gene_predictions:\n","            if verbose:\n","                print(\"\\nFiltering predictions with mmseqs2...\")\n","\n","            # get amino acid sequence from coherent ORFs\n","            # 3' TO 5'\n","            aa_sorted = [ORF_seq_enc[j] for j in sortidx]\n","            aa_tensor = [aa_sorted[j] for j in max_ORF_PATH]\n","            aa_seq = [tensor_to_seq([int(y) for y in x]) for x in aa_tensor]\n","\n","            # make temp dir to store mmseqs stuff\n","            finding_empty_dir = True\n","            dir_idx = 0\n","            while finding_empty_dir:\n","                dirpath = os.path.join(\"/content/mmseqs2/tmp\", str(dir_idx))\n","                if os.path.isdir(dirpath):\n","                    dir_idx += 1\n","                    continue\n","                else:\n","                    !mkdir {dirpath}\n","                    finding_empty_dir = False\n","            \n","            # mmseqs create query DB     3' to 5'\n","            query_fasta_path_35 = os.path.join(dirpath, \"candidate_genes_35.fasta\")\n","            with open(query_fasta_path_35, \"w\") as f:\n","                for i, s in enumerate(aa_seq):\n","                    f.writelines(\">\" + str(i) + \"\\n\")\n","                    f.writelines(str(s) + \"\\n\")\n","\n","            # mmseqs create query DB     5' to 3'\n","            query_fasta_path_53 = os.path.join(dirpath, \"candidate_genes_53.fasta\")\n","            with open(query_fasta_path_53, \"w\") as f:\n","                for i, s in enumerate(aa_seq):\n","                    f.writelines(\">\" + str(i) + \"\\n\")\n","                    f.writelines(str(s)[::-1] + \"\\n\")\n","\n","            query_DB_path_35 = os.path.join(dirpath, \"candidateDB_35\")\n","            !/content/mmseqs2/mmseqs/bin/mmseqs createdb {query_fasta_path_35} {query_DB_path_35}\n","            \n","            query_DB_path_53 = os.path.join(dirpath, \"candidateDB_53\")\n","            !/content/mmseqs2/mmseqs/bin/mmseqs createdb {query_fasta_path_53} {query_DB_path_53}\n","            \n","\n","            \n","            # mmseqs search\n","            results_DB_path_35 = os.path.join(dirpath, \"resultsDB_35\")\n","            !/content/mmseqs2/mmseqs/bin/mmseqs search -s 7.0 {query_DB_path_35} {genexa_DB_path} {results_DB_path_35} {dirpath}\n","\n","            # convert to readable format\n","            m8_path_genexa = os.path.join(dirpath, \"resultDB_genexa.m8\")\n","            !/content/mmseqs2/mmseqs/bin/mmseqs convertalis {query_DB_path_35} {genexa_DB_path} {results_DB_path_35} {m8_path_genexa} --format-output \"query,target,evalue,raw\"\n","\n","            # load search results\n","            mmseqs_results_genexa = pandas.read_table(m8_path_genexa, header=None, names=[\"query\", \"target\", \"evalue\", \"raw\"]).to_numpy()\n","\n","            # get hits\n","            hit_idx_genexa = np.unique(mmseqs_results_genexa[:, 0]).astype(int)\n","\n","            # mmseqs search\n","            results_DB_path_53 = os.path.join(dirpath, \"resultsDB_53\")\n","            !/content/mmseqs2/mmseqs/bin/mmseqs search -s 7.0 {query_DB_path_53} {swissprot_DB_path} {results_DB_path_53} {dirpath}\n","\n","            # convert to readable format\n","            m8_path_secondary = os.path.join(dirpath, \"resultDB_secondary.m8\")\n","            !/content/mmseqs2/mmseqs/bin/mmseqs convertalis {query_DB_path_53} {swissprot_DB_path} {results_DB_path_53} {m8_path_secondary} --format-output \"query,target,evalue,raw\"\n","\n","            # load search results\n","            mmseqs_results_secondary = pandas.read_table(m8_path_secondary, header=None, names=[\"query\", \"target\", \"evalue\", \"raw\"]).to_numpy()\n","\n","            # get hits\n","            hit_idx_secondary = np.unique(mmseqs_results_secondary[:, 0]).astype(int)\n","\n","            # filter gene predictions, keep if mmseqs hit or high enough gene score\n","            # cutoff = 240\n","            # cutoff = 190\n","            cutoff = 200\n","\n","            cutoffpath = [x for i, x in enumerate(max_ORF_PATH) if (scores_sorted[x] > cutoff or (i in hit_idx_genexa or i in hit_idx_secondary))]\n","            gene_predict_coords = [coords_sorted[j] for j in cutoffpath]\n","            gene_predict_strand = [strands_sorted[j] for j in cutoffpath]\n","\n","            graph_score_cutoff = [scores_sorted[j] for j in cutoffpath]\n","            contig_gene_coord.append(gene_predict_coords)\n","            contig_gene_strand.append(gene_predict_strand)\n","\n","            n_genes = len(gene_predict_coords)\n","            if verbose:\n","                print(\"found\", n_genes, \"genes\\n\\n\")\n","\n","\n","        else:\n","            # just give all predictions\n","            cutoff = 0\n","            cutoffpath = [x for x in max_ORF_PATH if scores_sorted[x] > cutoff]\n","            gene_predict_coords = [coords_sorted[j] for j in cutoffpath]\n","            gene_predict_strand = [strands_sorted[j] for j in cutoffpath]\n","\n","            graph_score_cutoff = [scores_sorted[j] for j in cutoffpath]\n","            contig_gene_coord.append(gene_predict_coords)\n","            contig_gene_strand.append(gene_predict_strand)\n","\n","            n_genes = len(gene_predict_coords)\n","            if verbose:\n","                print(\"found\", n_genes, \"genes\\n\\n\")\n","    contig_gene_coord_list.append(contig_gene_coord)\n","    contig_gene_strand_list.append(contig_gene_strand)\n","\n","print(\"Done\")"],"execution_count":17,"outputs":[{"output_type":"stream","text":["Reading fasta file GCF_900465355.1_Ran1-1_genomic.fna.gz...\n","\n","Finding and translating open reading frames...\n","\n","Encoding amino acids...\n","\n","Applying kmer prefilter...\n","\n","Scoring ORFs with temporal convolutional network...\n","\n"],"name":"stdout"},{"output_type":"display_data","data":{"application/vnd.jupyter.widget-view+json":{"model_id":"da4cdb519f6b4666beb6c961d72ec00f","version_minor":0,"version_major":2},"text/plain":["HBox(children=(FloatProgress(value=0.0, max=25.0), HTML(value='')))"]},"metadata":{"tags":[]}},{"output_type":"stream","text":["\n","Scoring translation initiation sites...\n","\n"],"name":"stdout"},{"output_type":"display_data","data":{"application/vnd.jupyter.widget-view+json":{"model_id":"b14bd3ff1365460fa6b77069a7d29d9d","version_minor":0,"version_major":2},"text/plain":["HBox(children=(FloatProgress(value=0.0, max=8.0), HTML(value='')))"]},"metadata":{"tags":[]}},{"output_type":"stream","text":["\n","Creating weighted directed acyclic graph of all ORFs and subORFs on contig 0...\n","\n"],"name":"stdout"},{"output_type":"display_data","data":{"application/vnd.jupyter.widget-view+json":{"model_id":"7097bf4a7d214150b67139f710b78906","version_minor":0,"version_major":2},"text/plain":["HBox(children=(FloatProgress(value=0.0, max=72662.0), HTML(value='')))"]},"metadata":{"tags":[]}},{"output_type":"stream","text":["\n","Maximizing geneiness...\n","\n","Filtering predictions with mmseqs2...\n","createdb /content/mmseqs2/tmp/6/candidate_genes_35.fasta /content/mmseqs2/tmp/6/candidateDB_35 \n","\n","MMseqs Version:       \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Database type         \t0\n","Shuffle input database\ttrue\n","Createdb mode         \t0\n","Write lookup file     \t1\n","Offset of numeric ids \t0\n","Compressed            \t0\n","Verbosity             \t3\n","\n","Converting sequences\n","[1718] 0s 4ms\n","Time for merging to candidateDB_35_h: 0h 0m 0s 6ms\n","Time for merging to candidateDB_35: 0h 0m 0s 3ms\n","Database type: Aminoacid\n","Time for processing: 0h 0m 0s 18ms\n","createdb /content/mmseqs2/tmp/6/candidate_genes_53.fasta /content/mmseqs2/tmp/6/candidateDB_53 \n","\n","MMseqs Version:       \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Database type         \t0\n","Shuffle input database\ttrue\n","Createdb mode         \t0\n","Write lookup file     \t1\n","Offset of numeric ids \t0\n","Compressed            \t0\n","Verbosity             \t3\n","\n","Converting sequences\n","[1718] 0s 4ms\n","Time for merging to candidateDB_53_h: 0h 0m 0s 3ms\n","Time for merging to candidateDB_53: 0h 0m 0s 3ms\n","Database type: Aminoacid\n","Time for processing: 0h 0m 0s 14ms\n","search -s 7.0 /content/mmseqs2/tmp/6/candidateDB_35 /content/mmseqs2/genexa/genexaDB /content/mmseqs2/tmp/6/resultsDB_35 /content/mmseqs2/tmp/6 \n","\n","MMseqs Version:                     \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Substitution matrix                 \tnucl:nucleotide.out,aa:blosum62.out\n","Add backtrace                       \tfalse\n","Alignment mode                      \t2\n","Allow wrapped scoring               \tfalse\n","E-value threshold                   \t0.001\n","Seq. id. threshold                  \t0\n","Min alignment length                \t0\n","Seq. id. mode                       \t0\n","Alternative alignments              \t0\n","Coverage threshold                  \t0\n","Coverage mode                       \t0\n","Max sequence length                 \t65535\n","Compositional bias                  \t1\n","Realign hits                        \tfalse\n","Max reject                          \t2147483647\n","Max accept                          \t2147483647\n","Include identical seq. id.          \tfalse\n","Preload mode                        \t0\n","Pseudo count a                      \t1\n","Pseudo count b                      \t1.5\n","Score bias                          \t0\n","Gap open cost                       \tnucl:5,aa:11\n","Gap extension cost                  \tnucl:2,aa:1\n","Zdrop                               \t40\n","Threads                             \t4\n","Compressed                          \t0\n","Verbosity                           \t3\n","Seed substitution matrix            \tnucl:nucleotide.out,aa:VTML80.out\n","Sensitivity                         \t7\n","k-mer length                        \t0\n","k-score                             \t2147483647\n","Alphabet size                       \tnucl:5,aa:21\n","Max results per query               \t300\n","Split database                      \t0\n","Split mode                          \t2\n","Split memory limit                  \t0\n","Diagonal scoring                    \ttrue\n","Exact k-mer matching                \t0\n","Mask residues                       \t1\n","Mask lower case residues            \t0\n","Minimum diagonal score              \t15\n","Spaced k-mers                       \t1\n","Spaced k-mer pattern                \t\n","Local temporary path                \t\n","Rescore mode                        \t0\n","Remove hits by seq. id. and coverage\tfalse\n","Sort results                        \t0\n","Mask profile                        \t1\n","Profile e-value threshold           \t0.1\n","Global sequence weighting           \tfalse\n","Allow deletions                     \tfalse\n","Filter MSA                          \t1\n","Maximum seq. id. threshold          \t0.9\n","Minimum seq. id.                    \t0\n","Minimum score per column            \t-20\n","Minimum coverage                    \t0\n","Select N most diverse seqs          \t1000\n","Omit consensus                      \tfalse\n","Min codons in orf                   \t30\n","Max codons in length                \t32734\n","Max orf gaps                        \t2147483647\n","Contig start mode                   \t2\n","Contig end mode                     \t2\n","Orf start mode                      \t1\n","Forward frames                      \t1,2,3\n","Reverse frames                      \t1,2,3\n","Translation table                   \t1\n","Translate orf                       \t0\n","Use all table starts                \tfalse\n","Offset of numeric ids               \t0\n","Create lookup                       \t0\n","Add orf stop                        \tfalse\n","Chain overlapping alignments        \t0\n","Merge query                         \t1\n","Search type                         \t0\n","Search iterations                   \t1\n","Start sensitivity                   \t4\n","Search steps                        \t1\n","Slice search mode                   \tfalse\n","Strand selection                    \t1\n","Disk space limit                    \t0\n","MPI runner                          \t\n","Force restart with latest tmp       \tfalse\n","Remove temporary files              \tfalse\n","\n","prefilter /content/mmseqs2/tmp/6/candidateDB_35 /content/mmseqs2/genexa/genexaDB.idx /content/mmseqs2/tmp/6/16459183689663617183/pref_0 --sub-mat nucl:nucleotide.out,aa:blosum62.out --seed-sub-mat nucl:nucleotide.out,aa:VTML80.out -k 0 --k-score 2147483647 --alph-size nucl:5,aa:21 --max-seq-len 65535 --max-seqs 300 --split 0 --split-mode 2 --split-memory-limit 0 -c 0 --cov-mode 0 --comp-bias-corr 1 --diag-score 1 --exact-kmer-matching 0 --mask 1 --mask-lower-case 0 --min-ungapped-score 15 --add-self-matches 0 --spaced-kmer-mode 1 --db-load-mode 0 --pca 1 --pcb 1.5 --threads 4 --compressed 0 -v 3 -s 7.0 \n","\n","Index version: 16\n","Generated by:  16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","ScoreMatrix:  VTML80.out\n","Query database size: 1729 type: Aminoacid\n","Estimated memory consumption: 2G\n","Target database size: 468351 type: Aminoacid\n","Process prefiltering step 1 of 1\n","\n","k-mer similarity threshold: 100\n","Starting prefiltering scores calculation (step 1 of 1)\n","Query db start 1 to 1729\n","Target db start 1 to 468351\n","[=================================================================] 100.00% 1.73K 15s 608ms\n","\n","861.061374 k-mers per position\n","747223 DB matches per sequence\n","108 overflows\n","0 queries produce too many hits (truncated result)\n","280 sequences passed prefiltering per query sequence\n","300 median result list length\n","1 sequences with 0 size result lists\n","Time for merging to pref_0: 0h 0m 0s 0ms\n","Time for processing: 0h 0m 17s 43ms\n","align /content/mmseqs2/tmp/6/candidateDB_35 /content/mmseqs2/genexa/genexaDB.idx /content/mmseqs2/tmp/6/16459183689663617183/pref_0 /content/mmseqs2/tmp/6/resultsDB_35 --sub-mat nucl:nucleotide.out,aa:blosum62.out -a 0 --alignment-mode 2 --wrapped-scoring 0 -e 0.001 --min-seq-id 0 --min-aln-len 0 --seq-id-mode 0 --alt-ali 0 -c 0 --cov-mode 0 --max-seq-len 65535 --comp-bias-corr 1 --realign 0 --max-rejected 2147483647 --max-accept 2147483647 --add-self-matches 0 --db-load-mode 0 --pca 1 --pcb 1.5 --score-bias 0 --gap-open nucl:5,aa:11 --gap-extend nucl:2,aa:1 --zdrop 40 --threads 4 --compressed 0 -v 3 \n","\n","Index version: 16\n","Generated by:  16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","ScoreMatrix:  VTML80.out\n","Compute score and coverage\n","Query database size: 1729 type: Aminoacid\n","Target database size: 468351 type: Aminoacid\n","Calculation of alignments\n","[=================================================================] 100.00% 1.73K 13s 645ms\n","Time for merging to resultsDB_35: 0h 0m 0s 0ms\n","\n","484839 alignments calculated.\n","156278 sequence pairs passed the thresholds (0.322330 of overall calculated).\n","90.386353 hits per query sequence.\n","Time for processing: 0h 0m 13s 691ms\n","convertalis /content/mmseqs2/tmp/6/candidateDB_35 /content/mmseqs2/genexa/genexaDB /content/mmseqs2/tmp/6/resultsDB_35 /content/mmseqs2/tmp/6/resultDB_genexa.m8 --format-output query,target,evalue,raw \n","\n","MMseqs Version:        \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Substitution matrix    \tnucl:nucleotide.out,aa:blosum62.out\n","Alignment format       \t0\n","Format alignment output\tquery,target,evalue,raw\n","Translation table      \t1\n","Gap open cost          \tnucl:5,aa:11\n","Gap extension cost     \tnucl:2,aa:1\n","Database output        \tfalse\n","Preload mode           \t0\n","Search type            \t0\n","Threads                \t4\n","Compressed             \t0\n","Verbosity              \t3\n","\n","[=================================================================] 100.00% 1.73K 0s 116ms\n","Time for merging to resultDB_genexa.m8: 0h 0m 0s 4ms\n","Time for processing: 0h 0m 0s 186ms\n","search -s 7.0 /content/mmseqs2/tmp/6/candidateDB_53 /content/mmseqs2/swissprot/swissprotDB /content/mmseqs2/tmp/6/resultsDB_53 /content/mmseqs2/tmp/6 \n","\n","MMseqs Version:                     \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Substitution matrix                 \tnucl:nucleotide.out,aa:blosum62.out\n","Add backtrace                       \tfalse\n","Alignment mode                      \t2\n","Allow wrapped scoring               \tfalse\n","E-value threshold                   \t0.001\n","Seq. id. threshold                  \t0\n","Min alignment length                \t0\n","Seq. id. mode                       \t0\n","Alternative alignments              \t0\n","Coverage threshold                  \t0\n","Coverage mode                       \t0\n","Max sequence length                 \t65535\n","Compositional bias                  \t1\n","Realign hits                        \tfalse\n","Max reject                          \t2147483647\n","Max accept                          \t2147483647\n","Include identical seq. id.          \tfalse\n","Preload mode                        \t0\n","Pseudo count a                      \t1\n","Pseudo count b                      \t1.5\n","Score bias                          \t0\n","Gap open cost                       \tnucl:5,aa:11\n","Gap extension cost                  \tnucl:2,aa:1\n","Zdrop                               \t40\n","Threads                             \t4\n","Compressed                          \t0\n","Verbosity                           \t3\n","Seed substitution matrix            \tnucl:nucleotide.out,aa:VTML80.out\n","Sensitivity                         \t7\n","k-mer length                        \t0\n","k-score                             \t2147483647\n","Alphabet size                       \tnucl:5,aa:21\n","Max results per query               \t300\n","Split database                      \t0\n","Split mode                          \t2\n","Split memory limit                  \t0\n","Diagonal scoring                    \ttrue\n","Exact k-mer matching                \t0\n","Mask residues                       \t1\n","Mask lower case residues            \t0\n","Minimum diagonal score              \t15\n","Spaced k-mers                       \t1\n","Spaced k-mer pattern                \t\n","Local temporary path                \t\n","Rescore mode                        \t0\n","Remove hits by seq. id. and coverage\tfalse\n","Sort results                        \t0\n","Mask profile                        \t1\n","Profile e-value threshold           \t0.1\n","Global sequence weighting           \tfalse\n","Allow deletions                     \tfalse\n","Filter MSA                          \t1\n","Maximum seq. id. threshold          \t0.9\n","Minimum seq. id.                    \t0\n","Minimum score per column            \t-20\n","Minimum coverage                    \t0\n","Select N most diverse seqs          \t1000\n","Omit consensus                      \tfalse\n","Min codons in orf                   \t30\n","Max codons in length                \t32734\n","Max orf gaps                        \t2147483647\n","Contig start mode                   \t2\n","Contig end mode                     \t2\n","Orf start mode                      \t1\n","Forward frames                      \t1,2,3\n","Reverse frames                      \t1,2,3\n","Translation table                   \t1\n","Translate orf                       \t0\n","Use all table starts                \tfalse\n","Offset of numeric ids               \t0\n","Create lookup                       \t0\n","Add orf stop                        \tfalse\n","Chain overlapping alignments        \t0\n","Merge query                         \t1\n","Search type                         \t0\n","Search iterations                   \t1\n","Start sensitivity                   \t4\n","Search steps                        \t1\n","Slice search mode                   \tfalse\n","Strand selection                    \t1\n","Disk space limit                    \t0\n","MPI runner                          \t\n","Force restart with latest tmp       \tfalse\n","Remove temporary files              \tfalse\n","\n","prefilter /content/mmseqs2/tmp/6/candidateDB_53 /content/mmseqs2/swissprot/swissprotDB /content/mmseqs2/tmp/6/16568021189535338629/pref_0 --sub-mat nucl:nucleotide.out,aa:blosum62.out --seed-sub-mat nucl:nucleotide.out,aa:VTML80.out -k 0 --k-score 2147483647 --alph-size nucl:5,aa:21 --max-seq-len 65535 --max-seqs 300 --split 0 --split-mode 2 --split-memory-limit 0 -c 0 --cov-mode 0 --comp-bias-corr 1 --diag-score 1 --exact-kmer-matching 0 --mask 1 --mask-lower-case 0 --min-ungapped-score 15 --add-self-matches 0 --spaced-kmer-mode 1 --db-load-mode 0 --pca 1 --pcb 1.5 --threads 4 --compressed 0 -v 3 -s 7.0 \n","\n","Query database size: 1729 type: Aminoacid\n","Estimated memory consumption: 2G\n","Target database size: 562755 type: Aminoacid\n","Index table k-mer threshold: 100 at k-mer size 6 \n","Index table: counting k-mers\n","[=================================================================] 100.00% 562.76K 24s 902ms\n","Index table: Masked residues: 3164874\n","Index table: fill\n","[=================================================================] 100.00% 562.76K 18s 2ms\n","Index statistics\n","Entries:          192757622\n","DB size:          1591 MB\n","Avg k-mer size:   3.011838\n","Top 10 k-mers\n","    GPGGTL\t1835\n","    GQSWTV\t1706\n","    WGMFAT\t1638\n","    PGVFEV\t1634\n","    VLWQFW\t1623\n","    AYIRPN\t1587\n","    RSPKGV\t1585\n","    TPHKWY\t1560\n","    HIRVGW\t1548\n","    ITLSPY\t1541\n","Time for index table init: 0h 0m 45s 70ms\n","Process prefiltering step 1 of 1\n","\n","k-mer similarity threshold: 100\n","Starting prefiltering scores calculation (step 1 of 1)\n","Query db start 1 to 1729\n","Target db start 1 to 562755\n","[=================================================================] 100.00% 1.73K 17s 252ms\n","\n","860.238443 k-mers per position\n","840031 DB matches per sequence\n","153 overflows\n","0 queries produce too many hits (truncated result)\n","284 sequences passed prefiltering per query sequence\n","300 median result list length\n","0 sequences with 0 size result lists\n","Time for merging to pref_0: 0h 0m 0s 0ms\n","Time for processing: 0h 1m 3s 470ms\n","align /content/mmseqs2/tmp/6/candidateDB_53 /content/mmseqs2/swissprot/swissprotDB /content/mmseqs2/tmp/6/16568021189535338629/pref_0 /content/mmseqs2/tmp/6/resultsDB_53 --sub-mat nucl:nucleotide.out,aa:blosum62.out -a 0 --alignment-mode 2 --wrapped-scoring 0 -e 0.001 --min-seq-id 0 --min-aln-len 0 --seq-id-mode 0 --alt-ali 0 -c 0 --cov-mode 0 --max-seq-len 65535 --comp-bias-corr 1 --realign 0 --max-rejected 2147483647 --max-accept 2147483647 --add-self-matches 0 --db-load-mode 0 --pca 1 --pcb 1.5 --score-bias 0 --gap-open nucl:5,aa:11 --gap-extend nucl:2,aa:1 --zdrop 40 --threads 4 --compressed 0 -v 3 \n","\n","Compute score and coverage\n","Query database size: 1729 type: Aminoacid\n","Target database size: 562755 type: Aminoacid\n","Calculation of alignments\n","[=================================================================] 100.00% 1.73K 13s 179ms\n","Time for merging to resultsDB_53: 0h 0m 0s 0ms\n","\n","491859 alignments calculated.\n","125129 sequence pairs passed the thresholds (0.254400 of overall calculated).\n","72.370735 hits per query sequence.\n","Time for processing: 0h 0m 13s 251ms\n","convertalis /content/mmseqs2/tmp/6/candidateDB_53 /content/mmseqs2/swissprot/swissprotDB /content/mmseqs2/tmp/6/resultsDB_53 /content/mmseqs2/tmp/6/resultDB_secondary.m8 --format-output query,target,evalue,raw \n","\n","MMseqs Version:        \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Substitution matrix    \tnucl:nucleotide.out,aa:blosum62.out\n","Alignment format       \t0\n","Format alignment output\tquery,target,evalue,raw\n","Translation table      \t1\n","Gap open cost          \tnucl:5,aa:11\n","Gap extension cost     \tnucl:2,aa:1\n","Database output        \tfalse\n","Preload mode           \t0\n","Search type            \t0\n","Threads                \t4\n","Compressed             \t0\n","Verbosity              \t3\n","\n","[=================================================================] 100.00% 1.73K 0s 86ms\n","Time for merging to resultDB_secondary.m8: 0h 0m 0s 4ms\n","Time for processing: 0h 0m 0s 176ms\n","found 1241 genes\n","\n","\n","Reading fasta file GCF_003047205.1_ASM304720v1_genomic.fna.gz...\n","\n","Finding and translating open reading frames...\n","\n","Encoding amino acids...\n","\n","Applying kmer prefilter...\n","\n","Scoring ORFs with temporal convolutional network...\n","\n"],"name":"stdout"},{"output_type":"display_data","data":{"application/vnd.jupyter.widget-view+json":{"model_id":"6952fed2527b44f5b8c3c4baf6e42323","version_minor":0,"version_major":2},"text/plain":["HBox(children=(FloatProgress(value=0.0, max=42.0), HTML(value='')))"]},"metadata":{"tags":[]}},{"output_type":"stream","text":["\n","Scoring translation initiation sites...\n","\n"],"name":"stdout"},{"output_type":"display_data","data":{"application/vnd.jupyter.widget-view+json":{"model_id":"8e1e5a8fb3d340be9050c46787dfa567","version_minor":0,"version_major":2},"text/plain":["HBox(children=(FloatProgress(value=0.0, max=13.0), HTML(value='')))"]},"metadata":{"tags":[]}},{"output_type":"stream","text":["\n","Creating weighted directed acyclic graph of all ORFs and subORFs on contig 0...\n","\n"],"name":"stdout"},{"output_type":"display_data","data":{"application/vnd.jupyter.widget-view+json":{"model_id":"e4f66d6bc4eb4de9b13c1ca3cba2dfd1","version_minor":0,"version_major":2},"text/plain":["HBox(children=(FloatProgress(value=0.0, max=123251.0), HTML(value='')))"]},"metadata":{"tags":[]}},{"output_type":"stream","text":["\n","Maximizing geneiness...\n","\n","Filtering predictions with mmseqs2...\n","createdb /content/mmseqs2/tmp/7/candidate_genes_35.fasta /content/mmseqs2/tmp/7/candidateDB_35 \n","\n","MMseqs Version:       \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Database type         \t0\n","Shuffle input database\ttrue\n","Createdb mode         \t0\n","Write lookup file     \t1\n","Offset of numeric ids \t0\n","Compressed            \t0\n","Verbosity             \t3\n","\n","Converting sequences\n","[3940] 0s 7ms\n","Time for merging to candidateDB_35_h: 0h 0m 0s 6ms\n","Time for merging to candidateDB_35: 0h 0m 0s 4ms\n","Database type: Aminoacid\n","Time for processing: 0h 0m 0s 24ms\n","createdb /content/mmseqs2/tmp/7/candidate_genes_53.fasta /content/mmseqs2/tmp/7/candidateDB_53 \n","\n","MMseqs Version:       \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Database type         \t0\n","Shuffle input database\ttrue\n","Createdb mode         \t0\n","Write lookup file     \t1\n","Offset of numeric ids \t0\n","Compressed            \t0\n","Verbosity             \t3\n","\n","Converting sequences\n","[3940] 0s 6ms\n","Time for merging to candidateDB_53_h: 0h 0m 0s 6ms\n","Time for merging to candidateDB_53: 0h 0m 0s 4ms\n","Database type: Aminoacid\n","Time for processing: 0h 0m 0s 22ms\n","search -s 7.0 /content/mmseqs2/tmp/7/candidateDB_35 /content/mmseqs2/genexa/genexaDB /content/mmseqs2/tmp/7/resultsDB_35 /content/mmseqs2/tmp/7 \n","\n","MMseqs Version:                     \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Substitution matrix                 \tnucl:nucleotide.out,aa:blosum62.out\n","Add backtrace                       \tfalse\n","Alignment mode                      \t2\n","Allow wrapped scoring               \tfalse\n","E-value threshold                   \t0.001\n","Seq. id. threshold                  \t0\n","Min alignment length                \t0\n","Seq. id. mode                       \t0\n","Alternative alignments              \t0\n","Coverage threshold                  \t0\n","Coverage mode                       \t0\n","Max sequence length                 \t65535\n","Compositional bias                  \t1\n","Realign hits                        \tfalse\n","Max reject                          \t2147483647\n","Max accept                          \t2147483647\n","Include identical seq. id.          \tfalse\n","Preload mode                        \t0\n","Pseudo count a                      \t1\n","Pseudo count b                      \t1.5\n","Score bias                          \t0\n","Gap open cost                       \tnucl:5,aa:11\n","Gap extension cost                  \tnucl:2,aa:1\n","Zdrop                               \t40\n","Threads                             \t4\n","Compressed                          \t0\n","Verbosity                           \t3\n","Seed substitution matrix            \tnucl:nucleotide.out,aa:VTML80.out\n","Sensitivity                         \t7\n","k-mer length                        \t0\n","k-score                             \t2147483647\n","Alphabet size                       \tnucl:5,aa:21\n","Max results per query               \t300\n","Split database                      \t0\n","Split mode                          \t2\n","Split memory limit                  \t0\n","Diagonal scoring                    \ttrue\n","Exact k-mer matching                \t0\n","Mask residues                       \t1\n","Mask lower case residues            \t0\n","Minimum diagonal score              \t15\n","Spaced k-mers                       \t1\n","Spaced k-mer pattern                \t\n","Local temporary path                \t\n","Rescore mode                        \t0\n","Remove hits by seq. id. and coverage\tfalse\n","Sort results                        \t0\n","Mask profile                        \t1\n","Profile e-value threshold           \t0.1\n","Global sequence weighting           \tfalse\n","Allow deletions                     \tfalse\n","Filter MSA                          \t1\n","Maximum seq. id. threshold          \t0.9\n","Minimum seq. id.                    \t0\n","Minimum score per column            \t-20\n","Minimum coverage                    \t0\n","Select N most diverse seqs          \t1000\n","Omit consensus                      \tfalse\n","Min codons in orf                   \t30\n","Max codons in length                \t32734\n","Max orf gaps                        \t2147483647\n","Contig start mode                   \t2\n","Contig end mode                     \t2\n","Orf start mode                      \t1\n","Forward frames                      \t1,2,3\n","Reverse frames                      \t1,2,3\n","Translation table                   \t1\n","Translate orf                       \t0\n","Use all table starts                \tfalse\n","Offset of numeric ids               \t0\n","Create lookup                       \t0\n","Add orf stop                        \tfalse\n","Chain overlapping alignments        \t0\n","Merge query                         \t1\n","Search type                         \t0\n","Search iterations                   \t1\n","Start sensitivity                   \t4\n","Search steps                        \t1\n","Slice search mode                   \tfalse\n","Strand selection                    \t1\n","Disk space limit                    \t0\n","MPI runner                          \t\n","Force restart with latest tmp       \tfalse\n","Remove temporary files              \tfalse\n","\n","prefilter /content/mmseqs2/tmp/7/candidateDB_35 /content/mmseqs2/genexa/genexaDB.idx /content/mmseqs2/tmp/7/10117974647565127616/pref_0 --sub-mat nucl:nucleotide.out,aa:blosum62.out --seed-sub-mat nucl:nucleotide.out,aa:VTML80.out -k 0 --k-score 2147483647 --alph-size nucl:5,aa:21 --max-seq-len 65535 --max-seqs 300 --split 0 --split-mode 2 --split-memory-limit 0 -c 0 --cov-mode 0 --comp-bias-corr 1 --diag-score 1 --exact-kmer-matching 0 --mask 1 --mask-lower-case 0 --min-ungapped-score 15 --add-self-matches 0 --spaced-kmer-mode 1 --db-load-mode 0 --pca 1 --pcb 1.5 --threads 4 --compressed 0 -v 3 -s 7.0 \n","\n","Index version: 16\n","Generated by:  16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","ScoreMatrix:  VTML80.out\n","Query database size: 3967 type: Aminoacid\n","Estimated memory consumption: 2G\n","Target database size: 468351 type: Aminoacid\n","Process prefiltering step 1 of 1\n","\n","k-mer similarity threshold: 100\n","Starting prefiltering scores calculation (step 1 of 1)\n","Query db start 1 to 3967\n","Target db start 1 to 468351\n","[=================================================================] 100.00% 3.97K 30s 704ms\n","\n","977.847120 k-mers per position\n","606597 DB matches per sequence\n","138 overflows\n","0 queries produce too many hits (truncated result)\n","280 sequences passed prefiltering per query sequence\n","300 median result list length\n","3 sequences with 0 size result lists\n","Time for merging to pref_0: 0h 0m 0s 1ms\n","Time for processing: 0h 0m 32s 213ms\n","align /content/mmseqs2/tmp/7/candidateDB_35 /content/mmseqs2/genexa/genexaDB.idx /content/mmseqs2/tmp/7/10117974647565127616/pref_0 /content/mmseqs2/tmp/7/resultsDB_35 --sub-mat nucl:nucleotide.out,aa:blosum62.out -a 0 --alignment-mode 2 --wrapped-scoring 0 -e 0.001 --min-seq-id 0 --min-aln-len 0 --seq-id-mode 0 --alt-ali 0 -c 0 --cov-mode 0 --max-seq-len 65535 --comp-bias-corr 1 --realign 0 --max-rejected 2147483647 --max-accept 2147483647 --add-self-matches 0 --db-load-mode 0 --pca 1 --pcb 1.5 --score-bias 0 --gap-open nucl:5,aa:11 --gap-extend nucl:2,aa:1 --zdrop 40 --threads 4 --compressed 0 -v 3 \n","\n","Index version: 16\n","Generated by:  16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","ScoreMatrix:  VTML80.out\n","Compute score and coverage\n","Query database size: 3967 type: Aminoacid\n","Target database size: 468351 type: Aminoacid\n","Calculation of alignments\n","[=================================================================] 100.00% 3.97K 27s 969ms\n","Time for merging to resultsDB_35: 0h 0m 0s 1ms\n","\n","1111011 alignments calculated.\n","313426 sequence pairs passed the thresholds (0.282109 of overall calculated).\n","79.008316 hits per query sequence.\n","Time for processing: 0h 0m 28s 62ms\n","convertalis /content/mmseqs2/tmp/7/candidateDB_35 /content/mmseqs2/genexa/genexaDB /content/mmseqs2/tmp/7/resultsDB_35 /content/mmseqs2/tmp/7/resultDB_genexa.m8 --format-output query,target,evalue,raw \n","\n","MMseqs Version:        \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Substitution matrix    \tnucl:nucleotide.out,aa:blosum62.out\n","Alignment format       \t0\n","Format alignment output\tquery,target,evalue,raw\n","Translation table      \t1\n","Gap open cost          \tnucl:5,aa:11\n","Gap extension cost     \tnucl:2,aa:1\n","Database output        \tfalse\n","Preload mode           \t0\n","Search type            \t0\n","Threads                \t4\n","Compressed             \t0\n","Verbosity              \t3\n","\n","[=================================================================] 100.00% 3.97K 0s 231ms\n","Time for merging to resultDB_genexa.m8: 0h 0m 0s 8ms\n","Time for processing: 0h 0m 0s 311ms\n","search -s 7.0 /content/mmseqs2/tmp/7/candidateDB_53 /content/mmseqs2/swissprot/swissprotDB /content/mmseqs2/tmp/7/resultsDB_53 /content/mmseqs2/tmp/7 \n","\n","MMseqs Version:                     \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Substitution matrix                 \tnucl:nucleotide.out,aa:blosum62.out\n","Add backtrace                       \tfalse\n","Alignment mode                      \t2\n","Allow wrapped scoring               \tfalse\n","E-value threshold                   \t0.001\n","Seq. id. threshold                  \t0\n","Min alignment length                \t0\n","Seq. id. mode                       \t0\n","Alternative alignments              \t0\n","Coverage threshold                  \t0\n","Coverage mode                       \t0\n","Max sequence length                 \t65535\n","Compositional bias                  \t1\n","Realign hits                        \tfalse\n","Max reject                          \t2147483647\n","Max accept                          \t2147483647\n","Include identical seq. id.          \tfalse\n","Preload mode                        \t0\n","Pseudo count a                      \t1\n","Pseudo count b                      \t1.5\n","Score bias                          \t0\n","Gap open cost                       \tnucl:5,aa:11\n","Gap extension cost                  \tnucl:2,aa:1\n","Zdrop                               \t40\n","Threads                             \t4\n","Compressed                          \t0\n","Verbosity                           \t3\n","Seed substitution matrix            \tnucl:nucleotide.out,aa:VTML80.out\n","Sensitivity                         \t7\n","k-mer length                        \t0\n","k-score                             \t2147483647\n","Alphabet size                       \tnucl:5,aa:21\n","Max results per query               \t300\n","Split database                      \t0\n","Split mode                          \t2\n","Split memory limit                  \t0\n","Diagonal scoring                    \ttrue\n","Exact k-mer matching                \t0\n","Mask residues                       \t1\n","Mask lower case residues            \t0\n","Minimum diagonal score              \t15\n","Spaced k-mers                       \t1\n","Spaced k-mer pattern                \t\n","Local temporary path                \t\n","Rescore mode                        \t0\n","Remove hits by seq. id. and coverage\tfalse\n","Sort results                        \t0\n","Mask profile                        \t1\n","Profile e-value threshold           \t0.1\n","Global sequence weighting           \tfalse\n","Allow deletions                     \tfalse\n","Filter MSA                          \t1\n","Maximum seq. id. threshold          \t0.9\n","Minimum seq. id.                    \t0\n","Minimum score per column            \t-20\n","Minimum coverage                    \t0\n","Select N most diverse seqs          \t1000\n","Omit consensus                      \tfalse\n","Min codons in orf                   \t30\n","Max codons in length                \t32734\n","Max orf gaps                        \t2147483647\n","Contig start mode                   \t2\n","Contig end mode                     \t2\n","Orf start mode                      \t1\n","Forward frames                      \t1,2,3\n","Reverse frames                      \t1,2,3\n","Translation table                   \t1\n","Translate orf                       \t0\n","Use all table starts                \tfalse\n","Offset of numeric ids               \t0\n","Create lookup                       \t0\n","Add orf stop                        \tfalse\n","Chain overlapping alignments        \t0\n","Merge query                         \t1\n","Search type                         \t0\n","Search iterations                   \t1\n","Start sensitivity                   \t4\n","Search steps                        \t1\n","Slice search mode                   \tfalse\n","Strand selection                    \t1\n","Disk space limit                    \t0\n","MPI runner                          \t\n","Force restart with latest tmp       \tfalse\n","Remove temporary files              \tfalse\n","\n","prefilter /content/mmseqs2/tmp/7/candidateDB_53 /content/mmseqs2/swissprot/swissprotDB /content/mmseqs2/tmp/7/4627345322845762662/pref_0 --sub-mat nucl:nucleotide.out,aa:blosum62.out --seed-sub-mat nucl:nucleotide.out,aa:VTML80.out -k 0 --k-score 2147483647 --alph-size nucl:5,aa:21 --max-seq-len 65535 --max-seqs 300 --split 0 --split-mode 2 --split-memory-limit 0 -c 0 --cov-mode 0 --comp-bias-corr 1 --diag-score 1 --exact-kmer-matching 0 --mask 1 --mask-lower-case 0 --min-ungapped-score 15 --add-self-matches 0 --spaced-kmer-mode 1 --db-load-mode 0 --pca 1 --pcb 1.5 --threads 4 --compressed 0 -v 3 -s 7.0 \n","\n","Query database size: 3967 type: Aminoacid\n","Estimated memory consumption: 2G\n","Target database size: 562755 type: Aminoacid\n","Index table k-mer threshold: 100 at k-mer size 6 \n","Index table: counting k-mers\n","[=================================================================] 100.00% 562.76K 24s 688ms\n","Index table: Masked residues: 3164874\n","Index table: fill\n","[=================================================================] 100.00% 562.76K 17s 678ms\n","Index statistics\n","Entries:          192757622\n","DB size:          1591 MB\n","Avg k-mer size:   3.011838\n","Top 10 k-mers\n","    GPGGTL\t1835\n","    GQSWTV\t1706\n","    WGMFAT\t1638\n","    PGVFEV\t1634\n","    VLWQFW\t1623\n","    AYIRPN\t1587\n","    RSPKGV\t1585\n","    TPHKWY\t1560\n","    HIRVGW\t1548\n","    ITLSPY\t1541\n","Time for index table init: 0h 0m 44s 418ms\n","Process prefiltering step 1 of 1\n","\n","k-mer similarity threshold: 100\n","Starting prefiltering scores calculation (step 1 of 1)\n","Query db start 1 to 3967\n","Target db start 1 to 562755\n","[=================================================================] 100.00% 3.97K 34s 595ms\n","\n","978.111202 k-mers per position\n","727633 DB matches per sequence\n","236 overflows\n","0 queries produce too many hits (truncated result)\n","287 sequences passed prefiltering per query sequence\n","300 median result list length\n","0 sequences with 0 size result lists\n","Time for merging to pref_0: 0h 0m 0s 1ms\n","Time for processing: 0h 1m 20s 213ms\n","align /content/mmseqs2/tmp/7/candidateDB_53 /content/mmseqs2/swissprot/swissprotDB /content/mmseqs2/tmp/7/4627345322845762662/pref_0 /content/mmseqs2/tmp/7/resultsDB_53 --sub-mat nucl:nucleotide.out,aa:blosum62.out -a 0 --alignment-mode 2 --wrapped-scoring 0 -e 0.001 --min-seq-id 0 --min-aln-len 0 --seq-id-mode 0 --alt-ali 0 -c 0 --cov-mode 0 --max-seq-len 65535 --comp-bias-corr 1 --realign 0 --max-rejected 2147483647 --max-accept 2147483647 --add-self-matches 0 --db-load-mode 0 --pca 1 --pcb 1.5 --score-bias 0 --gap-open nucl:5,aa:11 --gap-extend nucl:2,aa:1 --zdrop 40 --threads 4 --compressed 0 -v 3 \n","\n","Compute score and coverage\n","Query database size: 3967 type: Aminoacid\n","Target database size: 562755 type: Aminoacid\n","Calculation of alignments\n","[=================================================================] 100.00% 3.97K 26s 422ms\n","Time for merging to resultsDB_53: 0h 0m 0s 1ms\n","\n","1139921 alignments calculated.\n","237843 sequence pairs passed the thresholds (0.208649 of overall calculated).\n","59.955383 hits per query sequence.\n","Time for processing: 0h 0m 26s 490ms\n","convertalis /content/mmseqs2/tmp/7/candidateDB_53 /content/mmseqs2/swissprot/swissprotDB /content/mmseqs2/tmp/7/resultsDB_53 /content/mmseqs2/tmp/7/resultDB_secondary.m8 --format-output query,target,evalue,raw \n","\n","MMseqs Version:        \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Substitution matrix    \tnucl:nucleotide.out,aa:blosum62.out\n","Alignment format       \t0\n","Format alignment output\tquery,target,evalue,raw\n","Translation table      \t1\n","Gap open cost          \tnucl:5,aa:11\n","Gap extension cost     \tnucl:2,aa:1\n","Database output        \tfalse\n","Preload mode           \t0\n","Search type            \t0\n","Threads                \t4\n","Compressed             \t0\n","Verbosity              \t3\n","\n","[=================================================================] 100.00% 3.97K 0s 163ms\n","Time for merging to resultDB_secondary.m8: 0h 0m 0s 8ms\n","Time for processing: 0h 0m 0s 260ms\n","found 2635 genes\n","\n","\n","Reading fasta file GCF_002142475.1_ASM214247v1_genomic.fna.gz...\n","\n","Finding and translating open reading frames...\n","\n","Encoding amino acids...\n","\n","Applying kmer prefilter...\n","\n","Scoring ORFs with temporal convolutional network...\n","\n"],"name":"stdout"},{"output_type":"display_data","data":{"application/vnd.jupyter.widget-view+json":{"model_id":"514c76651fe8496093dce5ec30a93235","version_minor":0,"version_major":2},"text/plain":["HBox(children=(FloatProgress(value=0.0, max=112.0), HTML(value='')))"]},"metadata":{"tags":[]}},{"output_type":"stream","text":["\n","Scoring translation initiation sites...\n","\n"],"name":"stdout"},{"output_type":"display_data","data":{"application/vnd.jupyter.widget-view+json":{"model_id":"8b2846b7778741789dca148ee24644c8","version_minor":0,"version_major":2},"text/plain":["HBox(children=(FloatProgress(value=0.0, max=34.0), HTML(value='')))"]},"metadata":{"tags":[]}},{"output_type":"stream","text":["\n","Creating weighted directed acyclic graph of all ORFs and subORFs on contig 0...\n","\n"],"name":"stdout"},{"output_type":"display_data","data":{"application/vnd.jupyter.widget-view+json":{"model_id":"178edc8405c24d979b6b7ee906db4adc","version_minor":0,"version_major":2},"text/plain":["HBox(children=(FloatProgress(value=0.0, max=302978.0), HTML(value='')))"]},"metadata":{"tags":[]}},{"output_type":"stream","text":["\n","Maximizing geneiness...\n","\n","Filtering predictions with mmseqs2...\n","createdb /content/mmseqs2/tmp/8/candidate_genes_35.fasta /content/mmseqs2/tmp/8/candidateDB_35 \n","\n","MMseqs Version:       \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Database type         \t0\n","Shuffle input database\ttrue\n","Createdb mode         \t0\n","Write lookup file     \t1\n","Offset of numeric ids \t0\n","Compressed            \t0\n","Verbosity             \t3\n","\n","Converting sequences\n","[10808] 0s 14ms\n","Time for merging to candidateDB_35_h: 0h 0m 0s 6ms\n","Time for merging to candidateDB_35: 0h 0m 0s 7ms\n","Database type: Aminoacid\n","Time for processing: 0h 0m 0s 41ms\n","createdb /content/mmseqs2/tmp/8/candidate_genes_53.fasta /content/mmseqs2/tmp/8/candidateDB_53 \n","\n","MMseqs Version:       \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Database type         \t0\n","Shuffle input database\ttrue\n","Createdb mode         \t0\n","Write lookup file     \t1\n","Offset of numeric ids \t0\n","Compressed            \t0\n","Verbosity             \t3\n","\n","Converting sequences\n","[10808] 0s 11ms\n","Time for merging to candidateDB_53_h: 0h 0m 0s 5ms\n","Time for merging to candidateDB_53: 0h 0m 0s 7ms\n","Database type: Aminoacid\n","Time for processing: 0h 0m 0s 36ms\n","search -s 7.0 /content/mmseqs2/tmp/8/candidateDB_35 /content/mmseqs2/genexa/genexaDB /content/mmseqs2/tmp/8/resultsDB_35 /content/mmseqs2/tmp/8 \n","\n","MMseqs Version:                     \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Substitution matrix                 \tnucl:nucleotide.out,aa:blosum62.out\n","Add backtrace                       \tfalse\n","Alignment mode                      \t2\n","Allow wrapped scoring               \tfalse\n","E-value threshold                   \t0.001\n","Seq. id. threshold                  \t0\n","Min alignment length                \t0\n","Seq. id. mode                       \t0\n","Alternative alignments              \t0\n","Coverage threshold                  \t0\n","Coverage mode                       \t0\n","Max sequence length                 \t65535\n","Compositional bias                  \t1\n","Realign hits                        \tfalse\n","Max reject                          \t2147483647\n","Max accept                          \t2147483647\n","Include identical seq. id.          \tfalse\n","Preload mode                        \t0\n","Pseudo count a                      \t1\n","Pseudo count b                      \t1.5\n","Score bias                          \t0\n","Gap open cost                       \tnucl:5,aa:11\n","Gap extension cost                  \tnucl:2,aa:1\n","Zdrop                               \t40\n","Threads                             \t4\n","Compressed                          \t0\n","Verbosity                           \t3\n","Seed substitution matrix            \tnucl:nucleotide.out,aa:VTML80.out\n","Sensitivity                         \t7\n","k-mer length                        \t0\n","k-score                             \t2147483647\n","Alphabet size                       \tnucl:5,aa:21\n","Max results per query               \t300\n","Split database                      \t0\n","Split mode                          \t2\n","Split memory limit                  \t0\n","Diagonal scoring                    \ttrue\n","Exact k-mer matching                \t0\n","Mask residues                       \t1\n","Mask lower case residues            \t0\n","Minimum diagonal score              \t15\n","Spaced k-mers                       \t1\n","Spaced k-mer pattern                \t\n","Local temporary path                \t\n","Rescore mode                        \t0\n","Remove hits by seq. id. and coverage\tfalse\n","Sort results                        \t0\n","Mask profile                        \t1\n","Profile e-value threshold           \t0.1\n","Global sequence weighting           \tfalse\n","Allow deletions                     \tfalse\n","Filter MSA                          \t1\n","Maximum seq. id. threshold          \t0.9\n","Minimum seq. id.                    \t0\n","Minimum score per column            \t-20\n","Minimum coverage                    \t0\n","Select N most diverse seqs          \t1000\n","Omit consensus                      \tfalse\n","Min codons in orf                   \t30\n","Max codons in length                \t32734\n","Max orf gaps                        \t2147483647\n","Contig start mode                   \t2\n","Contig end mode                     \t2\n","Orf start mode                      \t1\n","Forward frames                      \t1,2,3\n","Reverse frames                      \t1,2,3\n","Translation table                   \t1\n","Translate orf                       \t0\n","Use all table starts                \tfalse\n","Offset of numeric ids               \t0\n","Create lookup                       \t0\n","Add orf stop                        \tfalse\n","Chain overlapping alignments        \t0\n","Merge query                         \t1\n","Search type                         \t0\n","Search iterations                   \t1\n","Start sensitivity                   \t4\n","Search steps                        \t1\n","Slice search mode                   \tfalse\n","Strand selection                    \t1\n","Disk space limit                    \t0\n","MPI runner                          \t\n","Force restart with latest tmp       \tfalse\n","Remove temporary files              \tfalse\n","\n","prefilter /content/mmseqs2/tmp/8/candidateDB_35 /content/mmseqs2/genexa/genexaDB.idx /content/mmseqs2/tmp/8/3776765605466638049/pref_0 --sub-mat nucl:nucleotide.out,aa:blosum62.out --seed-sub-mat nucl:nucleotide.out,aa:VTML80.out -k 0 --k-score 2147483647 --alph-size nucl:5,aa:21 --max-seq-len 65535 --max-seqs 300 --split 0 --split-mode 2 --split-memory-limit 0 -c 0 --cov-mode 0 --comp-bias-corr 1 --diag-score 1 --exact-kmer-matching 0 --mask 1 --mask-lower-case 0 --min-ungapped-score 15 --add-self-matches 0 --spaced-kmer-mode 1 --db-load-mode 0 --pca 1 --pcb 1.5 --threads 4 --compressed 0 -v 3 -s 7.0 \n","\n","Index version: 16\n","Generated by:  16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","ScoreMatrix:  VTML80.out\n","Query database size: 10811 type: Aminoacid\n","Estimated memory consumption: 2G\n","Target database size: 468351 type: Aminoacid\n","Process prefiltering step 1 of 1\n","\n","k-mer similarity threshold: 100\n","Starting prefiltering scores calculation (step 1 of 1)\n","Query db start 1 to 10811\n","Target db start 1 to 468351\n","[=================================================================] 100.00% 10.81K 1m 5s 707ms\n","\n","918.938066 k-mers per position\n","468769 DB matches per sequence\n","331 overflows\n","0 queries produce too many hits (truncated result)\n","267 sequences passed prefiltering per query sequence\n","300 median result list length\n","1 sequences with 0 size result lists\n","Time for merging to pref_0: 0h 0m 0s 3ms\n","Time for processing: 0h 1m 7s 158ms\n","align /content/mmseqs2/tmp/8/candidateDB_35 /content/mmseqs2/genexa/genexaDB.idx /content/mmseqs2/tmp/8/3776765605466638049/pref_0 /content/mmseqs2/tmp/8/resultsDB_35 --sub-mat nucl:nucleotide.out,aa:blosum62.out -a 0 --alignment-mode 2 --wrapped-scoring 0 -e 0.001 --min-seq-id 0 --min-aln-len 0 --seq-id-mode 0 --alt-ali 0 -c 0 --cov-mode 0 --max-seq-len 65535 --comp-bias-corr 1 --realign 0 --max-rejected 2147483647 --max-accept 2147483647 --add-self-matches 0 --db-load-mode 0 --pca 1 --pcb 1.5 --score-bias 0 --gap-open nucl:5,aa:11 --gap-extend nucl:2,aa:1 --zdrop 40 --threads 4 --compressed 0 -v 3 \n","\n","Index version: 16\n","Generated by:  16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","ScoreMatrix:  VTML80.out\n","Compute score and coverage\n","Query database size: 10811 type: Aminoacid\n","Target database size: 468351 type: Aminoacid\n","Calculation of alignments\n","[=================================================================] 100.00% 10.81K 51s 153ms\n","Time for merging to resultsDB_35: 0h 0m 0s 3ms\n","\n","2896447 alignments calculated.\n","509317 sequence pairs passed the thresholds (0.175842 of overall calculated).\n","47.110996 hits per query sequence.\n","Time for processing: 0h 0m 51s 200ms\n","convertalis /content/mmseqs2/tmp/8/candidateDB_35 /content/mmseqs2/genexa/genexaDB /content/mmseqs2/tmp/8/resultsDB_35 /content/mmseqs2/tmp/8/resultDB_genexa.m8 --format-output query,target,evalue,raw \n","\n","MMseqs Version:        \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Substitution matrix    \tnucl:nucleotide.out,aa:blosum62.out\n","Alignment format       \t0\n","Format alignment output\tquery,target,evalue,raw\n","Translation table      \t1\n","Gap open cost          \tnucl:5,aa:11\n","Gap extension cost     \tnucl:2,aa:1\n","Database output        \tfalse\n","Preload mode           \t0\n","Search type            \t0\n","Threads                \t4\n","Compressed             \t0\n","Verbosity              \t3\n","\n","[=================================================================] 100.00% 10.81K 0s 390ms\n","Time for merging to resultDB_genexa.m8: 0h 0m 0s 17ms\n","Time for processing: 0h 0m 0s 489ms\n","search -s 7.0 /content/mmseqs2/tmp/8/candidateDB_53 /content/mmseqs2/swissprot/swissprotDB /content/mmseqs2/tmp/8/resultsDB_53 /content/mmseqs2/tmp/8 \n","\n","MMseqs Version:                     \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Substitution matrix                 \tnucl:nucleotide.out,aa:blosum62.out\n","Add backtrace                       \tfalse\n","Alignment mode                      \t2\n","Allow wrapped scoring               \tfalse\n","E-value threshold                   \t0.001\n","Seq. id. threshold                  \t0\n","Min alignment length                \t0\n","Seq. id. mode                       \t0\n","Alternative alignments              \t0\n","Coverage threshold                  \t0\n","Coverage mode                       \t0\n","Max sequence length                 \t65535\n","Compositional bias                  \t1\n","Realign hits                        \tfalse\n","Max reject                          \t2147483647\n","Max accept                          \t2147483647\n","Include identical seq. id.          \tfalse\n","Preload mode                        \t0\n","Pseudo count a                      \t1\n","Pseudo count b                      \t1.5\n","Score bias                          \t0\n","Gap open cost                       \tnucl:5,aa:11\n","Gap extension cost                  \tnucl:2,aa:1\n","Zdrop                               \t40\n","Threads                             \t4\n","Compressed                          \t0\n","Verbosity                           \t3\n","Seed substitution matrix            \tnucl:nucleotide.out,aa:VTML80.out\n","Sensitivity                         \t7\n","k-mer length                        \t0\n","k-score                             \t2147483647\n","Alphabet size                       \tnucl:5,aa:21\n","Max results per query               \t300\n","Split database                      \t0\n","Split mode                          \t2\n","Split memory limit                  \t0\n","Diagonal scoring                    \ttrue\n","Exact k-mer matching                \t0\n","Mask residues                       \t1\n","Mask lower case residues            \t0\n","Minimum diagonal score              \t15\n","Spaced k-mers                       \t1\n","Spaced k-mer pattern                \t\n","Local temporary path                \t\n","Rescore mode                        \t0\n","Remove hits by seq. id. and coverage\tfalse\n","Sort results                        \t0\n","Mask profile                        \t1\n","Profile e-value threshold           \t0.1\n","Global sequence weighting           \tfalse\n","Allow deletions                     \tfalse\n","Filter MSA                          \t1\n","Maximum seq. id. threshold          \t0.9\n","Minimum seq. id.                    \t0\n","Minimum score per column            \t-20\n","Minimum coverage                    \t0\n","Select N most diverse seqs          \t1000\n","Omit consensus                      \tfalse\n","Min codons in orf                   \t30\n","Max codons in length                \t32734\n","Max orf gaps                        \t2147483647\n","Contig start mode                   \t2\n","Contig end mode                     \t2\n","Orf start mode                      \t1\n","Forward frames                      \t1,2,3\n","Reverse frames                      \t1,2,3\n","Translation table                   \t1\n","Translate orf                       \t0\n","Use all table starts                \tfalse\n","Offset of numeric ids               \t0\n","Create lookup                       \t0\n","Add orf stop                        \tfalse\n","Chain overlapping alignments        \t0\n","Merge query                         \t1\n","Search type                         \t0\n","Search iterations                   \t1\n","Start sensitivity                   \t4\n","Search steps                        \t1\n","Slice search mode                   \tfalse\n","Strand selection                    \t1\n","Disk space limit                    \t0\n","MPI runner                          \t\n","Force restart with latest tmp       \tfalse\n","Remove temporary files              \tfalse\n","\n","prefilter /content/mmseqs2/tmp/8/candidateDB_53 /content/mmseqs2/swissprot/swissprotDB /content/mmseqs2/tmp/8/11133413529865738311/pref_0 --sub-mat nucl:nucleotide.out,aa:blosum62.out --seed-sub-mat nucl:nucleotide.out,aa:VTML80.out -k 0 --k-score 2147483647 --alph-size nucl:5,aa:21 --max-seq-len 65535 --max-seqs 300 --split 0 --split-mode 2 --split-memory-limit 0 -c 0 --cov-mode 0 --comp-bias-corr 1 --diag-score 1 --exact-kmer-matching 0 --mask 1 --mask-lower-case 0 --min-ungapped-score 15 --add-self-matches 0 --spaced-kmer-mode 1 --db-load-mode 0 --pca 1 --pcb 1.5 --threads 4 --compressed 0 -v 3 -s 7.0 \n","\n","Query database size: 10811 type: Aminoacid\n","Estimated memory consumption: 2G\n","Target database size: 562755 type: Aminoacid\n","Index table k-mer threshold: 100 at k-mer size 6 \n","Index table: counting k-mers\n","[=================================================================] 100.00% 562.76K 24s 692ms\n","Index table: Masked residues: 3164874\n","Index table: fill\n","[=================================================================] 100.00% 562.76K 17s 662ms\n","Index statistics\n","Entries:          192757622\n","DB size:          1591 MB\n","Avg k-mer size:   3.011838\n","Top 10 k-mers\n","    GPGGTL\t1835\n","    GQSWTV\t1706\n","    WGMFAT\t1638\n","    PGVFEV\t1634\n","    VLWQFW\t1623\n","    AYIRPN\t1587\n","    RSPKGV\t1585\n","    TPHKWY\t1560\n","    HIRVGW\t1548\n","    ITLSPY\t1541\n","Time for index table init: 0h 0m 44s 463ms\n","Process prefiltering step 1 of 1\n","\n","k-mer similarity threshold: 100\n","Starting prefiltering scores calculation (step 1 of 1)\n","Query db start 1 to 10811\n","Target db start 1 to 562755\n","[=================================================================] 100.00% 10.81K 1m 13s 693ms\n","\n","917.870758 k-mers per position\n","562638 DB matches per sequence\n","553 overflows\n","0 queries produce too many hits (truncated result)\n","279 sequences passed prefiltering per query sequence\n","300 median result list length\n","2 sequences with 0 size result lists\n","Time for merging to pref_0: 0h 0m 0s 2ms\n","Time for processing: 0h 1m 59s 325ms\n","align /content/mmseqs2/tmp/8/candidateDB_53 /content/mmseqs2/swissprot/swissprotDB /content/mmseqs2/tmp/8/11133413529865738311/pref_0 /content/mmseqs2/tmp/8/resultsDB_53 --sub-mat nucl:nucleotide.out,aa:blosum62.out -a 0 --alignment-mode 2 --wrapped-scoring 0 -e 0.001 --min-seq-id 0 --min-aln-len 0 --seq-id-mode 0 --alt-ali 0 -c 0 --cov-mode 0 --max-seq-len 65535 --comp-bias-corr 1 --realign 0 --max-rejected 2147483647 --max-accept 2147483647 --add-self-matches 0 --db-load-mode 0 --pca 1 --pcb 1.5 --score-bias 0 --gap-open nucl:5,aa:11 --gap-extend nucl:2,aa:1 --zdrop 40 --threads 4 --compressed 0 -v 3 \n","\n","Compute score and coverage\n","Query database size: 10811 type: Aminoacid\n","Target database size: 562755 type: Aminoacid\n","Calculation of alignments\n","[=================================================================] 100.00% 10.81K 49s 746ms\n","Time for merging to resultsDB_53: 0h 0m 0s 3ms\n","\n","3017685 alignments calculated.\n","393960 sequence pairs passed the thresholds (0.130550 of overall calculated).\n","36.440662 hits per query sequence.\n","Time for processing: 0h 0m 49s 819ms\n","convertalis /content/mmseqs2/tmp/8/candidateDB_53 /content/mmseqs2/swissprot/swissprotDB /content/mmseqs2/tmp/8/resultsDB_53 /content/mmseqs2/tmp/8/resultDB_secondary.m8 --format-output query,target,evalue,raw \n","\n","MMseqs Version:        \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Substitution matrix    \tnucl:nucleotide.out,aa:blosum62.out\n","Alignment format       \t0\n","Format alignment output\tquery,target,evalue,raw\n","Translation table      \t1\n","Gap open cost          \tnucl:5,aa:11\n","Gap extension cost     \tnucl:2,aa:1\n","Database output        \tfalse\n","Preload mode           \t0\n","Search type            \t0\n","Threads                \t4\n","Compressed             \t0\n","Verbosity              \t3\n","\n","[=================================================================] 100.00% 10.81K 0s 268ms\n","Time for merging to resultDB_secondary.m8: 0h 0m 0s 16ms\n","Time for processing: 0h 0m 0s 378ms\n","found 5564 genes\n","\n","\n","Creating weighted directed acyclic graph of all ORFs and subORFs on contig 1...\n","\n"],"name":"stdout"},{"output_type":"display_data","data":{"application/vnd.jupyter.widget-view+json":{"model_id":"33254081567441e8a0a886e191604f93","version_minor":0,"version_major":2},"text/plain":["HBox(children=(FloatProgress(value=0.0, max=3700.0), HTML(value='')))"]},"metadata":{"tags":[]}},{"output_type":"stream","text":["\n","Maximizing geneiness...\n","\n","Filtering predictions with mmseqs2...\n","createdb /content/mmseqs2/tmp/9/candidate_genes_35.fasta /content/mmseqs2/tmp/9/candidateDB_35 \n","\n","MMseqs Version:       \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Database type         \t0\n","Shuffle input database\ttrue\n","Createdb mode         \t0\n","Write lookup file     \t1\n","Offset of numeric ids \t0\n","Compressed            \t0\n","Verbosity             \t3\n","\n","Converting sequences\n","[203] 0s 2ms\n","Time for merging to candidateDB_35_h: 0h 0m 0s 1ms\n","Time for merging to candidateDB_35: 0h 0m 0s 1ms\n","Database type: Aminoacid\n","Time for processing: 0h 0m 0s 8ms\n","createdb /content/mmseqs2/tmp/9/candidate_genes_53.fasta /content/mmseqs2/tmp/9/candidateDB_53 \n","\n","MMseqs Version:       \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Database type         \t0\n","Shuffle input database\ttrue\n","Createdb mode         \t0\n","Write lookup file     \t1\n","Offset of numeric ids \t0\n","Compressed            \t0\n","Verbosity             \t3\n","\n","Converting sequences\n","[203] 0s 2ms\n","Time for merging to candidateDB_53_h: 0h 0m 0s 2ms\n","Time for merging to candidateDB_53: 0h 0m 0s 1ms\n","Database type: Aminoacid\n","Time for processing: 0h 0m 0s 8ms\n","search -s 7.0 /content/mmseqs2/tmp/9/candidateDB_35 /content/mmseqs2/genexa/genexaDB /content/mmseqs2/tmp/9/resultsDB_35 /content/mmseqs2/tmp/9 \n","\n","MMseqs Version:                     \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Substitution matrix                 \tnucl:nucleotide.out,aa:blosum62.out\n","Add backtrace                       \tfalse\n","Alignment mode                      \t2\n","Allow wrapped scoring               \tfalse\n","E-value threshold                   \t0.001\n","Seq. id. threshold                  \t0\n","Min alignment length                \t0\n","Seq. id. mode                       \t0\n","Alternative alignments              \t0\n","Coverage threshold                  \t0\n","Coverage mode                       \t0\n","Max sequence length                 \t65535\n","Compositional bias                  \t1\n","Realign hits                        \tfalse\n","Max reject                          \t2147483647\n","Max accept                          \t2147483647\n","Include identical seq. id.          \tfalse\n","Preload mode                        \t0\n","Pseudo count a                      \t1\n","Pseudo count b                      \t1.5\n","Score bias                          \t0\n","Gap open cost                       \tnucl:5,aa:11\n","Gap extension cost                  \tnucl:2,aa:1\n","Zdrop                               \t40\n","Threads                             \t4\n","Compressed                          \t0\n","Verbosity                           \t3\n","Seed substitution matrix            \tnucl:nucleotide.out,aa:VTML80.out\n","Sensitivity                         \t7\n","k-mer length                        \t0\n","k-score                             \t2147483647\n","Alphabet size                       \tnucl:5,aa:21\n","Max results per query               \t300\n","Split database                      \t0\n","Split mode                          \t2\n","Split memory limit                  \t0\n","Diagonal scoring                    \ttrue\n","Exact k-mer matching                \t0\n","Mask residues                       \t1\n","Mask lower case residues            \t0\n","Minimum diagonal score              \t15\n","Spaced k-mers                       \t1\n","Spaced k-mer pattern                \t\n","Local temporary path                \t\n","Rescore mode                        \t0\n","Remove hits by seq. id. and coverage\tfalse\n","Sort results                        \t0\n","Mask profile                        \t1\n","Profile e-value threshold           \t0.1\n","Global sequence weighting           \tfalse\n","Allow deletions                     \tfalse\n","Filter MSA                          \t1\n","Maximum seq. id. threshold          \t0.9\n","Minimum seq. id.                    \t0\n","Minimum score per column            \t-20\n","Minimum coverage                    \t0\n","Select N most diverse seqs          \t1000\n","Omit consensus                      \tfalse\n","Min codons in orf                   \t30\n","Max codons in length                \t32734\n","Max orf gaps                        \t2147483647\n","Contig start mode                   \t2\n","Contig end mode                     \t2\n","Orf start mode                      \t1\n","Forward frames                      \t1,2,3\n","Reverse frames                      \t1,2,3\n","Translation table                   \t1\n","Translate orf                       \t0\n","Use all table starts                \tfalse\n","Offset of numeric ids               \t0\n","Create lookup                       \t0\n","Add orf stop                        \tfalse\n","Chain overlapping alignments        \t0\n","Merge query                         \t1\n","Search type                         \t0\n","Search iterations                   \t1\n","Start sensitivity                   \t4\n","Search steps                        \t1\n","Slice search mode                   \tfalse\n","Strand selection                    \t1\n","Disk space limit                    \t0\n","MPI runner                          \t\n","Force restart with latest tmp       \tfalse\n","Remove temporary files              \tfalse\n","\n","prefilter /content/mmseqs2/tmp/9/candidateDB_35 /content/mmseqs2/genexa/genexaDB.idx /content/mmseqs2/tmp/9/15882300637077700098/pref_0 --sub-mat nucl:nucleotide.out,aa:blosum62.out --seed-sub-mat nucl:nucleotide.out,aa:VTML80.out -k 0 --k-score 2147483647 --alph-size nucl:5,aa:21 --max-seq-len 65535 --max-seqs 300 --split 0 --split-mode 2 --split-memory-limit 0 -c 0 --cov-mode 0 --comp-bias-corr 1 --diag-score 1 --exact-kmer-matching 0 --mask 1 --mask-lower-case 0 --min-ungapped-score 15 --add-self-matches 0 --spaced-kmer-mode 1 --db-load-mode 0 --pca 1 --pcb 1.5 --threads 4 --compressed 0 -v 3 -s 7.0 \n","\n","Index version: 16\n","Generated by:  16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","ScoreMatrix:  VTML80.out\n","Query database size: 294 type: Aminoacid\n","Estimated memory consumption: 2G\n","Target database size: 468351 type: Aminoacid\n","Process prefiltering step 1 of 1\n","\n","k-mer similarity threshold: 100\n","Starting prefiltering scores calculation (step 1 of 1)\n","Query db start 1 to 294\n","Target db start 1 to 468351\n","[=================================================================] 100.00% 294 1s 589ms\n","\n","875.873677 k-mers per position\n","402987 DB matches per sequence\n","1 overflows\n","0 queries produce too many hits (truncated result)\n","259 sequences passed prefiltering per query sequence\n","300 median result list length\n","1 sequences with 0 size result lists\n","Time for merging to pref_0: 0h 0m 0s 0ms\n","Time for processing: 0h 0m 3s 2ms\n","align /content/mmseqs2/tmp/9/candidateDB_35 /content/mmseqs2/genexa/genexaDB.idx /content/mmseqs2/tmp/9/15882300637077700098/pref_0 /content/mmseqs2/tmp/9/resultsDB_35 --sub-mat nucl:nucleotide.out,aa:blosum62.out -a 0 --alignment-mode 2 --wrapped-scoring 0 -e 0.001 --min-seq-id 0 --min-aln-len 0 --seq-id-mode 0 --alt-ali 0 -c 0 --cov-mode 0 --max-seq-len 65535 --comp-bias-corr 1 --realign 0 --max-rejected 2147483647 --max-accept 2147483647 --add-self-matches 0 --db-load-mode 0 --pca 1 --pcb 1.5 --score-bias 0 --gap-open nucl:5,aa:11 --gap-extend nucl:2,aa:1 --zdrop 40 --threads 4 --compressed 0 -v 3 \n","\n","Index version: 16\n","Generated by:  16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","ScoreMatrix:  VTML80.out\n","Compute score and coverage\n","Query database size: 294 type: Aminoacid\n","Target database size: 468351 type: Aminoacid\n","Calculation of alignments\n","[=================================================================] 100.00% 294 1s 499ms\n","Time for merging to resultsDB_35: 0h 0m 0s 0ms\n","\n","76294 alignments calculated.\n","20213 sequence pairs passed the thresholds (0.264936 of overall calculated).\n","68.751701 hits per query sequence.\n","Time for processing: 0h 0m 1s 512ms\n","convertalis /content/mmseqs2/tmp/9/candidateDB_35 /content/mmseqs2/genexa/genexaDB /content/mmseqs2/tmp/9/resultsDB_35 /content/mmseqs2/tmp/9/resultDB_genexa.m8 --format-output query,target,evalue,raw \n","\n","MMseqs Version:        \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Substitution matrix    \tnucl:nucleotide.out,aa:blosum62.out\n","Alignment format       \t0\n","Format alignment output\tquery,target,evalue,raw\n","Translation table      \t1\n","Gap open cost          \tnucl:5,aa:11\n","Gap extension cost     \tnucl:2,aa:1\n","Database output        \tfalse\n","Preload mode           \t0\n","Search type            \t0\n","Threads                \t4\n","Compressed             \t0\n","Verbosity              \t3\n","\n","[=================================================================] 100.00% 294 0s 20ms\n","Time for merging to resultDB_genexa.m8: 0h 0m 0s 1ms\n","Time for processing: 0h 0m 0s 87ms\n","search -s 7.0 /content/mmseqs2/tmp/9/candidateDB_53 /content/mmseqs2/swissprot/swissprotDB /content/mmseqs2/tmp/9/resultsDB_53 /content/mmseqs2/tmp/9 \n","\n","MMseqs Version:                     \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Substitution matrix                 \tnucl:nucleotide.out,aa:blosum62.out\n","Add backtrace                       \tfalse\n","Alignment mode                      \t2\n","Allow wrapped scoring               \tfalse\n","E-value threshold                   \t0.001\n","Seq. id. threshold                  \t0\n","Min alignment length                \t0\n","Seq. id. mode                       \t0\n","Alternative alignments              \t0\n","Coverage threshold                  \t0\n","Coverage mode                       \t0\n","Max sequence length                 \t65535\n","Compositional bias                  \t1\n","Realign hits                        \tfalse\n","Max reject                          \t2147483647\n","Max accept                          \t2147483647\n","Include identical seq. id.          \tfalse\n","Preload mode                        \t0\n","Pseudo count a                      \t1\n","Pseudo count b                      \t1.5\n","Score bias                          \t0\n","Gap open cost                       \tnucl:5,aa:11\n","Gap extension cost                  \tnucl:2,aa:1\n","Zdrop                               \t40\n","Threads                             \t4\n","Compressed                          \t0\n","Verbosity                           \t3\n","Seed substitution matrix            \tnucl:nucleotide.out,aa:VTML80.out\n","Sensitivity                         \t7\n","k-mer length                        \t0\n","k-score                             \t2147483647\n","Alphabet size                       \tnucl:5,aa:21\n","Max results per query               \t300\n","Split database                      \t0\n","Split mode                          \t2\n","Split memory limit                  \t0\n","Diagonal scoring                    \ttrue\n","Exact k-mer matching                \t0\n","Mask residues                       \t1\n","Mask lower case residues            \t0\n","Minimum diagonal score              \t15\n","Spaced k-mers                       \t1\n","Spaced k-mer pattern                \t\n","Local temporary path                \t\n","Rescore mode                        \t0\n","Remove hits by seq. id. and coverage\tfalse\n","Sort results                        \t0\n","Mask profile                        \t1\n","Profile e-value threshold           \t0.1\n","Global sequence weighting           \tfalse\n","Allow deletions                     \tfalse\n","Filter MSA                          \t1\n","Maximum seq. id. threshold          \t0.9\n","Minimum seq. id.                    \t0\n","Minimum score per column            \t-20\n","Minimum coverage                    \t0\n","Select N most diverse seqs          \t1000\n","Omit consensus                      \tfalse\n","Min codons in orf                   \t30\n","Max codons in length                \t32734\n","Max orf gaps                        \t2147483647\n","Contig start mode                   \t2\n","Contig end mode                     \t2\n","Orf start mode                      \t1\n","Forward frames                      \t1,2,3\n","Reverse frames                      \t1,2,3\n","Translation table                   \t1\n","Translate orf                       \t0\n","Use all table starts                \tfalse\n","Offset of numeric ids               \t0\n","Create lookup                       \t0\n","Add orf stop                        \tfalse\n","Chain overlapping alignments        \t0\n","Merge query                         \t1\n","Search type                         \t0\n","Search iterations                   \t1\n","Start sensitivity                   \t4\n","Search steps                        \t1\n","Slice search mode                   \tfalse\n","Strand selection                    \t1\n","Disk space limit                    \t0\n","MPI runner                          \t\n","Force restart with latest tmp       \tfalse\n","Remove temporary files              \tfalse\n","\n","prefilter /content/mmseqs2/tmp/9/candidateDB_53 /content/mmseqs2/swissprot/swissprotDB /content/mmseqs2/tmp/9/17639481736885713960/pref_0 --sub-mat nucl:nucleotide.out,aa:blosum62.out --seed-sub-mat nucl:nucleotide.out,aa:VTML80.out -k 0 --k-score 2147483647 --alph-size nucl:5,aa:21 --max-seq-len 65535 --max-seqs 300 --split 0 --split-mode 2 --split-memory-limit 0 -c 0 --cov-mode 0 --comp-bias-corr 1 --diag-score 1 --exact-kmer-matching 0 --mask 1 --mask-lower-case 0 --min-ungapped-score 15 --add-self-matches 0 --spaced-kmer-mode 1 --db-load-mode 0 --pca 1 --pcb 1.5 --threads 4 --compressed 0 -v 3 -s 7.0 \n","\n","Query database size: 294 type: Aminoacid\n","Estimated memory consumption: 2G\n","Target database size: 562755 type: Aminoacid\n","Index table k-mer threshold: 100 at k-mer size 6 \n","Index table: counting k-mers\n","[=================================================================] 100.00% 562.76K 24s 597ms\n","Index table: Masked residues: 3164874\n","Index table: fill\n","[=================================================================] 100.00% 562.76K 17s 971ms\n","Index statistics\n","Entries:          192757622\n","DB size:          1591 MB\n","Avg k-mer size:   3.011838\n","Top 10 k-mers\n","    GPGGTL\t1835\n","    GQSWTV\t1706\n","    WGMFAT\t1638\n","    PGVFEV\t1634\n","    VLWQFW\t1623\n","    AYIRPN\t1587\n","    RSPKGV\t1585\n","    TPHKWY\t1560\n","    HIRVGW\t1548\n","    ITLSPY\t1541\n","Time for index table init: 0h 0m 44s 708ms\n","Process prefiltering step 1 of 1\n","\n","k-mer similarity threshold: 100\n","Starting prefiltering scores calculation (step 1 of 1)\n","Query db start 1 to 294\n","Target db start 1 to 562755\n","[=================================================================] 100.00% 294 1s 792ms\n","\n","879.924287 k-mers per position\n","490777 DB matches per sequence\n","7 overflows\n","0 queries produce too many hits (truncated result)\n","270 sequences passed prefiltering per query sequence\n","300 median result list length\n","1 sequences with 0 size result lists\n","Time for merging to pref_0: 0h 0m 0s 0ms\n","Time for processing: 0h 0m 47s 607ms\n","align /content/mmseqs2/tmp/9/candidateDB_53 /content/mmseqs2/swissprot/swissprotDB /content/mmseqs2/tmp/9/17639481736885713960/pref_0 /content/mmseqs2/tmp/9/resultsDB_53 --sub-mat nucl:nucleotide.out,aa:blosum62.out -a 0 --alignment-mode 2 --wrapped-scoring 0 -e 0.001 --min-seq-id 0 --min-aln-len 0 --seq-id-mode 0 --alt-ali 0 -c 0 --cov-mode 0 --max-seq-len 65535 --comp-bias-corr 1 --realign 0 --max-rejected 2147483647 --max-accept 2147483647 --add-self-matches 0 --db-load-mode 0 --pca 1 --pcb 1.5 --score-bias 0 --gap-open nucl:5,aa:11 --gap-extend nucl:2,aa:1 --zdrop 40 --threads 4 --compressed 0 -v 3 \n","\n","Compute score and coverage\n","Query database size: 294 type: Aminoacid\n","Target database size: 562755 type: Aminoacid\n","Calculation of alignments\n","[=================================================================] 100.00% 294 1s 751ms\n","Time for merging to resultsDB_53: 0h 0m 0s 0ms\n","\n","79660 alignments calculated.\n","19402 sequence pairs passed the thresholds (0.243560 of overall calculated).\n","65.993195 hits per query sequence.\n","Time for processing: 0h 0m 1s 813ms\n","convertalis /content/mmseqs2/tmp/9/candidateDB_53 /content/mmseqs2/swissprot/swissprotDB /content/mmseqs2/tmp/9/resultsDB_53 /content/mmseqs2/tmp/9/resultDB_secondary.m8 --format-output query,target,evalue,raw \n","\n","MMseqs Version:        \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Substitution matrix    \tnucl:nucleotide.out,aa:blosum62.out\n","Alignment format       \t0\n","Format alignment output\tquery,target,evalue,raw\n","Translation table      \t1\n","Gap open cost          \tnucl:5,aa:11\n","Gap extension cost     \tnucl:2,aa:1\n","Database output        \tfalse\n","Preload mode           \t0\n","Search type            \t0\n","Threads                \t4\n","Compressed             \t0\n","Verbosity              \t3\n","\n","[=================================================================] 100.00% 294 0s 13ms\n","Time for merging to resultDB_secondary.m8: 0h 0m 0s 0ms\n","Time for processing: 0h 0m 0s 100ms\n","found 188 genes\n","\n","\n","Creating weighted directed acyclic graph of all ORFs and subORFs on contig 2...\n","\n"],"name":"stdout"},{"output_type":"display_data","data":{"application/vnd.jupyter.widget-view+json":{"model_id":"59be591aa4e04fbf9cebacfd3c697dd4","version_minor":0,"version_major":2},"text/plain":["HBox(children=(FloatProgress(value=0.0, max=24280.0), HTML(value='')))"]},"metadata":{"tags":[]}},{"output_type":"stream","text":["\n","Maximizing geneiness...\n","\n","Filtering predictions with mmseqs2...\n","createdb /content/mmseqs2/tmp/10/candidate_genes_35.fasta /content/mmseqs2/tmp/10/candidateDB_35 \n","\n","MMseqs Version:       \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Database type         \t0\n","Shuffle input database\ttrue\n","Createdb mode         \t0\n","Write lookup file     \t1\n","Offset of numeric ids \t0\n","Compressed            \t0\n","Verbosity             \t3\n","\n","Converting sequences\n","[2425] 0s 4ms\n","Time for merging to candidateDB_35_h: 0h 0m 0s 5ms\n","Time for merging to candidateDB_35: 0h 0m 0s 3ms\n","Database type: Aminoacid\n","Time for processing: 0h 0m 0s 17ms\n","createdb /content/mmseqs2/tmp/10/candidate_genes_53.fasta /content/mmseqs2/tmp/10/candidateDB_53 \n","\n","MMseqs Version:       \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Database type         \t0\n","Shuffle input database\ttrue\n","Createdb mode         \t0\n","Write lookup file     \t1\n","Offset of numeric ids \t0\n","Compressed            \t0\n","Verbosity             \t3\n","\n","Converting sequences\n","[2425] 0s 4ms\n","Time for merging to candidateDB_53_h: 0h 0m 0s 3ms\n","Time for merging to candidateDB_53: 0h 0m 0s 3ms\n","Database type: Aminoacid\n","Time for processing: 0h 0m 0s 15ms\n","search -s 7.0 /content/mmseqs2/tmp/10/candidateDB_35 /content/mmseqs2/genexa/genexaDB /content/mmseqs2/tmp/10/resultsDB_35 /content/mmseqs2/tmp/10 \n","\n","MMseqs Version:                     \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Substitution matrix                 \tnucl:nucleotide.out,aa:blosum62.out\n","Add backtrace                       \tfalse\n","Alignment mode                      \t2\n","Allow wrapped scoring               \tfalse\n","E-value threshold                   \t0.001\n","Seq. id. threshold                  \t0\n","Min alignment length                \t0\n","Seq. id. mode                       \t0\n","Alternative alignments              \t0\n","Coverage threshold                  \t0\n","Coverage mode                       \t0\n","Max sequence length                 \t65535\n","Compositional bias                  \t1\n","Realign hits                        \tfalse\n","Max reject                          \t2147483647\n","Max accept                          \t2147483647\n","Include identical seq. id.          \tfalse\n","Preload mode                        \t0\n","Pseudo count a                      \t1\n","Pseudo count b                      \t1.5\n","Score bias                          \t0\n","Gap open cost                       \tnucl:5,aa:11\n","Gap extension cost                  \tnucl:2,aa:1\n","Zdrop                               \t40\n","Threads                             \t4\n","Compressed                          \t0\n","Verbosity                           \t3\n","Seed substitution matrix            \tnucl:nucleotide.out,aa:VTML80.out\n","Sensitivity                         \t7\n","k-mer length                        \t0\n","k-score                             \t2147483647\n","Alphabet size                       \tnucl:5,aa:21\n","Max results per query               \t300\n","Split database                      \t0\n","Split mode                          \t2\n","Split memory limit                  \t0\n","Diagonal scoring                    \ttrue\n","Exact k-mer matching                \t0\n","Mask residues                       \t1\n","Mask lower case residues            \t0\n","Minimum diagonal score              \t15\n","Spaced k-mers                       \t1\n","Spaced k-mer pattern                \t\n","Local temporary path                \t\n","Rescore mode                        \t0\n","Remove hits by seq. id. and coverage\tfalse\n","Sort results                        \t0\n","Mask profile                        \t1\n","Profile e-value threshold           \t0.1\n","Global sequence weighting           \tfalse\n","Allow deletions                     \tfalse\n","Filter MSA                          \t1\n","Maximum seq. id. threshold          \t0.9\n","Minimum seq. id.                    \t0\n","Minimum score per column            \t-20\n","Minimum coverage                    \t0\n","Select N most diverse seqs          \t1000\n","Omit consensus                      \tfalse\n","Min codons in orf                   \t30\n","Max codons in length                \t32734\n","Max orf gaps                        \t2147483647\n","Contig start mode                   \t2\n","Contig end mode                     \t2\n","Orf start mode                      \t1\n","Forward frames                      \t1,2,3\n","Reverse frames                      \t1,2,3\n","Translation table                   \t1\n","Translate orf                       \t0\n","Use all table starts                \tfalse\n","Offset of numeric ids               \t0\n","Create lookup                       \t0\n","Add orf stop                        \tfalse\n","Chain overlapping alignments        \t0\n","Merge query                         \t1\n","Search type                         \t0\n","Search iterations                   \t1\n","Start sensitivity                   \t4\n","Search steps                        \t1\n","Slice search mode                   \tfalse\n","Strand selection                    \t1\n","Disk space limit                    \t0\n","MPI runner                          \t\n","Force restart with latest tmp       \tfalse\n","Remove temporary files              \tfalse\n","\n","prefilter /content/mmseqs2/tmp/10/candidateDB_35 /content/mmseqs2/genexa/genexaDB.idx /content/mmseqs2/tmp/10/7260291378005499074/pref_0 --sub-mat nucl:nucleotide.out,aa:blosum62.out --seed-sub-mat nucl:nucleotide.out,aa:VTML80.out -k 0 --k-score 2147483647 --alph-size nucl:5,aa:21 --max-seq-len 65535 --max-seqs 300 --split 0 --split-mode 2 --split-memory-limit 0 -c 0 --cov-mode 0 --comp-bias-corr 1 --diag-score 1 --exact-kmer-matching 0 --mask 1 --mask-lower-case 0 --min-ungapped-score 15 --add-self-matches 0 --spaced-kmer-mode 1 --db-load-mode 0 --pca 1 --pcb 1.5 --threads 4 --compressed 0 -v 3 -s 7.0 \n","\n","Index version: 16\n","Generated by:  16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","ScoreMatrix:  VTML80.out\n","Query database size: 2487 type: Aminoacid\n","Estimated memory consumption: 2G\n","Target database size: 468351 type: Aminoacid\n","Process prefiltering step 1 of 1\n","\n","k-mer similarity threshold: 100\n","Starting prefiltering scores calculation (step 1 of 1)\n","Query db start 1 to 2487\n","Target db start 1 to 468351\n","[=================================================================] 100.00% 2.49K 16s 416ms\n","\n","898.849265 k-mers per position\n","515814 DB matches per sequence\n","98 overflows\n","0 queries produce too many hits (truncated result)\n","268 sequences passed prefiltering per query sequence\n","300 median result list length\n","2 sequences with 0 size result lists\n","Time for merging to pref_0: 0h 0m 0s 1ms\n","Time for processing: 0h 0m 18s 0ms\n","align /content/mmseqs2/tmp/10/candidateDB_35 /content/mmseqs2/genexa/genexaDB.idx /content/mmseqs2/tmp/10/7260291378005499074/pref_0 /content/mmseqs2/tmp/10/resultsDB_35 --sub-mat nucl:nucleotide.out,aa:blosum62.out -a 0 --alignment-mode 2 --wrapped-scoring 0 -e 0.001 --min-seq-id 0 --min-aln-len 0 --seq-id-mode 0 --alt-ali 0 -c 0 --cov-mode 0 --max-seq-len 65535 --comp-bias-corr 1 --realign 0 --max-rejected 2147483647 --max-accept 2147483647 --add-self-matches 0 --db-load-mode 0 --pca 1 --pcb 1.5 --score-bias 0 --gap-open nucl:5,aa:11 --gap-extend nucl:2,aa:1 --zdrop 40 --threads 4 --compressed 0 -v 3 \n","\n","Index version: 16\n","Generated by:  16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","ScoreMatrix:  VTML80.out\n","Compute score and coverage\n","Query database size: 2487 type: Aminoacid\n","Target database size: 468351 type: Aminoacid\n","Calculation of alignments\n","[=================================================================] 100.00% 2.49K 15s 395ms\n","Time for merging to resultsDB_35: 0h 0m 0s 1ms\n","\n","666981 alignments calculated.\n","199378 sequence pairs passed the thresholds (0.298926 of overall calculated).\n","80.168076 hits per query sequence.\n","Time for processing: 0h 0m 15s 448ms\n","convertalis /content/mmseqs2/tmp/10/candidateDB_35 /content/mmseqs2/genexa/genexaDB /content/mmseqs2/tmp/10/resultsDB_35 /content/mmseqs2/tmp/10/resultDB_genexa.m8 --format-output query,target,evalue,raw \n","\n","MMseqs Version:        \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Substitution matrix    \tnucl:nucleotide.out,aa:blosum62.out\n","Alignment format       \t0\n","Format alignment output\tquery,target,evalue,raw\n","Translation table      \t1\n","Gap open cost          \tnucl:5,aa:11\n","Gap extension cost     \tnucl:2,aa:1\n","Database output        \tfalse\n","Preload mode           \t0\n","Search type            \t0\n","Threads                \t4\n","Compressed             \t0\n","Verbosity              \t3\n","\n","[=================================================================] 100.00% 2.49K 0s 145ms\n","Time for merging to resultDB_genexa.m8: 0h 0m 0s 6ms\n","Time for processing: 0h 0m 0s 222ms\n","search -s 7.0 /content/mmseqs2/tmp/10/candidateDB_53 /content/mmseqs2/swissprot/swissprotDB /content/mmseqs2/tmp/10/resultsDB_53 /content/mmseqs2/tmp/10 \n","\n","MMseqs Version:                     \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Substitution matrix                 \tnucl:nucleotide.out,aa:blosum62.out\n","Add backtrace                       \tfalse\n","Alignment mode                      \t2\n","Allow wrapped scoring               \tfalse\n","E-value threshold                   \t0.001\n","Seq. id. threshold                  \t0\n","Min alignment length                \t0\n","Seq. id. mode                       \t0\n","Alternative alignments              \t0\n","Coverage threshold                  \t0\n","Coverage mode                       \t0\n","Max sequence length                 \t65535\n","Compositional bias                  \t1\n","Realign hits                        \tfalse\n","Max reject                          \t2147483647\n","Max accept                          \t2147483647\n","Include identical seq. id.          \tfalse\n","Preload mode                        \t0\n","Pseudo count a                      \t1\n","Pseudo count b                      \t1.5\n","Score bias                          \t0\n","Gap open cost                       \tnucl:5,aa:11\n","Gap extension cost                  \tnucl:2,aa:1\n","Zdrop                               \t40\n","Threads                             \t4\n","Compressed                          \t0\n","Verbosity                           \t3\n","Seed substitution matrix            \tnucl:nucleotide.out,aa:VTML80.out\n","Sensitivity                         \t7\n","k-mer length                        \t0\n","k-score                             \t2147483647\n","Alphabet size                       \tnucl:5,aa:21\n","Max results per query               \t300\n","Split database                      \t0\n","Split mode                          \t2\n","Split memory limit                  \t0\n","Diagonal scoring                    \ttrue\n","Exact k-mer matching                \t0\n","Mask residues                       \t1\n","Mask lower case residues            \t0\n","Minimum diagonal score              \t15\n","Spaced k-mers                       \t1\n","Spaced k-mer pattern                \t\n","Local temporary path                \t\n","Rescore mode                        \t0\n","Remove hits by seq. id. and coverage\tfalse\n","Sort results                        \t0\n","Mask profile                        \t1\n","Profile e-value threshold           \t0.1\n","Global sequence weighting           \tfalse\n","Allow deletions                     \tfalse\n","Filter MSA                          \t1\n","Maximum seq. id. threshold          \t0.9\n","Minimum seq. id.                    \t0\n","Minimum score per column            \t-20\n","Minimum coverage                    \t0\n","Select N most diverse seqs          \t1000\n","Omit consensus                      \tfalse\n","Min codons in orf                   \t30\n","Max codons in length                \t32734\n","Max orf gaps                        \t2147483647\n","Contig start mode                   \t2\n","Contig end mode                     \t2\n","Orf start mode                      \t1\n","Forward frames                      \t1,2,3\n","Reverse frames                      \t1,2,3\n","Translation table                   \t1\n","Translate orf                       \t0\n","Use all table starts                \tfalse\n","Offset of numeric ids               \t0\n","Create lookup                       \t0\n","Add orf stop                        \tfalse\n","Chain overlapping alignments        \t0\n","Merge query                         \t1\n","Search type                         \t0\n","Search iterations                   \t1\n","Start sensitivity                   \t4\n","Search steps                        \t1\n","Slice search mode                   \tfalse\n","Strand selection                    \t1\n","Disk space limit                    \t0\n","MPI runner                          \t\n","Force restart with latest tmp       \tfalse\n","Remove temporary files              \tfalse\n","\n","prefilter /content/mmseqs2/tmp/10/candidateDB_53 /content/mmseqs2/swissprot/swissprotDB /content/mmseqs2/tmp/10/4212223201576341984/pref_0 --sub-mat nucl:nucleotide.out,aa:blosum62.out --seed-sub-mat nucl:nucleotide.out,aa:VTML80.out -k 0 --k-score 2147483647 --alph-size nucl:5,aa:21 --max-seq-len 65535 --max-seqs 300 --split 0 --split-mode 2 --split-memory-limit 0 -c 0 --cov-mode 0 --comp-bias-corr 1 --diag-score 1 --exact-kmer-matching 0 --mask 1 --mask-lower-case 0 --min-ungapped-score 15 --add-self-matches 0 --spaced-kmer-mode 1 --db-load-mode 0 --pca 1 --pcb 1.5 --threads 4 --compressed 0 -v 3 -s 7.0 \n","\n","Query database size: 2487 type: Aminoacid\n","Estimated memory consumption: 2G\n","Target database size: 562755 type: Aminoacid\n","Index table k-mer threshold: 100 at k-mer size 6 \n","Index table: counting k-mers\n","[=================================================================] 100.00% 562.76K 24s 575ms\n","Index table: Masked residues: 3164874\n","Index table: fill\n","[=================================================================] 100.00% 562.76K 18s 43ms\n","Index statistics\n","Entries:          192757622\n","DB size:          1591 MB\n","Avg k-mer size:   3.011838\n","Top 10 k-mers\n","    GPGGTL\t1835\n","    GQSWTV\t1706\n","    WGMFAT\t1638\n","    PGVFEV\t1634\n","    VLWQFW\t1623\n","    AYIRPN\t1587\n","    RSPKGV\t1585\n","    TPHKWY\t1560\n","    HIRVGW\t1548\n","    ITLSPY\t1541\n","Time for index table init: 0h 0m 44s 757ms\n","Process prefiltering step 1 of 1\n","\n","k-mer similarity threshold: 100\n","Starting prefiltering scores calculation (step 1 of 1)\n","Query db start 1 to 2487\n","Target db start 1 to 562755\n","[=================================================================] 100.00% 2.49K 18s 361ms\n","\n","899.888710 k-mers per position\n","615920 DB matches per sequence\n","159 overflows\n","0 queries produce too many hits (truncated result)\n","278 sequences passed prefiltering per query sequence\n","300 median result list length\n","2 sequences with 0 size result lists\n","Time for merging to pref_0: 0h 0m 0s 1ms\n","Time for processing: 0h 1m 4s 395ms\n","align /content/mmseqs2/tmp/10/candidateDB_53 /content/mmseqs2/swissprot/swissprotDB /content/mmseqs2/tmp/10/4212223201576341984/pref_0 /content/mmseqs2/tmp/10/resultsDB_53 --sub-mat nucl:nucleotide.out,aa:blosum62.out -a 0 --alignment-mode 2 --wrapped-scoring 0 -e 0.001 --min-seq-id 0 --min-aln-len 0 --seq-id-mode 0 --alt-ali 0 -c 0 --cov-mode 0 --max-seq-len 65535 --comp-bias-corr 1 --realign 0 --max-rejected 2147483647 --max-accept 2147483647 --add-self-matches 0 --db-load-mode 0 --pca 1 --pcb 1.5 --score-bias 0 --gap-open nucl:5,aa:11 --gap-extend nucl:2,aa:1 --zdrop 40 --threads 4 --compressed 0 -v 3 \n","\n","Compute score and coverage\n","Query database size: 2487 type: Aminoacid\n","Target database size: 562755 type: Aminoacid\n","Calculation of alignments\n","[=================================================================] 100.00% 2.49K 17s 392ms\n","Time for merging to resultsDB_53: 0h 0m 0s 0ms\n","\n","692659 alignments calculated.\n","170362 sequence pairs passed the thresholds (0.245954 of overall calculated).\n","68.501007 hits per query sequence.\n","Time for processing: 0h 0m 17s 504ms\n","convertalis /content/mmseqs2/tmp/10/candidateDB_53 /content/mmseqs2/swissprot/swissprotDB /content/mmseqs2/tmp/10/resultsDB_53 /content/mmseqs2/tmp/10/resultDB_secondary.m8 --format-output query,target,evalue,raw \n","\n","MMseqs Version:        \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Substitution matrix    \tnucl:nucleotide.out,aa:blosum62.out\n","Alignment format       \t0\n","Format alignment output\tquery,target,evalue,raw\n","Translation table      \t1\n","Gap open cost          \tnucl:5,aa:11\n","Gap extension cost     \tnucl:2,aa:1\n","Database output        \tfalse\n","Preload mode           \t0\n","Search type            \t0\n","Threads                \t4\n","Compressed             \t0\n","Verbosity              \t3\n","\n","[=================================================================] 100.00% 2.49K 0s 119ms\n","Time for merging to resultDB_secondary.m8: 0h 0m 0s 6ms\n","Time for processing: 0h 0m 0s 217ms\n","found 1398 genes\n","\n","\n","Creating weighted directed acyclic graph of all ORFs and subORFs on contig 3...\n","\n"],"name":"stdout"},{"output_type":"display_data","data":{"application/vnd.jupyter.widget-view+json":{"model_id":"14da851f7f264917b338e5c2ea0cd230","version_minor":0,"version_major":2},"text/plain":["HBox(children=(FloatProgress(value=0.0, max=2086.0), HTML(value='')))"]},"metadata":{"tags":[]}},{"output_type":"stream","text":["\n","Maximizing geneiness...\n","\n","Filtering predictions with mmseqs2...\n","createdb /content/mmseqs2/tmp/11/candidate_genes_35.fasta /content/mmseqs2/tmp/11/candidateDB_35 \n","\n","MMseqs Version:       \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Database type         \t0\n","Shuffle input database\ttrue\n","Createdb mode         \t0\n","Write lookup file     \t1\n","Offset of numeric ids \t0\n","Compressed            \t0\n","Verbosity             \t3\n","\n","Converting sequences\n","[203] 0s 2ms\n","Time for merging to candidateDB_35_h: 0h 0m 0s 1ms\n","Time for merging to candidateDB_35: 0h 0m 0s 1ms\n","Database type: Aminoacid\n","Time for processing: 0h 0m 0s 8ms\n","createdb /content/mmseqs2/tmp/11/candidate_genes_53.fasta /content/mmseqs2/tmp/11/candidateDB_53 \n","\n","MMseqs Version:       \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Database type         \t0\n","Shuffle input database\ttrue\n","Createdb mode         \t0\n","Write lookup file     \t1\n","Offset of numeric ids \t0\n","Compressed            \t0\n","Verbosity             \t3\n","\n","Converting sequences\n","[203] 0s 2ms\n","Time for merging to candidateDB_53_h: 0h 0m 0s 2ms\n","Time for merging to candidateDB_53: 0h 0m 0s 1ms\n","Database type: Aminoacid\n","Time for processing: 0h 0m 0s 8ms\n","search -s 7.0 /content/mmseqs2/tmp/11/candidateDB_35 /content/mmseqs2/genexa/genexaDB /content/mmseqs2/tmp/11/resultsDB_35 /content/mmseqs2/tmp/11 \n","\n","MMseqs Version:                     \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Substitution matrix                 \tnucl:nucleotide.out,aa:blosum62.out\n","Add backtrace                       \tfalse\n","Alignment mode                      \t2\n","Allow wrapped scoring               \tfalse\n","E-value threshold                   \t0.001\n","Seq. id. threshold                  \t0\n","Min alignment length                \t0\n","Seq. id. mode                       \t0\n","Alternative alignments              \t0\n","Coverage threshold                  \t0\n","Coverage mode                       \t0\n","Max sequence length                 \t65535\n","Compositional bias                  \t1\n","Realign hits                        \tfalse\n","Max reject                          \t2147483647\n","Max accept                          \t2147483647\n","Include identical seq. id.          \tfalse\n","Preload mode                        \t0\n","Pseudo count a                      \t1\n","Pseudo count b                      \t1.5\n","Score bias                          \t0\n","Gap open cost                       \tnucl:5,aa:11\n","Gap extension cost                  \tnucl:2,aa:1\n","Zdrop                               \t40\n","Threads                             \t4\n","Compressed                          \t0\n","Verbosity                           \t3\n","Seed substitution matrix            \tnucl:nucleotide.out,aa:VTML80.out\n","Sensitivity                         \t7\n","k-mer length                        \t0\n","k-score                             \t2147483647\n","Alphabet size                       \tnucl:5,aa:21\n","Max results per query               \t300\n","Split database                      \t0\n","Split mode                          \t2\n","Split memory limit                  \t0\n","Diagonal scoring                    \ttrue\n","Exact k-mer matching                \t0\n","Mask residues                       \t1\n","Mask lower case residues            \t0\n","Minimum diagonal score              \t15\n","Spaced k-mers                       \t1\n","Spaced k-mer pattern                \t\n","Local temporary path                \t\n","Rescore mode                        \t0\n","Remove hits by seq. id. and coverage\tfalse\n","Sort results                        \t0\n","Mask profile                        \t1\n","Profile e-value threshold           \t0.1\n","Global sequence weighting           \tfalse\n","Allow deletions                     \tfalse\n","Filter MSA                          \t1\n","Maximum seq. id. threshold          \t0.9\n","Minimum seq. id.                    \t0\n","Minimum score per column            \t-20\n","Minimum coverage                    \t0\n","Select N most diverse seqs          \t1000\n","Omit consensus                      \tfalse\n","Min codons in orf                   \t30\n","Max codons in length                \t32734\n","Max orf gaps                        \t2147483647\n","Contig start mode                   \t2\n","Contig end mode                     \t2\n","Orf start mode                      \t1\n","Forward frames                      \t1,2,3\n","Reverse frames                      \t1,2,3\n","Translation table                   \t1\n","Translate orf                       \t0\n","Use all table starts                \tfalse\n","Offset of numeric ids               \t0\n","Create lookup                       \t0\n","Add orf stop                        \tfalse\n","Chain overlapping alignments        \t0\n","Merge query                         \t1\n","Search type                         \t0\n","Search iterations                   \t1\n","Start sensitivity                   \t4\n","Search steps                        \t1\n","Slice search mode                   \tfalse\n","Strand selection                    \t1\n","Disk space limit                    \t0\n","MPI runner                          \t\n","Force restart with latest tmp       \tfalse\n","Remove temporary files              \tfalse\n","\n","prefilter /content/mmseqs2/tmp/11/candidateDB_35 /content/mmseqs2/genexa/genexaDB.idx /content/mmseqs2/tmp/11/2000005274895803585/pref_0 --sub-mat nucl:nucleotide.out,aa:blosum62.out --seed-sub-mat nucl:nucleotide.out,aa:VTML80.out -k 0 --k-score 2147483647 --alph-size nucl:5,aa:21 --max-seq-len 65535 --max-seqs 300 --split 0 --split-mode 2 --split-memory-limit 0 -c 0 --cov-mode 0 --comp-bias-corr 1 --diag-score 1 --exact-kmer-matching 0 --mask 1 --mask-lower-case 0 --min-ungapped-score 15 --add-self-matches 0 --spaced-kmer-mode 1 --db-load-mode 0 --pca 1 --pcb 1.5 --threads 4 --compressed 0 -v 3 -s 7.0 \n","\n","Index version: 16\n","Generated by:  16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","ScoreMatrix:  VTML80.out\n","Query database size: 212 type: Aminoacid\n","Estimated memory consumption: 2G\n","Target database size: 468351 type: Aminoacid\n","Process prefiltering step 1 of 1\n","\n","k-mer similarity threshold: 100\n","Starting prefiltering scores calculation (step 1 of 1)\n","Query db start 1 to 212\n","Target db start 1 to 468351\n","[=================================================================] 100.00% 212 1s 6ms\n","\n","903.817253 k-mers per position\n","360552 DB matches per sequence\n","1 overflows\n","0 queries produce too many hits (truncated result)\n","259 sequences passed prefiltering per query sequence\n","300 median result list length\n","0 sequences with 0 size result lists\n","Time for merging to pref_0: 0h 0m 0s 0ms\n","Time for processing: 0h 0m 2s 416ms\n","align /content/mmseqs2/tmp/11/candidateDB_35 /content/mmseqs2/genexa/genexaDB.idx /content/mmseqs2/tmp/11/2000005274895803585/pref_0 /content/mmseqs2/tmp/11/resultsDB_35 --sub-mat nucl:nucleotide.out,aa:blosum62.out -a 0 --alignment-mode 2 --wrapped-scoring 0 -e 0.001 --min-seq-id 0 --min-aln-len 0 --seq-id-mode 0 --alt-ali 0 -c 0 --cov-mode 0 --max-seq-len 65535 --comp-bias-corr 1 --realign 0 --max-rejected 2147483647 --max-accept 2147483647 --add-self-matches 0 --db-load-mode 0 --pca 1 --pcb 1.5 --score-bias 0 --gap-open nucl:5,aa:11 --gap-extend nucl:2,aa:1 --zdrop 40 --threads 4 --compressed 0 -v 3 \n","\n","Index version: 16\n","Generated by:  16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","ScoreMatrix:  VTML80.out\n","Compute score and coverage\n","Query database size: 212 type: Aminoacid\n","Target database size: 468351 type: Aminoacid\n","Calculation of alignments\n","[=================================================================] 100.00% 212 0s 983ms\n","Time for merging to resultsDB_35: 0h 0m 0s 0ms\n","\n","55044 alignments calculated.\n","13016 sequence pairs passed the thresholds (0.236465 of overall calculated).\n","61.396225 hits per query sequence.\n","Time for processing: 0h 0m 1s 40ms\n","convertalis /content/mmseqs2/tmp/11/candidateDB_35 /content/mmseqs2/genexa/genexaDB /content/mmseqs2/tmp/11/resultsDB_35 /content/mmseqs2/tmp/11/resultDB_genexa.m8 --format-output query,target,evalue,raw \n","\n","MMseqs Version:        \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Substitution matrix    \tnucl:nucleotide.out,aa:blosum62.out\n","Alignment format       \t0\n","Format alignment output\tquery,target,evalue,raw\n","Translation table      \t1\n","Gap open cost          \tnucl:5,aa:11\n","Gap extension cost     \tnucl:2,aa:1\n","Database output        \tfalse\n","Preload mode           \t0\n","Search type            \t0\n","Threads                \t4\n","Compressed             \t0\n","Verbosity              \t3\n","\n","[=================================================================] 100.00% 212 0s 10ms\n","Time for merging to resultDB_genexa.m8: 0h 0m 0s 0ms\n","Time for processing: 0h 0m 0s 74ms\n","search -s 7.0 /content/mmseqs2/tmp/11/candidateDB_53 /content/mmseqs2/swissprot/swissprotDB /content/mmseqs2/tmp/11/resultsDB_53 /content/mmseqs2/tmp/11 \n","\n","MMseqs Version:                     \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Substitution matrix                 \tnucl:nucleotide.out,aa:blosum62.out\n","Add backtrace                       \tfalse\n","Alignment mode                      \t2\n","Allow wrapped scoring               \tfalse\n","E-value threshold                   \t0.001\n","Seq. id. threshold                  \t0\n","Min alignment length                \t0\n","Seq. id. mode                       \t0\n","Alternative alignments              \t0\n","Coverage threshold                  \t0\n","Coverage mode                       \t0\n","Max sequence length                 \t65535\n","Compositional bias                  \t1\n","Realign hits                        \tfalse\n","Max reject                          \t2147483647\n","Max accept                          \t2147483647\n","Include identical seq. id.          \tfalse\n","Preload mode                        \t0\n","Pseudo count a                      \t1\n","Pseudo count b                      \t1.5\n","Score bias                          \t0\n","Gap open cost                       \tnucl:5,aa:11\n","Gap extension cost                  \tnucl:2,aa:1\n","Zdrop                               \t40\n","Threads                             \t4\n","Compressed                          \t0\n","Verbosity                           \t3\n","Seed substitution matrix            \tnucl:nucleotide.out,aa:VTML80.out\n","Sensitivity                         \t7\n","k-mer length                        \t0\n","k-score                             \t2147483647\n","Alphabet size                       \tnucl:5,aa:21\n","Max results per query               \t300\n","Split database                      \t0\n","Split mode                          \t2\n","Split memory limit                  \t0\n","Diagonal scoring                    \ttrue\n","Exact k-mer matching                \t0\n","Mask residues                       \t1\n","Mask lower case residues            \t0\n","Minimum diagonal score              \t15\n","Spaced k-mers                       \t1\n","Spaced k-mer pattern                \t\n","Local temporary path                \t\n","Rescore mode                        \t0\n","Remove hits by seq. id. and coverage\tfalse\n","Sort results                        \t0\n","Mask profile                        \t1\n","Profile e-value threshold           \t0.1\n","Global sequence weighting           \tfalse\n","Allow deletions                     \tfalse\n","Filter MSA                          \t1\n","Maximum seq. id. threshold          \t0.9\n","Minimum seq. id.                    \t0\n","Minimum score per column            \t-20\n","Minimum coverage                    \t0\n","Select N most diverse seqs          \t1000\n","Omit consensus                      \tfalse\n","Min codons in orf                   \t30\n","Max codons in length                \t32734\n","Max orf gaps                        \t2147483647\n","Contig start mode                   \t2\n","Contig end mode                     \t2\n","Orf start mode                      \t1\n","Forward frames                      \t1,2,3\n","Reverse frames                      \t1,2,3\n","Translation table                   \t1\n","Translate orf                       \t0\n","Use all table starts                \tfalse\n","Offset of numeric ids               \t0\n","Create lookup                       \t0\n","Add orf stop                        \tfalse\n","Chain overlapping alignments        \t0\n","Merge query                         \t1\n","Search type                         \t0\n","Search iterations                   \t1\n","Start sensitivity                   \t4\n","Search steps                        \t1\n","Slice search mode                   \tfalse\n","Strand selection                    \t1\n","Disk space limit                    \t0\n","MPI runner                          \t\n","Force restart with latest tmp       \tfalse\n","Remove temporary files              \tfalse\n","\n","prefilter /content/mmseqs2/tmp/11/candidateDB_53 /content/mmseqs2/swissprot/swissprotDB /content/mmseqs2/tmp/11/4313588189621687967/pref_0 --sub-mat nucl:nucleotide.out,aa:blosum62.out --seed-sub-mat nucl:nucleotide.out,aa:VTML80.out -k 0 --k-score 2147483647 --alph-size nucl:5,aa:21 --max-seq-len 65535 --max-seqs 300 --split 0 --split-mode 2 --split-memory-limit 0 -c 0 --cov-mode 0 --comp-bias-corr 1 --diag-score 1 --exact-kmer-matching 0 --mask 1 --mask-lower-case 0 --min-ungapped-score 15 --add-self-matches 0 --spaced-kmer-mode 1 --db-load-mode 0 --pca 1 --pcb 1.5 --threads 4 --compressed 0 -v 3 -s 7.0 \n","\n","Query database size: 212 type: Aminoacid\n","Estimated memory consumption: 2G\n","Target database size: 562755 type: Aminoacid\n","Index table k-mer threshold: 100 at k-mer size 6 \n","Index table: counting k-mers\n","[=================================================================] 100.00% 562.76K 24s 559ms\n","Index table: Masked residues: 3164874\n","Index table: fill\n","[=================================================================] 100.00% 562.76K 18s 1ms\n","Index statistics\n","Entries:          192757622\n","DB size:          1591 MB\n","Avg k-mer size:   3.011838\n","Top 10 k-mers\n","    GPGGTL\t1835\n","    GQSWTV\t1706\n","    WGMFAT\t1638\n","    PGVFEV\t1634\n","    VLWQFW\t1623\n","    AYIRPN\t1587\n","    RSPKGV\t1585\n","    TPHKWY\t1560\n","    HIRVGW\t1548\n","    ITLSPY\t1541\n","Time for index table init: 0h 0m 44s 624ms\n","Process prefiltering step 1 of 1\n","\n","k-mer similarity threshold: 100\n","Starting prefiltering scores calculation (step 1 of 1)\n","Query db start 1 to 212\n","Target db start 1 to 562755\n","[=================================================================] 100.00% 212 1s 170ms\n","\n","904.078855 k-mers per position\n","448614 DB matches per sequence\n","6 overflows\n","0 queries produce too many hits (truncated result)\n","268 sequences passed prefiltering per query sequence\n","300 median result list length\n","0 sequences with 0 size result lists\n","Time for merging to pref_0: 0h 0m 0s 0ms\n","Time for processing: 0h 0m 46s 915ms\n","align /content/mmseqs2/tmp/11/candidateDB_53 /content/mmseqs2/swissprot/swissprotDB /content/mmseqs2/tmp/11/4313588189621687967/pref_0 /content/mmseqs2/tmp/11/resultsDB_53 --sub-mat nucl:nucleotide.out,aa:blosum62.out -a 0 --alignment-mode 2 --wrapped-scoring 0 -e 0.001 --min-seq-id 0 --min-aln-len 0 --seq-id-mode 0 --alt-ali 0 -c 0 --cov-mode 0 --max-seq-len 65535 --comp-bias-corr 1 --realign 0 --max-rejected 2147483647 --max-accept 2147483647 --add-self-matches 0 --db-load-mode 0 --pca 1 --pcb 1.5 --score-bias 0 --gap-open nucl:5,aa:11 --gap-extend nucl:2,aa:1 --zdrop 40 --threads 4 --compressed 0 -v 3 \n","\n","Compute score and coverage\n","Query database size: 212 type: Aminoacid\n","Target database size: 562755 type: Aminoacid\n","Calculation of alignments\n","[=================================================================] 100.00% 212 1s 387ms\n","Time for merging to resultsDB_53: 0h 0m 0s 0ms\n","\n","56877 alignments calculated.\n","14254 sequence pairs passed the thresholds (0.250611 of overall calculated).\n","67.235847 hits per query sequence.\n","Time for processing: 0h 0m 1s 457ms\n","convertalis /content/mmseqs2/tmp/11/candidateDB_53 /content/mmseqs2/swissprot/swissprotDB /content/mmseqs2/tmp/11/resultsDB_53 /content/mmseqs2/tmp/11/resultDB_secondary.m8 --format-output query,target,evalue,raw \n","\n","MMseqs Version:        \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Substitution matrix    \tnucl:nucleotide.out,aa:blosum62.out\n","Alignment format       \t0\n","Format alignment output\tquery,target,evalue,raw\n","Translation table      \t1\n","Gap open cost          \tnucl:5,aa:11\n","Gap extension cost     \tnucl:2,aa:1\n","Database output        \tfalse\n","Preload mode           \t0\n","Search type            \t0\n","Threads                \t4\n","Compressed             \t0\n","Verbosity              \t3\n","\n","[=================================================================] 100.00% 212 0s 12ms\n","Time for merging to resultDB_secondary.m8: 0h 0m 0s 0ms\n","Time for processing: 0h 0m 0s 96ms\n","found 110 genes\n","\n","\n","Reading fasta file GCF_002113825.1_ASM211382v1_genomic.fna.gz...\n","\n","Finding and translating open reading frames...\n","\n","Encoding amino acids...\n","\n","Applying kmer prefilter...\n","\n","Scoring ORFs with temporal convolutional network...\n","\n"],"name":"stdout"},{"output_type":"display_data","data":{"application/vnd.jupyter.widget-view+json":{"model_id":"ad34d5fafbea45598758d5a9658ebdc6","version_minor":0,"version_major":2},"text/plain":["HBox(children=(FloatProgress(value=0.0, max=44.0), HTML(value='')))"]},"metadata":{"tags":[]}},{"output_type":"stream","text":["\n","Scoring translation initiation sites...\n","\n"],"name":"stdout"},{"output_type":"display_data","data":{"application/vnd.jupyter.widget-view+json":{"model_id":"b5536e67d2a641debb7a27a6e3c1764a","version_minor":0,"version_major":2},"text/plain":["HBox(children=(FloatProgress(value=0.0, max=14.0), HTML(value='')))"]},"metadata":{"tags":[]}},{"output_type":"stream","text":["\n","Creating weighted directed acyclic graph of all ORFs and subORFs on contig 0...\n","\n"],"name":"stdout"},{"output_type":"display_data","data":{"application/vnd.jupyter.widget-view+json":{"model_id":"e0dc1c3d48334cbfa8d6730667496ee5","version_minor":0,"version_major":2},"text/plain":["HBox(children=(FloatProgress(value=0.0, max=130767.0), HTML(value='')))"]},"metadata":{"tags":[]}},{"output_type":"stream","text":["\n","Maximizing geneiness...\n","\n","Filtering predictions with mmseqs2...\n","createdb /content/mmseqs2/tmp/12/candidate_genes_35.fasta /content/mmseqs2/tmp/12/candidateDB_35 \n","\n","MMseqs Version:       \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Database type         \t0\n","Shuffle input database\ttrue\n","Createdb mode         \t0\n","Write lookup file     \t1\n","Offset of numeric ids \t0\n","Compressed            \t0\n","Verbosity             \t3\n","\n","Converting sequences\n","[3839] 0s 6ms\n","Time for merging to candidateDB_35_h: 0h 0m 0s 2ms\n","Time for merging to candidateDB_35: 0h 0m 0s 3ms\n","Database type: Aminoacid\n","Time for processing: 0h 0m 0s 18ms\n","createdb /content/mmseqs2/tmp/12/candidate_genes_53.fasta /content/mmseqs2/tmp/12/candidateDB_53 \n","\n","MMseqs Version:       \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Database type         \t0\n","Shuffle input database\ttrue\n","Createdb mode         \t0\n","Write lookup file     \t1\n","Offset of numeric ids \t0\n","Compressed            \t0\n","Verbosity             \t3\n","\n","Converting sequences\n","[3839] 0s 5ms\n","Time for merging to candidateDB_53_h: 0h 0m 0s 5ms\n","Time for merging to candidateDB_53: 0h 0m 0s 4ms\n","Database type: Aminoacid\n","Time for processing: 0h 0m 0s 20ms\n","search -s 7.0 /content/mmseqs2/tmp/12/candidateDB_35 /content/mmseqs2/genexa/genexaDB /content/mmseqs2/tmp/12/resultsDB_35 /content/mmseqs2/tmp/12 \n","\n","MMseqs Version:                     \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Substitution matrix                 \tnucl:nucleotide.out,aa:blosum62.out\n","Add backtrace                       \tfalse\n","Alignment mode                      \t2\n","Allow wrapped scoring               \tfalse\n","E-value threshold                   \t0.001\n","Seq. id. threshold                  \t0\n","Min alignment length                \t0\n","Seq. id. mode                       \t0\n","Alternative alignments              \t0\n","Coverage threshold                  \t0\n","Coverage mode                       \t0\n","Max sequence length                 \t65535\n","Compositional bias                  \t1\n","Realign hits                        \tfalse\n","Max reject                          \t2147483647\n","Max accept                          \t2147483647\n","Include identical seq. id.          \tfalse\n","Preload mode                        \t0\n","Pseudo count a                      \t1\n","Pseudo count b                      \t1.5\n","Score bias                          \t0\n","Gap open cost                       \tnucl:5,aa:11\n","Gap extension cost                  \tnucl:2,aa:1\n","Zdrop                               \t40\n","Threads                             \t4\n","Compressed                          \t0\n","Verbosity                           \t3\n","Seed substitution matrix            \tnucl:nucleotide.out,aa:VTML80.out\n","Sensitivity                         \t7\n","k-mer length                        \t0\n","k-score                             \t2147483647\n","Alphabet size                       \tnucl:5,aa:21\n","Max results per query               \t300\n","Split database                      \t0\n","Split mode                          \t2\n","Split memory limit                  \t0\n","Diagonal scoring                    \ttrue\n","Exact k-mer matching                \t0\n","Mask residues                       \t1\n","Mask lower case residues            \t0\n","Minimum diagonal score              \t15\n","Spaced k-mers                       \t1\n","Spaced k-mer pattern                \t\n","Local temporary path                \t\n","Rescore mode                        \t0\n","Remove hits by seq. id. and coverage\tfalse\n","Sort results                        \t0\n","Mask profile                        \t1\n","Profile e-value threshold           \t0.1\n","Global sequence weighting           \tfalse\n","Allow deletions                     \tfalse\n","Filter MSA                          \t1\n","Maximum seq. id. threshold          \t0.9\n","Minimum seq. id.                    \t0\n","Minimum score per column            \t-20\n","Minimum coverage                    \t0\n","Select N most diverse seqs          \t1000\n","Omit consensus                      \tfalse\n","Min codons in orf                   \t30\n","Max codons in length                \t32734\n","Max orf gaps                        \t2147483647\n","Contig start mode                   \t2\n","Contig end mode                     \t2\n","Orf start mode                      \t1\n","Forward frames                      \t1,2,3\n","Reverse frames                      \t1,2,3\n","Translation table                   \t1\n","Translate orf                       \t0\n","Use all table starts                \tfalse\n","Offset of numeric ids               \t0\n","Create lookup                       \t0\n","Add orf stop                        \tfalse\n","Chain overlapping alignments        \t0\n","Merge query                         \t1\n","Search type                         \t0\n","Search iterations                   \t1\n","Start sensitivity                   \t4\n","Search steps                        \t1\n","Slice search mode                   \tfalse\n","Strand selection                    \t1\n","Disk space limit                    \t0\n","MPI runner                          \t\n","Force restart with latest tmp       \tfalse\n","Remove temporary files              \tfalse\n","\n","prefilter /content/mmseqs2/tmp/12/candidateDB_35 /content/mmseqs2/genexa/genexaDB.idx /content/mmseqs2/tmp/12/15186463245495659712/pref_0 --sub-mat nucl:nucleotide.out,aa:blosum62.out --seed-sub-mat nucl:nucleotide.out,aa:VTML80.out -k 0 --k-score 2147483647 --alph-size nucl:5,aa:21 --max-seq-len 65535 --max-seqs 300 --split 0 --split-mode 2 --split-memory-limit 0 -c 0 --cov-mode 0 --comp-bias-corr 1 --diag-score 1 --exact-kmer-matching 0 --mask 1 --mask-lower-case 0 --min-ungapped-score 15 --add-self-matches 0 --spaced-kmer-mode 1 --db-load-mode 0 --pca 1 --pcb 1.5 --threads 4 --compressed 0 -v 3 -s 7.0 \n","\n","Index version: 16\n","Generated by:  16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","ScoreMatrix:  VTML80.out\n","Query database size: 3933 type: Aminoacid\n","Estimated memory consumption: 2G\n","Target database size: 468351 type: Aminoacid\n","Process prefiltering step 1 of 1\n","\n","k-mer similarity threshold: 100\n","Starting prefiltering scores calculation (step 1 of 1)\n","Query db start 1 to 3933\n","Target db start 1 to 468351\n","[=================================================================] 100.00% 3.93K 27s 925ms\n","\n","984.262236 k-mers per position\n","548252 DB matches per sequence\n","145 overflows\n","0 queries produce too many hits (truncated result)\n","275 sequences passed prefiltering per query sequence\n","300 median result list length\n","9 sequences with 0 size result lists\n","Time for merging to pref_0: 0h 0m 0s 1ms\n","Time for processing: 0h 0m 29s 387ms\n","align /content/mmseqs2/tmp/12/candidateDB_35 /content/mmseqs2/genexa/genexaDB.idx /content/mmseqs2/tmp/12/15186463245495659712/pref_0 /content/mmseqs2/tmp/12/resultsDB_35 --sub-mat nucl:nucleotide.out,aa:blosum62.out -a 0 --alignment-mode 2 --wrapped-scoring 0 -e 0.001 --min-seq-id 0 --min-aln-len 0 --seq-id-mode 0 --alt-ali 0 -c 0 --cov-mode 0 --max-seq-len 65535 --comp-bias-corr 1 --realign 0 --max-rejected 2147483647 --max-accept 2147483647 --add-self-matches 0 --db-load-mode 0 --pca 1 --pcb 1.5 --score-bias 0 --gap-open nucl:5,aa:11 --gap-extend nucl:2,aa:1 --zdrop 40 --threads 4 --compressed 0 -v 3 \n","\n","Index version: 16\n","Generated by:  16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","ScoreMatrix:  VTML80.out\n","Compute score and coverage\n","Query database size: 3933 type: Aminoacid\n","Target database size: 468351 type: Aminoacid\n","Calculation of alignments\n","[=================================================================] 100.00% 3.93K 23s 118ms\n","Time for merging to resultsDB_35: 0h 0m 0s 1ms\n","\n","1083363 alignments calculated.\n","262802 sequence pairs passed the thresholds (0.242580 of overall calculated).\n","66.819733 hits per query sequence.\n","Time for processing: 0h 0m 23s 335ms\n","convertalis /content/mmseqs2/tmp/12/candidateDB_35 /content/mmseqs2/genexa/genexaDB /content/mmseqs2/tmp/12/resultsDB_35 /content/mmseqs2/tmp/12/resultDB_genexa.m8 --format-output query,target,evalue,raw \n","\n","MMseqs Version:        \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Substitution matrix    \tnucl:nucleotide.out,aa:blosum62.out\n","Alignment format       \t0\n","Format alignment output\tquery,target,evalue,raw\n","Translation table      \t1\n","Gap open cost          \tnucl:5,aa:11\n","Gap extension cost     \tnucl:2,aa:1\n","Database output        \tfalse\n","Preload mode           \t0\n","Search type            \t0\n","Threads                \t4\n","Compressed             \t0\n","Verbosity              \t3\n","\n","[=================================================================] 100.00% 3.93K 0s 200ms\n","Time for merging to resultDB_genexa.m8: 0h 0m 0s 7ms\n","Time for processing: 0h 0m 0s 277ms\n","search -s 7.0 /content/mmseqs2/tmp/12/candidateDB_53 /content/mmseqs2/swissprot/swissprotDB /content/mmseqs2/tmp/12/resultsDB_53 /content/mmseqs2/tmp/12 \n","\n","MMseqs Version:                     \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Substitution matrix                 \tnucl:nucleotide.out,aa:blosum62.out\n","Add backtrace                       \tfalse\n","Alignment mode                      \t2\n","Allow wrapped scoring               \tfalse\n","E-value threshold                   \t0.001\n","Seq. id. threshold                  \t0\n","Min alignment length                \t0\n","Seq. id. mode                       \t0\n","Alternative alignments              \t0\n","Coverage threshold                  \t0\n","Coverage mode                       \t0\n","Max sequence length                 \t65535\n","Compositional bias                  \t1\n","Realign hits                        \tfalse\n","Max reject                          \t2147483647\n","Max accept                          \t2147483647\n","Include identical seq. id.          \tfalse\n","Preload mode                        \t0\n","Pseudo count a                      \t1\n","Pseudo count b                      \t1.5\n","Score bias                          \t0\n","Gap open cost                       \tnucl:5,aa:11\n","Gap extension cost                  \tnucl:2,aa:1\n","Zdrop                               \t40\n","Threads                             \t4\n","Compressed                          \t0\n","Verbosity                           \t3\n","Seed substitution matrix            \tnucl:nucleotide.out,aa:VTML80.out\n","Sensitivity                         \t7\n","k-mer length                        \t0\n","k-score                             \t2147483647\n","Alphabet size                       \tnucl:5,aa:21\n","Max results per query               \t300\n","Split database                      \t0\n","Split mode                          \t2\n","Split memory limit                  \t0\n","Diagonal scoring                    \ttrue\n","Exact k-mer matching                \t0\n","Mask residues                       \t1\n","Mask lower case residues            \t0\n","Minimum diagonal score              \t15\n","Spaced k-mers                       \t1\n","Spaced k-mer pattern                \t\n","Local temporary path                \t\n","Rescore mode                        \t0\n","Remove hits by seq. id. and coverage\tfalse\n","Sort results                        \t0\n","Mask profile                        \t1\n","Profile e-value threshold           \t0.1\n","Global sequence weighting           \tfalse\n","Allow deletions                     \tfalse\n","Filter MSA                          \t1\n","Maximum seq. id. threshold          \t0.9\n","Minimum seq. id.                    \t0\n","Minimum score per column            \t-20\n","Minimum coverage                    \t0\n","Select N most diverse seqs          \t1000\n","Omit consensus                      \tfalse\n","Min codons in orf                   \t30\n","Max codons in length                \t32734\n","Max orf gaps                        \t2147483647\n","Contig start mode                   \t2\n","Contig end mode                     \t2\n","Orf start mode                      \t1\n","Forward frames                      \t1,2,3\n","Reverse frames                      \t1,2,3\n","Translation table                   \t1\n","Translate orf                       \t0\n","Use all table starts                \tfalse\n","Offset of numeric ids               \t0\n","Create lookup                       \t0\n","Add orf stop                        \tfalse\n","Chain overlapping alignments        \t0\n","Merge query                         \t1\n","Search type                         \t0\n","Search iterations                   \t1\n","Start sensitivity                   \t4\n","Search steps                        \t1\n","Slice search mode                   \tfalse\n","Strand selection                    \t1\n","Disk space limit                    \t0\n","MPI runner                          \t\n","Force restart with latest tmp       \tfalse\n","Remove temporary files              \tfalse\n","\n","prefilter /content/mmseqs2/tmp/12/candidateDB_53 /content/mmseqs2/swissprot/swissprotDB /content/mmseqs2/tmp/12/4414953177667033950/pref_0 --sub-mat nucl:nucleotide.out,aa:blosum62.out --seed-sub-mat nucl:nucleotide.out,aa:VTML80.out -k 0 --k-score 2147483647 --alph-size nucl:5,aa:21 --max-seq-len 65535 --max-seqs 300 --split 0 --split-mode 2 --split-memory-limit 0 -c 0 --cov-mode 0 --comp-bias-corr 1 --diag-score 1 --exact-kmer-matching 0 --mask 1 --mask-lower-case 0 --min-ungapped-score 15 --add-self-matches 0 --spaced-kmer-mode 1 --db-load-mode 0 --pca 1 --pcb 1.5 --threads 4 --compressed 0 -v 3 -s 7.0 \n","\n","Query database size: 3933 type: Aminoacid\n","Estimated memory consumption: 2G\n","Target database size: 562755 type: Aminoacid\n","Index table k-mer threshold: 100 at k-mer size 6 \n","Index table: counting k-mers\n","[=================================================================] 100.00% 562.76K 24s 733ms\n","Index table: Masked residues: 3164874\n","Index table: fill\n","[=================================================================] 100.00% 562.76K 17s 823ms\n","Index statistics\n","Entries:          192757622\n","DB size:          1591 MB\n","Avg k-mer size:   3.011838\n","Top 10 k-mers\n","    GPGGTL\t1835\n","    GQSWTV\t1706\n","    WGMFAT\t1638\n","    PGVFEV\t1634\n","    VLWQFW\t1623\n","    AYIRPN\t1587\n","    RSPKGV\t1585\n","    TPHKWY\t1560\n","    HIRVGW\t1548\n","    ITLSPY\t1541\n","Time for index table init: 0h 0m 44s 661ms\n","Process prefiltering step 1 of 1\n","\n","k-mer similarity threshold: 100\n","Starting prefiltering scores calculation (step 1 of 1)\n","Query db start 1 to 3933\n","Target db start 1 to 562755\n","[=================================================================] 100.00% 3.93K 31s 733ms\n","\n","985.144879 k-mers per position\n","661101 DB matches per sequence\n","232 overflows\n","0 queries produce too many hits (truncated result)\n","284 sequences passed prefiltering per query sequence\n","300 median result list length\n","8 sequences with 0 size result lists\n","Time for merging to pref_0: 0h 0m 0s 1ms\n","Time for processing: 0h 1m 17s 570ms\n","align /content/mmseqs2/tmp/12/candidateDB_53 /content/mmseqs2/swissprot/swissprotDB /content/mmseqs2/tmp/12/4414953177667033950/pref_0 /content/mmseqs2/tmp/12/resultsDB_53 --sub-mat nucl:nucleotide.out,aa:blosum62.out -a 0 --alignment-mode 2 --wrapped-scoring 0 -e 0.001 --min-seq-id 0 --min-aln-len 0 --seq-id-mode 0 --alt-ali 0 -c 0 --cov-mode 0 --max-seq-len 65535 --comp-bias-corr 1 --realign 0 --max-rejected 2147483647 --max-accept 2147483647 --add-self-matches 0 --db-load-mode 0 --pca 1 --pcb 1.5 --score-bias 0 --gap-open nucl:5,aa:11 --gap-extend nucl:2,aa:1 --zdrop 40 --threads 4 --compressed 0 -v 3 \n","\n","Compute score and coverage\n","Query database size: 3933 type: Aminoacid\n","Target database size: 562755 type: Aminoacid\n","Calculation of alignments\n","[=================================================================] 100.00% 3.93K 24s 388ms\n","Time for merging to resultsDB_53: 0h 0m 0s 1ms\n","\n","1118936 alignments calculated.\n","217218 sequence pairs passed the thresholds (0.194129 of overall calculated).\n","55.229595 hits per query sequence.\n","Time for processing: 0h 0m 24s 551ms\n","convertalis /content/mmseqs2/tmp/12/candidateDB_53 /content/mmseqs2/swissprot/swissprotDB /content/mmseqs2/tmp/12/resultsDB_53 /content/mmseqs2/tmp/12/resultDB_secondary.m8 --format-output query,target,evalue,raw \n","\n","MMseqs Version:        \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Substitution matrix    \tnucl:nucleotide.out,aa:blosum62.out\n","Alignment format       \t0\n","Format alignment output\tquery,target,evalue,raw\n","Translation table      \t1\n","Gap open cost          \tnucl:5,aa:11\n","Gap extension cost     \tnucl:2,aa:1\n","Database output        \tfalse\n","Preload mode           \t0\n","Search type            \t0\n","Threads                \t4\n","Compressed             \t0\n","Verbosity              \t3\n","\n","[=================================================================] 100.00% 3.93K 0s 153ms\n","Time for merging to resultDB_secondary.m8: 0h 0m 0s 7ms\n","Time for processing: 0h 0m 0s 250ms\n","found 2332 genes\n","\n","\n","Reading fasta file GCF_002009335.2_ASM200933v2_genomic.fna.gz...\n","\n","Finding and translating open reading frames...\n","\n","Encoding amino acids...\n","\n","Applying kmer prefilter...\n","\n","Scoring ORFs with temporal convolutional network...\n","\n"],"name":"stdout"},{"output_type":"display_data","data":{"application/vnd.jupyter.widget-view+json":{"model_id":"61764a2ad59c421c821478b2ac42f19b","version_minor":0,"version_major":2},"text/plain":["HBox(children=(FloatProgress(value=0.0, max=83.0), HTML(value='')))"]},"metadata":{"tags":[]}},{"output_type":"stream","text":["\n","Scoring translation initiation sites...\n","\n"],"name":"stdout"},{"output_type":"display_data","data":{"application/vnd.jupyter.widget-view+json":{"model_id":"53e48adaac844b999781e04c08ecbda6","version_minor":0,"version_major":2},"text/plain":["HBox(children=(FloatProgress(value=0.0, max=25.0), HTML(value='')))"]},"metadata":{"tags":[]}},{"output_type":"stream","text":["\n","Creating weighted directed acyclic graph of all ORFs and subORFs on contig 0...\n","\n"],"name":"stdout"},{"output_type":"display_data","data":{"application/vnd.jupyter.widget-view+json":{"model_id":"ae9787fe5de34d368d18fd66b8140f23","version_minor":0,"version_major":2},"text/plain":["HBox(children=(FloatProgress(value=0.0, max=248007.0), HTML(value='')))"]},"metadata":{"tags":[]}},{"output_type":"stream","text":["\n","Maximizing geneiness...\n","\n","Filtering predictions with mmseqs2...\n","createdb /content/mmseqs2/tmp/13/candidate_genes_35.fasta /content/mmseqs2/tmp/13/candidateDB_35 \n","\n","MMseqs Version:       \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Database type         \t0\n","Shuffle input database\ttrue\n","Createdb mode         \t0\n","Write lookup file     \t1\n","Offset of numeric ids \t0\n","Compressed            \t0\n","Verbosity             \t3\n","\n","Converting sequences\n","[7273] 0s 8ms\n","Time for merging to candidateDB_35_h: 0h 0m 0s 5ms\n","Time for merging to candidateDB_35: 0h 0m 0s 5ms\n","Database type: Aminoacid\n","Time for processing: 0h 0m 0s 27ms\n","createdb /content/mmseqs2/tmp/13/candidate_genes_53.fasta /content/mmseqs2/tmp/13/candidateDB_53 \n","\n","MMseqs Version:       \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Database type         \t0\n","Shuffle input database\ttrue\n","Createdb mode         \t0\n","Write lookup file     \t1\n","Offset of numeric ids \t0\n","Compressed            \t0\n","Verbosity             \t3\n","\n","Converting sequences\n","[7273] 0s 10ms\n","Time for merging to candidateDB_53_h: 0h 0m 0s 4ms\n","Time for merging to candidateDB_53: 0h 0m 0s 5ms\n","Database type: Aminoacid\n","Time for processing: 0h 0m 0s 28ms\n","search -s 7.0 /content/mmseqs2/tmp/13/candidateDB_35 /content/mmseqs2/genexa/genexaDB /content/mmseqs2/tmp/13/resultsDB_35 /content/mmseqs2/tmp/13 \n","\n","MMseqs Version:                     \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Substitution matrix                 \tnucl:nucleotide.out,aa:blosum62.out\n","Add backtrace                       \tfalse\n","Alignment mode                      \t2\n","Allow wrapped scoring               \tfalse\n","E-value threshold                   \t0.001\n","Seq. id. threshold                  \t0\n","Min alignment length                \t0\n","Seq. id. mode                       \t0\n","Alternative alignments              \t0\n","Coverage threshold                  \t0\n","Coverage mode                       \t0\n","Max sequence length                 \t65535\n","Compositional bias                  \t1\n","Realign hits                        \tfalse\n","Max reject                          \t2147483647\n","Max accept                          \t2147483647\n","Include identical seq. id.          \tfalse\n","Preload mode                        \t0\n","Pseudo count a                      \t1\n","Pseudo count b                      \t1.5\n","Score bias                          \t0\n","Gap open cost                       \tnucl:5,aa:11\n","Gap extension cost                  \tnucl:2,aa:1\n","Zdrop                               \t40\n","Threads                             \t4\n","Compressed                          \t0\n","Verbosity                           \t3\n","Seed substitution matrix            \tnucl:nucleotide.out,aa:VTML80.out\n","Sensitivity                         \t7\n","k-mer length                        \t0\n","k-score                             \t2147483647\n","Alphabet size                       \tnucl:5,aa:21\n","Max results per query               \t300\n","Split database                      \t0\n","Split mode                          \t2\n","Split memory limit                  \t0\n","Diagonal scoring                    \ttrue\n","Exact k-mer matching                \t0\n","Mask residues                       \t1\n","Mask lower case residues            \t0\n","Minimum diagonal score              \t15\n","Spaced k-mers                       \t1\n","Spaced k-mer pattern                \t\n","Local temporary path                \t\n","Rescore mode                        \t0\n","Remove hits by seq. id. and coverage\tfalse\n","Sort results                        \t0\n","Mask profile                        \t1\n","Profile e-value threshold           \t0.1\n","Global sequence weighting           \tfalse\n","Allow deletions                     \tfalse\n","Filter MSA                          \t1\n","Maximum seq. id. threshold          \t0.9\n","Minimum seq. id.                    \t0\n","Minimum score per column            \t-20\n","Minimum coverage                    \t0\n","Select N most diverse seqs          \t1000\n","Omit consensus                      \tfalse\n","Min codons in orf                   \t30\n","Max codons in length                \t32734\n","Max orf gaps                        \t2147483647\n","Contig start mode                   \t2\n","Contig end mode                     \t2\n","Orf start mode                      \t1\n","Forward frames                      \t1,2,3\n","Reverse frames                      \t1,2,3\n","Translation table                   \t1\n","Translate orf                       \t0\n","Use all table starts                \tfalse\n","Offset of numeric ids               \t0\n","Create lookup                       \t0\n","Add orf stop                        \tfalse\n","Chain overlapping alignments        \t0\n","Merge query                         \t1\n","Search type                         \t0\n","Search iterations                   \t1\n","Start sensitivity                   \t4\n","Search steps                        \t1\n","Slice search mode                   \tfalse\n","Strand selection                    \t1\n","Disk space limit                    \t0\n","MPI runner                          \t\n","Force restart with latest tmp       \tfalse\n","Remove temporary files              \tfalse\n","\n","prefilter /content/mmseqs2/tmp/13/candidateDB_35 /content/mmseqs2/genexa/genexaDB.idx /content/mmseqs2/tmp/13/9926177142385964223/pref_0 --sub-mat nucl:nucleotide.out,aa:blosum62.out --seed-sub-mat nucl:nucleotide.out,aa:VTML80.out -k 0 --k-score 2147483647 --alph-size nucl:5,aa:21 --max-seq-len 65535 --max-seqs 300 --split 0 --split-mode 2 --split-memory-limit 0 -c 0 --cov-mode 0 --comp-bias-corr 1 --diag-score 1 --exact-kmer-matching 0 --mask 1 --mask-lower-case 0 --min-ungapped-score 15 --add-self-matches 0 --spaced-kmer-mode 1 --db-load-mode 0 --pca 1 --pcb 1.5 --threads 4 --compressed 0 -v 3 -s 7.0 \n","\n","Index version: 16\n","Generated by:  16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","ScoreMatrix:  VTML80.out\n","Query database size: 7285 type: Aminoacid\n","Estimated memory consumption: 2G\n","Target database size: 468351 type: Aminoacid\n","Process prefiltering step 1 of 1\n","\n","k-mer similarity threshold: 100\n","Starting prefiltering scores calculation (step 1 of 1)\n","Query db start 1 to 7285\n","Target db start 1 to 468351\n","[=================================================================] 100.00% 7.28K 54s 727ms\n","\n","1008.792357 k-mers per position\n","587299 DB matches per sequence\n","399 overflows\n","0 queries produce too many hits (truncated result)\n","277 sequences passed prefiltering per query sequence\n","300 median result list length\n","1 sequences with 0 size result lists\n","Time for merging to pref_0: 0h 0m 0s 2ms\n","Time for processing: 0h 0m 56s 169ms\n","align /content/mmseqs2/tmp/13/candidateDB_35 /content/mmseqs2/genexa/genexaDB.idx /content/mmseqs2/tmp/13/9926177142385964223/pref_0 /content/mmseqs2/tmp/13/resultsDB_35 --sub-mat nucl:nucleotide.out,aa:blosum62.out -a 0 --alignment-mode 2 --wrapped-scoring 0 -e 0.001 --min-seq-id 0 --min-aln-len 0 --seq-id-mode 0 --alt-ali 0 -c 0 --cov-mode 0 --max-seq-len 65535 --comp-bias-corr 1 --realign 0 --max-rejected 2147483647 --max-accept 2147483647 --add-self-matches 0 --db-load-mode 0 --pca 1 --pcb 1.5 --score-bias 0 --gap-open nucl:5,aa:11 --gap-extend nucl:2,aa:1 --zdrop 40 --threads 4 --compressed 0 -v 3 \n","\n","Index version: 16\n","Generated by:  16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","ScoreMatrix:  VTML80.out\n","Compute score and coverage\n","Query database size: 7285 type: Aminoacid\n","Target database size: 468351 type: Aminoacid\n","Calculation of alignments\n","[=================================================================] 100.00% 7.28K 42s 407ms\n","Time for merging to resultsDB_35: 0h 0m 0s 2ms\n","\n","2018366 alignments calculated.\n","426992 sequence pairs passed the thresholds (0.211553 of overall calculated).\n","58.612492 hits per query sequence.\n","Time for processing: 0h 0m 42s 494ms\n","convertalis /content/mmseqs2/tmp/13/candidateDB_35 /content/mmseqs2/genexa/genexaDB /content/mmseqs2/tmp/13/resultsDB_35 /content/mmseqs2/tmp/13/resultDB_genexa.m8 --format-output query,target,evalue,raw \n","\n","MMseqs Version:        \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Substitution matrix    \tnucl:nucleotide.out,aa:blosum62.out\n","Alignment format       \t0\n","Format alignment output\tquery,target,evalue,raw\n","Translation table      \t1\n","Gap open cost          \tnucl:5,aa:11\n","Gap extension cost     \tnucl:2,aa:1\n","Database output        \tfalse\n","Preload mode           \t0\n","Search type            \t0\n","Threads                \t4\n","Compressed             \t0\n","Verbosity              \t3\n","\n","[=================================================================] 100.00% 7.28K 0s 315ms\n","Time for merging to resultDB_genexa.m8: 0h 0m 0s 12ms\n","Time for processing: 0h 0m 0s 401ms\n","search -s 7.0 /content/mmseqs2/tmp/13/candidateDB_53 /content/mmseqs2/swissprot/swissprotDB /content/mmseqs2/tmp/13/resultsDB_53 /content/mmseqs2/tmp/13 \n","\n","MMseqs Version:                     \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Substitution matrix                 \tnucl:nucleotide.out,aa:blosum62.out\n","Add backtrace                       \tfalse\n","Alignment mode                      \t2\n","Allow wrapped scoring               \tfalse\n","E-value threshold                   \t0.001\n","Seq. id. threshold                  \t0\n","Min alignment length                \t0\n","Seq. id. mode                       \t0\n","Alternative alignments              \t0\n","Coverage threshold                  \t0\n","Coverage mode                       \t0\n","Max sequence length                 \t65535\n","Compositional bias                  \t1\n","Realign hits                        \tfalse\n","Max reject                          \t2147483647\n","Max accept                          \t2147483647\n","Include identical seq. id.          \tfalse\n","Preload mode                        \t0\n","Pseudo count a                      \t1\n","Pseudo count b                      \t1.5\n","Score bias                          \t0\n","Gap open cost                       \tnucl:5,aa:11\n","Gap extension cost                  \tnucl:2,aa:1\n","Zdrop                               \t40\n","Threads                             \t4\n","Compressed                          \t0\n","Verbosity                           \t3\n","Seed substitution matrix            \tnucl:nucleotide.out,aa:VTML80.out\n","Sensitivity                         \t7\n","k-mer length                        \t0\n","k-score                             \t2147483647\n","Alphabet size                       \tnucl:5,aa:21\n","Max results per query               \t300\n","Split database                      \t0\n","Split mode                          \t2\n","Split memory limit                  \t0\n","Diagonal scoring                    \ttrue\n","Exact k-mer matching                \t0\n","Mask residues                       \t1\n","Mask lower case residues            \t0\n","Minimum diagonal score              \t15\n","Spaced k-mers                       \t1\n","Spaced k-mer pattern                \t\n","Local temporary path                \t\n","Rescore mode                        \t0\n","Remove hits by seq. id. and coverage\tfalse\n","Sort results                        \t0\n","Mask profile                        \t1\n","Profile e-value threshold           \t0.1\n","Global sequence weighting           \tfalse\n","Allow deletions                     \tfalse\n","Filter MSA                          \t1\n","Maximum seq. id. threshold          \t0.9\n","Minimum seq. id.                    \t0\n","Minimum score per column            \t-20\n","Minimum coverage                    \t0\n","Select N most diverse seqs          \t1000\n","Omit consensus                      \tfalse\n","Min codons in orf                   \t30\n","Max codons in length                \t32734\n","Max orf gaps                        \t2147483647\n","Contig start mode                   \t2\n","Contig end mode                     \t2\n","Orf start mode                      \t1\n","Forward frames                      \t1,2,3\n","Reverse frames                      \t1,2,3\n","Translation table                   \t1\n","Translate orf                       \t0\n","Use all table starts                \tfalse\n","Offset of numeric ids               \t0\n","Create lookup                       \t0\n","Add orf stop                        \tfalse\n","Chain overlapping alignments        \t0\n","Merge query                         \t1\n","Search type                         \t0\n","Search iterations                   \t1\n","Start sensitivity                   \t4\n","Search steps                        \t1\n","Slice search mode                   \tfalse\n","Strand selection                    \t1\n","Disk space limit                    \t0\n","MPI runner                          \t\n","Force restart with latest tmp       \tfalse\n","Remove temporary files              \tfalse\n","\n","prefilter /content/mmseqs2/tmp/13/candidateDB_53 /content/mmseqs2/swissprot/swissprotDB /content/mmseqs2/tmp/13/4516318165712379933/pref_0 --sub-mat nucl:nucleotide.out,aa:blosum62.out --seed-sub-mat nucl:nucleotide.out,aa:VTML80.out -k 0 --k-score 2147483647 --alph-size nucl:5,aa:21 --max-seq-len 65535 --max-seqs 300 --split 0 --split-mode 2 --split-memory-limit 0 -c 0 --cov-mode 0 --comp-bias-corr 1 --diag-score 1 --exact-kmer-matching 0 --mask 1 --mask-lower-case 0 --min-ungapped-score 15 --add-self-matches 0 --spaced-kmer-mode 1 --db-load-mode 0 --pca 1 --pcb 1.5 --threads 4 --compressed 0 -v 3 -s 7.0 \n","\n","Query database size: 7285 type: Aminoacid\n","Estimated memory consumption: 2G\n","Target database size: 562755 type: Aminoacid\n","Index table k-mer threshold: 100 at k-mer size 6 \n","Index table: counting k-mers\n","[=================================================================] 100.00% 562.76K 24s 437ms\n","Index table: Masked residues: 3164874\n","Index table: fill\n","[=================================================================] 100.00% 562.76K 17s 457ms\n","Index statistics\n","Entries:          192757622\n","DB size:          1591 MB\n","Avg k-mer size:   3.011838\n","Top 10 k-mers\n","    GPGGTL\t1835\n","    GQSWTV\t1706\n","    WGMFAT\t1638\n","    PGVFEV\t1634\n","    VLWQFW\t1623\n","    AYIRPN\t1587\n","    RSPKGV\t1585\n","    TPHKWY\t1560\n","    HIRVGW\t1548\n","    ITLSPY\t1541\n","Time for index table init: 0h 0m 43s 956ms\n","Process prefiltering step 1 of 1\n","\n","k-mer similarity threshold: 100\n","Starting prefiltering scores calculation (step 1 of 1)\n","Query db start 1 to 7285\n","Target db start 1 to 562755\n","[=================================================================] 100.00% 7.28K 1m 0s 651ms\n","\n","1009.606974 k-mers per position\n","697483 DB matches per sequence\n","628 overflows\n","0 queries produce too many hits (truncated result)\n","284 sequences passed prefiltering per query sequence\n","300 median result list length\n","5 sequences with 0 size result lists\n","Time for merging to pref_0: 0h 0m 0s 2ms\n","Time for processing: 0h 1m 45s 759ms\n","align /content/mmseqs2/tmp/13/candidateDB_53 /content/mmseqs2/swissprot/swissprotDB /content/mmseqs2/tmp/13/4516318165712379933/pref_0 /content/mmseqs2/tmp/13/resultsDB_53 --sub-mat nucl:nucleotide.out,aa:blosum62.out -a 0 --alignment-mode 2 --wrapped-scoring 0 -e 0.001 --min-seq-id 0 --min-aln-len 0 --seq-id-mode 0 --alt-ali 0 -c 0 --cov-mode 0 --max-seq-len 65535 --comp-bias-corr 1 --realign 0 --max-rejected 2147483647 --max-accept 2147483647 --add-self-matches 0 --db-load-mode 0 --pca 1 --pcb 1.5 --score-bias 0 --gap-open nucl:5,aa:11 --gap-extend nucl:2,aa:1 --zdrop 40 --threads 4 --compressed 0 -v 3 \n","\n","Compute score and coverage\n","Query database size: 7285 type: Aminoacid\n","Target database size: 562755 type: Aminoacid\n","Calculation of alignments\n","[=================================================================] 100.00% 7.28K 40s 111ms\n","Time for merging to resultsDB_53: 0h 0m 0s 2ms\n","\n","2071378 alignments calculated.\n","316708 sequence pairs passed the thresholds (0.152897 of overall calculated).\n","43.473988 hits per query sequence.\n","Time for processing: 0h 0m 40s 176ms\n","convertalis /content/mmseqs2/tmp/13/candidateDB_53 /content/mmseqs2/swissprot/swissprotDB /content/mmseqs2/tmp/13/resultsDB_53 /content/mmseqs2/tmp/13/resultDB_secondary.m8 --format-output query,target,evalue,raw \n","\n","MMseqs Version:        \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Substitution matrix    \tnucl:nucleotide.out,aa:blosum62.out\n","Alignment format       \t0\n","Format alignment output\tquery,target,evalue,raw\n","Translation table      \t1\n","Gap open cost          \tnucl:5,aa:11\n","Gap extension cost     \tnucl:2,aa:1\n","Database output        \tfalse\n","Preload mode           \t0\n","Search type            \t0\n","Threads                \t4\n","Compressed             \t0\n","Verbosity              \t3\n","\n","[=================================================================] 100.00% 7.28K 0s 218ms\n","Time for merging to resultDB_secondary.m8: 0h 0m 0s 9ms\n","Time for processing: 0h 0m 0s 318ms\n","found 3772 genes\n","\n","\n","Reading fasta file GCF_002005165.1_ASM200516v1_genomic.fna.gz...\n","\n","Finding and translating open reading frames...\n","\n","Encoding amino acids...\n","\n","Applying kmer prefilter...\n","\n","Scoring ORFs with temporal convolutional network...\n","\n"],"name":"stdout"},{"output_type":"display_data","data":{"application/vnd.jupyter.widget-view+json":{"model_id":"e00e98f3550144f3aad0b161323907a0","version_minor":0,"version_major":2},"text/plain":["HBox(children=(FloatProgress(value=0.0, max=70.0), HTML(value='')))"]},"metadata":{"tags":[]}},{"output_type":"stream","text":["\n","Scoring translation initiation sites...\n","\n"],"name":"stdout"},{"output_type":"display_data","data":{"application/vnd.jupyter.widget-view+json":{"model_id":"0991db3a6cf442acb396adda7cf1d9e7","version_minor":0,"version_major":2},"text/plain":["HBox(children=(FloatProgress(value=0.0, max=21.0), HTML(value='')))"]},"metadata":{"tags":[]}},{"output_type":"stream","text":["\n","Creating weighted directed acyclic graph of all ORFs and subORFs on contig 0...\n","\n"],"name":"stdout"},{"output_type":"display_data","data":{"application/vnd.jupyter.widget-view+json":{"model_id":"0eda738208ca4abfad9bdf7c7242e420","version_minor":0,"version_major":2},"text/plain":["HBox(children=(FloatProgress(value=0.0, max=209623.0), HTML(value='')))"]},"metadata":{"tags":[]}},{"output_type":"stream","text":["\n","Maximizing geneiness...\n","\n","Filtering predictions with mmseqs2...\n","createdb /content/mmseqs2/tmp/14/candidate_genes_35.fasta /content/mmseqs2/tmp/14/candidateDB_35 \n","\n","MMseqs Version:       \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Database type         \t0\n","Shuffle input database\ttrue\n","Createdb mode         \t0\n","Write lookup file     \t1\n","Offset of numeric ids \t0\n","Compressed            \t0\n","Verbosity             \t3\n","\n","Converting sequences\n","[6263] 0s 7ms\n","Time for merging to candidateDB_35_h: 0h 0m 0s 4ms\n","Time for merging to candidateDB_35: 0h 0m 0s 5ms\n","Database type: Aminoacid\n","Time for processing: 0h 0m 0s 25ms\n","createdb /content/mmseqs2/tmp/14/candidate_genes_53.fasta /content/mmseqs2/tmp/14/candidateDB_53 \n","\n","MMseqs Version:       \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Database type         \t0\n","Shuffle input database\ttrue\n","Createdb mode         \t0\n","Write lookup file     \t1\n","Offset of numeric ids \t0\n","Compressed            \t0\n","Verbosity             \t3\n","\n","Converting sequences\n","[6263] 0s 7ms\n","Time for merging to candidateDB_53_h: 0h 0m 0s 3ms\n","Time for merging to candidateDB_53: 0h 0m 0s 5ms\n","Database type: Aminoacid\n","Time for processing: 0h 0m 0s 24ms\n","search -s 7.0 /content/mmseqs2/tmp/14/candidateDB_35 /content/mmseqs2/genexa/genexaDB /content/mmseqs2/tmp/14/resultsDB_35 /content/mmseqs2/tmp/14 \n","\n","MMseqs Version:                     \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Substitution matrix                 \tnucl:nucleotide.out,aa:blosum62.out\n","Add backtrace                       \tfalse\n","Alignment mode                      \t2\n","Allow wrapped scoring               \tfalse\n","E-value threshold                   \t0.001\n","Seq. id. threshold                  \t0\n","Min alignment length                \t0\n","Seq. id. mode                       \t0\n","Alternative alignments              \t0\n","Coverage threshold                  \t0\n","Coverage mode                       \t0\n","Max sequence length                 \t65535\n","Compositional bias                  \t1\n","Realign hits                        \tfalse\n","Max reject                          \t2147483647\n","Max accept                          \t2147483647\n","Include identical seq. id.          \tfalse\n","Preload mode                        \t0\n","Pseudo count a                      \t1\n","Pseudo count b                      \t1.5\n","Score bias                          \t0\n","Gap open cost                       \tnucl:5,aa:11\n","Gap extension cost                  \tnucl:2,aa:1\n","Zdrop                               \t40\n","Threads                             \t4\n","Compressed                          \t0\n","Verbosity                           \t3\n","Seed substitution matrix            \tnucl:nucleotide.out,aa:VTML80.out\n","Sensitivity                         \t7\n","k-mer length                        \t0\n","k-score                             \t2147483647\n","Alphabet size                       \tnucl:5,aa:21\n","Max results per query               \t300\n","Split database                      \t0\n","Split mode                          \t2\n","Split memory limit                  \t0\n","Diagonal scoring                    \ttrue\n","Exact k-mer matching                \t0\n","Mask residues                       \t1\n","Mask lower case residues            \t0\n","Minimum diagonal score              \t15\n","Spaced k-mers                       \t1\n","Spaced k-mer pattern                \t\n","Local temporary path                \t\n","Rescore mode                        \t0\n","Remove hits by seq. id. and coverage\tfalse\n","Sort results                        \t0\n","Mask profile                        \t1\n","Profile e-value threshold           \t0.1\n","Global sequence weighting           \tfalse\n","Allow deletions                     \tfalse\n","Filter MSA                          \t1\n","Maximum seq. id. threshold          \t0.9\n","Minimum seq. id.                    \t0\n","Minimum score per column            \t-20\n","Minimum coverage                    \t0\n","Select N most diverse seqs          \t1000\n","Omit consensus                      \tfalse\n","Min codons in orf                   \t30\n","Max codons in length                \t32734\n","Max orf gaps                        \t2147483647\n","Contig start mode                   \t2\n","Contig end mode                     \t2\n","Orf start mode                      \t1\n","Forward frames                      \t1,2,3\n","Reverse frames                      \t1,2,3\n","Translation table                   \t1\n","Translate orf                       \t0\n","Use all table starts                \tfalse\n","Offset of numeric ids               \t0\n","Create lookup                       \t0\n","Add orf stop                        \tfalse\n","Chain overlapping alignments        \t0\n","Merge query                         \t1\n","Search type                         \t0\n","Search iterations                   \t1\n","Start sensitivity                   \t4\n","Search steps                        \t1\n","Slice search mode                   \tfalse\n","Strand selection                    \t1\n","Disk space limit                    \t0\n","MPI runner                          \t\n","Force restart with latest tmp       \tfalse\n","Remove temporary files              \tfalse\n","\n","prefilter /content/mmseqs2/tmp/14/candidateDB_35 /content/mmseqs2/genexa/genexaDB.idx /content/mmseqs2/tmp/14/4665891039276268734/pref_0 --sub-mat nucl:nucleotide.out,aa:blosum62.out --seed-sub-mat nucl:nucleotide.out,aa:VTML80.out -k 0 --k-score 2147483647 --alph-size nucl:5,aa:21 --max-seq-len 65535 --max-seqs 300 --split 0 --split-mode 2 --split-memory-limit 0 -c 0 --cov-mode 0 --comp-bias-corr 1 --diag-score 1 --exact-kmer-matching 0 --mask 1 --mask-lower-case 0 --min-ungapped-score 15 --add-self-matches 0 --spaced-kmer-mode 1 --db-load-mode 0 --pca 1 --pcb 1.5 --threads 4 --compressed 0 -v 3 -s 7.0 \n","\n","Index version: 16\n","Generated by:  16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","ScoreMatrix:  VTML80.out\n","Query database size: 6333 type: Aminoacid\n","Estimated memory consumption: 2G\n","Target database size: 468351 type: Aminoacid\n","Process prefiltering step 1 of 1\n","\n","k-mer similarity threshold: 100\n","Starting prefiltering scores calculation (step 1 of 1)\n","Query db start 1 to 6333\n","Target db start 1 to 468351\n","[=================================================================] 100.00% 6.33K 40s 291ms\n","\n","955.078283 k-mers per position\n","491434 DB matches per sequence\n","135 overflows\n","0 queries produce too many hits (truncated result)\n","274 sequences passed prefiltering per query sequence\n","300 median result list length\n","13 sequences with 0 size result lists\n","Time for merging to pref_0: 0h 0m 0s 1ms\n","Time for processing: 0h 0m 41s 731ms\n","align /content/mmseqs2/tmp/14/candidateDB_35 /content/mmseqs2/genexa/genexaDB.idx /content/mmseqs2/tmp/14/4665891039276268734/pref_0 /content/mmseqs2/tmp/14/resultsDB_35 --sub-mat nucl:nucleotide.out,aa:blosum62.out -a 0 --alignment-mode 2 --wrapped-scoring 0 -e 0.001 --min-seq-id 0 --min-aln-len 0 --seq-id-mode 0 --alt-ali 0 -c 0 --cov-mode 0 --max-seq-len 65535 --comp-bias-corr 1 --realign 0 --max-rejected 2147483647 --max-accept 2147483647 --add-self-matches 0 --db-load-mode 0 --pca 1 --pcb 1.5 --score-bias 0 --gap-open nucl:5,aa:11 --gap-extend nucl:2,aa:1 --zdrop 40 --threads 4 --compressed 0 -v 3 \n","\n","Index version: 16\n","Generated by:  16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","ScoreMatrix:  VTML80.out\n","Compute score and coverage\n","Query database size: 6333 type: Aminoacid\n","Target database size: 468351 type: Aminoacid\n","Calculation of alignments\n","[=================================================================] 100.00% 6.33K 36s 414ms\n","Time for merging to resultsDB_35: 0h 0m 0s 2ms\n","\n","1735790 alignments calculated.\n","449004 sequence pairs passed the thresholds (0.258674 of overall calculated).\n","70.899101 hits per query sequence.\n","Time for processing: 0h 0m 36s 509ms\n","convertalis /content/mmseqs2/tmp/14/candidateDB_35 /content/mmseqs2/genexa/genexaDB /content/mmseqs2/tmp/14/resultsDB_35 /content/mmseqs2/tmp/14/resultDB_genexa.m8 --format-output query,target,evalue,raw \n","\n","MMseqs Version:        \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Substitution matrix    \tnucl:nucleotide.out,aa:blosum62.out\n","Alignment format       \t0\n","Format alignment output\tquery,target,evalue,raw\n","Translation table      \t1\n","Gap open cost          \tnucl:5,aa:11\n","Gap extension cost     \tnucl:2,aa:1\n","Database output        \tfalse\n","Preload mode           \t0\n","Search type            \t0\n","Threads                \t4\n","Compressed             \t0\n","Verbosity              \t3\n","\n","[=================================================================] 100.00% 6.33K 0s 346ms\n","Time for merging to resultDB_genexa.m8: 0h 0m 0s 15ms\n","Time for processing: 0h 0m 0s 436ms\n","search -s 7.0 /content/mmseqs2/tmp/14/candidateDB_53 /content/mmseqs2/swissprot/swissprotDB /content/mmseqs2/tmp/14/resultsDB_53 /content/mmseqs2/tmp/14 \n","\n","MMseqs Version:                     \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Substitution matrix                 \tnucl:nucleotide.out,aa:blosum62.out\n","Add backtrace                       \tfalse\n","Alignment mode                      \t2\n","Allow wrapped scoring               \tfalse\n","E-value threshold                   \t0.001\n","Seq. id. threshold                  \t0\n","Min alignment length                \t0\n","Seq. id. mode                       \t0\n","Alternative alignments              \t0\n","Coverage threshold                  \t0\n","Coverage mode                       \t0\n","Max sequence length                 \t65535\n","Compositional bias                  \t1\n","Realign hits                        \tfalse\n","Max reject                          \t2147483647\n","Max accept                          \t2147483647\n","Include identical seq. id.          \tfalse\n","Preload mode                        \t0\n","Pseudo count a                      \t1\n","Pseudo count b                      \t1.5\n","Score bias                          \t0\n","Gap open cost                       \tnucl:5,aa:11\n","Gap extension cost                  \tnucl:2,aa:1\n","Zdrop                               \t40\n","Threads                             \t4\n","Compressed                          \t0\n","Verbosity                           \t3\n","Seed substitution matrix            \tnucl:nucleotide.out,aa:VTML80.out\n","Sensitivity                         \t7\n","k-mer length                        \t0\n","k-score                             \t2147483647\n","Alphabet size                       \tnucl:5,aa:21\n","Max results per query               \t300\n","Split database                      \t0\n","Split mode                          \t2\n","Split memory limit                  \t0\n","Diagonal scoring                    \ttrue\n","Exact k-mer matching                \t0\n","Mask residues                       \t1\n","Mask lower case residues            \t0\n","Minimum diagonal score              \t15\n","Spaced k-mers                       \t1\n","Spaced k-mer pattern                \t\n","Local temporary path                \t\n","Rescore mode                        \t0\n","Remove hits by seq. id. and coverage\tfalse\n","Sort results                        \t0\n","Mask profile                        \t1\n","Profile e-value threshold           \t0.1\n","Global sequence weighting           \tfalse\n","Allow deletions                     \tfalse\n","Filter MSA                          \t1\n","Maximum seq. id. threshold          \t0.9\n","Minimum seq. id.                    \t0\n","Minimum score per column            \t-20\n","Minimum coverage                    \t0\n","Select N most diverse seqs          \t1000\n","Omit consensus                      \tfalse\n","Min codons in orf                   \t30\n","Max codons in length                \t32734\n","Max orf gaps                        \t2147483647\n","Contig start mode                   \t2\n","Contig end mode                     \t2\n","Orf start mode                      \t1\n","Forward frames                      \t1,2,3\n","Reverse frames                      \t1,2,3\n","Translation table                   \t1\n","Translate orf                       \t0\n","Use all table starts                \tfalse\n","Offset of numeric ids               \t0\n","Create lookup                       \t0\n","Add orf stop                        \tfalse\n","Chain overlapping alignments        \t0\n","Merge query                         \t1\n","Search type                         \t0\n","Search iterations                   \t1\n","Start sensitivity                   \t4\n","Search steps                        \t1\n","Slice search mode                   \tfalse\n","Strand selection                    \t1\n","Disk space limit                    \t0\n","MPI runner                          \t\n","Force restart with latest tmp       \tfalse\n","Remove temporary files              \tfalse\n","\n","prefilter /content/mmseqs2/tmp/14/candidateDB_53 /content/mmseqs2/swissprot/swissprotDB /content/mmseqs2/tmp/14/4617683153757725916/pref_0 --sub-mat nucl:nucleotide.out,aa:blosum62.out --seed-sub-mat nucl:nucleotide.out,aa:VTML80.out -k 0 --k-score 2147483647 --alph-size nucl:5,aa:21 --max-seq-len 65535 --max-seqs 300 --split 0 --split-mode 2 --split-memory-limit 0 -c 0 --cov-mode 0 --comp-bias-corr 1 --diag-score 1 --exact-kmer-matching 0 --mask 1 --mask-lower-case 0 --min-ungapped-score 15 --add-self-matches 0 --spaced-kmer-mode 1 --db-load-mode 0 --pca 1 --pcb 1.5 --threads 4 --compressed 0 -v 3 -s 7.0 \n","\n","Query database size: 6333 type: Aminoacid\n","Estimated memory consumption: 2G\n","Target database size: 562755 type: Aminoacid\n","Index table k-mer threshold: 100 at k-mer size 6 \n","Index table: counting k-mers\n","[=================================================================] 100.00% 562.76K 24s 538ms\n","Index table: Masked residues: 3164874\n","Index table: fill\n","[=================================================================] 100.00% 562.76K 17s 801ms\n","Index statistics\n","Entries:          192757622\n","DB size:          1591 MB\n","Avg k-mer size:   3.011838\n","Top 10 k-mers\n","    GPGGTL\t1835\n","    GQSWTV\t1706\n","    WGMFAT\t1638\n","    PGVFEV\t1634\n","    VLWQFW\t1623\n","    AYIRPN\t1587\n","    RSPKGV\t1585\n","    TPHKWY\t1560\n","    HIRVGW\t1548\n","    ITLSPY\t1541\n","Time for index table init: 0h 0m 44s 403ms\n","Process prefiltering step 1 of 1\n","\n","k-mer similarity threshold: 100\n","Starting prefiltering scores calculation (step 1 of 1)\n","Query db start 1 to 6333\n","Target db start 1 to 562755\n","[=================================================================] 100.00% 6.33K 45s 70ms\n","\n","954.926194 k-mers per position\n","592149 DB matches per sequence\n","234 overflows\n","0 queries produce too many hits (truncated result)\n","283 sequences passed prefiltering per query sequence\n","300 median result list length\n","2 sequences with 0 size result lists\n","Time for merging to pref_0: 0h 0m 0s 1ms\n","Time for processing: 0h 1m 30s 618ms\n","align /content/mmseqs2/tmp/14/candidateDB_53 /content/mmseqs2/swissprot/swissprotDB /content/mmseqs2/tmp/14/4617683153757725916/pref_0 /content/mmseqs2/tmp/14/resultsDB_53 --sub-mat nucl:nucleotide.out,aa:blosum62.out -a 0 --alignment-mode 2 --wrapped-scoring 0 -e 0.001 --min-seq-id 0 --min-aln-len 0 --seq-id-mode 0 --alt-ali 0 -c 0 --cov-mode 0 --max-seq-len 65535 --comp-bias-corr 1 --realign 0 --max-rejected 2147483647 --max-accept 2147483647 --add-self-matches 0 --db-load-mode 0 --pca 1 --pcb 1.5 --score-bias 0 --gap-open nucl:5,aa:11 --gap-extend nucl:2,aa:1 --zdrop 40 --threads 4 --compressed 0 -v 3 \n","\n","Compute score and coverage\n","Query database size: 6333 type: Aminoacid\n","Target database size: 562755 type: Aminoacid\n","Calculation of alignments\n","[=================================================================] 100.00% 6.33K 34s 240ms\n","Time for merging to resultsDB_53: 0h 0m 0s 2ms\n","\n","1793007 alignments calculated.\n","315297 sequence pairs passed the thresholds (0.175848 of overall calculated).\n","49.786358 hits per query sequence.\n","Time for processing: 0h 0m 34s 309ms\n","convertalis /content/mmseqs2/tmp/14/candidateDB_53 /content/mmseqs2/swissprot/swissprotDB /content/mmseqs2/tmp/14/resultsDB_53 /content/mmseqs2/tmp/14/resultDB_secondary.m8 --format-output query,target,evalue,raw \n","\n","MMseqs Version:        \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Substitution matrix    \tnucl:nucleotide.out,aa:blosum62.out\n","Alignment format       \t0\n","Format alignment output\tquery,target,evalue,raw\n","Translation table      \t1\n","Gap open cost          \tnucl:5,aa:11\n","Gap extension cost     \tnucl:2,aa:1\n","Database output        \tfalse\n","Preload mode           \t0\n","Search type            \t0\n","Threads                \t4\n","Compressed             \t0\n","Verbosity              \t3\n","\n","[=================================================================] 100.00% 6.33K 0s 215ms\n","Time for merging to resultDB_secondary.m8: 0h 0m 0s 9ms\n","Time for processing: 0h 0m 0s 315ms\n","found 3559 genes\n","\n","\n","Reading fasta file GCF_001729945.1_ASM172994v1_genomic.fna.gz...\n","\n","Finding and translating open reading frames...\n","\n","Encoding amino acids...\n","\n","Applying kmer prefilter...\n","\n","Scoring ORFs with temporal convolutional network...\n","\n"],"name":"stdout"},{"output_type":"display_data","data":{"application/vnd.jupyter.widget-view+json":{"model_id":"bb83e6df8656425fb855240b40d559d9","version_minor":0,"version_major":2},"text/plain":["HBox(children=(FloatProgress(value=0.0, max=43.0), HTML(value='')))"]},"metadata":{"tags":[]}},{"output_type":"stream","text":["\n","Scoring translation initiation sites...\n","\n"],"name":"stdout"},{"output_type":"display_data","data":{"application/vnd.jupyter.widget-view+json":{"model_id":"49abb9be43d348dfa5df3012b39a6853","version_minor":0,"version_major":2},"text/plain":["HBox(children=(FloatProgress(value=0.0, max=13.0), HTML(value='')))"]},"metadata":{"tags":[]}},{"output_type":"stream","text":["\n","Creating weighted directed acyclic graph of all ORFs and subORFs on contig 0...\n","\n"],"name":"stdout"},{"output_type":"display_data","data":{"application/vnd.jupyter.widget-view+json":{"model_id":"b602f01384d74070bcc76e312e63d85a","version_minor":0,"version_major":2},"text/plain":["HBox(children=(FloatProgress(value=0.0, max=126841.0), HTML(value='')))"]},"metadata":{"tags":[]}},{"output_type":"stream","text":["\n","Maximizing geneiness...\n","\n","Filtering predictions with mmseqs2...\n","createdb /content/mmseqs2/tmp/15/candidate_genes_35.fasta /content/mmseqs2/tmp/15/candidateDB_35 \n","\n","MMseqs Version:       \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Database type         \t0\n","Shuffle input database\ttrue\n","Createdb mode         \t0\n","Write lookup file     \t1\n","Offset of numeric ids \t0\n","Compressed            \t0\n","Verbosity             \t3\n","\n","Converting sequences\n","[4950] 0s 6ms\n","Time for merging to candidateDB_35_h: 0h 0m 0s 3ms\n","Time for merging to candidateDB_35: 0h 0m 0s 4ms\n","Database type: Aminoacid\n","Time for processing: 0h 0m 0s 22ms\n","createdb /content/mmseqs2/tmp/15/candidate_genes_53.fasta /content/mmseqs2/tmp/15/candidateDB_53 \n","\n","MMseqs Version:       \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Database type         \t0\n","Shuffle input database\ttrue\n","Createdb mode         \t0\n","Write lookup file     \t1\n","Offset of numeric ids \t0\n","Compressed            \t0\n","Verbosity             \t3\n","\n","Converting sequences\n","[4950] 0s 6ms\n","Time for merging to candidateDB_53_h: 0h 0m 0s 4ms\n","Time for merging to candidateDB_53: 0h 0m 0s 4ms\n","Database type: Aminoacid\n","Time for processing: 0h 0m 0s 22ms\n","search -s 7.0 /content/mmseqs2/tmp/15/candidateDB_35 /content/mmseqs2/genexa/genexaDB /content/mmseqs2/tmp/15/resultsDB_35 /content/mmseqs2/tmp/15 \n","\n","MMseqs Version:                     \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Substitution matrix                 \tnucl:nucleotide.out,aa:blosum62.out\n","Add backtrace                       \tfalse\n","Alignment mode                      \t2\n","Allow wrapped scoring               \tfalse\n","E-value threshold                   \t0.001\n","Seq. id. threshold                  \t0\n","Min alignment length                \t0\n","Seq. id. mode                       \t0\n","Alternative alignments              \t0\n","Coverage threshold                  \t0\n","Coverage mode                       \t0\n","Max sequence length                 \t65535\n","Compositional bias                  \t1\n","Realign hits                        \tfalse\n","Max reject                          \t2147483647\n","Max accept                          \t2147483647\n","Include identical seq. id.          \tfalse\n","Preload mode                        \t0\n","Pseudo count a                      \t1\n","Pseudo count b                      \t1.5\n","Score bias                          \t0\n","Gap open cost                       \tnucl:5,aa:11\n","Gap extension cost                  \tnucl:2,aa:1\n","Zdrop                               \t40\n","Threads                             \t4\n","Compressed                          \t0\n","Verbosity                           \t3\n","Seed substitution matrix            \tnucl:nucleotide.out,aa:VTML80.out\n","Sensitivity                         \t7\n","k-mer length                        \t0\n","k-score                             \t2147483647\n","Alphabet size                       \tnucl:5,aa:21\n","Max results per query               \t300\n","Split database                      \t0\n","Split mode                          \t2\n","Split memory limit                  \t0\n","Diagonal scoring                    \ttrue\n","Exact k-mer matching                \t0\n","Mask residues                       \t1\n","Mask lower case residues            \t0\n","Minimum diagonal score              \t15\n","Spaced k-mers                       \t1\n","Spaced k-mer pattern                \t\n","Local temporary path                \t\n","Rescore mode                        \t0\n","Remove hits by seq. id. and coverage\tfalse\n","Sort results                        \t0\n","Mask profile                        \t1\n","Profile e-value threshold           \t0.1\n","Global sequence weighting           \tfalse\n","Allow deletions                     \tfalse\n","Filter MSA                          \t1\n","Maximum seq. id. threshold          \t0.9\n","Minimum seq. id.                    \t0\n","Minimum score per column            \t-20\n","Minimum coverage                    \t0\n","Select N most diverse seqs          \t1000\n","Omit consensus                      \tfalse\n","Min codons in orf                   \t30\n","Max codons in length                \t32734\n","Max orf gaps                        \t2147483647\n","Contig start mode                   \t2\n","Contig end mode                     \t2\n","Orf start mode                      \t1\n","Forward frames                      \t1,2,3\n","Reverse frames                      \t1,2,3\n","Translation table                   \t1\n","Translate orf                       \t0\n","Use all table starts                \tfalse\n","Offset of numeric ids               \t0\n","Create lookup                       \t0\n","Add orf stop                        \tfalse\n","Chain overlapping alignments        \t0\n","Merge query                         \t1\n","Search type                         \t0\n","Search iterations                   \t1\n","Start sensitivity                   \t4\n","Search steps                        \t1\n","Slice search mode                   \tfalse\n","Strand selection                    \t1\n","Disk space limit                    \t0\n","MPI runner                          \t\n","Force restart with latest tmp       \tfalse\n","Remove temporary files              \tfalse\n","\n","prefilter /content/mmseqs2/tmp/15/candidateDB_35 /content/mmseqs2/genexa/genexaDB.idx /content/mmseqs2/tmp/15/17852349009876124861/pref_0 --sub-mat nucl:nucleotide.out,aa:blosum62.out --seed-sub-mat nucl:nucleotide.out,aa:VTML80.out -k 0 --k-score 2147483647 --alph-size nucl:5,aa:21 --max-seq-len 65535 --max-seqs 300 --split 0 --split-mode 2 --split-memory-limit 0 -c 0 --cov-mode 0 --comp-bias-corr 1 --diag-score 1 --exact-kmer-matching 0 --mask 1 --mask-lower-case 0 --min-ungapped-score 15 --add-self-matches 0 --spaced-kmer-mode 1 --db-load-mode 0 --pca 1 --pcb 1.5 --threads 4 --compressed 0 -v 3 -s 7.0 \n","\n","Index version: 16\n","Generated by:  16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","ScoreMatrix:  VTML80.out\n","Query database size: 5027 type: Aminoacid\n","Estimated memory consumption: 2G\n","Target database size: 468351 type: Aminoacid\n","Process prefiltering step 1 of 1\n","\n","k-mer similarity threshold: 100\n","Starting prefiltering scores calculation (step 1 of 1)\n","Query db start 1 to 5027\n","Target db start 1 to 468351\n","[=================================================================] 100.00% 5.03K 33s 741ms\n","\n","963.885834 k-mers per position\n","541504 DB matches per sequence\n","155 overflows\n","0 queries produce too many hits (truncated result)\n","282 sequences passed prefiltering per query sequence\n","300 median result list length\n","0 sequences with 0 size result lists\n","Time for merging to pref_0: 0h 0m 0s 1ms\n","Time for processing: 0h 0m 35s 148ms\n","align /content/mmseqs2/tmp/15/candidateDB_35 /content/mmseqs2/genexa/genexaDB.idx /content/mmseqs2/tmp/15/17852349009876124861/pref_0 /content/mmseqs2/tmp/15/resultsDB_35 --sub-mat nucl:nucleotide.out,aa:blosum62.out -a 0 --alignment-mode 2 --wrapped-scoring 0 -e 0.001 --min-seq-id 0 --min-aln-len 0 --seq-id-mode 0 --alt-ali 0 -c 0 --cov-mode 0 --max-seq-len 65535 --comp-bias-corr 1 --realign 0 --max-rejected 2147483647 --max-accept 2147483647 --add-self-matches 0 --db-load-mode 0 --pca 1 --pcb 1.5 --score-bias 0 --gap-open nucl:5,aa:11 --gap-extend nucl:2,aa:1 --zdrop 40 --threads 4 --compressed 0 -v 3 \n","\n","Index version: 16\n","Generated by:  16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","ScoreMatrix:  VTML80.out\n","Compute score and coverage\n","Query database size: 5027 type: Aminoacid\n","Target database size: 468351 type: Aminoacid\n","Calculation of alignments\n","[=================================================================] 100.00% 5.03K 28s 810ms\n","Time for merging to resultsDB_35: 0h 0m 0s 1ms\n","\n","1418844 alignments calculated.\n","320555 sequence pairs passed the thresholds (0.225927 of overall calculated).\n","63.766659 hits per query sequence.\n","Time for processing: 0h 0m 28s 885ms\n","convertalis /content/mmseqs2/tmp/15/candidateDB_35 /content/mmseqs2/genexa/genexaDB /content/mmseqs2/tmp/15/resultsDB_35 /content/mmseqs2/tmp/15/resultDB_genexa.m8 --format-output query,target,evalue,raw \n","\n","MMseqs Version:        \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Substitution matrix    \tnucl:nucleotide.out,aa:blosum62.out\n","Alignment format       \t0\n","Format alignment output\tquery,target,evalue,raw\n","Translation table      \t1\n","Gap open cost          \tnucl:5,aa:11\n","Gap extension cost     \tnucl:2,aa:1\n","Database output        \tfalse\n","Preload mode           \t0\n","Search type            \t0\n","Threads                \t4\n","Compressed             \t0\n","Verbosity              \t3\n","\n","[=================================================================] 100.00% 5.03K 0s 238ms\n","Time for merging to resultDB_genexa.m8: 0h 0m 0s 10ms\n","Time for processing: 0h 0m 0s 321ms\n","search -s 7.0 /content/mmseqs2/tmp/15/candidateDB_53 /content/mmseqs2/swissprot/swissprotDB /content/mmseqs2/tmp/15/resultsDB_53 /content/mmseqs2/tmp/15 \n","\n","MMseqs Version:                     \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Substitution matrix                 \tnucl:nucleotide.out,aa:blosum62.out\n","Add backtrace                       \tfalse\n","Alignment mode                      \t2\n","Allow wrapped scoring               \tfalse\n","E-value threshold                   \t0.001\n","Seq. id. threshold                  \t0\n","Min alignment length                \t0\n","Seq. id. mode                       \t0\n","Alternative alignments              \t0\n","Coverage threshold                  \t0\n","Coverage mode                       \t0\n","Max sequence length                 \t65535\n","Compositional bias                  \t1\n","Realign hits                        \tfalse\n","Max reject                          \t2147483647\n","Max accept                          \t2147483647\n","Include identical seq. id.          \tfalse\n","Preload mode                        \t0\n","Pseudo count a                      \t1\n","Pseudo count b                      \t1.5\n","Score bias                          \t0\n","Gap open cost                       \tnucl:5,aa:11\n","Gap extension cost                  \tnucl:2,aa:1\n","Zdrop                               \t40\n","Threads                             \t4\n","Compressed                          \t0\n","Verbosity                           \t3\n","Seed substitution matrix            \tnucl:nucleotide.out,aa:VTML80.out\n","Sensitivity                         \t7\n","k-mer length                        \t0\n","k-score                             \t2147483647\n","Alphabet size                       \tnucl:5,aa:21\n","Max results per query               \t300\n","Split database                      \t0\n","Split mode                          \t2\n","Split memory limit                  \t0\n","Diagonal scoring                    \ttrue\n","Exact k-mer matching                \t0\n","Mask residues                       \t1\n","Mask lower case residues            \t0\n","Minimum diagonal score              \t15\n","Spaced k-mers                       \t1\n","Spaced k-mer pattern                \t\n","Local temporary path                \t\n","Rescore mode                        \t0\n","Remove hits by seq. id. and coverage\tfalse\n","Sort results                        \t0\n","Mask profile                        \t1\n","Profile e-value threshold           \t0.1\n","Global sequence weighting           \tfalse\n","Allow deletions                     \tfalse\n","Filter MSA                          \t1\n","Maximum seq. id. threshold          \t0.9\n","Minimum seq. id.                    \t0\n","Minimum score per column            \t-20\n","Minimum coverage                    \t0\n","Select N most diverse seqs          \t1000\n","Omit consensus                      \tfalse\n","Min codons in orf                   \t30\n","Max codons in length                \t32734\n","Max orf gaps                        \t2147483647\n","Contig start mode                   \t2\n","Contig end mode                     \t2\n","Orf start mode                      \t1\n","Forward frames                      \t1,2,3\n","Reverse frames                      \t1,2,3\n","Translation table                   \t1\n","Translate orf                       \t0\n","Use all table starts                \tfalse\n","Offset of numeric ids               \t0\n","Create lookup                       \t0\n","Add orf stop                        \tfalse\n","Chain overlapping alignments        \t0\n","Merge query                         \t1\n","Search type                         \t0\n","Search iterations                   \t1\n","Start sensitivity                   \t4\n","Search steps                        \t1\n","Slice search mode                   \tfalse\n","Strand selection                    \t1\n","Disk space limit                    \t0\n","MPI runner                          \t\n","Force restart with latest tmp       \tfalse\n","Remove temporary files              \tfalse\n","\n","prefilter /content/mmseqs2/tmp/15/candidateDB_53 /content/mmseqs2/swissprot/swissprotDB /content/mmseqs2/tmp/15/4719048141803071899/pref_0 --sub-mat nucl:nucleotide.out,aa:blosum62.out --seed-sub-mat nucl:nucleotide.out,aa:VTML80.out -k 0 --k-score 2147483647 --alph-size nucl:5,aa:21 --max-seq-len 65535 --max-seqs 300 --split 0 --split-mode 2 --split-memory-limit 0 -c 0 --cov-mode 0 --comp-bias-corr 1 --diag-score 1 --exact-kmer-matching 0 --mask 1 --mask-lower-case 0 --min-ungapped-score 15 --add-self-matches 0 --spaced-kmer-mode 1 --db-load-mode 0 --pca 1 --pcb 1.5 --threads 4 --compressed 0 -v 3 -s 7.0 \n","\n","Query database size: 5027 type: Aminoacid\n","Estimated memory consumption: 2G\n","Target database size: 562755 type: Aminoacid\n","Index table k-mer threshold: 100 at k-mer size 6 \n","Index table: counting k-mers\n","[=================================================================] 100.00% 562.76K 24s 321ms\n","Index table: Masked residues: 3164874\n","Index table: fill\n","[=================================================================] 100.00% 562.76K 17s 936ms\n","Index statistics\n","Entries:          192757622\n","DB size:          1591 MB\n","Avg k-mer size:   3.011838\n","Top 10 k-mers\n","    GPGGTL\t1835\n","    GQSWTV\t1706\n","    WGMFAT\t1638\n","    PGVFEV\t1634\n","    VLWQFW\t1623\n","    AYIRPN\t1587\n","    RSPKGV\t1585\n","    TPHKWY\t1560\n","    HIRVGW\t1548\n","    ITLSPY\t1541\n","Time for index table init: 0h 0m 44s 363ms\n","Process prefiltering step 1 of 1\n","\n","k-mer similarity threshold: 100\n","Starting prefiltering scores calculation (step 1 of 1)\n","Query db start 1 to 5027\n","Target db start 1 to 562755\n","[=================================================================] 100.00% 5.03K 37s 786ms\n","\n","962.246698 k-mers per position\n","637307 DB matches per sequence\n","274 overflows\n","0 queries produce too many hits (truncated result)\n","287 sequences passed prefiltering per query sequence\n","300 median result list length\n","0 sequences with 0 size result lists\n","Time for merging to pref_0: 0h 0m 0s 2ms\n","Time for processing: 0h 1m 23s 265ms\n","align /content/mmseqs2/tmp/15/candidateDB_53 /content/mmseqs2/swissprot/swissprotDB /content/mmseqs2/tmp/15/4719048141803071899/pref_0 /content/mmseqs2/tmp/15/resultsDB_53 --sub-mat nucl:nucleotide.out,aa:blosum62.out -a 0 --alignment-mode 2 --wrapped-scoring 0 -e 0.001 --min-seq-id 0 --min-aln-len 0 --seq-id-mode 0 --alt-ali 0 -c 0 --cov-mode 0 --max-seq-len 65535 --comp-bias-corr 1 --realign 0 --max-rejected 2147483647 --max-accept 2147483647 --add-self-matches 0 --db-load-mode 0 --pca 1 --pcb 1.5 --score-bias 0 --gap-open nucl:5,aa:11 --gap-extend nucl:2,aa:1 --zdrop 40 --threads 4 --compressed 0 -v 3 \n","\n","Compute score and coverage\n","Query database size: 5027 type: Aminoacid\n","Target database size: 562755 type: Aminoacid\n","Calculation of alignments\n","[=================================================================] 100.00% 5.03K 27s 786ms\n","Time for merging to resultsDB_53: 0h 0m 0s 1ms\n","\n","1447507 alignments calculated.\n","237303 sequence pairs passed the thresholds (0.163939 of overall calculated).\n","47.205688 hits per query sequence.\n","Time for processing: 0h 0m 27s 848ms\n","convertalis /content/mmseqs2/tmp/15/candidateDB_53 /content/mmseqs2/swissprot/swissprotDB /content/mmseqs2/tmp/15/resultsDB_53 /content/mmseqs2/tmp/15/resultDB_secondary.m8 --format-output query,target,evalue,raw \n","\n","MMseqs Version:        \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Substitution matrix    \tnucl:nucleotide.out,aa:blosum62.out\n","Alignment format       \t0\n","Format alignment output\tquery,target,evalue,raw\n","Translation table      \t1\n","Gap open cost          \tnucl:5,aa:11\n","Gap extension cost     \tnucl:2,aa:1\n","Database output        \tfalse\n","Preload mode           \t0\n","Search type            \t0\n","Threads                \t4\n","Compressed             \t0\n","Verbosity              \t3\n","\n","[=================================================================] 100.00% 5.03K 0s 162ms\n","Time for merging to resultDB_secondary.m8: 0h 0m 0s 7ms\n","Time for processing: 0h 0m 0s 260ms\n","found 2920 genes\n","\n","\n","Reading fasta file GCF_001547735.1_ASM154773v1_genomic.fna.gz...\n","\n","Finding and translating open reading frames...\n","\n","Encoding amino acids...\n","\n","Applying kmer prefilter...\n","\n","Scoring ORFs with temporal convolutional network...\n","\n"],"name":"stdout"},{"output_type":"display_data","data":{"application/vnd.jupyter.widget-view+json":{"model_id":"6827cf9798a849939a10a17e4306c4da","version_minor":0,"version_major":2},"text/plain":["HBox(children=(FloatProgress(value=0.0, max=28.0), HTML(value='')))"]},"metadata":{"tags":[]}},{"output_type":"stream","text":["\n","Scoring translation initiation sites...\n","\n"],"name":"stdout"},{"output_type":"display_data","data":{"application/vnd.jupyter.widget-view+json":{"model_id":"305a0d07e2f841759d1a07379bc042f1","version_minor":0,"version_major":2},"text/plain":["HBox(children=(FloatProgress(value=0.0, max=9.0), HTML(value='')))"]},"metadata":{"tags":[]}},{"output_type":"stream","text":["\n","Creating weighted directed acyclic graph of all ORFs and subORFs on contig 0...\n","\n"],"name":"stdout"},{"output_type":"display_data","data":{"application/vnd.jupyter.widget-view+json":{"model_id":"c4b1be860dec4cad817d25d78dd3a568","version_minor":0,"version_major":2},"text/plain":["HBox(children=(FloatProgress(value=0.0, max=82782.0), HTML(value='')))"]},"metadata":{"tags":[]}},{"output_type":"stream","text":["\n","Maximizing geneiness...\n","\n","Filtering predictions with mmseqs2...\n","createdb /content/mmseqs2/tmp/16/candidate_genes_35.fasta /content/mmseqs2/tmp/16/candidateDB_35 \n","\n","MMseqs Version:       \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Database type         \t0\n","Shuffle input database\ttrue\n","Createdb mode         \t0\n","Write lookup file     \t1\n","Offset of numeric ids \t0\n","Compressed            \t0\n","Verbosity             \t3\n","\n","Converting sequences\n","[2223] 0s 4ms\n","Time for merging to candidateDB_35_h: 0h 0m 0s 4ms\n","Time for merging to candidateDB_35: 0h 0m 0s 3ms\n","Database type: Aminoacid\n","Time for processing: 0h 0m 0s 16ms\n","createdb /content/mmseqs2/tmp/16/candidate_genes_53.fasta /content/mmseqs2/tmp/16/candidateDB_53 \n","\n","MMseqs Version:       \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Database type         \t0\n","Shuffle input database\ttrue\n","Createdb mode         \t0\n","Write lookup file     \t1\n","Offset of numeric ids \t0\n","Compressed            \t0\n","Verbosity             \t3\n","\n","Converting sequences\n","[2223] 0s 4ms\n","Time for merging to candidateDB_53_h: 0h 0m 0s 2ms\n","Time for merging to candidateDB_53: 0h 0m 0s 3ms\n","Database type: Aminoacid\n","Time for processing: 0h 0m 0s 15ms\n","search -s 7.0 /content/mmseqs2/tmp/16/candidateDB_35 /content/mmseqs2/genexa/genexaDB /content/mmseqs2/tmp/16/resultsDB_35 /content/mmseqs2/tmp/16 \n","\n","MMseqs Version:                     \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Substitution matrix                 \tnucl:nucleotide.out,aa:blosum62.out\n","Add backtrace                       \tfalse\n","Alignment mode                      \t2\n","Allow wrapped scoring               \tfalse\n","E-value threshold                   \t0.001\n","Seq. id. threshold                  \t0\n","Min alignment length                \t0\n","Seq. id. mode                       \t0\n","Alternative alignments              \t0\n","Coverage threshold                  \t0\n","Coverage mode                       \t0\n","Max sequence length                 \t65535\n","Compositional bias                  \t1\n","Realign hits                        \tfalse\n","Max reject                          \t2147483647\n","Max accept                          \t2147483647\n","Include identical seq. id.          \tfalse\n","Preload mode                        \t0\n","Pseudo count a                      \t1\n","Pseudo count b                      \t1.5\n","Score bias                          \t0\n","Gap open cost                       \tnucl:5,aa:11\n","Gap extension cost                  \tnucl:2,aa:1\n","Zdrop                               \t40\n","Threads                             \t4\n","Compressed                          \t0\n","Verbosity                           \t3\n","Seed substitution matrix            \tnucl:nucleotide.out,aa:VTML80.out\n","Sensitivity                         \t7\n","k-mer length                        \t0\n","k-score                             \t2147483647\n","Alphabet size                       \tnucl:5,aa:21\n","Max results per query               \t300\n","Split database                      \t0\n","Split mode                          \t2\n","Split memory limit                  \t0\n","Diagonal scoring                    \ttrue\n","Exact k-mer matching                \t0\n","Mask residues                       \t1\n","Mask lower case residues            \t0\n","Minimum diagonal score              \t15\n","Spaced k-mers                       \t1\n","Spaced k-mer pattern                \t\n","Local temporary path                \t\n","Rescore mode                        \t0\n","Remove hits by seq. id. and coverage\tfalse\n","Sort results                        \t0\n","Mask profile                        \t1\n","Profile e-value threshold           \t0.1\n","Global sequence weighting           \tfalse\n","Allow deletions                     \tfalse\n","Filter MSA                          \t1\n","Maximum seq. id. threshold          \t0.9\n","Minimum seq. id.                    \t0\n","Minimum score per column            \t-20\n","Minimum coverage                    \t0\n","Select N most diverse seqs          \t1000\n","Omit consensus                      \tfalse\n","Min codons in orf                   \t30\n","Max codons in length                \t32734\n","Max orf gaps                        \t2147483647\n","Contig start mode                   \t2\n","Contig end mode                     \t2\n","Orf start mode                      \t1\n","Forward frames                      \t1,2,3\n","Reverse frames                      \t1,2,3\n","Translation table                   \t1\n","Translate orf                       \t0\n","Use all table starts                \tfalse\n","Offset of numeric ids               \t0\n","Create lookup                       \t0\n","Add orf stop                        \tfalse\n","Chain overlapping alignments        \t0\n","Merge query                         \t1\n","Search type                         \t0\n","Search iterations                   \t1\n","Start sensitivity                   \t4\n","Search steps                        \t1\n","Slice search mode                   \tfalse\n","Strand selection                    \t1\n","Disk space limit                    \t0\n","MPI runner                          \t\n","Force restart with latest tmp       \tfalse\n","Remove temporary files              \tfalse\n","\n","prefilter /content/mmseqs2/tmp/16/candidateDB_35 /content/mmseqs2/genexa/genexaDB.idx /content/mmseqs2/tmp/16/12592062906766429372/pref_0 --sub-mat nucl:nucleotide.out,aa:blosum62.out --seed-sub-mat nucl:nucleotide.out,aa:VTML80.out -k 0 --k-score 2147483647 --alph-size nucl:5,aa:21 --max-seq-len 65535 --max-seqs 300 --split 0 --split-mode 2 --split-memory-limit 0 -c 0 --cov-mode 0 --comp-bias-corr 1 --diag-score 1 --exact-kmer-matching 0 --mask 1 --mask-lower-case 0 --min-ungapped-score 15 --add-self-matches 0 --spaced-kmer-mode 1 --db-load-mode 0 --pca 1 --pcb 1.5 --threads 4 --compressed 0 -v 3 -s 7.0 \n","\n","Index version: 16\n","Generated by:  16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","ScoreMatrix:  VTML80.out\n","Query database size: 2251 type: Aminoacid\n","Estimated memory consumption: 2G\n","Target database size: 468351 type: Aminoacid\n","Process prefiltering step 1 of 1\n","\n","k-mer similarity threshold: 100\n","Starting prefiltering scores calculation (step 1 of 1)\n","Query db start 1 to 2251\n","Target db start 1 to 468351\n","[=================================================================] 100.00% 2.25K 22s 322ms\n","\n","1026.009243 k-mers per position\n","764452 DB matches per sequence\n","119 overflows\n","0 queries produce too many hits (truncated result)\n","287 sequences passed prefiltering per query sequence\n","300 median result list length\n","1 sequences with 0 size result lists\n","Time for merging to pref_0: 0h 0m 0s 1ms\n","Time for processing: 0h 0m 23s 828ms\n","align /content/mmseqs2/tmp/16/candidateDB_35 /content/mmseqs2/genexa/genexaDB.idx /content/mmseqs2/tmp/16/12592062906766429372/pref_0 /content/mmseqs2/tmp/16/resultsDB_35 --sub-mat nucl:nucleotide.out,aa:blosum62.out -a 0 --alignment-mode 2 --wrapped-scoring 0 -e 0.001 --min-seq-id 0 --min-aln-len 0 --seq-id-mode 0 --alt-ali 0 -c 0 --cov-mode 0 --max-seq-len 65535 --comp-bias-corr 1 --realign 0 --max-rejected 2147483647 --max-accept 2147483647 --add-self-matches 0 --db-load-mode 0 --pca 1 --pcb 1.5 --score-bias 0 --gap-open nucl:5,aa:11 --gap-extend nucl:2,aa:1 --zdrop 40 --threads 4 --compressed 0 -v 3 \n","\n","Index version: 16\n","Generated by:  16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","ScoreMatrix:  VTML80.out\n","Compute score and coverage\n","Query database size: 2251 type: Aminoacid\n","Target database size: 468351 type: Aminoacid\n","Calculation of alignments\n","[=================================================================] 100.00% 2.25K 19s 646ms\n","Time for merging to resultsDB_35: 0h 0m 0s 0ms\n","\n","648191 alignments calculated.\n","233402 sequence pairs passed the thresholds (0.360082 of overall calculated).\n","103.688141 hits per query sequence.\n","Time for processing: 0h 0m 19s 676ms\n","convertalis /content/mmseqs2/tmp/16/candidateDB_35 /content/mmseqs2/genexa/genexaDB /content/mmseqs2/tmp/16/resultsDB_35 /content/mmseqs2/tmp/16/resultDB_genexa.m8 --format-output query,target,evalue,raw \n","\n","MMseqs Version:        \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Substitution matrix    \tnucl:nucleotide.out,aa:blosum62.out\n","Alignment format       \t0\n","Format alignment output\tquery,target,evalue,raw\n","Translation table      \t1\n","Gap open cost          \tnucl:5,aa:11\n","Gap extension cost     \tnucl:2,aa:1\n","Database output        \tfalse\n","Preload mode           \t0\n","Search type            \t0\n","Threads                \t4\n","Compressed             \t0\n","Verbosity              \t3\n","\n","[=================================================================] 100.00% 2.25K 0s 175ms\n","Time for merging to resultDB_genexa.m8: 0h 0m 0s 6ms\n","Time for processing: 0h 0m 0s 250ms\n","search -s 7.0 /content/mmseqs2/tmp/16/candidateDB_53 /content/mmseqs2/swissprot/swissprotDB /content/mmseqs2/tmp/16/resultsDB_53 /content/mmseqs2/tmp/16 \n","\n","MMseqs Version:                     \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Substitution matrix                 \tnucl:nucleotide.out,aa:blosum62.out\n","Add backtrace                       \tfalse\n","Alignment mode                      \t2\n","Allow wrapped scoring               \tfalse\n","E-value threshold                   \t0.001\n","Seq. id. threshold                  \t0\n","Min alignment length                \t0\n","Seq. id. mode                       \t0\n","Alternative alignments              \t0\n","Coverage threshold                  \t0\n","Coverage mode                       \t0\n","Max sequence length                 \t65535\n","Compositional bias                  \t1\n","Realign hits                        \tfalse\n","Max reject                          \t2147483647\n","Max accept                          \t2147483647\n","Include identical seq. id.          \tfalse\n","Preload mode                        \t0\n","Pseudo count a                      \t1\n","Pseudo count b                      \t1.5\n","Score bias                          \t0\n","Gap open cost                       \tnucl:5,aa:11\n","Gap extension cost                  \tnucl:2,aa:1\n","Zdrop                               \t40\n","Threads                             \t4\n","Compressed                          \t0\n","Verbosity                           \t3\n","Seed substitution matrix            \tnucl:nucleotide.out,aa:VTML80.out\n","Sensitivity                         \t7\n","k-mer length                        \t0\n","k-score                             \t2147483647\n","Alphabet size                       \tnucl:5,aa:21\n","Max results per query               \t300\n","Split database                      \t0\n","Split mode                          \t2\n","Split memory limit                  \t0\n","Diagonal scoring                    \ttrue\n","Exact k-mer matching                \t0\n","Mask residues                       \t1\n","Mask lower case residues            \t0\n","Minimum diagonal score              \t15\n","Spaced k-mers                       \t1\n","Spaced k-mer pattern                \t\n","Local temporary path                \t\n","Rescore mode                        \t0\n","Remove hits by seq. id. and coverage\tfalse\n","Sort results                        \t0\n","Mask profile                        \t1\n","Profile e-value threshold           \t0.1\n","Global sequence weighting           \tfalse\n","Allow deletions                     \tfalse\n","Filter MSA                          \t1\n","Maximum seq. id. threshold          \t0.9\n","Minimum seq. id.                    \t0\n","Minimum score per column            \t-20\n","Minimum coverage                    \t0\n","Select N most diverse seqs          \t1000\n","Omit consensus                      \tfalse\n","Min codons in orf                   \t30\n","Max codons in length                \t32734\n","Max orf gaps                        \t2147483647\n","Contig start mode                   \t2\n","Contig end mode                     \t2\n","Orf start mode                      \t1\n","Forward frames                      \t1,2,3\n","Reverse frames                      \t1,2,3\n","Translation table                   \t1\n","Translate orf                       \t0\n","Use all table starts                \tfalse\n","Offset of numeric ids               \t0\n","Create lookup                       \t0\n","Add orf stop                        \tfalse\n","Chain overlapping alignments        \t0\n","Merge query                         \t1\n","Search type                         \t0\n","Search iterations                   \t1\n","Start sensitivity                   \t4\n","Search steps                        \t1\n","Slice search mode                   \tfalse\n","Strand selection                    \t1\n","Disk space limit                    \t0\n","MPI runner                          \t\n","Force restart with latest tmp       \tfalse\n","Remove temporary files              \tfalse\n","\n","prefilter /content/mmseqs2/tmp/16/candidateDB_53 /content/mmseqs2/swissprot/swissprotDB /content/mmseqs2/tmp/16/4820413129848417882/pref_0 --sub-mat nucl:nucleotide.out,aa:blosum62.out --seed-sub-mat nucl:nucleotide.out,aa:VTML80.out -k 0 --k-score 2147483647 --alph-size nucl:5,aa:21 --max-seq-len 65535 --max-seqs 300 --split 0 --split-mode 2 --split-memory-limit 0 -c 0 --cov-mode 0 --comp-bias-corr 1 --diag-score 1 --exact-kmer-matching 0 --mask 1 --mask-lower-case 0 --min-ungapped-score 15 --add-self-matches 0 --spaced-kmer-mode 1 --db-load-mode 0 --pca 1 --pcb 1.5 --threads 4 --compressed 0 -v 3 -s 7.0 \n","\n","Query database size: 2251 type: Aminoacid\n","Estimated memory consumption: 2G\n","Target database size: 562755 type: Aminoacid\n","Index table k-mer threshold: 100 at k-mer size 6 \n","Index table: counting k-mers\n","[=================================================================] 100.00% 562.76K 24s 413ms\n","Index table: Masked residues: 3164874\n","Index table: fill\n","[=================================================================] 100.00% 562.76K 17s 847ms\n","Index statistics\n","Entries:          192757622\n","DB size:          1591 MB\n","Avg k-mer size:   3.011838\n","Top 10 k-mers\n","    GPGGTL\t1835\n","    GQSWTV\t1706\n","    WGMFAT\t1638\n","    PGVFEV\t1634\n","    VLWQFW\t1623\n","    AYIRPN\t1587\n","    RSPKGV\t1585\n","    TPHKWY\t1560\n","    HIRVGW\t1548\n","    ITLSPY\t1541\n","Time for index table init: 0h 0m 44s 386ms\n","Process prefiltering step 1 of 1\n","\n","k-mer similarity threshold: 100\n","Starting prefiltering scores calculation (step 1 of 1)\n","Query db start 1 to 2251\n","Target db start 1 to 562755\n","[=================================================================] 100.00% 2.25K 25s 705ms\n","\n","1026.583280 k-mers per position\n","950000 DB matches per sequence\n","235 overflows\n","0 queries produce too many hits (truncated result)\n","291 sequences passed prefiltering per query sequence\n","300 median result list length\n","1 sequences with 0 size result lists\n","Time for merging to pref_0: 0h 0m 0s 0ms\n","Time for processing: 0h 1m 11s 312ms\n","align /content/mmseqs2/tmp/16/candidateDB_53 /content/mmseqs2/swissprot/swissprotDB /content/mmseqs2/tmp/16/4820413129848417882/pref_0 /content/mmseqs2/tmp/16/resultsDB_53 --sub-mat nucl:nucleotide.out,aa:blosum62.out -a 0 --alignment-mode 2 --wrapped-scoring 0 -e 0.001 --min-seq-id 0 --min-aln-len 0 --seq-id-mode 0 --alt-ali 0 -c 0 --cov-mode 0 --max-seq-len 65535 --comp-bias-corr 1 --realign 0 --max-rejected 2147483647 --max-accept 2147483647 --add-self-matches 0 --db-load-mode 0 --pca 1 --pcb 1.5 --score-bias 0 --gap-open nucl:5,aa:11 --gap-extend nucl:2,aa:1 --zdrop 40 --threads 4 --compressed 0 -v 3 \n","\n","Compute score and coverage\n","Query database size: 2251 type: Aminoacid\n","Target database size: 562755 type: Aminoacid\n","Calculation of alignments\n","[=================================================================] 100.00% 2.25K 19s 535ms\n","Time for merging to resultsDB_53: 0h 0m 0s 0ms\n","\n","655930 alignments calculated.\n","195509 sequence pairs passed the thresholds (0.298064 of overall calculated).\n","86.854286 hits per query sequence.\n","Time for processing: 0h 0m 19s 641ms\n","convertalis /content/mmseqs2/tmp/16/candidateDB_53 /content/mmseqs2/swissprot/swissprotDB /content/mmseqs2/tmp/16/resultsDB_53 /content/mmseqs2/tmp/16/resultDB_secondary.m8 --format-output query,target,evalue,raw \n","\n","MMseqs Version:        \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Substitution matrix    \tnucl:nucleotide.out,aa:blosum62.out\n","Alignment format       \t0\n","Format alignment output\tquery,target,evalue,raw\n","Translation table      \t1\n","Gap open cost          \tnucl:5,aa:11\n","Gap extension cost     \tnucl:2,aa:1\n","Database output        \tfalse\n","Preload mode           \t0\n","Search type            \t0\n","Threads                \t4\n","Compressed             \t0\n","Verbosity              \t3\n","\n","[=================================================================] 100.00% 2.25K 0s 139ms\n","Time for merging to resultDB_secondary.m8: 0h 0m 0s 6ms\n","Time for processing: 0h 0m 0s 235ms\n","found 1818 genes\n","\n","\n","Reading fasta file GCF_001263175.1_ASM126317v1_genomic.fna.gz...\n","\n","Finding and translating open reading frames...\n","\n","Encoding amino acids...\n","\n","Applying kmer prefilter...\n","\n","Scoring ORFs with temporal convolutional network...\n","\n"],"name":"stdout"},{"output_type":"display_data","data":{"application/vnd.jupyter.widget-view+json":{"model_id":"b170446e153a470a8c0a960dda54c53c","version_minor":0,"version_major":2},"text/plain":["HBox(children=(FloatProgress(value=0.0, max=68.0), HTML(value='')))"]},"metadata":{"tags":[]}},{"output_type":"stream","text":["\n","Scoring translation initiation sites...\n","\n"],"name":"stdout"},{"output_type":"display_data","data":{"application/vnd.jupyter.widget-view+json":{"model_id":"16e15acd67614143aea32b54880875e6","version_minor":0,"version_major":2},"text/plain":["HBox(children=(FloatProgress(value=0.0, max=21.0), HTML(value='')))"]},"metadata":{"tags":[]}},{"output_type":"stream","text":["\n","Creating weighted directed acyclic graph of all ORFs and subORFs on contig 0...\n","\n"],"name":"stdout"},{"output_type":"display_data","data":{"application/vnd.jupyter.widget-view+json":{"model_id":"5d55188da98243b88ea68e297494abe9","version_minor":0,"version_major":2},"text/plain":["HBox(children=(FloatProgress(value=0.0, max=201861.0), HTML(value='')))"]},"metadata":{"tags":[]}},{"output_type":"stream","text":["\n","Maximizing geneiness...\n","\n","Filtering predictions with mmseqs2...\n","createdb /content/mmseqs2/tmp/17/candidate_genes_35.fasta /content/mmseqs2/tmp/17/candidateDB_35 \n","\n","MMseqs Version:       \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Database type         \t0\n","Shuffle input database\ttrue\n","Createdb mode         \t0\n","Write lookup file     \t1\n","Offset of numeric ids \t0\n","Compressed            \t0\n","Verbosity             \t3\n","\n","Converting sequences\n","[5354] 0s 8ms\n","Time for merging to candidateDB_35_h: 0h 0m 0s 4ms\n","Time for merging to candidateDB_35: 0h 0m 0s 4ms\n","Database type: Aminoacid\n","Time for processing: 0h 0m 0s 25ms\n","createdb /content/mmseqs2/tmp/17/candidate_genes_53.fasta /content/mmseqs2/tmp/17/candidateDB_53 \n","\n","MMseqs Version:       \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Database type         \t0\n","Shuffle input database\ttrue\n","Createdb mode         \t0\n","Write lookup file     \t1\n","Offset of numeric ids \t0\n","Compressed            \t0\n","Verbosity             \t3\n","\n","Converting sequences\n","[5354] 0s 8ms\n","Time for merging to candidateDB_53_h: 0h 0m 0s 3ms\n","Time for merging to candidateDB_53: 0h 0m 0s 5ms\n","Database type: Aminoacid\n","Time for processing: 0h 0m 0s 23ms\n","search -s 7.0 /content/mmseqs2/tmp/17/candidateDB_35 /content/mmseqs2/genexa/genexaDB /content/mmseqs2/tmp/17/resultsDB_35 /content/mmseqs2/tmp/17 \n","\n","MMseqs Version:                     \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Substitution matrix                 \tnucl:nucleotide.out,aa:blosum62.out\n","Add backtrace                       \tfalse\n","Alignment mode                      \t2\n","Allow wrapped scoring               \tfalse\n","E-value threshold                   \t0.001\n","Seq. id. threshold                  \t0\n","Min alignment length                \t0\n","Seq. id. mode                       \t0\n","Alternative alignments              \t0\n","Coverage threshold                  \t0\n","Coverage mode                       \t0\n","Max sequence length                 \t65535\n","Compositional bias                  \t1\n","Realign hits                        \tfalse\n","Max reject                          \t2147483647\n","Max accept                          \t2147483647\n","Include identical seq. id.          \tfalse\n","Preload mode                        \t0\n","Pseudo count a                      \t1\n","Pseudo count b                      \t1.5\n","Score bias                          \t0\n","Gap open cost                       \tnucl:5,aa:11\n","Gap extension cost                  \tnucl:2,aa:1\n","Zdrop                               \t40\n","Threads                             \t4\n","Compressed                          \t0\n","Verbosity                           \t3\n","Seed substitution matrix            \tnucl:nucleotide.out,aa:VTML80.out\n","Sensitivity                         \t7\n","k-mer length                        \t0\n","k-score                             \t2147483647\n","Alphabet size                       \tnucl:5,aa:21\n","Max results per query               \t300\n","Split database                      \t0\n","Split mode                          \t2\n","Split memory limit                  \t0\n","Diagonal scoring                    \ttrue\n","Exact k-mer matching                \t0\n","Mask residues                       \t1\n","Mask lower case residues            \t0\n","Minimum diagonal score              \t15\n","Spaced k-mers                       \t1\n","Spaced k-mer pattern                \t\n","Local temporary path                \t\n","Rescore mode                        \t0\n","Remove hits by seq. id. and coverage\tfalse\n","Sort results                        \t0\n","Mask profile                        \t1\n","Profile e-value threshold           \t0.1\n","Global sequence weighting           \tfalse\n","Allow deletions                     \tfalse\n","Filter MSA                          \t1\n","Maximum seq. id. threshold          \t0.9\n","Minimum seq. id.                    \t0\n","Minimum score per column            \t-20\n","Minimum coverage                    \t0\n","Select N most diverse seqs          \t1000\n","Omit consensus                      \tfalse\n","Min codons in orf                   \t30\n","Max codons in length                \t32734\n","Max orf gaps                        \t2147483647\n","Contig start mode                   \t2\n","Contig end mode                     \t2\n","Orf start mode                      \t1\n","Forward frames                      \t1,2,3\n","Reverse frames                      \t1,2,3\n","Translation table                   \t1\n","Translate orf                       \t0\n","Use all table starts                \tfalse\n","Offset of numeric ids               \t0\n","Create lookup                       \t0\n","Add orf stop                        \tfalse\n","Chain overlapping alignments        \t0\n","Merge query                         \t1\n","Search type                         \t0\n","Search iterations                   \t1\n","Start sensitivity                   \t4\n","Search steps                        \t1\n","Slice search mode                   \tfalse\n","Strand selection                    \t1\n","Disk space limit                    \t0\n","MPI runner                          \t\n","Force restart with latest tmp       \tfalse\n","Remove temporary files              \tfalse\n","\n","prefilter /content/mmseqs2/tmp/17/candidateDB_35 /content/mmseqs2/genexa/genexaDB.idx /content/mmseqs2/tmp/17/7331776803656733883/pref_0 --sub-mat nucl:nucleotide.out,aa:blosum62.out --seed-sub-mat nucl:nucleotide.out,aa:VTML80.out -k 0 --k-score 2147483647 --alph-size nucl:5,aa:21 --max-seq-len 65535 --max-seqs 300 --split 0 --split-mode 2 --split-memory-limit 0 -c 0 --cov-mode 0 --comp-bias-corr 1 --diag-score 1 --exact-kmer-matching 0 --mask 1 --mask-lower-case 0 --min-ungapped-score 15 --add-self-matches 0 --spaced-kmer-mode 1 --db-load-mode 0 --pca 1 --pcb 1.5 --threads 4 --compressed 0 -v 3 -s 7.0 \n","\n","Index version: 16\n","Generated by:  16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","ScoreMatrix:  VTML80.out\n","Query database size: 5436 type: Aminoacid\n","Estimated memory consumption: 2G\n","Target database size: 468351 type: Aminoacid\n","Process prefiltering step 1 of 1\n","\n","k-mer similarity threshold: 100\n","Starting prefiltering scores calculation (step 1 of 1)\n","Query db start 1 to 5436\n","Target db start 1 to 468351\n","[=================================================================] 100.00% 5.44K 44s 299ms\n","\n","777.124615 k-mers per position\n","691687 DB matches per sequence\n","315 overflows\n","0 queries produce too many hits (truncated result)\n","274 sequences passed prefiltering per query sequence\n","300 median result list length\n","6 sequences with 0 size result lists\n","Time for merging to pref_0: 0h 0m 0s 1ms\n","Time for processing: 0h 0m 45s 755ms\n","align /content/mmseqs2/tmp/17/candidateDB_35 /content/mmseqs2/genexa/genexaDB.idx /content/mmseqs2/tmp/17/7331776803656733883/pref_0 /content/mmseqs2/tmp/17/resultsDB_35 --sub-mat nucl:nucleotide.out,aa:blosum62.out -a 0 --alignment-mode 2 --wrapped-scoring 0 -e 0.001 --min-seq-id 0 --min-aln-len 0 --seq-id-mode 0 --alt-ali 0 -c 0 --cov-mode 0 --max-seq-len 65535 --comp-bias-corr 1 --realign 0 --max-rejected 2147483647 --max-accept 2147483647 --add-self-matches 0 --db-load-mode 0 --pca 1 --pcb 1.5 --score-bias 0 --gap-open nucl:5,aa:11 --gap-extend nucl:2,aa:1 --zdrop 40 --threads 4 --compressed 0 -v 3 \n","\n","Index version: 16\n","Generated by:  16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","ScoreMatrix:  VTML80.out\n","Compute score and coverage\n","Query database size: 5436 type: Aminoacid\n","Target database size: 468351 type: Aminoacid\n","Calculation of alignments\n","[=================================================================] 100.00% 5.44K 35s 253ms\n","Time for merging to resultsDB_35: 0h 0m 0s 1ms\n","\n","1492088 alignments calculated.\n","356200 sequence pairs passed the thresholds (0.238726 of overall calculated).\n","65.526123 hits per query sequence.\n","Time for processing: 0h 0m 35s 290ms\n","convertalis /content/mmseqs2/tmp/17/candidateDB_35 /content/mmseqs2/genexa/genexaDB /content/mmseqs2/tmp/17/resultsDB_35 /content/mmseqs2/tmp/17/resultDB_genexa.m8 --format-output query,target,evalue,raw \n","\n","MMseqs Version:        \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Substitution matrix    \tnucl:nucleotide.out,aa:blosum62.out\n","Alignment format       \t0\n","Format alignment output\tquery,target,evalue,raw\n","Translation table      \t1\n","Gap open cost          \tnucl:5,aa:11\n","Gap extension cost     \tnucl:2,aa:1\n","Database output        \tfalse\n","Preload mode           \t0\n","Search type            \t0\n","Threads                \t4\n","Compressed             \t0\n","Verbosity              \t3\n","\n","[=================================================================] 100.00% 5.44K 0s 264ms\n","Time for merging to resultDB_genexa.m8: 0h 0m 0s 10ms\n","Time for processing: 0h 0m 0s 345ms\n","search -s 7.0 /content/mmseqs2/tmp/17/candidateDB_53 /content/mmseqs2/swissprot/swissprotDB /content/mmseqs2/tmp/17/resultsDB_53 /content/mmseqs2/tmp/17 \n","\n","MMseqs Version:                     \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Substitution matrix                 \tnucl:nucleotide.out,aa:blosum62.out\n","Add backtrace                       \tfalse\n","Alignment mode                      \t2\n","Allow wrapped scoring               \tfalse\n","E-value threshold                   \t0.001\n","Seq. id. threshold                  \t0\n","Min alignment length                \t0\n","Seq. id. mode                       \t0\n","Alternative alignments              \t0\n","Coverage threshold                  \t0\n","Coverage mode                       \t0\n","Max sequence length                 \t65535\n","Compositional bias                  \t1\n","Realign hits                        \tfalse\n","Max reject                          \t2147483647\n","Max accept                          \t2147483647\n","Include identical seq. id.          \tfalse\n","Preload mode                        \t0\n","Pseudo count a                      \t1\n","Pseudo count b                      \t1.5\n","Score bias                          \t0\n","Gap open cost                       \tnucl:5,aa:11\n","Gap extension cost                  \tnucl:2,aa:1\n","Zdrop                               \t40\n","Threads                             \t4\n","Compressed                          \t0\n","Verbosity                           \t3\n","Seed substitution matrix            \tnucl:nucleotide.out,aa:VTML80.out\n","Sensitivity                         \t7\n","k-mer length                        \t0\n","k-score                             \t2147483647\n","Alphabet size                       \tnucl:5,aa:21\n","Max results per query               \t300\n","Split database                      \t0\n","Split mode                          \t2\n","Split memory limit                  \t0\n","Diagonal scoring                    \ttrue\n","Exact k-mer matching                \t0\n","Mask residues                       \t1\n","Mask lower case residues            \t0\n","Minimum diagonal score              \t15\n","Spaced k-mers                       \t1\n","Spaced k-mer pattern                \t\n","Local temporary path                \t\n","Rescore mode                        \t0\n","Remove hits by seq. id. and coverage\tfalse\n","Sort results                        \t0\n","Mask profile                        \t1\n","Profile e-value threshold           \t0.1\n","Global sequence weighting           \tfalse\n","Allow deletions                     \tfalse\n","Filter MSA                          \t1\n","Maximum seq. id. threshold          \t0.9\n","Minimum seq. id.                    \t0\n","Minimum score per column            \t-20\n","Minimum coverage                    \t0\n","Select N most diverse seqs          \t1000\n","Omit consensus                      \tfalse\n","Min codons in orf                   \t30\n","Max codons in length                \t32734\n","Max orf gaps                        \t2147483647\n","Contig start mode                   \t2\n","Contig end mode                     \t2\n","Orf start mode                      \t1\n","Forward frames                      \t1,2,3\n","Reverse frames                      \t1,2,3\n","Translation table                   \t1\n","Translate orf                       \t0\n","Use all table starts                \tfalse\n","Offset of numeric ids               \t0\n","Create lookup                       \t0\n","Add orf stop                        \tfalse\n","Chain overlapping alignments        \t0\n","Merge query                         \t1\n","Search type                         \t0\n","Search iterations                   \t1\n","Start sensitivity                   \t4\n","Search steps                        \t1\n","Slice search mode                   \tfalse\n","Strand selection                    \t1\n","Disk space limit                    \t0\n","MPI runner                          \t\n","Force restart with latest tmp       \tfalse\n","Remove temporary files              \tfalse\n","\n","prefilter /content/mmseqs2/tmp/17/candidateDB_53 /content/mmseqs2/swissprot/swissprotDB /content/mmseqs2/tmp/17/4921778117893763865/pref_0 --sub-mat nucl:nucleotide.out,aa:blosum62.out --seed-sub-mat nucl:nucleotide.out,aa:VTML80.out -k 0 --k-score 2147483647 --alph-size nucl:5,aa:21 --max-seq-len 65535 --max-seqs 300 --split 0 --split-mode 2 --split-memory-limit 0 -c 0 --cov-mode 0 --comp-bias-corr 1 --diag-score 1 --exact-kmer-matching 0 --mask 1 --mask-lower-case 0 --min-ungapped-score 15 --add-self-matches 0 --spaced-kmer-mode 1 --db-load-mode 0 --pca 1 --pcb 1.5 --threads 4 --compressed 0 -v 3 -s 7.0 \n","\n","Query database size: 5436 type: Aminoacid\n","Estimated memory consumption: 2G\n","Target database size: 562755 type: Aminoacid\n","Index table k-mer threshold: 100 at k-mer size 6 \n","Index table: counting k-mers\n","[=================================================================] 100.00% 562.76K 24s 475ms\n","Index table: Masked residues: 3164874\n","Index table: fill\n","[=================================================================] 100.00% 562.76K 17s 773ms\n","Index statistics\n","Entries:          192757622\n","DB size:          1591 MB\n","Avg k-mer size:   3.011838\n","Top 10 k-mers\n","    GPGGTL\t1835\n","    GQSWTV\t1706\n","    WGMFAT\t1638\n","    PGVFEV\t1634\n","    VLWQFW\t1623\n","    AYIRPN\t1587\n","    RSPKGV\t1585\n","    TPHKWY\t1560\n","    HIRVGW\t1548\n","    ITLSPY\t1541\n","Time for index table init: 0h 0m 44s 332ms\n","Process prefiltering step 1 of 1\n","\n","k-mer similarity threshold: 100\n","Starting prefiltering scores calculation (step 1 of 1)\n","Query db start 1 to 5436\n","Target db start 1 to 562755\n","[=================================================================] 100.00% 5.44K 48s 667ms\n","\n","776.846731 k-mers per position\n","783741 DB matches per sequence\n","457 overflows\n","0 queries produce too many hits (truncated result)\n","279 sequences passed prefiltering per query sequence\n","300 median result list length\n","4 sequences with 0 size result lists\n","Time for merging to pref_0: 0h 0m 0s 1ms\n","Time for processing: 0h 1m 34s 176ms\n","align /content/mmseqs2/tmp/17/candidateDB_53 /content/mmseqs2/swissprot/swissprotDB /content/mmseqs2/tmp/17/4921778117893763865/pref_0 /content/mmseqs2/tmp/17/resultsDB_53 --sub-mat nucl:nucleotide.out,aa:blosum62.out -a 0 --alignment-mode 2 --wrapped-scoring 0 -e 0.001 --min-seq-id 0 --min-aln-len 0 --seq-id-mode 0 --alt-ali 0 -c 0 --cov-mode 0 --max-seq-len 65535 --comp-bias-corr 1 --realign 0 --max-rejected 2147483647 --max-accept 2147483647 --add-self-matches 0 --db-load-mode 0 --pca 1 --pcb 1.5 --score-bias 0 --gap-open nucl:5,aa:11 --gap-extend nucl:2,aa:1 --zdrop 40 --threads 4 --compressed 0 -v 3 \n","\n","Compute score and coverage\n","Query database size: 5436 type: Aminoacid\n","Target database size: 562755 type: Aminoacid\n","Calculation of alignments\n","[=================================================================] 100.00% 5.44K 34s 658ms\n","Time for merging to resultsDB_53: 0h 0m 0s 1ms\n","\n","1517844 alignments calculated.\n","274910 sequence pairs passed the thresholds (0.181119 of overall calculated).\n","50.572113 hits per query sequence.\n","Time for processing: 0h 0m 34s 733ms\n","convertalis /content/mmseqs2/tmp/17/candidateDB_53 /content/mmseqs2/swissprot/swissprotDB /content/mmseqs2/tmp/17/resultsDB_53 /content/mmseqs2/tmp/17/resultDB_secondary.m8 --format-output query,target,evalue,raw \n","\n","MMseqs Version:        \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Substitution matrix    \tnucl:nucleotide.out,aa:blosum62.out\n","Alignment format       \t0\n","Format alignment output\tquery,target,evalue,raw\n","Translation table      \t1\n","Gap open cost          \tnucl:5,aa:11\n","Gap extension cost     \tnucl:2,aa:1\n","Database output        \tfalse\n","Preload mode           \t0\n","Search type            \t0\n","Threads                \t4\n","Compressed             \t0\n","Verbosity              \t3\n","\n","[=================================================================] 100.00% 5.44K 0s 193ms\n","Time for merging to resultDB_secondary.m8: 0h 0m 0s 8ms\n","Time for processing: 0h 0m 0s 293ms\n","found 3602 genes\n","\n","\n","Reading fasta file GCF_001027025.1_ASM102702v1_genomic.fna.gz...\n","\n","Finding and translating open reading frames...\n","\n","Encoding amino acids...\n","\n","Applying kmer prefilter...\n","\n","Scoring ORFs with temporal convolutional network...\n","\n"],"name":"stdout"},{"output_type":"display_data","data":{"application/vnd.jupyter.widget-view+json":{"model_id":"6ea4c17aa5464fd89f91f787d0a6ad32","version_minor":0,"version_major":2},"text/plain":["HBox(children=(FloatProgress(value=0.0, max=32.0), HTML(value='')))"]},"metadata":{"tags":[]}},{"output_type":"stream","text":["\n","Scoring translation initiation sites...\n","\n"],"name":"stdout"},{"output_type":"display_data","data":{"application/vnd.jupyter.widget-view+json":{"model_id":"e9cd6a10e5f54cbf97a4580e36e04237","version_minor":0,"version_major":2},"text/plain":["HBox(children=(FloatProgress(value=0.0, max=10.0), HTML(value='')))"]},"metadata":{"tags":[]}},{"output_type":"stream","text":["\n","Creating weighted directed acyclic graph of all ORFs and subORFs on contig 0...\n","\n"],"name":"stdout"},{"output_type":"display_data","data":{"application/vnd.jupyter.widget-view+json":{"model_id":"ba22ecbbf35d4d1fb453c08123191325","version_minor":0,"version_major":2},"text/plain":["HBox(children=(FloatProgress(value=0.0, max=95634.0), HTML(value='')))"]},"metadata":{"tags":[]}},{"output_type":"stream","text":["\n","Maximizing geneiness...\n","\n","Filtering predictions with mmseqs2...\n","createdb /content/mmseqs2/tmp/18/candidate_genes_35.fasta /content/mmseqs2/tmp/18/candidateDB_35 \n","\n","MMseqs Version:       \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Database type         \t0\n","Shuffle input database\ttrue\n","Createdb mode         \t0\n","Write lookup file     \t1\n","Offset of numeric ids \t0\n","Compressed            \t0\n","Verbosity             \t3\n","\n","Converting sequences\n","[3132] 0s 6ms\n","Time for merging to candidateDB_35_h: 0h 0m 0s 5ms\n","Time for merging to candidateDB_35: 0h 0m 0s 3ms\n","Database type: Aminoacid\n","Time for processing: 0h 0m 0s 20ms\n","createdb /content/mmseqs2/tmp/18/candidate_genes_53.fasta /content/mmseqs2/tmp/18/candidateDB_53 \n","\n","MMseqs Version:       \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Database type         \t0\n","Shuffle input database\ttrue\n","Createdb mode         \t0\n","Write lookup file     \t1\n","Offset of numeric ids \t0\n","Compressed            \t0\n","Verbosity             \t3\n","\n","Converting sequences\n","[3132] 0s 5ms\n","Time for merging to candidateDB_53_h: 0h 0m 0s 4ms\n","Time for merging to candidateDB_53: 0h 0m 0s 3ms\n","Database type: Aminoacid\n","Time for processing: 0h 0m 0s 17ms\n","search -s 7.0 /content/mmseqs2/tmp/18/candidateDB_35 /content/mmseqs2/genexa/genexaDB /content/mmseqs2/tmp/18/resultsDB_35 /content/mmseqs2/tmp/18 \n","\n","MMseqs Version:                     \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Substitution matrix                 \tnucl:nucleotide.out,aa:blosum62.out\n","Add backtrace                       \tfalse\n","Alignment mode                      \t2\n","Allow wrapped scoring               \tfalse\n","E-value threshold                   \t0.001\n","Seq. id. threshold                  \t0\n","Min alignment length                \t0\n","Seq. id. mode                       \t0\n","Alternative alignments              \t0\n","Coverage threshold                  \t0\n","Coverage mode                       \t0\n","Max sequence length                 \t65535\n","Compositional bias                  \t1\n","Realign hits                        \tfalse\n","Max reject                          \t2147483647\n","Max accept                          \t2147483647\n","Include identical seq. id.          \tfalse\n","Preload mode                        \t0\n","Pseudo count a                      \t1\n","Pseudo count b                      \t1.5\n","Score bias                          \t0\n","Gap open cost                       \tnucl:5,aa:11\n","Gap extension cost                  \tnucl:2,aa:1\n","Zdrop                               \t40\n","Threads                             \t4\n","Compressed                          \t0\n","Verbosity                           \t3\n","Seed substitution matrix            \tnucl:nucleotide.out,aa:VTML80.out\n","Sensitivity                         \t7\n","k-mer length                        \t0\n","k-score                             \t2147483647\n","Alphabet size                       \tnucl:5,aa:21\n","Max results per query               \t300\n","Split database                      \t0\n","Split mode                          \t2\n","Split memory limit                  \t0\n","Diagonal scoring                    \ttrue\n","Exact k-mer matching                \t0\n","Mask residues                       \t1\n","Mask lower case residues            \t0\n","Minimum diagonal score              \t15\n","Spaced k-mers                       \t1\n","Spaced k-mer pattern                \t\n","Local temporary path                \t\n","Rescore mode                        \t0\n","Remove hits by seq. id. and coverage\tfalse\n","Sort results                        \t0\n","Mask profile                        \t1\n","Profile e-value threshold           \t0.1\n","Global sequence weighting           \tfalse\n","Allow deletions                     \tfalse\n","Filter MSA                          \t1\n","Maximum seq. id. threshold          \t0.9\n","Minimum seq. id.                    \t0\n","Minimum score per column            \t-20\n","Minimum coverage                    \t0\n","Select N most diverse seqs          \t1000\n","Omit consensus                      \tfalse\n","Min codons in orf                   \t30\n","Max codons in length                \t32734\n","Max orf gaps                        \t2147483647\n","Contig start mode                   \t2\n","Contig end mode                     \t2\n","Orf start mode                      \t1\n","Forward frames                      \t1,2,3\n","Reverse frames                      \t1,2,3\n","Translation table                   \t1\n","Translate orf                       \t0\n","Use all table starts                \tfalse\n","Offset of numeric ids               \t0\n","Create lookup                       \t0\n","Add orf stop                        \tfalse\n","Chain overlapping alignments        \t0\n","Merge query                         \t1\n","Search type                         \t0\n","Search iterations                   \t1\n","Start sensitivity                   \t4\n","Search steps                        \t1\n","Slice search mode                   \tfalse\n","Strand selection                    \t1\n","Disk space limit                    \t0\n","MPI runner                          \t\n","Force restart with latest tmp       \tfalse\n","Remove temporary files              \tfalse\n","\n","prefilter /content/mmseqs2/tmp/18/candidateDB_35 /content/mmseqs2/genexa/genexaDB.idx /content/mmseqs2/tmp/18/2071490700547038394/pref_0 --sub-mat nucl:nucleotide.out,aa:blosum62.out --seed-sub-mat nucl:nucleotide.out,aa:VTML80.out -k 0 --k-score 2147483647 --alph-size nucl:5,aa:21 --max-seq-len 65535 --max-seqs 300 --split 0 --split-mode 2 --split-memory-limit 0 -c 0 --cov-mode 0 --comp-bias-corr 1 --diag-score 1 --exact-kmer-matching 0 --mask 1 --mask-lower-case 0 --min-ungapped-score 15 --add-self-matches 0 --spaced-kmer-mode 1 --db-load-mode 0 --pca 1 --pcb 1.5 --threads 4 --compressed 0 -v 3 -s 7.0 \n","\n","Index version: 16\n","Generated by:  16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","ScoreMatrix:  VTML80.out\n","Query database size: 3152 type: Aminoacid\n","Estimated memory consumption: 2G\n","Target database size: 468351 type: Aminoacid\n","Process prefiltering step 1 of 1\n","\n","k-mer similarity threshold: 100\n","Starting prefiltering scores calculation (step 1 of 1)\n","Query db start 1 to 3152\n","Target db start 1 to 468351\n","[=================================================================] 100.00% 3.15K 26s 53ms\n","\n","962.921177 k-mers per position\n","639842 DB matches per sequence\n","127 overflows\n","0 queries produce too many hits (truncated result)\n","276 sequences passed prefiltering per query sequence\n","300 median result list length\n","2 sequences with 0 size result lists\n","Time for merging to pref_0: 0h 0m 0s 1ms\n","Time for processing: 0h 0m 27s 506ms\n","align /content/mmseqs2/tmp/18/candidateDB_35 /content/mmseqs2/genexa/genexaDB.idx /content/mmseqs2/tmp/18/2071490700547038394/pref_0 /content/mmseqs2/tmp/18/resultsDB_35 --sub-mat nucl:nucleotide.out,aa:blosum62.out -a 0 --alignment-mode 2 --wrapped-scoring 0 -e 0.001 --min-seq-id 0 --min-aln-len 0 --seq-id-mode 0 --alt-ali 0 -c 0 --cov-mode 0 --max-seq-len 65535 --comp-bias-corr 1 --realign 0 --max-rejected 2147483647 --max-accept 2147483647 --add-self-matches 0 --db-load-mode 0 --pca 1 --pcb 1.5 --score-bias 0 --gap-open nucl:5,aa:11 --gap-extend nucl:2,aa:1 --zdrop 40 --threads 4 --compressed 0 -v 3 \n","\n","Index version: 16\n","Generated by:  16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","ScoreMatrix:  VTML80.out\n","Compute score and coverage\n","Query database size: 3152 type: Aminoacid\n","Target database size: 468351 type: Aminoacid\n","Calculation of alignments\n","[=================================================================] 100.00% 3.15K 20s 834ms\n","Time for merging to resultsDB_35: 0h 0m 0s 1ms\n","\n","870226 alignments calculated.\n","246892 sequence pairs passed the thresholds (0.283710 of overall calculated).\n","78.328682 hits per query sequence.\n","Time for processing: 0h 0m 20s 861ms\n","convertalis /content/mmseqs2/tmp/18/candidateDB_35 /content/mmseqs2/genexa/genexaDB /content/mmseqs2/tmp/18/resultsDB_35 /content/mmseqs2/tmp/18/resultDB_genexa.m8 --format-output query,target,evalue,raw \n","\n","MMseqs Version:        \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Substitution matrix    \tnucl:nucleotide.out,aa:blosum62.out\n","Alignment format       \t0\n","Format alignment output\tquery,target,evalue,raw\n","Translation table      \t1\n","Gap open cost          \tnucl:5,aa:11\n","Gap extension cost     \tnucl:2,aa:1\n","Database output        \tfalse\n","Preload mode           \t0\n","Search type            \t0\n","Threads                \t4\n","Compressed             \t0\n","Verbosity              \t3\n","\n","[=================================================================] 100.00% 3.15K 0s 179ms\n","Time for merging to resultDB_genexa.m8: 0h 0m 0s 7ms\n","Time for processing: 0h 0m 0s 256ms\n","search -s 7.0 /content/mmseqs2/tmp/18/candidateDB_53 /content/mmseqs2/swissprot/swissprotDB /content/mmseqs2/tmp/18/resultsDB_53 /content/mmseqs2/tmp/18 \n","\n","MMseqs Version:                     \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Substitution matrix                 \tnucl:nucleotide.out,aa:blosum62.out\n","Add backtrace                       \tfalse\n","Alignment mode                      \t2\n","Allow wrapped scoring               \tfalse\n","E-value threshold                   \t0.001\n","Seq. id. threshold                  \t0\n","Min alignment length                \t0\n","Seq. id. mode                       \t0\n","Alternative alignments              \t0\n","Coverage threshold                  \t0\n","Coverage mode                       \t0\n","Max sequence length                 \t65535\n","Compositional bias                  \t1\n","Realign hits                        \tfalse\n","Max reject                          \t2147483647\n","Max accept                          \t2147483647\n","Include identical seq. id.          \tfalse\n","Preload mode                        \t0\n","Pseudo count a                      \t1\n","Pseudo count b                      \t1.5\n","Score bias                          \t0\n","Gap open cost                       \tnucl:5,aa:11\n","Gap extension cost                  \tnucl:2,aa:1\n","Zdrop                               \t40\n","Threads                             \t4\n","Compressed                          \t0\n","Verbosity                           \t3\n","Seed substitution matrix            \tnucl:nucleotide.out,aa:VTML80.out\n","Sensitivity                         \t7\n","k-mer length                        \t0\n","k-score                             \t2147483647\n","Alphabet size                       \tnucl:5,aa:21\n","Max results per query               \t300\n","Split database                      \t0\n","Split mode                          \t2\n","Split memory limit                  \t0\n","Diagonal scoring                    \ttrue\n","Exact k-mer matching                \t0\n","Mask residues                       \t1\n","Mask lower case residues            \t0\n","Minimum diagonal score              \t15\n","Spaced k-mers                       \t1\n","Spaced k-mer pattern                \t\n","Local temporary path                \t\n","Rescore mode                        \t0\n","Remove hits by seq. id. and coverage\tfalse\n","Sort results                        \t0\n","Mask profile                        \t1\n","Profile e-value threshold           \t0.1\n","Global sequence weighting           \tfalse\n","Allow deletions                     \tfalse\n","Filter MSA                          \t1\n","Maximum seq. id. threshold          \t0.9\n","Minimum seq. id.                    \t0\n","Minimum score per column            \t-20\n","Minimum coverage                    \t0\n","Select N most diverse seqs          \t1000\n","Omit consensus                      \tfalse\n","Min codons in orf                   \t30\n","Max codons in length                \t32734\n","Max orf gaps                        \t2147483647\n","Contig start mode                   \t2\n","Contig end mode                     \t2\n","Orf start mode                      \t1\n","Forward frames                      \t1,2,3\n","Reverse frames                      \t1,2,3\n","Translation table                   \t1\n","Translate orf                       \t0\n","Use all table starts                \tfalse\n","Offset of numeric ids               \t0\n","Create lookup                       \t0\n","Add orf stop                        \tfalse\n","Chain overlapping alignments        \t0\n","Merge query                         \t1\n","Search type                         \t0\n","Search iterations                   \t1\n","Start sensitivity                   \t4\n","Search steps                        \t1\n","Slice search mode                   \tfalse\n","Strand selection                    \t1\n","Disk space limit                    \t0\n","MPI runner                          \t\n","Force restart with latest tmp       \tfalse\n","Remove temporary files              \tfalse\n","\n","prefilter /content/mmseqs2/tmp/18/candidateDB_53 /content/mmseqs2/swissprot/swissprotDB /content/mmseqs2/tmp/18/5023143105939109848/pref_0 --sub-mat nucl:nucleotide.out,aa:blosum62.out --seed-sub-mat nucl:nucleotide.out,aa:VTML80.out -k 0 --k-score 2147483647 --alph-size nucl:5,aa:21 --max-seq-len 65535 --max-seqs 300 --split 0 --split-mode 2 --split-memory-limit 0 -c 0 --cov-mode 0 --comp-bias-corr 1 --diag-score 1 --exact-kmer-matching 0 --mask 1 --mask-lower-case 0 --min-ungapped-score 15 --add-self-matches 0 --spaced-kmer-mode 1 --db-load-mode 0 --pca 1 --pcb 1.5 --threads 4 --compressed 0 -v 3 -s 7.0 \n","\n","Query database size: 3152 type: Aminoacid\n","Estimated memory consumption: 2G\n","Target database size: 562755 type: Aminoacid\n","Index table k-mer threshold: 100 at k-mer size 6 \n","Index table: counting k-mers\n","[=================================================================] 100.00% 562.76K 24s 569ms\n","Index table: Masked residues: 3164874\n","Index table: fill\n","[=================================================================] 100.00% 562.76K 17s 918ms\n","Index statistics\n","Entries:          192757622\n","DB size:          1591 MB\n","Avg k-mer size:   3.011838\n","Top 10 k-mers\n","    GPGGTL\t1835\n","    GQSWTV\t1706\n","    WGMFAT\t1638\n","    PGVFEV\t1634\n","    VLWQFW\t1623\n","    AYIRPN\t1587\n","    RSPKGV\t1585\n","    TPHKWY\t1560\n","    HIRVGW\t1548\n","    ITLSPY\t1541\n","Time for index table init: 0h 0m 44s 537ms\n","Process prefiltering step 1 of 1\n","\n","k-mer similarity threshold: 100\n","Starting prefiltering scores calculation (step 1 of 1)\n","Query db start 1 to 3152\n","Target db start 1 to 562755\n","[=================================================================] 100.00% 3.15K 29s 930ms\n","\n","962.386850 k-mers per position\n","793392 DB matches per sequence\n","244 overflows\n","0 queries produce too many hits (truncated result)\n","283 sequences passed prefiltering per query sequence\n","300 median result list length\n","2 sequences with 0 size result lists\n","Time for merging to pref_0: 0h 0m 0s 1ms\n","Time for processing: 0h 1m 15s 631ms\n","align /content/mmseqs2/tmp/18/candidateDB_53 /content/mmseqs2/swissprot/swissprotDB /content/mmseqs2/tmp/18/5023143105939109848/pref_0 /content/mmseqs2/tmp/18/resultsDB_53 --sub-mat nucl:nucleotide.out,aa:blosum62.out -a 0 --alignment-mode 2 --wrapped-scoring 0 -e 0.001 --min-seq-id 0 --min-aln-len 0 --seq-id-mode 0 --alt-ali 0 -c 0 --cov-mode 0 --max-seq-len 65535 --comp-bias-corr 1 --realign 0 --max-rejected 2147483647 --max-accept 2147483647 --add-self-matches 0 --db-load-mode 0 --pca 1 --pcb 1.5 --score-bias 0 --gap-open nucl:5,aa:11 --gap-extend nucl:2,aa:1 --zdrop 40 --threads 4 --compressed 0 -v 3 \n","\n","Compute score and coverage\n","Query database size: 3152 type: Aminoacid\n","Target database size: 562755 type: Aminoacid\n","Calculation of alignments\n","[=================================================================] 100.00% 3.15K 20s 364ms\n","Time for merging to resultsDB_53: 0h 0m 0s 1ms\n","\n","894507 alignments calculated.\n","187760 sequence pairs passed the thresholds (0.209903 of overall calculated).\n","59.568527 hits per query sequence.\n","Time for processing: 0h 0m 20s 427ms\n","convertalis /content/mmseqs2/tmp/18/candidateDB_53 /content/mmseqs2/swissprot/swissprotDB /content/mmseqs2/tmp/18/resultsDB_53 /content/mmseqs2/tmp/18/resultDB_secondary.m8 --format-output query,target,evalue,raw \n","\n","MMseqs Version:        \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Substitution matrix    \tnucl:nucleotide.out,aa:blosum62.out\n","Alignment format       \t0\n","Format alignment output\tquery,target,evalue,raw\n","Translation table      \t1\n","Gap open cost          \tnucl:5,aa:11\n","Gap extension cost     \tnucl:2,aa:1\n","Database output        \tfalse\n","Preload mode           \t0\n","Search type            \t0\n","Threads                \t4\n","Compressed             \t0\n","Verbosity              \t3\n","\n","[=================================================================] 100.00% 3.15K 0s 132ms\n","Time for merging to resultDB_secondary.m8: 0h 0m 0s 5ms\n","Time for processing: 0h 0m 0s 226ms\n","found 2013 genes\n","\n","\n","Reading fasta file GCF_000814265.1_ASM81426v1_genomic.fna.gz...\n","\n","Finding and translating open reading frames...\n","\n","Encoding amino acids...\n","\n","Applying kmer prefilter...\n","\n","Scoring ORFs with temporal convolutional network...\n","\n"],"name":"stdout"},{"output_type":"display_data","data":{"application/vnd.jupyter.widget-view+json":{"model_id":"7ff6a0f775b04308bdd6d3b4fce51764","version_minor":0,"version_major":2},"text/plain":["HBox(children=(FloatProgress(value=0.0, max=17.0), HTML(value='')))"]},"metadata":{"tags":[]}},{"output_type":"stream","text":["\n","Scoring translation initiation sites...\n","\n"],"name":"stdout"},{"output_type":"display_data","data":{"application/vnd.jupyter.widget-view+json":{"model_id":"ba1e0ca3afb34fa6a1f6a024745b5dc0","version_minor":0,"version_major":2},"text/plain":["HBox(children=(FloatProgress(value=0.0, max=5.0), HTML(value='')))"]},"metadata":{"tags":[]}},{"output_type":"stream","text":["\n","Creating weighted directed acyclic graph of all ORFs and subORFs on contig 0...\n","\n"],"name":"stdout"},{"output_type":"display_data","data":{"application/vnd.jupyter.widget-view+json":{"model_id":"3dc805dc47c643e19f9b9d222f7a7f0d","version_minor":0,"version_major":2},"text/plain":["HBox(children=(FloatProgress(value=0.0, max=49809.0), HTML(value='')))"]},"metadata":{"tags":[]}},{"output_type":"stream","text":["\n","Maximizing geneiness...\n","\n","Filtering predictions with mmseqs2...\n","createdb /content/mmseqs2/tmp/19/candidate_genes_35.fasta /content/mmseqs2/tmp/19/candidateDB_35 \n","\n","MMseqs Version:       \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Database type         \t0\n","Shuffle input database\ttrue\n","Createdb mode         \t0\n","Write lookup file     \t1\n","Offset of numeric ids \t0\n","Compressed            \t0\n","Verbosity             \t3\n","\n","Converting sequences\n","[2324] 0s 4ms\n","Time for merging to candidateDB_35_h: 0h 0m 0s 3ms\n","Time for merging to candidateDB_35: 0h 0m 0s 3ms\n","Database type: Aminoacid\n","Time for processing: 0h 0m 0s 15ms\n","createdb /content/mmseqs2/tmp/19/candidate_genes_53.fasta /content/mmseqs2/tmp/19/candidateDB_53 \n","\n","MMseqs Version:       \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Database type         \t0\n","Shuffle input database\ttrue\n","Createdb mode         \t0\n","Write lookup file     \t1\n","Offset of numeric ids \t0\n","Compressed            \t0\n","Verbosity             \t3\n","\n","Converting sequences\n","[2324] 0s 5ms\n","Time for merging to candidateDB_53_h: 0h 0m 0s 3ms\n","Time for merging to candidateDB_53: 0h 0m 0s 4ms\n","Database type: Aminoacid\n","Time for processing: 0h 0m 0s 17ms\n","search -s 7.0 /content/mmseqs2/tmp/19/candidateDB_35 /content/mmseqs2/genexa/genexaDB /content/mmseqs2/tmp/19/resultsDB_35 /content/mmseqs2/tmp/19 \n","\n","MMseqs Version:                     \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Substitution matrix                 \tnucl:nucleotide.out,aa:blosum62.out\n","Add backtrace                       \tfalse\n","Alignment mode                      \t2\n","Allow wrapped scoring               \tfalse\n","E-value threshold                   \t0.001\n","Seq. id. threshold                  \t0\n","Min alignment length                \t0\n","Seq. id. mode                       \t0\n","Alternative alignments              \t0\n","Coverage threshold                  \t0\n","Coverage mode                       \t0\n","Max sequence length                 \t65535\n","Compositional bias                  \t1\n","Realign hits                        \tfalse\n","Max reject                          \t2147483647\n","Max accept                          \t2147483647\n","Include identical seq. id.          \tfalse\n","Preload mode                        \t0\n","Pseudo count a                      \t1\n","Pseudo count b                      \t1.5\n","Score bias                          \t0\n","Gap open cost                       \tnucl:5,aa:11\n","Gap extension cost                  \tnucl:2,aa:1\n","Zdrop                               \t40\n","Threads                             \t4\n","Compressed                          \t0\n","Verbosity                           \t3\n","Seed substitution matrix            \tnucl:nucleotide.out,aa:VTML80.out\n","Sensitivity                         \t7\n","k-mer length                        \t0\n","k-score                             \t2147483647\n","Alphabet size                       \tnucl:5,aa:21\n","Max results per query               \t300\n","Split database                      \t0\n","Split mode                          \t2\n","Split memory limit                  \t0\n","Diagonal scoring                    \ttrue\n","Exact k-mer matching                \t0\n","Mask residues                       \t1\n","Mask lower case residues            \t0\n","Minimum diagonal score              \t15\n","Spaced k-mers                       \t1\n","Spaced k-mer pattern                \t\n","Local temporary path                \t\n","Rescore mode                        \t0\n","Remove hits by seq. id. and coverage\tfalse\n","Sort results                        \t0\n","Mask profile                        \t1\n","Profile e-value threshold           \t0.1\n","Global sequence weighting           \tfalse\n","Allow deletions                     \tfalse\n","Filter MSA                          \t1\n","Maximum seq. id. threshold          \t0.9\n","Minimum seq. id.                    \t0\n","Minimum score per column            \t-20\n","Minimum coverage                    \t0\n","Select N most diverse seqs          \t1000\n","Omit consensus                      \tfalse\n","Min codons in orf                   \t30\n","Max codons in length                \t32734\n","Max orf gaps                        \t2147483647\n","Contig start mode                   \t2\n","Contig end mode                     \t2\n","Orf start mode                      \t1\n","Forward frames                      \t1,2,3\n","Reverse frames                      \t1,2,3\n","Translation table                   \t1\n","Translate orf                       \t0\n","Use all table starts                \tfalse\n","Offset of numeric ids               \t0\n","Create lookup                       \t0\n","Add orf stop                        \tfalse\n","Chain overlapping alignments        \t0\n","Merge query                         \t1\n","Search type                         \t0\n","Search iterations                   \t1\n","Start sensitivity                   \t4\n","Search steps                        \t1\n","Slice search mode                   \tfalse\n","Strand selection                    \t1\n","Disk space limit                    \t0\n","MPI runner                          \t\n","Force restart with latest tmp       \tfalse\n","Remove temporary files              \tfalse\n","\n","prefilter /content/mmseqs2/tmp/19/candidateDB_35 /content/mmseqs2/genexa/genexaDB.idx /content/mmseqs2/tmp/19/15257948671146894521/pref_0 --sub-mat nucl:nucleotide.out,aa:blosum62.out --seed-sub-mat nucl:nucleotide.out,aa:VTML80.out -k 0 --k-score 2147483647 --alph-size nucl:5,aa:21 --max-seq-len 65535 --max-seqs 300 --split 0 --split-mode 2 --split-memory-limit 0 -c 0 --cov-mode 0 --comp-bias-corr 1 --diag-score 1 --exact-kmer-matching 0 --mask 1 --mask-lower-case 0 --min-ungapped-score 15 --add-self-matches 0 --spaced-kmer-mode 1 --db-load-mode 0 --pca 1 --pcb 1.5 --threads 4 --compressed 0 -v 3 -s 7.0 \n","\n","Index version: 16\n","Generated by:  16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","ScoreMatrix:  VTML80.out\n","Query database size: 2325 type: Aminoacid\n","Estimated memory consumption: 2G\n","Target database size: 468351 type: Aminoacid\n","Process prefiltering step 1 of 1\n","\n","k-mer similarity threshold: 100\n","Starting prefiltering scores calculation (step 1 of 1)\n","Query db start 1 to 2325\n","Target db start 1 to 468351\n","[=================================================================] 100.00% 2.33K 18s 131ms\n","\n","832.181558 k-mers per position\n","594121 DB matches per sequence\n","50 overflows\n","0 queries produce too many hits (truncated result)\n","263 sequences passed prefiltering per query sequence\n","300 median result list length\n","8 sequences with 0 size result lists\n","Time for merging to pref_0: 0h 0m 0s 1ms\n","Time for processing: 0h 0m 19s 588ms\n","align /content/mmseqs2/tmp/19/candidateDB_35 /content/mmseqs2/genexa/genexaDB.idx /content/mmseqs2/tmp/19/15257948671146894521/pref_0 /content/mmseqs2/tmp/19/resultsDB_35 --sub-mat nucl:nucleotide.out,aa:blosum62.out -a 0 --alignment-mode 2 --wrapped-scoring 0 -e 0.001 --min-seq-id 0 --min-aln-len 0 --seq-id-mode 0 --alt-ali 0 -c 0 --cov-mode 0 --max-seq-len 65535 --comp-bias-corr 1 --realign 0 --max-rejected 2147483647 --max-accept 2147483647 --add-self-matches 0 --db-load-mode 0 --pca 1 --pcb 1.5 --score-bias 0 --gap-open nucl:5,aa:11 --gap-extend nucl:2,aa:1 --zdrop 40 --threads 4 --compressed 0 -v 3 \n","\n","Index version: 16\n","Generated by:  16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","ScoreMatrix:  VTML80.out\n","Compute score and coverage\n","Query database size: 2325 type: Aminoacid\n","Target database size: 468351 type: Aminoacid\n","Calculation of alignments\n","[=================================================================] 100.00% 2.33K 16s 948ms\n","Time for merging to resultsDB_35: 0h 0m 0s 0ms\n","\n","613349 alignments calculated.\n","197569 sequence pairs passed the thresholds (0.322115 of overall calculated).\n","84.975914 hits per query sequence.\n","Time for processing: 0h 0m 17s 23ms\n","convertalis /content/mmseqs2/tmp/19/candidateDB_35 /content/mmseqs2/genexa/genexaDB /content/mmseqs2/tmp/19/resultsDB_35 /content/mmseqs2/tmp/19/resultDB_genexa.m8 --format-output query,target,evalue,raw \n","\n","MMseqs Version:        \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Substitution matrix    \tnucl:nucleotide.out,aa:blosum62.out\n","Alignment format       \t0\n","Format alignment output\tquery,target,evalue,raw\n","Translation table      \t1\n","Gap open cost          \tnucl:5,aa:11\n","Gap extension cost     \tnucl:2,aa:1\n","Database output        \tfalse\n","Preload mode           \t0\n","Search type            \t0\n","Threads                \t4\n","Compressed             \t0\n","Verbosity              \t3\n","\n","[=================================================================] 100.00% 2.33K 0s 148ms\n","Time for merging to resultDB_genexa.m8: 0h 0m 0s 6ms\n","Time for processing: 0h 0m 0s 222ms\n","search -s 7.0 /content/mmseqs2/tmp/19/candidateDB_53 /content/mmseqs2/swissprot/swissprotDB /content/mmseqs2/tmp/19/resultsDB_53 /content/mmseqs2/tmp/19 \n","\n","MMseqs Version:                     \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Substitution matrix                 \tnucl:nucleotide.out,aa:blosum62.out\n","Add backtrace                       \tfalse\n","Alignment mode                      \t2\n","Allow wrapped scoring               \tfalse\n","E-value threshold                   \t0.001\n","Seq. id. threshold                  \t0\n","Min alignment length                \t0\n","Seq. id. mode                       \t0\n","Alternative alignments              \t0\n","Coverage threshold                  \t0\n","Coverage mode                       \t0\n","Max sequence length                 \t65535\n","Compositional bias                  \t1\n","Realign hits                        \tfalse\n","Max reject                          \t2147483647\n","Max accept                          \t2147483647\n","Include identical seq. id.          \tfalse\n","Preload mode                        \t0\n","Pseudo count a                      \t1\n","Pseudo count b                      \t1.5\n","Score bias                          \t0\n","Gap open cost                       \tnucl:5,aa:11\n","Gap extension cost                  \tnucl:2,aa:1\n","Zdrop                               \t40\n","Threads                             \t4\n","Compressed                          \t0\n","Verbosity                           \t3\n","Seed substitution matrix            \tnucl:nucleotide.out,aa:VTML80.out\n","Sensitivity                         \t7\n","k-mer length                        \t0\n","k-score                             \t2147483647\n","Alphabet size                       \tnucl:5,aa:21\n","Max results per query               \t300\n","Split database                      \t0\n","Split mode                          \t2\n","Split memory limit                  \t0\n","Diagonal scoring                    \ttrue\n","Exact k-mer matching                \t0\n","Mask residues                       \t1\n","Mask lower case residues            \t0\n","Minimum diagonal score              \t15\n","Spaced k-mers                       \t1\n","Spaced k-mer pattern                \t\n","Local temporary path                \t\n","Rescore mode                        \t0\n","Remove hits by seq. id. and coverage\tfalse\n","Sort results                        \t0\n","Mask profile                        \t1\n","Profile e-value threshold           \t0.1\n","Global sequence weighting           \tfalse\n","Allow deletions                     \tfalse\n","Filter MSA                          \t1\n","Maximum seq. id. threshold          \t0.9\n","Minimum seq. id.                    \t0\n","Minimum score per column            \t-20\n","Minimum coverage                    \t0\n","Select N most diverse seqs          \t1000\n","Omit consensus                      \tfalse\n","Min codons in orf                   \t30\n","Max codons in length                \t32734\n","Max orf gaps                        \t2147483647\n","Contig start mode                   \t2\n","Contig end mode                     \t2\n","Orf start mode                      \t1\n","Forward frames                      \t1,2,3\n","Reverse frames                      \t1,2,3\n","Translation table                   \t1\n","Translate orf                       \t0\n","Use all table starts                \tfalse\n","Offset of numeric ids               \t0\n","Create lookup                       \t0\n","Add orf stop                        \tfalse\n","Chain overlapping alignments        \t0\n","Merge query                         \t1\n","Search type                         \t0\n","Search iterations                   \t1\n","Start sensitivity                   \t4\n","Search steps                        \t1\n","Slice search mode                   \tfalse\n","Strand selection                    \t1\n","Disk space limit                    \t0\n","MPI runner                          \t\n","Force restart with latest tmp       \tfalse\n","Remove temporary files              \tfalse\n","\n","prefilter /content/mmseqs2/tmp/19/candidateDB_53 /content/mmseqs2/swissprot/swissprotDB /content/mmseqs2/tmp/19/5124508093984455831/pref_0 --sub-mat nucl:nucleotide.out,aa:blosum62.out --seed-sub-mat nucl:nucleotide.out,aa:VTML80.out -k 0 --k-score 2147483647 --alph-size nucl:5,aa:21 --max-seq-len 65535 --max-seqs 300 --split 0 --split-mode 2 --split-memory-limit 0 -c 0 --cov-mode 0 --comp-bias-corr 1 --diag-score 1 --exact-kmer-matching 0 --mask 1 --mask-lower-case 0 --min-ungapped-score 15 --add-self-matches 0 --spaced-kmer-mode 1 --db-load-mode 0 --pca 1 --pcb 1.5 --threads 4 --compressed 0 -v 3 -s 7.0 \n","\n","Query database size: 2325 type: Aminoacid\n","Estimated memory consumption: 2G\n","Target database size: 562755 type: Aminoacid\n","Index table k-mer threshold: 100 at k-mer size 6 \n","Index table: counting k-mers\n","[=================================================================] 100.00% 562.76K 24s 598ms\n","Index table: Masked residues: 3164874\n","Index table: fill\n","[=================================================================] 100.00% 562.76K 17s 929ms\n","Index statistics\n","Entries:          192757622\n","DB size:          1591 MB\n","Avg k-mer size:   3.011838\n","Top 10 k-mers\n","    GPGGTL\t1835\n","    GQSWTV\t1706\n","    WGMFAT\t1638\n","    PGVFEV\t1634\n","    VLWQFW\t1623\n","    AYIRPN\t1587\n","    RSPKGV\t1585\n","    TPHKWY\t1560\n","    HIRVGW\t1548\n","    ITLSPY\t1541\n","Time for index table init: 0h 0m 44s 587ms\n","Process prefiltering step 1 of 1\n","\n","k-mer similarity threshold: 100\n","Starting prefiltering scores calculation (step 1 of 1)\n","Query db start 1 to 2325\n","Target db start 1 to 562755\n","[=================================================================] 100.00% 2.33K 21s 71ms\n","\n","832.502546 k-mers per position\n","760798 DB matches per sequence\n","142 overflows\n","0 queries produce too many hits (truncated result)\n","272 sequences passed prefiltering per query sequence\n","300 median result list length\n","6 sequences with 0 size result lists\n","Time for merging to pref_0: 0h 0m 0s 0ms\n","Time for processing: 0h 1m 6s 821ms\n","align /content/mmseqs2/tmp/19/candidateDB_53 /content/mmseqs2/swissprot/swissprotDB /content/mmseqs2/tmp/19/5124508093984455831/pref_0 /content/mmseqs2/tmp/19/resultsDB_53 --sub-mat nucl:nucleotide.out,aa:blosum62.out -a 0 --alignment-mode 2 --wrapped-scoring 0 -e 0.001 --min-seq-id 0 --min-aln-len 0 --seq-id-mode 0 --alt-ali 0 -c 0 --cov-mode 0 --max-seq-len 65535 --comp-bias-corr 1 --realign 0 --max-rejected 2147483647 --max-accept 2147483647 --add-self-matches 0 --db-load-mode 0 --pca 1 --pcb 1.5 --score-bias 0 --gap-open nucl:5,aa:11 --gap-extend nucl:2,aa:1 --zdrop 40 --threads 4 --compressed 0 -v 3 \n","\n","Compute score and coverage\n","Query database size: 2325 type: Aminoacid\n","Target database size: 562755 type: Aminoacid\n","Calculation of alignments\n","[=================================================================] 100.00% 2.33K 18s 784ms\n","Time for merging to resultsDB_53: 0h 0m 0s 0ms\n","\n","632726 alignments calculated.\n","171401 sequence pairs passed the thresholds (0.270893 of overall calculated).\n","73.720863 hits per query sequence.\n","Time for processing: 0h 0m 18s 852ms\n","convertalis /content/mmseqs2/tmp/19/candidateDB_53 /content/mmseqs2/swissprot/swissprotDB /content/mmseqs2/tmp/19/resultsDB_53 /content/mmseqs2/tmp/19/resultDB_secondary.m8 --format-output query,target,evalue,raw \n","\n","MMseqs Version:        \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Substitution matrix    \tnucl:nucleotide.out,aa:blosum62.out\n","Alignment format       \t0\n","Format alignment output\tquery,target,evalue,raw\n","Translation table      \t1\n","Gap open cost          \tnucl:5,aa:11\n","Gap extension cost     \tnucl:2,aa:1\n","Database output        \tfalse\n","Preload mode           \t0\n","Search type            \t0\n","Threads                \t4\n","Compressed             \t0\n","Verbosity              \t3\n","\n","[=================================================================] 100.00% 2.33K 0s 120ms\n","Time for merging to resultDB_secondary.m8: 0h 0m 0s 6ms\n","Time for processing: 0h 0m 0s 218ms\n","found 1693 genes\n","\n","\n","Reading fasta file GCF_000737325.1_ASM73732v2_genomic.fna.gz...\n","\n","Finding and translating open reading frames...\n","\n","Encoding amino acids...\n","\n","Applying kmer prefilter...\n","\n","Scoring ORFs with temporal convolutional network...\n","\n"],"name":"stdout"},{"output_type":"display_data","data":{"application/vnd.jupyter.widget-view+json":{"model_id":"3811d8b5a11c46adb189f822412a4872","version_minor":0,"version_major":2},"text/plain":["HBox(children=(FloatProgress(value=0.0, max=170.0), HTML(value='')))"]},"metadata":{"tags":[]}},{"output_type":"stream","text":["\n","Scoring translation initiation sites...\n","\n"],"name":"stdout"},{"output_type":"display_data","data":{"application/vnd.jupyter.widget-view+json":{"model_id":"a148a06e19654464a603b25d760833c0","version_minor":0,"version_major":2},"text/plain":["HBox(children=(FloatProgress(value=0.0, max=51.0), HTML(value='')))"]},"metadata":{"tags":[]}},{"output_type":"stream","text":["\n","Creating weighted directed acyclic graph of all ORFs and subORFs on contig 0...\n","\n"],"name":"stdout"},{"output_type":"display_data","data":{"application/vnd.jupyter.widget-view+json":{"model_id":"c0e4c752431f4e3593dbd77e001eee62","version_minor":0,"version_major":2},"text/plain":["HBox(children=(FloatProgress(value=0.0, max=508244.0), HTML(value='')))"]},"metadata":{"tags":[]}},{"output_type":"stream","text":["\n","Maximizing geneiness...\n","\n","Filtering predictions with mmseqs2...\n","createdb /content/mmseqs2/tmp/20/candidate_genes_35.fasta /content/mmseqs2/tmp/20/candidateDB_35 \n","\n","MMseqs Version:       \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Database type         \t0\n","Shuffle input database\ttrue\n","Createdb mode         \t0\n","Write lookup file     \t1\n","Offset of numeric ids \t0\n","Compressed            \t0\n","Verbosity             \t3\n","\n","Converting sequences\n","[11616] 0s 14ms\n","Time for merging to candidateDB_35_h: 0h 0m 0s 4ms\n","Time for merging to candidateDB_35: 0h 0m 0s 13ms\n","Database type: Aminoacid\n","Time for processing: 0h 0m 0s 48ms\n","createdb /content/mmseqs2/tmp/20/candidate_genes_53.fasta /content/mmseqs2/tmp/20/candidateDB_53 \n","\n","MMseqs Version:       \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Database type         \t0\n","Shuffle input database\ttrue\n","Createdb mode         \t0\n","Write lookup file     \t1\n","Offset of numeric ids \t0\n","Compressed            \t0\n","Verbosity             \t3\n","\n","Converting sequences\n","[11616] 0s 14ms\n","Time for merging to candidateDB_53_h: 0h 0m 0s 5ms\n","Time for merging to candidateDB_53: 0h 0m 0s 8ms\n","Database type: Aminoacid\n","Time for processing: 0h 0m 0s 42ms\n","search -s 7.0 /content/mmseqs2/tmp/20/candidateDB_35 /content/mmseqs2/genexa/genexaDB /content/mmseqs2/tmp/20/resultsDB_35 /content/mmseqs2/tmp/20 \n","\n","MMseqs Version:                     \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Substitution matrix                 \tnucl:nucleotide.out,aa:blosum62.out\n","Add backtrace                       \tfalse\n","Alignment mode                      \t2\n","Allow wrapped scoring               \tfalse\n","E-value threshold                   \t0.001\n","Seq. id. threshold                  \t0\n","Min alignment length                \t0\n","Seq. id. mode                       \t0\n","Alternative alignments              \t0\n","Coverage threshold                  \t0\n","Coverage mode                       \t0\n","Max sequence length                 \t65535\n","Compositional bias                  \t1\n","Realign hits                        \tfalse\n","Max reject                          \t2147483647\n","Max accept                          \t2147483647\n","Include identical seq. id.          \tfalse\n","Preload mode                        \t0\n","Pseudo count a                      \t1\n","Pseudo count b                      \t1.5\n","Score bias                          \t0\n","Gap open cost                       \tnucl:5,aa:11\n","Gap extension cost                  \tnucl:2,aa:1\n","Zdrop                               \t40\n","Threads                             \t4\n","Compressed                          \t0\n","Verbosity                           \t3\n","Seed substitution matrix            \tnucl:nucleotide.out,aa:VTML80.out\n","Sensitivity                         \t7\n","k-mer length                        \t0\n","k-score                             \t2147483647\n","Alphabet size                       \tnucl:5,aa:21\n","Max results per query               \t300\n","Split database                      \t0\n","Split mode                          \t2\n","Split memory limit                  \t0\n","Diagonal scoring                    \ttrue\n","Exact k-mer matching                \t0\n","Mask residues                       \t1\n","Mask lower case residues            \t0\n","Minimum diagonal score              \t15\n","Spaced k-mers                       \t1\n","Spaced k-mer pattern                \t\n","Local temporary path                \t\n","Rescore mode                        \t0\n","Remove hits by seq. id. and coverage\tfalse\n","Sort results                        \t0\n","Mask profile                        \t1\n","Profile e-value threshold           \t0.1\n","Global sequence weighting           \tfalse\n","Allow deletions                     \tfalse\n","Filter MSA                          \t1\n","Maximum seq. id. threshold          \t0.9\n","Minimum seq. id.                    \t0\n","Minimum score per column            \t-20\n","Minimum coverage                    \t0\n","Select N most diverse seqs          \t1000\n","Omit consensus                      \tfalse\n","Min codons in orf                   \t30\n","Max codons in length                \t32734\n","Max orf gaps                        \t2147483647\n","Contig start mode                   \t2\n","Contig end mode                     \t2\n","Orf start mode                      \t1\n","Forward frames                      \t1,2,3\n","Reverse frames                      \t1,2,3\n","Translation table                   \t1\n","Translate orf                       \t0\n","Use all table starts                \tfalse\n","Offset of numeric ids               \t0\n","Create lookup                       \t0\n","Add orf stop                        \tfalse\n","Chain overlapping alignments        \t0\n","Merge query                         \t1\n","Search type                         \t0\n","Search iterations                   \t1\n","Start sensitivity                   \t4\n","Search steps                        \t1\n","Slice search mode                   \tfalse\n","Strand selection                    \t1\n","Disk space limit                    \t0\n","MPI runner                          \t\n","Force restart with latest tmp       \tfalse\n","Remove temporary files              \tfalse\n","\n","prefilter /content/mmseqs2/tmp/20/candidateDB_35 /content/mmseqs2/genexa/genexaDB.idx /content/mmseqs2/tmp/20/10212118844990903459/pref_0 --sub-mat nucl:nucleotide.out,aa:blosum62.out --seed-sub-mat nucl:nucleotide.out,aa:VTML80.out -k 0 --k-score 2147483647 --alph-size nucl:5,aa:21 --max-seq-len 65535 --max-seqs 300 --split 0 --split-mode 2 --split-memory-limit 0 -c 0 --cov-mode 0 --comp-bias-corr 1 --diag-score 1 --exact-kmer-matching 0 --mask 1 --mask-lower-case 0 --min-ungapped-score 15 --add-self-matches 0 --spaced-kmer-mode 1 --db-load-mode 0 --pca 1 --pcb 1.5 --threads 4 --compressed 0 -v 3 -s 7.0 \n","\n","Index version: 16\n","Generated by:  16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","ScoreMatrix:  VTML80.out\n","Query database size: 11633 type: Aminoacid\n","Estimated memory consumption: 2G\n","Target database size: 468351 type: Aminoacid\n","Process prefiltering step 1 of 1\n","\n","k-mer similarity threshold: 100\n","Starting prefiltering scores calculation (step 1 of 1)\n","Query db start 1 to 11633\n","Target db start 1 to 468351\n","[=================================================================] 100.00% 11.63K 1m 39s 813ms\n","\n","713.209439 k-mers per position\n","729753 DB matches per sequence\n","624 overflows\n","0 queries produce too many hits (truncated result)\n","268 sequences passed prefiltering per query sequence\n","300 median result list length\n","25 sequences with 0 size result lists\n","Time for merging to pref_0: 0h 0m 0s 3ms\n","Time for processing: 0h 1m 41s 326ms\n","align /content/mmseqs2/tmp/20/candidateDB_35 /content/mmseqs2/genexa/genexaDB.idx /content/mmseqs2/tmp/20/10212118844990903459/pref_0 /content/mmseqs2/tmp/20/resultsDB_35 --sub-mat nucl:nucleotide.out,aa:blosum62.out -a 0 --alignment-mode 2 --wrapped-scoring 0 -e 0.001 --min-seq-id 0 --min-aln-len 0 --seq-id-mode 0 --alt-ali 0 -c 0 --cov-mode 0 --max-seq-len 65535 --comp-bias-corr 1 --realign 0 --max-rejected 2147483647 --max-accept 2147483647 --add-self-matches 0 --db-load-mode 0 --pca 1 --pcb 1.5 --score-bias 0 --gap-open nucl:5,aa:11 --gap-extend nucl:2,aa:1 --zdrop 40 --threads 4 --compressed 0 -v 3 \n","\n","Index version: 16\n","Generated by:  16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","ScoreMatrix:  VTML80.out\n","Compute score and coverage\n","Query database size: 11633 type: Aminoacid\n","Target database size: 468351 type: Aminoacid\n","Calculation of alignments\n","[=================================================================] 100.00% 11.63K 1m 9s 934ms\n","Time for merging to resultsDB_35: 0h 0m 0s 3ms\n","\n","3122079 alignments calculated.\n","690754 sequence pairs passed the thresholds (0.221248 of overall calculated).\n","59.378838 hits per query sequence.\n","Time for processing: 0h 1m 9s 980ms\n","convertalis /content/mmseqs2/tmp/20/candidateDB_35 /content/mmseqs2/genexa/genexaDB /content/mmseqs2/tmp/20/resultsDB_35 /content/mmseqs2/tmp/20/resultDB_genexa.m8 --format-output query,target,evalue,raw \n","\n","MMseqs Version:        \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Substitution matrix    \tnucl:nucleotide.out,aa:blosum62.out\n","Alignment format       \t0\n","Format alignment output\tquery,target,evalue,raw\n","Translation table      \t1\n","Gap open cost          \tnucl:5,aa:11\n","Gap extension cost     \tnucl:2,aa:1\n","Database output        \tfalse\n","Preload mode           \t0\n","Search type            \t0\n","Threads                \t4\n","Compressed             \t0\n","Verbosity              \t3\n","\n","[=================================================================] 100.00% 11.63K 0s 506ms\n","Time for merging to resultDB_genexa.m8: 0h 0m 0s 19ms\n","Time for processing: 0h 0m 0s 607ms\n","search -s 7.0 /content/mmseqs2/tmp/20/candidateDB_53 /content/mmseqs2/swissprot/swissprotDB /content/mmseqs2/tmp/20/resultsDB_53 /content/mmseqs2/tmp/20 \n","\n","MMseqs Version:                     \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Substitution matrix                 \tnucl:nucleotide.out,aa:blosum62.out\n","Add backtrace                       \tfalse\n","Alignment mode                      \t2\n","Allow wrapped scoring               \tfalse\n","E-value threshold                   \t0.001\n","Seq. id. threshold                  \t0\n","Min alignment length                \t0\n","Seq. id. mode                       \t0\n","Alternative alignments              \t0\n","Coverage threshold                  \t0\n","Coverage mode                       \t0\n","Max sequence length                 \t65535\n","Compositional bias                  \t1\n","Realign hits                        \tfalse\n","Max reject                          \t2147483647\n","Max accept                          \t2147483647\n","Include identical seq. id.          \tfalse\n","Preload mode                        \t0\n","Pseudo count a                      \t1\n","Pseudo count b                      \t1.5\n","Score bias                          \t0\n","Gap open cost                       \tnucl:5,aa:11\n","Gap extension cost                  \tnucl:2,aa:1\n","Zdrop                               \t40\n","Threads                             \t4\n","Compressed                          \t0\n","Verbosity                           \t3\n","Seed substitution matrix            \tnucl:nucleotide.out,aa:VTML80.out\n","Sensitivity                         \t7\n","k-mer length                        \t0\n","k-score                             \t2147483647\n","Alphabet size                       \tnucl:5,aa:21\n","Max results per query               \t300\n","Split database                      \t0\n","Split mode                          \t2\n","Split memory limit                  \t0\n","Diagonal scoring                    \ttrue\n","Exact k-mer matching                \t0\n","Mask residues                       \t1\n","Mask lower case residues            \t0\n","Minimum diagonal score              \t15\n","Spaced k-mers                       \t1\n","Spaced k-mer pattern                \t\n","Local temporary path                \t\n","Rescore mode                        \t0\n","Remove hits by seq. id. and coverage\tfalse\n","Sort results                        \t0\n","Mask profile                        \t1\n","Profile e-value threshold           \t0.1\n","Global sequence weighting           \tfalse\n","Allow deletions                     \tfalse\n","Filter MSA                          \t1\n","Maximum seq. id. threshold          \t0.9\n","Minimum seq. id.                    \t0\n","Minimum score per column            \t-20\n","Minimum coverage                    \t0\n","Select N most diverse seqs          \t1000\n","Omit consensus                      \tfalse\n","Min codons in orf                   \t30\n","Max codons in length                \t32734\n","Max orf gaps                        \t2147483647\n","Contig start mode                   \t2\n","Contig end mode                     \t2\n","Orf start mode                      \t1\n","Forward frames                      \t1,2,3\n","Reverse frames                      \t1,2,3\n","Translation table                   \t1\n","Translate orf                       \t0\n","Use all table starts                \tfalse\n","Offset of numeric ids               \t0\n","Create lookup                       \t0\n","Add orf stop                        \tfalse\n","Chain overlapping alignments        \t0\n","Merge query                         \t1\n","Search type                         \t0\n","Search iterations                   \t1\n","Start sensitivity                   \t4\n","Search steps                        \t1\n","Slice search mode                   \tfalse\n","Strand selection                    \t1\n","Disk space limit                    \t0\n","MPI runner                          \t\n","Force restart with latest tmp       \tfalse\n","Remove temporary files              \tfalse\n","\n","prefilter /content/mmseqs2/tmp/20/candidateDB_53 /content/mmseqs2/swissprot/swissprotDB /content/mmseqs2/tmp/20/7354537830982067457/pref_0 --sub-mat nucl:nucleotide.out,aa:blosum62.out --seed-sub-mat nucl:nucleotide.out,aa:VTML80.out -k 0 --k-score 2147483647 --alph-size nucl:5,aa:21 --max-seq-len 65535 --max-seqs 300 --split 0 --split-mode 2 --split-memory-limit 0 -c 0 --cov-mode 0 --comp-bias-corr 1 --diag-score 1 --exact-kmer-matching 0 --mask 1 --mask-lower-case 0 --min-ungapped-score 15 --add-self-matches 0 --spaced-kmer-mode 1 --db-load-mode 0 --pca 1 --pcb 1.5 --threads 4 --compressed 0 -v 3 -s 7.0 \n","\n","Query database size: 11633 type: Aminoacid\n","Estimated memory consumption: 2G\n","Target database size: 562755 type: Aminoacid\n","Index table k-mer threshold: 100 at k-mer size 6 \n","Index table: counting k-mers\n","[=================================================================] 100.00% 562.76K 24s 388ms\n","Index table: Masked residues: 3164874\n","Index table: fill\n","[=================================================================] 100.00% 562.76K 17s 885ms\n","Index statistics\n","Entries:          192757622\n","DB size:          1591 MB\n","Avg k-mer size:   3.011838\n","Top 10 k-mers\n","    GPGGTL\t1835\n","    GQSWTV\t1706\n","    WGMFAT\t1638\n","    PGVFEV\t1634\n","    VLWQFW\t1623\n","    AYIRPN\t1587\n","    RSPKGV\t1585\n","    TPHKWY\t1560\n","    HIRVGW\t1548\n","    ITLSPY\t1541\n","Time for index table init: 0h 0m 44s 394ms\n","Process prefiltering step 1 of 1\n","\n","k-mer similarity threshold: 100\n","Starting prefiltering scores calculation (step 1 of 1)\n","Query db start 1 to 11633\n","Target db start 1 to 562755\n","[=================================================================] 100.00% 11.63K 1m 48s 449ms\n","\n","712.896521 k-mers per position\n","807293 DB matches per sequence\n","886 overflows\n","0 queries produce too many hits (truncated result)\n","272 sequences passed prefiltering per query sequence\n","300 median result list length\n","41 sequences with 0 size result lists\n","Time for merging to pref_0: 0h 0m 0s 3ms\n","Time for processing: 0h 2m 34s 95ms\n","align /content/mmseqs2/tmp/20/candidateDB_53 /content/mmseqs2/swissprot/swissprotDB /content/mmseqs2/tmp/20/7354537830982067457/pref_0 /content/mmseqs2/tmp/20/resultsDB_53 --sub-mat nucl:nucleotide.out,aa:blosum62.out -a 0 --alignment-mode 2 --wrapped-scoring 0 -e 0.001 --min-seq-id 0 --min-aln-len 0 --seq-id-mode 0 --alt-ali 0 -c 0 --cov-mode 0 --max-seq-len 65535 --comp-bias-corr 1 --realign 0 --max-rejected 2147483647 --max-accept 2147483647 --add-self-matches 0 --db-load-mode 0 --pca 1 --pcb 1.5 --score-bias 0 --gap-open nucl:5,aa:11 --gap-extend nucl:2,aa:1 --zdrop 40 --threads 4 --compressed 0 -v 3 \n","\n","Compute score and coverage\n","Query database size: 11633 type: Aminoacid\n","Target database size: 562755 type: Aminoacid\n","Calculation of alignments\n","[=================================================================] 100.00% 11.63K 1m 2s 836ms\n","Time for merging to resultsDB_53: 0h 0m 0s 3ms\n","\n","3166096 alignments calculated.\n","486324 sequence pairs passed the thresholds (0.153604 of overall calculated).\n","41.805553 hits per query sequence.\n","Time for processing: 0h 1m 2s 933ms\n","convertalis /content/mmseqs2/tmp/20/candidateDB_53 /content/mmseqs2/swissprot/swissprotDB /content/mmseqs2/tmp/20/resultsDB_53 /content/mmseqs2/tmp/20/resultDB_secondary.m8 --format-output query,target,evalue,raw \n","\n","MMseqs Version:        \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Substitution matrix    \tnucl:nucleotide.out,aa:blosum62.out\n","Alignment format       \t0\n","Format alignment output\tquery,target,evalue,raw\n","Translation table      \t1\n","Gap open cost          \tnucl:5,aa:11\n","Gap extension cost     \tnucl:2,aa:1\n","Database output        \tfalse\n","Preload mode           \t0\n","Search type            \t0\n","Threads                \t4\n","Compressed             \t0\n","Verbosity              \t3\n","\n","[=================================================================] 100.00% 11.63K 0s 336ms\n","Time for merging to resultDB_secondary.m8: 0h 0m 0s 18ms\n","Time for processing: 0h 0m 0s 455ms\n","found 8399 genes\n","\n","\n","Reading fasta file GCF_000725345.1_ASM72534v1_genomic.fna.gz...\n","\n","Finding and translating open reading frames...\n","\n","Encoding amino acids...\n","\n","Applying kmer prefilter...\n","\n","Scoring ORFs with temporal convolutional network...\n","\n"],"name":"stdout"},{"output_type":"display_data","data":{"application/vnd.jupyter.widget-view+json":{"model_id":"7477b8c7e1f6469aa9c868870bff7fe1","version_minor":0,"version_major":2},"text/plain":["HBox(children=(FloatProgress(value=0.0, max=67.0), HTML(value='')))"]},"metadata":{"tags":[]}},{"output_type":"stream","text":["\n","Scoring translation initiation sites...\n","\n"],"name":"stdout"},{"output_type":"display_data","data":{"application/vnd.jupyter.widget-view+json":{"model_id":"8b88b8c9bcce4d47b17a69226d365f4f","version_minor":0,"version_major":2},"text/plain":["HBox(children=(FloatProgress(value=0.0, max=20.0), HTML(value='')))"]},"metadata":{"tags":[]}},{"output_type":"stream","text":["\n","Creating weighted directed acyclic graph of all ORFs and subORFs on contig 0...\n","\n"],"name":"stdout"},{"output_type":"display_data","data":{"application/vnd.jupyter.widget-view+json":{"model_id":"87030ead123f4fe699f7235fe8b401d2","version_minor":0,"version_major":2},"text/plain":["HBox(children=(FloatProgress(value=0.0, max=199251.0), HTML(value='')))"]},"metadata":{"tags":[]}},{"output_type":"stream","text":["\n","Maximizing geneiness...\n","\n","Filtering predictions with mmseqs2...\n","createdb /content/mmseqs2/tmp/21/candidate_genes_35.fasta /content/mmseqs2/tmp/21/candidateDB_35 \n","\n","MMseqs Version:       \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Database type         \t0\n","Shuffle input database\ttrue\n","Createdb mode         \t0\n","Write lookup file     \t1\n","Offset of numeric ids \t0\n","Compressed            \t0\n","Verbosity             \t3\n","\n","Converting sequences\n","[9495] 0s 10ms\n","Time for merging to candidateDB_35_h: 0h 0m 0s 5ms\n","Time for merging to candidateDB_35: 0h 0m 0s 6ms\n","Database type: Aminoacid\n","Time for processing: 0h 0m 0s 32ms\n","createdb /content/mmseqs2/tmp/21/candidate_genes_53.fasta /content/mmseqs2/tmp/21/candidateDB_53 \n","\n","MMseqs Version:       \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Database type         \t0\n","Shuffle input database\ttrue\n","Createdb mode         \t0\n","Write lookup file     \t1\n","Offset of numeric ids \t0\n","Compressed            \t0\n","Verbosity             \t3\n","\n","Converting sequences\n","[9495] 0s 11ms\n","Time for merging to candidateDB_53_h: 0h 0m 0s 5ms\n","Time for merging to candidateDB_53: 0h 0m 0s 6ms\n","Database type: Aminoacid\n","Time for processing: 0h 0m 0s 34ms\n","search -s 7.0 /content/mmseqs2/tmp/21/candidateDB_35 /content/mmseqs2/genexa/genexaDB /content/mmseqs2/tmp/21/resultsDB_35 /content/mmseqs2/tmp/21 \n","\n","MMseqs Version:                     \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Substitution matrix                 \tnucl:nucleotide.out,aa:blosum62.out\n","Add backtrace                       \tfalse\n","Alignment mode                      \t2\n","Allow wrapped scoring               \tfalse\n","E-value threshold                   \t0.001\n","Seq. id. threshold                  \t0\n","Min alignment length                \t0\n","Seq. id. mode                       \t0\n","Alternative alignments              \t0\n","Coverage threshold                  \t0\n","Coverage mode                       \t0\n","Max sequence length                 \t65535\n","Compositional bias                  \t1\n","Realign hits                        \tfalse\n","Max reject                          \t2147483647\n","Max accept                          \t2147483647\n","Include identical seq. id.          \tfalse\n","Preload mode                        \t0\n","Pseudo count a                      \t1\n","Pseudo count b                      \t1.5\n","Score bias                          \t0\n","Gap open cost                       \tnucl:5,aa:11\n","Gap extension cost                  \tnucl:2,aa:1\n","Zdrop                               \t40\n","Threads                             \t4\n","Compressed                          \t0\n","Verbosity                           \t3\n","Seed substitution matrix            \tnucl:nucleotide.out,aa:VTML80.out\n","Sensitivity                         \t7\n","k-mer length                        \t0\n","k-score                             \t2147483647\n","Alphabet size                       \tnucl:5,aa:21\n","Max results per query               \t300\n","Split database                      \t0\n","Split mode                          \t2\n","Split memory limit                  \t0\n","Diagonal scoring                    \ttrue\n","Exact k-mer matching                \t0\n","Mask residues                       \t1\n","Mask lower case residues            \t0\n","Minimum diagonal score              \t15\n","Spaced k-mers                       \t1\n","Spaced k-mer pattern                \t\n","Local temporary path                \t\n","Rescore mode                        \t0\n","Remove hits by seq. id. and coverage\tfalse\n","Sort results                        \t0\n","Mask profile                        \t1\n","Profile e-value threshold           \t0.1\n","Global sequence weighting           \tfalse\n","Allow deletions                     \tfalse\n","Filter MSA                          \t1\n","Maximum seq. id. threshold          \t0.9\n","Minimum seq. id.                    \t0\n","Minimum score per column            \t-20\n","Minimum coverage                    \t0\n","Select N most diverse seqs          \t1000\n","Omit consensus                      \tfalse\n","Min codons in orf                   \t30\n","Max codons in length                \t32734\n","Max orf gaps                        \t2147483647\n","Contig start mode                   \t2\n","Contig end mode                     \t2\n","Orf start mode                      \t1\n","Forward frames                      \t1,2,3\n","Reverse frames                      \t1,2,3\n","Translation table                   \t1\n","Translate orf                       \t0\n","Use all table starts                \tfalse\n","Offset of numeric ids               \t0\n","Create lookup                       \t0\n","Add orf stop                        \tfalse\n","Chain overlapping alignments        \t0\n","Merge query                         \t1\n","Search type                         \t0\n","Search iterations                   \t1\n","Start sensitivity                   \t4\n","Search steps                        \t1\n","Slice search mode                   \tfalse\n","Strand selection                    \t1\n","Disk space limit                    \t0\n","MPI runner                          \t\n","Force restart with latest tmp       \tfalse\n","Remove temporary files              \tfalse\n","\n","prefilter /content/mmseqs2/tmp/21/candidateDB_35 /content/mmseqs2/genexa/genexaDB.idx /content/mmseqs2/tmp/21/4951832741881207970/pref_0 --sub-mat nucl:nucleotide.out,aa:blosum62.out --seed-sub-mat nucl:nucleotide.out,aa:VTML80.out -k 0 --k-score 2147483647 --alph-size nucl:5,aa:21 --max-seq-len 65535 --max-seqs 300 --split 0 --split-mode 2 --split-memory-limit 0 -c 0 --cov-mode 0 --comp-bias-corr 1 --diag-score 1 --exact-kmer-matching 0 --mask 1 --mask-lower-case 0 --min-ungapped-score 15 --add-self-matches 0 --spaced-kmer-mode 1 --db-load-mode 0 --pca 1 --pcb 1.5 --threads 4 --compressed 0 -v 3 -s 7.0 \n","\n","Index version: 16\n","Generated by:  16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","ScoreMatrix:  VTML80.out\n","Query database size: 9547 type: Aminoacid\n","Estimated memory consumption: 2G\n","Target database size: 468351 type: Aminoacid\n","Process prefiltering step 1 of 1\n","\n","k-mer similarity threshold: 100\n","Starting prefiltering scores calculation (step 1 of 1)\n","Query db start 1 to 9547\n","Target db start 1 to 468351\n","[=================================================================] 100.00% 9.55K 52s 588ms\n","\n","888.836027 k-mers per position\n","419115 DB matches per sequence\n","143 overflows\n","0 queries produce too many hits (truncated result)\n","257 sequences passed prefiltering per query sequence\n","300 median result list length\n","13 sequences with 0 size result lists\n","Time for merging to pref_0: 0h 0m 0s 2ms\n","Time for processing: 0h 0m 54s 15ms\n","align /content/mmseqs2/tmp/21/candidateDB_35 /content/mmseqs2/genexa/genexaDB.idx /content/mmseqs2/tmp/21/4951832741881207970/pref_0 /content/mmseqs2/tmp/21/resultsDB_35 --sub-mat nucl:nucleotide.out,aa:blosum62.out -a 0 --alignment-mode 2 --wrapped-scoring 0 -e 0.001 --min-seq-id 0 --min-aln-len 0 --seq-id-mode 0 --alt-ali 0 -c 0 --cov-mode 0 --max-seq-len 65535 --comp-bias-corr 1 --realign 0 --max-rejected 2147483647 --max-accept 2147483647 --add-self-matches 0 --db-load-mode 0 --pca 1 --pcb 1.5 --score-bias 0 --gap-open nucl:5,aa:11 --gap-extend nucl:2,aa:1 --zdrop 40 --threads 4 --compressed 0 -v 3 \n","\n","Index version: 16\n","Generated by:  16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","ScoreMatrix:  VTML80.out\n","Compute score and coverage\n","Query database size: 9547 type: Aminoacid\n","Target database size: 468351 type: Aminoacid\n","Calculation of alignments\n","[=================================================================] 100.00% 9.55K 49s 915ms\n","Time for merging to resultsDB_35: 0h 0m 0s 2ms\n","\n","2459868 alignments calculated.\n","568507 sequence pairs passed the thresholds (0.231113 of overall calculated).\n","59.548237 hits per query sequence.\n","Time for processing: 0h 0m 50s 27ms\n","convertalis /content/mmseqs2/tmp/21/candidateDB_35 /content/mmseqs2/genexa/genexaDB /content/mmseqs2/tmp/21/resultsDB_35 /content/mmseqs2/tmp/21/resultDB_genexa.m8 --format-output query,target,evalue,raw \n","\n","MMseqs Version:        \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Substitution matrix    \tnucl:nucleotide.out,aa:blosum62.out\n","Alignment format       \t0\n","Format alignment output\tquery,target,evalue,raw\n","Translation table      \t1\n","Gap open cost          \tnucl:5,aa:11\n","Gap extension cost     \tnucl:2,aa:1\n","Database output        \tfalse\n","Preload mode           \t0\n","Search type            \t0\n","Threads                \t4\n","Compressed             \t0\n","Verbosity              \t3\n","\n","[=================================================================] 100.00% 9.55K 0s 423ms\n","Time for merging to resultDB_genexa.m8: 0h 0m 0s 15ms\n","Time for processing: 0h 0m 0s 517ms\n","search -s 7.0 /content/mmseqs2/tmp/21/candidateDB_53 /content/mmseqs2/swissprot/swissprotDB /content/mmseqs2/tmp/21/resultsDB_53 /content/mmseqs2/tmp/21 \n","\n","MMseqs Version:                     \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Substitution matrix                 \tnucl:nucleotide.out,aa:blosum62.out\n","Add backtrace                       \tfalse\n","Alignment mode                      \t2\n","Allow wrapped scoring               \tfalse\n","E-value threshold                   \t0.001\n","Seq. id. threshold                  \t0\n","Min alignment length                \t0\n","Seq. id. mode                       \t0\n","Alternative alignments              \t0\n","Coverage threshold                  \t0\n","Coverage mode                       \t0\n","Max sequence length                 \t65535\n","Compositional bias                  \t1\n","Realign hits                        \tfalse\n","Max reject                          \t2147483647\n","Max accept                          \t2147483647\n","Include identical seq. id.          \tfalse\n","Preload mode                        \t0\n","Pseudo count a                      \t1\n","Pseudo count b                      \t1.5\n","Score bias                          \t0\n","Gap open cost                       \tnucl:5,aa:11\n","Gap extension cost                  \tnucl:2,aa:1\n","Zdrop                               \t40\n","Threads                             \t4\n","Compressed                          \t0\n","Verbosity                           \t3\n","Seed substitution matrix            \tnucl:nucleotide.out,aa:VTML80.out\n","Sensitivity                         \t7\n","k-mer length                        \t0\n","k-score                             \t2147483647\n","Alphabet size                       \tnucl:5,aa:21\n","Max results per query               \t300\n","Split database                      \t0\n","Split mode                          \t2\n","Split memory limit                  \t0\n","Diagonal scoring                    \ttrue\n","Exact k-mer matching                \t0\n","Mask residues                       \t1\n","Mask lower case residues            \t0\n","Minimum diagonal score              \t15\n","Spaced k-mers                       \t1\n","Spaced k-mer pattern                \t\n","Local temporary path                \t\n","Rescore mode                        \t0\n","Remove hits by seq. id. and coverage\tfalse\n","Sort results                        \t0\n","Mask profile                        \t1\n","Profile e-value threshold           \t0.1\n","Global sequence weighting           \tfalse\n","Allow deletions                     \tfalse\n","Filter MSA                          \t1\n","Maximum seq. id. threshold          \t0.9\n","Minimum seq. id.                    \t0\n","Minimum score per column            \t-20\n","Minimum coverage                    \t0\n","Select N most diverse seqs          \t1000\n","Omit consensus                      \tfalse\n","Min codons in orf                   \t30\n","Max codons in length                \t32734\n","Max orf gaps                        \t2147483647\n","Contig start mode                   \t2\n","Contig end mode                     \t2\n","Orf start mode                      \t1\n","Forward frames                      \t1,2,3\n","Reverse frames                      \t1,2,3\n","Translation table                   \t1\n","Translate orf                       \t0\n","Use all table starts                \tfalse\n","Offset of numeric ids               \t0\n","Create lookup                       \t0\n","Add orf stop                        \tfalse\n","Chain overlapping alignments        \t0\n","Merge query                         \t1\n","Search type                         \t0\n","Search iterations                   \t1\n","Start sensitivity                   \t4\n","Search steps                        \t1\n","Slice search mode                   \tfalse\n","Strand selection                    \t1\n","Disk space limit                    \t0\n","MPI runner                          \t\n","Force restart with latest tmp       \tfalse\n","Remove temporary files              \tfalse\n","\n","prefilter /content/mmseqs2/tmp/21/candidateDB_53 /content/mmseqs2/swissprot/swissprotDB /content/mmseqs2/tmp/21/7455902819027413440/pref_0 --sub-mat nucl:nucleotide.out,aa:blosum62.out --seed-sub-mat nucl:nucleotide.out,aa:VTML80.out -k 0 --k-score 2147483647 --alph-size nucl:5,aa:21 --max-seq-len 65535 --max-seqs 300 --split 0 --split-mode 2 --split-memory-limit 0 -c 0 --cov-mode 0 --comp-bias-corr 1 --diag-score 1 --exact-kmer-matching 0 --mask 1 --mask-lower-case 0 --min-ungapped-score 15 --add-self-matches 0 --spaced-kmer-mode 1 --db-load-mode 0 --pca 1 --pcb 1.5 --threads 4 --compressed 0 -v 3 -s 7.0 \n","\n","Query database size: 9547 type: Aminoacid\n","Estimated memory consumption: 2G\n","Target database size: 562755 type: Aminoacid\n","Index table k-mer threshold: 100 at k-mer size 6 \n","Index table: counting k-mers\n","[=================================================================] 100.00% 562.76K 24s 309ms\n","Index table: Masked residues: 3164874\n","Index table: fill\n","[=================================================================] 100.00% 562.76K 18s 63ms\n","Index statistics\n","Entries:          192757622\n","DB size:          1591 MB\n","Avg k-mer size:   3.011838\n","Top 10 k-mers\n","    GPGGTL\t1835\n","    GQSWTV\t1706\n","    WGMFAT\t1638\n","    PGVFEV\t1634\n","    VLWQFW\t1623\n","    AYIRPN\t1587\n","    RSPKGV\t1585\n","    TPHKWY\t1560\n","    HIRVGW\t1548\n","    ITLSPY\t1541\n","Time for index table init: 0h 0m 44s 429ms\n","Process prefiltering step 1 of 1\n","\n","k-mer similarity threshold: 100\n","Starting prefiltering scores calculation (step 1 of 1)\n","Query db start 1 to 9547\n","Target db start 1 to 562755\n","[=================================================================] 100.00% 9.55K 59s 473ms\n","\n","888.776234 k-mers per position\n","516590 DB matches per sequence\n","335 overflows\n","0 queries produce too many hits (truncated result)\n","271 sequences passed prefiltering per query sequence\n","300 median result list length\n","8 sequences with 0 size result lists\n","Time for merging to pref_0: 0h 0m 0s 2ms\n","Time for processing: 0h 1m 45s 28ms\n","align /content/mmseqs2/tmp/21/candidateDB_53 /content/mmseqs2/swissprot/swissprotDB /content/mmseqs2/tmp/21/7455902819027413440/pref_0 /content/mmseqs2/tmp/21/resultsDB_53 --sub-mat nucl:nucleotide.out,aa:blosum62.out -a 0 --alignment-mode 2 --wrapped-scoring 0 -e 0.001 --min-seq-id 0 --min-aln-len 0 --seq-id-mode 0 --alt-ali 0 -c 0 --cov-mode 0 --max-seq-len 65535 --comp-bias-corr 1 --realign 0 --max-rejected 2147483647 --max-accept 2147483647 --add-self-matches 0 --db-load-mode 0 --pca 1 --pcb 1.5 --score-bias 0 --gap-open nucl:5,aa:11 --gap-extend nucl:2,aa:1 --zdrop 40 --threads 4 --compressed 0 -v 3 \n","\n","Compute score and coverage\n","Query database size: 9547 type: Aminoacid\n","Target database size: 562755 type: Aminoacid\n","Calculation of alignments\n","[=================================================================] 100.00% 9.55K 43s 763ms\n","Time for merging to resultsDB_53: 0h 0m 0s 2ms\n","\n","2591282 alignments calculated.\n","380814 sequence pairs passed the thresholds (0.146960 of overall calculated).\n","39.888344 hits per query sequence.\n","Time for processing: 0h 0m 43s 837ms\n","convertalis /content/mmseqs2/tmp/21/candidateDB_53 /content/mmseqs2/swissprot/swissprotDB /content/mmseqs2/tmp/21/resultsDB_53 /content/mmseqs2/tmp/21/resultDB_secondary.m8 --format-output query,target,evalue,raw \n","\n","MMseqs Version:        \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Substitution matrix    \tnucl:nucleotide.out,aa:blosum62.out\n","Alignment format       \t0\n","Format alignment output\tquery,target,evalue,raw\n","Translation table      \t1\n","Gap open cost          \tnucl:5,aa:11\n","Gap extension cost     \tnucl:2,aa:1\n","Database output        \tfalse\n","Preload mode           \t0\n","Search type            \t0\n","Threads                \t4\n","Compressed             \t0\n","Verbosity              \t3\n","\n","[=================================================================] 100.00% 9.55K 0s 270ms\n","Time for merging to resultDB_secondary.m8: 0h 0m 0s 15ms\n","Time for processing: 0h 0m 0s 379ms\n","found 4576 genes\n","\n","\n","Reading fasta file GCF_000661895.1_ASM66189v1_genomic.fna.gz...\n","\n","Finding and translating open reading frames...\n","\n","Encoding amino acids...\n","\n","Applying kmer prefilter...\n","\n","Scoring ORFs with temporal convolutional network...\n","\n"],"name":"stdout"},{"output_type":"display_data","data":{"application/vnd.jupyter.widget-view+json":{"model_id":"7c6d6006f1684eb783858af378569668","version_minor":0,"version_major":2},"text/plain":["HBox(children=(FloatProgress(value=0.0, max=42.0), HTML(value='')))"]},"metadata":{"tags":[]}},{"output_type":"stream","text":["\n","Scoring translation initiation sites...\n","\n"],"name":"stdout"},{"output_type":"display_data","data":{"application/vnd.jupyter.widget-view+json":{"model_id":"3876f9ce8b284399b2b9649839e1fe02","version_minor":0,"version_major":2},"text/plain":["HBox(children=(FloatProgress(value=0.0, max=13.0), HTML(value='')))"]},"metadata":{"tags":[]}},{"output_type":"stream","text":["\n","Creating weighted directed acyclic graph of all ORFs and subORFs on contig 0...\n","\n"],"name":"stdout"},{"output_type":"display_data","data":{"application/vnd.jupyter.widget-view+json":{"model_id":"1b46dca39e954201b0852959a1c4c10c","version_minor":0,"version_major":2},"text/plain":["HBox(children=(FloatProgress(value=0.0, max=106600.0), HTML(value='')))"]},"metadata":{"tags":[]}},{"output_type":"stream","text":["\n","Maximizing geneiness...\n","\n","Filtering predictions with mmseqs2...\n","createdb /content/mmseqs2/tmp/22/candidate_genes_35.fasta /content/mmseqs2/tmp/22/candidateDB_35 \n","\n","MMseqs Version:       \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Database type         \t0\n","Shuffle input database\ttrue\n","Createdb mode         \t0\n","Write lookup file     \t1\n","Offset of numeric ids \t0\n","Compressed            \t0\n","Verbosity             \t3\n","\n","Converting sequences\n","[3839] 0s 5ms\n","Time for merging to candidateDB_35_h: 0h 0m 0s 2ms\n","Time for merging to candidateDB_35: 0h 0m 0s 4ms\n","Database type: Aminoacid\n","Time for processing: 0h 0m 0s 19ms\n","createdb /content/mmseqs2/tmp/22/candidate_genes_53.fasta /content/mmseqs2/tmp/22/candidateDB_53 \n","\n","MMseqs Version:       \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Database type         \t0\n","Shuffle input database\ttrue\n","Createdb mode         \t0\n","Write lookup file     \t1\n","Offset of numeric ids \t0\n","Compressed            \t0\n","Verbosity             \t3\n","\n","Converting sequences\n","[3839] 0s 6ms\n","Time for merging to candidateDB_53_h: 0h 0m 0s 3ms\n","Time for merging to candidateDB_53: 0h 0m 0s 4ms\n","Database type: Aminoacid\n","Time for processing: 0h 0m 0s 20ms\n","search -s 7.0 /content/mmseqs2/tmp/22/candidateDB_35 /content/mmseqs2/genexa/genexaDB /content/mmseqs2/tmp/22/resultsDB_35 /content/mmseqs2/tmp/22 \n","\n","MMseqs Version:                     \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Substitution matrix                 \tnucl:nucleotide.out,aa:blosum62.out\n","Add backtrace                       \tfalse\n","Alignment mode                      \t2\n","Allow wrapped scoring               \tfalse\n","E-value threshold                   \t0.001\n","Seq. id. threshold                  \t0\n","Min alignment length                \t0\n","Seq. id. mode                       \t0\n","Alternative alignments              \t0\n","Coverage threshold                  \t0\n","Coverage mode                       \t0\n","Max sequence length                 \t65535\n","Compositional bias                  \t1\n","Realign hits                        \tfalse\n","Max reject                          \t2147483647\n","Max accept                          \t2147483647\n","Include identical seq. id.          \tfalse\n","Preload mode                        \t0\n","Pseudo count a                      \t1\n","Pseudo count b                      \t1.5\n","Score bias                          \t0\n","Gap open cost                       \tnucl:5,aa:11\n","Gap extension cost                  \tnucl:2,aa:1\n","Zdrop                               \t40\n","Threads                             \t4\n","Compressed                          \t0\n","Verbosity                           \t3\n","Seed substitution matrix            \tnucl:nucleotide.out,aa:VTML80.out\n","Sensitivity                         \t7\n","k-mer length                        \t0\n","k-score                             \t2147483647\n","Alphabet size                       \tnucl:5,aa:21\n","Max results per query               \t300\n","Split database                      \t0\n","Split mode                          \t2\n","Split memory limit                  \t0\n","Diagonal scoring                    \ttrue\n","Exact k-mer matching                \t0\n","Mask residues                       \t1\n","Mask lower case residues            \t0\n","Minimum diagonal score              \t15\n","Spaced k-mers                       \t1\n","Spaced k-mer pattern                \t\n","Local temporary path                \t\n","Rescore mode                        \t0\n","Remove hits by seq. id. and coverage\tfalse\n","Sort results                        \t0\n","Mask profile                        \t1\n","Profile e-value threshold           \t0.1\n","Global sequence weighting           \tfalse\n","Allow deletions                     \tfalse\n","Filter MSA                          \t1\n","Maximum seq. id. threshold          \t0.9\n","Minimum seq. id.                    \t0\n","Minimum score per column            \t-20\n","Minimum coverage                    \t0\n","Select N most diverse seqs          \t1000\n","Omit consensus                      \tfalse\n","Min codons in orf                   \t30\n","Max codons in length                \t32734\n","Max orf gaps                        \t2147483647\n","Contig start mode                   \t2\n","Contig end mode                     \t2\n","Orf start mode                      \t1\n","Forward frames                      \t1,2,3\n","Reverse frames                      \t1,2,3\n","Translation table                   \t1\n","Translate orf                       \t0\n","Use all table starts                \tfalse\n","Offset of numeric ids               \t0\n","Create lookup                       \t0\n","Add orf stop                        \tfalse\n","Chain overlapping alignments        \t0\n","Merge query                         \t1\n","Search type                         \t0\n","Search iterations                   \t1\n","Start sensitivity                   \t4\n","Search steps                        \t1\n","Slice search mode                   \tfalse\n","Strand selection                    \t1\n","Disk space limit                    \t0\n","MPI runner                          \t\n","Force restart with latest tmp       \tfalse\n","Remove temporary files              \tfalse\n","\n","prefilter /content/mmseqs2/tmp/22/candidateDB_35 /content/mmseqs2/genexa/genexaDB.idx /content/mmseqs2/tmp/22/18138290712481064097/pref_0 --sub-mat nucl:nucleotide.out,aa:blosum62.out --seed-sub-mat nucl:nucleotide.out,aa:VTML80.out -k 0 --k-score 2147483647 --alph-size nucl:5,aa:21 --max-seq-len 65535 --max-seqs 300 --split 0 --split-mode 2 --split-memory-limit 0 -c 0 --cov-mode 0 --comp-bias-corr 1 --diag-score 1 --exact-kmer-matching 0 --mask 1 --mask-lower-case 0 --min-ungapped-score 15 --add-self-matches 0 --spaced-kmer-mode 1 --db-load-mode 0 --pca 1 --pcb 1.5 --threads 4 --compressed 0 -v 3 -s 7.0 \n","\n","Index version: 16\n","Generated by:  16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","ScoreMatrix:  VTML80.out\n","Query database size: 3920 type: Aminoacid\n","Estimated memory consumption: 2G\n","Target database size: 468351 type: Aminoacid\n","Process prefiltering step 1 of 1\n","\n","k-mer similarity threshold: 100\n","Starting prefiltering scores calculation (step 1 of 1)\n","Query db start 1 to 3920\n","Target db start 1 to 468351\n","[=================================================================] 100.00% 3.92K 30s 191ms\n","\n","742.222163 k-mers per position\n","641595 DB matches per sequence\n","129 overflows\n","0 queries produce too many hits (truncated result)\n","277 sequences passed prefiltering per query sequence\n","300 median result list length\n","5 sequences with 0 size result lists\n","Time for merging to pref_0: 0h 0m 0s 1ms\n","Time for processing: 0h 0m 31s 667ms\n","align /content/mmseqs2/tmp/22/candidateDB_35 /content/mmseqs2/genexa/genexaDB.idx /content/mmseqs2/tmp/22/18138290712481064097/pref_0 /content/mmseqs2/tmp/22/resultsDB_35 --sub-mat nucl:nucleotide.out,aa:blosum62.out -a 0 --alignment-mode 2 --wrapped-scoring 0 -e 0.001 --min-seq-id 0 --min-aln-len 0 --seq-id-mode 0 --alt-ali 0 -c 0 --cov-mode 0 --max-seq-len 65535 --comp-bias-corr 1 --realign 0 --max-rejected 2147483647 --max-accept 2147483647 --add-self-matches 0 --db-load-mode 0 --pca 1 --pcb 1.5 --score-bias 0 --gap-open nucl:5,aa:11 --gap-extend nucl:2,aa:1 --zdrop 40 --threads 4 --compressed 0 -v 3 \n","\n","Index version: 16\n","Generated by:  16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","ScoreMatrix:  VTML80.out\n","Compute score and coverage\n","Query database size: 3920 type: Aminoacid\n","Target database size: 468351 type: Aminoacid\n","Calculation of alignments\n","[=================================================================] 100.00% 3.92K 29s 597ms\n","Time for merging to resultsDB_35: 0h 0m 0s 1ms\n","\n","1086512 alignments calculated.\n","337317 sequence pairs passed the thresholds (0.310459 of overall calculated).\n","86.050255 hits per query sequence.\n","Time for processing: 0h 0m 29s 632ms\n","convertalis /content/mmseqs2/tmp/22/candidateDB_35 /content/mmseqs2/genexa/genexaDB /content/mmseqs2/tmp/22/resultsDB_35 /content/mmseqs2/tmp/22/resultDB_genexa.m8 --format-output query,target,evalue,raw \n","\n","MMseqs Version:        \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Substitution matrix    \tnucl:nucleotide.out,aa:blosum62.out\n","Alignment format       \t0\n","Format alignment output\tquery,target,evalue,raw\n","Translation table      \t1\n","Gap open cost          \tnucl:5,aa:11\n","Gap extension cost     \tnucl:2,aa:1\n","Database output        \tfalse\n","Preload mode           \t0\n","Search type            \t0\n","Threads                \t4\n","Compressed             \t0\n","Verbosity              \t3\n","\n","[=================================================================] 100.00% 3.92K 0s 259ms\n","Time for merging to resultDB_genexa.m8: 0h 0m 0s 9ms\n","Time for processing: 0h 0m 0s 339ms\n","search -s 7.0 /content/mmseqs2/tmp/22/candidateDB_53 /content/mmseqs2/swissprot/swissprotDB /content/mmseqs2/tmp/22/resultsDB_53 /content/mmseqs2/tmp/22 \n","\n","MMseqs Version:                     \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Substitution matrix                 \tnucl:nucleotide.out,aa:blosum62.out\n","Add backtrace                       \tfalse\n","Alignment mode                      \t2\n","Allow wrapped scoring               \tfalse\n","E-value threshold                   \t0.001\n","Seq. id. threshold                  \t0\n","Min alignment length                \t0\n","Seq. id. mode                       \t0\n","Alternative alignments              \t0\n","Coverage threshold                  \t0\n","Coverage mode                       \t0\n","Max sequence length                 \t65535\n","Compositional bias                  \t1\n","Realign hits                        \tfalse\n","Max reject                          \t2147483647\n","Max accept                          \t2147483647\n","Include identical seq. id.          \tfalse\n","Preload mode                        \t0\n","Pseudo count a                      \t1\n","Pseudo count b                      \t1.5\n","Score bias                          \t0\n","Gap open cost                       \tnucl:5,aa:11\n","Gap extension cost                  \tnucl:2,aa:1\n","Zdrop                               \t40\n","Threads                             \t4\n","Compressed                          \t0\n","Verbosity                           \t3\n","Seed substitution matrix            \tnucl:nucleotide.out,aa:VTML80.out\n","Sensitivity                         \t7\n","k-mer length                        \t0\n","k-score                             \t2147483647\n","Alphabet size                       \tnucl:5,aa:21\n","Max results per query               \t300\n","Split database                      \t0\n","Split mode                          \t2\n","Split memory limit                  \t0\n","Diagonal scoring                    \ttrue\n","Exact k-mer matching                \t0\n","Mask residues                       \t1\n","Mask lower case residues            \t0\n","Minimum diagonal score              \t15\n","Spaced k-mers                       \t1\n","Spaced k-mer pattern                \t\n","Local temporary path                \t\n","Rescore mode                        \t0\n","Remove hits by seq. id. and coverage\tfalse\n","Sort results                        \t0\n","Mask profile                        \t1\n","Profile e-value threshold           \t0.1\n","Global sequence weighting           \tfalse\n","Allow deletions                     \tfalse\n","Filter MSA                          \t1\n","Maximum seq. id. threshold          \t0.9\n","Minimum seq. id.                    \t0\n","Minimum score per column            \t-20\n","Minimum coverage                    \t0\n","Select N most diverse seqs          \t1000\n","Omit consensus                      \tfalse\n","Min codons in orf                   \t30\n","Max codons in length                \t32734\n","Max orf gaps                        \t2147483647\n","Contig start mode                   \t2\n","Contig end mode                     \t2\n","Orf start mode                      \t1\n","Forward frames                      \t1,2,3\n","Reverse frames                      \t1,2,3\n","Translation table                   \t1\n","Translate orf                       \t0\n","Use all table starts                \tfalse\n","Offset of numeric ids               \t0\n","Create lookup                       \t0\n","Add orf stop                        \tfalse\n","Chain overlapping alignments        \t0\n","Merge query                         \t1\n","Search type                         \t0\n","Search iterations                   \t1\n","Start sensitivity                   \t4\n","Search steps                        \t1\n","Slice search mode                   \tfalse\n","Strand selection                    \t1\n","Disk space limit                    \t0\n","MPI runner                          \t\n","Force restart with latest tmp       \tfalse\n","Remove temporary files              \tfalse\n","\n","prefilter /content/mmseqs2/tmp/22/candidateDB_53 /content/mmseqs2/swissprot/swissprotDB /content/mmseqs2/tmp/22/7557267807072759423/pref_0 --sub-mat nucl:nucleotide.out,aa:blosum62.out --seed-sub-mat nucl:nucleotide.out,aa:VTML80.out -k 0 --k-score 2147483647 --alph-size nucl:5,aa:21 --max-seq-len 65535 --max-seqs 300 --split 0 --split-mode 2 --split-memory-limit 0 -c 0 --cov-mode 0 --comp-bias-corr 1 --diag-score 1 --exact-kmer-matching 0 --mask 1 --mask-lower-case 0 --min-ungapped-score 15 --add-self-matches 0 --spaced-kmer-mode 1 --db-load-mode 0 --pca 1 --pcb 1.5 --threads 4 --compressed 0 -v 3 -s 7.0 \n","\n","Query database size: 3920 type: Aminoacid\n","Estimated memory consumption: 2G\n","Target database size: 562755 type: Aminoacid\n","Index table k-mer threshold: 100 at k-mer size 6 \n","Index table: counting k-mers\n","[=================================================================] 100.00% 562.76K 25s 0ms\n","Index table: Masked residues: 3164874\n","Index table: fill\n","[=================================================================] 100.00% 562.76K 18s 340ms\n","Index statistics\n","Entries:          192757622\n","DB size:          1591 MB\n","Avg k-mer size:   3.011838\n","Top 10 k-mers\n","    GPGGTL\t1835\n","    GQSWTV\t1706\n","    WGMFAT\t1638\n","    PGVFEV\t1634\n","    VLWQFW\t1623\n","    AYIRPN\t1587\n","    RSPKGV\t1585\n","    TPHKWY\t1560\n","    HIRVGW\t1548\n","    ITLSPY\t1541\n","Time for index table init: 0h 0m 45s 506ms\n","Process prefiltering step 1 of 1\n","\n","k-mer similarity threshold: 100\n","Starting prefiltering scores calculation (step 1 of 1)\n","Query db start 1 to 3920\n","Target db start 1 to 562755\n","[=================================================================] 100.00% 3.92K 33s 644ms\n","\n","742.122829 k-mers per position\n","729077 DB matches per sequence\n","210 overflows\n","0 queries produce too many hits (truncated result)\n","282 sequences passed prefiltering per query sequence\n","300 median result list length\n","5 sequences with 0 size result lists\n","Time for merging to pref_0: 0h 0m 0s 1ms\n","Time for processing: 0h 1m 20s 342ms\n","align /content/mmseqs2/tmp/22/candidateDB_53 /content/mmseqs2/swissprot/swissprotDB /content/mmseqs2/tmp/22/7557267807072759423/pref_0 /content/mmseqs2/tmp/22/resultsDB_53 --sub-mat nucl:nucleotide.out,aa:blosum62.out -a 0 --alignment-mode 2 --wrapped-scoring 0 -e 0.001 --min-seq-id 0 --min-aln-len 0 --seq-id-mode 0 --alt-ali 0 -c 0 --cov-mode 0 --max-seq-len 65535 --comp-bias-corr 1 --realign 0 --max-rejected 2147483647 --max-accept 2147483647 --add-self-matches 0 --db-load-mode 0 --pca 1 --pcb 1.5 --score-bias 0 --gap-open nucl:5,aa:11 --gap-extend nucl:2,aa:1 --zdrop 40 --threads 4 --compressed 0 -v 3 \n","\n","Compute score and coverage\n","Query database size: 3920 type: Aminoacid\n","Target database size: 562755 type: Aminoacid\n","Calculation of alignments\n","[=================================================================] 100.00% 3.92K 27s 782ms\n","Time for merging to resultsDB_53: 0h 0m 0s 2ms\n","\n","1106615 alignments calculated.\n","248837 sequence pairs passed the thresholds (0.224863 of overall calculated).\n","63.478828 hits per query sequence.\n","Time for processing: 0h 0m 27s 846ms\n","convertalis /content/mmseqs2/tmp/22/candidateDB_53 /content/mmseqs2/swissprot/swissprotDB /content/mmseqs2/tmp/22/resultsDB_53 /content/mmseqs2/tmp/22/resultDB_secondary.m8 --format-output query,target,evalue,raw \n","\n","MMseqs Version:        \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Substitution matrix    \tnucl:nucleotide.out,aa:blosum62.out\n","Alignment format       \t0\n","Format alignment output\tquery,target,evalue,raw\n","Translation table      \t1\n","Gap open cost          \tnucl:5,aa:11\n","Gap extension cost     \tnucl:2,aa:1\n","Database output        \tfalse\n","Preload mode           \t0\n","Search type            \t0\n","Threads                \t4\n","Compressed             \t0\n","Verbosity              \t3\n","\n","[=================================================================] 100.00% 3.92K 0s 194ms\n","Time for merging to resultDB_secondary.m8: 0h 0m 0s 8ms\n","Time for processing: 0h 0m 0s 305ms\n","found 2728 genes\n","\n","\n","Creating weighted directed acyclic graph of all ORFs and subORFs on contig 1...\n","\n"],"name":"stdout"},{"output_type":"display_data","data":{"application/vnd.jupyter.widget-view+json":{"model_id":"fe61a9a41ba347b9ba49a5e7d5d0f5b8","version_minor":0,"version_major":2},"text/plain":["HBox(children=(FloatProgress(value=0.0, max=8425.0), HTML(value='')))"]},"metadata":{"tags":[]}},{"output_type":"stream","text":["\n","Maximizing geneiness...\n","\n","Filtering predictions with mmseqs2...\n","createdb /content/mmseqs2/tmp/23/candidate_genes_35.fasta /content/mmseqs2/tmp/23/candidateDB_35 \n","\n","MMseqs Version:       \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Database type         \t0\n","Shuffle input database\ttrue\n","Createdb mode         \t0\n","Write lookup file     \t1\n","Offset of numeric ids \t0\n","Compressed            \t0\n","Verbosity             \t3\n","\n","Converting sequences\n","[708] 0s 3ms\n","Time for merging to candidateDB_35_h: 0h 0m 0s 2ms\n","Time for merging to candidateDB_35: 0h 0m 0s 2ms\n","Database type: Aminoacid\n","Time for processing: 0h 0m 0s 11ms\n","createdb /content/mmseqs2/tmp/23/candidate_genes_53.fasta /content/mmseqs2/tmp/23/candidateDB_53 \n","\n","MMseqs Version:       \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Database type         \t0\n","Shuffle input database\ttrue\n","Createdb mode         \t0\n","Write lookup file     \t1\n","Offset of numeric ids \t0\n","Compressed            \t0\n","Verbosity             \t3\n","\n","Converting sequences\n","[708] 0s 3ms\n","Time for merging to candidateDB_53_h: 0h 0m 0s 3ms\n","Time for merging to candidateDB_53: 0h 0m 0s 2ms\n","Database type: Aminoacid\n","Time for processing: 0h 0m 0s 12ms\n","search -s 7.0 /content/mmseqs2/tmp/23/candidateDB_35 /content/mmseqs2/genexa/genexaDB /content/mmseqs2/tmp/23/resultsDB_35 /content/mmseqs2/tmp/23 \n","\n","MMseqs Version:                     \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Substitution matrix                 \tnucl:nucleotide.out,aa:blosum62.out\n","Add backtrace                       \tfalse\n","Alignment mode                      \t2\n","Allow wrapped scoring               \tfalse\n","E-value threshold                   \t0.001\n","Seq. id. threshold                  \t0\n","Min alignment length                \t0\n","Seq. id. mode                       \t0\n","Alternative alignments              \t0\n","Coverage threshold                  \t0\n","Coverage mode                       \t0\n","Max sequence length                 \t65535\n","Compositional bias                  \t1\n","Realign hits                        \tfalse\n","Max reject                          \t2147483647\n","Max accept                          \t2147483647\n","Include identical seq. id.          \tfalse\n","Preload mode                        \t0\n","Pseudo count a                      \t1\n","Pseudo count b                      \t1.5\n","Score bias                          \t0\n","Gap open cost                       \tnucl:5,aa:11\n","Gap extension cost                  \tnucl:2,aa:1\n","Zdrop                               \t40\n","Threads                             \t4\n","Compressed                          \t0\n","Verbosity                           \t3\n","Seed substitution matrix            \tnucl:nucleotide.out,aa:VTML80.out\n","Sensitivity                         \t7\n","k-mer length                        \t0\n","k-score                             \t2147483647\n","Alphabet size                       \tnucl:5,aa:21\n","Max results per query               \t300\n","Split database                      \t0\n","Split mode                          \t2\n","Split memory limit                  \t0\n","Diagonal scoring                    \ttrue\n","Exact k-mer matching                \t0\n","Mask residues                       \t1\n","Mask lower case residues            \t0\n","Minimum diagonal score              \t15\n","Spaced k-mers                       \t1\n","Spaced k-mer pattern                \t\n","Local temporary path                \t\n","Rescore mode                        \t0\n","Remove hits by seq. id. and coverage\tfalse\n","Sort results                        \t0\n","Mask profile                        \t1\n","Profile e-value threshold           \t0.1\n","Global sequence weighting           \tfalse\n","Allow deletions                     \tfalse\n","Filter MSA                          \t1\n","Maximum seq. id. threshold          \t0.9\n","Minimum seq. id.                    \t0\n","Minimum score per column            \t-20\n","Minimum coverage                    \t0\n","Select N most diverse seqs          \t1000\n","Omit consensus                      \tfalse\n","Min codons in orf                   \t30\n","Max codons in length                \t32734\n","Max orf gaps                        \t2147483647\n","Contig start mode                   \t2\n","Contig end mode                     \t2\n","Orf start mode                      \t1\n","Forward frames                      \t1,2,3\n","Reverse frames                      \t1,2,3\n","Translation table                   \t1\n","Translate orf                       \t0\n","Use all table starts                \tfalse\n","Offset of numeric ids               \t0\n","Create lookup                       \t0\n","Add orf stop                        \tfalse\n","Chain overlapping alignments        \t0\n","Merge query                         \t1\n","Search type                         \t0\n","Search iterations                   \t1\n","Start sensitivity                   \t4\n","Search steps                        \t1\n","Slice search mode                   \tfalse\n","Strand selection                    \t1\n","Disk space limit                    \t0\n","MPI runner                          \t\n","Force restart with latest tmp       \tfalse\n","Remove temporary files              \tfalse\n","\n","prefilter /content/mmseqs2/tmp/23/candidateDB_35 /content/mmseqs2/genexa/genexaDB.idx /content/mmseqs2/tmp/23/12878004609371368608/pref_0 --sub-mat nucl:nucleotide.out,aa:blosum62.out --seed-sub-mat nucl:nucleotide.out,aa:VTML80.out -k 0 --k-score 2147483647 --alph-size nucl:5,aa:21 --max-seq-len 65535 --max-seqs 300 --split 0 --split-mode 2 --split-memory-limit 0 -c 0 --cov-mode 0 --comp-bias-corr 1 --diag-score 1 --exact-kmer-matching 0 --mask 1 --mask-lower-case 0 --min-ungapped-score 15 --add-self-matches 0 --spaced-kmer-mode 1 --db-load-mode 0 --pca 1 --pcb 1.5 --threads 4 --compressed 0 -v 3 -s 7.0 \n","\n","Index version: 16\n","Generated by:  16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","ScoreMatrix:  VTML80.out\n","Query database size: 756 type: Aminoacid\n","Estimated memory consumption: 2G\n","Target database size: 468351 type: Aminoacid\n","Process prefiltering step 1 of 1\n","\n","k-mer similarity threshold: 100\n","Starting prefiltering scores calculation (step 1 of 1)\n","Query db start 1 to 756\n","Target db start 1 to 468351\n","[=================================================================] 100.00% 756 5s 946ms\n","\n","701.418422 k-mers per position\n","619823 DB matches per sequence\n","28 overflows\n","0 queries produce too many hits (truncated result)\n","264 sequences passed prefiltering per query sequence\n","300 median result list length\n","2 sequences with 0 size result lists\n","Time for merging to pref_0: 0h 0m 0s 0ms\n","Time for processing: 0h 0m 7s 447ms\n","align /content/mmseqs2/tmp/23/candidateDB_35 /content/mmseqs2/genexa/genexaDB.idx /content/mmseqs2/tmp/23/12878004609371368608/pref_0 /content/mmseqs2/tmp/23/resultsDB_35 --sub-mat nucl:nucleotide.out,aa:blosum62.out -a 0 --alignment-mode 2 --wrapped-scoring 0 -e 0.001 --min-seq-id 0 --min-aln-len 0 --seq-id-mode 0 --alt-ali 0 -c 0 --cov-mode 0 --max-seq-len 65535 --comp-bias-corr 1 --realign 0 --max-rejected 2147483647 --max-accept 2147483647 --add-self-matches 0 --db-load-mode 0 --pca 1 --pcb 1.5 --score-bias 0 --gap-open nucl:5,aa:11 --gap-extend nucl:2,aa:1 --zdrop 40 --threads 4 --compressed 0 -v 3 \n","\n","Index version: 16\n","Generated by:  16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","ScoreMatrix:  VTML80.out\n","Compute score and coverage\n","Query database size: 756 type: Aminoacid\n","Target database size: 468351 type: Aminoacid\n","Calculation of alignments\n","[=================================================================] 100.00% 756 7s 403ms\n","Time for merging to resultsDB_35: 0h 0m 0s 0ms\n","\n","199895 alignments calculated.\n","86501 sequence pairs passed the thresholds (0.432732 of overall calculated).\n","114.419312 hits per query sequence.\n","Time for processing: 0h 0m 7s 487ms\n","convertalis /content/mmseqs2/tmp/23/candidateDB_35 /content/mmseqs2/genexa/genexaDB /content/mmseqs2/tmp/23/resultsDB_35 /content/mmseqs2/tmp/23/resultDB_genexa.m8 --format-output query,target,evalue,raw \n","\n","MMseqs Version:        \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Substitution matrix    \tnucl:nucleotide.out,aa:blosum62.out\n","Alignment format       \t0\n","Format alignment output\tquery,target,evalue,raw\n","Translation table      \t1\n","Gap open cost          \tnucl:5,aa:11\n","Gap extension cost     \tnucl:2,aa:1\n","Database output        \tfalse\n","Preload mode           \t0\n","Search type            \t0\n","Threads                \t4\n","Compressed             \t0\n","Verbosity              \t3\n","\n","[=================================================================] 100.00% 756 0s 71ms\n","Time for merging to resultDB_genexa.m8: 0h 0m 0s 3ms\n","Time for processing: 0h 0m 0s 141ms\n","search -s 7.0 /content/mmseqs2/tmp/23/candidateDB_53 /content/mmseqs2/swissprot/swissprotDB /content/mmseqs2/tmp/23/resultsDB_53 /content/mmseqs2/tmp/23 \n","\n","MMseqs Version:                     \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Substitution matrix                 \tnucl:nucleotide.out,aa:blosum62.out\n","Add backtrace                       \tfalse\n","Alignment mode                      \t2\n","Allow wrapped scoring               \tfalse\n","E-value threshold                   \t0.001\n","Seq. id. threshold                  \t0\n","Min alignment length                \t0\n","Seq. id. mode                       \t0\n","Alternative alignments              \t0\n","Coverage threshold                  \t0\n","Coverage mode                       \t0\n","Max sequence length                 \t65535\n","Compositional bias                  \t1\n","Realign hits                        \tfalse\n","Max reject                          \t2147483647\n","Max accept                          \t2147483647\n","Include identical seq. id.          \tfalse\n","Preload mode                        \t0\n","Pseudo count a                      \t1\n","Pseudo count b                      \t1.5\n","Score bias                          \t0\n","Gap open cost                       \tnucl:5,aa:11\n","Gap extension cost                  \tnucl:2,aa:1\n","Zdrop                               \t40\n","Threads                             \t4\n","Compressed                          \t0\n","Verbosity                           \t3\n","Seed substitution matrix            \tnucl:nucleotide.out,aa:VTML80.out\n","Sensitivity                         \t7\n","k-mer length                        \t0\n","k-score                             \t2147483647\n","Alphabet size                       \tnucl:5,aa:21\n","Max results per query               \t300\n","Split database                      \t0\n","Split mode                          \t2\n","Split memory limit                  \t0\n","Diagonal scoring                    \ttrue\n","Exact k-mer matching                \t0\n","Mask residues                       \t1\n","Mask lower case residues            \t0\n","Minimum diagonal score              \t15\n","Spaced k-mers                       \t1\n","Spaced k-mer pattern                \t\n","Local temporary path                \t\n","Rescore mode                        \t0\n","Remove hits by seq. id. and coverage\tfalse\n","Sort results                        \t0\n","Mask profile                        \t1\n","Profile e-value threshold           \t0.1\n","Global sequence weighting           \tfalse\n","Allow deletions                     \tfalse\n","Filter MSA                          \t1\n","Maximum seq. id. threshold          \t0.9\n","Minimum seq. id.                    \t0\n","Minimum score per column            \t-20\n","Minimum coverage                    \t0\n","Select N most diverse seqs          \t1000\n","Omit consensus                      \tfalse\n","Min codons in orf                   \t30\n","Max codons in length                \t32734\n","Max orf gaps                        \t2147483647\n","Contig start mode                   \t2\n","Contig end mode                     \t2\n","Orf start mode                      \t1\n","Forward frames                      \t1,2,3\n","Reverse frames                      \t1,2,3\n","Translation table                   \t1\n","Translate orf                       \t0\n","Use all table starts                \tfalse\n","Offset of numeric ids               \t0\n","Create lookup                       \t0\n","Add orf stop                        \tfalse\n","Chain overlapping alignments        \t0\n","Merge query                         \t1\n","Search type                         \t0\n","Search iterations                   \t1\n","Start sensitivity                   \t4\n","Search steps                        \t1\n","Slice search mode                   \tfalse\n","Strand selection                    \t1\n","Disk space limit                    \t0\n","MPI runner                          \t\n","Force restart with latest tmp       \tfalse\n","Remove temporary files              \tfalse\n","\n","prefilter /content/mmseqs2/tmp/23/candidateDB_53 /content/mmseqs2/swissprot/swissprotDB /content/mmseqs2/tmp/23/7658632795118105406/pref_0 --sub-mat nucl:nucleotide.out,aa:blosum62.out --seed-sub-mat nucl:nucleotide.out,aa:VTML80.out -k 0 --k-score 2147483647 --alph-size nucl:5,aa:21 --max-seq-len 65535 --max-seqs 300 --split 0 --split-mode 2 --split-memory-limit 0 -c 0 --cov-mode 0 --comp-bias-corr 1 --diag-score 1 --exact-kmer-matching 0 --mask 1 --mask-lower-case 0 --min-ungapped-score 15 --add-self-matches 0 --spaced-kmer-mode 1 --db-load-mode 0 --pca 1 --pcb 1.5 --threads 4 --compressed 0 -v 3 -s 7.0 \n","\n","Query database size: 756 type: Aminoacid\n","Estimated memory consumption: 2G\n","Target database size: 562755 type: Aminoacid\n","Index table k-mer threshold: 100 at k-mer size 6 \n","Index table: counting k-mers\n","[=================================================================] 100.00% 562.76K 24s 652ms\n","Index table: Masked residues: 3164874\n","Index table: fill\n","[=================================================================] 100.00% 562.76K 18s 162ms\n","Index statistics\n","Entries:          192757622\n","DB size:          1591 MB\n","Avg k-mer size:   3.011838\n","Top 10 k-mers\n","    GPGGTL\t1835\n","    GQSWTV\t1706\n","    WGMFAT\t1638\n","    PGVFEV\t1634\n","    VLWQFW\t1623\n","    AYIRPN\t1587\n","    RSPKGV\t1585\n","    TPHKWY\t1560\n","    HIRVGW\t1548\n","    ITLSPY\t1541\n","Time for index table init: 0h 0m 44s 910ms\n","Process prefiltering step 1 of 1\n","\n","k-mer similarity threshold: 100\n","Starting prefiltering scores calculation (step 1 of 1)\n","Query db start 1 to 756\n","Target db start 1 to 562755\n","[=================================================================] 100.00% 756 6s 304ms\n","\n","699.874917 k-mers per position\n","707830 DB matches per sequence\n","44 overflows\n","0 queries produce too many hits (truncated result)\n","267 sequences passed prefiltering per query sequence\n","300 median result list length\n","2 sequences with 0 size result lists\n","Time for merging to pref_0: 0h 0m 0s 0ms\n","Time for processing: 0h 0m 52s 370ms\n","align /content/mmseqs2/tmp/23/candidateDB_53 /content/mmseqs2/swissprot/swissprotDB /content/mmseqs2/tmp/23/7658632795118105406/pref_0 /content/mmseqs2/tmp/23/resultsDB_53 --sub-mat nucl:nucleotide.out,aa:blosum62.out -a 0 --alignment-mode 2 --wrapped-scoring 0 -e 0.001 --min-seq-id 0 --min-aln-len 0 --seq-id-mode 0 --alt-ali 0 -c 0 --cov-mode 0 --max-seq-len 65535 --comp-bias-corr 1 --realign 0 --max-rejected 2147483647 --max-accept 2147483647 --add-self-matches 0 --db-load-mode 0 --pca 1 --pcb 1.5 --score-bias 0 --gap-open nucl:5,aa:11 --gap-extend nucl:2,aa:1 --zdrop 40 --threads 4 --compressed 0 -v 3 \n","\n","Compute score and coverage\n","Query database size: 756 type: Aminoacid\n","Target database size: 562755 type: Aminoacid\n","Calculation of alignments\n","[=================================================================] 100.00% 756 6s 648ms\n","Time for merging to resultsDB_53: 0h 0m 0s 0ms\n","\n","201987 alignments calculated.\n","59953 sequence pairs passed the thresholds (0.296816 of overall calculated).\n","79.302910 hits per query sequence.\n","Time for processing: 0h 0m 6s 701ms\n","convertalis /content/mmseqs2/tmp/23/candidateDB_53 /content/mmseqs2/swissprot/swissprotDB /content/mmseqs2/tmp/23/resultsDB_53 /content/mmseqs2/tmp/23/resultDB_secondary.m8 --format-output query,target,evalue,raw \n","\n","MMseqs Version:        \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Substitution matrix    \tnucl:nucleotide.out,aa:blosum62.out\n","Alignment format       \t0\n","Format alignment output\tquery,target,evalue,raw\n","Translation table      \t1\n","Gap open cost          \tnucl:5,aa:11\n","Gap extension cost     \tnucl:2,aa:1\n","Database output        \tfalse\n","Preload mode           \t0\n","Search type            \t0\n","Threads                \t4\n","Compressed             \t0\n","Verbosity              \t3\n","\n","[=================================================================] 100.00% 756 0s 46ms\n","Time for merging to resultDB_secondary.m8: 0h 0m 0s 2ms\n","Time for processing: 0h 0m 0s 136ms\n","found 533 genes\n","\n","\n","Creating weighted directed acyclic graph of all ORFs and subORFs on contig 2...\n","\n"],"name":"stdout"},{"output_type":"display_data","data":{"application/vnd.jupyter.widget-view+json":{"model_id":"498794a26d4349288751a0bd1224b452","version_minor":0,"version_major":2},"text/plain":["HBox(children=(FloatProgress(value=0.0, max=5811.0), HTML(value='')))"]},"metadata":{"tags":[]}},{"output_type":"stream","text":["\n","Maximizing geneiness...\n","\n","Filtering predictions with mmseqs2...\n","createdb /content/mmseqs2/tmp/24/candidate_genes_35.fasta /content/mmseqs2/tmp/24/candidateDB_35 \n","\n","MMseqs Version:       \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Database type         \t0\n","Shuffle input database\ttrue\n","Createdb mode         \t0\n","Write lookup file     \t1\n","Offset of numeric ids \t0\n","Compressed            \t0\n","Verbosity             \t3\n","\n","Converting sequences\n","[506] 0s 2ms\n","Time for merging to candidateDB_35_h: 0h 0m 0s 2ms\n","Time for merging to candidateDB_35: 0h 0m 0s 2ms\n","Database type: Aminoacid\n","Time for processing: 0h 0m 0s 10ms\n","createdb /content/mmseqs2/tmp/24/candidate_genes_53.fasta /content/mmseqs2/tmp/24/candidateDB_53 \n","\n","MMseqs Version:       \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Database type         \t0\n","Shuffle input database\ttrue\n","Createdb mode         \t0\n","Write lookup file     \t1\n","Offset of numeric ids \t0\n","Compressed            \t0\n","Verbosity             \t3\n","\n","Converting sequences\n","[506] 0s 3ms\n","Time for merging to candidateDB_53_h: 0h 0m 0s 3ms\n","Time for merging to candidateDB_53: 0h 0m 0s 2ms\n","Database type: Aminoacid\n","Time for processing: 0h 0m 0s 11ms\n","search -s 7.0 /content/mmseqs2/tmp/24/candidateDB_35 /content/mmseqs2/genexa/genexaDB /content/mmseqs2/tmp/24/resultsDB_35 /content/mmseqs2/tmp/24 \n","\n","MMseqs Version:                     \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Substitution matrix                 \tnucl:nucleotide.out,aa:blosum62.out\n","Add backtrace                       \tfalse\n","Alignment mode                      \t2\n","Allow wrapped scoring               \tfalse\n","E-value threshold                   \t0.001\n","Seq. id. threshold                  \t0\n","Min alignment length                \t0\n","Seq. id. mode                       \t0\n","Alternative alignments              \t0\n","Coverage threshold                  \t0\n","Coverage mode                       \t0\n","Max sequence length                 \t65535\n","Compositional bias                  \t1\n","Realign hits                        \tfalse\n","Max reject                          \t2147483647\n","Max accept                          \t2147483647\n","Include identical seq. id.          \tfalse\n","Preload mode                        \t0\n","Pseudo count a                      \t1\n","Pseudo count b                      \t1.5\n","Score bias                          \t0\n","Gap open cost                       \tnucl:5,aa:11\n","Gap extension cost                  \tnucl:2,aa:1\n","Zdrop                               \t40\n","Threads                             \t4\n","Compressed                          \t0\n","Verbosity                           \t3\n","Seed substitution matrix            \tnucl:nucleotide.out,aa:VTML80.out\n","Sensitivity                         \t7\n","k-mer length                        \t0\n","k-score                             \t2147483647\n","Alphabet size                       \tnucl:5,aa:21\n","Max results per query               \t300\n","Split database                      \t0\n","Split mode                          \t2\n","Split memory limit                  \t0\n","Diagonal scoring                    \ttrue\n","Exact k-mer matching                \t0\n","Mask residues                       \t1\n","Mask lower case residues            \t0\n","Minimum diagonal score              \t15\n","Spaced k-mers                       \t1\n","Spaced k-mer pattern                \t\n","Local temporary path                \t\n","Rescore mode                        \t0\n","Remove hits by seq. id. and coverage\tfalse\n","Sort results                        \t0\n","Mask profile                        \t1\n","Profile e-value threshold           \t0.1\n","Global sequence weighting           \tfalse\n","Allow deletions                     \tfalse\n","Filter MSA                          \t1\n","Maximum seq. id. threshold          \t0.9\n","Minimum seq. id.                    \t0\n","Minimum score per column            \t-20\n","Minimum coverage                    \t0\n","Select N most diverse seqs          \t1000\n","Omit consensus                      \tfalse\n","Min codons in orf                   \t30\n","Max codons in length                \t32734\n","Max orf gaps                        \t2147483647\n","Contig start mode                   \t2\n","Contig end mode                     \t2\n","Orf start mode                      \t1\n","Forward frames                      \t1,2,3\n","Reverse frames                      \t1,2,3\n","Translation table                   \t1\n","Translate orf                       \t0\n","Use all table starts                \tfalse\n","Offset of numeric ids               \t0\n","Create lookup                       \t0\n","Add orf stop                        \tfalse\n","Chain overlapping alignments        \t0\n","Merge query                         \t1\n","Search type                         \t0\n","Search iterations                   \t1\n","Start sensitivity                   \t4\n","Search steps                        \t1\n","Slice search mode                   \tfalse\n","Strand selection                    \t1\n","Disk space limit                    \t0\n","MPI runner                          \t\n","Force restart with latest tmp       \tfalse\n","Remove temporary files              \tfalse\n","\n","prefilter /content/mmseqs2/tmp/24/candidateDB_35 /content/mmseqs2/genexa/genexaDB.idx /content/mmseqs2/tmp/24/7617718506261673119/pref_0 --sub-mat nucl:nucleotide.out,aa:blosum62.out --seed-sub-mat nucl:nucleotide.out,aa:VTML80.out -k 0 --k-score 2147483647 --alph-size nucl:5,aa:21 --max-seq-len 65535 --max-seqs 300 --split 0 --split-mode 2 --split-memory-limit 0 -c 0 --cov-mode 0 --comp-bias-corr 1 --diag-score 1 --exact-kmer-matching 0 --mask 1 --mask-lower-case 0 --min-ungapped-score 15 --add-self-matches 0 --spaced-kmer-mode 1 --db-load-mode 0 --pca 1 --pcb 1.5 --threads 4 --compressed 0 -v 3 -s 7.0 \n","\n","Index version: 16\n","Generated by:  16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","ScoreMatrix:  VTML80.out\n","Query database size: 522 type: Aminoacid\n","Estimated memory consumption: 2G\n","Target database size: 468351 type: Aminoacid\n","Process prefiltering step 1 of 1\n","\n","k-mer similarity threshold: 100\n","Starting prefiltering scores calculation (step 1 of 1)\n","Query db start 1 to 522\n","Target db start 1 to 468351\n","[=================================================================] 100.00% 522 3s 966ms\n","\n","718.033756 k-mers per position\n","633004 DB matches per sequence\n","22 overflows\n","0 queries produce too many hits (truncated result)\n","270 sequences passed prefiltering per query sequence\n","300 median result list length\n","1 sequences with 0 size result lists\n","Time for merging to pref_0: 0h 0m 0s 0ms\n","Time for processing: 0h 0m 5s 390ms\n","align /content/mmseqs2/tmp/24/candidateDB_35 /content/mmseqs2/genexa/genexaDB.idx /content/mmseqs2/tmp/24/7617718506261673119/pref_0 /content/mmseqs2/tmp/24/resultsDB_35 --sub-mat nucl:nucleotide.out,aa:blosum62.out -a 0 --alignment-mode 2 --wrapped-scoring 0 -e 0.001 --min-seq-id 0 --min-aln-len 0 --seq-id-mode 0 --alt-ali 0 -c 0 --cov-mode 0 --max-seq-len 65535 --comp-bias-corr 1 --realign 0 --max-rejected 2147483647 --max-accept 2147483647 --add-self-matches 0 --db-load-mode 0 --pca 1 --pcb 1.5 --score-bias 0 --gap-open nucl:5,aa:11 --gap-extend nucl:2,aa:1 --zdrop 40 --threads 4 --compressed 0 -v 3 \n","\n","Index version: 16\n","Generated by:  16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","ScoreMatrix:  VTML80.out\n","Compute score and coverage\n","Query database size: 522 type: Aminoacid\n","Target database size: 468351 type: Aminoacid\n","Calculation of alignments\n","[=================================================================] 100.00% 522 5s 352ms\n","Time for merging to resultsDB_35: 0h 0m 0s 0ms\n","\n","140982 alignments calculated.\n","61227 sequence pairs passed the thresholds (0.434289 of overall calculated).\n","117.293106 hits per query sequence.\n","Time for processing: 0h 0m 5s 375ms\n","convertalis /content/mmseqs2/tmp/24/candidateDB_35 /content/mmseqs2/genexa/genexaDB /content/mmseqs2/tmp/24/resultsDB_35 /content/mmseqs2/tmp/24/resultDB_genexa.m8 --format-output query,target,evalue,raw \n","\n","MMseqs Version:        \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Substitution matrix    \tnucl:nucleotide.out,aa:blosum62.out\n","Alignment format       \t0\n","Format alignment output\tquery,target,evalue,raw\n","Translation table      \t1\n","Gap open cost          \tnucl:5,aa:11\n","Gap extension cost     \tnucl:2,aa:1\n","Database output        \tfalse\n","Preload mode           \t0\n","Search type            \t0\n","Threads                \t4\n","Compressed             \t0\n","Verbosity              \t3\n","\n","[=================================================================] 100.00% 522 0s 49ms\n","Time for merging to resultDB_genexa.m8: 0h 0m 0s 2ms\n","Time for processing: 0h 0m 0s 122ms\n","search -s 7.0 /content/mmseqs2/tmp/24/candidateDB_53 /content/mmseqs2/swissprot/swissprotDB /content/mmseqs2/tmp/24/resultsDB_53 /content/mmseqs2/tmp/24 \n","\n","MMseqs Version:                     \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Substitution matrix                 \tnucl:nucleotide.out,aa:blosum62.out\n","Add backtrace                       \tfalse\n","Alignment mode                      \t2\n","Allow wrapped scoring               \tfalse\n","E-value threshold                   \t0.001\n","Seq. id. threshold                  \t0\n","Min alignment length                \t0\n","Seq. id. mode                       \t0\n","Alternative alignments              \t0\n","Coverage threshold                  \t0\n","Coverage mode                       \t0\n","Max sequence length                 \t65535\n","Compositional bias                  \t1\n","Realign hits                        \tfalse\n","Max reject                          \t2147483647\n","Max accept                          \t2147483647\n","Include identical seq. id.          \tfalse\n","Preload mode                        \t0\n","Pseudo count a                      \t1\n","Pseudo count b                      \t1.5\n","Score bias                          \t0\n","Gap open cost                       \tnucl:5,aa:11\n","Gap extension cost                  \tnucl:2,aa:1\n","Zdrop                               \t40\n","Threads                             \t4\n","Compressed                          \t0\n","Verbosity                           \t3\n","Seed substitution matrix            \tnucl:nucleotide.out,aa:VTML80.out\n","Sensitivity                         \t7\n","k-mer length                        \t0\n","k-score                             \t2147483647\n","Alphabet size                       \tnucl:5,aa:21\n","Max results per query               \t300\n","Split database                      \t0\n","Split mode                          \t2\n","Split memory limit                  \t0\n","Diagonal scoring                    \ttrue\n","Exact k-mer matching                \t0\n","Mask residues                       \t1\n","Mask lower case residues            \t0\n","Minimum diagonal score              \t15\n","Spaced k-mers                       \t1\n","Spaced k-mer pattern                \t\n","Local temporary path                \t\n","Rescore mode                        \t0\n","Remove hits by seq. id. and coverage\tfalse\n","Sort results                        \t0\n","Mask profile                        \t1\n","Profile e-value threshold           \t0.1\n","Global sequence weighting           \tfalse\n","Allow deletions                     \tfalse\n","Filter MSA                          \t1\n","Maximum seq. id. threshold          \t0.9\n","Minimum seq. id.                    \t0\n","Minimum score per column            \t-20\n","Minimum coverage                    \t0\n","Select N most diverse seqs          \t1000\n","Omit consensus                      \tfalse\n","Min codons in orf                   \t30\n","Max codons in length                \t32734\n","Max orf gaps                        \t2147483647\n","Contig start mode                   \t2\n","Contig end mode                     \t2\n","Orf start mode                      \t1\n","Forward frames                      \t1,2,3\n","Reverse frames                      \t1,2,3\n","Translation table                   \t1\n","Translate orf                       \t0\n","Use all table starts                \tfalse\n","Offset of numeric ids               \t0\n","Create lookup                       \t0\n","Add orf stop                        \tfalse\n","Chain overlapping alignments        \t0\n","Merge query                         \t1\n","Search type                         \t0\n","Search iterations                   \t1\n","Start sensitivity                   \t4\n","Search steps                        \t1\n","Slice search mode                   \tfalse\n","Strand selection                    \t1\n","Disk space limit                    \t0\n","MPI runner                          \t\n","Force restart with latest tmp       \tfalse\n","Remove temporary files              \tfalse\n","\n","prefilter /content/mmseqs2/tmp/24/candidateDB_53 /content/mmseqs2/swissprot/swissprotDB /content/mmseqs2/tmp/24/7759997783163451389/pref_0 --sub-mat nucl:nucleotide.out,aa:blosum62.out --seed-sub-mat nucl:nucleotide.out,aa:VTML80.out -k 0 --k-score 2147483647 --alph-size nucl:5,aa:21 --max-seq-len 65535 --max-seqs 300 --split 0 --split-mode 2 --split-memory-limit 0 -c 0 --cov-mode 0 --comp-bias-corr 1 --diag-score 1 --exact-kmer-matching 0 --mask 1 --mask-lower-case 0 --min-ungapped-score 15 --add-self-matches 0 --spaced-kmer-mode 1 --db-load-mode 0 --pca 1 --pcb 1.5 --threads 4 --compressed 0 -v 3 -s 7.0 \n","\n","Query database size: 522 type: Aminoacid\n","Estimated memory consumption: 2G\n","Target database size: 562755 type: Aminoacid\n","Index table k-mer threshold: 100 at k-mer size 6 \n","Index table: counting k-mers\n","[=================================================================] 100.00% 562.76K 24s 773ms\n","Index table: Masked residues: 3164874\n","Index table: fill\n","[=================================================================] 100.00% 562.76K 18s 177ms\n","Index statistics\n","Entries:          192757622\n","DB size:          1591 MB\n","Avg k-mer size:   3.011838\n","Top 10 k-mers\n","    GPGGTL\t1835\n","    GQSWTV\t1706\n","    WGMFAT\t1638\n","    PGVFEV\t1634\n","    VLWQFW\t1623\n","    AYIRPN\t1587\n","    RSPKGV\t1585\n","    TPHKWY\t1560\n","    HIRVGW\t1548\n","    ITLSPY\t1541\n","Time for index table init: 0h 0m 45s 135ms\n","Process prefiltering step 1 of 1\n","\n","k-mer similarity threshold: 100\n","Starting prefiltering scores calculation (step 1 of 1)\n","Query db start 1 to 522\n","Target db start 1 to 562755\n","[=================================================================] 100.00% 522 4s 358ms\n","\n","717.793852 k-mers per position\n","723438 DB matches per sequence\n","35 overflows\n","0 queries produce too many hits (truncated result)\n","273 sequences passed prefiltering per query sequence\n","300 median result list length\n","3 sequences with 0 size result lists\n","Time for merging to pref_0: 0h 0m 0s 0ms\n","Time for processing: 0h 0m 50s 620ms\n","align /content/mmseqs2/tmp/24/candidateDB_53 /content/mmseqs2/swissprot/swissprotDB /content/mmseqs2/tmp/24/7759997783163451389/pref_0 /content/mmseqs2/tmp/24/resultsDB_53 --sub-mat nucl:nucleotide.out,aa:blosum62.out -a 0 --alignment-mode 2 --wrapped-scoring 0 -e 0.001 --min-seq-id 0 --min-aln-len 0 --seq-id-mode 0 --alt-ali 0 -c 0 --cov-mode 0 --max-seq-len 65535 --comp-bias-corr 1 --realign 0 --max-rejected 2147483647 --max-accept 2147483647 --add-self-matches 0 --db-load-mode 0 --pca 1 --pcb 1.5 --score-bias 0 --gap-open nucl:5,aa:11 --gap-extend nucl:2,aa:1 --zdrop 40 --threads 4 --compressed 0 -v 3 \n","\n","Compute score and coverage\n","Query database size: 522 type: Aminoacid\n","Target database size: 562755 type: Aminoacid\n","Calculation of alignments\n","[=================================================================] 100.00% 522 4s 641ms\n","Time for merging to resultsDB_53: 0h 0m 0s 0ms\n","\n","142813 alignments calculated.\n","44298 sequence pairs passed the thresholds (0.310182 of overall calculated).\n","84.862068 hits per query sequence.\n","Time for processing: 0h 0m 4s 701ms\n","convertalis /content/mmseqs2/tmp/24/candidateDB_53 /content/mmseqs2/swissprot/swissprotDB /content/mmseqs2/tmp/24/resultsDB_53 /content/mmseqs2/tmp/24/resultDB_secondary.m8 --format-output query,target,evalue,raw \n","\n","MMseqs Version:        \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Substitution matrix    \tnucl:nucleotide.out,aa:blosum62.out\n","Alignment format       \t0\n","Format alignment output\tquery,target,evalue,raw\n","Translation table      \t1\n","Gap open cost          \tnucl:5,aa:11\n","Gap extension cost     \tnucl:2,aa:1\n","Database output        \tfalse\n","Preload mode           \t0\n","Search type            \t0\n","Threads                \t4\n","Compressed             \t0\n","Verbosity              \t3\n","\n","[=================================================================] 100.00% 522 0s 33ms\n","Time for merging to resultDB_secondary.m8: 0h 0m 0s 1ms\n","Time for processing: 0h 0m 0s 125ms\n","found 369 genes\n","\n","\n","Creating weighted directed acyclic graph of all ORFs and subORFs on contig 3...\n","\n"],"name":"stdout"},{"output_type":"display_data","data":{"application/vnd.jupyter.widget-view+json":{"model_id":"e60d3a18c6ae4a948beb4c89e82f1fa5","version_minor":0,"version_major":2},"text/plain":["HBox(children=(FloatProgress(value=0.0, max=2159.0), HTML(value='')))"]},"metadata":{"tags":[]}},{"output_type":"stream","text":["\n","Maximizing geneiness...\n","\n","Filtering predictions with mmseqs2...\n","createdb /content/mmseqs2/tmp/25/candidate_genes_35.fasta /content/mmseqs2/tmp/25/candidateDB_35 \n","\n","MMseqs Version:       \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Database type         \t0\n","Shuffle input database\ttrue\n","Createdb mode         \t0\n","Write lookup file     \t1\n","Offset of numeric ids \t0\n","Compressed            \t0\n","Verbosity             \t3\n","\n","Converting sequences\n","[203] 0s 2ms\n","Time for merging to candidateDB_35_h: 0h 0m 0s 2ms\n","Time for merging to candidateDB_35: 0h 0m 0s 2ms\n","Database type: Aminoacid\n","Time for processing: 0h 0m 0s 8ms\n","createdb /content/mmseqs2/tmp/25/candidate_genes_53.fasta /content/mmseqs2/tmp/25/candidateDB_53 \n","\n","MMseqs Version:       \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Database type         \t0\n","Shuffle input database\ttrue\n","Createdb mode         \t0\n","Write lookup file     \t1\n","Offset of numeric ids \t0\n","Compressed            \t0\n","Verbosity             \t3\n","\n","Converting sequences\n","[203] 0s 2ms\n","Time for merging to candidateDB_53_h: 0h 0m 0s 2ms\n","Time for merging to candidateDB_53: 0h 0m 0s 2ms\n","Database type: Aminoacid\n","Time for processing: 0h 0m 0s 8ms\n","search -s 7.0 /content/mmseqs2/tmp/25/candidateDB_35 /content/mmseqs2/genexa/genexaDB /content/mmseqs2/tmp/25/resultsDB_35 /content/mmseqs2/tmp/25 \n","\n","MMseqs Version:                     \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Substitution matrix                 \tnucl:nucleotide.out,aa:blosum62.out\n","Add backtrace                       \tfalse\n","Alignment mode                      \t2\n","Allow wrapped scoring               \tfalse\n","E-value threshold                   \t0.001\n","Seq. id. threshold                  \t0\n","Min alignment length                \t0\n","Seq. id. mode                       \t0\n","Alternative alignments              \t0\n","Coverage threshold                  \t0\n","Coverage mode                       \t0\n","Max sequence length                 \t65535\n","Compositional bias                  \t1\n","Realign hits                        \tfalse\n","Max reject                          \t2147483647\n","Max accept                          \t2147483647\n","Include identical seq. id.          \tfalse\n","Preload mode                        \t0\n","Pseudo count a                      \t1\n","Pseudo count b                      \t1.5\n","Score bias                          \t0\n","Gap open cost                       \tnucl:5,aa:11\n","Gap extension cost                  \tnucl:2,aa:1\n","Zdrop                               \t40\n","Threads                             \t4\n","Compressed                          \t0\n","Verbosity                           \t3\n","Seed substitution matrix            \tnucl:nucleotide.out,aa:VTML80.out\n","Sensitivity                         \t7\n","k-mer length                        \t0\n","k-score                             \t2147483647\n","Alphabet size                       \tnucl:5,aa:21\n","Max results per query               \t300\n","Split database                      \t0\n","Split mode                          \t2\n","Split memory limit                  \t0\n","Diagonal scoring                    \ttrue\n","Exact k-mer matching                \t0\n","Mask residues                       \t1\n","Mask lower case residues            \t0\n","Minimum diagonal score              \t15\n","Spaced k-mers                       \t1\n","Spaced k-mer pattern                \t\n","Local temporary path                \t\n","Rescore mode                        \t0\n","Remove hits by seq. id. and coverage\tfalse\n","Sort results                        \t0\n","Mask profile                        \t1\n","Profile e-value threshold           \t0.1\n","Global sequence weighting           \tfalse\n","Allow deletions                     \tfalse\n","Filter MSA                          \t1\n","Maximum seq. id. threshold          \t0.9\n","Minimum seq. id.                    \t0\n","Minimum score per column            \t-20\n","Minimum coverage                    \t0\n","Select N most diverse seqs          \t1000\n","Omit consensus                      \tfalse\n","Min codons in orf                   \t30\n","Max codons in length                \t32734\n","Max orf gaps                        \t2147483647\n","Contig start mode                   \t2\n","Contig end mode                     \t2\n","Orf start mode                      \t1\n","Forward frames                      \t1,2,3\n","Reverse frames                      \t1,2,3\n","Translation table                   \t1\n","Translate orf                       \t0\n","Use all table starts                \tfalse\n","Offset of numeric ids               \t0\n","Create lookup                       \t0\n","Add orf stop                        \tfalse\n","Chain overlapping alignments        \t0\n","Merge query                         \t1\n","Search type                         \t0\n","Search iterations                   \t1\n","Start sensitivity                   \t4\n","Search steps                        \t1\n","Slice search mode                   \tfalse\n","Strand selection                    \t1\n","Disk space limit                    \t0\n","MPI runner                          \t\n","Force restart with latest tmp       \tfalse\n","Remove temporary files              \tfalse\n","\n","prefilter /content/mmseqs2/tmp/25/candidateDB_35 /content/mmseqs2/genexa/genexaDB.idx /content/mmseqs2/tmp/25/2357432403151977630/pref_0 --sub-mat nucl:nucleotide.out,aa:blosum62.out --seed-sub-mat nucl:nucleotide.out,aa:VTML80.out -k 0 --k-score 2147483647 --alph-size nucl:5,aa:21 --max-seq-len 65535 --max-seqs 300 --split 0 --split-mode 2 --split-memory-limit 0 -c 0 --cov-mode 0 --comp-bias-corr 1 --diag-score 1 --exact-kmer-matching 0 --mask 1 --mask-lower-case 0 --min-ungapped-score 15 --add-self-matches 0 --spaced-kmer-mode 1 --db-load-mode 0 --pca 1 --pcb 1.5 --threads 4 --compressed 0 -v 3 -s 7.0 \n","\n","Index version: 16\n","Generated by:  16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","ScoreMatrix:  VTML80.out\n","Query database size: 219 type: Aminoacid\n","Estimated memory consumption: 2G\n","Target database size: 468351 type: Aminoacid\n","Process prefiltering step 1 of 1\n","\n","k-mer similarity threshold: 100\n","Starting prefiltering scores calculation (step 1 of 1)\n","Query db start 1 to 219\n","Target db start 1 to 468351\n","[=================================================================] 100.00% 219 1s 950ms\n","\n","741.960418 k-mers per position\n","727090 DB matches per sequence\n","15 overflows\n","0 queries produce too many hits (truncated result)\n","273 sequences passed prefiltering per query sequence\n","300 median result list length\n","1 sequences with 0 size result lists\n","Time for merging to pref_0: 0h 0m 0s 0ms\n","Time for processing: 0h 0m 3s 393ms\n","align /content/mmseqs2/tmp/25/candidateDB_35 /content/mmseqs2/genexa/genexaDB.idx /content/mmseqs2/tmp/25/2357432403151977630/pref_0 /content/mmseqs2/tmp/25/resultsDB_35 --sub-mat nucl:nucleotide.out,aa:blosum62.out -a 0 --alignment-mode 2 --wrapped-scoring 0 -e 0.001 --min-seq-id 0 --min-aln-len 0 --seq-id-mode 0 --alt-ali 0 -c 0 --cov-mode 0 --max-seq-len 65535 --comp-bias-corr 1 --realign 0 --max-rejected 2147483647 --max-accept 2147483647 --add-self-matches 0 --db-load-mode 0 --pca 1 --pcb 1.5 --score-bias 0 --gap-open nucl:5,aa:11 --gap-extend nucl:2,aa:1 --zdrop 40 --threads 4 --compressed 0 -v 3 \n","\n","Index version: 16\n","Generated by:  16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","ScoreMatrix:  VTML80.out\n","Compute score and coverage\n","Query database size: 219 type: Aminoacid\n","Target database size: 468351 type: Aminoacid\n","Calculation of alignments\n","[=================================================================] 100.00% 219 2s 985ms\n","Time for merging to resultsDB_35: 0h 0m 0s 0ms\n","\n","59828 alignments calculated.\n","26907 sequence pairs passed the thresholds (0.449739 of overall calculated).\n","122.863014 hits per query sequence.\n","Time for processing: 0h 0m 3s 0ms\n","convertalis /content/mmseqs2/tmp/25/candidateDB_35 /content/mmseqs2/genexa/genexaDB /content/mmseqs2/tmp/25/resultsDB_35 /content/mmseqs2/tmp/25/resultDB_genexa.m8 --format-output query,target,evalue,raw \n","\n","MMseqs Version:        \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Substitution matrix    \tnucl:nucleotide.out,aa:blosum62.out\n","Alignment format       \t0\n","Format alignment output\tquery,target,evalue,raw\n","Translation table      \t1\n","Gap open cost          \tnucl:5,aa:11\n","Gap extension cost     \tnucl:2,aa:1\n","Database output        \tfalse\n","Preload mode           \t0\n","Search type            \t0\n","Threads                \t4\n","Compressed             \t0\n","Verbosity              \t3\n","\n","[=================================================================] 100.00% 219 0s 23ms\n","Time for merging to resultDB_genexa.m8: 0h 0m 0s 1ms\n","Time for processing: 0h 0m 0s 90ms\n","search -s 7.0 /content/mmseqs2/tmp/25/candidateDB_53 /content/mmseqs2/swissprot/swissprotDB /content/mmseqs2/tmp/25/resultsDB_53 /content/mmseqs2/tmp/25 \n","\n","MMseqs Version:                     \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Substitution matrix                 \tnucl:nucleotide.out,aa:blosum62.out\n","Add backtrace                       \tfalse\n","Alignment mode                      \t2\n","Allow wrapped scoring               \tfalse\n","E-value threshold                   \t0.001\n","Seq. id. threshold                  \t0\n","Min alignment length                \t0\n","Seq. id. mode                       \t0\n","Alternative alignments              \t0\n","Coverage threshold                  \t0\n","Coverage mode                       \t0\n","Max sequence length                 \t65535\n","Compositional bias                  \t1\n","Realign hits                        \tfalse\n","Max reject                          \t2147483647\n","Max accept                          \t2147483647\n","Include identical seq. id.          \tfalse\n","Preload mode                        \t0\n","Pseudo count a                      \t1\n","Pseudo count b                      \t1.5\n","Score bias                          \t0\n","Gap open cost                       \tnucl:5,aa:11\n","Gap extension cost                  \tnucl:2,aa:1\n","Zdrop                               \t40\n","Threads                             \t4\n","Compressed                          \t0\n","Verbosity                           \t3\n","Seed substitution matrix            \tnucl:nucleotide.out,aa:VTML80.out\n","Sensitivity                         \t7\n","k-mer length                        \t0\n","k-score                             \t2147483647\n","Alphabet size                       \tnucl:5,aa:21\n","Max results per query               \t300\n","Split database                      \t0\n","Split mode                          \t2\n","Split memory limit                  \t0\n","Diagonal scoring                    \ttrue\n","Exact k-mer matching                \t0\n","Mask residues                       \t1\n","Mask lower case residues            \t0\n","Minimum diagonal score              \t15\n","Spaced k-mers                       \t1\n","Spaced k-mer pattern                \t\n","Local temporary path                \t\n","Rescore mode                        \t0\n","Remove hits by seq. id. and coverage\tfalse\n","Sort results                        \t0\n","Mask profile                        \t1\n","Profile e-value threshold           \t0.1\n","Global sequence weighting           \tfalse\n","Allow deletions                     \tfalse\n","Filter MSA                          \t1\n","Maximum seq. id. threshold          \t0.9\n","Minimum seq. id.                    \t0\n","Minimum score per column            \t-20\n","Minimum coverage                    \t0\n","Select N most diverse seqs          \t1000\n","Omit consensus                      \tfalse\n","Min codons in orf                   \t30\n","Max codons in length                \t32734\n","Max orf gaps                        \t2147483647\n","Contig start mode                   \t2\n","Contig end mode                     \t2\n","Orf start mode                      \t1\n","Forward frames                      \t1,2,3\n","Reverse frames                      \t1,2,3\n","Translation table                   \t1\n","Translate orf                       \t0\n","Use all table starts                \tfalse\n","Offset of numeric ids               \t0\n","Create lookup                       \t0\n","Add orf stop                        \tfalse\n","Chain overlapping alignments        \t0\n","Merge query                         \t1\n","Search type                         \t0\n","Search iterations                   \t1\n","Start sensitivity                   \t4\n","Search steps                        \t1\n","Slice search mode                   \tfalse\n","Strand selection                    \t1\n","Disk space limit                    \t0\n","MPI runner                          \t\n","Force restart with latest tmp       \tfalse\n","Remove temporary files              \tfalse\n","\n","prefilter /content/mmseqs2/tmp/25/candidateDB_53 /content/mmseqs2/swissprot/swissprotDB /content/mmseqs2/tmp/25/7861362771208797372/pref_0 --sub-mat nucl:nucleotide.out,aa:blosum62.out --seed-sub-mat nucl:nucleotide.out,aa:VTML80.out -k 0 --k-score 2147483647 --alph-size nucl:5,aa:21 --max-seq-len 65535 --max-seqs 300 --split 0 --split-mode 2 --split-memory-limit 0 -c 0 --cov-mode 0 --comp-bias-corr 1 --diag-score 1 --exact-kmer-matching 0 --mask 1 --mask-lower-case 0 --min-ungapped-score 15 --add-self-matches 0 --spaced-kmer-mode 1 --db-load-mode 0 --pca 1 --pcb 1.5 --threads 4 --compressed 0 -v 3 -s 7.0 \n","\n","Query database size: 219 type: Aminoacid\n","Estimated memory consumption: 2G\n","Target database size: 562755 type: Aminoacid\n","Index table k-mer threshold: 100 at k-mer size 6 \n","Index table: counting k-mers\n","[=================================================================] 100.00% 562.76K 24s 688ms\n","Index table: Masked residues: 3164874\n","Index table: fill\n","[=================================================================] 100.00% 562.76K 18s 310ms\n","Index statistics\n","Entries:          192757622\n","DB size:          1591 MB\n","Avg k-mer size:   3.011838\n","Top 10 k-mers\n","    GPGGTL\t1835\n","    GQSWTV\t1706\n","    WGMFAT\t1638\n","    PGVFEV\t1634\n","    VLWQFW\t1623\n","    AYIRPN\t1587\n","    RSPKGV\t1585\n","    TPHKWY\t1560\n","    HIRVGW\t1548\n","    ITLSPY\t1541\n","Time for index table init: 0h 0m 45s 126ms\n","Process prefiltering step 1 of 1\n","\n","k-mer similarity threshold: 100\n","Starting prefiltering scores calculation (step 1 of 1)\n","Query db start 1 to 219\n","Target db start 1 to 562755\n","[=================================================================] 100.00% 219 2s 119ms\n","\n","743.927231 k-mers per position\n","834139 DB matches per sequence\n","25 overflows\n","0 queries produce too many hits (truncated result)\n","276 sequences passed prefiltering per query sequence\n","300 median result list length\n","1 sequences with 0 size result lists\n","Time for merging to pref_0: 0h 0m 0s 0ms\n","Time for processing: 0h 0m 48s 405ms\n","align /content/mmseqs2/tmp/25/candidateDB_53 /content/mmseqs2/swissprot/swissprotDB /content/mmseqs2/tmp/25/7861362771208797372/pref_0 /content/mmseqs2/tmp/25/resultsDB_53 --sub-mat nucl:nucleotide.out,aa:blosum62.out -a 0 --alignment-mode 2 --wrapped-scoring 0 -e 0.001 --min-seq-id 0 --min-aln-len 0 --seq-id-mode 0 --alt-ali 0 -c 0 --cov-mode 0 --max-seq-len 65535 --comp-bias-corr 1 --realign 0 --max-rejected 2147483647 --max-accept 2147483647 --add-self-matches 0 --db-load-mode 0 --pca 1 --pcb 1.5 --score-bias 0 --gap-open nucl:5,aa:11 --gap-extend nucl:2,aa:1 --zdrop 40 --threads 4 --compressed 0 -v 3 \n","\n","Compute score and coverage\n","Query database size: 219 type: Aminoacid\n","Target database size: 562755 type: Aminoacid\n","Calculation of alignments\n","[=================================================================] 100.00% 219 2s 294ms\n","Time for merging to resultsDB_53: 0h 0m 0s 0ms\n","\n","60475 alignments calculated.\n","18198 sequence pairs passed the thresholds (0.300918 of overall calculated).\n","83.095894 hits per query sequence.\n","Time for processing: 0h 0m 2s 342ms\n","convertalis /content/mmseqs2/tmp/25/candidateDB_53 /content/mmseqs2/swissprot/swissprotDB /content/mmseqs2/tmp/25/resultsDB_53 /content/mmseqs2/tmp/25/resultDB_secondary.m8 --format-output query,target,evalue,raw \n","\n","MMseqs Version:        \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Substitution matrix    \tnucl:nucleotide.out,aa:blosum62.out\n","Alignment format       \t0\n","Format alignment output\tquery,target,evalue,raw\n","Translation table      \t1\n","Gap open cost          \tnucl:5,aa:11\n","Gap extension cost     \tnucl:2,aa:1\n","Database output        \tfalse\n","Preload mode           \t0\n","Search type            \t0\n","Threads                \t4\n","Compressed             \t0\n","Verbosity              \t3\n","\n","[=================================================================] 100.00% 219 0s 15ms\n","Time for merging to resultDB_secondary.m8: 0h 0m 0s 3ms\n","Time for processing: 0h 0m 0s 108ms\n","found 142 genes\n","\n","\n","Reading fasta file GCF_000525675.1_ASM52567v1_genomic.fna.gz...\n","\n","Finding and translating open reading frames...\n","\n","Encoding amino acids...\n","\n","Applying kmer prefilter...\n","\n","Scoring ORFs with temporal convolutional network...\n","\n"],"name":"stdout"},{"output_type":"display_data","data":{"application/vnd.jupyter.widget-view+json":{"model_id":"42c7bcfeeba04f34910e1b23a09b1ff5","version_minor":0,"version_major":2},"text/plain":["HBox(children=(FloatProgress(value=0.0, max=45.0), HTML(value='')))"]},"metadata":{"tags":[]}},{"output_type":"stream","text":["\n","Scoring translation initiation sites...\n","\n"],"name":"stdout"},{"output_type":"display_data","data":{"application/vnd.jupyter.widget-view+json":{"model_id":"b56a4f4aea4e4125b30b44c47fa59bcd","version_minor":0,"version_major":2},"text/plain":["HBox(children=(FloatProgress(value=0.0, max=14.0), HTML(value='')))"]},"metadata":{"tags":[]}},{"output_type":"stream","text":["\n","Creating weighted directed acyclic graph of all ORFs and subORFs on contig 0...\n","\n"],"name":"stdout"},{"output_type":"display_data","data":{"application/vnd.jupyter.widget-view+json":{"model_id":"c1005321f2dd4114bc203299b14edd55","version_minor":0,"version_major":2},"text/plain":["HBox(children=(FloatProgress(value=0.0, max=134931.0), HTML(value='')))"]},"metadata":{"tags":[]}},{"output_type":"stream","text":["\n","Maximizing geneiness...\n","\n","Filtering predictions with mmseqs2...\n","createdb /content/mmseqs2/tmp/26/candidate_genes_35.fasta /content/mmseqs2/tmp/26/candidateDB_35 \n","\n","MMseqs Version:       \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Database type         \t0\n","Shuffle input database\ttrue\n","Createdb mode         \t0\n","Write lookup file     \t1\n","Offset of numeric ids \t0\n","Compressed            \t0\n","Verbosity             \t3\n","\n","Converting sequences\n","[4344] 0s 6ms\n","Time for merging to candidateDB_35_h: 0h 0m 0s 6ms\n","Time for merging to candidateDB_35: 0h 0m 0s 4ms\n","Database type: Aminoacid\n","Time for processing: 0h 0m 0s 23ms\n","createdb /content/mmseqs2/tmp/26/candidate_genes_53.fasta /content/mmseqs2/tmp/26/candidateDB_53 \n","\n","MMseqs Version:       \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Database type         \t0\n","Shuffle input database\ttrue\n","Createdb mode         \t0\n","Write lookup file     \t1\n","Offset of numeric ids \t0\n","Compressed            \t0\n","Verbosity             \t3\n","\n","Converting sequences\n","[4344] 0s 7ms\n","Time for merging to candidateDB_53_h: 0h 0m 0s 3ms\n","Time for merging to candidateDB_53: 0h 0m 0s 5ms\n","Database type: Aminoacid\n","Time for processing: 0h 0m 0s 24ms\n","search -s 7.0 /content/mmseqs2/tmp/26/candidateDB_35 /content/mmseqs2/genexa/genexaDB /content/mmseqs2/tmp/26/resultsDB_35 /content/mmseqs2/tmp/26 \n","\n","MMseqs Version:                     \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Substitution matrix                 \tnucl:nucleotide.out,aa:blosum62.out\n","Add backtrace                       \tfalse\n","Alignment mode                      \t2\n","Allow wrapped scoring               \tfalse\n","E-value threshold                   \t0.001\n","Seq. id. threshold                  \t0\n","Min alignment length                \t0\n","Seq. id. mode                       \t0\n","Alternative alignments              \t0\n","Coverage threshold                  \t0\n","Coverage mode                       \t0\n","Max sequence length                 \t65535\n","Compositional bias                  \t1\n","Realign hits                        \tfalse\n","Max reject                          \t2147483647\n","Max accept                          \t2147483647\n","Include identical seq. id.          \tfalse\n","Preload mode                        \t0\n","Pseudo count a                      \t1\n","Pseudo count b                      \t1.5\n","Score bias                          \t0\n","Gap open cost                       \tnucl:5,aa:11\n","Gap extension cost                  \tnucl:2,aa:1\n","Zdrop                               \t40\n","Threads                             \t4\n","Compressed                          \t0\n","Verbosity                           \t3\n","Seed substitution matrix            \tnucl:nucleotide.out,aa:VTML80.out\n","Sensitivity                         \t7\n","k-mer length                        \t0\n","k-score                             \t2147483647\n","Alphabet size                       \tnucl:5,aa:21\n","Max results per query               \t300\n","Split database                      \t0\n","Split mode                          \t2\n","Split memory limit                  \t0\n","Diagonal scoring                    \ttrue\n","Exact k-mer matching                \t0\n","Mask residues                       \t1\n","Mask lower case residues            \t0\n","Minimum diagonal score              \t15\n","Spaced k-mers                       \t1\n","Spaced k-mer pattern                \t\n","Local temporary path                \t\n","Rescore mode                        \t0\n","Remove hits by seq. id. and coverage\tfalse\n","Sort results                        \t0\n","Mask profile                        \t1\n","Profile e-value threshold           \t0.1\n","Global sequence weighting           \tfalse\n","Allow deletions                     \tfalse\n","Filter MSA                          \t1\n","Maximum seq. id. threshold          \t0.9\n","Minimum seq. id.                    \t0\n","Minimum score per column            \t-20\n","Minimum coverage                    \t0\n","Select N most diverse seqs          \t1000\n","Omit consensus                      \tfalse\n","Min codons in orf                   \t30\n","Max codons in length                \t32734\n","Max orf gaps                        \t2147483647\n","Contig start mode                   \t2\n","Contig end mode                     \t2\n","Orf start mode                      \t1\n","Forward frames                      \t1,2,3\n","Reverse frames                      \t1,2,3\n","Translation table                   \t1\n","Translate orf                       \t0\n","Use all table starts                \tfalse\n","Offset of numeric ids               \t0\n","Create lookup                       \t0\n","Add orf stop                        \tfalse\n","Chain overlapping alignments        \t0\n","Merge query                         \t1\n","Search type                         \t0\n","Search iterations                   \t1\n","Start sensitivity                   \t4\n","Search steps                        \t1\n","Slice search mode                   \tfalse\n","Strand selection                    \t1\n","Disk space limit                    \t0\n","MPI runner                          \t\n","Force restart with latest tmp       \tfalse\n","Remove temporary files              \tfalse\n","\n","prefilter /content/mmseqs2/tmp/26/candidateDB_35 /content/mmseqs2/genexa/genexaDB.idx /content/mmseqs2/tmp/26/15543890373751833757/pref_0 --sub-mat nucl:nucleotide.out,aa:blosum62.out --seed-sub-mat nucl:nucleotide.out,aa:VTML80.out -k 0 --k-score 2147483647 --alph-size nucl:5,aa:21 --max-seq-len 65535 --max-seqs 300 --split 0 --split-mode 2 --split-memory-limit 0 -c 0 --cov-mode 0 --comp-bias-corr 1 --diag-score 1 --exact-kmer-matching 0 --mask 1 --mask-lower-case 0 --min-ungapped-score 15 --add-self-matches 0 --spaced-kmer-mode 1 --db-load-mode 0 --pca 1 --pcb 1.5 --threads 4 --compressed 0 -v 3 -s 7.0 \n","\n","Index version: 16\n","Generated by:  16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","ScoreMatrix:  VTML80.out\n","Query database size: 4411 type: Aminoacid\n","Estimated memory consumption: 2G\n","Target database size: 468351 type: Aminoacid\n","Process prefiltering step 1 of 1\n","\n","k-mer similarity threshold: 100\n","Starting prefiltering scores calculation (step 1 of 1)\n","Query db start 1 to 4411\n","Target db start 1 to 468351\n","[=================================================================] 100.00% 4.41K 32s 106ms\n","\n","914.280035 k-mers per position\n","546359 DB matches per sequence\n","137 overflows\n","0 queries produce too many hits (truncated result)\n","275 sequences passed prefiltering per query sequence\n","300 median result list length\n","2 sequences with 0 size result lists\n","Time for merging to pref_0: 0h 0m 0s 1ms\n","Time for processing: 0h 0m 33s 549ms\n","align /content/mmseqs2/tmp/26/candidateDB_35 /content/mmseqs2/genexa/genexaDB.idx /content/mmseqs2/tmp/26/15543890373751833757/pref_0 /content/mmseqs2/tmp/26/resultsDB_35 --sub-mat nucl:nucleotide.out,aa:blosum62.out -a 0 --alignment-mode 2 --wrapped-scoring 0 -e 0.001 --min-seq-id 0 --min-aln-len 0 --seq-id-mode 0 --alt-ali 0 -c 0 --cov-mode 0 --max-seq-len 65535 --comp-bias-corr 1 --realign 0 --max-rejected 2147483647 --max-accept 2147483647 --add-self-matches 0 --db-load-mode 0 --pca 1 --pcb 1.5 --score-bias 0 --gap-open nucl:5,aa:11 --gap-extend nucl:2,aa:1 --zdrop 40 --threads 4 --compressed 0 -v 3 \n","\n","Index version: 16\n","Generated by:  16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","ScoreMatrix:  VTML80.out\n","Compute score and coverage\n","Query database size: 4411 type: Aminoacid\n","Target database size: 468351 type: Aminoacid\n","Calculation of alignments\n","[=================================================================] 100.00% 4.41K 26s 313ms\n","Time for merging to resultsDB_35: 0h 0m 0s 1ms\n","\n","1215631 alignments calculated.\n","257853 sequence pairs passed the thresholds (0.212115 of overall calculated).\n","58.456814 hits per query sequence.\n","Time for processing: 0h 0m 26s 348ms\n","convertalis /content/mmseqs2/tmp/26/candidateDB_35 /content/mmseqs2/genexa/genexaDB /content/mmseqs2/tmp/26/resultsDB_35 /content/mmseqs2/tmp/26/resultDB_genexa.m8 --format-output query,target,evalue,raw \n","\n","MMseqs Version:        \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Substitution matrix    \tnucl:nucleotide.out,aa:blosum62.out\n","Alignment format       \t0\n","Format alignment output\tquery,target,evalue,raw\n","Translation table      \t1\n","Gap open cost          \tnucl:5,aa:11\n","Gap extension cost     \tnucl:2,aa:1\n","Database output        \tfalse\n","Preload mode           \t0\n","Search type            \t0\n","Threads                \t4\n","Compressed             \t0\n","Verbosity              \t3\n","\n","[=================================================================] 100.00% 4.41K 0s 198ms\n","Time for merging to resultDB_genexa.m8: 0h 0m 0s 8ms\n","Time for processing: 0h 0m 0s 274ms\n","search -s 7.0 /content/mmseqs2/tmp/26/candidateDB_53 /content/mmseqs2/swissprot/swissprotDB /content/mmseqs2/tmp/26/resultsDB_53 /content/mmseqs2/tmp/26 \n","\n","MMseqs Version:                     \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Substitution matrix                 \tnucl:nucleotide.out,aa:blosum62.out\n","Add backtrace                       \tfalse\n","Alignment mode                      \t2\n","Allow wrapped scoring               \tfalse\n","E-value threshold                   \t0.001\n","Seq. id. threshold                  \t0\n","Min alignment length                \t0\n","Seq. id. mode                       \t0\n","Alternative alignments              \t0\n","Coverage threshold                  \t0\n","Coverage mode                       \t0\n","Max sequence length                 \t65535\n","Compositional bias                  \t1\n","Realign hits                        \tfalse\n","Max reject                          \t2147483647\n","Max accept                          \t2147483647\n","Include identical seq. id.          \tfalse\n","Preload mode                        \t0\n","Pseudo count a                      \t1\n","Pseudo count b                      \t1.5\n","Score bias                          \t0\n","Gap open cost                       \tnucl:5,aa:11\n","Gap extension cost                  \tnucl:2,aa:1\n","Zdrop                               \t40\n","Threads                             \t4\n","Compressed                          \t0\n","Verbosity                           \t3\n","Seed substitution matrix            \tnucl:nucleotide.out,aa:VTML80.out\n","Sensitivity                         \t7\n","k-mer length                        \t0\n","k-score                             \t2147483647\n","Alphabet size                       \tnucl:5,aa:21\n","Max results per query               \t300\n","Split database                      \t0\n","Split mode                          \t2\n","Split memory limit                  \t0\n","Diagonal scoring                    \ttrue\n","Exact k-mer matching                \t0\n","Mask residues                       \t1\n","Mask lower case residues            \t0\n","Minimum diagonal score              \t15\n","Spaced k-mers                       \t1\n","Spaced k-mer pattern                \t\n","Local temporary path                \t\n","Rescore mode                        \t0\n","Remove hits by seq. id. and coverage\tfalse\n","Sort results                        \t0\n","Mask profile                        \t1\n","Profile e-value threshold           \t0.1\n","Global sequence weighting           \tfalse\n","Allow deletions                     \tfalse\n","Filter MSA                          \t1\n","Maximum seq. id. threshold          \t0.9\n","Minimum seq. id.                    \t0\n","Minimum score per column            \t-20\n","Minimum coverage                    \t0\n","Select N most diverse seqs          \t1000\n","Omit consensus                      \tfalse\n","Min codons in orf                   \t30\n","Max codons in length                \t32734\n","Max orf gaps                        \t2147483647\n","Contig start mode                   \t2\n","Contig end mode                     \t2\n","Orf start mode                      \t1\n","Forward frames                      \t1,2,3\n","Reverse frames                      \t1,2,3\n","Translation table                   \t1\n","Translate orf                       \t0\n","Use all table starts                \tfalse\n","Offset of numeric ids               \t0\n","Create lookup                       \t0\n","Add orf stop                        \tfalse\n","Chain overlapping alignments        \t0\n","Merge query                         \t1\n","Search type                         \t0\n","Search iterations                   \t1\n","Start sensitivity                   \t4\n","Search steps                        \t1\n","Slice search mode                   \tfalse\n","Strand selection                    \t1\n","Disk space limit                    \t0\n","MPI runner                          \t\n","Force restart with latest tmp       \tfalse\n","Remove temporary files              \tfalse\n","\n","prefilter /content/mmseqs2/tmp/26/candidateDB_53 /content/mmseqs2/swissprot/swissprotDB /content/mmseqs2/tmp/26/7962727759254143355/pref_0 --sub-mat nucl:nucleotide.out,aa:blosum62.out --seed-sub-mat nucl:nucleotide.out,aa:VTML80.out -k 0 --k-score 2147483647 --alph-size nucl:5,aa:21 --max-seq-len 65535 --max-seqs 300 --split 0 --split-mode 2 --split-memory-limit 0 -c 0 --cov-mode 0 --comp-bias-corr 1 --diag-score 1 --exact-kmer-matching 0 --mask 1 --mask-lower-case 0 --min-ungapped-score 15 --add-self-matches 0 --spaced-kmer-mode 1 --db-load-mode 0 --pca 1 --pcb 1.5 --threads 4 --compressed 0 -v 3 -s 7.0 \n","\n","Query database size: 4411 type: Aminoacid\n","Estimated memory consumption: 2G\n","Target database size: 562755 type: Aminoacid\n","Index table k-mer threshold: 100 at k-mer size 6 \n","Index table: counting k-mers\n","[=================================================================] 100.00% 562.76K 25s 161ms\n","Index table: Masked residues: 3164874\n","Index table: fill\n","[=================================================================] 100.00% 562.76K 19s 549ms\n","Index statistics\n","Entries:          192757622\n","DB size:          1591 MB\n","Avg k-mer size:   3.011838\n","Top 10 k-mers\n","    GPGGTL\t1835\n","    GQSWTV\t1706\n","    WGMFAT\t1638\n","    PGVFEV\t1634\n","    VLWQFW\t1623\n","    AYIRPN\t1587\n","    RSPKGV\t1585\n","    TPHKWY\t1560\n","    HIRVGW\t1548\n","    ITLSPY\t1541\n","Time for index table init: 0h 0m 47s 63ms\n","Process prefiltering step 1 of 1\n","\n","k-mer similarity threshold: 100\n","Starting prefiltering scores calculation (step 1 of 1)\n","Query db start 1 to 4411\n","Target db start 1 to 562755\n","[=================================================================] 100.00% 4.41K 37s 99ms\n","\n","914.404673 k-mers per position\n","676733 DB matches per sequence\n","260 overflows\n","0 queries produce too many hits (truncated result)\n","285 sequences passed prefiltering per query sequence\n","300 median result list length\n","2 sequences with 0 size result lists\n","Time for merging to pref_0: 0h 0m 0s 1ms\n","Time for processing: 0h 1m 25s 422ms\n","align /content/mmseqs2/tmp/26/candidateDB_53 /content/mmseqs2/swissprot/swissprotDB /content/mmseqs2/tmp/26/7962727759254143355/pref_0 /content/mmseqs2/tmp/26/resultsDB_53 --sub-mat nucl:nucleotide.out,aa:blosum62.out -a 0 --alignment-mode 2 --wrapped-scoring 0 -e 0.001 --min-seq-id 0 --min-aln-len 0 --seq-id-mode 0 --alt-ali 0 -c 0 --cov-mode 0 --max-seq-len 65535 --comp-bias-corr 1 --realign 0 --max-rejected 2147483647 --max-accept 2147483647 --add-self-matches 0 --db-load-mode 0 --pca 1 --pcb 1.5 --score-bias 0 --gap-open nucl:5,aa:11 --gap-extend nucl:2,aa:1 --zdrop 40 --threads 4 --compressed 0 -v 3 \n","\n","Compute score and coverage\n","Query database size: 4411 type: Aminoacid\n","Target database size: 562755 type: Aminoacid\n","Calculation of alignments\n","[=================================================================] 100.00% 4.41K 27s 92ms\n","Time for merging to resultsDB_53: 0h 0m 0s 1ms\n","\n","1257838 alignments calculated.\n","211661 sequence pairs passed the thresholds (0.168274 of overall calculated).\n","47.984810 hits per query sequence.\n","Time for processing: 0h 0m 27s 197ms\n","convertalis /content/mmseqs2/tmp/26/candidateDB_53 /content/mmseqs2/swissprot/swissprotDB /content/mmseqs2/tmp/26/resultsDB_53 /content/mmseqs2/tmp/26/resultDB_secondary.m8 --format-output query,target,evalue,raw \n","\n","MMseqs Version:        \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Substitution matrix    \tnucl:nucleotide.out,aa:blosum62.out\n","Alignment format       \t0\n","Format alignment output\tquery,target,evalue,raw\n","Translation table      \t1\n","Gap open cost          \tnucl:5,aa:11\n","Gap extension cost     \tnucl:2,aa:1\n","Database output        \tfalse\n","Preload mode           \t0\n","Search type            \t0\n","Threads                \t4\n","Compressed             \t0\n","Verbosity              \t3\n","\n","[=================================================================] 100.00% 4.41K 0s 151ms\n","Time for merging to resultDB_secondary.m8: 0h 0m 0s 6ms\n","Time for processing: 0h 0m 0s 245ms\n","found 2738 genes\n","\n","\n","Reading fasta file GCF_000219875.1_ASM21987v1_genomic.fna.gz...\n","\n","Finding and translating open reading frames...\n","\n","Encoding amino acids...\n","\n","Applying kmer prefilter...\n","\n","Scoring ORFs with temporal convolutional network...\n","\n"],"name":"stdout"},{"output_type":"display_data","data":{"application/vnd.jupyter.widget-view+json":{"model_id":"7f4a050d4b8742daac16ef0ad9f09807","version_minor":0,"version_major":2},"text/plain":["HBox(children=(FloatProgress(value=0.0, max=70.0), HTML(value='')))"]},"metadata":{"tags":[]}},{"output_type":"stream","text":["\n","Scoring translation initiation sites...\n","\n"],"name":"stdout"},{"output_type":"display_data","data":{"application/vnd.jupyter.widget-view+json":{"model_id":"be0fe09d5db044368c3590f366fa4df6","version_minor":0,"version_major":2},"text/plain":["HBox(children=(FloatProgress(value=0.0, max=21.0), HTML(value='')))"]},"metadata":{"tags":[]}},{"output_type":"stream","text":["\n","Creating weighted directed acyclic graph of all ORFs and subORFs on contig 0...\n","\n"],"name":"stdout"},{"output_type":"display_data","data":{"application/vnd.jupyter.widget-view+json":{"model_id":"f8c70f32e2b84044b4b72d6ded8cb5f1","version_minor":0,"version_major":2},"text/plain":["HBox(children=(FloatProgress(value=0.0, max=209617.0), HTML(value='')))"]},"metadata":{"tags":[]}},{"output_type":"stream","text":["\n","Maximizing geneiness...\n","\n","Filtering predictions with mmseqs2...\n","createdb /content/mmseqs2/tmp/27/candidate_genes_35.fasta /content/mmseqs2/tmp/27/candidateDB_35 \n","\n","MMseqs Version:       \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Database type         \t0\n","Shuffle input database\ttrue\n","Createdb mode         \t0\n","Write lookup file     \t1\n","Offset of numeric ids \t0\n","Compressed            \t0\n","Verbosity             \t3\n","\n","Converting sequences\n","[4950] 0s 8ms\n","Time for merging to candidateDB_35_h: 0h 0m 0s 5ms\n","Time for merging to candidateDB_35: 0h 0m 0s 4ms\n","Database type: Aminoacid\n","Time for processing: 0h 0m 0s 24ms\n","createdb /content/mmseqs2/tmp/27/candidate_genes_53.fasta /content/mmseqs2/tmp/27/candidateDB_53 \n","\n","MMseqs Version:       \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Database type         \t0\n","Shuffle input database\ttrue\n","Createdb mode         \t0\n","Write lookup file     \t1\n","Offset of numeric ids \t0\n","Compressed            \t0\n","Verbosity             \t3\n","\n","Converting sequences\n","[4950] 0s 8ms\n","Time for merging to candidateDB_53_h: 0h 0m 0s 3ms\n","Time for merging to candidateDB_53: 0h 0m 0s 4ms\n","Database type: Aminoacid\n","Time for processing: 0h 0m 0s 23ms\n","search -s 7.0 /content/mmseqs2/tmp/27/candidateDB_35 /content/mmseqs2/genexa/genexaDB /content/mmseqs2/tmp/27/resultsDB_35 /content/mmseqs2/tmp/27 \n","\n","MMseqs Version:                     \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Substitution matrix                 \tnucl:nucleotide.out,aa:blosum62.out\n","Add backtrace                       \tfalse\n","Alignment mode                      \t2\n","Allow wrapped scoring               \tfalse\n","E-value threshold                   \t0.001\n","Seq. id. threshold                  \t0\n","Min alignment length                \t0\n","Seq. id. mode                       \t0\n","Alternative alignments              \t0\n","Coverage threshold                  \t0\n","Coverage mode                       \t0\n","Max sequence length                 \t65535\n","Compositional bias                  \t1\n","Realign hits                        \tfalse\n","Max reject                          \t2147483647\n","Max accept                          \t2147483647\n","Include identical seq. id.          \tfalse\n","Preload mode                        \t0\n","Pseudo count a                      \t1\n","Pseudo count b                      \t1.5\n","Score bias                          \t0\n","Gap open cost                       \tnucl:5,aa:11\n","Gap extension cost                  \tnucl:2,aa:1\n","Zdrop                               \t40\n","Threads                             \t4\n","Compressed                          \t0\n","Verbosity                           \t3\n","Seed substitution matrix            \tnucl:nucleotide.out,aa:VTML80.out\n","Sensitivity                         \t7\n","k-mer length                        \t0\n","k-score                             \t2147483647\n","Alphabet size                       \tnucl:5,aa:21\n","Max results per query               \t300\n","Split database                      \t0\n","Split mode                          \t2\n","Split memory limit                  \t0\n","Diagonal scoring                    \ttrue\n","Exact k-mer matching                \t0\n","Mask residues                       \t1\n","Mask lower case residues            \t0\n","Minimum diagonal score              \t15\n","Spaced k-mers                       \t1\n","Spaced k-mer pattern                \t\n","Local temporary path                \t\n","Rescore mode                        \t0\n","Remove hits by seq. id. and coverage\tfalse\n","Sort results                        \t0\n","Mask profile                        \t1\n","Profile e-value threshold           \t0.1\n","Global sequence weighting           \tfalse\n","Allow deletions                     \tfalse\n","Filter MSA                          \t1\n","Maximum seq. id. threshold          \t0.9\n","Minimum seq. id.                    \t0\n","Minimum score per column            \t-20\n","Minimum coverage                    \t0\n","Select N most diverse seqs          \t1000\n","Omit consensus                      \tfalse\n","Min codons in orf                   \t30\n","Max codons in length                \t32734\n","Max orf gaps                        \t2147483647\n","Contig start mode                   \t2\n","Contig end mode                     \t2\n","Orf start mode                      \t1\n","Forward frames                      \t1,2,3\n","Reverse frames                      \t1,2,3\n","Translation table                   \t1\n","Translate orf                       \t0\n","Use all table starts                \tfalse\n","Offset of numeric ids               \t0\n","Create lookup                       \t0\n","Add orf stop                        \tfalse\n","Chain overlapping alignments        \t0\n","Merge query                         \t1\n","Search type                         \t0\n","Search iterations                   \t1\n","Start sensitivity                   \t4\n","Search steps                        \t1\n","Slice search mode                   \tfalse\n","Strand selection                    \t1\n","Disk space limit                    \t0\n","MPI runner                          \t\n","Force restart with latest tmp       \tfalse\n","Remove temporary files              \tfalse\n","\n","prefilter /content/mmseqs2/tmp/27/candidateDB_35 /content/mmseqs2/genexa/genexaDB.idx /content/mmseqs2/tmp/27/10283604270642138268/pref_0 --sub-mat nucl:nucleotide.out,aa:blosum62.out --seed-sub-mat nucl:nucleotide.out,aa:VTML80.out -k 0 --k-score 2147483647 --alph-size nucl:5,aa:21 --max-seq-len 65535 --max-seqs 300 --split 0 --split-mode 2 --split-memory-limit 0 -c 0 --cov-mode 0 --comp-bias-corr 1 --diag-score 1 --exact-kmer-matching 0 --mask 1 --mask-lower-case 0 --min-ungapped-score 15 --add-self-matches 0 --spaced-kmer-mode 1 --db-load-mode 0 --pca 1 --pcb 1.5 --threads 4 --compressed 0 -v 3 -s 7.0 \n","\n","Index version: 16\n","Generated by:  16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","ScoreMatrix:  VTML80.out\n","Query database size: 4984 type: Aminoacid\n","Estimated memory consumption: 2G\n","Target database size: 468351 type: Aminoacid\n","Process prefiltering step 1 of 1\n","\n","k-mer similarity threshold: 100\n","Starting prefiltering scores calculation (step 1 of 1)\n","Query db start 1 to 4984\n","Target db start 1 to 468351\n","[=================================================================] 100.00% 4.98K 35s 666ms\n","\n","954.356182 k-mers per position\n","559188 DB matches per sequence\n","138 overflows\n","0 queries produce too many hits (truncated result)\n","279 sequences passed prefiltering per query sequence\n","300 median result list length\n","2 sequences with 0 size result lists\n","Time for merging to pref_0: 0h 0m 0s 1ms\n","Time for processing: 0h 0m 37s 96ms\n","align /content/mmseqs2/tmp/27/candidateDB_35 /content/mmseqs2/genexa/genexaDB.idx /content/mmseqs2/tmp/27/10283604270642138268/pref_0 /content/mmseqs2/tmp/27/resultsDB_35 --sub-mat nucl:nucleotide.out,aa:blosum62.out -a 0 --alignment-mode 2 --wrapped-scoring 0 -e 0.001 --min-seq-id 0 --min-aln-len 0 --seq-id-mode 0 --alt-ali 0 -c 0 --cov-mode 0 --max-seq-len 65535 --comp-bias-corr 1 --realign 0 --max-rejected 2147483647 --max-accept 2147483647 --add-self-matches 0 --db-load-mode 0 --pca 1 --pcb 1.5 --score-bias 0 --gap-open nucl:5,aa:11 --gap-extend nucl:2,aa:1 --zdrop 40 --threads 4 --compressed 0 -v 3 \n","\n","Index version: 16\n","Generated by:  16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","ScoreMatrix:  VTML80.out\n","Compute score and coverage\n","Query database size: 4984 type: Aminoacid\n","Target database size: 468351 type: Aminoacid\n","Calculation of alignments\n","[=================================================================] 100.00% 4.98K 32s 237ms\n","Time for merging to resultsDB_35: 0h 0m 0s 1ms\n","\n","1393144 alignments calculated.\n","365413 sequence pairs passed the thresholds (0.262294 of overall calculated).\n","73.317215 hits per query sequence.\n","Time for processing: 0h 0m 32s 266ms\n","convertalis /content/mmseqs2/tmp/27/candidateDB_35 /content/mmseqs2/genexa/genexaDB /content/mmseqs2/tmp/27/resultsDB_35 /content/mmseqs2/tmp/27/resultDB_genexa.m8 --format-output query,target,evalue,raw \n","\n","MMseqs Version:        \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Substitution matrix    \tnucl:nucleotide.out,aa:blosum62.out\n","Alignment format       \t0\n","Format alignment output\tquery,target,evalue,raw\n","Translation table      \t1\n","Gap open cost          \tnucl:5,aa:11\n","Gap extension cost     \tnucl:2,aa:1\n","Database output        \tfalse\n","Preload mode           \t0\n","Search type            \t0\n","Threads                \t4\n","Compressed             \t0\n","Verbosity              \t3\n","\n","[=================================================================] 100.00% 4.98K 0s 277ms\n","Time for merging to resultDB_genexa.m8: 0h 0m 0s 10ms\n","Time for processing: 0h 0m 0s 358ms\n","search -s 7.0 /content/mmseqs2/tmp/27/candidateDB_53 /content/mmseqs2/swissprot/swissprotDB /content/mmseqs2/tmp/27/resultsDB_53 /content/mmseqs2/tmp/27 \n","\n","MMseqs Version:                     \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Substitution matrix                 \tnucl:nucleotide.out,aa:blosum62.out\n","Add backtrace                       \tfalse\n","Alignment mode                      \t2\n","Allow wrapped scoring               \tfalse\n","E-value threshold                   \t0.001\n","Seq. id. threshold                  \t0\n","Min alignment length                \t0\n","Seq. id. mode                       \t0\n","Alternative alignments              \t0\n","Coverage threshold                  \t0\n","Coverage mode                       \t0\n","Max sequence length                 \t65535\n","Compositional bias                  \t1\n","Realign hits                        \tfalse\n","Max reject                          \t2147483647\n","Max accept                          \t2147483647\n","Include identical seq. id.          \tfalse\n","Preload mode                        \t0\n","Pseudo count a                      \t1\n","Pseudo count b                      \t1.5\n","Score bias                          \t0\n","Gap open cost                       \tnucl:5,aa:11\n","Gap extension cost                  \tnucl:2,aa:1\n","Zdrop                               \t40\n","Threads                             \t4\n","Compressed                          \t0\n","Verbosity                           \t3\n","Seed substitution matrix            \tnucl:nucleotide.out,aa:VTML80.out\n","Sensitivity                         \t7\n","k-mer length                        \t0\n","k-score                             \t2147483647\n","Alphabet size                       \tnucl:5,aa:21\n","Max results per query               \t300\n","Split database                      \t0\n","Split mode                          \t2\n","Split memory limit                  \t0\n","Diagonal scoring                    \ttrue\n","Exact k-mer matching                \t0\n","Mask residues                       \t1\n","Mask lower case residues            \t0\n","Minimum diagonal score              \t15\n","Spaced k-mers                       \t1\n","Spaced k-mer pattern                \t\n","Local temporary path                \t\n","Rescore mode                        \t0\n","Remove hits by seq. id. and coverage\tfalse\n","Sort results                        \t0\n","Mask profile                        \t1\n","Profile e-value threshold           \t0.1\n","Global sequence weighting           \tfalse\n","Allow deletions                     \tfalse\n","Filter MSA                          \t1\n","Maximum seq. id. threshold          \t0.9\n","Minimum seq. id.                    \t0\n","Minimum score per column            \t-20\n","Minimum coverage                    \t0\n","Select N most diverse seqs          \t1000\n","Omit consensus                      \tfalse\n","Min codons in orf                   \t30\n","Max codons in length                \t32734\n","Max orf gaps                        \t2147483647\n","Contig start mode                   \t2\n","Contig end mode                     \t2\n","Orf start mode                      \t1\n","Forward frames                      \t1,2,3\n","Reverse frames                      \t1,2,3\n","Translation table                   \t1\n","Translate orf                       \t0\n","Use all table starts                \tfalse\n","Offset of numeric ids               \t0\n","Create lookup                       \t0\n","Add orf stop                        \tfalse\n","Chain overlapping alignments        \t0\n","Merge query                         \t1\n","Search type                         \t0\n","Search iterations                   \t1\n","Start sensitivity                   \t4\n","Search steps                        \t1\n","Slice search mode                   \tfalse\n","Strand selection                    \t1\n","Disk space limit                    \t0\n","MPI runner                          \t\n","Force restart with latest tmp       \tfalse\n","Remove temporary files              \tfalse\n","\n","prefilter /content/mmseqs2/tmp/27/candidateDB_53 /content/mmseqs2/swissprot/swissprotDB /content/mmseqs2/tmp/27/8064092747299489338/pref_0 --sub-mat nucl:nucleotide.out,aa:blosum62.out --seed-sub-mat nucl:nucleotide.out,aa:VTML80.out -k 0 --k-score 2147483647 --alph-size nucl:5,aa:21 --max-seq-len 65535 --max-seqs 300 --split 0 --split-mode 2 --split-memory-limit 0 -c 0 --cov-mode 0 --comp-bias-corr 1 --diag-score 1 --exact-kmer-matching 0 --mask 1 --mask-lower-case 0 --min-ungapped-score 15 --add-self-matches 0 --spaced-kmer-mode 1 --db-load-mode 0 --pca 1 --pcb 1.5 --threads 4 --compressed 0 -v 3 -s 7.0 \n","\n","Query database size: 4984 type: Aminoacid\n","Estimated memory consumption: 2G\n","Target database size: 562755 type: Aminoacid\n","Index table k-mer threshold: 100 at k-mer size 6 \n","Index table: counting k-mers\n","[=================================================================] 100.00% 562.76K 24s 598ms\n","Index table: Masked residues: 3164874\n","Index table: fill\n","[=================================================================] 100.00% 562.76K 17s 788ms\n","Index statistics\n","Entries:          192757622\n","DB size:          1591 MB\n","Avg k-mer size:   3.011838\n","Top 10 k-mers\n","    GPGGTL\t1835\n","    GQSWTV\t1706\n","    WGMFAT\t1638\n","    PGVFEV\t1634\n","    VLWQFW\t1623\n","    AYIRPN\t1587\n","    RSPKGV\t1585\n","    TPHKWY\t1560\n","    HIRVGW\t1548\n","    ITLSPY\t1541\n","Time for index table init: 0h 0m 44s 447ms\n","Process prefiltering step 1 of 1\n","\n","k-mer similarity threshold: 100\n","Starting prefiltering scores calculation (step 1 of 1)\n","Query db start 1 to 4984\n","Target db start 1 to 562755\n","[=================================================================] 100.00% 4.98K 39s 616ms\n","\n","953.996833 k-mers per position\n","645360 DB matches per sequence\n","226 overflows\n","0 queries produce too many hits (truncated result)\n","285 sequences passed prefiltering per query sequence\n","300 median result list length\n","1 sequences with 0 size result lists\n","Time for merging to pref_0: 0h 0m 0s 1ms\n","Time for processing: 0h 1m 25s 210ms\n","align /content/mmseqs2/tmp/27/candidateDB_53 /content/mmseqs2/swissprot/swissprotDB /content/mmseqs2/tmp/27/8064092747299489338/pref_0 /content/mmseqs2/tmp/27/resultsDB_53 --sub-mat nucl:nucleotide.out,aa:blosum62.out -a 0 --alignment-mode 2 --wrapped-scoring 0 -e 0.001 --min-seq-id 0 --min-aln-len 0 --seq-id-mode 0 --alt-ali 0 -c 0 --cov-mode 0 --max-seq-len 65535 --comp-bias-corr 1 --realign 0 --max-rejected 2147483647 --max-accept 2147483647 --add-self-matches 0 --db-load-mode 0 --pca 1 --pcb 1.5 --score-bias 0 --gap-open nucl:5,aa:11 --gap-extend nucl:2,aa:1 --zdrop 40 --threads 4 --compressed 0 -v 3 \n","\n","Compute score and coverage\n","Query database size: 4984 type: Aminoacid\n","Target database size: 562755 type: Aminoacid\n","Calculation of alignments\n","[=================================================================] 100.00% 4.98K 30s 579ms\n","Time for merging to resultsDB_53: 0h 0m 0s 1ms\n","\n","1420624 alignments calculated.\n","270545 sequence pairs passed the thresholds (0.190441 of overall calculated).\n","54.282703 hits per query sequence.\n","Time for processing: 0h 0m 30s 656ms\n","convertalis /content/mmseqs2/tmp/27/candidateDB_53 /content/mmseqs2/swissprot/swissprotDB /content/mmseqs2/tmp/27/resultsDB_53 /content/mmseqs2/tmp/27/resultDB_secondary.m8 --format-output query,target,evalue,raw \n","\n","MMseqs Version:        \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Substitution matrix    \tnucl:nucleotide.out,aa:blosum62.out\n","Alignment format       \t0\n","Format alignment output\tquery,target,evalue,raw\n","Translation table      \t1\n","Gap open cost          \tnucl:5,aa:11\n","Gap extension cost     \tnucl:2,aa:1\n","Database output        \tfalse\n","Preload mode           \t0\n","Search type            \t0\n","Threads                \t4\n","Compressed             \t0\n","Verbosity              \t3\n","\n","[=================================================================] 100.00% 4.98K 0s 193ms\n","Time for merging to resultDB_secondary.m8: 0h 0m 0s 8ms\n","Time for processing: 0h 0m 0s 291ms\n","found 3025 genes\n","\n","\n","Reading fasta file GCF_000212395.1_ASM21239v1_genomic.fna.gz...\n","\n","Finding and translating open reading frames...\n","\n","Encoding amino acids...\n","\n","Applying kmer prefilter...\n","\n","Scoring ORFs with temporal convolutional network...\n","\n"],"name":"stdout"},{"output_type":"display_data","data":{"application/vnd.jupyter.widget-view+json":{"model_id":"b6b4a12026d443bb93e1e812242210b3","version_minor":0,"version_major":2},"text/plain":["HBox(children=(FloatProgress(value=0.0, max=21.0), HTML(value='')))"]},"metadata":{"tags":[]}},{"output_type":"stream","text":["\n","Scoring translation initiation sites...\n","\n"],"name":"stdout"},{"output_type":"display_data","data":{"application/vnd.jupyter.widget-view+json":{"model_id":"9e44a9be7c92461885888134efc64624","version_minor":0,"version_major":2},"text/plain":["HBox(children=(FloatProgress(value=0.0, max=7.0), HTML(value='')))"]},"metadata":{"tags":[]}},{"output_type":"stream","text":["\n","Creating weighted directed acyclic graph of all ORFs and subORFs on contig 0...\n","\n"],"name":"stdout"},{"output_type":"display_data","data":{"application/vnd.jupyter.widget-view+json":{"model_id":"2683050fae9240ad9710728db708861f","version_minor":0,"version_major":2},"text/plain":["HBox(children=(FloatProgress(value=0.0, max=61429.0), HTML(value='')))"]},"metadata":{"tags":[]}},{"output_type":"stream","text":["\n","Maximizing geneiness...\n","\n","Filtering predictions with mmseqs2...\n","createdb /content/mmseqs2/tmp/28/candidate_genes_35.fasta /content/mmseqs2/tmp/28/candidateDB_35 \n","\n","MMseqs Version:       \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Database type         \t0\n","Shuffle input database\ttrue\n","Createdb mode         \t0\n","Write lookup file     \t1\n","Offset of numeric ids \t0\n","Compressed            \t0\n","Verbosity             \t3\n","\n","Converting sequences\n","[2425] 0s 4ms\n","Time for merging to candidateDB_35_h: 0h 0m 0s 4ms\n","Time for merging to candidateDB_35: 0h 0m 0s 3ms\n","Database type: Aminoacid\n","Time for processing: 0h 0m 0s 18ms\n","createdb /content/mmseqs2/tmp/28/candidate_genes_53.fasta /content/mmseqs2/tmp/28/candidateDB_53 \n","\n","MMseqs Version:       \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Database type         \t0\n","Shuffle input database\ttrue\n","Createdb mode         \t0\n","Write lookup file     \t1\n","Offset of numeric ids \t0\n","Compressed            \t0\n","Verbosity             \t3\n","\n","Converting sequences\n","[2425] 0s 5ms\n","Time for merging to candidateDB_53_h: 0h 0m 0s 4ms\n","Time for merging to candidateDB_53: 0h 0m 0s 5ms\n","Database type: Aminoacid\n","Time for processing: 0h 0m 0s 20ms\n","search -s 7.0 /content/mmseqs2/tmp/28/candidateDB_35 /content/mmseqs2/genexa/genexaDB /content/mmseqs2/tmp/28/resultsDB_35 /content/mmseqs2/tmp/28 \n","\n","MMseqs Version:                     \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Substitution matrix                 \tnucl:nucleotide.out,aa:blosum62.out\n","Add backtrace                       \tfalse\n","Alignment mode                      \t2\n","Allow wrapped scoring               \tfalse\n","E-value threshold                   \t0.001\n","Seq. id. threshold                  \t0\n","Min alignment length                \t0\n","Seq. id. mode                       \t0\n","Alternative alignments              \t0\n","Coverage threshold                  \t0\n","Coverage mode                       \t0\n","Max sequence length                 \t65535\n","Compositional bias                  \t1\n","Realign hits                        \tfalse\n","Max reject                          \t2147483647\n","Max accept                          \t2147483647\n","Include identical seq. id.          \tfalse\n","Preload mode                        \t0\n","Pseudo count a                      \t1\n","Pseudo count b                      \t1.5\n","Score bias                          \t0\n","Gap open cost                       \tnucl:5,aa:11\n","Gap extension cost                  \tnucl:2,aa:1\n","Zdrop                               \t40\n","Threads                             \t4\n","Compressed                          \t0\n","Verbosity                           \t3\n","Seed substitution matrix            \tnucl:nucleotide.out,aa:VTML80.out\n","Sensitivity                         \t7\n","k-mer length                        \t0\n","k-score                             \t2147483647\n","Alphabet size                       \tnucl:5,aa:21\n","Max results per query               \t300\n","Split database                      \t0\n","Split mode                          \t2\n","Split memory limit                  \t0\n","Diagonal scoring                    \ttrue\n","Exact k-mer matching                \t0\n","Mask residues                       \t1\n","Mask lower case residues            \t0\n","Minimum diagonal score              \t15\n","Spaced k-mers                       \t1\n","Spaced k-mer pattern                \t\n","Local temporary path                \t\n","Rescore mode                        \t0\n","Remove hits by seq. id. and coverage\tfalse\n","Sort results                        \t0\n","Mask profile                        \t1\n","Profile e-value threshold           \t0.1\n","Global sequence weighting           \tfalse\n","Allow deletions                     \tfalse\n","Filter MSA                          \t1\n","Maximum seq. id. threshold          \t0.9\n","Minimum seq. id.                    \t0\n","Minimum score per column            \t-20\n","Minimum coverage                    \t0\n","Select N most diverse seqs          \t1000\n","Omit consensus                      \tfalse\n","Min codons in orf                   \t30\n","Max codons in length                \t32734\n","Max orf gaps                        \t2147483647\n","Contig start mode                   \t2\n","Contig end mode                     \t2\n","Orf start mode                      \t1\n","Forward frames                      \t1,2,3\n","Reverse frames                      \t1,2,3\n","Translation table                   \t1\n","Translate orf                       \t0\n","Use all table starts                \tfalse\n","Offset of numeric ids               \t0\n","Create lookup                       \t0\n","Add orf stop                        \tfalse\n","Chain overlapping alignments        \t0\n","Merge query                         \t1\n","Search type                         \t0\n","Search iterations                   \t1\n","Start sensitivity                   \t4\n","Search steps                        \t1\n","Slice search mode                   \tfalse\n","Strand selection                    \t1\n","Disk space limit                    \t0\n","MPI runner                          \t\n","Force restart with latest tmp       \tfalse\n","Remove temporary files              \tfalse\n","\n","prefilter /content/mmseqs2/tmp/28/candidateDB_35 /content/mmseqs2/genexa/genexaDB.idx /content/mmseqs2/tmp/28/5023318167532442779/pref_0 --sub-mat nucl:nucleotide.out,aa:blosum62.out --seed-sub-mat nucl:nucleotide.out,aa:VTML80.out -k 0 --k-score 2147483647 --alph-size nucl:5,aa:21 --max-seq-len 65535 --max-seqs 300 --split 0 --split-mode 2 --split-memory-limit 0 -c 0 --cov-mode 0 --comp-bias-corr 1 --diag-score 1 --exact-kmer-matching 0 --mask 1 --mask-lower-case 0 --min-ungapped-score 15 --add-self-matches 0 --spaced-kmer-mode 1 --db-load-mode 0 --pca 1 --pcb 1.5 --threads 4 --compressed 0 -v 3 -s 7.0 \n","\n","Index version: 16\n","Generated by:  16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","ScoreMatrix:  VTML80.out\n","Query database size: 2513 type: Aminoacid\n","Estimated memory consumption: 2G\n","Target database size: 468351 type: Aminoacid\n","Process prefiltering step 1 of 1\n","\n","k-mer similarity threshold: 100\n","Starting prefiltering scores calculation (step 1 of 1)\n","Query db start 1 to 2513\n","Target db start 1 to 468351\n","[=================================================================] 100.00% 2.51K 21s 323ms\n","\n","912.400200 k-mers per position\n","646608 DB matches per sequence\n","95 overflows\n","0 queries produce too many hits (truncated result)\n","273 sequences passed prefiltering per query sequence\n","300 median result list length\n","4 sequences with 0 size result lists\n","Time for merging to pref_0: 0h 0m 0s 1ms\n","Time for processing: 0h 0m 22s 797ms\n","align /content/mmseqs2/tmp/28/candidateDB_35 /content/mmseqs2/genexa/genexaDB.idx /content/mmseqs2/tmp/28/5023318167532442779/pref_0 /content/mmseqs2/tmp/28/resultsDB_35 --sub-mat nucl:nucleotide.out,aa:blosum62.out -a 0 --alignment-mode 2 --wrapped-scoring 0 -e 0.001 --min-seq-id 0 --min-aln-len 0 --seq-id-mode 0 --alt-ali 0 -c 0 --cov-mode 0 --max-seq-len 65535 --comp-bias-corr 1 --realign 0 --max-rejected 2147483647 --max-accept 2147483647 --add-self-matches 0 --db-load-mode 0 --pca 1 --pcb 1.5 --score-bias 0 --gap-open nucl:5,aa:11 --gap-extend nucl:2,aa:1 --zdrop 40 --threads 4 --compressed 0 -v 3 \n","\n","Index version: 16\n","Generated by:  16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","ScoreMatrix:  VTML80.out\n","Compute score and coverage\n","Query database size: 2513 type: Aminoacid\n","Target database size: 468351 type: Aminoacid\n","Calculation of alignments\n","[=================================================================] 100.00% 2.51K 18s 981ms\n","Time for merging to resultsDB_35: 0h 0m 0s 1ms\n","\n","687229 alignments calculated.\n","219651 sequence pairs passed the thresholds (0.319618 of overall calculated).\n","87.405891 hits per query sequence.\n","Time for processing: 0h 0m 19s 54ms\n","convertalis /content/mmseqs2/tmp/28/candidateDB_35 /content/mmseqs2/genexa/genexaDB /content/mmseqs2/tmp/28/resultsDB_35 /content/mmseqs2/tmp/28/resultDB_genexa.m8 --format-output query,target,evalue,raw \n","\n","MMseqs Version:        \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Substitution matrix    \tnucl:nucleotide.out,aa:blosum62.out\n","Alignment format       \t0\n","Format alignment output\tquery,target,evalue,raw\n","Translation table      \t1\n","Gap open cost          \tnucl:5,aa:11\n","Gap extension cost     \tnucl:2,aa:1\n","Database output        \tfalse\n","Preload mode           \t0\n","Search type            \t0\n","Threads                \t4\n","Compressed             \t0\n","Verbosity              \t3\n","\n","[=================================================================] 100.00% 2.51K 0s 163ms\n","Time for merging to resultDB_genexa.m8: 0h 0m 0s 6ms\n","Time for processing: 0h 0m 0s 241ms\n","search -s 7.0 /content/mmseqs2/tmp/28/candidateDB_53 /content/mmseqs2/swissprot/swissprotDB /content/mmseqs2/tmp/28/resultsDB_53 /content/mmseqs2/tmp/28 \n","\n","MMseqs Version:                     \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Substitution matrix                 \tnucl:nucleotide.out,aa:blosum62.out\n","Add backtrace                       \tfalse\n","Alignment mode                      \t2\n","Allow wrapped scoring               \tfalse\n","E-value threshold                   \t0.001\n","Seq. id. threshold                  \t0\n","Min alignment length                \t0\n","Seq. id. mode                       \t0\n","Alternative alignments              \t0\n","Coverage threshold                  \t0\n","Coverage mode                       \t0\n","Max sequence length                 \t65535\n","Compositional bias                  \t1\n","Realign hits                        \tfalse\n","Max reject                          \t2147483647\n","Max accept                          \t2147483647\n","Include identical seq. id.          \tfalse\n","Preload mode                        \t0\n","Pseudo count a                      \t1\n","Pseudo count b                      \t1.5\n","Score bias                          \t0\n","Gap open cost                       \tnucl:5,aa:11\n","Gap extension cost                  \tnucl:2,aa:1\n","Zdrop                               \t40\n","Threads                             \t4\n","Compressed                          \t0\n","Verbosity                           \t3\n","Seed substitution matrix            \tnucl:nucleotide.out,aa:VTML80.out\n","Sensitivity                         \t7\n","k-mer length                        \t0\n","k-score                             \t2147483647\n","Alphabet size                       \tnucl:5,aa:21\n","Max results per query               \t300\n","Split database                      \t0\n","Split mode                          \t2\n","Split memory limit                  \t0\n","Diagonal scoring                    \ttrue\n","Exact k-mer matching                \t0\n","Mask residues                       \t1\n","Mask lower case residues            \t0\n","Minimum diagonal score              \t15\n","Spaced k-mers                       \t1\n","Spaced k-mer pattern                \t\n","Local temporary path                \t\n","Rescore mode                        \t0\n","Remove hits by seq. id. and coverage\tfalse\n","Sort results                        \t0\n","Mask profile                        \t1\n","Profile e-value threshold           \t0.1\n","Global sequence weighting           \tfalse\n","Allow deletions                     \tfalse\n","Filter MSA                          \t1\n","Maximum seq. id. threshold          \t0.9\n","Minimum seq. id.                    \t0\n","Minimum score per column            \t-20\n","Minimum coverage                    \t0\n","Select N most diverse seqs          \t1000\n","Omit consensus                      \tfalse\n","Min codons in orf                   \t30\n","Max codons in length                \t32734\n","Max orf gaps                        \t2147483647\n","Contig start mode                   \t2\n","Contig end mode                     \t2\n","Orf start mode                      \t1\n","Forward frames                      \t1,2,3\n","Reverse frames                      \t1,2,3\n","Translation table                   \t1\n","Translate orf                       \t0\n","Use all table starts                \tfalse\n","Offset of numeric ids               \t0\n","Create lookup                       \t0\n","Add orf stop                        \tfalse\n","Chain overlapping alignments        \t0\n","Merge query                         \t1\n","Search type                         \t0\n","Search iterations                   \t1\n","Start sensitivity                   \t4\n","Search steps                        \t1\n","Slice search mode                   \tfalse\n","Strand selection                    \t1\n","Disk space limit                    \t0\n","MPI runner                          \t\n","Force restart with latest tmp       \tfalse\n","Remove temporary files              \tfalse\n","\n","prefilter /content/mmseqs2/tmp/28/candidateDB_53 /content/mmseqs2/swissprot/swissprotDB /content/mmseqs2/tmp/28/8165457735344835321/pref_0 --sub-mat nucl:nucleotide.out,aa:blosum62.out --seed-sub-mat nucl:nucleotide.out,aa:VTML80.out -k 0 --k-score 2147483647 --alph-size nucl:5,aa:21 --max-seq-len 65535 --max-seqs 300 --split 0 --split-mode 2 --split-memory-limit 0 -c 0 --cov-mode 0 --comp-bias-corr 1 --diag-score 1 --exact-kmer-matching 0 --mask 1 --mask-lower-case 0 --min-ungapped-score 15 --add-self-matches 0 --spaced-kmer-mode 1 --db-load-mode 0 --pca 1 --pcb 1.5 --threads 4 --compressed 0 -v 3 -s 7.0 \n","\n","Query database size: 2513 type: Aminoacid\n","Estimated memory consumption: 2G\n","Target database size: 562755 type: Aminoacid\n","Index table k-mer threshold: 100 at k-mer size 6 \n","Index table: counting k-mers\n","[=================================================================] 100.00% 562.76K 24s 710ms\n","Index table: Masked residues: 3164874\n","Index table: fill\n","[=================================================================] 100.00% 562.76K 18s 51ms\n","Index statistics\n","Entries:          192757622\n","DB size:          1591 MB\n","Avg k-mer size:   3.011838\n","Top 10 k-mers\n","    GPGGTL\t1835\n","    GQSWTV\t1706\n","    WGMFAT\t1638\n","    PGVFEV\t1634\n","    VLWQFW\t1623\n","    AYIRPN\t1587\n","    RSPKGV\t1585\n","    TPHKWY\t1560\n","    HIRVGW\t1548\n","    ITLSPY\t1541\n","Time for index table init: 0h 0m 44s 880ms\n","Process prefiltering step 1 of 1\n","\n","k-mer similarity threshold: 100\n","Starting prefiltering scores calculation (step 1 of 1)\n","Query db start 1 to 2513\n","Target db start 1 to 562755\n","[=================================================================] 100.00% 2.51K 24s 922ms\n","\n","911.164976 k-mers per position\n","825909 DB matches per sequence\n","192 overflows\n","0 queries produce too many hits (truncated result)\n","281 sequences passed prefiltering per query sequence\n","300 median result list length\n","5 sequences with 0 size result lists\n","Time for merging to pref_0: 0h 0m 0s 1ms\n","Time for processing: 0h 1m 10s 981ms\n","align /content/mmseqs2/tmp/28/candidateDB_53 /content/mmseqs2/swissprot/swissprotDB /content/mmseqs2/tmp/28/8165457735344835321/pref_0 /content/mmseqs2/tmp/28/resultsDB_53 --sub-mat nucl:nucleotide.out,aa:blosum62.out -a 0 --alignment-mode 2 --wrapped-scoring 0 -e 0.001 --min-seq-id 0 --min-aln-len 0 --seq-id-mode 0 --alt-ali 0 -c 0 --cov-mode 0 --max-seq-len 65535 --comp-bias-corr 1 --realign 0 --max-rejected 2147483647 --max-accept 2147483647 --add-self-matches 0 --db-load-mode 0 --pca 1 --pcb 1.5 --score-bias 0 --gap-open nucl:5,aa:11 --gap-extend nucl:2,aa:1 --zdrop 40 --threads 4 --compressed 0 -v 3 \n","\n","Compute score and coverage\n","Query database size: 2513 type: Aminoacid\n","Target database size: 562755 type: Aminoacid\n","Calculation of alignments\n","[=================================================================] 100.00% 2.51K 19s 64ms\n","Time for merging to resultsDB_53: 0h 0m 0s 1ms\n","\n","706240 alignments calculated.\n","182086 sequence pairs passed the thresholds (0.257825 of overall calculated).\n","72.457619 hits per query sequence.\n","Time for processing: 0h 0m 19s 158ms\n","convertalis /content/mmseqs2/tmp/28/candidateDB_53 /content/mmseqs2/swissprot/swissprotDB /content/mmseqs2/tmp/28/resultsDB_53 /content/mmseqs2/tmp/28/resultDB_secondary.m8 --format-output query,target,evalue,raw \n","\n","MMseqs Version:        \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Substitution matrix    \tnucl:nucleotide.out,aa:blosum62.out\n","Alignment format       \t0\n","Format alignment output\tquery,target,evalue,raw\n","Translation table      \t1\n","Gap open cost          \tnucl:5,aa:11\n","Gap extension cost     \tnucl:2,aa:1\n","Database output        \tfalse\n","Preload mode           \t0\n","Search type            \t0\n","Threads                \t4\n","Compressed             \t0\n","Verbosity              \t3\n","\n","[=================================================================] 100.00% 2.51K 0s 130ms\n","Time for merging to resultDB_secondary.m8: 0h 0m 0s 5ms\n","Time for processing: 0h 0m 0s 226ms\n","found 1834 genes\n","\n","\n","Reading fasta file GCF_000195295.1_ASM19529v1_genomic.fna.gz...\n","\n","Finding and translating open reading frames...\n","\n","Encoding amino acids...\n","\n","Applying kmer prefilter...\n","\n","Scoring ORFs with temporal convolutional network...\n","\n"],"name":"stdout"},{"output_type":"display_data","data":{"application/vnd.jupyter.widget-view+json":{"model_id":"b2d6d7bd6bf648d6a740aa7a70402d1b","version_minor":0,"version_major":2},"text/plain":["HBox(children=(FloatProgress(value=0.0, max=54.0), HTML(value='')))"]},"metadata":{"tags":[]}},{"output_type":"stream","text":["\n","Scoring translation initiation sites...\n","\n"],"name":"stdout"},{"output_type":"display_data","data":{"application/vnd.jupyter.widget-view+json":{"model_id":"b1a23333630047b0bffb03ba74441786","version_minor":0,"version_major":2},"text/plain":["HBox(children=(FloatProgress(value=0.0, max=17.0), HTML(value='')))"]},"metadata":{"tags":[]}},{"output_type":"stream","text":["\n","Creating weighted directed acyclic graph of all ORFs and subORFs on contig 0...\n","\n"],"name":"stdout"},{"output_type":"display_data","data":{"application/vnd.jupyter.widget-view+json":{"model_id":"1a840f291ae745a19988416f7cc68b12","version_minor":0,"version_major":2},"text/plain":["HBox(children=(FloatProgress(value=0.0, max=161843.0), HTML(value='')))"]},"metadata":{"tags":[]}},{"output_type":"stream","text":["\n","Maximizing geneiness...\n","\n","Filtering predictions with mmseqs2...\n","createdb /content/mmseqs2/tmp/29/candidate_genes_35.fasta /content/mmseqs2/tmp/29/candidateDB_35 \n","\n","MMseqs Version:       \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Database type         \t0\n","Shuffle input database\ttrue\n","Createdb mode         \t0\n","Write lookup file     \t1\n","Offset of numeric ids \t0\n","Compressed            \t0\n","Verbosity             \t3\n","\n","Converting sequences\n","[5758] 0s 8ms\n","Time for merging to candidateDB_35_h: 0h 0m 0s 3ms\n","Time for merging to candidateDB_35: 0h 0m 0s 5ms\n","Database type: Aminoacid\n","Time for processing: 0h 0m 0s 25ms\n","createdb /content/mmseqs2/tmp/29/candidate_genes_53.fasta /content/mmseqs2/tmp/29/candidateDB_53 \n","\n","MMseqs Version:       \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Database type         \t0\n","Shuffle input database\ttrue\n","Createdb mode         \t0\n","Write lookup file     \t1\n","Offset of numeric ids \t0\n","Compressed            \t0\n","Verbosity             \t3\n","\n","Converting sequences\n","[5758] 0s 7ms\n","Time for merging to candidateDB_53_h: 0h 0m 0s 4ms\n","Time for merging to candidateDB_53: 0h 0m 0s 4ms\n","Database type: Aminoacid\n","Time for processing: 0h 0m 0s 23ms\n","search -s 7.0 /content/mmseqs2/tmp/29/candidateDB_35 /content/mmseqs2/genexa/genexaDB /content/mmseqs2/tmp/29/resultsDB_35 /content/mmseqs2/tmp/29 \n","\n","MMseqs Version:                     \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Substitution matrix                 \tnucl:nucleotide.out,aa:blosum62.out\n","Add backtrace                       \tfalse\n","Alignment mode                      \t2\n","Allow wrapped scoring               \tfalse\n","E-value threshold                   \t0.001\n","Seq. id. threshold                  \t0\n","Min alignment length                \t0\n","Seq. id. mode                       \t0\n","Alternative alignments              \t0\n","Coverage threshold                  \t0\n","Coverage mode                       \t0\n","Max sequence length                 \t65535\n","Compositional bias                  \t1\n","Realign hits                        \tfalse\n","Max reject                          \t2147483647\n","Max accept                          \t2147483647\n","Include identical seq. id.          \tfalse\n","Preload mode                        \t0\n","Pseudo count a                      \t1\n","Pseudo count b                      \t1.5\n","Score bias                          \t0\n","Gap open cost                       \tnucl:5,aa:11\n","Gap extension cost                  \tnucl:2,aa:1\n","Zdrop                               \t40\n","Threads                             \t4\n","Compressed                          \t0\n","Verbosity                           \t3\n","Seed substitution matrix            \tnucl:nucleotide.out,aa:VTML80.out\n","Sensitivity                         \t7\n","k-mer length                        \t0\n","k-score                             \t2147483647\n","Alphabet size                       \tnucl:5,aa:21\n","Max results per query               \t300\n","Split database                      \t0\n","Split mode                          \t2\n","Split memory limit                  \t0\n","Diagonal scoring                    \ttrue\n","Exact k-mer matching                \t0\n","Mask residues                       \t1\n","Mask lower case residues            \t0\n","Minimum diagonal score              \t15\n","Spaced k-mers                       \t1\n","Spaced k-mer pattern                \t\n","Local temporary path                \t\n","Rescore mode                        \t0\n","Remove hits by seq. id. and coverage\tfalse\n","Sort results                        \t0\n","Mask profile                        \t1\n","Profile e-value threshold           \t0.1\n","Global sequence weighting           \tfalse\n","Allow deletions                     \tfalse\n","Filter MSA                          \t1\n","Maximum seq. id. threshold          \t0.9\n","Minimum seq. id.                    \t0\n","Minimum score per column            \t-20\n","Minimum coverage                    \t0\n","Select N most diverse seqs          \t1000\n","Omit consensus                      \tfalse\n","Min codons in orf                   \t30\n","Max codons in length                \t32734\n","Max orf gaps                        \t2147483647\n","Contig start mode                   \t2\n","Contig end mode                     \t2\n","Orf start mode                      \t1\n","Forward frames                      \t1,2,3\n","Reverse frames                      \t1,2,3\n","Translation table                   \t1\n","Translate orf                       \t0\n","Use all table starts                \tfalse\n","Offset of numeric ids               \t0\n","Create lookup                       \t0\n","Add orf stop                        \tfalse\n","Chain overlapping alignments        \t0\n","Merge query                         \t1\n","Search type                         \t0\n","Search iterations                   \t1\n","Start sensitivity                   \t4\n","Search steps                        \t1\n","Slice search mode                   \tfalse\n","Strand selection                    \t1\n","Disk space limit                    \t0\n","MPI runner                          \t\n","Force restart with latest tmp       \tfalse\n","Remove temporary files              \tfalse\n","\n","prefilter /content/mmseqs2/tmp/29/candidateDB_35 /content/mmseqs2/genexa/genexaDB.idx /content/mmseqs2/tmp/29/18209776138132298906/pref_0 --sub-mat nucl:nucleotide.out,aa:blosum62.out --seed-sub-mat nucl:nucleotide.out,aa:VTML80.out -k 0 --k-score 2147483647 --alph-size nucl:5,aa:21 --max-seq-len 65535 --max-seqs 300 --split 0 --split-mode 2 --split-memory-limit 0 -c 0 --cov-mode 0 --comp-bias-corr 1 --diag-score 1 --exact-kmer-matching 0 --mask 1 --mask-lower-case 0 --min-ungapped-score 15 --add-self-matches 0 --spaced-kmer-mode 1 --db-load-mode 0 --pca 1 --pcb 1.5 --threads 4 --compressed 0 -v 3 -s 7.0 \n","\n","Index version: 16\n","Generated by:  16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","ScoreMatrix:  VTML80.out\n","Query database size: 5833 type: Aminoacid\n","Estimated memory consumption: 2G\n","Target database size: 468351 type: Aminoacid\n","Process prefiltering step 1 of 1\n","\n","k-mer similarity threshold: 100\n","Starting prefiltering scores calculation (step 1 of 1)\n","Query db start 1 to 5833\n","Target db start 1 to 468351\n","[=================================================================] 100.00% 5.83K 39s 23ms\n","\n","1020.478558 k-mers per position\n","514947 DB matches per sequence\n","244 overflows\n","0 queries produce too many hits (truncated result)\n","277 sequences passed prefiltering per query sequence\n","300 median result list length\n","1 sequences with 0 size result lists\n","Time for merging to pref_0: 0h 0m 0s 1ms\n","Time for processing: 0h 0m 40s 430ms\n","align /content/mmseqs2/tmp/29/candidateDB_35 /content/mmseqs2/genexa/genexaDB.idx /content/mmseqs2/tmp/29/18209776138132298906/pref_0 /content/mmseqs2/tmp/29/resultsDB_35 --sub-mat nucl:nucleotide.out,aa:blosum62.out -a 0 --alignment-mode 2 --wrapped-scoring 0 -e 0.001 --min-seq-id 0 --min-aln-len 0 --seq-id-mode 0 --alt-ali 0 -c 0 --cov-mode 0 --max-seq-len 65535 --comp-bias-corr 1 --realign 0 --max-rejected 2147483647 --max-accept 2147483647 --add-self-matches 0 --db-load-mode 0 --pca 1 --pcb 1.5 --score-bias 0 --gap-open nucl:5,aa:11 --gap-extend nucl:2,aa:1 --zdrop 40 --threads 4 --compressed 0 -v 3 \n","\n","Index version: 16\n","Generated by:  16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","ScoreMatrix:  VTML80.out\n","Compute score and coverage\n","Query database size: 5833 type: Aminoacid\n","Target database size: 468351 type: Aminoacid\n","Calculation of alignments\n","[=================================================================] 100.00% 5.83K 31s 3ms\n","Time for merging to resultsDB_35: 0h 0m 0s 1ms\n","\n","1617527 alignments calculated.\n","303666 sequence pairs passed the thresholds (0.187735 of overall calculated).\n","52.060005 hits per query sequence.\n","Time for processing: 0h 0m 31s 34ms\n","convertalis /content/mmseqs2/tmp/29/candidateDB_35 /content/mmseqs2/genexa/genexaDB /content/mmseqs2/tmp/29/resultsDB_35 /content/mmseqs2/tmp/29/resultDB_genexa.m8 --format-output query,target,evalue,raw \n","\n","MMseqs Version:        \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Substitution matrix    \tnucl:nucleotide.out,aa:blosum62.out\n","Alignment format       \t0\n","Format alignment output\tquery,target,evalue,raw\n","Translation table      \t1\n","Gap open cost          \tnucl:5,aa:11\n","Gap extension cost     \tnucl:2,aa:1\n","Database output        \tfalse\n","Preload mode           \t0\n","Search type            \t0\n","Threads                \t4\n","Compressed             \t0\n","Verbosity              \t3\n","\n","[=================================================================] 100.00% 5.83K 0s 228ms\n","Time for merging to resultDB_genexa.m8: 0h 0m 0s 10ms\n","Time for processing: 0h 0m 0s 308ms\n","search -s 7.0 /content/mmseqs2/tmp/29/candidateDB_53 /content/mmseqs2/swissprot/swissprotDB /content/mmseqs2/tmp/29/resultsDB_53 /content/mmseqs2/tmp/29 \n","\n","MMseqs Version:                     \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Substitution matrix                 \tnucl:nucleotide.out,aa:blosum62.out\n","Add backtrace                       \tfalse\n","Alignment mode                      \t2\n","Allow wrapped scoring               \tfalse\n","E-value threshold                   \t0.001\n","Seq. id. threshold                  \t0\n","Min alignment length                \t0\n","Seq. id. mode                       \t0\n","Alternative alignments              \t0\n","Coverage threshold                  \t0\n","Coverage mode                       \t0\n","Max sequence length                 \t65535\n","Compositional bias                  \t1\n","Realign hits                        \tfalse\n","Max reject                          \t2147483647\n","Max accept                          \t2147483647\n","Include identical seq. id.          \tfalse\n","Preload mode                        \t0\n","Pseudo count a                      \t1\n","Pseudo count b                      \t1.5\n","Score bias                          \t0\n","Gap open cost                       \tnucl:5,aa:11\n","Gap extension cost                  \tnucl:2,aa:1\n","Zdrop                               \t40\n","Threads                             \t4\n","Compressed                          \t0\n","Verbosity                           \t3\n","Seed substitution matrix            \tnucl:nucleotide.out,aa:VTML80.out\n","Sensitivity                         \t7\n","k-mer length                        \t0\n","k-score                             \t2147483647\n","Alphabet size                       \tnucl:5,aa:21\n","Max results per query               \t300\n","Split database                      \t0\n","Split mode                          \t2\n","Split memory limit                  \t0\n","Diagonal scoring                    \ttrue\n","Exact k-mer matching                \t0\n","Mask residues                       \t1\n","Mask lower case residues            \t0\n","Minimum diagonal score              \t15\n","Spaced k-mers                       \t1\n","Spaced k-mer pattern                \t\n","Local temporary path                \t\n","Rescore mode                        \t0\n","Remove hits by seq. id. and coverage\tfalse\n","Sort results                        \t0\n","Mask profile                        \t1\n","Profile e-value threshold           \t0.1\n","Global sequence weighting           \tfalse\n","Allow deletions                     \tfalse\n","Filter MSA                          \t1\n","Maximum seq. id. threshold          \t0.9\n","Minimum seq. id.                    \t0\n","Minimum score per column            \t-20\n","Minimum coverage                    \t0\n","Select N most diverse seqs          \t1000\n","Omit consensus                      \tfalse\n","Min codons in orf                   \t30\n","Max codons in length                \t32734\n","Max orf gaps                        \t2147483647\n","Contig start mode                   \t2\n","Contig end mode                     \t2\n","Orf start mode                      \t1\n","Forward frames                      \t1,2,3\n","Reverse frames                      \t1,2,3\n","Translation table                   \t1\n","Translate orf                       \t0\n","Use all table starts                \tfalse\n","Offset of numeric ids               \t0\n","Create lookup                       \t0\n","Add orf stop                        \tfalse\n","Chain overlapping alignments        \t0\n","Merge query                         \t1\n","Search type                         \t0\n","Search iterations                   \t1\n","Start sensitivity                   \t4\n","Search steps                        \t1\n","Slice search mode                   \tfalse\n","Strand selection                    \t1\n","Disk space limit                    \t0\n","MPI runner                          \t\n","Force restart with latest tmp       \tfalse\n","Remove temporary files              \tfalse\n","\n","prefilter /content/mmseqs2/tmp/29/candidateDB_53 /content/mmseqs2/swissprot/swissprotDB /content/mmseqs2/tmp/29/8266822723390181304/pref_0 --sub-mat nucl:nucleotide.out,aa:blosum62.out --seed-sub-mat nucl:nucleotide.out,aa:VTML80.out -k 0 --k-score 2147483647 --alph-size nucl:5,aa:21 --max-seq-len 65535 --max-seqs 300 --split 0 --split-mode 2 --split-memory-limit 0 -c 0 --cov-mode 0 --comp-bias-corr 1 --diag-score 1 --exact-kmer-matching 0 --mask 1 --mask-lower-case 0 --min-ungapped-score 15 --add-self-matches 0 --spaced-kmer-mode 1 --db-load-mode 0 --pca 1 --pcb 1.5 --threads 4 --compressed 0 -v 3 -s 7.0 \n","\n","Query database size: 5833 type: Aminoacid\n","Estimated memory consumption: 2G\n","Target database size: 562755 type: Aminoacid\n","Index table k-mer threshold: 100 at k-mer size 6 \n","Index table: counting k-mers\n","[=================================================================] 100.00% 562.76K 24s 530ms\n","Index table: Masked residues: 3164874\n","Index table: fill\n","[=================================================================] 100.00% 562.76K 17s 996ms\n","Index statistics\n","Entries:          192757622\n","DB size:          1591 MB\n","Avg k-mer size:   3.011838\n","Top 10 k-mers\n","    GPGGTL\t1835\n","    GQSWTV\t1706\n","    WGMFAT\t1638\n","    PGVFEV\t1634\n","    VLWQFW\t1623\n","    AYIRPN\t1587\n","    RSPKGV\t1585\n","    TPHKWY\t1560\n","    HIRVGW\t1548\n","    ITLSPY\t1541\n","Time for index table init: 0h 0m 44s 647ms\n","Process prefiltering step 1 of 1\n","\n","k-mer similarity threshold: 100\n","Starting prefiltering scores calculation (step 1 of 1)\n","Query db start 1 to 5833\n","Target db start 1 to 562755\n","[=================================================================] 100.00% 5.83K 43s 744ms\n","\n","1021.851134 k-mers per position\n","616984 DB matches per sequence\n","403 overflows\n","0 queries produce too many hits (truncated result)\n","285 sequences passed prefiltering per query sequence\n","300 median result list length\n","2 sequences with 0 size result lists\n","Time for merging to pref_0: 0h 0m 0s 1ms\n","Time for processing: 0h 1m 29s 520ms\n","align /content/mmseqs2/tmp/29/candidateDB_53 /content/mmseqs2/swissprot/swissprotDB /content/mmseqs2/tmp/29/8266822723390181304/pref_0 /content/mmseqs2/tmp/29/resultsDB_53 --sub-mat nucl:nucleotide.out,aa:blosum62.out -a 0 --alignment-mode 2 --wrapped-scoring 0 -e 0.001 --min-seq-id 0 --min-aln-len 0 --seq-id-mode 0 --alt-ali 0 -c 0 --cov-mode 0 --max-seq-len 65535 --comp-bias-corr 1 --realign 0 --max-rejected 2147483647 --max-accept 2147483647 --add-self-matches 0 --db-load-mode 0 --pca 1 --pcb 1.5 --score-bias 0 --gap-open nucl:5,aa:11 --gap-extend nucl:2,aa:1 --zdrop 40 --threads 4 --compressed 0 -v 3 \n","\n","Compute score and coverage\n","Query database size: 5833 type: Aminoacid\n","Target database size: 562755 type: Aminoacid\n","Calculation of alignments\n","[=================================================================] 100.00% 5.83K 30s 986ms\n","Time for merging to resultsDB_53: 0h 0m 0s 2ms\n","\n","1665651 alignments calculated.\n","239646 sequence pairs passed the thresholds (0.143875 of overall calculated).\n","41.084518 hits per query sequence.\n","Time for processing: 0h 0m 31s 58ms\n","convertalis /content/mmseqs2/tmp/29/candidateDB_53 /content/mmseqs2/swissprot/swissprotDB /content/mmseqs2/tmp/29/resultsDB_53 /content/mmseqs2/tmp/29/resultDB_secondary.m8 --format-output query,target,evalue,raw \n","\n","MMseqs Version:        \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Substitution matrix    \tnucl:nucleotide.out,aa:blosum62.out\n","Alignment format       \t0\n","Format alignment output\tquery,target,evalue,raw\n","Translation table      \t1\n","Gap open cost          \tnucl:5,aa:11\n","Gap extension cost     \tnucl:2,aa:1\n","Database output        \tfalse\n","Preload mode           \t0\n","Search type            \t0\n","Threads                \t4\n","Compressed             \t0\n","Verbosity              \t3\n","\n","[=================================================================] 100.00% 5.83K 0s 172ms\n","Time for merging to resultDB_secondary.m8: 0h 0m 0s 8ms\n","Time for processing: 0h 0m 0s 270ms\n","found 2859 genes\n","\n","\n","Reading fasta file GCF_000191045.1_ASM19104v1_genomic.fna.gz...\n","\n","Finding and translating open reading frames...\n","\n","Encoding amino acids...\n","\n","Applying kmer prefilter...\n","\n","Scoring ORFs with temporal convolutional network...\n","\n"],"name":"stdout"},{"output_type":"display_data","data":{"application/vnd.jupyter.widget-view+json":{"model_id":"6aefc36b55a6463792803e9f96b30729","version_minor":0,"version_major":2},"text/plain":["HBox(children=(FloatProgress(value=0.0, max=14.0), HTML(value='')))"]},"metadata":{"tags":[]}},{"output_type":"stream","text":["\n","Scoring translation initiation sites...\n","\n"],"name":"stdout"},{"output_type":"display_data","data":{"application/vnd.jupyter.widget-view+json":{"model_id":"b6115ec16a4e446f873fb9861cb2759e","version_minor":0,"version_major":2},"text/plain":["HBox(children=(FloatProgress(value=0.0, max=5.0), HTML(value='')))"]},"metadata":{"tags":[]}},{"output_type":"stream","text":["\n","Creating weighted directed acyclic graph of all ORFs and subORFs on contig 0...\n","\n"],"name":"stdout"},{"output_type":"display_data","data":{"application/vnd.jupyter.widget-view+json":{"model_id":"3e59f5aad0ba4248a195040d52f58961","version_minor":0,"version_major":2},"text/plain":["HBox(children=(FloatProgress(value=0.0, max=41193.0), HTML(value='')))"]},"metadata":{"tags":[]}},{"output_type":"stream","text":["\n","Maximizing geneiness...\n","\n","Filtering predictions with mmseqs2...\n","createdb /content/mmseqs2/tmp/30/candidate_genes_35.fasta /content/mmseqs2/tmp/30/candidateDB_35 \n","\n","MMseqs Version:       \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Database type         \t0\n","Shuffle input database\ttrue\n","Createdb mode         \t0\n","Write lookup file     \t1\n","Offset of numeric ids \t0\n","Compressed            \t0\n","Verbosity             \t3\n","\n","Converting sequences\n","[1920] 0s 4ms\n","Time for merging to candidateDB_35_h: 0h 0m 0s 3ms\n","Time for merging to candidateDB_35: 0h 0m 0s 3ms\n","Database type: Aminoacid\n","Time for processing: 0h 0m 0s 14ms\n","createdb /content/mmseqs2/tmp/30/candidate_genes_53.fasta /content/mmseqs2/tmp/30/candidateDB_53 \n","\n","MMseqs Version:       \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Database type         \t0\n","Shuffle input database\ttrue\n","Createdb mode         \t0\n","Write lookup file     \t1\n","Offset of numeric ids \t0\n","Compressed            \t0\n","Verbosity             \t3\n","\n","Converting sequences\n","[1920] 0s 4ms\n","Time for merging to candidateDB_53_h: 0h 0m 0s 3ms\n","Time for merging to candidateDB_53: 0h 0m 0s 3ms\n","Database type: Aminoacid\n","Time for processing: 0h 0m 0s 14ms\n","search -s 7.0 /content/mmseqs2/tmp/30/candidateDB_35 /content/mmseqs2/genexa/genexaDB /content/mmseqs2/tmp/30/resultsDB_35 /content/mmseqs2/tmp/30 \n","\n","MMseqs Version:                     \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Substitution matrix                 \tnucl:nucleotide.out,aa:blosum62.out\n","Add backtrace                       \tfalse\n","Alignment mode                      \t2\n","Allow wrapped scoring               \tfalse\n","E-value threshold                   \t0.001\n","Seq. id. threshold                  \t0\n","Min alignment length                \t0\n","Seq. id. mode                       \t0\n","Alternative alignments              \t0\n","Coverage threshold                  \t0\n","Coverage mode                       \t0\n","Max sequence length                 \t65535\n","Compositional bias                  \t1\n","Realign hits                        \tfalse\n","Max reject                          \t2147483647\n","Max accept                          \t2147483647\n","Include identical seq. id.          \tfalse\n","Preload mode                        \t0\n","Pseudo count a                      \t1\n","Pseudo count b                      \t1.5\n","Score bias                          \t0\n","Gap open cost                       \tnucl:5,aa:11\n","Gap extension cost                  \tnucl:2,aa:1\n","Zdrop                               \t40\n","Threads                             \t4\n","Compressed                          \t0\n","Verbosity                           \t3\n","Seed substitution matrix            \tnucl:nucleotide.out,aa:VTML80.out\n","Sensitivity                         \t7\n","k-mer length                        \t0\n","k-score                             \t2147483647\n","Alphabet size                       \tnucl:5,aa:21\n","Max results per query               \t300\n","Split database                      \t0\n","Split mode                          \t2\n","Split memory limit                  \t0\n","Diagonal scoring                    \ttrue\n","Exact k-mer matching                \t0\n","Mask residues                       \t1\n","Mask lower case residues            \t0\n","Minimum diagonal score              \t15\n","Spaced k-mers                       \t1\n","Spaced k-mer pattern                \t\n","Local temporary path                \t\n","Rescore mode                        \t0\n","Remove hits by seq. id. and coverage\tfalse\n","Sort results                        \t0\n","Mask profile                        \t1\n","Profile e-value threshold           \t0.1\n","Global sequence weighting           \tfalse\n","Allow deletions                     \tfalse\n","Filter MSA                          \t1\n","Maximum seq. id. threshold          \t0.9\n","Minimum seq. id.                    \t0\n","Minimum score per column            \t-20\n","Minimum coverage                    \t0\n","Select N most diverse seqs          \t1000\n","Omit consensus                      \tfalse\n","Min codons in orf                   \t30\n","Max codons in length                \t32734\n","Max orf gaps                        \t2147483647\n","Contig start mode                   \t2\n","Contig end mode                     \t2\n","Orf start mode                      \t1\n","Forward frames                      \t1,2,3\n","Reverse frames                      \t1,2,3\n","Translation table                   \t1\n","Translate orf                       \t0\n","Use all table starts                \tfalse\n","Offset of numeric ids               \t0\n","Create lookup                       \t0\n","Add orf stop                        \tfalse\n","Chain overlapping alignments        \t0\n","Merge query                         \t1\n","Search type                         \t0\n","Search iterations                   \t1\n","Start sensitivity                   \t4\n","Search steps                        \t1\n","Slice search mode                   \tfalse\n","Strand selection                    \t1\n","Disk space limit                    \t0\n","MPI runner                          \t\n","Force restart with latest tmp       \tfalse\n","Remove temporary files              \tfalse\n","\n","prefilter /content/mmseqs2/tmp/30/candidateDB_35 /content/mmseqs2/genexa/genexaDB.idx /content/mmseqs2/tmp/30/13163946311976307844/pref_0 --sub-mat nucl:nucleotide.out,aa:blosum62.out --seed-sub-mat nucl:nucleotide.out,aa:VTML80.out -k 0 --k-score 2147483647 --alph-size nucl:5,aa:21 --max-seq-len 65535 --max-seqs 300 --split 0 --split-mode 2 --split-memory-limit 0 -c 0 --cov-mode 0 --comp-bias-corr 1 --diag-score 1 --exact-kmer-matching 0 --mask 1 --mask-lower-case 0 --min-ungapped-score 15 --add-self-matches 0 --spaced-kmer-mode 1 --db-load-mode 0 --pca 1 --pcb 1.5 --threads 4 --compressed 0 -v 3 -s 7.0 \n","\n","Index version: 16\n","Generated by:  16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","ScoreMatrix:  VTML80.out\n","Query database size: 1935 type: Aminoacid\n","Estimated memory consumption: 2G\n","Target database size: 468351 type: Aminoacid\n","Process prefiltering step 1 of 1\n","\n","k-mer similarity threshold: 100\n","Starting prefiltering scores calculation (step 1 of 1)\n","Query db start 1 to 1935\n","Target db start 1 to 468351\n","[=================================================================] 100.00% 1.93K 17s 267ms\n","\n","894.793943 k-mers per position\n","701429 DB matches per sequence\n","71 overflows\n","0 queries produce too many hits (truncated result)\n","281 sequences passed prefiltering per query sequence\n","300 median result list length\n","2 sequences with 0 size result lists\n","Time for merging to pref_0: 0h 0m 0s 0ms\n","Time for processing: 0h 0m 18s 697ms\n","align /content/mmseqs2/tmp/30/candidateDB_35 /content/mmseqs2/genexa/genexaDB.idx /content/mmseqs2/tmp/30/13163946311976307844/pref_0 /content/mmseqs2/tmp/30/resultsDB_35 --sub-mat nucl:nucleotide.out,aa:blosum62.out -a 0 --alignment-mode 2 --wrapped-scoring 0 -e 0.001 --min-seq-id 0 --min-aln-len 0 --seq-id-mode 0 --alt-ali 0 -c 0 --cov-mode 0 --max-seq-len 65535 --comp-bias-corr 1 --realign 0 --max-rejected 2147483647 --max-accept 2147483647 --add-self-matches 0 --db-load-mode 0 --pca 1 --pcb 1.5 --score-bias 0 --gap-open nucl:5,aa:11 --gap-extend nucl:2,aa:1 --zdrop 40 --threads 4 --compressed 0 -v 3 \n","\n","Index version: 16\n","Generated by:  16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","ScoreMatrix:  VTML80.out\n","Compute score and coverage\n","Query database size: 1935 type: Aminoacid\n","Target database size: 468351 type: Aminoacid\n","Calculation of alignments\n","[=================================================================] 100.00% 1.93K 16s 62ms\n","Time for merging to resultsDB_35: 0h 0m 0s 0ms\n","\n","544322 alignments calculated.\n","185321 sequence pairs passed the thresholds (0.340462 of overall calculated).\n","95.773125 hits per query sequence.\n","Time for processing: 0h 0m 16s 96ms\n","convertalis /content/mmseqs2/tmp/30/candidateDB_35 /content/mmseqs2/genexa/genexaDB /content/mmseqs2/tmp/30/resultsDB_35 /content/mmseqs2/tmp/30/resultDB_genexa.m8 --format-output query,target,evalue,raw \n","\n","MMseqs Version:        \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Substitution matrix    \tnucl:nucleotide.out,aa:blosum62.out\n","Alignment format       \t0\n","Format alignment output\tquery,target,evalue,raw\n","Translation table      \t1\n","Gap open cost          \tnucl:5,aa:11\n","Gap extension cost     \tnucl:2,aa:1\n","Database output        \tfalse\n","Preload mode           \t0\n","Search type            \t0\n","Threads                \t4\n","Compressed             \t0\n","Verbosity              \t3\n","\n","[=================================================================] 100.00% 1.93K 0s 137ms\n","Time for merging to resultDB_genexa.m8: 0h 0m 0s 5ms\n","Time for processing: 0h 0m 0s 210ms\n","search -s 7.0 /content/mmseqs2/tmp/30/candidateDB_53 /content/mmseqs2/swissprot/swissprotDB /content/mmseqs2/tmp/30/resultsDB_53 /content/mmseqs2/tmp/30 \n","\n","MMseqs Version:                     \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Substitution matrix                 \tnucl:nucleotide.out,aa:blosum62.out\n","Add backtrace                       \tfalse\n","Alignment mode                      \t2\n","Allow wrapped scoring               \tfalse\n","E-value threshold                   \t0.001\n","Seq. id. threshold                  \t0\n","Min alignment length                \t0\n","Seq. id. mode                       \t0\n","Alternative alignments              \t0\n","Coverage threshold                  \t0\n","Coverage mode                       \t0\n","Max sequence length                 \t65535\n","Compositional bias                  \t1\n","Realign hits                        \tfalse\n","Max reject                          \t2147483647\n","Max accept                          \t2147483647\n","Include identical seq. id.          \tfalse\n","Preload mode                        \t0\n","Pseudo count a                      \t1\n","Pseudo count b                      \t1.5\n","Score bias                          \t0\n","Gap open cost                       \tnucl:5,aa:11\n","Gap extension cost                  \tnucl:2,aa:1\n","Zdrop                               \t40\n","Threads                             \t4\n","Compressed                          \t0\n","Verbosity                           \t3\n","Seed substitution matrix            \tnucl:nucleotide.out,aa:VTML80.out\n","Sensitivity                         \t7\n","k-mer length                        \t0\n","k-score                             \t2147483647\n","Alphabet size                       \tnucl:5,aa:21\n","Max results per query               \t300\n","Split database                      \t0\n","Split mode                          \t2\n","Split memory limit                  \t0\n","Diagonal scoring                    \ttrue\n","Exact k-mer matching                \t0\n","Mask residues                       \t1\n","Mask lower case residues            \t0\n","Minimum diagonal score              \t15\n","Spaced k-mers                       \t1\n","Spaced k-mer pattern                \t\n","Local temporary path                \t\n","Rescore mode                        \t0\n","Remove hits by seq. id. and coverage\tfalse\n","Sort results                        \t0\n","Mask profile                        \t1\n","Profile e-value threshold           \t0.1\n","Global sequence weighting           \tfalse\n","Allow deletions                     \tfalse\n","Filter MSA                          \t1\n","Maximum seq. id. threshold          \t0.9\n","Minimum seq. id.                    \t0\n","Minimum score per column            \t-20\n","Minimum coverage                    \t0\n","Select N most diverse seqs          \t1000\n","Omit consensus                      \tfalse\n","Min codons in orf                   \t30\n","Max codons in length                \t32734\n","Max orf gaps                        \t2147483647\n","Contig start mode                   \t2\n","Contig end mode                     \t2\n","Orf start mode                      \t1\n","Forward frames                      \t1,2,3\n","Reverse frames                      \t1,2,3\n","Translation table                   \t1\n","Translate orf                       \t0\n","Use all table starts                \tfalse\n","Offset of numeric ids               \t0\n","Create lookup                       \t0\n","Add orf stop                        \tfalse\n","Chain overlapping alignments        \t0\n","Merge query                         \t1\n","Search type                         \t0\n","Search iterations                   \t1\n","Start sensitivity                   \t4\n","Search steps                        \t1\n","Slice search mode                   \tfalse\n","Strand selection                    \t1\n","Disk space limit                    \t0\n","MPI runner                          \t\n","Force restart with latest tmp       \tfalse\n","Remove temporary files              \tfalse\n","\n","prefilter /content/mmseqs2/tmp/30/candidateDB_53 /content/mmseqs2/swissprot/swissprotDB /content/mmseqs2/tmp/30/10496852460387792930/pref_0 --sub-mat nucl:nucleotide.out,aa:blosum62.out --seed-sub-mat nucl:nucleotide.out,aa:VTML80.out -k 0 --k-score 2147483647 --alph-size nucl:5,aa:21 --max-seq-len 65535 --max-seqs 300 --split 0 --split-mode 2 --split-memory-limit 0 -c 0 --cov-mode 0 --comp-bias-corr 1 --diag-score 1 --exact-kmer-matching 0 --mask 1 --mask-lower-case 0 --min-ungapped-score 15 --add-self-matches 0 --spaced-kmer-mode 1 --db-load-mode 0 --pca 1 --pcb 1.5 --threads 4 --compressed 0 -v 3 -s 7.0 \n","\n","Query database size: 1935 type: Aminoacid\n","Estimated memory consumption: 2G\n","Target database size: 562755 type: Aminoacid\n","Index table k-mer threshold: 100 at k-mer size 6 \n","Index table: counting k-mers\n","[=================================================================] 100.00% 562.76K 24s 771ms\n","Index table: Masked residues: 3164874\n","Index table: fill\n","[=================================================================] 100.00% 562.76K 18s 11ms\n","Index statistics\n","Entries:          192757622\n","DB size:          1591 MB\n","Avg k-mer size:   3.011838\n","Top 10 k-mers\n","    GPGGTL\t1835\n","    GQSWTV\t1706\n","    WGMFAT\t1638\n","    PGVFEV\t1634\n","    VLWQFW\t1623\n","    AYIRPN\t1587\n","    RSPKGV\t1585\n","    TPHKWY\t1560\n","    HIRVGW\t1548\n","    ITLSPY\t1541\n","Time for index table init: 0h 0m 44s 910ms\n","Process prefiltering step 1 of 1\n","\n","k-mer similarity threshold: 100\n","Starting prefiltering scores calculation (step 1 of 1)\n","Query db start 1 to 1935\n","Target db start 1 to 562755\n","[=================================================================] 100.00% 1.93K 20s 257ms\n","\n","894.275739 k-mers per position\n","891159 DB matches per sequence\n","148 overflows\n","0 queries produce too many hits (truncated result)\n","286 sequences passed prefiltering per query sequence\n","300 median result list length\n","2 sequences with 0 size result lists\n","Time for merging to pref_0: 0h 0m 0s 0ms\n","Time for processing: 0h 1m 6s 317ms\n","align /content/mmseqs2/tmp/30/candidateDB_53 /content/mmseqs2/swissprot/swissprotDB /content/mmseqs2/tmp/30/10496852460387792930/pref_0 /content/mmseqs2/tmp/30/resultsDB_53 --sub-mat nucl:nucleotide.out,aa:blosum62.out -a 0 --alignment-mode 2 --wrapped-scoring 0 -e 0.001 --min-seq-id 0 --min-aln-len 0 --seq-id-mode 0 --alt-ali 0 -c 0 --cov-mode 0 --max-seq-len 65535 --comp-bias-corr 1 --realign 0 --max-rejected 2147483647 --max-accept 2147483647 --add-self-matches 0 --db-load-mode 0 --pca 1 --pcb 1.5 --score-bias 0 --gap-open nucl:5,aa:11 --gap-extend nucl:2,aa:1 --zdrop 40 --threads 4 --compressed 0 -v 3 \n","\n","Compute score and coverage\n","Query database size: 1935 type: Aminoacid\n","Target database size: 562755 type: Aminoacid\n","Calculation of alignments\n","[=================================================================] 100.00% 1.93K 16s 563ms\n","Time for merging to resultsDB_53: 0h 0m 0s 0ms\n","\n","554951 alignments calculated.\n","163026 sequence pairs passed the thresholds (0.293766 of overall calculated).\n","84.251160 hits per query sequence.\n","Time for processing: 0h 0m 16s 676ms\n","convertalis /content/mmseqs2/tmp/30/candidateDB_53 /content/mmseqs2/swissprot/swissprotDB /content/mmseqs2/tmp/30/resultsDB_53 /content/mmseqs2/tmp/30/resultDB_secondary.m8 --format-output query,target,evalue,raw \n","\n","MMseqs Version:        \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Substitution matrix    \tnucl:nucleotide.out,aa:blosum62.out\n","Alignment format       \t0\n","Format alignment output\tquery,target,evalue,raw\n","Translation table      \t1\n","Gap open cost          \tnucl:5,aa:11\n","Gap extension cost     \tnucl:2,aa:1\n","Database output        \tfalse\n","Preload mode           \t0\n","Search type            \t0\n","Threads                \t4\n","Compressed             \t0\n","Verbosity              \t3\n","\n","[=================================================================] 100.00% 1.93K 0s 112ms\n","Time for merging to resultDB_secondary.m8: 0h 0m 0s 5ms\n","Time for processing: 0h 0m 0s 203ms\n","found 1535 genes\n","\n","\n","Reading fasta file GCF_000189295.2_ASM18929v2_genomic.fna.gz...\n","\n","Finding and translating open reading frames...\n","\n","Encoding amino acids...\n","\n","Applying kmer prefilter...\n","\n","Scoring ORFs with temporal convolutional network...\n","\n"],"name":"stdout"},{"output_type":"display_data","data":{"application/vnd.jupyter.widget-view+json":{"model_id":"3023cac4c45946a290dc588f01dc67dc","version_minor":0,"version_major":2},"text/plain":["HBox(children=(FloatProgress(value=0.0, max=80.0), HTML(value='')))"]},"metadata":{"tags":[]}},{"output_type":"stream","text":["\n","Scoring translation initiation sites...\n","\n"],"name":"stdout"},{"output_type":"display_data","data":{"application/vnd.jupyter.widget-view+json":{"model_id":"3aae4efe6f1a43ba9f273945b0ab88ce","version_minor":0,"version_major":2},"text/plain":["HBox(children=(FloatProgress(value=0.0, max=24.0), HTML(value='')))"]},"metadata":{"tags":[]}},{"output_type":"stream","text":["\n","Creating weighted directed acyclic graph of all ORFs and subORFs on contig 0...\n","\n"],"name":"stdout"},{"output_type":"display_data","data":{"application/vnd.jupyter.widget-view+json":{"model_id":"1c3ef650221348099251f50de17d63d2","version_minor":0,"version_major":2},"text/plain":["HBox(children=(FloatProgress(value=0.0, max=238297.0), HTML(value='')))"]},"metadata":{"tags":[]}},{"output_type":"stream","text":["\n","Maximizing geneiness...\n","\n","Filtering predictions with mmseqs2...\n","createdb /content/mmseqs2/tmp/31/candidate_genes_35.fasta /content/mmseqs2/tmp/31/candidateDB_35 \n","\n","MMseqs Version:       \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Database type         \t0\n","Shuffle input database\ttrue\n","Createdb mode         \t0\n","Write lookup file     \t1\n","Offset of numeric ids \t0\n","Compressed            \t0\n","Verbosity             \t3\n","\n","Converting sequences\n","[5758] 0s 9ms\n","Time for merging to candidateDB_35_h: 0h 0m 0s 4ms\n","Time for merging to candidateDB_35: 0h 0m 0s 4ms\n","Database type: Aminoacid\n","Time for processing: 0h 0m 0s 26ms\n","createdb /content/mmseqs2/tmp/31/candidate_genes_53.fasta /content/mmseqs2/tmp/31/candidateDB_53 \n","\n","MMseqs Version:       \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Database type         \t0\n","Shuffle input database\ttrue\n","Createdb mode         \t0\n","Write lookup file     \t1\n","Offset of numeric ids \t0\n","Compressed            \t0\n","Verbosity             \t3\n","\n","Converting sequences\n","[5758] 0s 7ms\n","Time for merging to candidateDB_53_h: 0h 0m 0s 3ms\n","Time for merging to candidateDB_53: 0h 0m 0s 4ms\n","Database type: Aminoacid\n","Time for processing: 0h 0m 0s 23ms\n","search -s 7.0 /content/mmseqs2/tmp/31/candidateDB_35 /content/mmseqs2/genexa/genexaDB /content/mmseqs2/tmp/31/resultsDB_35 /content/mmseqs2/tmp/31 \n","\n","MMseqs Version:                     \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Substitution matrix                 \tnucl:nucleotide.out,aa:blosum62.out\n","Add backtrace                       \tfalse\n","Alignment mode                      \t2\n","Allow wrapped scoring               \tfalse\n","E-value threshold                   \t0.001\n","Seq. id. threshold                  \t0\n","Min alignment length                \t0\n","Seq. id. mode                       \t0\n","Alternative alignments              \t0\n","Coverage threshold                  \t0\n","Coverage mode                       \t0\n","Max sequence length                 \t65535\n","Compositional bias                  \t1\n","Realign hits                        \tfalse\n","Max reject                          \t2147483647\n","Max accept                          \t2147483647\n","Include identical seq. id.          \tfalse\n","Preload mode                        \t0\n","Pseudo count a                      \t1\n","Pseudo count b                      \t1.5\n","Score bias                          \t0\n","Gap open cost                       \tnucl:5,aa:11\n","Gap extension cost                  \tnucl:2,aa:1\n","Zdrop                               \t40\n","Threads                             \t4\n","Compressed                          \t0\n","Verbosity                           \t3\n","Seed substitution matrix            \tnucl:nucleotide.out,aa:VTML80.out\n","Sensitivity                         \t7\n","k-mer length                        \t0\n","k-score                             \t2147483647\n","Alphabet size                       \tnucl:5,aa:21\n","Max results per query               \t300\n","Split database                      \t0\n","Split mode                          \t2\n","Split memory limit                  \t0\n","Diagonal scoring                    \ttrue\n","Exact k-mer matching                \t0\n","Mask residues                       \t1\n","Mask lower case residues            \t0\n","Minimum diagonal score              \t15\n","Spaced k-mers                       \t1\n","Spaced k-mer pattern                \t\n","Local temporary path                \t\n","Rescore mode                        \t0\n","Remove hits by seq. id. and coverage\tfalse\n","Sort results                        \t0\n","Mask profile                        \t1\n","Profile e-value threshold           \t0.1\n","Global sequence weighting           \tfalse\n","Allow deletions                     \tfalse\n","Filter MSA                          \t1\n","Maximum seq. id. threshold          \t0.9\n","Minimum seq. id.                    \t0\n","Minimum score per column            \t-20\n","Minimum coverage                    \t0\n","Select N most diverse seqs          \t1000\n","Omit consensus                      \tfalse\n","Min codons in orf                   \t30\n","Max codons in length                \t32734\n","Max orf gaps                        \t2147483647\n","Contig start mode                   \t2\n","Contig end mode                     \t2\n","Orf start mode                      \t1\n","Forward frames                      \t1,2,3\n","Reverse frames                      \t1,2,3\n","Translation table                   \t1\n","Translate orf                       \t0\n","Use all table starts                \tfalse\n","Offset of numeric ids               \t0\n","Create lookup                       \t0\n","Add orf stop                        \tfalse\n","Chain overlapping alignments        \t0\n","Merge query                         \t1\n","Search type                         \t0\n","Search iterations                   \t1\n","Start sensitivity                   \t4\n","Search steps                        \t1\n","Slice search mode                   \tfalse\n","Strand selection                    \t1\n","Disk space limit                    \t0\n","MPI runner                          \t\n","Force restart with latest tmp       \tfalse\n","Remove temporary files              \tfalse\n","\n","prefilter /content/mmseqs2/tmp/31/candidateDB_35 /content/mmseqs2/genexa/genexaDB.idx /content/mmseqs2/tmp/31/7903660208866612355/pref_0 --sub-mat nucl:nucleotide.out,aa:blosum62.out --seed-sub-mat nucl:nucleotide.out,aa:VTML80.out -k 0 --k-score 2147483647 --alph-size nucl:5,aa:21 --max-seq-len 65535 --max-seqs 300 --split 0 --split-mode 2 --split-memory-limit 0 -c 0 --cov-mode 0 --comp-bias-corr 1 --diag-score 1 --exact-kmer-matching 0 --mask 1 --mask-lower-case 0 --min-ungapped-score 15 --add-self-matches 0 --spaced-kmer-mode 1 --db-load-mode 0 --pca 1 --pcb 1.5 --threads 4 --compressed 0 -v 3 -s 7.0 \n","\n","Index version: 16\n","Generated by:  16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","ScoreMatrix:  VTML80.out\n","Query database size: 5766 type: Aminoacid\n","Estimated memory consumption: 2G\n","Target database size: 468351 type: Aminoacid\n","Process prefiltering step 1 of 1\n","\n","k-mer similarity threshold: 100\n","Starting prefiltering scores calculation (step 1 of 1)\n","Query db start 1 to 5766\n","Target db start 1 to 468351\n","[=================================================================] 100.00% 5.77K 45s 602ms\n","\n","962.106069 k-mers per position\n","631310 DB matches per sequence\n","294 overflows\n","0 queries produce too many hits (truncated result)\n","279 sequences passed prefiltering per query sequence\n","300 median result list length\n","1 sequences with 0 size result lists\n","Time for merging to pref_0: 0h 0m 0s 1ms\n","Time for processing: 0h 0m 47s 46ms\n","align /content/mmseqs2/tmp/31/candidateDB_35 /content/mmseqs2/genexa/genexaDB.idx /content/mmseqs2/tmp/31/7903660208866612355/pref_0 /content/mmseqs2/tmp/31/resultsDB_35 --sub-mat nucl:nucleotide.out,aa:blosum62.out -a 0 --alignment-mode 2 --wrapped-scoring 0 -e 0.001 --min-seq-id 0 --min-aln-len 0 --seq-id-mode 0 --alt-ali 0 -c 0 --cov-mode 0 --max-seq-len 65535 --comp-bias-corr 1 --realign 0 --max-rejected 2147483647 --max-accept 2147483647 --add-self-matches 0 --db-load-mode 0 --pca 1 --pcb 1.5 --score-bias 0 --gap-open nucl:5,aa:11 --gap-extend nucl:2,aa:1 --zdrop 40 --threads 4 --compressed 0 -v 3 \n","\n","Index version: 16\n","Generated by:  16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","ScoreMatrix:  VTML80.out\n","Compute score and coverage\n","Query database size: 5766 type: Aminoacid\n","Target database size: 468351 type: Aminoacid\n","Calculation of alignments\n","[=================================================================] 100.00% 5.77K 36s 108ms\n","Time for merging to resultsDB_35: 0h 0m 0s 1ms\n","\n","1611624 alignments calculated.\n","406373 sequence pairs passed the thresholds (0.252151 of overall calculated).\n","70.477455 hits per query sequence.\n","Time for processing: 0h 0m 36s 150ms\n","convertalis /content/mmseqs2/tmp/31/candidateDB_35 /content/mmseqs2/genexa/genexaDB /content/mmseqs2/tmp/31/resultsDB_35 /content/mmseqs2/tmp/31/resultDB_genexa.m8 --format-output query,target,evalue,raw \n","\n","MMseqs Version:        \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Substitution matrix    \tnucl:nucleotide.out,aa:blosum62.out\n","Alignment format       \t0\n","Format alignment output\tquery,target,evalue,raw\n","Translation table      \t1\n","Gap open cost          \tnucl:5,aa:11\n","Gap extension cost     \tnucl:2,aa:1\n","Database output        \tfalse\n","Preload mode           \t0\n","Search type            \t0\n","Threads                \t4\n","Compressed             \t0\n","Verbosity              \t3\n","\n","[=================================================================] 100.00% 5.77K 0s 291ms\n","Time for merging to resultDB_genexa.m8: 0h 0m 0s 11ms\n","Time for processing: 0h 0m 0s 373ms\n","search -s 7.0 /content/mmseqs2/tmp/31/candidateDB_53 /content/mmseqs2/swissprot/swissprotDB /content/mmseqs2/tmp/31/resultsDB_53 /content/mmseqs2/tmp/31 \n","\n","MMseqs Version:                     \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Substitution matrix                 \tnucl:nucleotide.out,aa:blosum62.out\n","Add backtrace                       \tfalse\n","Alignment mode                      \t2\n","Allow wrapped scoring               \tfalse\n","E-value threshold                   \t0.001\n","Seq. id. threshold                  \t0\n","Min alignment length                \t0\n","Seq. id. mode                       \t0\n","Alternative alignments              \t0\n","Coverage threshold                  \t0\n","Coverage mode                       \t0\n","Max sequence length                 \t65535\n","Compositional bias                  \t1\n","Realign hits                        \tfalse\n","Max reject                          \t2147483647\n","Max accept                          \t2147483647\n","Include identical seq. id.          \tfalse\n","Preload mode                        \t0\n","Pseudo count a                      \t1\n","Pseudo count b                      \t1.5\n","Score bias                          \t0\n","Gap open cost                       \tnucl:5,aa:11\n","Gap extension cost                  \tnucl:2,aa:1\n","Zdrop                               \t40\n","Threads                             \t4\n","Compressed                          \t0\n","Verbosity                           \t3\n","Seed substitution matrix            \tnucl:nucleotide.out,aa:VTML80.out\n","Sensitivity                         \t7\n","k-mer length                        \t0\n","k-score                             \t2147483647\n","Alphabet size                       \tnucl:5,aa:21\n","Max results per query               \t300\n","Split database                      \t0\n","Split mode                          \t2\n","Split memory limit                  \t0\n","Diagonal scoring                    \ttrue\n","Exact k-mer matching                \t0\n","Mask residues                       \t1\n","Mask lower case residues            \t0\n","Minimum diagonal score              \t15\n","Spaced k-mers                       \t1\n","Spaced k-mer pattern                \t\n","Local temporary path                \t\n","Rescore mode                        \t0\n","Remove hits by seq. id. and coverage\tfalse\n","Sort results                        \t0\n","Mask profile                        \t1\n","Profile e-value threshold           \t0.1\n","Global sequence weighting           \tfalse\n","Allow deletions                     \tfalse\n","Filter MSA                          \t1\n","Maximum seq. id. threshold          \t0.9\n","Minimum seq. id.                    \t0\n","Minimum score per column            \t-20\n","Minimum coverage                    \t0\n","Select N most diverse seqs          \t1000\n","Omit consensus                      \tfalse\n","Min codons in orf                   \t30\n","Max codons in length                \t32734\n","Max orf gaps                        \t2147483647\n","Contig start mode                   \t2\n","Contig end mode                     \t2\n","Orf start mode                      \t1\n","Forward frames                      \t1,2,3\n","Reverse frames                      \t1,2,3\n","Translation table                   \t1\n","Translate orf                       \t0\n","Use all table starts                \tfalse\n","Offset of numeric ids               \t0\n","Create lookup                       \t0\n","Add orf stop                        \tfalse\n","Chain overlapping alignments        \t0\n","Merge query                         \t1\n","Search type                         \t0\n","Search iterations                   \t1\n","Start sensitivity                   \t4\n","Search steps                        \t1\n","Slice search mode                   \tfalse\n","Strand selection                    \t1\n","Disk space limit                    \t0\n","MPI runner                          \t\n","Force restart with latest tmp       \tfalse\n","Remove temporary files              \tfalse\n","\n","prefilter /content/mmseqs2/tmp/31/candidateDB_53 /content/mmseqs2/swissprot/swissprotDB /content/mmseqs2/tmp/31/10598217448433138913/pref_0 --sub-mat nucl:nucleotide.out,aa:blosum62.out --seed-sub-mat nucl:nucleotide.out,aa:VTML80.out -k 0 --k-score 2147483647 --alph-size nucl:5,aa:21 --max-seq-len 65535 --max-seqs 300 --split 0 --split-mode 2 --split-memory-limit 0 -c 0 --cov-mode 0 --comp-bias-corr 1 --diag-score 1 --exact-kmer-matching 0 --mask 1 --mask-lower-case 0 --min-ungapped-score 15 --add-self-matches 0 --spaced-kmer-mode 1 --db-load-mode 0 --pca 1 --pcb 1.5 --threads 4 --compressed 0 -v 3 -s 7.0 \n","\n","Query database size: 5766 type: Aminoacid\n","Estimated memory consumption: 2G\n","Target database size: 562755 type: Aminoacid\n","Index table k-mer threshold: 100 at k-mer size 6 \n","Index table: counting k-mers\n","[=================================================================] 100.00% 562.76K 24s 38ms\n","Index table: Masked residues: 3164874\n","Index table: fill\n","[=================================================================] 100.00% 562.76K 17s 722ms\n","Index statistics\n","Entries:          192757622\n","DB size:          1591 MB\n","Avg k-mer size:   3.011838\n","Top 10 k-mers\n","    GPGGTL\t1835\n","    GQSWTV\t1706\n","    WGMFAT\t1638\n","    PGVFEV\t1634\n","    VLWQFW\t1623\n","    AYIRPN\t1587\n","    RSPKGV\t1585\n","    TPHKWY\t1560\n","    HIRVGW\t1548\n","    ITLSPY\t1541\n","Time for index table init: 0h 0m 43s 817ms\n","Process prefiltering step 1 of 1\n","\n","k-mer similarity threshold: 100\n","Starting prefiltering scores calculation (step 1 of 1)\n","Query db start 1 to 5766\n","Target db start 1 to 562755\n","[=================================================================] 100.00% 5.77K 50s 469ms\n","\n","962.411777 k-mers per position\n","739554 DB matches per sequence\n","453 overflows\n","0 queries produce too many hits (truncated result)\n","285 sequences passed prefiltering per query sequence\n","300 median result list length\n","0 sequences with 0 size result lists\n","Time for merging to pref_0: 0h 0m 0s 1ms\n","Time for processing: 0h 1m 35s 448ms\n","align /content/mmseqs2/tmp/31/candidateDB_53 /content/mmseqs2/swissprot/swissprotDB /content/mmseqs2/tmp/31/10598217448433138913/pref_0 /content/mmseqs2/tmp/31/resultsDB_53 --sub-mat nucl:nucleotide.out,aa:blosum62.out -a 0 --alignment-mode 2 --wrapped-scoring 0 -e 0.001 --min-seq-id 0 --min-aln-len 0 --seq-id-mode 0 --alt-ali 0 -c 0 --cov-mode 0 --max-seq-len 65535 --comp-bias-corr 1 --realign 0 --max-rejected 2147483647 --max-accept 2147483647 --add-self-matches 0 --db-load-mode 0 --pca 1 --pcb 1.5 --score-bias 0 --gap-open nucl:5,aa:11 --gap-extend nucl:2,aa:1 --zdrop 40 --threads 4 --compressed 0 -v 3 \n","\n","Compute score and coverage\n","Query database size: 5766 type: Aminoacid\n","Target database size: 562755 type: Aminoacid\n","Calculation of alignments\n","[=================================================================] 100.00% 5.77K 33s 3ms\n","Time for merging to resultsDB_53: 0h 0m 0s 1ms\n","\n","1648014 alignments calculated.\n","287577 sequence pairs passed the thresholds (0.174499 of overall calculated).\n","49.874611 hits per query sequence.\n","Time for processing: 0h 0m 33s 79ms\n","convertalis /content/mmseqs2/tmp/31/candidateDB_53 /content/mmseqs2/swissprot/swissprotDB /content/mmseqs2/tmp/31/resultsDB_53 /content/mmseqs2/tmp/31/resultDB_secondary.m8 --format-output query,target,evalue,raw \n","\n","MMseqs Version:        \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Substitution matrix    \tnucl:nucleotide.out,aa:blosum62.out\n","Alignment format       \t0\n","Format alignment output\tquery,target,evalue,raw\n","Translation table      \t1\n","Gap open cost          \tnucl:5,aa:11\n","Gap extension cost     \tnucl:2,aa:1\n","Database output        \tfalse\n","Preload mode           \t0\n","Search type            \t0\n","Threads                \t4\n","Compressed             \t0\n","Verbosity              \t3\n","\n","[=================================================================] 100.00% 5.77K 0s 202ms\n","Time for merging to resultDB_secondary.m8: 0h 0m 0s 9ms\n","Time for processing: 0h 0m 0s 301ms\n","found 3448 genes\n","\n","\n","Reading fasta file GCF_000185805.1_ASM18580v1_genomic.fna.gz...\n","\n","Finding and translating open reading frames...\n","\n","Encoding amino acids...\n","\n","Applying kmer prefilter...\n","\n","Scoring ORFs with temporal convolutional network...\n","\n"],"name":"stdout"},{"output_type":"display_data","data":{"application/vnd.jupyter.widget-view+json":{"model_id":"2fced687a73844b290a7cc87b85a8ace","version_minor":0,"version_major":2},"text/plain":["HBox(children=(FloatProgress(value=0.0, max=20.0), HTML(value='')))"]},"metadata":{"tags":[]}},{"output_type":"stream","text":["\n","Scoring translation initiation sites...\n","\n"],"name":"stdout"},{"output_type":"display_data","data":{"application/vnd.jupyter.widget-view+json":{"model_id":"82b536966beb414699990fb2ff46d4c6","version_minor":0,"version_major":2},"text/plain":["HBox(children=(FloatProgress(value=0.0, max=6.0), HTML(value='')))"]},"metadata":{"tags":[]}},{"output_type":"stream","text":["\n","Creating weighted directed acyclic graph of all ORFs and subORFs on contig 0...\n","\n"],"name":"stdout"},{"output_type":"display_data","data":{"application/vnd.jupyter.widget-view+json":{"model_id":"ef69d21fcd854d6bb5cb864741b0f7af","version_minor":0,"version_major":2},"text/plain":["HBox(children=(FloatProgress(value=0.0, max=56484.0), HTML(value='')))"]},"metadata":{"tags":[]}},{"output_type":"stream","text":["\n","Maximizing geneiness...\n","\n","Filtering predictions with mmseqs2...\n","createdb /content/mmseqs2/tmp/32/candidate_genes_35.fasta /content/mmseqs2/tmp/32/candidateDB_35 \n","\n","MMseqs Version:       \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Database type         \t0\n","Shuffle input database\ttrue\n","Createdb mode         \t0\n","Write lookup file     \t1\n","Offset of numeric ids \t0\n","Compressed            \t0\n","Verbosity             \t3\n","\n","Converting sequences\n","[2122] 0s 4ms\n","Time for merging to candidateDB_35_h: 0h 0m 0s 2ms\n","Time for merging to candidateDB_35: 0h 0m 0s 3ms\n","Database type: Aminoacid\n","Time for processing: 0h 0m 0s 14ms\n","createdb /content/mmseqs2/tmp/32/candidate_genes_53.fasta /content/mmseqs2/tmp/32/candidateDB_53 \n","\n","MMseqs Version:       \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Database type         \t0\n","Shuffle input database\ttrue\n","Createdb mode         \t0\n","Write lookup file     \t1\n","Offset of numeric ids \t0\n","Compressed            \t0\n","Verbosity             \t3\n","\n","Converting sequences\n","[2122] 0s 5ms\n","Time for merging to candidateDB_53_h: 0h 0m 0s 6ms\n","Time for merging to candidateDB_53: 0h 0m 0s 3ms\n","Database type: Aminoacid\n","Time for processing: 0h 0m 0s 18ms\n","search -s 7.0 /content/mmseqs2/tmp/32/candidateDB_35 /content/mmseqs2/genexa/genexaDB /content/mmseqs2/tmp/32/resultsDB_35 /content/mmseqs2/tmp/32 \n","\n","MMseqs Version:                     \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Substitution matrix                 \tnucl:nucleotide.out,aa:blosum62.out\n","Add backtrace                       \tfalse\n","Alignment mode                      \t2\n","Allow wrapped scoring               \tfalse\n","E-value threshold                   \t0.001\n","Seq. id. threshold                  \t0\n","Min alignment length                \t0\n","Seq. id. mode                       \t0\n","Alternative alignments              \t0\n","Coverage threshold                  \t0\n","Coverage mode                       \t0\n","Max sequence length                 \t65535\n","Compositional bias                  \t1\n","Realign hits                        \tfalse\n","Max reject                          \t2147483647\n","Max accept                          \t2147483647\n","Include identical seq. id.          \tfalse\n","Preload mode                        \t0\n","Pseudo count a                      \t1\n","Pseudo count b                      \t1.5\n","Score bias                          \t0\n","Gap open cost                       \tnucl:5,aa:11\n","Gap extension cost                  \tnucl:2,aa:1\n","Zdrop                               \t40\n","Threads                             \t4\n","Compressed                          \t0\n","Verbosity                           \t3\n","Seed substitution matrix            \tnucl:nucleotide.out,aa:VTML80.out\n","Sensitivity                         \t7\n","k-mer length                        \t0\n","k-score                             \t2147483647\n","Alphabet size                       \tnucl:5,aa:21\n","Max results per query               \t300\n","Split database                      \t0\n","Split mode                          \t2\n","Split memory limit                  \t0\n","Diagonal scoring                    \ttrue\n","Exact k-mer matching                \t0\n","Mask residues                       \t1\n","Mask lower case residues            \t0\n","Minimum diagonal score              \t15\n","Spaced k-mers                       \t1\n","Spaced k-mer pattern                \t\n","Local temporary path                \t\n","Rescore mode                        \t0\n","Remove hits by seq. id. and coverage\tfalse\n","Sort results                        \t0\n","Mask profile                        \t1\n","Profile e-value threshold           \t0.1\n","Global sequence weighting           \tfalse\n","Allow deletions                     \tfalse\n","Filter MSA                          \t1\n","Maximum seq. id. threshold          \t0.9\n","Minimum seq. id.                    \t0\n","Minimum score per column            \t-20\n","Minimum coverage                    \t0\n","Select N most diverse seqs          \t1000\n","Omit consensus                      \tfalse\n","Min codons in orf                   \t30\n","Max codons in length                \t32734\n","Max orf gaps                        \t2147483647\n","Contig start mode                   \t2\n","Contig end mode                     \t2\n","Orf start mode                      \t1\n","Forward frames                      \t1,2,3\n","Reverse frames                      \t1,2,3\n","Translation table                   \t1\n","Translate orf                       \t0\n","Use all table starts                \tfalse\n","Offset of numeric ids               \t0\n","Create lookup                       \t0\n","Add orf stop                        \tfalse\n","Chain overlapping alignments        \t0\n","Merge query                         \t1\n","Search type                         \t0\n","Search iterations                   \t1\n","Start sensitivity                   \t4\n","Search steps                        \t1\n","Slice search mode                   \tfalse\n","Strand selection                    \t1\n","Disk space limit                    \t0\n","MPI runner                          \t\n","Force restart with latest tmp       \tfalse\n","Remove temporary files              \tfalse\n","\n","prefilter /content/mmseqs2/tmp/32/candidateDB_35 /content/mmseqs2/genexa/genexaDB.idx /content/mmseqs2/tmp/32/2643374105756916866/pref_0 --sub-mat nucl:nucleotide.out,aa:blosum62.out --seed-sub-mat nucl:nucleotide.out,aa:VTML80.out -k 0 --k-score 2147483647 --alph-size nucl:5,aa:21 --max-seq-len 65535 --max-seqs 300 --split 0 --split-mode 2 --split-memory-limit 0 -c 0 --cov-mode 0 --comp-bias-corr 1 --diag-score 1 --exact-kmer-matching 0 --mask 1 --mask-lower-case 0 --min-ungapped-score 15 --add-self-matches 0 --spaced-kmer-mode 1 --db-load-mode 0 --pca 1 --pcb 1.5 --threads 4 --compressed 0 -v 3 -s 7.0 \n","\n","Index version: 16\n","Generated by:  16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","ScoreMatrix:  VTML80.out\n","Query database size: 2159 type: Aminoacid\n","Estimated memory consumption: 2G\n","Target database size: 468351 type: Aminoacid\n","Process prefiltering step 1 of 1\n","\n","k-mer similarity threshold: 100\n","Starting prefiltering scores calculation (step 1 of 1)\n","Query db start 1 to 2159\n","Target db start 1 to 468351\n","[=================================================================] 100.00% 2.16K 18s 944ms\n","\n","899.608149 k-mers per position\n","706188 DB matches per sequence\n","97 overflows\n","0 queries produce too many hits (truncated result)\n","287 sequences passed prefiltering per query sequence\n","300 median result list length\n","1 sequences with 0 size result lists\n","Time for merging to pref_0: 0h 0m 0s 0ms\n","Time for processing: 0h 0m 20s 334ms\n","align /content/mmseqs2/tmp/32/candidateDB_35 /content/mmseqs2/genexa/genexaDB.idx /content/mmseqs2/tmp/32/2643374105756916866/pref_0 /content/mmseqs2/tmp/32/resultsDB_35 --sub-mat nucl:nucleotide.out,aa:blosum62.out -a 0 --alignment-mode 2 --wrapped-scoring 0 -e 0.001 --min-seq-id 0 --min-aln-len 0 --seq-id-mode 0 --alt-ali 0 -c 0 --cov-mode 0 --max-seq-len 65535 --comp-bias-corr 1 --realign 0 --max-rejected 2147483647 --max-accept 2147483647 --add-self-matches 0 --db-load-mode 0 --pca 1 --pcb 1.5 --score-bias 0 --gap-open nucl:5,aa:11 --gap-extend nucl:2,aa:1 --zdrop 40 --threads 4 --compressed 0 -v 3 \n","\n","Index version: 16\n","Generated by:  16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","ScoreMatrix:  VTML80.out\n","Compute score and coverage\n","Query database size: 2159 type: Aminoacid\n","Target database size: 468351 type: Aminoacid\n","Calculation of alignments\n","[=================================================================] 100.00% 2.16K 16s 881ms\n","Time for merging to resultsDB_35: 0h 0m 0s 0ms\n","\n","620245 alignments calculated.\n","190375 sequence pairs passed the thresholds (0.306935 of overall calculated).\n","88.177399 hits per query sequence.\n","Time for processing: 0h 0m 16s 915ms\n","convertalis /content/mmseqs2/tmp/32/candidateDB_35 /content/mmseqs2/genexa/genexaDB /content/mmseqs2/tmp/32/resultsDB_35 /content/mmseqs2/tmp/32/resultDB_genexa.m8 --format-output query,target,evalue,raw \n","\n","MMseqs Version:        \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Substitution matrix    \tnucl:nucleotide.out,aa:blosum62.out\n","Alignment format       \t0\n","Format alignment output\tquery,target,evalue,raw\n","Translation table      \t1\n","Gap open cost          \tnucl:5,aa:11\n","Gap extension cost     \tnucl:2,aa:1\n","Database output        \tfalse\n","Preload mode           \t0\n","Search type            \t0\n","Threads                \t4\n","Compressed             \t0\n","Verbosity              \t3\n","\n","[=================================================================] 100.00% 2.16K 0s 141ms\n","Time for merging to resultDB_genexa.m8: 0h 0m 0s 5ms\n","Time for processing: 0h 0m 0s 214ms\n","search -s 7.0 /content/mmseqs2/tmp/32/candidateDB_53 /content/mmseqs2/swissprot/swissprotDB /content/mmseqs2/tmp/32/resultsDB_53 /content/mmseqs2/tmp/32 \n","\n","MMseqs Version:                     \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Substitution matrix                 \tnucl:nucleotide.out,aa:blosum62.out\n","Add backtrace                       \tfalse\n","Alignment mode                      \t2\n","Allow wrapped scoring               \tfalse\n","E-value threshold                   \t0.001\n","Seq. id. threshold                  \t0\n","Min alignment length                \t0\n","Seq. id. mode                       \t0\n","Alternative alignments              \t0\n","Coverage threshold                  \t0\n","Coverage mode                       \t0\n","Max sequence length                 \t65535\n","Compositional bias                  \t1\n","Realign hits                        \tfalse\n","Max reject                          \t2147483647\n","Max accept                          \t2147483647\n","Include identical seq. id.          \tfalse\n","Preload mode                        \t0\n","Pseudo count a                      \t1\n","Pseudo count b                      \t1.5\n","Score bias                          \t0\n","Gap open cost                       \tnucl:5,aa:11\n","Gap extension cost                  \tnucl:2,aa:1\n","Zdrop                               \t40\n","Threads                             \t4\n","Compressed                          \t0\n","Verbosity                           \t3\n","Seed substitution matrix            \tnucl:nucleotide.out,aa:VTML80.out\n","Sensitivity                         \t7\n","k-mer length                        \t0\n","k-score                             \t2147483647\n","Alphabet size                       \tnucl:5,aa:21\n","Max results per query               \t300\n","Split database                      \t0\n","Split mode                          \t2\n","Split memory limit                  \t0\n","Diagonal scoring                    \ttrue\n","Exact k-mer matching                \t0\n","Mask residues                       \t1\n","Mask lower case residues            \t0\n","Minimum diagonal score              \t15\n","Spaced k-mers                       \t1\n","Spaced k-mer pattern                \t\n","Local temporary path                \t\n","Rescore mode                        \t0\n","Remove hits by seq. id. and coverage\tfalse\n","Sort results                        \t0\n","Mask profile                        \t1\n","Profile e-value threshold           \t0.1\n","Global sequence weighting           \tfalse\n","Allow deletions                     \tfalse\n","Filter MSA                          \t1\n","Maximum seq. id. threshold          \t0.9\n","Minimum seq. id.                    \t0\n","Minimum score per column            \t-20\n","Minimum coverage                    \t0\n","Select N most diverse seqs          \t1000\n","Omit consensus                      \tfalse\n","Min codons in orf                   \t30\n","Max codons in length                \t32734\n","Max orf gaps                        \t2147483647\n","Contig start mode                   \t2\n","Contig end mode                     \t2\n","Orf start mode                      \t1\n","Forward frames                      \t1,2,3\n","Reverse frames                      \t1,2,3\n","Translation table                   \t1\n","Translate orf                       \t0\n","Use all table starts                \tfalse\n","Offset of numeric ids               \t0\n","Create lookup                       \t0\n","Add orf stop                        \tfalse\n","Chain overlapping alignments        \t0\n","Merge query                         \t1\n","Search type                         \t0\n","Search iterations                   \t1\n","Start sensitivity                   \t4\n","Search steps                        \t1\n","Slice search mode                   \tfalse\n","Strand selection                    \t1\n","Disk space limit                    \t0\n","MPI runner                          \t\n","Force restart with latest tmp       \tfalse\n","Remove temporary files              \tfalse\n","\n","prefilter /content/mmseqs2/tmp/32/candidateDB_53 /content/mmseqs2/swissprot/swissprotDB /content/mmseqs2/tmp/32/10699582436478484896/pref_0 --sub-mat nucl:nucleotide.out,aa:blosum62.out --seed-sub-mat nucl:nucleotide.out,aa:VTML80.out -k 0 --k-score 2147483647 --alph-size nucl:5,aa:21 --max-seq-len 65535 --max-seqs 300 --split 0 --split-mode 2 --split-memory-limit 0 -c 0 --cov-mode 0 --comp-bias-corr 1 --diag-score 1 --exact-kmer-matching 0 --mask 1 --mask-lower-case 0 --min-ungapped-score 15 --add-self-matches 0 --spaced-kmer-mode 1 --db-load-mode 0 --pca 1 --pcb 1.5 --threads 4 --compressed 0 -v 3 -s 7.0 \n","\n","Query database size: 2159 type: Aminoacid\n","Estimated memory consumption: 2G\n","Target database size: 562755 type: Aminoacid\n","Index table k-mer threshold: 100 at k-mer size 6 \n","Index table: counting k-mers\n","[=================================================================] 100.00% 562.76K 24s 90ms\n","Index table: Masked residues: 3164874\n","Index table: fill\n","[=================================================================] 100.00% 562.76K 17s 789ms\n","Index statistics\n","Entries:          192757622\n","DB size:          1591 MB\n","Avg k-mer size:   3.011838\n","Top 10 k-mers\n","    GPGGTL\t1835\n","    GQSWTV\t1706\n","    WGMFAT\t1638\n","    PGVFEV\t1634\n","    VLWQFW\t1623\n","    AYIRPN\t1587\n","    RSPKGV\t1585\n","    TPHKWY\t1560\n","    HIRVGW\t1548\n","    ITLSPY\t1541\n","Time for index table init: 0h 0m 43s 915ms\n","Process prefiltering step 1 of 1\n","\n","k-mer similarity threshold: 100\n","Starting prefiltering scores calculation (step 1 of 1)\n","Query db start 1 to 2159\n","Target db start 1 to 562755\n","[=================================================================] 100.00% 2.16K 21s 873ms\n","\n","899.175160 k-mers per position\n","870434 DB matches per sequence\n","171 overflows\n","0 queries produce too many hits (truncated result)\n","291 sequences passed prefiltering per query sequence\n","300 median result list length\n","0 sequences with 0 size result lists\n","Time for merging to pref_0: 0h 0m 0s 0ms\n","Time for processing: 0h 1m 6s 897ms\n","align /content/mmseqs2/tmp/32/candidateDB_53 /content/mmseqs2/swissprot/swissprotDB /content/mmseqs2/tmp/32/10699582436478484896/pref_0 /content/mmseqs2/tmp/32/resultsDB_53 --sub-mat nucl:nucleotide.out,aa:blosum62.out -a 0 --alignment-mode 2 --wrapped-scoring 0 -e 0.001 --min-seq-id 0 --min-aln-len 0 --seq-id-mode 0 --alt-ali 0 -c 0 --cov-mode 0 --max-seq-len 65535 --comp-bias-corr 1 --realign 0 --max-rejected 2147483647 --max-accept 2147483647 --add-self-matches 0 --db-load-mode 0 --pca 1 --pcb 1.5 --score-bias 0 --gap-open nucl:5,aa:11 --gap-extend nucl:2,aa:1 --zdrop 40 --threads 4 --compressed 0 -v 3 \n","\n","Compute score and coverage\n","Query database size: 2159 type: Aminoacid\n","Target database size: 562755 type: Aminoacid\n","Calculation of alignments\n","[=================================================================] 100.00% 2.16K 17s 353ms\n","Time for merging to resultsDB_53: 0h 0m 0s 0ms\n","\n","628615 alignments calculated.\n","172228 sequence pairs passed the thresholds (0.273980 of overall calculated).\n","79.772118 hits per query sequence.\n","Time for processing: 0h 0m 17s 419ms\n","convertalis /content/mmseqs2/tmp/32/candidateDB_53 /content/mmseqs2/swissprot/swissprotDB /content/mmseqs2/tmp/32/resultsDB_53 /content/mmseqs2/tmp/32/resultDB_secondary.m8 --format-output query,target,evalue,raw \n","\n","MMseqs Version:        \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Substitution matrix    \tnucl:nucleotide.out,aa:blosum62.out\n","Alignment format       \t0\n","Format alignment output\tquery,target,evalue,raw\n","Translation table      \t1\n","Gap open cost          \tnucl:5,aa:11\n","Gap extension cost     \tnucl:2,aa:1\n","Database output        \tfalse\n","Preload mode           \t0\n","Search type            \t0\n","Threads                \t4\n","Compressed             \t0\n","Verbosity              \t3\n","\n","[=================================================================] 100.00% 2.16K 0s 121ms\n","Time for merging to resultDB_secondary.m8: 0h 0m 0s 5ms\n","Time for processing: 0h 0m 0s 212ms\n","found 1689 genes\n","\n","\n","Creating weighted directed acyclic graph of all ORFs and subORFs on contig 1...\n","\n"],"name":"stdout"},{"output_type":"display_data","data":{"application/vnd.jupyter.widget-view+json":{"model_id":"f13b81222bfa4e74a80096f3e476fe79","version_minor":0,"version_major":2},"text/plain":["HBox(children=(FloatProgress(value=0.0, max=2629.0), HTML(value='')))"]},"metadata":{"tags":[]}},{"output_type":"stream","text":["\n","Maximizing geneiness...\n","\n","Filtering predictions with mmseqs2...\n","createdb /content/mmseqs2/tmp/33/candidate_genes_35.fasta /content/mmseqs2/tmp/33/candidateDB_35 \n","\n","MMseqs Version:       \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Database type         \t0\n","Shuffle input database\ttrue\n","Createdb mode         \t0\n","Write lookup file     \t1\n","Offset of numeric ids \t0\n","Compressed            \t0\n","Verbosity             \t3\n","\n","Converting sequences\n","[304] 0s 2ms\n","Time for merging to candidateDB_35_h: 0h 0m 0s 1ms\n","Time for merging to candidateDB_35: 0h 0m 0s 1ms\n","Database type: Aminoacid\n","Time for processing: 0h 0m 0s 8ms\n","createdb /content/mmseqs2/tmp/33/candidate_genes_53.fasta /content/mmseqs2/tmp/33/candidateDB_53 \n","\n","MMseqs Version:       \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Database type         \t0\n","Shuffle input database\ttrue\n","Createdb mode         \t0\n","Write lookup file     \t1\n","Offset of numeric ids \t0\n","Compressed            \t0\n","Verbosity             \t3\n","\n","Converting sequences\n","[304] 0s 2ms\n","Time for merging to candidateDB_53_h: 0h 0m 0s 1ms\n","Time for merging to candidateDB_53: 0h 0m 0s 1ms\n","Database type: Aminoacid\n","Time for processing: 0h 0m 0s 8ms\n","search -s 7.0 /content/mmseqs2/tmp/33/candidateDB_35 /content/mmseqs2/genexa/genexaDB /content/mmseqs2/tmp/33/resultsDB_35 /content/mmseqs2/tmp/33 \n","\n","MMseqs Version:                     \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Substitution matrix                 \tnucl:nucleotide.out,aa:blosum62.out\n","Add backtrace                       \tfalse\n","Alignment mode                      \t2\n","Allow wrapped scoring               \tfalse\n","E-value threshold                   \t0.001\n","Seq. id. threshold                  \t0\n","Min alignment length                \t0\n","Seq. id. mode                       \t0\n","Alternative alignments              \t0\n","Coverage threshold                  \t0\n","Coverage mode                       \t0\n","Max sequence length                 \t65535\n","Compositional bias                  \t1\n","Realign hits                        \tfalse\n","Max reject                          \t2147483647\n","Max accept                          \t2147483647\n","Include identical seq. id.          \tfalse\n","Preload mode                        \t0\n","Pseudo count a                      \t1\n","Pseudo count b                      \t1.5\n","Score bias                          \t0\n","Gap open cost                       \tnucl:5,aa:11\n","Gap extension cost                  \tnucl:2,aa:1\n","Zdrop                               \t40\n","Threads                             \t4\n","Compressed                          \t0\n","Verbosity                           \t3\n","Seed substitution matrix            \tnucl:nucleotide.out,aa:VTML80.out\n","Sensitivity                         \t7\n","k-mer length                        \t0\n","k-score                             \t2147483647\n","Alphabet size                       \tnucl:5,aa:21\n","Max results per query               \t300\n","Split database                      \t0\n","Split mode                          \t2\n","Split memory limit                  \t0\n","Diagonal scoring                    \ttrue\n","Exact k-mer matching                \t0\n","Mask residues                       \t1\n","Mask lower case residues            \t0\n","Minimum diagonal score              \t15\n","Spaced k-mers                       \t1\n","Spaced k-mer pattern                \t\n","Local temporary path                \t\n","Rescore mode                        \t0\n","Remove hits by seq. id. and coverage\tfalse\n","Sort results                        \t0\n","Mask profile                        \t1\n","Profile e-value threshold           \t0.1\n","Global sequence weighting           \tfalse\n","Allow deletions                     \tfalse\n","Filter MSA                          \t1\n","Maximum seq. id. threshold          \t0.9\n","Minimum seq. id.                    \t0\n","Minimum score per column            \t-20\n","Minimum coverage                    \t0\n","Select N most diverse seqs          \t1000\n","Omit consensus                      \tfalse\n","Min codons in orf                   \t30\n","Max codons in length                \t32734\n","Max orf gaps                        \t2147483647\n","Contig start mode                   \t2\n","Contig end mode                     \t2\n","Orf start mode                      \t1\n","Forward frames                      \t1,2,3\n","Reverse frames                      \t1,2,3\n","Translation table                   \t1\n","Translate orf                       \t0\n","Use all table starts                \tfalse\n","Offset of numeric ids               \t0\n","Create lookup                       \t0\n","Add orf stop                        \tfalse\n","Chain overlapping alignments        \t0\n","Merge query                         \t1\n","Search type                         \t0\n","Search iterations                   \t1\n","Start sensitivity                   \t4\n","Search steps                        \t1\n","Slice search mode                   \tfalse\n","Strand selection                    \t1\n","Disk space limit                    \t0\n","MPI runner                          \t\n","Force restart with latest tmp       \tfalse\n","Remove temporary files              \tfalse\n","\n","prefilter /content/mmseqs2/tmp/33/candidateDB_35 /content/mmseqs2/genexa/genexaDB.idx /content/mmseqs2/tmp/33/15829832076356772993/pref_0 --sub-mat nucl:nucleotide.out,aa:blosum62.out --seed-sub-mat nucl:nucleotide.out,aa:VTML80.out -k 0 --k-score 2147483647 --alph-size nucl:5,aa:21 --max-seq-len 65535 --max-seqs 300 --split 0 --split-mode 2 --split-memory-limit 0 -c 0 --cov-mode 0 --comp-bias-corr 1 --diag-score 1 --exact-kmer-matching 0 --mask 1 --mask-lower-case 0 --min-ungapped-score 15 --add-self-matches 0 --spaced-kmer-mode 1 --db-load-mode 0 --pca 1 --pcb 1.5 --threads 4 --compressed 0 -v 3 -s 7.0 \n","\n","Index version: 16\n","Generated by:  16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","ScoreMatrix:  VTML80.out\n","Query database size: 337 type: Aminoacid\n","Estimated memory consumption: 2G\n","Target database size: 468351 type: Aminoacid\n","Process prefiltering step 1 of 1\n","\n","k-mer similarity threshold: 100\n","Starting prefiltering scores calculation (step 1 of 1)\n","Query db start 1 to 337\n","Target db start 1 to 468351\n","[=================================================================] 100.00% 337 2s 91ms\n","\n","775.994105 k-mers per position\n","524129 DB matches per sequence\n","18 overflows\n","0 queries produce too many hits (truncated result)\n","274 sequences passed prefiltering per query sequence\n","300 median result list length\n","0 sequences with 0 size result lists\n","Time for merging to pref_0: 0h 0m 0s 0ms\n","Time for processing: 0h 0m 3s 568ms\n","align /content/mmseqs2/tmp/33/candidateDB_35 /content/mmseqs2/genexa/genexaDB.idx /content/mmseqs2/tmp/33/15829832076356772993/pref_0 /content/mmseqs2/tmp/33/resultsDB_35 --sub-mat nucl:nucleotide.out,aa:blosum62.out -a 0 --alignment-mode 2 --wrapped-scoring 0 -e 0.001 --min-seq-id 0 --min-aln-len 0 --seq-id-mode 0 --alt-ali 0 -c 0 --cov-mode 0 --max-seq-len 65535 --comp-bias-corr 1 --realign 0 --max-rejected 2147483647 --max-accept 2147483647 --add-self-matches 0 --db-load-mode 0 --pca 1 --pcb 1.5 --score-bias 0 --gap-open nucl:5,aa:11 --gap-extend nucl:2,aa:1 --zdrop 40 --threads 4 --compressed 0 -v 3 \n","\n","Index version: 16\n","Generated by:  16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","ScoreMatrix:  VTML80.out\n","Compute score and coverage\n","Query database size: 337 type: Aminoacid\n","Target database size: 468351 type: Aminoacid\n","Calculation of alignments\n","[=================================================================] 100.00% 337 3s 163ms\n","Time for merging to resultsDB_35: 0h 0m 0s 0ms\n","\n","92507 alignments calculated.\n","35742 sequence pairs passed the thresholds (0.386371 of overall calculated).\n","106.059349 hits per query sequence.\n","Time for processing: 0h 0m 3s 196ms\n","convertalis /content/mmseqs2/tmp/33/candidateDB_35 /content/mmseqs2/genexa/genexaDB /content/mmseqs2/tmp/33/resultsDB_35 /content/mmseqs2/tmp/33/resultDB_genexa.m8 --format-output query,target,evalue,raw \n","\n","MMseqs Version:        \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Substitution matrix    \tnucl:nucleotide.out,aa:blosum62.out\n","Alignment format       \t0\n","Format alignment output\tquery,target,evalue,raw\n","Translation table      \t1\n","Gap open cost          \tnucl:5,aa:11\n","Gap extension cost     \tnucl:2,aa:1\n","Database output        \tfalse\n","Preload mode           \t0\n","Search type            \t0\n","Threads                \t4\n","Compressed             \t0\n","Verbosity              \t3\n","\n","[=================================================================] 100.00% 337 0s 27ms\n","Time for merging to resultDB_genexa.m8: 0h 0m 0s 1ms\n","Time for processing: 0h 0m 0s 91ms\n","search -s 7.0 /content/mmseqs2/tmp/33/candidateDB_53 /content/mmseqs2/swissprot/swissprotDB /content/mmseqs2/tmp/33/resultsDB_53 /content/mmseqs2/tmp/33 \n","\n","MMseqs Version:                     \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Substitution matrix                 \tnucl:nucleotide.out,aa:blosum62.out\n","Add backtrace                       \tfalse\n","Alignment mode                      \t2\n","Allow wrapped scoring               \tfalse\n","E-value threshold                   \t0.001\n","Seq. id. threshold                  \t0\n","Min alignment length                \t0\n","Seq. id. mode                       \t0\n","Alternative alignments              \t0\n","Coverage threshold                  \t0\n","Coverage mode                       \t0\n","Max sequence length                 \t65535\n","Compositional bias                  \t1\n","Realign hits                        \tfalse\n","Max reject                          \t2147483647\n","Max accept                          \t2147483647\n","Include identical seq. id.          \tfalse\n","Preload mode                        \t0\n","Pseudo count a                      \t1\n","Pseudo count b                      \t1.5\n","Score bias                          \t0\n","Gap open cost                       \tnucl:5,aa:11\n","Gap extension cost                  \tnucl:2,aa:1\n","Zdrop                               \t40\n","Threads                             \t4\n","Compressed                          \t0\n","Verbosity                           \t3\n","Seed substitution matrix            \tnucl:nucleotide.out,aa:VTML80.out\n","Sensitivity                         \t7\n","k-mer length                        \t0\n","k-score                             \t2147483647\n","Alphabet size                       \tnucl:5,aa:21\n","Max results per query               \t300\n","Split database                      \t0\n","Split mode                          \t2\n","Split memory limit                  \t0\n","Diagonal scoring                    \ttrue\n","Exact k-mer matching                \t0\n","Mask residues                       \t1\n","Mask lower case residues            \t0\n","Minimum diagonal score              \t15\n","Spaced k-mers                       \t1\n","Spaced k-mer pattern                \t\n","Local temporary path                \t\n","Rescore mode                        \t0\n","Remove hits by seq. id. and coverage\tfalse\n","Sort results                        \t0\n","Mask profile                        \t1\n","Profile e-value threshold           \t0.1\n","Global sequence weighting           \tfalse\n","Allow deletions                     \tfalse\n","Filter MSA                          \t1\n","Maximum seq. id. threshold          \t0.9\n","Minimum seq. id.                    \t0\n","Minimum score per column            \t-20\n","Minimum coverage                    \t0\n","Select N most diverse seqs          \t1000\n","Omit consensus                      \tfalse\n","Min codons in orf                   \t30\n","Max codons in length                \t32734\n","Max orf gaps                        \t2147483647\n","Contig start mode                   \t2\n","Contig end mode                     \t2\n","Orf start mode                      \t1\n","Forward frames                      \t1,2,3\n","Reverse frames                      \t1,2,3\n","Translation table                   \t1\n","Translate orf                       \t0\n","Use all table starts                \tfalse\n","Offset of numeric ids               \t0\n","Create lookup                       \t0\n","Add orf stop                        \tfalse\n","Chain overlapping alignments        \t0\n","Merge query                         \t1\n","Search type                         \t0\n","Search iterations                   \t1\n","Start sensitivity                   \t4\n","Search steps                        \t1\n","Slice search mode                   \tfalse\n","Strand selection                    \t1\n","Disk space limit                    \t0\n","MPI runner                          \t\n","Force restart with latest tmp       \tfalse\n","Remove temporary files              \tfalse\n","\n","prefilter /content/mmseqs2/tmp/33/candidateDB_53 /content/mmseqs2/swissprot/swissprotDB /content/mmseqs2/tmp/33/10800947424523830879/pref_0 --sub-mat nucl:nucleotide.out,aa:blosum62.out --seed-sub-mat nucl:nucleotide.out,aa:VTML80.out -k 0 --k-score 2147483647 --alph-size nucl:5,aa:21 --max-seq-len 65535 --max-seqs 300 --split 0 --split-mode 2 --split-memory-limit 0 -c 0 --cov-mode 0 --comp-bias-corr 1 --diag-score 1 --exact-kmer-matching 0 --mask 1 --mask-lower-case 0 --min-ungapped-score 15 --add-self-matches 0 --spaced-kmer-mode 1 --db-load-mode 0 --pca 1 --pcb 1.5 --threads 4 --compressed 0 -v 3 -s 7.0 \n","\n","Query database size: 337 type: Aminoacid\n","Estimated memory consumption: 2G\n","Target database size: 562755 type: Aminoacid\n","Index table k-mer threshold: 100 at k-mer size 6 \n","Index table: counting k-mers\n","[=================================================================] 100.00% 562.76K 24s 102ms\n","Index table: Masked residues: 3164874\n","Index table: fill\n","[=================================================================] 100.00% 562.76K 17s 795ms\n","Index statistics\n","Entries:          192757622\n","DB size:          1591 MB\n","Avg k-mer size:   3.011838\n","Top 10 k-mers\n","    GPGGTL\t1835\n","    GQSWTV\t1706\n","    WGMFAT\t1638\n","    PGVFEV\t1634\n","    VLWQFW\t1623\n","    AYIRPN\t1587\n","    RSPKGV\t1585\n","    TPHKWY\t1560\n","    HIRVGW\t1548\n","    ITLSPY\t1541\n","Time for index table init: 0h 0m 43s 996ms\n","Process prefiltering step 1 of 1\n","\n","k-mer similarity threshold: 100\n","Starting prefiltering scores calculation (step 1 of 1)\n","Query db start 1 to 337\n","Target db start 1 to 562755\n","[=================================================================] 100.00% 337 2s 418ms\n","\n","772.574858 k-mers per position\n","640119 DB matches per sequence\n","23 overflows\n","0 queries produce too many hits (truncated result)\n","280 sequences passed prefiltering per query sequence\n","300 median result list length\n","0 sequences with 0 size result lists\n","Time for merging to pref_0: 0h 0m 0s 0ms\n","Time for processing: 0h 0m 47s 619ms\n","align /content/mmseqs2/tmp/33/candidateDB_53 /content/mmseqs2/swissprot/swissprotDB /content/mmseqs2/tmp/33/10800947424523830879/pref_0 /content/mmseqs2/tmp/33/resultsDB_53 --sub-mat nucl:nucleotide.out,aa:blosum62.out -a 0 --alignment-mode 2 --wrapped-scoring 0 -e 0.001 --min-seq-id 0 --min-aln-len 0 --seq-id-mode 0 --alt-ali 0 -c 0 --cov-mode 0 --max-seq-len 65535 --comp-bias-corr 1 --realign 0 --max-rejected 2147483647 --max-accept 2147483647 --add-self-matches 0 --db-load-mode 0 --pca 1 --pcb 1.5 --score-bias 0 --gap-open nucl:5,aa:11 --gap-extend nucl:2,aa:1 --zdrop 40 --threads 4 --compressed 0 -v 3 \n","\n","Compute score and coverage\n","Query database size: 337 type: Aminoacid\n","Target database size: 562755 type: Aminoacid\n","Calculation of alignments\n","[=================================================================] 100.00% 337 2s 838ms\n","Time for merging to resultsDB_53: 0h 0m 0s 0ms\n","\n","94508 alignments calculated.\n","33480 sequence pairs passed the thresholds (0.354256 of overall calculated).\n","99.347183 hits per query sequence.\n","Time for processing: 0h 0m 2s 966ms\n","convertalis /content/mmseqs2/tmp/33/candidateDB_53 /content/mmseqs2/swissprot/swissprotDB /content/mmseqs2/tmp/33/resultsDB_53 /content/mmseqs2/tmp/33/resultDB_secondary.m8 --format-output query,target,evalue,raw \n","\n","MMseqs Version:        \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Substitution matrix    \tnucl:nucleotide.out,aa:blosum62.out\n","Alignment format       \t0\n","Format alignment output\tquery,target,evalue,raw\n","Translation table      \t1\n","Gap open cost          \tnucl:5,aa:11\n","Gap extension cost     \tnucl:2,aa:1\n","Database output        \tfalse\n","Preload mode           \t0\n","Search type            \t0\n","Threads                \t4\n","Compressed             \t0\n","Verbosity              \t3\n","\n","[=================================================================] 100.00% 337 0s 24ms\n","Time for merging to resultDB_secondary.m8: 0h 0m 0s 1ms\n","Time for processing: 0h 0m 0s 108ms\n","found 222 genes\n","\n","\n","Reading fasta file GCF_000184705.1_ASM18470v1_genomic.fna.gz...\n","\n","Finding and translating open reading frames...\n","\n","Encoding amino acids...\n","\n","Applying kmer prefilter...\n","\n","Scoring ORFs with temporal convolutional network...\n","\n"],"name":"stdout"},{"output_type":"display_data","data":{"application/vnd.jupyter.widget-view+json":{"model_id":"10ee481c890b4dc4af099c17b074adaf","version_minor":0,"version_major":2},"text/plain":["HBox(children=(FloatProgress(value=0.0, max=54.0), HTML(value='')))"]},"metadata":{"tags":[]}},{"output_type":"stream","text":["\n","Scoring translation initiation sites...\n","\n"],"name":"stdout"},{"output_type":"display_data","data":{"application/vnd.jupyter.widget-view+json":{"model_id":"a525832e1581473b9c905d43ac755765","version_minor":0,"version_major":2},"text/plain":["HBox(children=(FloatProgress(value=0.0, max=16.0), HTML(value='')))"]},"metadata":{"tags":[]}},{"output_type":"stream","text":["\n","Creating weighted directed acyclic graph of all ORFs and subORFs on contig 0...\n","\n"],"name":"stdout"},{"output_type":"display_data","data":{"application/vnd.jupyter.widget-view+json":{"model_id":"a52dda18eb02436aa05d93ee6cd2d293","version_minor":0,"version_major":2},"text/plain":["HBox(children=(FloatProgress(value=0.0, max=159003.0), HTML(value='')))"]},"metadata":{"tags":[]}},{"output_type":"stream","text":["\n","Maximizing geneiness...\n","\n","Filtering predictions with mmseqs2...\n","createdb /content/mmseqs2/tmp/34/candidate_genes_35.fasta /content/mmseqs2/tmp/34/candidateDB_35 \n","\n","MMseqs Version:       \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Database type         \t0\n","Shuffle input database\ttrue\n","Createdb mode         \t0\n","Write lookup file     \t1\n","Offset of numeric ids \t0\n","Compressed            \t0\n","Verbosity             \t3\n","\n","Converting sequences\n","[3940] 0s 6ms\n","Time for merging to candidateDB_35_h: 0h 0m 0s 3ms\n","Time for merging to candidateDB_35: 0h 0m 0s 4ms\n","Database type: Aminoacid\n","Time for processing: 0h 0m 0s 20ms\n","createdb /content/mmseqs2/tmp/34/candidate_genes_53.fasta /content/mmseqs2/tmp/34/candidateDB_53 \n","\n","MMseqs Version:       \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Database type         \t0\n","Shuffle input database\ttrue\n","Createdb mode         \t0\n","Write lookup file     \t1\n","Offset of numeric ids \t0\n","Compressed            \t0\n","Verbosity             \t3\n","\n","Converting sequences\n","[3940] 0s 5ms\n","Time for merging to candidateDB_53_h: 0h 0m 0s 3ms\n","Time for merging to candidateDB_53: 0h 0m 0s 4ms\n","Database type: Aminoacid\n","Time for processing: 0h 0m 0s 19ms\n","search -s 7.0 /content/mmseqs2/tmp/34/candidateDB_35 /content/mmseqs2/genexa/genexaDB /content/mmseqs2/tmp/34/resultsDB_35 /content/mmseqs2/tmp/34 \n","\n","MMseqs Version:                     \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Substitution matrix                 \tnucl:nucleotide.out,aa:blosum62.out\n","Add backtrace                       \tfalse\n","Alignment mode                      \t2\n","Allow wrapped scoring               \tfalse\n","E-value threshold                   \t0.001\n","Seq. id. threshold                  \t0\n","Min alignment length                \t0\n","Seq. id. mode                       \t0\n","Alternative alignments              \t0\n","Coverage threshold                  \t0\n","Coverage mode                       \t0\n","Max sequence length                 \t65535\n","Compositional bias                  \t1\n","Realign hits                        \tfalse\n","Max reject                          \t2147483647\n","Max accept                          \t2147483647\n","Include identical seq. id.          \tfalse\n","Preload mode                        \t0\n","Pseudo count a                      \t1\n","Pseudo count b                      \t1.5\n","Score bias                          \t0\n","Gap open cost                       \tnucl:5,aa:11\n","Gap extension cost                  \tnucl:2,aa:1\n","Zdrop                               \t40\n","Threads                             \t4\n","Compressed                          \t0\n","Verbosity                           \t3\n","Seed substitution matrix            \tnucl:nucleotide.out,aa:VTML80.out\n","Sensitivity                         \t7\n","k-mer length                        \t0\n","k-score                             \t2147483647\n","Alphabet size                       \tnucl:5,aa:21\n","Max results per query               \t300\n","Split database                      \t0\n","Split mode                          \t2\n","Split memory limit                  \t0\n","Diagonal scoring                    \ttrue\n","Exact k-mer matching                \t0\n","Mask residues                       \t1\n","Mask lower case residues            \t0\n","Minimum diagonal score              \t15\n","Spaced k-mers                       \t1\n","Spaced k-mer pattern                \t\n","Local temporary path                \t\n","Rescore mode                        \t0\n","Remove hits by seq. id. and coverage\tfalse\n","Sort results                        \t0\n","Mask profile                        \t1\n","Profile e-value threshold           \t0.1\n","Global sequence weighting           \tfalse\n","Allow deletions                     \tfalse\n","Filter MSA                          \t1\n","Maximum seq. id. threshold          \t0.9\n","Minimum seq. id.                    \t0\n","Minimum score per column            \t-20\n","Minimum coverage                    \t0\n","Select N most diverse seqs          \t1000\n","Omit consensus                      \tfalse\n","Min codons in orf                   \t30\n","Max codons in length                \t32734\n","Max orf gaps                        \t2147483647\n","Contig start mode                   \t2\n","Contig end mode                     \t2\n","Orf start mode                      \t1\n","Forward frames                      \t1,2,3\n","Reverse frames                      \t1,2,3\n","Translation table                   \t1\n","Translate orf                       \t0\n","Use all table starts                \tfalse\n","Offset of numeric ids               \t0\n","Create lookup                       \t0\n","Add orf stop                        \tfalse\n","Chain overlapping alignments        \t0\n","Merge query                         \t1\n","Search type                         \t0\n","Search iterations                   \t1\n","Start sensitivity                   \t4\n","Search steps                        \t1\n","Slice search mode                   \tfalse\n","Strand selection                    \t1\n","Disk space limit                    \t0\n","MPI runner                          \t\n","Force restart with latest tmp       \tfalse\n","Remove temporary files              \tfalse\n","\n","prefilter /content/mmseqs2/tmp/34/candidateDB_35 /content/mmseqs2/genexa/genexaDB.idx /content/mmseqs2/tmp/34/10569545973247077504/pref_0 --sub-mat nucl:nucleotide.out,aa:blosum62.out --seed-sub-mat nucl:nucleotide.out,aa:VTML80.out -k 0 --k-score 2147483647 --alph-size nucl:5,aa:21 --max-seq-len 65535 --max-seqs 300 --split 0 --split-mode 2 --split-memory-limit 0 -c 0 --cov-mode 0 --comp-bias-corr 1 --diag-score 1 --exact-kmer-matching 0 --mask 1 --mask-lower-case 0 --min-ungapped-score 15 --add-self-matches 0 --spaced-kmer-mode 1 --db-load-mode 0 --pca 1 --pcb 1.5 --threads 4 --compressed 0 -v 3 -s 7.0 \n","\n","Index version: 16\n","Generated by:  16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","ScoreMatrix:  VTML80.out\n","Query database size: 3981 type: Aminoacid\n","Estimated memory consumption: 2G\n","Target database size: 468351 type: Aminoacid\n","Process prefiltering step 1 of 1\n","\n","k-mer similarity threshold: 100\n","Starting prefiltering scores calculation (step 1 of 1)\n","Query db start 1 to 3981\n","Target db start 1 to 468351\n","[=================================================================] 100.00% 3.98K 27s 713ms\n","\n","698.905400 k-mers per position\n","586850 DB matches per sequence\n","139 overflows\n","0 queries produce too many hits (truncated result)\n","262 sequences passed prefiltering per query sequence\n","300 median result list length\n","2 sequences with 0 size result lists\n","Time for merging to pref_0: 0h 0m 0s 1ms\n","Time for processing: 0h 0m 29s 134ms\n","align /content/mmseqs2/tmp/34/candidateDB_35 /content/mmseqs2/genexa/genexaDB.idx /content/mmseqs2/tmp/34/10569545973247077504/pref_0 /content/mmseqs2/tmp/34/resultsDB_35 --sub-mat nucl:nucleotide.out,aa:blosum62.out -a 0 --alignment-mode 2 --wrapped-scoring 0 -e 0.001 --min-seq-id 0 --min-aln-len 0 --seq-id-mode 0 --alt-ali 0 -c 0 --cov-mode 0 --max-seq-len 65535 --comp-bias-corr 1 --realign 0 --max-rejected 2147483647 --max-accept 2147483647 --add-self-matches 0 --db-load-mode 0 --pca 1 --pcb 1.5 --score-bias 0 --gap-open nucl:5,aa:11 --gap-extend nucl:2,aa:1 --zdrop 40 --threads 4 --compressed 0 -v 3 \n","\n","Index version: 16\n","Generated by:  16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","ScoreMatrix:  VTML80.out\n","Compute score and coverage\n","Query database size: 3981 type: Aminoacid\n","Target database size: 468351 type: Aminoacid\n","Calculation of alignments\n","[=================================================================] 100.00% 3.98K 26s 897ms\n","Time for merging to resultsDB_35: 0h 0m 0s 1ms\n","\n","1044662 alignments calculated.\n","305638 sequence pairs passed the thresholds (0.292571 of overall calculated).\n","76.774178 hits per query sequence.\n","Time for processing: 0h 0m 26s 934ms\n","convertalis /content/mmseqs2/tmp/34/candidateDB_35 /content/mmseqs2/genexa/genexaDB /content/mmseqs2/tmp/34/resultsDB_35 /content/mmseqs2/tmp/34/resultDB_genexa.m8 --format-output query,target,evalue,raw \n","\n","MMseqs Version:        \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Substitution matrix    \tnucl:nucleotide.out,aa:blosum62.out\n","Alignment format       \t0\n","Format alignment output\tquery,target,evalue,raw\n","Translation table      \t1\n","Gap open cost          \tnucl:5,aa:11\n","Gap extension cost     \tnucl:2,aa:1\n","Database output        \tfalse\n","Preload mode           \t0\n","Search type            \t0\n","Threads                \t4\n","Compressed             \t0\n","Verbosity              \t3\n","\n","[=================================================================] 100.00% 3.98K 0s 224ms\n","Time for merging to resultDB_genexa.m8: 0h 0m 0s 8ms\n","Time for processing: 0h 0m 0s 301ms\n","search -s 7.0 /content/mmseqs2/tmp/34/candidateDB_53 /content/mmseqs2/swissprot/swissprotDB /content/mmseqs2/tmp/34/resultsDB_53 /content/mmseqs2/tmp/34 \n","\n","MMseqs Version:                     \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Substitution matrix                 \tnucl:nucleotide.out,aa:blosum62.out\n","Add backtrace                       \tfalse\n","Alignment mode                      \t2\n","Allow wrapped scoring               \tfalse\n","E-value threshold                   \t0.001\n","Seq. id. threshold                  \t0\n","Min alignment length                \t0\n","Seq. id. mode                       \t0\n","Alternative alignments              \t0\n","Coverage threshold                  \t0\n","Coverage mode                       \t0\n","Max sequence length                 \t65535\n","Compositional bias                  \t1\n","Realign hits                        \tfalse\n","Max reject                          \t2147483647\n","Max accept                          \t2147483647\n","Include identical seq. id.          \tfalse\n","Preload mode                        \t0\n","Pseudo count a                      \t1\n","Pseudo count b                      \t1.5\n","Score bias                          \t0\n","Gap open cost                       \tnucl:5,aa:11\n","Gap extension cost                  \tnucl:2,aa:1\n","Zdrop                               \t40\n","Threads                             \t4\n","Compressed                          \t0\n","Verbosity                           \t3\n","Seed substitution matrix            \tnucl:nucleotide.out,aa:VTML80.out\n","Sensitivity                         \t7\n","k-mer length                        \t0\n","k-score                             \t2147483647\n","Alphabet size                       \tnucl:5,aa:21\n","Max results per query               \t300\n","Split database                      \t0\n","Split mode                          \t2\n","Split memory limit                  \t0\n","Diagonal scoring                    \ttrue\n","Exact k-mer matching                \t0\n","Mask residues                       \t1\n","Mask lower case residues            \t0\n","Minimum diagonal score              \t15\n","Spaced k-mers                       \t1\n","Spaced k-mer pattern                \t\n","Local temporary path                \t\n","Rescore mode                        \t0\n","Remove hits by seq. id. and coverage\tfalse\n","Sort results                        \t0\n","Mask profile                        \t1\n","Profile e-value threshold           \t0.1\n","Global sequence weighting           \tfalse\n","Allow deletions                     \tfalse\n","Filter MSA                          \t1\n","Maximum seq. id. threshold          \t0.9\n","Minimum seq. id.                    \t0\n","Minimum score per column            \t-20\n","Minimum coverage                    \t0\n","Select N most diverse seqs          \t1000\n","Omit consensus                      \tfalse\n","Min codons in orf                   \t30\n","Max codons in length                \t32734\n","Max orf gaps                        \t2147483647\n","Contig start mode                   \t2\n","Contig end mode                     \t2\n","Orf start mode                      \t1\n","Forward frames                      \t1,2,3\n","Reverse frames                      \t1,2,3\n","Translation table                   \t1\n","Translate orf                       \t0\n","Use all table starts                \tfalse\n","Offset of numeric ids               \t0\n","Create lookup                       \t0\n","Add orf stop                        \tfalse\n","Chain overlapping alignments        \t0\n","Merge query                         \t1\n","Search type                         \t0\n","Search iterations                   \t1\n","Start sensitivity                   \t4\n","Search steps                        \t1\n","Slice search mode                   \tfalse\n","Strand selection                    \t1\n","Disk space limit                    \t0\n","MPI runner                          \t\n","Force restart with latest tmp       \tfalse\n","Remove temporary files              \tfalse\n","\n","prefilter /content/mmseqs2/tmp/34/candidateDB_53 /content/mmseqs2/swissprot/swissprotDB /content/mmseqs2/tmp/34/10902312412569176862/pref_0 --sub-mat nucl:nucleotide.out,aa:blosum62.out --seed-sub-mat nucl:nucleotide.out,aa:VTML80.out -k 0 --k-score 2147483647 --alph-size nucl:5,aa:21 --max-seq-len 65535 --max-seqs 300 --split 0 --split-mode 2 --split-memory-limit 0 -c 0 --cov-mode 0 --comp-bias-corr 1 --diag-score 1 --exact-kmer-matching 0 --mask 1 --mask-lower-case 0 --min-ungapped-score 15 --add-self-matches 0 --spaced-kmer-mode 1 --db-load-mode 0 --pca 1 --pcb 1.5 --threads 4 --compressed 0 -v 3 -s 7.0 \n","\n","Query database size: 3981 type: Aminoacid\n","Estimated memory consumption: 2G\n","Target database size: 562755 type: Aminoacid\n","Index table k-mer threshold: 100 at k-mer size 6 \n","Index table: counting k-mers\n","[=================================================================] 100.00% 562.76K 24s 105ms\n","Index table: Masked residues: 3164874\n","Index table: fill\n","[=================================================================] 100.00% 562.76K 17s 814ms\n","Index statistics\n","Entries:          192757622\n","DB size:          1591 MB\n","Avg k-mer size:   3.011838\n","Top 10 k-mers\n","    GPGGTL\t1835\n","    GQSWTV\t1706\n","    WGMFAT\t1638\n","    PGVFEV\t1634\n","    VLWQFW\t1623\n","    AYIRPN\t1587\n","    RSPKGV\t1585\n","    TPHKWY\t1560\n","    HIRVGW\t1548\n","    ITLSPY\t1541\n","Time for index table init: 0h 0m 44s 63ms\n","Process prefiltering step 1 of 1\n","\n","k-mer similarity threshold: 100\n","Starting prefiltering scores calculation (step 1 of 1)\n","Query db start 1 to 3981\n","Target db start 1 to 562755\n","[=================================================================] 100.00% 3.98K 29s 482ms\n","\n","699.595770 k-mers per position\n","644215 DB matches per sequence\n","195 overflows\n","0 queries produce too many hits (truncated result)\n","267 sequences passed prefiltering per query sequence\n","300 median result list length\n","1 sequences with 0 size result lists\n","Time for merging to pref_0: 0h 0m 0s 1ms\n","Time for processing: 0h 1m 14s 686ms\n","align /content/mmseqs2/tmp/34/candidateDB_53 /content/mmseqs2/swissprot/swissprotDB /content/mmseqs2/tmp/34/10902312412569176862/pref_0 /content/mmseqs2/tmp/34/resultsDB_53 --sub-mat nucl:nucleotide.out,aa:blosum62.out -a 0 --alignment-mode 2 --wrapped-scoring 0 -e 0.001 --min-seq-id 0 --min-aln-len 0 --seq-id-mode 0 --alt-ali 0 -c 0 --cov-mode 0 --max-seq-len 65535 --comp-bias-corr 1 --realign 0 --max-rejected 2147483647 --max-accept 2147483647 --add-self-matches 0 --db-load-mode 0 --pca 1 --pcb 1.5 --score-bias 0 --gap-open nucl:5,aa:11 --gap-extend nucl:2,aa:1 --zdrop 40 --threads 4 --compressed 0 -v 3 \n","\n","Compute score and coverage\n","Query database size: 3981 type: Aminoacid\n","Target database size: 562755 type: Aminoacid\n","Calculation of alignments\n","[=================================================================] 100.00% 3.98K 25s 274ms\n","Time for merging to resultsDB_53: 0h 0m 0s 1ms\n","\n","1063566 alignments calculated.\n","232455 sequence pairs passed the thresholds (0.218562 of overall calculated).\n","58.391109 hits per query sequence.\n","Time for processing: 0h 0m 25s 343ms\n","convertalis /content/mmseqs2/tmp/34/candidateDB_53 /content/mmseqs2/swissprot/swissprotDB /content/mmseqs2/tmp/34/resultsDB_53 /content/mmseqs2/tmp/34/resultDB_secondary.m8 --format-output query,target,evalue,raw \n","\n","MMseqs Version:        \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Substitution matrix    \tnucl:nucleotide.out,aa:blosum62.out\n","Alignment format       \t0\n","Format alignment output\tquery,target,evalue,raw\n","Translation table      \t1\n","Gap open cost          \tnucl:5,aa:11\n","Gap extension cost     \tnucl:2,aa:1\n","Database output        \tfalse\n","Preload mode           \t0\n","Search type            \t0\n","Threads                \t4\n","Compressed             \t0\n","Verbosity              \t3\n","\n","[=================================================================] 100.00% 3.98K 0s 161ms\n","Time for merging to resultDB_secondary.m8: 0h 0m 0s 6ms\n","Time for processing: 0h 0m 0s 254ms\n","found 2381 genes\n","\n","\n","Reading fasta file GCF_000177635.2_ASM17763v2_genomic.fna.gz...\n","\n","Finding and translating open reading frames...\n","\n","Encoding amino acids...\n","\n","Applying kmer prefilter...\n","\n","Scoring ORFs with temporal convolutional network...\n","\n"],"name":"stdout"},{"output_type":"display_data","data":{"application/vnd.jupyter.widget-view+json":{"model_id":"d1d7312e6a8748c882ad3d46a918e629","version_minor":0,"version_major":2},"text/plain":["HBox(children=(FloatProgress(value=0.0, max=61.0), HTML(value='')))"]},"metadata":{"tags":[]}},{"output_type":"stream","text":["\n","Scoring translation initiation sites...\n","\n"],"name":"stdout"},{"output_type":"display_data","data":{"application/vnd.jupyter.widget-view+json":{"model_id":"6a8fc129d6344bab9c879b3e1cb9b199","version_minor":0,"version_major":2},"text/plain":["HBox(children=(FloatProgress(value=0.0, max=19.0), HTML(value='')))"]},"metadata":{"tags":[]}},{"output_type":"stream","text":["\n","Creating weighted directed acyclic graph of all ORFs and subORFs on contig 0...\n","\n"],"name":"stdout"},{"output_type":"display_data","data":{"application/vnd.jupyter.widget-view+json":{"model_id":"11c0335cfc7c4a249343656f5f9b9d39","version_minor":0,"version_major":2},"text/plain":["HBox(children=(FloatProgress(value=0.0, max=182262.0), HTML(value='')))"]},"metadata":{"tags":[]}},{"output_type":"stream","text":["\n","Maximizing geneiness...\n","\n","Filtering predictions with mmseqs2...\n","createdb /content/mmseqs2/tmp/35/candidate_genes_35.fasta /content/mmseqs2/tmp/35/candidateDB_35 \n","\n","MMseqs Version:       \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Database type         \t0\n","Shuffle input database\ttrue\n","Createdb mode         \t0\n","Write lookup file     \t1\n","Offset of numeric ids \t0\n","Compressed            \t0\n","Verbosity             \t3\n","\n","Converting sequences\n","[4445] 0s 6ms\n","Time for merging to candidateDB_35_h: 0h 0m 0s 4ms\n","Time for merging to candidateDB_35: 0h 0m 0s 4ms\n","Database type: Aminoacid\n","Time for processing: 0h 0m 0s 21ms\n","createdb /content/mmseqs2/tmp/35/candidate_genes_53.fasta /content/mmseqs2/tmp/35/candidateDB_53 \n","\n","MMseqs Version:       \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Database type         \t0\n","Shuffle input database\ttrue\n","Createdb mode         \t0\n","Write lookup file     \t1\n","Offset of numeric ids \t0\n","Compressed            \t0\n","Verbosity             \t3\n","\n","Converting sequences\n","[4445] 0s 7ms\n","Time for merging to candidateDB_53_h: 0h 0m 0s 3ms\n","Time for merging to candidateDB_53: 0h 0m 0s 4ms\n","Database type: Aminoacid\n","Time for processing: 0h 0m 0s 22ms\n","search -s 7.0 /content/mmseqs2/tmp/35/candidateDB_35 /content/mmseqs2/genexa/genexaDB /content/mmseqs2/tmp/35/resultsDB_35 /content/mmseqs2/tmp/35 \n","\n","MMseqs Version:                     \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Substitution matrix                 \tnucl:nucleotide.out,aa:blosum62.out\n","Add backtrace                       \tfalse\n","Alignment mode                      \t2\n","Allow wrapped scoring               \tfalse\n","E-value threshold                   \t0.001\n","Seq. id. threshold                  \t0\n","Min alignment length                \t0\n","Seq. id. mode                       \t0\n","Alternative alignments              \t0\n","Coverage threshold                  \t0\n","Coverage mode                       \t0\n","Max sequence length                 \t65535\n","Compositional bias                  \t1\n","Realign hits                        \tfalse\n","Max reject                          \t2147483647\n","Max accept                          \t2147483647\n","Include identical seq. id.          \tfalse\n","Preload mode                        \t0\n","Pseudo count a                      \t1\n","Pseudo count b                      \t1.5\n","Score bias                          \t0\n","Gap open cost                       \tnucl:5,aa:11\n","Gap extension cost                  \tnucl:2,aa:1\n","Zdrop                               \t40\n","Threads                             \t4\n","Compressed                          \t0\n","Verbosity                           \t3\n","Seed substitution matrix            \tnucl:nucleotide.out,aa:VTML80.out\n","Sensitivity                         \t7\n","k-mer length                        \t0\n","k-score                             \t2147483647\n","Alphabet size                       \tnucl:5,aa:21\n","Max results per query               \t300\n","Split database                      \t0\n","Split mode                          \t2\n","Split memory limit                  \t0\n","Diagonal scoring                    \ttrue\n","Exact k-mer matching                \t0\n","Mask residues                       \t1\n","Mask lower case residues            \t0\n","Minimum diagonal score              \t15\n","Spaced k-mers                       \t1\n","Spaced k-mer pattern                \t\n","Local temporary path                \t\n","Rescore mode                        \t0\n","Remove hits by seq. id. and coverage\tfalse\n","Sort results                        \t0\n","Mask profile                        \t1\n","Profile e-value threshold           \t0.1\n","Global sequence weighting           \tfalse\n","Allow deletions                     \tfalse\n","Filter MSA                          \t1\n","Maximum seq. id. threshold          \t0.9\n","Minimum seq. id.                    \t0\n","Minimum score per column            \t-20\n","Minimum coverage                    \t0\n","Select N most diverse seqs          \t1000\n","Omit consensus                      \tfalse\n","Min codons in orf                   \t30\n","Max codons in length                \t32734\n","Max orf gaps                        \t2147483647\n","Contig start mode                   \t2\n","Contig end mode                     \t2\n","Orf start mode                      \t1\n","Forward frames                      \t1,2,3\n","Reverse frames                      \t1,2,3\n","Translation table                   \t1\n","Translate orf                       \t0\n","Use all table starts                \tfalse\n","Offset of numeric ids               \t0\n","Create lookup                       \t0\n","Add orf stop                        \tfalse\n","Chain overlapping alignments        \t0\n","Merge query                         \t1\n","Search type                         \t0\n","Search iterations                   \t1\n","Start sensitivity                   \t4\n","Search steps                        \t1\n","Slice search mode                   \tfalse\n","Strand selection                    \t1\n","Disk space limit                    \t0\n","MPI runner                          \t\n","Force restart with latest tmp       \tfalse\n","Remove temporary files              \tfalse\n","\n","prefilter /content/mmseqs2/tmp/35/candidateDB_35 /content/mmseqs2/genexa/genexaDB.idx /content/mmseqs2/tmp/35/5309259870137382015/pref_0 --sub-mat nucl:nucleotide.out,aa:blosum62.out --seed-sub-mat nucl:nucleotide.out,aa:VTML80.out -k 0 --k-score 2147483647 --alph-size nucl:5,aa:21 --max-seq-len 65535 --max-seqs 300 --split 0 --split-mode 2 --split-memory-limit 0 -c 0 --cov-mode 0 --comp-bias-corr 1 --diag-score 1 --exact-kmer-matching 0 --mask 1 --mask-lower-case 0 --min-ungapped-score 15 --add-self-matches 0 --spaced-kmer-mode 1 --db-load-mode 0 --pca 1 --pcb 1.5 --threads 4 --compressed 0 -v 3 -s 7.0 \n","\n","Index version: 16\n","Generated by:  16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","ScoreMatrix:  VTML80.out\n","Query database size: 4495 type: Aminoacid\n","Estimated memory consumption: 2G\n","Target database size: 468351 type: Aminoacid\n","Process prefiltering step 1 of 1\n","\n","k-mer similarity threshold: 100\n","Starting prefiltering scores calculation (step 1 of 1)\n","Query db start 1 to 4495\n","Target db start 1 to 468351\n","[=================================================================] 100.00% 4.49K 33s 337ms\n","\n","1002.798133 k-mers per position\n","570180 DB matches per sequence\n","185 overflows\n","0 queries produce too many hits (truncated result)\n","278 sequences passed prefiltering per query sequence\n","300 median result list length\n","1 sequences with 0 size result lists\n","Time for merging to pref_0: 0h 0m 0s 1ms\n","Time for processing: 0h 0m 34s 755ms\n","align /content/mmseqs2/tmp/35/candidateDB_35 /content/mmseqs2/genexa/genexaDB.idx /content/mmseqs2/tmp/35/5309259870137382015/pref_0 /content/mmseqs2/tmp/35/resultsDB_35 --sub-mat nucl:nucleotide.out,aa:blosum62.out -a 0 --alignment-mode 2 --wrapped-scoring 0 -e 0.001 --min-seq-id 0 --min-aln-len 0 --seq-id-mode 0 --alt-ali 0 -c 0 --cov-mode 0 --max-seq-len 65535 --comp-bias-corr 1 --realign 0 --max-rejected 2147483647 --max-accept 2147483647 --add-self-matches 0 --db-load-mode 0 --pca 1 --pcb 1.5 --score-bias 0 --gap-open nucl:5,aa:11 --gap-extend nucl:2,aa:1 --zdrop 40 --threads 4 --compressed 0 -v 3 \n","\n","Index version: 16\n","Generated by:  16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","ScoreMatrix:  VTML80.out\n","Compute score and coverage\n","Query database size: 4495 type: Aminoacid\n","Target database size: 468351 type: Aminoacid\n","Calculation of alignments\n","[=================================================================] 100.00% 4.49K 28s 600ms\n","Time for merging to resultsDB_35: 0h 0m 0s 1ms\n","\n","1253724 alignments calculated.\n","313744 sequence pairs passed the thresholds (0.250250 of overall calculated).\n","69.798439 hits per query sequence.\n","Time for processing: 0h 0m 28s 632ms\n","convertalis /content/mmseqs2/tmp/35/candidateDB_35 /content/mmseqs2/genexa/genexaDB /content/mmseqs2/tmp/35/resultsDB_35 /content/mmseqs2/tmp/35/resultDB_genexa.m8 --format-output query,target,evalue,raw \n","\n","MMseqs Version:        \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Substitution matrix    \tnucl:nucleotide.out,aa:blosum62.out\n","Alignment format       \t0\n","Format alignment output\tquery,target,evalue,raw\n","Translation table      \t1\n","Gap open cost          \tnucl:5,aa:11\n","Gap extension cost     \tnucl:2,aa:1\n","Database output        \tfalse\n","Preload mode           \t0\n","Search type            \t0\n","Threads                \t4\n","Compressed             \t0\n","Verbosity              \t3\n","\n","[=================================================================] 100.00% 4.49K 0s 227ms\n","Time for merging to resultDB_genexa.m8: 0h 0m 0s 9ms\n","Time for processing: 0h 0m 0s 304ms\n","search -s 7.0 /content/mmseqs2/tmp/35/candidateDB_53 /content/mmseqs2/swissprot/swissprotDB /content/mmseqs2/tmp/35/resultsDB_53 /content/mmseqs2/tmp/35 \n","\n","MMseqs Version:                     \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Substitution matrix                 \tnucl:nucleotide.out,aa:blosum62.out\n","Add backtrace                       \tfalse\n","Alignment mode                      \t2\n","Allow wrapped scoring               \tfalse\n","E-value threshold                   \t0.001\n","Seq. id. threshold                  \t0\n","Min alignment length                \t0\n","Seq. id. mode                       \t0\n","Alternative alignments              \t0\n","Coverage threshold                  \t0\n","Coverage mode                       \t0\n","Max sequence length                 \t65535\n","Compositional bias                  \t1\n","Realign hits                        \tfalse\n","Max reject                          \t2147483647\n","Max accept                          \t2147483647\n","Include identical seq. id.          \tfalse\n","Preload mode                        \t0\n","Pseudo count a                      \t1\n","Pseudo count b                      \t1.5\n","Score bias                          \t0\n","Gap open cost                       \tnucl:5,aa:11\n","Gap extension cost                  \tnucl:2,aa:1\n","Zdrop                               \t40\n","Threads                             \t4\n","Compressed                          \t0\n","Verbosity                           \t3\n","Seed substitution matrix            \tnucl:nucleotide.out,aa:VTML80.out\n","Sensitivity                         \t7\n","k-mer length                        \t0\n","k-score                             \t2147483647\n","Alphabet size                       \tnucl:5,aa:21\n","Max results per query               \t300\n","Split database                      \t0\n","Split mode                          \t2\n","Split memory limit                  \t0\n","Diagonal scoring                    \ttrue\n","Exact k-mer matching                \t0\n","Mask residues                       \t1\n","Mask lower case residues            \t0\n","Minimum diagonal score              \t15\n","Spaced k-mers                       \t1\n","Spaced k-mer pattern                \t\n","Local temporary path                \t\n","Rescore mode                        \t0\n","Remove hits by seq. id. and coverage\tfalse\n","Sort results                        \t0\n","Mask profile                        \t1\n","Profile e-value threshold           \t0.1\n","Global sequence weighting           \tfalse\n","Allow deletions                     \tfalse\n","Filter MSA                          \t1\n","Maximum seq. id. threshold          \t0.9\n","Minimum seq. id.                    \t0\n","Minimum score per column            \t-20\n","Minimum coverage                    \t0\n","Select N most diverse seqs          \t1000\n","Omit consensus                      \tfalse\n","Min codons in orf                   \t30\n","Max codons in length                \t32734\n","Max orf gaps                        \t2147483647\n","Contig start mode                   \t2\n","Contig end mode                     \t2\n","Orf start mode                      \t1\n","Forward frames                      \t1,2,3\n","Reverse frames                      \t1,2,3\n","Translation table                   \t1\n","Translate orf                       \t0\n","Use all table starts                \tfalse\n","Offset of numeric ids               \t0\n","Create lookup                       \t0\n","Add orf stop                        \tfalse\n","Chain overlapping alignments        \t0\n","Merge query                         \t1\n","Search type                         \t0\n","Search iterations                   \t1\n","Start sensitivity                   \t4\n","Search steps                        \t1\n","Slice search mode                   \tfalse\n","Strand selection                    \t1\n","Disk space limit                    \t0\n","MPI runner                          \t\n","Force restart with latest tmp       \tfalse\n","Remove temporary files              \tfalse\n","\n","prefilter /content/mmseqs2/tmp/35/candidateDB_53 /content/mmseqs2/swissprot/swissprotDB /content/mmseqs2/tmp/35/11003677400614522845/pref_0 --sub-mat nucl:nucleotide.out,aa:blosum62.out --seed-sub-mat nucl:nucleotide.out,aa:VTML80.out -k 0 --k-score 2147483647 --alph-size nucl:5,aa:21 --max-seq-len 65535 --max-seqs 300 --split 0 --split-mode 2 --split-memory-limit 0 -c 0 --cov-mode 0 --comp-bias-corr 1 --diag-score 1 --exact-kmer-matching 0 --mask 1 --mask-lower-case 0 --min-ungapped-score 15 --add-self-matches 0 --spaced-kmer-mode 1 --db-load-mode 0 --pca 1 --pcb 1.5 --threads 4 --compressed 0 -v 3 -s 7.0 \n","\n","Query database size: 4495 type: Aminoacid\n","Estimated memory consumption: 2G\n","Target database size: 562755 type: Aminoacid\n","Index table k-mer threshold: 100 at k-mer size 6 \n","Index table: counting k-mers\n","[=================================================================] 100.00% 562.76K 23s 952ms\n","Index table: Masked residues: 3164874\n","Index table: fill\n","[=================================================================] 100.00% 562.76K 17s 715ms\n","Index statistics\n","Entries:          192757622\n","DB size:          1591 MB\n","Avg k-mer size:   3.011838\n","Top 10 k-mers\n","    GPGGTL\t1835\n","    GQSWTV\t1706\n","    WGMFAT\t1638\n","    PGVFEV\t1634\n","    VLWQFW\t1623\n","    AYIRPN\t1587\n","    RSPKGV\t1585\n","    TPHKWY\t1560\n","    HIRVGW\t1548\n","    ITLSPY\t1541\n","Time for index table init: 0h 0m 43s 719ms\n","Process prefiltering step 1 of 1\n","\n","k-mer similarity threshold: 100\n","Starting prefiltering scores calculation (step 1 of 1)\n","Query db start 1 to 4495\n","Target db start 1 to 562755\n","[=================================================================] 100.00% 4.49K 37s 242ms\n","\n","1002.359251 k-mers per position\n","684561 DB matches per sequence\n","297 overflows\n","0 queries produce too many hits (truncated result)\n","285 sequences passed prefiltering per query sequence\n","300 median result list length\n","1 sequences with 0 size result lists\n","Time for merging to pref_0: 0h 0m 0s 1ms\n","Time for processing: 0h 1m 22s 99ms\n","align /content/mmseqs2/tmp/35/candidateDB_53 /content/mmseqs2/swissprot/swissprotDB /content/mmseqs2/tmp/35/11003677400614522845/pref_0 /content/mmseqs2/tmp/35/resultsDB_53 --sub-mat nucl:nucleotide.out,aa:blosum62.out -a 0 --alignment-mode 2 --wrapped-scoring 0 -e 0.001 --min-seq-id 0 --min-aln-len 0 --seq-id-mode 0 --alt-ali 0 -c 0 --cov-mode 0 --max-seq-len 65535 --comp-bias-corr 1 --realign 0 --max-rejected 2147483647 --max-accept 2147483647 --add-self-matches 0 --db-load-mode 0 --pca 1 --pcb 1.5 --score-bias 0 --gap-open nucl:5,aa:11 --gap-extend nucl:2,aa:1 --zdrop 40 --threads 4 --compressed 0 -v 3 \n","\n","Compute score and coverage\n","Query database size: 4495 type: Aminoacid\n","Target database size: 562755 type: Aminoacid\n","Calculation of alignments\n","[=================================================================] 100.00% 4.49K 27s 112ms\n","Time for merging to resultsDB_53: 0h 0m 0s 1ms\n","\n","1283100 alignments calculated.\n","238585 sequence pairs passed the thresholds (0.185944 of overall calculated).\n","53.077866 hits per query sequence.\n","Time for processing: 0h 0m 27s 173ms\n","convertalis /content/mmseqs2/tmp/35/candidateDB_53 /content/mmseqs2/swissprot/swissprotDB /content/mmseqs2/tmp/35/resultsDB_53 /content/mmseqs2/tmp/35/resultDB_secondary.m8 --format-output query,target,evalue,raw \n","\n","MMseqs Version:        \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Substitution matrix    \tnucl:nucleotide.out,aa:blosum62.out\n","Alignment format       \t0\n","Format alignment output\tquery,target,evalue,raw\n","Translation table      \t1\n","Gap open cost          \tnucl:5,aa:11\n","Gap extension cost     \tnucl:2,aa:1\n","Database output        \tfalse\n","Preload mode           \t0\n","Search type            \t0\n","Threads                \t4\n","Compressed             \t0\n","Verbosity              \t3\n","\n","[=================================================================] 100.00% 4.49K 0s 163ms\n","Time for merging to resultDB_secondary.m8: 0h 0m 0s 7ms\n","Time for processing: 0h 0m 0s 258ms\n","found 2634 genes\n","\n","\n","Reading fasta file GCF_000147695.2_ASM14769v3_genomic.fna.gz...\n","\n","Finding and translating open reading frames...\n","\n","Encoding amino acids...\n","\n","Applying kmer prefilter...\n","\n","Scoring ORFs with temporal convolutional network...\n","\n"],"name":"stdout"},{"output_type":"display_data","data":{"application/vnd.jupyter.widget-view+json":{"model_id":"6b905dab19c24646ba210604f80e34e7","version_minor":0,"version_major":2},"text/plain":["HBox(children=(FloatProgress(value=0.0, max=32.0), HTML(value='')))"]},"metadata":{"tags":[]}},{"output_type":"stream","text":["\n","Scoring translation initiation sites...\n","\n"],"name":"stdout"},{"output_type":"display_data","data":{"application/vnd.jupyter.widget-view+json":{"model_id":"3b97a4d2484f4e75849d7bb799ce2367","version_minor":0,"version_major":2},"text/plain":["HBox(children=(FloatProgress(value=0.0, max=10.0), HTML(value='')))"]},"metadata":{"tags":[]}},{"output_type":"stream","text":["\n","Creating weighted directed acyclic graph of all ORFs and subORFs on contig 0...\n","\n"],"name":"stdout"},{"output_type":"display_data","data":{"application/vnd.jupyter.widget-view+json":{"model_id":"1c669e42110245e8a4da4bd82320bb12","version_minor":0,"version_major":2},"text/plain":["HBox(children=(FloatProgress(value=0.0, max=94855.0), HTML(value='')))"]},"metadata":{"tags":[]}},{"output_type":"stream","text":["\n","Maximizing geneiness...\n","\n","Filtering predictions with mmseqs2...\n","createdb /content/mmseqs2/tmp/36/candidate_genes_35.fasta /content/mmseqs2/tmp/36/candidateDB_35 \n","\n","MMseqs Version:       \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Database type         \t0\n","Shuffle input database\ttrue\n","Createdb mode         \t0\n","Write lookup file     \t1\n","Offset of numeric ids \t0\n","Compressed            \t0\n","Verbosity             \t3\n","\n","Converting sequences\n","[4748] 0s 6ms\n","Time for merging to candidateDB_35_h: 0h 0m 0s 5ms\n","Time for merging to candidateDB_35: 0h 0m 0s 4ms\n","Database type: Aminoacid\n","Time for processing: 0h 0m 0s 26ms\n","createdb /content/mmseqs2/tmp/36/candidate_genes_53.fasta /content/mmseqs2/tmp/36/candidateDB_53 \n","\n","MMseqs Version:       \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Database type         \t0\n","Shuffle input database\ttrue\n","Createdb mode         \t0\n","Write lookup file     \t1\n","Offset of numeric ids \t0\n","Compressed            \t0\n","Verbosity             \t3\n","\n","Converting sequences\n","[4748] 0s 6ms\n","Time for merging to candidateDB_53_h: 0h 0m 0s 3ms\n","Time for merging to candidateDB_53: 0h 0m 0s 4ms\n","Database type: Aminoacid\n","Time for processing: 0h 0m 0s 20ms\n","search -s 7.0 /content/mmseqs2/tmp/36/candidateDB_35 /content/mmseqs2/genexa/genexaDB /content/mmseqs2/tmp/36/resultsDB_35 /content/mmseqs2/tmp/36 \n","\n","MMseqs Version:                     \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Substitution matrix                 \tnucl:nucleotide.out,aa:blosum62.out\n","Add backtrace                       \tfalse\n","Alignment mode                      \t2\n","Allow wrapped scoring               \tfalse\n","E-value threshold                   \t0.001\n","Seq. id. threshold                  \t0\n","Min alignment length                \t0\n","Seq. id. mode                       \t0\n","Alternative alignments              \t0\n","Coverage threshold                  \t0\n","Coverage mode                       \t0\n","Max sequence length                 \t65535\n","Compositional bias                  \t1\n","Realign hits                        \tfalse\n","Max reject                          \t2147483647\n","Max accept                          \t2147483647\n","Include identical seq. id.          \tfalse\n","Preload mode                        \t0\n","Pseudo count a                      \t1\n","Pseudo count b                      \t1.5\n","Score bias                          \t0\n","Gap open cost                       \tnucl:5,aa:11\n","Gap extension cost                  \tnucl:2,aa:1\n","Zdrop                               \t40\n","Threads                             \t4\n","Compressed                          \t0\n","Verbosity                           \t3\n","Seed substitution matrix            \tnucl:nucleotide.out,aa:VTML80.out\n","Sensitivity                         \t7\n","k-mer length                        \t0\n","k-score                             \t2147483647\n","Alphabet size                       \tnucl:5,aa:21\n","Max results per query               \t300\n","Split database                      \t0\n","Split mode                          \t2\n","Split memory limit                  \t0\n","Diagonal scoring                    \ttrue\n","Exact k-mer matching                \t0\n","Mask residues                       \t1\n","Mask lower case residues            \t0\n","Minimum diagonal score              \t15\n","Spaced k-mers                       \t1\n","Spaced k-mer pattern                \t\n","Local temporary path                \t\n","Rescore mode                        \t0\n","Remove hits by seq. id. and coverage\tfalse\n","Sort results                        \t0\n","Mask profile                        \t1\n","Profile e-value threshold           \t0.1\n","Global sequence weighting           \tfalse\n","Allow deletions                     \tfalse\n","Filter MSA                          \t1\n","Maximum seq. id. threshold          \t0.9\n","Minimum seq. id.                    \t0\n","Minimum score per column            \t-20\n","Minimum coverage                    \t0\n","Select N most diverse seqs          \t1000\n","Omit consensus                      \tfalse\n","Min codons in orf                   \t30\n","Max codons in length                \t32734\n","Max orf gaps                        \t2147483647\n","Contig start mode                   \t2\n","Contig end mode                     \t2\n","Orf start mode                      \t1\n","Forward frames                      \t1,2,3\n","Reverse frames                      \t1,2,3\n","Translation table                   \t1\n","Translate orf                       \t0\n","Use all table starts                \tfalse\n","Offset of numeric ids               \t0\n","Create lookup                       \t0\n","Add orf stop                        \tfalse\n","Chain overlapping alignments        \t0\n","Merge query                         \t1\n","Search type                         \t0\n","Search iterations                   \t1\n","Start sensitivity                   \t4\n","Search steps                        \t1\n","Slice search mode                   \tfalse\n","Strand selection                    \t1\n","Disk space limit                    \t0\n","MPI runner                          \t\n","Force restart with latest tmp       \tfalse\n","Remove temporary files              \tfalse\n","\n","prefilter /content/mmseqs2/tmp/36/candidateDB_35 /content/mmseqs2/genexa/genexaDB.idx /content/mmseqs2/tmp/36/48973767027686526/pref_0 --sub-mat nucl:nucleotide.out,aa:blosum62.out --seed-sub-mat nucl:nucleotide.out,aa:VTML80.out -k 0 --k-score 2147483647 --alph-size nucl:5,aa:21 --max-seq-len 65535 --max-seqs 300 --split 0 --split-mode 2 --split-memory-limit 0 -c 0 --cov-mode 0 --comp-bias-corr 1 --diag-score 1 --exact-kmer-matching 0 --mask 1 --mask-lower-case 0 --min-ungapped-score 15 --add-self-matches 0 --spaced-kmer-mode 1 --db-load-mode 0 --pca 1 --pcb 1.5 --threads 4 --compressed 0 -v 3 -s 7.0 \n","\n","Index version: 16\n","Generated by:  16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","ScoreMatrix:  VTML80.out\n","Query database size: 4758 type: Aminoacid\n","Estimated memory consumption: 2G\n","Target database size: 468351 type: Aminoacid\n","Process prefiltering step 1 of 1\n","\n","k-mer similarity threshold: 100\n","Starting prefiltering scores calculation (step 1 of 1)\n","Query db start 1 to 4758\n","Target db start 1 to 468351\n","[=================================================================] 100.00% 4.76K 29s 101ms\n","\n","837.441184 k-mers per position\n","475488 DB matches per sequence\n","98 overflows\n","0 queries produce too many hits (truncated result)\n","258 sequences passed prefiltering per query sequence\n","300 median result list length\n","7 sequences with 0 size result lists\n","Time for merging to pref_0: 0h 0m 0s 1ms\n","Time for processing: 0h 0m 30s 519ms\n","align /content/mmseqs2/tmp/36/candidateDB_35 /content/mmseqs2/genexa/genexaDB.idx /content/mmseqs2/tmp/36/48973767027686526/pref_0 /content/mmseqs2/tmp/36/resultsDB_35 --sub-mat nucl:nucleotide.out,aa:blosum62.out -a 0 --alignment-mode 2 --wrapped-scoring 0 -e 0.001 --min-seq-id 0 --min-aln-len 0 --seq-id-mode 0 --alt-ali 0 -c 0 --cov-mode 0 --max-seq-len 65535 --comp-bias-corr 1 --realign 0 --max-rejected 2147483647 --max-accept 2147483647 --add-self-matches 0 --db-load-mode 0 --pca 1 --pcb 1.5 --score-bias 0 --gap-open nucl:5,aa:11 --gap-extend nucl:2,aa:1 --zdrop 40 --threads 4 --compressed 0 -v 3 \n","\n","Index version: 16\n","Generated by:  16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","ScoreMatrix:  VTML80.out\n","Compute score and coverage\n","Query database size: 4758 type: Aminoacid\n","Target database size: 468351 type: Aminoacid\n","Calculation of alignments\n","[=================================================================] 100.00% 4.76K 26s 801ms\n","Time for merging to resultsDB_35: 0h 0m 0s 1ms\n","\n","1230869 alignments calculated.\n","330729 sequence pairs passed the thresholds (0.268696 of overall calculated).\n","69.510086 hits per query sequence.\n","Time for processing: 0h 0m 26s 840ms\n","convertalis /content/mmseqs2/tmp/36/candidateDB_35 /content/mmseqs2/genexa/genexaDB /content/mmseqs2/tmp/36/resultsDB_35 /content/mmseqs2/tmp/36/resultDB_genexa.m8 --format-output query,target,evalue,raw \n","\n","MMseqs Version:        \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Substitution matrix    \tnucl:nucleotide.out,aa:blosum62.out\n","Alignment format       \t0\n","Format alignment output\tquery,target,evalue,raw\n","Translation table      \t1\n","Gap open cost          \tnucl:5,aa:11\n","Gap extension cost     \tnucl:2,aa:1\n","Database output        \tfalse\n","Preload mode           \t0\n","Search type            \t0\n","Threads                \t4\n","Compressed             \t0\n","Verbosity              \t3\n","\n","[=================================================================] 100.00% 4.76K 0s 240ms\n","Time for merging to resultDB_genexa.m8: 0h 0m 0s 9ms\n","Time for processing: 0h 0m 0s 323ms\n","search -s 7.0 /content/mmseqs2/tmp/36/candidateDB_53 /content/mmseqs2/swissprot/swissprotDB /content/mmseqs2/tmp/36/resultsDB_53 /content/mmseqs2/tmp/36 \n","\n","MMseqs Version:                     \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Substitution matrix                 \tnucl:nucleotide.out,aa:blosum62.out\n","Add backtrace                       \tfalse\n","Alignment mode                      \t2\n","Allow wrapped scoring               \tfalse\n","E-value threshold                   \t0.001\n","Seq. id. threshold                  \t0\n","Min alignment length                \t0\n","Seq. id. mode                       \t0\n","Alternative alignments              \t0\n","Coverage threshold                  \t0\n","Coverage mode                       \t0\n","Max sequence length                 \t65535\n","Compositional bias                  \t1\n","Realign hits                        \tfalse\n","Max reject                          \t2147483647\n","Max accept                          \t2147483647\n","Include identical seq. id.          \tfalse\n","Preload mode                        \t0\n","Pseudo count a                      \t1\n","Pseudo count b                      \t1.5\n","Score bias                          \t0\n","Gap open cost                       \tnucl:5,aa:11\n","Gap extension cost                  \tnucl:2,aa:1\n","Zdrop                               \t40\n","Threads                             \t4\n","Compressed                          \t0\n","Verbosity                           \t3\n","Seed substitution matrix            \tnucl:nucleotide.out,aa:VTML80.out\n","Sensitivity                         \t7\n","k-mer length                        \t0\n","k-score                             \t2147483647\n","Alphabet size                       \tnucl:5,aa:21\n","Max results per query               \t300\n","Split database                      \t0\n","Split mode                          \t2\n","Split memory limit                  \t0\n","Diagonal scoring                    \ttrue\n","Exact k-mer matching                \t0\n","Mask residues                       \t1\n","Mask lower case residues            \t0\n","Minimum diagonal score              \t15\n","Spaced k-mers                       \t1\n","Spaced k-mer pattern                \t\n","Local temporary path                \t\n","Rescore mode                        \t0\n","Remove hits by seq. id. and coverage\tfalse\n","Sort results                        \t0\n","Mask profile                        \t1\n","Profile e-value threshold           \t0.1\n","Global sequence weighting           \tfalse\n","Allow deletions                     \tfalse\n","Filter MSA                          \t1\n","Maximum seq. id. threshold          \t0.9\n","Minimum seq. id.                    \t0\n","Minimum score per column            \t-20\n","Minimum coverage                    \t0\n","Select N most diverse seqs          \t1000\n","Omit consensus                      \tfalse\n","Min codons in orf                   \t30\n","Max codons in length                \t32734\n","Max orf gaps                        \t2147483647\n","Contig start mode                   \t2\n","Contig end mode                     \t2\n","Orf start mode                      \t1\n","Forward frames                      \t1,2,3\n","Reverse frames                      \t1,2,3\n","Translation table                   \t1\n","Translate orf                       \t0\n","Use all table starts                \tfalse\n","Offset of numeric ids               \t0\n","Create lookup                       \t0\n","Add orf stop                        \tfalse\n","Chain overlapping alignments        \t0\n","Merge query                         \t1\n","Search type                         \t0\n","Search iterations                   \t1\n","Start sensitivity                   \t4\n","Search steps                        \t1\n","Slice search mode                   \tfalse\n","Strand selection                    \t1\n","Disk space limit                    \t0\n","MPI runner                          \t\n","Force restart with latest tmp       \tfalse\n","Remove temporary files              \tfalse\n","\n","prefilter /content/mmseqs2/tmp/36/candidateDB_53 /content/mmseqs2/swissprot/swissprotDB /content/mmseqs2/tmp/36/11105042388659868828/pref_0 --sub-mat nucl:nucleotide.out,aa:blosum62.out --seed-sub-mat nucl:nucleotide.out,aa:VTML80.out -k 0 --k-score 2147483647 --alph-size nucl:5,aa:21 --max-seq-len 65535 --max-seqs 300 --split 0 --split-mode 2 --split-memory-limit 0 -c 0 --cov-mode 0 --comp-bias-corr 1 --diag-score 1 --exact-kmer-matching 0 --mask 1 --mask-lower-case 0 --min-ungapped-score 15 --add-self-matches 0 --spaced-kmer-mode 1 --db-load-mode 0 --pca 1 --pcb 1.5 --threads 4 --compressed 0 -v 3 -s 7.0 \n","\n","Query database size: 4758 type: Aminoacid\n","Estimated memory consumption: 2G\n","Target database size: 562755 type: Aminoacid\n","Index table k-mer threshold: 100 at k-mer size 6 \n","Index table: counting k-mers\n","[=================================================================] 100.00% 562.76K 24s 50ms\n","Index table: Masked residues: 3164874\n","Index table: fill\n","[=================================================================] 100.00% 562.76K 17s 811ms\n","Index statistics\n","Entries:          192757622\n","DB size:          1591 MB\n","Avg k-mer size:   3.011838\n","Top 10 k-mers\n","    GPGGTL\t1835\n","    GQSWTV\t1706\n","    WGMFAT\t1638\n","    PGVFEV\t1634\n","    VLWQFW\t1623\n","    AYIRPN\t1587\n","    RSPKGV\t1585\n","    TPHKWY\t1560\n","    HIRVGW\t1548\n","    ITLSPY\t1541\n","Time for index table init: 0h 0m 43s 939ms\n","Process prefiltering step 1 of 1\n","\n","k-mer similarity threshold: 100\n","Starting prefiltering scores calculation (step 1 of 1)\n","Query db start 1 to 4758\n","Target db start 1 to 562755\n","[=================================================================] 100.00% 4.76K 33s 340ms\n","\n","837.662222 k-mers per position\n","600236 DB matches per sequence\n","210 overflows\n","0 queries produce too many hits (truncated result)\n","270 sequences passed prefiltering per query sequence\n","300 median result list length\n","3 sequences with 0 size result lists\n","Time for merging to pref_0: 0h 0m 0s 1ms\n","Time for processing: 0h 1m 18s 429ms\n","align /content/mmseqs2/tmp/36/candidateDB_53 /content/mmseqs2/swissprot/swissprotDB /content/mmseqs2/tmp/36/11105042388659868828/pref_0 /content/mmseqs2/tmp/36/resultsDB_53 --sub-mat nucl:nucleotide.out,aa:blosum62.out -a 0 --alignment-mode 2 --wrapped-scoring 0 -e 0.001 --min-seq-id 0 --min-aln-len 0 --seq-id-mode 0 --alt-ali 0 -c 0 --cov-mode 0 --max-seq-len 65535 --comp-bias-corr 1 --realign 0 --max-rejected 2147483647 --max-accept 2147483647 --add-self-matches 0 --db-load-mode 0 --pca 1 --pcb 1.5 --score-bias 0 --gap-open nucl:5,aa:11 --gap-extend nucl:2,aa:1 --zdrop 40 --threads 4 --compressed 0 -v 3 \n","\n","Compute score and coverage\n","Query database size: 4758 type: Aminoacid\n","Target database size: 562755 type: Aminoacid\n","Calculation of alignments\n","[=================================================================] 100.00% 4.76K 26s 126ms\n","Time for merging to resultsDB_53: 0h 0m 0s 1ms\n","\n","1285198 alignments calculated.\n","249223 sequence pairs passed the thresholds (0.193918 of overall calculated).\n","52.379780 hits per query sequence.\n","Time for processing: 0h 0m 26s 191ms\n","convertalis /content/mmseqs2/tmp/36/candidateDB_53 /content/mmseqs2/swissprot/swissprotDB /content/mmseqs2/tmp/36/resultsDB_53 /content/mmseqs2/tmp/36/resultDB_secondary.m8 --format-output query,target,evalue,raw \n","\n","MMseqs Version:        \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Substitution matrix    \tnucl:nucleotide.out,aa:blosum62.out\n","Alignment format       \t0\n","Format alignment output\tquery,target,evalue,raw\n","Translation table      \t1\n","Gap open cost          \tnucl:5,aa:11\n","Gap extension cost     \tnucl:2,aa:1\n","Database output        \tfalse\n","Preload mode           \t0\n","Search type            \t0\n","Threads                \t4\n","Compressed             \t0\n","Verbosity              \t3\n","\n","[=================================================================] 100.00% 4.76K 0s 170ms\n","Time for merging to resultDB_secondary.m8: 0h 0m 0s 7ms\n","Time for processing: 0h 0m 0s 266ms\n","found 2778 genes\n","\n","\n","Reading fasta file GCF_000144645.1_ASM14464v1_genomic.fna.gz...\n","\n","Finding and translating open reading frames...\n","\n","Encoding amino acids...\n","\n","Applying kmer prefilter...\n","\n","Scoring ORFs with temporal convolutional network...\n","\n"],"name":"stdout"},{"output_type":"display_data","data":{"application/vnd.jupyter.widget-view+json":{"model_id":"96df2e0cbe3c4bf690bde1aaab48fb07","version_minor":0,"version_major":2},"text/plain":["HBox(children=(FloatProgress(value=0.0, max=32.0), HTML(value='')))"]},"metadata":{"tags":[]}},{"output_type":"stream","text":["\n","Scoring translation initiation sites...\n","\n"],"name":"stdout"},{"output_type":"display_data","data":{"application/vnd.jupyter.widget-view+json":{"model_id":"5e5b167cdae742b383abd1d5ce530e8e","version_minor":0,"version_major":2},"text/plain":["HBox(children=(FloatProgress(value=0.0, max=10.0), HTML(value='')))"]},"metadata":{"tags":[]}},{"output_type":"stream","text":["\n","Creating weighted directed acyclic graph of all ORFs and subORFs on contig 0...\n","\n"],"name":"stdout"},{"output_type":"display_data","data":{"application/vnd.jupyter.widget-view+json":{"model_id":"2f8725285e434f4e973d69cd223bdd71","version_minor":0,"version_major":2},"text/plain":["HBox(children=(FloatProgress(value=0.0, max=95683.0), HTML(value='')))"]},"metadata":{"tags":[]}},{"output_type":"stream","text":["\n","Maximizing geneiness...\n","\n","Filtering predictions with mmseqs2...\n","createdb /content/mmseqs2/tmp/37/candidate_genes_35.fasta /content/mmseqs2/tmp/37/candidateDB_35 \n","\n","MMseqs Version:       \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Database type         \t0\n","Shuffle input database\ttrue\n","Createdb mode         \t0\n","Write lookup file     \t1\n","Offset of numeric ids \t0\n","Compressed            \t0\n","Verbosity             \t3\n","\n","Converting sequences\n","[3637] 0s 5ms\n","Time for merging to candidateDB_35_h: 0h 0m 0s 4ms\n","Time for merging to candidateDB_35: 0h 0m 0s 4ms\n","Database type: Aminoacid\n","Time for processing: 0h 0m 0s 19ms\n","createdb /content/mmseqs2/tmp/37/candidate_genes_53.fasta /content/mmseqs2/tmp/37/candidateDB_53 \n","\n","MMseqs Version:       \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Database type         \t0\n","Shuffle input database\ttrue\n","Createdb mode         \t0\n","Write lookup file     \t1\n","Offset of numeric ids \t0\n","Compressed            \t0\n","Verbosity             \t3\n","\n","Converting sequences\n","[3637] 0s 5ms\n","Time for merging to candidateDB_53_h: 0h 0m 0s 4ms\n","Time for merging to candidateDB_53: 0h 0m 0s 3ms\n","Database type: Aminoacid\n","Time for processing: 0h 0m 0s 19ms\n","search -s 7.0 /content/mmseqs2/tmp/37/candidateDB_35 /content/mmseqs2/genexa/genexaDB /content/mmseqs2/tmp/37/resultsDB_35 /content/mmseqs2/tmp/37 \n","\n","MMseqs Version:                     \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Substitution matrix                 \tnucl:nucleotide.out,aa:blosum62.out\n","Add backtrace                       \tfalse\n","Alignment mode                      \t2\n","Allow wrapped scoring               \tfalse\n","E-value threshold                   \t0.001\n","Seq. id. threshold                  \t0\n","Min alignment length                \t0\n","Seq. id. mode                       \t0\n","Alternative alignments              \t0\n","Coverage threshold                  \t0\n","Coverage mode                       \t0\n","Max sequence length                 \t65535\n","Compositional bias                  \t1\n","Realign hits                        \tfalse\n","Max reject                          \t2147483647\n","Max accept                          \t2147483647\n","Include identical seq. id.          \tfalse\n","Preload mode                        \t0\n","Pseudo count a                      \t1\n","Pseudo count b                      \t1.5\n","Score bias                          \t0\n","Gap open cost                       \tnucl:5,aa:11\n","Gap extension cost                  \tnucl:2,aa:1\n","Zdrop                               \t40\n","Threads                             \t4\n","Compressed                          \t0\n","Verbosity                           \t3\n","Seed substitution matrix            \tnucl:nucleotide.out,aa:VTML80.out\n","Sensitivity                         \t7\n","k-mer length                        \t0\n","k-score                             \t2147483647\n","Alphabet size                       \tnucl:5,aa:21\n","Max results per query               \t300\n","Split database                      \t0\n","Split mode                          \t2\n","Split memory limit                  \t0\n","Diagonal scoring                    \ttrue\n","Exact k-mer matching                \t0\n","Mask residues                       \t1\n","Mask lower case residues            \t0\n","Minimum diagonal score              \t15\n","Spaced k-mers                       \t1\n","Spaced k-mer pattern                \t\n","Local temporary path                \t\n","Rescore mode                        \t0\n","Remove hits by seq. id. and coverage\tfalse\n","Sort results                        \t0\n","Mask profile                        \t1\n","Profile e-value threshold           \t0.1\n","Global sequence weighting           \tfalse\n","Allow deletions                     \tfalse\n","Filter MSA                          \t1\n","Maximum seq. id. threshold          \t0.9\n","Minimum seq. id.                    \t0\n","Minimum score per column            \t-20\n","Minimum coverage                    \t0\n","Select N most diverse seqs          \t1000\n","Omit consensus                      \tfalse\n","Min codons in orf                   \t30\n","Max codons in length                \t32734\n","Max orf gaps                        \t2147483647\n","Contig start mode                   \t2\n","Contig end mode                     \t2\n","Orf start mode                      \t1\n","Forward frames                      \t1,2,3\n","Reverse frames                      \t1,2,3\n","Translation table                   \t1\n","Translate orf                       \t0\n","Use all table starts                \tfalse\n","Offset of numeric ids               \t0\n","Create lookup                       \t0\n","Add orf stop                        \tfalse\n","Chain overlapping alignments        \t0\n","Merge query                         \t1\n","Search type                         \t0\n","Search iterations                   \t1\n","Start sensitivity                   \t4\n","Search steps                        \t1\n","Slice search mode                   \tfalse\n","Strand selection                    \t1\n","Disk space limit                    \t0\n","MPI runner                          \t\n","Force restart with latest tmp       \tfalse\n","Remove temporary files              \tfalse\n","\n","prefilter /content/mmseqs2/tmp/37/candidateDB_35 /content/mmseqs2/genexa/genexaDB.idx /content/mmseqs2/tmp/37/13235431737627542653/pref_0 --sub-mat nucl:nucleotide.out,aa:blosum62.out --seed-sub-mat nucl:nucleotide.out,aa:VTML80.out -k 0 --k-score 2147483647 --alph-size nucl:5,aa:21 --max-seq-len 65535 --max-seqs 300 --split 0 --split-mode 2 --split-memory-limit 0 -c 0 --cov-mode 0 --comp-bias-corr 1 --diag-score 1 --exact-kmer-matching 0 --mask 1 --mask-lower-case 0 --min-ungapped-score 15 --add-self-matches 0 --spaced-kmer-mode 1 --db-load-mode 0 --pca 1 --pcb 1.5 --threads 4 --compressed 0 -v 3 -s 7.0 \n","\n","Index version: 16\n","Generated by:  16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","ScoreMatrix:  VTML80.out\n","Query database size: 3737 type: Aminoacid\n","Estimated memory consumption: 2G\n","Target database size: 468351 type: Aminoacid\n","Process prefiltering step 1 of 1\n","\n","k-mer similarity threshold: 100\n","Starting prefiltering scores calculation (step 1 of 1)\n","Query db start 1 to 3737\n","Target db start 1 to 468351\n","[=================================================================] 100.00% 3.74K 25s 724ms\n","\n","917.319852 k-mers per position\n","549209 DB matches per sequence\n","109 overflows\n","0 queries produce too many hits (truncated result)\n","278 sequences passed prefiltering per query sequence\n","300 median result list length\n","3 sequences with 0 size result lists\n","Time for merging to pref_0: 0h 0m 0s 1ms\n","Time for processing: 0h 0m 27s 147ms\n","align /content/mmseqs2/tmp/37/candidateDB_35 /content/mmseqs2/genexa/genexaDB.idx /content/mmseqs2/tmp/37/13235431737627542653/pref_0 /content/mmseqs2/tmp/37/resultsDB_35 --sub-mat nucl:nucleotide.out,aa:blosum62.out -a 0 --alignment-mode 2 --wrapped-scoring 0 -e 0.001 --min-seq-id 0 --min-aln-len 0 --seq-id-mode 0 --alt-ali 0 -c 0 --cov-mode 0 --max-seq-len 65535 --comp-bias-corr 1 --realign 0 --max-rejected 2147483647 --max-accept 2147483647 --add-self-matches 0 --db-load-mode 0 --pca 1 --pcb 1.5 --score-bias 0 --gap-open nucl:5,aa:11 --gap-extend nucl:2,aa:1 --zdrop 40 --threads 4 --compressed 0 -v 3 \n","\n","Index version: 16\n","Generated by:  16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","ScoreMatrix:  VTML80.out\n","Compute score and coverage\n","Query database size: 3737 type: Aminoacid\n","Target database size: 468351 type: Aminoacid\n","Calculation of alignments\n","[=================================================================] 100.00% 3.74K 22s 519ms\n","Time for merging to resultsDB_35: 0h 0m 0s 1ms\n","\n","1038909 alignments calculated.\n","264280 sequence pairs passed the thresholds (0.254382 of overall calculated).\n","70.719826 hits per query sequence.\n","Time for processing: 0h 0m 22s 547ms\n","convertalis /content/mmseqs2/tmp/37/candidateDB_35 /content/mmseqs2/genexa/genexaDB /content/mmseqs2/tmp/37/resultsDB_35 /content/mmseqs2/tmp/37/resultDB_genexa.m8 --format-output query,target,evalue,raw \n","\n","MMseqs Version:        \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Substitution matrix    \tnucl:nucleotide.out,aa:blosum62.out\n","Alignment format       \t0\n","Format alignment output\tquery,target,evalue,raw\n","Translation table      \t1\n","Gap open cost          \tnucl:5,aa:11\n","Gap extension cost     \tnucl:2,aa:1\n","Database output        \tfalse\n","Preload mode           \t0\n","Search type            \t0\n","Threads                \t4\n","Compressed             \t0\n","Verbosity              \t3\n","\n","[=================================================================] 100.00% 3.74K 0s 196ms\n","Time for merging to resultDB_genexa.m8: 0h 0m 0s 9ms\n","Time for processing: 0h 0m 0s 276ms\n","search -s 7.0 /content/mmseqs2/tmp/37/candidateDB_53 /content/mmseqs2/swissprot/swissprotDB /content/mmseqs2/tmp/37/resultsDB_53 /content/mmseqs2/tmp/37 \n","\n","MMseqs Version:                     \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Substitution matrix                 \tnucl:nucleotide.out,aa:blosum62.out\n","Add backtrace                       \tfalse\n","Alignment mode                      \t2\n","Allow wrapped scoring               \tfalse\n","E-value threshold                   \t0.001\n","Seq. id. threshold                  \t0\n","Min alignment length                \t0\n","Seq. id. mode                       \t0\n","Alternative alignments              \t0\n","Coverage threshold                  \t0\n","Coverage mode                       \t0\n","Max sequence length                 \t65535\n","Compositional bias                  \t1\n","Realign hits                        \tfalse\n","Max reject                          \t2147483647\n","Max accept                          \t2147483647\n","Include identical seq. id.          \tfalse\n","Preload mode                        \t0\n","Pseudo count a                      \t1\n","Pseudo count b                      \t1.5\n","Score bias                          \t0\n","Gap open cost                       \tnucl:5,aa:11\n","Gap extension cost                  \tnucl:2,aa:1\n","Zdrop                               \t40\n","Threads                             \t4\n","Compressed                          \t0\n","Verbosity                           \t3\n","Seed substitution matrix            \tnucl:nucleotide.out,aa:VTML80.out\n","Sensitivity                         \t7\n","k-mer length                        \t0\n","k-score                             \t2147483647\n","Alphabet size                       \tnucl:5,aa:21\n","Max results per query               \t300\n","Split database                      \t0\n","Split mode                          \t2\n","Split memory limit                  \t0\n","Diagonal scoring                    \ttrue\n","Exact k-mer matching                \t0\n","Mask residues                       \t1\n","Mask lower case residues            \t0\n","Minimum diagonal score              \t15\n","Spaced k-mers                       \t1\n","Spaced k-mer pattern                \t\n","Local temporary path                \t\n","Rescore mode                        \t0\n","Remove hits by seq. id. and coverage\tfalse\n","Sort results                        \t0\n","Mask profile                        \t1\n","Profile e-value threshold           \t0.1\n","Global sequence weighting           \tfalse\n","Allow deletions                     \tfalse\n","Filter MSA                          \t1\n","Maximum seq. id. threshold          \t0.9\n","Minimum seq. id.                    \t0\n","Minimum score per column            \t-20\n","Minimum coverage                    \t0\n","Select N most diverse seqs          \t1000\n","Omit consensus                      \tfalse\n","Min codons in orf                   \t30\n","Max codons in length                \t32734\n","Max orf gaps                        \t2147483647\n","Contig start mode                   \t2\n","Contig end mode                     \t2\n","Orf start mode                      \t1\n","Forward frames                      \t1,2,3\n","Reverse frames                      \t1,2,3\n","Translation table                   \t1\n","Translate orf                       \t0\n","Use all table starts                \tfalse\n","Offset of numeric ids               \t0\n","Create lookup                       \t0\n","Add orf stop                        \tfalse\n","Chain overlapping alignments        \t0\n","Merge query                         \t1\n","Search type                         \t0\n","Search iterations                   \t1\n","Start sensitivity                   \t4\n","Search steps                        \t1\n","Slice search mode                   \tfalse\n","Strand selection                    \t1\n","Disk space limit                    \t0\n","MPI runner                          \t\n","Force restart with latest tmp       \tfalse\n","Remove temporary files              \tfalse\n","\n","prefilter /content/mmseqs2/tmp/37/candidateDB_53 /content/mmseqs2/swissprot/swissprotDB /content/mmseqs2/tmp/37/11206407376705214811/pref_0 --sub-mat nucl:nucleotide.out,aa:blosum62.out --seed-sub-mat nucl:nucleotide.out,aa:VTML80.out -k 0 --k-score 2147483647 --alph-size nucl:5,aa:21 --max-seq-len 65535 --max-seqs 300 --split 0 --split-mode 2 --split-memory-limit 0 -c 0 --cov-mode 0 --comp-bias-corr 1 --diag-score 1 --exact-kmer-matching 0 --mask 1 --mask-lower-case 0 --min-ungapped-score 15 --add-self-matches 0 --spaced-kmer-mode 1 --db-load-mode 0 --pca 1 --pcb 1.5 --threads 4 --compressed 0 -v 3 -s 7.0 \n","\n","Query database size: 3737 type: Aminoacid\n","Estimated memory consumption: 2G\n","Target database size: 562755 type: Aminoacid\n","Index table k-mer threshold: 100 at k-mer size 6 \n","Index table: counting k-mers\n","[=================================================================] 100.00% 562.76K 24s 124ms\n","Index table: Masked residues: 3164874\n","Index table: fill\n","[=================================================================] 100.00% 562.76K 17s 928ms\n","Index statistics\n","Entries:          192757622\n","DB size:          1591 MB\n","Avg k-mer size:   3.011838\n","Top 10 k-mers\n","    GPGGTL\t1835\n","    GQSWTV\t1706\n","    WGMFAT\t1638\n","    PGVFEV\t1634\n","    VLWQFW\t1623\n","    AYIRPN\t1587\n","    RSPKGV\t1585\n","    TPHKWY\t1560\n","    HIRVGW\t1548\n","    ITLSPY\t1541\n","Time for index table init: 0h 0m 44s 180ms\n","Process prefiltering step 1 of 1\n","\n","k-mer similarity threshold: 100\n","Starting prefiltering scores calculation (step 1 of 1)\n","Query db start 1 to 3737\n","Target db start 1 to 562755\n","[=================================================================] 100.00% 3.74K 29s 553ms\n","\n","916.831461 k-mers per position\n","674951 DB matches per sequence\n","229 overflows\n","0 queries produce too many hits (truncated result)\n","285 sequences passed prefiltering per query sequence\n","300 median result list length\n","1 sequences with 0 size result lists\n","Time for merging to pref_0: 0h 0m 0s 1ms\n","Time for processing: 0h 1m 14s 864ms\n","align /content/mmseqs2/tmp/37/candidateDB_53 /content/mmseqs2/swissprot/swissprotDB /content/mmseqs2/tmp/37/11206407376705214811/pref_0 /content/mmseqs2/tmp/37/resultsDB_53 --sub-mat nucl:nucleotide.out,aa:blosum62.out -a 0 --alignment-mode 2 --wrapped-scoring 0 -e 0.001 --min-seq-id 0 --min-aln-len 0 --seq-id-mode 0 --alt-ali 0 -c 0 --cov-mode 0 --max-seq-len 65535 --comp-bias-corr 1 --realign 0 --max-rejected 2147483647 --max-accept 2147483647 --add-self-matches 0 --db-load-mode 0 --pca 1 --pcb 1.5 --score-bias 0 --gap-open nucl:5,aa:11 --gap-extend nucl:2,aa:1 --zdrop 40 --threads 4 --compressed 0 -v 3 \n","\n","Compute score and coverage\n","Query database size: 3737 type: Aminoacid\n","Target database size: 562755 type: Aminoacid\n","Calculation of alignments\n","[=================================================================] 100.00% 3.74K 21s 740ms\n","Time for merging to resultsDB_53: 0h 0m 0s 1ms\n","\n","1065187 alignments calculated.\n","192868 sequence pairs passed the thresholds (0.181065 of overall calculated).\n","51.610382 hits per query sequence.\n","Time for processing: 0h 0m 21s 817ms\n","convertalis /content/mmseqs2/tmp/37/candidateDB_53 /content/mmseqs2/swissprot/swissprotDB /content/mmseqs2/tmp/37/resultsDB_53 /content/mmseqs2/tmp/37/resultDB_secondary.m8 --format-output query,target,evalue,raw \n","\n","MMseqs Version:        \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Substitution matrix    \tnucl:nucleotide.out,aa:blosum62.out\n","Alignment format       \t0\n","Format alignment output\tquery,target,evalue,raw\n","Translation table      \t1\n","Gap open cost          \tnucl:5,aa:11\n","Gap extension cost     \tnucl:2,aa:1\n","Database output        \tfalse\n","Preload mode           \t0\n","Search type            \t0\n","Threads                \t4\n","Compressed             \t0\n","Verbosity              \t3\n","\n","[=================================================================] 100.00% 3.74K 0s 130ms\n","Time for merging to resultDB_secondary.m8: 0h 0m 0s 6ms\n","Time for processing: 0h 0m 0s 224ms\n","found 2249 genes\n","\n","\n","Reading fasta file GCF_000020965.1_ASM2096v1_genomic.fna.gz...\n","\n","Finding and translating open reading frames...\n","\n","Encoding amino acids...\n","\n","Applying kmer prefilter...\n","\n","Scoring ORFs with temporal convolutional network...\n","\n"],"name":"stdout"},{"output_type":"display_data","data":{"application/vnd.jupyter.widget-view+json":{"model_id":"048e9cad5a1c4d058a0fc845706c88bc","version_minor":0,"version_major":2},"text/plain":["HBox(children=(FloatProgress(value=0.0, max=19.0), HTML(value='')))"]},"metadata":{"tags":[]}},{"output_type":"stream","text":["\n","Scoring translation initiation sites...\n","\n"],"name":"stdout"},{"output_type":"display_data","data":{"application/vnd.jupyter.widget-view+json":{"model_id":"b4e9dd7b7a2441cbbb462522ae5cb527","version_minor":0,"version_major":2},"text/plain":["HBox(children=(FloatProgress(value=0.0, max=6.0), HTML(value='')))"]},"metadata":{"tags":[]}},{"output_type":"stream","text":["\n","Creating weighted directed acyclic graph of all ORFs and subORFs on contig 0...\n","\n"],"name":"stdout"},{"output_type":"display_data","data":{"application/vnd.jupyter.widget-view+json":{"model_id":"13b85acb0a834129afec188f7e5f5609","version_minor":0,"version_major":2},"text/plain":["HBox(children=(FloatProgress(value=0.0, max=54299.0), HTML(value='')))"]},"metadata":{"tags":[]}},{"output_type":"stream","text":["\n","Maximizing geneiness...\n","\n","Filtering predictions with mmseqs2...\n","createdb /content/mmseqs2/tmp/38/candidate_genes_35.fasta /content/mmseqs2/tmp/38/candidateDB_35 \n","\n","MMseqs Version:       \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Database type         \t0\n","Shuffle input database\ttrue\n","Createdb mode         \t0\n","Write lookup file     \t1\n","Offset of numeric ids \t0\n","Compressed            \t0\n","Verbosity             \t3\n","\n","Converting sequences\n","[2425] 0s 4ms\n","Time for merging to candidateDB_35_h: 0h 0m 0s 6ms\n","Time for merging to candidateDB_35: 0h 0m 0s 3ms\n","Database type: Aminoacid\n","Time for processing: 0h 0m 0s 18ms\n","createdb /content/mmseqs2/tmp/38/candidate_genes_53.fasta /content/mmseqs2/tmp/38/candidateDB_53 \n","\n","MMseqs Version:       \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Database type         \t0\n","Shuffle input database\ttrue\n","Createdb mode         \t0\n","Write lookup file     \t1\n","Offset of numeric ids \t0\n","Compressed            \t0\n","Verbosity             \t3\n","\n","Converting sequences\n","[2425] 0s 4ms\n","Time for merging to candidateDB_53_h: 0h 0m 0s 4ms\n","Time for merging to candidateDB_53: 0h 0m 0s 3ms\n","Database type: Aminoacid\n","Time for processing: 0h 0m 0s 17ms\n","search -s 7.0 /content/mmseqs2/tmp/38/candidateDB_35 /content/mmseqs2/genexa/genexaDB /content/mmseqs2/tmp/38/resultsDB_35 /content/mmseqs2/tmp/38 \n","\n","MMseqs Version:                     \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Substitution matrix                 \tnucl:nucleotide.out,aa:blosum62.out\n","Add backtrace                       \tfalse\n","Alignment mode                      \t2\n","Allow wrapped scoring               \tfalse\n","E-value threshold                   \t0.001\n","Seq. id. threshold                  \t0\n","Min alignment length                \t0\n","Seq. id. mode                       \t0\n","Alternative alignments              \t0\n","Coverage threshold                  \t0\n","Coverage mode                       \t0\n","Max sequence length                 \t65535\n","Compositional bias                  \t1\n","Realign hits                        \tfalse\n","Max reject                          \t2147483647\n","Max accept                          \t2147483647\n","Include identical seq. id.          \tfalse\n","Preload mode                        \t0\n","Pseudo count a                      \t1\n","Pseudo count b                      \t1.5\n","Score bias                          \t0\n","Gap open cost                       \tnucl:5,aa:11\n","Gap extension cost                  \tnucl:2,aa:1\n","Zdrop                               \t40\n","Threads                             \t4\n","Compressed                          \t0\n","Verbosity                           \t3\n","Seed substitution matrix            \tnucl:nucleotide.out,aa:VTML80.out\n","Sensitivity                         \t7\n","k-mer length                        \t0\n","k-score                             \t2147483647\n","Alphabet size                       \tnucl:5,aa:21\n","Max results per query               \t300\n","Split database                      \t0\n","Split mode                          \t2\n","Split memory limit                  \t0\n","Diagonal scoring                    \ttrue\n","Exact k-mer matching                \t0\n","Mask residues                       \t1\n","Mask lower case residues            \t0\n","Minimum diagonal score              \t15\n","Spaced k-mers                       \t1\n","Spaced k-mer pattern                \t\n","Local temporary path                \t\n","Rescore mode                        \t0\n","Remove hits by seq. id. and coverage\tfalse\n","Sort results                        \t0\n","Mask profile                        \t1\n","Profile e-value threshold           \t0.1\n","Global sequence weighting           \tfalse\n","Allow deletions                     \tfalse\n","Filter MSA                          \t1\n","Maximum seq. id. threshold          \t0.9\n","Minimum seq. id.                    \t0\n","Minimum score per column            \t-20\n","Minimum coverage                    \t0\n","Select N most diverse seqs          \t1000\n","Omit consensus                      \tfalse\n","Min codons in orf                   \t30\n","Max codons in length                \t32734\n","Max orf gaps                        \t2147483647\n","Contig start mode                   \t2\n","Contig end mode                     \t2\n","Orf start mode                      \t1\n","Forward frames                      \t1,2,3\n","Reverse frames                      \t1,2,3\n","Translation table                   \t1\n","Translate orf                       \t0\n","Use all table starts                \tfalse\n","Offset of numeric ids               \t0\n","Create lookup                       \t0\n","Add orf stop                        \tfalse\n","Chain overlapping alignments        \t0\n","Merge query                         \t1\n","Search type                         \t0\n","Search iterations                   \t1\n","Start sensitivity                   \t4\n","Search steps                        \t1\n","Slice search mode                   \tfalse\n","Strand selection                    \t1\n","Disk space limit                    \t0\n","MPI runner                          \t\n","Force restart with latest tmp       \tfalse\n","Remove temporary files              \tfalse\n","\n","prefilter /content/mmseqs2/tmp/38/candidateDB_35 /content/mmseqs2/genexa/genexaDB.idx /content/mmseqs2/tmp/38/7975145634517847164/pref_0 --sub-mat nucl:nucleotide.out,aa:blosum62.out --seed-sub-mat nucl:nucleotide.out,aa:VTML80.out -k 0 --k-score 2147483647 --alph-size nucl:5,aa:21 --max-seq-len 65535 --max-seqs 300 --split 0 --split-mode 2 --split-memory-limit 0 -c 0 --cov-mode 0 --comp-bias-corr 1 --diag-score 1 --exact-kmer-matching 0 --mask 1 --mask-lower-case 0 --min-ungapped-score 15 --add-self-matches 0 --spaced-kmer-mode 1 --db-load-mode 0 --pca 1 --pcb 1.5 --threads 4 --compressed 0 -v 3 -s 7.0 \n","\n","Index version: 16\n","Generated by:  16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","ScoreMatrix:  VTML80.out\n","Query database size: 2485 type: Aminoacid\n","Estimated memory consumption: 2G\n","Target database size: 468351 type: Aminoacid\n","Process prefiltering step 1 of 1\n","\n","k-mer similarity threshold: 100\n","Starting prefiltering scores calculation (step 1 of 1)\n","Query db start 1 to 2485\n","Target db start 1 to 468351\n","[=================================================================] 100.00% 2.48K 24s 134ms\n","\n","967.754756 k-mers per position\n","753026 DB matches per sequence\n","136 overflows\n","0 queries produce too many hits (truncated result)\n","278 sequences passed prefiltering per query sequence\n","300 median result list length\n","4 sequences with 0 size result lists\n","Time for merging to pref_0: 0h 0m 0s 1ms\n","Time for processing: 0h 0m 25s 576ms\n","align /content/mmseqs2/tmp/38/candidateDB_35 /content/mmseqs2/genexa/genexaDB.idx /content/mmseqs2/tmp/38/7975145634517847164/pref_0 /content/mmseqs2/tmp/38/resultsDB_35 --sub-mat nucl:nucleotide.out,aa:blosum62.out -a 0 --alignment-mode 2 --wrapped-scoring 0 -e 0.001 --min-seq-id 0 --min-aln-len 0 --seq-id-mode 0 --alt-ali 0 -c 0 --cov-mode 0 --max-seq-len 65535 --comp-bias-corr 1 --realign 0 --max-rejected 2147483647 --max-accept 2147483647 --add-self-matches 0 --db-load-mode 0 --pca 1 --pcb 1.5 --score-bias 0 --gap-open nucl:5,aa:11 --gap-extend nucl:2,aa:1 --zdrop 40 --threads 4 --compressed 0 -v 3 \n","\n","Index version: 16\n","Generated by:  16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","ScoreMatrix:  VTML80.out\n","Compute score and coverage\n","Query database size: 2485 type: Aminoacid\n","Target database size: 468351 type: Aminoacid\n","Calculation of alignments\n","[=================================================================] 100.00% 2.48K 19s 825ms\n","Time for merging to resultsDB_35: 0h 0m 0s 1ms\n","\n","692178 alignments calculated.\n","245606 sequence pairs passed the thresholds (0.354831 of overall calculated).\n","98.835411 hits per query sequence.\n","Time for processing: 0h 0m 19s 910ms\n","convertalis /content/mmseqs2/tmp/38/candidateDB_35 /content/mmseqs2/genexa/genexaDB /content/mmseqs2/tmp/38/resultsDB_35 /content/mmseqs2/tmp/38/resultDB_genexa.m8 --format-output query,target,evalue,raw \n","\n","MMseqs Version:        \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Substitution matrix    \tnucl:nucleotide.out,aa:blosum62.out\n","Alignment format       \t0\n","Format alignment output\tquery,target,evalue,raw\n","Translation table      \t1\n","Gap open cost          \tnucl:5,aa:11\n","Gap extension cost     \tnucl:2,aa:1\n","Database output        \tfalse\n","Preload mode           \t0\n","Search type            \t0\n","Threads                \t4\n","Compressed             \t0\n","Verbosity              \t3\n","\n","[=================================================================] 100.00% 2.48K 0s 183ms\n","Time for merging to resultDB_genexa.m8: 0h 0m 0s 7ms\n","Time for processing: 0h 0m 0s 261ms\n","search -s 7.0 /content/mmseqs2/tmp/38/candidateDB_53 /content/mmseqs2/swissprot/swissprotDB /content/mmseqs2/tmp/38/resultsDB_53 /content/mmseqs2/tmp/38 \n","\n","MMseqs Version:                     \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Substitution matrix                 \tnucl:nucleotide.out,aa:blosum62.out\n","Add backtrace                       \tfalse\n","Alignment mode                      \t2\n","Allow wrapped scoring               \tfalse\n","E-value threshold                   \t0.001\n","Seq. id. threshold                  \t0\n","Min alignment length                \t0\n","Seq. id. mode                       \t0\n","Alternative alignments              \t0\n","Coverage threshold                  \t0\n","Coverage mode                       \t0\n","Max sequence length                 \t65535\n","Compositional bias                  \t1\n","Realign hits                        \tfalse\n","Max reject                          \t2147483647\n","Max accept                          \t2147483647\n","Include identical seq. id.          \tfalse\n","Preload mode                        \t0\n","Pseudo count a                      \t1\n","Pseudo count b                      \t1.5\n","Score bias                          \t0\n","Gap open cost                       \tnucl:5,aa:11\n","Gap extension cost                  \tnucl:2,aa:1\n","Zdrop                               \t40\n","Threads                             \t4\n","Compressed                          \t0\n","Verbosity                           \t3\n","Seed substitution matrix            \tnucl:nucleotide.out,aa:VTML80.out\n","Sensitivity                         \t7\n","k-mer length                        \t0\n","k-score                             \t2147483647\n","Alphabet size                       \tnucl:5,aa:21\n","Max results per query               \t300\n","Split database                      \t0\n","Split mode                          \t2\n","Split memory limit                  \t0\n","Diagonal scoring                    \ttrue\n","Exact k-mer matching                \t0\n","Mask residues                       \t1\n","Mask lower case residues            \t0\n","Minimum diagonal score              \t15\n","Spaced k-mers                       \t1\n","Spaced k-mer pattern                \t\n","Local temporary path                \t\n","Rescore mode                        \t0\n","Remove hits by seq. id. and coverage\tfalse\n","Sort results                        \t0\n","Mask profile                        \t1\n","Profile e-value threshold           \t0.1\n","Global sequence weighting           \tfalse\n","Allow deletions                     \tfalse\n","Filter MSA                          \t1\n","Maximum seq. id. threshold          \t0.9\n","Minimum seq. id.                    \t0\n","Minimum score per column            \t-20\n","Minimum coverage                    \t0\n","Select N most diverse seqs          \t1000\n","Omit consensus                      \tfalse\n","Min codons in orf                   \t30\n","Max codons in length                \t32734\n","Max orf gaps                        \t2147483647\n","Contig start mode                   \t2\n","Contig end mode                     \t2\n","Orf start mode                      \t1\n","Forward frames                      \t1,2,3\n","Reverse frames                      \t1,2,3\n","Translation table                   \t1\n","Translate orf                       \t0\n","Use all table starts                \tfalse\n","Offset of numeric ids               \t0\n","Create lookup                       \t0\n","Add orf stop                        \tfalse\n","Chain overlapping alignments        \t0\n","Merge query                         \t1\n","Search type                         \t0\n","Search iterations                   \t1\n","Start sensitivity                   \t4\n","Search steps                        \t1\n","Slice search mode                   \tfalse\n","Strand selection                    \t1\n","Disk space limit                    \t0\n","MPI runner                          \t\n","Force restart with latest tmp       \tfalse\n","Remove temporary files              \tfalse\n","\n","prefilter /content/mmseqs2/tmp/38/candidateDB_53 /content/mmseqs2/swissprot/swissprotDB /content/mmseqs2/tmp/38/11307772364750560794/pref_0 --sub-mat nucl:nucleotide.out,aa:blosum62.out --seed-sub-mat nucl:nucleotide.out,aa:VTML80.out -k 0 --k-score 2147483647 --alph-size nucl:5,aa:21 --max-seq-len 65535 --max-seqs 300 --split 0 --split-mode 2 --split-memory-limit 0 -c 0 --cov-mode 0 --comp-bias-corr 1 --diag-score 1 --exact-kmer-matching 0 --mask 1 --mask-lower-case 0 --min-ungapped-score 15 --add-self-matches 0 --spaced-kmer-mode 1 --db-load-mode 0 --pca 1 --pcb 1.5 --threads 4 --compressed 0 -v 3 -s 7.0 \n","\n","Query database size: 2485 type: Aminoacid\n","Estimated memory consumption: 2G\n","Target database size: 562755 type: Aminoacid\n","Index table k-mer threshold: 100 at k-mer size 6 \n","Index table: counting k-mers\n","[=================================================================] 100.00% 562.76K 24s 287ms\n","Index table: Masked residues: 3164874\n","Index table: fill\n","[=================================================================] 100.00% 562.76K 17s 854ms\n","Index statistics\n","Entries:          192757622\n","DB size:          1591 MB\n","Avg k-mer size:   3.011838\n","Top 10 k-mers\n","    GPGGTL\t1835\n","    GQSWTV\t1706\n","    WGMFAT\t1638\n","    PGVFEV\t1634\n","    VLWQFW\t1623\n","    AYIRPN\t1587\n","    RSPKGV\t1585\n","    TPHKWY\t1560\n","    HIRVGW\t1548\n","    ITLSPY\t1541\n","Time for index table init: 0h 0m 44s 224ms\n","Process prefiltering step 1 of 1\n","\n","k-mer similarity threshold: 100\n","Starting prefiltering scores calculation (step 1 of 1)\n","Query db start 1 to 2485\n","Target db start 1 to 562755\n","[=================================================================] 100.00% 2.48K 27s 984ms\n","\n","967.748846 k-mers per position\n","951634 DB matches per sequence\n","267 overflows\n","0 queries produce too many hits (truncated result)\n","285 sequences passed prefiltering per query sequence\n","300 median result list length\n","6 sequences with 0 size result lists\n","Time for merging to pref_0: 0h 0m 0s 1ms\n","Time for processing: 0h 1m 13s 361ms\n","align /content/mmseqs2/tmp/38/candidateDB_53 /content/mmseqs2/swissprot/swissprotDB /content/mmseqs2/tmp/38/11307772364750560794/pref_0 /content/mmseqs2/tmp/38/resultsDB_53 --sub-mat nucl:nucleotide.out,aa:blosum62.out -a 0 --alignment-mode 2 --wrapped-scoring 0 -e 0.001 --min-seq-id 0 --min-aln-len 0 --seq-id-mode 0 --alt-ali 0 -c 0 --cov-mode 0 --max-seq-len 65535 --comp-bias-corr 1 --realign 0 --max-rejected 2147483647 --max-accept 2147483647 --add-self-matches 0 --db-load-mode 0 --pca 1 --pcb 1.5 --score-bias 0 --gap-open nucl:5,aa:11 --gap-extend nucl:2,aa:1 --zdrop 40 --threads 4 --compressed 0 -v 3 \n","\n","Compute score and coverage\n","Query database size: 2485 type: Aminoacid\n","Target database size: 562755 type: Aminoacid\n","Calculation of alignments\n","[=================================================================] 100.00% 2.48K 19s 315ms\n","Time for merging to resultsDB_53: 0h 0m 0s 1ms\n","\n","709412 alignments calculated.\n","196845 sequence pairs passed the thresholds (0.277476 of overall calculated).\n","79.213280 hits per query sequence.\n","Time for processing: 0h 0m 19s 384ms\n","convertalis /content/mmseqs2/tmp/38/candidateDB_53 /content/mmseqs2/swissprot/swissprotDB /content/mmseqs2/tmp/38/resultsDB_53 /content/mmseqs2/tmp/38/resultDB_secondary.m8 --format-output query,target,evalue,raw \n","\n","MMseqs Version:        \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Substitution matrix    \tnucl:nucleotide.out,aa:blosum62.out\n","Alignment format       \t0\n","Format alignment output\tquery,target,evalue,raw\n","Translation table      \t1\n","Gap open cost          \tnucl:5,aa:11\n","Gap extension cost     \tnucl:2,aa:1\n","Database output        \tfalse\n","Preload mode           \t0\n","Search type            \t0\n","Threads                \t4\n","Compressed             \t0\n","Verbosity              \t3\n","\n","[=================================================================] 100.00% 2.48K 0s 133ms\n","Time for merging to resultDB_secondary.m8: 0h 0m 0s 5ms\n","Time for processing: 0h 0m 0s 227ms\n","found 1858 genes\n","\n","\n","Reading fasta file GCF_000020505.1_ASM2050v1_genomic.fna.gz...\n","\n","Finding and translating open reading frames...\n","\n","Encoding amino acids...\n","\n","Applying kmer prefilter...\n","\n","Scoring ORFs with temporal convolutional network...\n","\n"],"name":"stdout"},{"output_type":"display_data","data":{"application/vnd.jupyter.widget-view+json":{"model_id":"66720ecd5d564308ae9537abf796386c","version_minor":0,"version_major":2},"text/plain":["HBox(children=(FloatProgress(value=0.0, max=44.0), HTML(value='')))"]},"metadata":{"tags":[]}},{"output_type":"stream","text":["\n","Scoring translation initiation sites...\n","\n"],"name":"stdout"},{"output_type":"display_data","data":{"application/vnd.jupyter.widget-view+json":{"model_id":"aafcacd6301b45168b390b80669a4b20","version_minor":0,"version_major":2},"text/plain":["HBox(children=(FloatProgress(value=0.0, max=14.0), HTML(value='')))"]},"metadata":{"tags":[]}},{"output_type":"stream","text":["\n","Creating weighted directed acyclic graph of all ORFs and subORFs on contig 0...\n","\n"],"name":"stdout"},{"output_type":"display_data","data":{"application/vnd.jupyter.widget-view+json":{"model_id":"1dc41c6bde60429c867d1728ac1d9bcd","version_minor":0,"version_major":2},"text/plain":["HBox(children=(FloatProgress(value=0.0, max=130339.0), HTML(value='')))"]},"metadata":{"tags":[]}},{"output_type":"stream","text":["\n","Maximizing geneiness...\n","\n","Filtering predictions with mmseqs2...\n","createdb /content/mmseqs2/tmp/39/candidate_genes_35.fasta /content/mmseqs2/tmp/39/candidateDB_35 \n","\n","MMseqs Version:       \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Database type         \t0\n","Shuffle input database\ttrue\n","Createdb mode         \t0\n","Write lookup file     \t1\n","Offset of numeric ids \t0\n","Compressed            \t0\n","Verbosity             \t3\n","\n","Converting sequences\n","[3738] 0s 6ms\n","Time for merging to candidateDB_35_h: 0h 0m 0s 4ms\n","Time for merging to candidateDB_35: 0h 0m 0s 3ms\n","Database type: Aminoacid\n","Time for processing: 0h 0m 0s 20ms\n","createdb /content/mmseqs2/tmp/39/candidate_genes_53.fasta /content/mmseqs2/tmp/39/candidateDB_53 \n","\n","MMseqs Version:       \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Database type         \t0\n","Shuffle input database\ttrue\n","Createdb mode         \t0\n","Write lookup file     \t1\n","Offset of numeric ids \t0\n","Compressed            \t0\n","Verbosity             \t3\n","\n","Converting sequences\n","[3738] 0s 5ms\n","Time for merging to candidateDB_53_h: 0h 0m 0s 3ms\n","Time for merging to candidateDB_53: 0h 0m 0s 3ms\n","Database type: Aminoacid\n","Time for processing: 0h 0m 0s 18ms\n","search -s 7.0 /content/mmseqs2/tmp/39/candidateDB_35 /content/mmseqs2/genexa/genexaDB /content/mmseqs2/tmp/39/resultsDB_35 /content/mmseqs2/tmp/39 \n","\n","MMseqs Version:                     \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Substitution matrix                 \tnucl:nucleotide.out,aa:blosum62.out\n","Add backtrace                       \tfalse\n","Alignment mode                      \t2\n","Allow wrapped scoring               \tfalse\n","E-value threshold                   \t0.001\n","Seq. id. threshold                  \t0\n","Min alignment length                \t0\n","Seq. id. mode                       \t0\n","Alternative alignments              \t0\n","Coverage threshold                  \t0\n","Coverage mode                       \t0\n","Max sequence length                 \t65535\n","Compositional bias                  \t1\n","Realign hits                        \tfalse\n","Max reject                          \t2147483647\n","Max accept                          \t2147483647\n","Include identical seq. id.          \tfalse\n","Preload mode                        \t0\n","Pseudo count a                      \t1\n","Pseudo count b                      \t1.5\n","Score bias                          \t0\n","Gap open cost                       \tnucl:5,aa:11\n","Gap extension cost                  \tnucl:2,aa:1\n","Zdrop                               \t40\n","Threads                             \t4\n","Compressed                          \t0\n","Verbosity                           \t3\n","Seed substitution matrix            \tnucl:nucleotide.out,aa:VTML80.out\n","Sensitivity                         \t7\n","k-mer length                        \t0\n","k-score                             \t2147483647\n","Alphabet size                       \tnucl:5,aa:21\n","Max results per query               \t300\n","Split database                      \t0\n","Split mode                          \t2\n","Split memory limit                  \t0\n","Diagonal scoring                    \ttrue\n","Exact k-mer matching                \t0\n","Mask residues                       \t1\n","Mask lower case residues            \t0\n","Minimum diagonal score              \t15\n","Spaced k-mers                       \t1\n","Spaced k-mer pattern                \t\n","Local temporary path                \t\n","Rescore mode                        \t0\n","Remove hits by seq. id. and coverage\tfalse\n","Sort results                        \t0\n","Mask profile                        \t1\n","Profile e-value threshold           \t0.1\n","Global sequence weighting           \tfalse\n","Allow deletions                     \tfalse\n","Filter MSA                          \t1\n","Maximum seq. id. threshold          \t0.9\n","Minimum seq. id.                    \t0\n","Minimum score per column            \t-20\n","Minimum coverage                    \t0\n","Select N most diverse seqs          \t1000\n","Omit consensus                      \tfalse\n","Min codons in orf                   \t30\n","Max codons in length                \t32734\n","Max orf gaps                        \t2147483647\n","Contig start mode                   \t2\n","Contig end mode                     \t2\n","Orf start mode                      \t1\n","Forward frames                      \t1,2,3\n","Reverse frames                      \t1,2,3\n","Translation table                   \t1\n","Translate orf                       \t0\n","Use all table starts                \tfalse\n","Offset of numeric ids               \t0\n","Create lookup                       \t0\n","Add orf stop                        \tfalse\n","Chain overlapping alignments        \t0\n","Merge query                         \t1\n","Search type                         \t0\n","Search iterations                   \t1\n","Start sensitivity                   \t4\n","Search steps                        \t1\n","Slice search mode                   \tfalse\n","Strand selection                    \t1\n","Disk space limit                    \t0\n","MPI runner                          \t\n","Force restart with latest tmp       \tfalse\n","Remove temporary files              \tfalse\n","\n","prefilter /content/mmseqs2/tmp/39/candidateDB_35 /content/mmseqs2/genexa/genexaDB.idx /content/mmseqs2/tmp/39/2714859531408151675/pref_0 --sub-mat nucl:nucleotide.out,aa:blosum62.out --seed-sub-mat nucl:nucleotide.out,aa:VTML80.out -k 0 --k-score 2147483647 --alph-size nucl:5,aa:21 --max-seq-len 65535 --max-seqs 300 --split 0 --split-mode 2 --split-memory-limit 0 -c 0 --cov-mode 0 --comp-bias-corr 1 --diag-score 1 --exact-kmer-matching 0 --mask 1 --mask-lower-case 0 --min-ungapped-score 15 --add-self-matches 0 --spaced-kmer-mode 1 --db-load-mode 0 --pca 1 --pcb 1.5 --threads 4 --compressed 0 -v 3 -s 7.0 \n","\n","Index version: 16\n","Generated by:  16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","ScoreMatrix:  VTML80.out\n","Query database size: 3743 type: Aminoacid\n","Estimated memory consumption: 2G\n","Target database size: 468351 type: Aminoacid\n","Process prefiltering step 1 of 1\n","\n","k-mer similarity threshold: 100\n","Starting prefiltering scores calculation (step 1 of 1)\n","Query db start 1 to 3743\n","Target db start 1 to 468351\n","[=================================================================] 100.00% 3.74K 26s 118ms\n","\n","954.050351 k-mers per position\n","539021 DB matches per sequence\n","130 overflows\n","0 queries produce too many hits (truncated result)\n","274 sequences passed prefiltering per query sequence\n","300 median result list length\n","3 sequences with 0 size result lists\n","Time for merging to pref_0: 0h 0m 0s 1ms\n","Time for processing: 0h 0m 27s 520ms\n","align /content/mmseqs2/tmp/39/candidateDB_35 /content/mmseqs2/genexa/genexaDB.idx /content/mmseqs2/tmp/39/2714859531408151675/pref_0 /content/mmseqs2/tmp/39/resultsDB_35 --sub-mat nucl:nucleotide.out,aa:blosum62.out -a 0 --alignment-mode 2 --wrapped-scoring 0 -e 0.001 --min-seq-id 0 --min-aln-len 0 --seq-id-mode 0 --alt-ali 0 -c 0 --cov-mode 0 --max-seq-len 65535 --comp-bias-corr 1 --realign 0 --max-rejected 2147483647 --max-accept 2147483647 --add-self-matches 0 --db-load-mode 0 --pca 1 --pcb 1.5 --score-bias 0 --gap-open nucl:5,aa:11 --gap-extend nucl:2,aa:1 --zdrop 40 --threads 4 --compressed 0 -v 3 \n","\n","Index version: 16\n","Generated by:  16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","ScoreMatrix:  VTML80.out\n","Compute score and coverage\n","Query database size: 3743 type: Aminoacid\n","Target database size: 468351 type: Aminoacid\n","Calculation of alignments\n","[=================================================================] 100.00% 3.74K 22s 579ms\n","Time for merging to resultsDB_35: 0h 0m 0s 1ms\n","\n","1025739 alignments calculated.\n","240491 sequence pairs passed the thresholds (0.234456 of overall calculated).\n","64.250870 hits per query sequence.\n","Time for processing: 0h 0m 22s 604ms\n","convertalis /content/mmseqs2/tmp/39/candidateDB_35 /content/mmseqs2/genexa/genexaDB /content/mmseqs2/tmp/39/resultsDB_35 /content/mmseqs2/tmp/39/resultDB_genexa.m8 --format-output query,target,evalue,raw \n","\n","MMseqs Version:        \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Substitution matrix    \tnucl:nucleotide.out,aa:blosum62.out\n","Alignment format       \t0\n","Format alignment output\tquery,target,evalue,raw\n","Translation table      \t1\n","Gap open cost          \tnucl:5,aa:11\n","Gap extension cost     \tnucl:2,aa:1\n","Database output        \tfalse\n","Preload mode           \t0\n","Search type            \t0\n","Threads                \t4\n","Compressed             \t0\n","Verbosity              \t3\n","\n","[=================================================================] 100.00% 3.74K 0s 174ms\n","Time for merging to resultDB_genexa.m8: 0h 0m 0s 7ms\n","Time for processing: 0h 0m 0s 251ms\n","search -s 7.0 /content/mmseqs2/tmp/39/candidateDB_53 /content/mmseqs2/swissprot/swissprotDB /content/mmseqs2/tmp/39/resultsDB_53 /content/mmseqs2/tmp/39 \n","\n","MMseqs Version:                     \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Substitution matrix                 \tnucl:nucleotide.out,aa:blosum62.out\n","Add backtrace                       \tfalse\n","Alignment mode                      \t2\n","Allow wrapped scoring               \tfalse\n","E-value threshold                   \t0.001\n","Seq. id. threshold                  \t0\n","Min alignment length                \t0\n","Seq. id. mode                       \t0\n","Alternative alignments              \t0\n","Coverage threshold                  \t0\n","Coverage mode                       \t0\n","Max sequence length                 \t65535\n","Compositional bias                  \t1\n","Realign hits                        \tfalse\n","Max reject                          \t2147483647\n","Max accept                          \t2147483647\n","Include identical seq. id.          \tfalse\n","Preload mode                        \t0\n","Pseudo count a                      \t1\n","Pseudo count b                      \t1.5\n","Score bias                          \t0\n","Gap open cost                       \tnucl:5,aa:11\n","Gap extension cost                  \tnucl:2,aa:1\n","Zdrop                               \t40\n","Threads                             \t4\n","Compressed                          \t0\n","Verbosity                           \t3\n","Seed substitution matrix            \tnucl:nucleotide.out,aa:VTML80.out\n","Sensitivity                         \t7\n","k-mer length                        \t0\n","k-score                             \t2147483647\n","Alphabet size                       \tnucl:5,aa:21\n","Max results per query               \t300\n","Split database                      \t0\n","Split mode                          \t2\n","Split memory limit                  \t0\n","Diagonal scoring                    \ttrue\n","Exact k-mer matching                \t0\n","Mask residues                       \t1\n","Mask lower case residues            \t0\n","Minimum diagonal score              \t15\n","Spaced k-mers                       \t1\n","Spaced k-mer pattern                \t\n","Local temporary path                \t\n","Rescore mode                        \t0\n","Remove hits by seq. id. and coverage\tfalse\n","Sort results                        \t0\n","Mask profile                        \t1\n","Profile e-value threshold           \t0.1\n","Global sequence weighting           \tfalse\n","Allow deletions                     \tfalse\n","Filter MSA                          \t1\n","Maximum seq. id. threshold          \t0.9\n","Minimum seq. id.                    \t0\n","Minimum score per column            \t-20\n","Minimum coverage                    \t0\n","Select N most diverse seqs          \t1000\n","Omit consensus                      \tfalse\n","Min codons in orf                   \t30\n","Max codons in length                \t32734\n","Max orf gaps                        \t2147483647\n","Contig start mode                   \t2\n","Contig end mode                     \t2\n","Orf start mode                      \t1\n","Forward frames                      \t1,2,3\n","Reverse frames                      \t1,2,3\n","Translation table                   \t1\n","Translate orf                       \t0\n","Use all table starts                \tfalse\n","Offset of numeric ids               \t0\n","Create lookup                       \t0\n","Add orf stop                        \tfalse\n","Chain overlapping alignments        \t0\n","Merge query                         \t1\n","Search type                         \t0\n","Search iterations                   \t1\n","Start sensitivity                   \t4\n","Search steps                        \t1\n","Slice search mode                   \tfalse\n","Strand selection                    \t1\n","Disk space limit                    \t0\n","MPI runner                          \t\n","Force restart with latest tmp       \tfalse\n","Remove temporary files              \tfalse\n","\n","prefilter /content/mmseqs2/tmp/39/candidateDB_53 /content/mmseqs2/swissprot/swissprotDB /content/mmseqs2/tmp/39/11409137352795906777/pref_0 --sub-mat nucl:nucleotide.out,aa:blosum62.out --seed-sub-mat nucl:nucleotide.out,aa:VTML80.out -k 0 --k-score 2147483647 --alph-size nucl:5,aa:21 --max-seq-len 65535 --max-seqs 300 --split 0 --split-mode 2 --split-memory-limit 0 -c 0 --cov-mode 0 --comp-bias-corr 1 --diag-score 1 --exact-kmer-matching 0 --mask 1 --mask-lower-case 0 --min-ungapped-score 15 --add-self-matches 0 --spaced-kmer-mode 1 --db-load-mode 0 --pca 1 --pcb 1.5 --threads 4 --compressed 0 -v 3 -s 7.0 \n","\n","Query database size: 3743 type: Aminoacid\n","Estimated memory consumption: 2G\n","Target database size: 562755 type: Aminoacid\n","Index table k-mer threshold: 100 at k-mer size 6 \n","Index table: counting k-mers\n","[=================================================================] 100.00% 562.76K 24s 227ms\n","Index table: Masked residues: 3164874\n","Index table: fill\n","[=================================================================] 100.00% 562.76K 17s 738ms\n","Index statistics\n","Entries:          192757622\n","DB size:          1591 MB\n","Avg k-mer size:   3.011838\n","Top 10 k-mers\n","    GPGGTL\t1835\n","    GQSWTV\t1706\n","    WGMFAT\t1638\n","    PGVFEV\t1634\n","    VLWQFW\t1623\n","    AYIRPN\t1587\n","    RSPKGV\t1585\n","    TPHKWY\t1560\n","    HIRVGW\t1548\n","    ITLSPY\t1541\n","Time for index table init: 0h 0m 44s 43ms\n","Process prefiltering step 1 of 1\n","\n","k-mer similarity threshold: 100\n","Starting prefiltering scores calculation (step 1 of 1)\n","Query db start 1 to 3743\n","Target db start 1 to 562755\n","[=================================================================] 100.00% 3.74K 29s 409ms\n","\n","955.994050 k-mers per position\n","650390 DB matches per sequence\n","229 overflows\n","0 queries produce too many hits (truncated result)\n","282 sequences passed prefiltering per query sequence\n","300 median result list length\n","0 sequences with 0 size result lists\n","Time for merging to pref_0: 0h 0m 0s 3ms\n","Time for processing: 0h 1m 14s 562ms\n","align /content/mmseqs2/tmp/39/candidateDB_53 /content/mmseqs2/swissprot/swissprotDB /content/mmseqs2/tmp/39/11409137352795906777/pref_0 /content/mmseqs2/tmp/39/resultsDB_53 --sub-mat nucl:nucleotide.out,aa:blosum62.out -a 0 --alignment-mode 2 --wrapped-scoring 0 -e 0.001 --min-seq-id 0 --min-aln-len 0 --seq-id-mode 0 --alt-ali 0 -c 0 --cov-mode 0 --max-seq-len 65535 --comp-bias-corr 1 --realign 0 --max-rejected 2147483647 --max-accept 2147483647 --add-self-matches 0 --db-load-mode 0 --pca 1 --pcb 1.5 --score-bias 0 --gap-open nucl:5,aa:11 --gap-extend nucl:2,aa:1 --zdrop 40 --threads 4 --compressed 0 -v 3 \n","\n","Compute score and coverage\n","Query database size: 3743 type: Aminoacid\n","Target database size: 562755 type: Aminoacid\n","Calculation of alignments\n","[=================================================================] 100.00% 3.74K 22s 910ms\n","Time for merging to resultsDB_53: 0h 0m 0s 1ms\n","\n","1058062 alignments calculated.\n","205783 sequence pairs passed the thresholds (0.194490 of overall calculated).\n","54.978092 hits per query sequence.\n","Time for processing: 0h 0m 22s 976ms\n","convertalis /content/mmseqs2/tmp/39/candidateDB_53 /content/mmseqs2/swissprot/swissprotDB /content/mmseqs2/tmp/39/resultsDB_53 /content/mmseqs2/tmp/39/resultDB_secondary.m8 --format-output query,target,evalue,raw \n","\n","MMseqs Version:        \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Substitution matrix    \tnucl:nucleotide.out,aa:blosum62.out\n","Alignment format       \t0\n","Format alignment output\tquery,target,evalue,raw\n","Translation table      \t1\n","Gap open cost          \tnucl:5,aa:11\n","Gap extension cost     \tnucl:2,aa:1\n","Database output        \tfalse\n","Preload mode           \t0\n","Search type            \t0\n","Threads                \t4\n","Compressed             \t0\n","Verbosity              \t3\n","\n","[=================================================================] 100.00% 3.74K 0s 138ms\n","Time for merging to resultDB_secondary.m8: 0h 0m 0s 6ms\n","Time for processing: 0h 0m 0s 235ms\n","found 2060 genes\n","\n","\n","Reading fasta file GCF_000020005.1_ASM2000v1_genomic.fna.gz...\n","\n","Finding and translating open reading frames...\n","\n","Encoding amino acids...\n","\n","Applying kmer prefilter...\n","\n","Scoring ORFs with temporal convolutional network...\n","\n"],"name":"stdout"},{"output_type":"display_data","data":{"application/vnd.jupyter.widget-view+json":{"model_id":"05347276863b4446bfe50360ce120ec3","version_minor":0,"version_major":2},"text/plain":["HBox(children=(FloatProgress(value=0.0, max=36.0), HTML(value='')))"]},"metadata":{"tags":[]}},{"output_type":"stream","text":["\n","Scoring translation initiation sites...\n","\n"],"name":"stdout"},{"output_type":"display_data","data":{"application/vnd.jupyter.widget-view+json":{"model_id":"4db70922ee6a461cbe2760331f1de5ad","version_minor":0,"version_major":2},"text/plain":["HBox(children=(FloatProgress(value=0.0, max=11.0), HTML(value='')))"]},"metadata":{"tags":[]}},{"output_type":"stream","text":["\n","Creating weighted directed acyclic graph of all ORFs and subORFs on contig 0...\n","\n"],"name":"stdout"},{"output_type":"display_data","data":{"application/vnd.jupyter.widget-view+json":{"model_id":"669d6a3b074545029153f230f2ce4be2","version_minor":0,"version_major":2},"text/plain":["HBox(children=(FloatProgress(value=0.0, max=105276.0), HTML(value='')))"]},"metadata":{"tags":[]}},{"output_type":"stream","text":["\n","Maximizing geneiness...\n","\n","Filtering predictions with mmseqs2...\n","createdb /content/mmseqs2/tmp/40/candidate_genes_35.fasta /content/mmseqs2/tmp/40/candidateDB_35 \n","\n","MMseqs Version:       \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Database type         \t0\n","Shuffle input database\ttrue\n","Createdb mode         \t0\n","Write lookup file     \t1\n","Offset of numeric ids \t0\n","Compressed            \t0\n","Verbosity             \t3\n","\n","Converting sequences\n","[4950] 0s 6ms\n","Time for merging to candidateDB_35_h: 0h 0m 0s 4ms\n","Time for merging to candidateDB_35: 0h 0m 0s 4ms\n","Database type: Aminoacid\n","Time for processing: 0h 0m 0s 22ms\n","createdb /content/mmseqs2/tmp/40/candidate_genes_53.fasta /content/mmseqs2/tmp/40/candidateDB_53 \n","\n","MMseqs Version:       \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Database type         \t0\n","Shuffle input database\ttrue\n","Createdb mode         \t0\n","Write lookup file     \t1\n","Offset of numeric ids \t0\n","Compressed            \t0\n","Verbosity             \t3\n","\n","Converting sequences\n","[4950] 0s 8ms\n","Time for merging to candidateDB_53_h: 0h 0m 0s 3ms\n","Time for merging to candidateDB_53: 0h 0m 0s 4ms\n","Database type: Aminoacid\n","Time for processing: 0h 0m 0s 22ms\n","search -s 7.0 /content/mmseqs2/tmp/40/candidateDB_35 /content/mmseqs2/genexa/genexaDB /content/mmseqs2/tmp/40/resultsDB_35 /content/mmseqs2/tmp/40 \n","\n","MMseqs Version:                     \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Substitution matrix                 \tnucl:nucleotide.out,aa:blosum62.out\n","Add backtrace                       \tfalse\n","Alignment mode                      \t2\n","Allow wrapped scoring               \tfalse\n","E-value threshold                   \t0.001\n","Seq. id. threshold                  \t0\n","Min alignment length                \t0\n","Seq. id. mode                       \t0\n","Alternative alignments              \t0\n","Coverage threshold                  \t0\n","Coverage mode                       \t0\n","Max sequence length                 \t65535\n","Compositional bias                  \t1\n","Realign hits                        \tfalse\n","Max reject                          \t2147483647\n","Max accept                          \t2147483647\n","Include identical seq. id.          \tfalse\n","Preload mode                        \t0\n","Pseudo count a                      \t1\n","Pseudo count b                      \t1.5\n","Score bias                          \t0\n","Gap open cost                       \tnucl:5,aa:11\n","Gap extension cost                  \tnucl:2,aa:1\n","Zdrop                               \t40\n","Threads                             \t4\n","Compressed                          \t0\n","Verbosity                           \t3\n","Seed substitution matrix            \tnucl:nucleotide.out,aa:VTML80.out\n","Sensitivity                         \t7\n","k-mer length                        \t0\n","k-score                             \t2147483647\n","Alphabet size                       \tnucl:5,aa:21\n","Max results per query               \t300\n","Split database                      \t0\n","Split mode                          \t2\n","Split memory limit                  \t0\n","Diagonal scoring                    \ttrue\n","Exact k-mer matching                \t0\n","Mask residues                       \t1\n","Mask lower case residues            \t0\n","Minimum diagonal score              \t15\n","Spaced k-mers                       \t1\n","Spaced k-mer pattern                \t\n","Local temporary path                \t\n","Rescore mode                        \t0\n","Remove hits by seq. id. and coverage\tfalse\n","Sort results                        \t0\n","Mask profile                        \t1\n","Profile e-value threshold           \t0.1\n","Global sequence weighting           \tfalse\n","Allow deletions                     \tfalse\n","Filter MSA                          \t1\n","Maximum seq. id. threshold          \t0.9\n","Minimum seq. id.                    \t0\n","Minimum score per column            \t-20\n","Minimum coverage                    \t0\n","Select N most diverse seqs          \t1000\n","Omit consensus                      \tfalse\n","Min codons in orf                   \t30\n","Max codons in length                \t32734\n","Max orf gaps                        \t2147483647\n","Contig start mode                   \t2\n","Contig end mode                     \t2\n","Orf start mode                      \t1\n","Forward frames                      \t1,2,3\n","Reverse frames                      \t1,2,3\n","Translation table                   \t1\n","Translate orf                       \t0\n","Use all table starts                \tfalse\n","Offset of numeric ids               \t0\n","Create lookup                       \t0\n","Add orf stop                        \tfalse\n","Chain overlapping alignments        \t0\n","Merge query                         \t1\n","Search type                         \t0\n","Search iterations                   \t1\n","Start sensitivity                   \t4\n","Search steps                        \t1\n","Slice search mode                   \tfalse\n","Strand selection                    \t1\n","Disk space limit                    \t0\n","MPI runner                          \t\n","Force restart with latest tmp       \tfalse\n","Remove temporary files              \tfalse\n","\n","prefilter /content/mmseqs2/tmp/40/candidateDB_35 /content/mmseqs2/genexa/genexaDB.idx /content/mmseqs2/tmp/40/16115773778961712229/pref_0 --sub-mat nucl:nucleotide.out,aa:blosum62.out --seed-sub-mat nucl:nucleotide.out,aa:VTML80.out -k 0 --k-score 2147483647 --alph-size nucl:5,aa:21 --max-seq-len 65535 --max-seqs 300 --split 0 --split-mode 2 --split-memory-limit 0 -c 0 --cov-mode 0 --comp-bias-corr 1 --diag-score 1 --exact-kmer-matching 0 --mask 1 --mask-lower-case 0 --min-ungapped-score 15 --add-self-matches 0 --spaced-kmer-mode 1 --db-load-mode 0 --pca 1 --pcb 1.5 --threads 4 --compressed 0 -v 3 -s 7.0 \n","\n","Index version: 16\n","Generated by:  16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","ScoreMatrix:  VTML80.out\n","Query database size: 5025 type: Aminoacid\n","Estimated memory consumption: 2G\n","Target database size: 468351 type: Aminoacid\n","Process prefiltering step 1 of 1\n","\n","k-mer similarity threshold: 100\n","Starting prefiltering scores calculation (step 1 of 1)\n","Query db start 1 to 5025\n","Target db start 1 to 468351\n","[=================================================================] 100.00% 5.03K 32s 142ms\n","\n","848.605500 k-mers per position\n","491515 DB matches per sequence\n","105 overflows\n","0 queries produce too many hits (truncated result)\n","260 sequences passed prefiltering per query sequence\n","300 median result list length\n","4 sequences with 0 size result lists\n","Time for merging to pref_0: 0h 0m 0s 1ms\n","Time for processing: 0h 0m 33s 607ms\n","align /content/mmseqs2/tmp/40/candidateDB_35 /content/mmseqs2/genexa/genexaDB.idx /content/mmseqs2/tmp/40/16115773778961712229/pref_0 /content/mmseqs2/tmp/40/resultsDB_35 --sub-mat nucl:nucleotide.out,aa:blosum62.out -a 0 --alignment-mode 2 --wrapped-scoring 0 -e 0.001 --min-seq-id 0 --min-aln-len 0 --seq-id-mode 0 --alt-ali 0 -c 0 --cov-mode 0 --max-seq-len 65535 --comp-bias-corr 1 --realign 0 --max-rejected 2147483647 --max-accept 2147483647 --add-self-matches 0 --db-load-mode 0 --pca 1 --pcb 1.5 --score-bias 0 --gap-open nucl:5,aa:11 --gap-extend nucl:2,aa:1 --zdrop 40 --threads 4 --compressed 0 -v 3 \n","\n","Index version: 16\n","Generated by:  16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","ScoreMatrix:  VTML80.out\n","Compute score and coverage\n","Query database size: 5025 type: Aminoacid\n","Target database size: 468351 type: Aminoacid\n","Calculation of alignments\n","[=================================================================] 100.00% 5.03K 28s 212ms\n","Time for merging to resultsDB_35: 0h 0m 0s 1ms\n","\n","1311468 alignments calculated.\n","320974 sequence pairs passed the thresholds (0.244744 of overall calculated).\n","63.875423 hits per query sequence.\n","Time for processing: 0h 0m 28s 266ms\n","convertalis /content/mmseqs2/tmp/40/candidateDB_35 /content/mmseqs2/genexa/genexaDB /content/mmseqs2/tmp/40/resultsDB_35 /content/mmseqs2/tmp/40/resultDB_genexa.m8 --format-output query,target,evalue,raw \n","\n","MMseqs Version:        \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Substitution matrix    \tnucl:nucleotide.out,aa:blosum62.out\n","Alignment format       \t0\n","Format alignment output\tquery,target,evalue,raw\n","Translation table      \t1\n","Gap open cost          \tnucl:5,aa:11\n","Gap extension cost     \tnucl:2,aa:1\n","Database output        \tfalse\n","Preload mode           \t0\n","Search type            \t0\n","Threads                \t4\n","Compressed             \t0\n","Verbosity              \t3\n","\n","[=================================================================] 100.00% 5.03K 0s 231ms\n","Time for merging to resultDB_genexa.m8: 0h 0m 0s 9ms\n","Time for processing: 0h 0m 0s 313ms\n","search -s 7.0 /content/mmseqs2/tmp/40/candidateDB_53 /content/mmseqs2/swissprot/swissprotDB /content/mmseqs2/tmp/40/resultsDB_53 /content/mmseqs2/tmp/40 \n","\n","MMseqs Version:                     \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Substitution matrix                 \tnucl:nucleotide.out,aa:blosum62.out\n","Add backtrace                       \tfalse\n","Alignment mode                      \t2\n","Allow wrapped scoring               \tfalse\n","E-value threshold                   \t0.001\n","Seq. id. threshold                  \t0\n","Min alignment length                \t0\n","Seq. id. mode                       \t0\n","Alternative alignments              \t0\n","Coverage threshold                  \t0\n","Coverage mode                       \t0\n","Max sequence length                 \t65535\n","Compositional bias                  \t1\n","Realign hits                        \tfalse\n","Max reject                          \t2147483647\n","Max accept                          \t2147483647\n","Include identical seq. id.          \tfalse\n","Preload mode                        \t0\n","Pseudo count a                      \t1\n","Pseudo count b                      \t1.5\n","Score bias                          \t0\n","Gap open cost                       \tnucl:5,aa:11\n","Gap extension cost                  \tnucl:2,aa:1\n","Zdrop                               \t40\n","Threads                             \t4\n","Compressed                          \t0\n","Verbosity                           \t3\n","Seed substitution matrix            \tnucl:nucleotide.out,aa:VTML80.out\n","Sensitivity                         \t7\n","k-mer length                        \t0\n","k-score                             \t2147483647\n","Alphabet size                       \tnucl:5,aa:21\n","Max results per query               \t300\n","Split database                      \t0\n","Split mode                          \t2\n","Split memory limit                  \t0\n","Diagonal scoring                    \ttrue\n","Exact k-mer matching                \t0\n","Mask residues                       \t1\n","Mask lower case residues            \t0\n","Minimum diagonal score              \t15\n","Spaced k-mers                       \t1\n","Spaced k-mer pattern                \t\n","Local temporary path                \t\n","Rescore mode                        \t0\n","Remove hits by seq. id. and coverage\tfalse\n","Sort results                        \t0\n","Mask profile                        \t1\n","Profile e-value threshold           \t0.1\n","Global sequence weighting           \tfalse\n","Allow deletions                     \tfalse\n","Filter MSA                          \t1\n","Maximum seq. id. threshold          \t0.9\n","Minimum seq. id.                    \t0\n","Minimum score per column            \t-20\n","Minimum coverage                    \t0\n","Select N most diverse seqs          \t1000\n","Omit consensus                      \tfalse\n","Min codons in orf                   \t30\n","Max codons in length                \t32734\n","Max orf gaps                        \t2147483647\n","Contig start mode                   \t2\n","Contig end mode                     \t2\n","Orf start mode                      \t1\n","Forward frames                      \t1,2,3\n","Reverse frames                      \t1,2,3\n","Translation table                   \t1\n","Translate orf                       \t0\n","Use all table starts                \tfalse\n","Offset of numeric ids               \t0\n","Create lookup                       \t0\n","Add orf stop                        \tfalse\n","Chain overlapping alignments        \t0\n","Merge query                         \t1\n","Search type                         \t0\n","Search iterations                   \t1\n","Start sensitivity                   \t4\n","Search steps                        \t1\n","Slice search mode                   \tfalse\n","Strand selection                    \t1\n","Disk space limit                    \t0\n","MPI runner                          \t\n","Force restart with latest tmp       \tfalse\n","Remove temporary files              \tfalse\n","\n","prefilter /content/mmseqs2/tmp/40/candidateDB_53 /content/mmseqs2/swissprot/swissprotDB /content/mmseqs2/tmp/40/13639167089793518403/pref_0 --sub-mat nucl:nucleotide.out,aa:blosum62.out --seed-sub-mat nucl:nucleotide.out,aa:VTML80.out -k 0 --k-score 2147483647 --alph-size nucl:5,aa:21 --max-seq-len 65535 --max-seqs 300 --split 0 --split-mode 2 --split-memory-limit 0 -c 0 --cov-mode 0 --comp-bias-corr 1 --diag-score 1 --exact-kmer-matching 0 --mask 1 --mask-lower-case 0 --min-ungapped-score 15 --add-self-matches 0 --spaced-kmer-mode 1 --db-load-mode 0 --pca 1 --pcb 1.5 --threads 4 --compressed 0 -v 3 -s 7.0 \n","\n","Query database size: 5025 type: Aminoacid\n","Estimated memory consumption: 2G\n","Target database size: 562755 type: Aminoacid\n","Index table k-mer threshold: 100 at k-mer size 6 \n","Index table: counting k-mers\n","[=================================================================] 100.00% 562.76K 24s 295ms\n","Index table: Masked residues: 3164874\n","Index table: fill\n","[=================================================================] 100.00% 562.76K 17s 765ms\n","Index statistics\n","Entries:          192757622\n","DB size:          1591 MB\n","Avg k-mer size:   3.011838\n","Top 10 k-mers\n","    GPGGTL\t1835\n","    GQSWTV\t1706\n","    WGMFAT\t1638\n","    PGVFEV\t1634\n","    VLWQFW\t1623\n","    AYIRPN\t1587\n","    RSPKGV\t1585\n","    TPHKWY\t1560\n","    HIRVGW\t1548\n","    ITLSPY\t1541\n","Time for index table init: 0h 0m 44s 163ms\n","Process prefiltering step 1 of 1\n","\n","k-mer similarity threshold: 100\n","Starting prefiltering scores calculation (step 1 of 1)\n","Query db start 1 to 5025\n","Target db start 1 to 562755\n","[=================================================================] 100.00% 5.03K 36s 743ms\n","\n","847.107806 k-mers per position\n","614969 DB matches per sequence\n","221 overflows\n","0 queries produce too many hits (truncated result)\n","273 sequences passed prefiltering per query sequence\n","300 median result list length\n","5 sequences with 0 size result lists\n","Time for merging to pref_0: 0h 0m 0s 1ms\n","Time for processing: 0h 1m 22s 93ms\n","align /content/mmseqs2/tmp/40/candidateDB_53 /content/mmseqs2/swissprot/swissprotDB /content/mmseqs2/tmp/40/13639167089793518403/pref_0 /content/mmseqs2/tmp/40/resultsDB_53 --sub-mat nucl:nucleotide.out,aa:blosum62.out -a 0 --alignment-mode 2 --wrapped-scoring 0 -e 0.001 --min-seq-id 0 --min-aln-len 0 --seq-id-mode 0 --alt-ali 0 -c 0 --cov-mode 0 --max-seq-len 65535 --comp-bias-corr 1 --realign 0 --max-rejected 2147483647 --max-accept 2147483647 --add-self-matches 0 --db-load-mode 0 --pca 1 --pcb 1.5 --score-bias 0 --gap-open nucl:5,aa:11 --gap-extend nucl:2,aa:1 --zdrop 40 --threads 4 --compressed 0 -v 3 \n","\n","Compute score and coverage\n","Query database size: 5025 type: Aminoacid\n","Target database size: 562755 type: Aminoacid\n","Calculation of alignments\n","[=================================================================] 100.00% 5.03K 26s 842ms\n","Time for merging to resultsDB_53: 0h 0m 0s 1ms\n","\n","1372437 alignments calculated.\n","232727 sequence pairs passed the thresholds (0.169572 of overall calculated).\n","46.313831 hits per query sequence.\n","Time for processing: 0h 0m 26s 987ms\n","convertalis /content/mmseqs2/tmp/40/candidateDB_53 /content/mmseqs2/swissprot/swissprotDB /content/mmseqs2/tmp/40/resultsDB_53 /content/mmseqs2/tmp/40/resultDB_secondary.m8 --format-output query,target,evalue,raw \n","\n","MMseqs Version:        \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Substitution matrix    \tnucl:nucleotide.out,aa:blosum62.out\n","Alignment format       \t0\n","Format alignment output\tquery,target,evalue,raw\n","Translation table      \t1\n","Gap open cost          \tnucl:5,aa:11\n","Gap extension cost     \tnucl:2,aa:1\n","Database output        \tfalse\n","Preload mode           \t0\n","Search type            \t0\n","Threads                \t4\n","Compressed             \t0\n","Verbosity              \t3\n","\n","[=================================================================] 100.00% 5.03K 0s 160ms\n","Time for merging to resultDB_secondary.m8: 0h 0m 0s 7ms\n","Time for processing: 0h 0m 0s 255ms\n","found 2882 genes\n","\n","\n","Creating weighted directed acyclic graph of all ORFs and subORFs on contig 1...\n","\n"],"name":"stdout"},{"output_type":"display_data","data":{"application/vnd.jupyter.widget-view+json":{"model_id":"d5be9d7e4fda47d9b59031db23e486dd","version_minor":0,"version_major":2},"text/plain":["HBox(children=(FloatProgress(value=0.0, max=462.0), HTML(value='')))"]},"metadata":{"tags":[]}},{"output_type":"stream","text":["\n","Maximizing geneiness...\n","\n","Filtering predictions with mmseqs2...\n","createdb /content/mmseqs2/tmp/41/candidate_genes_35.fasta /content/mmseqs2/tmp/41/candidateDB_35 \n","\n","MMseqs Version:       \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Database type         \t0\n","Shuffle input database\ttrue\n","Createdb mode         \t0\n","Write lookup file     \t1\n","Offset of numeric ids \t0\n","Compressed            \t0\n","Verbosity             \t3\n","\n","Converting sequences\n","[102] 0s 2ms\n","Time for merging to candidateDB_35_h: 0h 0m 0s 1ms\n","Time for merging to candidateDB_35: 0h 0m 0s 1ms\n","Database type: Aminoacid\n","Time for processing: 0h 0m 0s 7ms\n","createdb /content/mmseqs2/tmp/41/candidate_genes_53.fasta /content/mmseqs2/tmp/41/candidateDB_53 \n","\n","MMseqs Version:       \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Database type         \t0\n","Shuffle input database\ttrue\n","Createdb mode         \t0\n","Write lookup file     \t1\n","Offset of numeric ids \t0\n","Compressed            \t0\n","Verbosity             \t3\n","\n","Converting sequences\n","[102] 0s 2ms\n","Time for merging to candidateDB_53_h: 0h 0m 0s 1ms\n","Time for merging to candidateDB_53: 0h 0m 0s 1ms\n","Database type: Aminoacid\n","Time for processing: 0h 0m 0s 7ms\n","search -s 7.0 /content/mmseqs2/tmp/41/candidateDB_35 /content/mmseqs2/genexa/genexaDB /content/mmseqs2/tmp/41/resultsDB_35 /content/mmseqs2/tmp/41 \n","\n","MMseqs Version:                     \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Substitution matrix                 \tnucl:nucleotide.out,aa:blosum62.out\n","Add backtrace                       \tfalse\n","Alignment mode                      \t2\n","Allow wrapped scoring               \tfalse\n","E-value threshold                   \t0.001\n","Seq. id. threshold                  \t0\n","Min alignment length                \t0\n","Seq. id. mode                       \t0\n","Alternative alignments              \t0\n","Coverage threshold                  \t0\n","Coverage mode                       \t0\n","Max sequence length                 \t65535\n","Compositional bias                  \t1\n","Realign hits                        \tfalse\n","Max reject                          \t2147483647\n","Max accept                          \t2147483647\n","Include identical seq. id.          \tfalse\n","Preload mode                        \t0\n","Pseudo count a                      \t1\n","Pseudo count b                      \t1.5\n","Score bias                          \t0\n","Gap open cost                       \tnucl:5,aa:11\n","Gap extension cost                  \tnucl:2,aa:1\n","Zdrop                               \t40\n","Threads                             \t4\n","Compressed                          \t0\n","Verbosity                           \t3\n","Seed substitution matrix            \tnucl:nucleotide.out,aa:VTML80.out\n","Sensitivity                         \t7\n","k-mer length                        \t0\n","k-score                             \t2147483647\n","Alphabet size                       \tnucl:5,aa:21\n","Max results per query               \t300\n","Split database                      \t0\n","Split mode                          \t2\n","Split memory limit                  \t0\n","Diagonal scoring                    \ttrue\n","Exact k-mer matching                \t0\n","Mask residues                       \t1\n","Mask lower case residues            \t0\n","Minimum diagonal score              \t15\n","Spaced k-mers                       \t1\n","Spaced k-mer pattern                \t\n","Local temporary path                \t\n","Rescore mode                        \t0\n","Remove hits by seq. id. and coverage\tfalse\n","Sort results                        \t0\n","Mask profile                        \t1\n","Profile e-value threshold           \t0.1\n","Global sequence weighting           \tfalse\n","Allow deletions                     \tfalse\n","Filter MSA                          \t1\n","Maximum seq. id. threshold          \t0.9\n","Minimum seq. id.                    \t0\n","Minimum score per column            \t-20\n","Minimum coverage                    \t0\n","Select N most diverse seqs          \t1000\n","Omit consensus                      \tfalse\n","Min codons in orf                   \t30\n","Max codons in length                \t32734\n","Max orf gaps                        \t2147483647\n","Contig start mode                   \t2\n","Contig end mode                     \t2\n","Orf start mode                      \t1\n","Forward frames                      \t1,2,3\n","Reverse frames                      \t1,2,3\n","Translation table                   \t1\n","Translate orf                       \t0\n","Use all table starts                \tfalse\n","Offset of numeric ids               \t0\n","Create lookup                       \t0\n","Add orf stop                        \tfalse\n","Chain overlapping alignments        \t0\n","Merge query                         \t1\n","Search type                         \t0\n","Search iterations                   \t1\n","Start sensitivity                   \t4\n","Search steps                        \t1\n","Slice search mode                   \tfalse\n","Strand selection                    \t1\n","Disk space limit                    \t0\n","MPI runner                          \t\n","Force restart with latest tmp       \tfalse\n","Remove temporary files              \tfalse\n","\n","prefilter /content/mmseqs2/tmp/41/candidateDB_35 /content/mmseqs2/genexa/genexaDB.idx /content/mmseqs2/tmp/41/10855487675852016740/pref_0 --sub-mat nucl:nucleotide.out,aa:blosum62.out --seed-sub-mat nucl:nucleotide.out,aa:VTML80.out -k 0 --k-score 2147483647 --alph-size nucl:5,aa:21 --max-seq-len 65535 --max-seqs 300 --split 0 --split-mode 2 --split-memory-limit 0 -c 0 --cov-mode 0 --comp-bias-corr 1 --diag-score 1 --exact-kmer-matching 0 --mask 1 --mask-lower-case 0 --min-ungapped-score 15 --add-self-matches 0 --spaced-kmer-mode 1 --db-load-mode 0 --pca 1 --pcb 1.5 --threads 4 --compressed 0 -v 3 -s 7.0 \n","\n","Index version: 16\n","Generated by:  16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","ScoreMatrix:  VTML80.out\n","Query database size: 108 type: Aminoacid\n","Estimated memory consumption: 2G\n","Target database size: 468351 type: Aminoacid\n","Process prefiltering step 1 of 1\n","\n","k-mer similarity threshold: 100\n","Starting prefiltering scores calculation (step 1 of 1)\n","Query db start 1 to 108\n","Target db start 1 to 468351\n","[=================================================================] 100.00% 108 1s 491ms\n","\n","920.405422 k-mers per position\n","1035204 DB matches per sequence\n","13 overflows\n","0 queries produce too many hits (truncated result)\n","291 sequences passed prefiltering per query sequence\n","300 median result list length\n","0 sequences with 0 size result lists\n","Time for merging to pref_0: 0h 0m 0s 0ms\n","Time for processing: 0h 0m 2s 875ms\n","align /content/mmseqs2/tmp/41/candidateDB_35 /content/mmseqs2/genexa/genexaDB.idx /content/mmseqs2/tmp/41/10855487675852016740/pref_0 /content/mmseqs2/tmp/41/resultsDB_35 --sub-mat nucl:nucleotide.out,aa:blosum62.out -a 0 --alignment-mode 2 --wrapped-scoring 0 -e 0.001 --min-seq-id 0 --min-aln-len 0 --seq-id-mode 0 --alt-ali 0 -c 0 --cov-mode 0 --max-seq-len 65535 --comp-bias-corr 1 --realign 0 --max-rejected 2147483647 --max-accept 2147483647 --add-self-matches 0 --db-load-mode 0 --pca 1 --pcb 1.5 --score-bias 0 --gap-open nucl:5,aa:11 --gap-extend nucl:2,aa:1 --zdrop 40 --threads 4 --compressed 0 -v 3 \n","\n","Index version: 16\n","Generated by:  16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","ScoreMatrix:  VTML80.out\n","Compute score and coverage\n","Query database size: 108 type: Aminoacid\n","Target database size: 468351 type: Aminoacid\n","Calculation of alignments\n","[=================================================================] 100.00% 108 2s 265ms\n","Time for merging to resultsDB_35: 0h 0m 0s 0ms\n","\n","31501 alignments calculated.\n","14974 sequence pairs passed the thresholds (0.475350 of overall calculated).\n","138.648148 hits per query sequence.\n","Time for processing: 0h 0m 2s 324ms\n","convertalis /content/mmseqs2/tmp/41/candidateDB_35 /content/mmseqs2/genexa/genexaDB /content/mmseqs2/tmp/41/resultsDB_35 /content/mmseqs2/tmp/41/resultDB_genexa.m8 --format-output query,target,evalue,raw \n","\n","MMseqs Version:        \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Substitution matrix    \tnucl:nucleotide.out,aa:blosum62.out\n","Alignment format       \t0\n","Format alignment output\tquery,target,evalue,raw\n","Translation table      \t1\n","Gap open cost          \tnucl:5,aa:11\n","Gap extension cost     \tnucl:2,aa:1\n","Database output        \tfalse\n","Preload mode           \t0\n","Search type            \t0\n","Threads                \t4\n","Compressed             \t0\n","Verbosity              \t3\n","\n","[=================================================================] 100.00% 108 0s 12ms\n","Time for merging to resultDB_genexa.m8: 0h 0m 0s 0ms\n","Time for processing: 0h 0m 0s 75ms\n","search -s 7.0 /content/mmseqs2/tmp/41/candidateDB_53 /content/mmseqs2/swissprot/swissprotDB /content/mmseqs2/tmp/41/resultsDB_53 /content/mmseqs2/tmp/41 \n","\n","MMseqs Version:                     \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Substitution matrix                 \tnucl:nucleotide.out,aa:blosum62.out\n","Add backtrace                       \tfalse\n","Alignment mode                      \t2\n","Allow wrapped scoring               \tfalse\n","E-value threshold                   \t0.001\n","Seq. id. threshold                  \t0\n","Min alignment length                \t0\n","Seq. id. mode                       \t0\n","Alternative alignments              \t0\n","Coverage threshold                  \t0\n","Coverage mode                       \t0\n","Max sequence length                 \t65535\n","Compositional bias                  \t1\n","Realign hits                        \tfalse\n","Max reject                          \t2147483647\n","Max accept                          \t2147483647\n","Include identical seq. id.          \tfalse\n","Preload mode                        \t0\n","Pseudo count a                      \t1\n","Pseudo count b                      \t1.5\n","Score bias                          \t0\n","Gap open cost                       \tnucl:5,aa:11\n","Gap extension cost                  \tnucl:2,aa:1\n","Zdrop                               \t40\n","Threads                             \t4\n","Compressed                          \t0\n","Verbosity                           \t3\n","Seed substitution matrix            \tnucl:nucleotide.out,aa:VTML80.out\n","Sensitivity                         \t7\n","k-mer length                        \t0\n","k-score                             \t2147483647\n","Alphabet size                       \tnucl:5,aa:21\n","Max results per query               \t300\n","Split database                      \t0\n","Split mode                          \t2\n","Split memory limit                  \t0\n","Diagonal scoring                    \ttrue\n","Exact k-mer matching                \t0\n","Mask residues                       \t1\n","Mask lower case residues            \t0\n","Minimum diagonal score              \t15\n","Spaced k-mers                       \t1\n","Spaced k-mer pattern                \t\n","Local temporary path                \t\n","Rescore mode                        \t0\n","Remove hits by seq. id. and coverage\tfalse\n","Sort results                        \t0\n","Mask profile                        \t1\n","Profile e-value threshold           \t0.1\n","Global sequence weighting           \tfalse\n","Allow deletions                     \tfalse\n","Filter MSA                          \t1\n","Maximum seq. id. threshold          \t0.9\n","Minimum seq. id.                    \t0\n","Minimum score per column            \t-20\n","Minimum coverage                    \t0\n","Select N most diverse seqs          \t1000\n","Omit consensus                      \tfalse\n","Min codons in orf                   \t30\n","Max codons in length                \t32734\n","Max orf gaps                        \t2147483647\n","Contig start mode                   \t2\n","Contig end mode                     \t2\n","Orf start mode                      \t1\n","Forward frames                      \t1,2,3\n","Reverse frames                      \t1,2,3\n","Translation table                   \t1\n","Translate orf                       \t0\n","Use all table starts                \tfalse\n","Offset of numeric ids               \t0\n","Create lookup                       \t0\n","Add orf stop                        \tfalse\n","Chain overlapping alignments        \t0\n","Merge query                         \t1\n","Search type                         \t0\n","Search iterations                   \t1\n","Start sensitivity                   \t4\n","Search steps                        \t1\n","Slice search mode                   \tfalse\n","Strand selection                    \t1\n","Disk space limit                    \t0\n","MPI runner                          \t\n","Force restart with latest tmp       \tfalse\n","Remove temporary files              \tfalse\n","\n","prefilter /content/mmseqs2/tmp/41/candidateDB_53 /content/mmseqs2/swissprot/swissprotDB /content/mmseqs2/tmp/41/13740532077838864386/pref_0 --sub-mat nucl:nucleotide.out,aa:blosum62.out --seed-sub-mat nucl:nucleotide.out,aa:VTML80.out -k 0 --k-score 2147483647 --alph-size nucl:5,aa:21 --max-seq-len 65535 --max-seqs 300 --split 0 --split-mode 2 --split-memory-limit 0 -c 0 --cov-mode 0 --comp-bias-corr 1 --diag-score 1 --exact-kmer-matching 0 --mask 1 --mask-lower-case 0 --min-ungapped-score 15 --add-self-matches 0 --spaced-kmer-mode 1 --db-load-mode 0 --pca 1 --pcb 1.5 --threads 4 --compressed 0 -v 3 -s 7.0 \n","\n","Query database size: 108 type: Aminoacid\n","Estimated memory consumption: 2G\n","Target database size: 562755 type: Aminoacid\n","Index table k-mer threshold: 100 at k-mer size 6 \n","Index table: counting k-mers\n","[=================================================================] 100.00% 562.76K 24s 73ms\n","Index table: Masked residues: 3164874\n","Index table: fill\n","[=================================================================] 100.00% 562.76K 17s 790ms\n","Index statistics\n","Entries:          192757622\n","DB size:          1591 MB\n","Avg k-mer size:   3.011838\n","Top 10 k-mers\n","    GPGGTL\t1835\n","    GQSWTV\t1706\n","    WGMFAT\t1638\n","    PGVFEV\t1634\n","    VLWQFW\t1623\n","    AYIRPN\t1587\n","    RSPKGV\t1585\n","    TPHKWY\t1560\n","    HIRVGW\t1548\n","    ITLSPY\t1541\n","Time for index table init: 0h 0m 43s 991ms\n","Process prefiltering step 1 of 1\n","\n","k-mer similarity threshold: 100\n","Starting prefiltering scores calculation (step 1 of 1)\n","Query db start 1 to 108\n","Target db start 1 to 562755\n","[=================================================================] 100.00% 108 1s 732ms\n","\n","922.504061 k-mers per position\n","1282508 DB matches per sequence\n","23 overflows\n","0 queries produce too many hits (truncated result)\n","293 sequences passed prefiltering per query sequence\n","300 median result list length\n","0 sequences with 0 size result lists\n","Time for merging to pref_0: 0h 0m 0s 0ms\n","Time for processing: 0h 0m 46s 832ms\n","align /content/mmseqs2/tmp/41/candidateDB_53 /content/mmseqs2/swissprot/swissprotDB /content/mmseqs2/tmp/41/13740532077838864386/pref_0 /content/mmseqs2/tmp/41/resultsDB_53 --sub-mat nucl:nucleotide.out,aa:blosum62.out -a 0 --alignment-mode 2 --wrapped-scoring 0 -e 0.001 --min-seq-id 0 --min-aln-len 0 --seq-id-mode 0 --alt-ali 0 -c 0 --cov-mode 0 --max-seq-len 65535 --comp-bias-corr 1 --realign 0 --max-rejected 2147483647 --max-accept 2147483647 --add-self-matches 0 --db-load-mode 0 --pca 1 --pcb 1.5 --score-bias 0 --gap-open nucl:5,aa:11 --gap-extend nucl:2,aa:1 --zdrop 40 --threads 4 --compressed 0 -v 3 \n","\n","Compute score and coverage\n","Query database size: 108 type: Aminoacid\n","Target database size: 562755 type: Aminoacid\n","Calculation of alignments\n","[=================================================================] 100.00% 108 1s 427ms\n","Time for merging to resultsDB_53: 0h 0m 0s 0ms\n","\n","31718 alignments calculated.\n","8896 sequence pairs passed the thresholds (0.280472 of overall calculated).\n","82.370369 hits per query sequence.\n","Time for processing: 0h 0m 1s 480ms\n","convertalis /content/mmseqs2/tmp/41/candidateDB_53 /content/mmseqs2/swissprot/swissprotDB /content/mmseqs2/tmp/41/resultsDB_53 /content/mmseqs2/tmp/41/resultDB_secondary.m8 --format-output query,target,evalue,raw \n","\n","MMseqs Version:        \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Substitution matrix    \tnucl:nucleotide.out,aa:blosum62.out\n","Alignment format       \t0\n","Format alignment output\tquery,target,evalue,raw\n","Translation table      \t1\n","Gap open cost          \tnucl:5,aa:11\n","Gap extension cost     \tnucl:2,aa:1\n","Database output        \tfalse\n","Preload mode           \t0\n","Search type            \t0\n","Threads                \t4\n","Compressed             \t0\n","Verbosity              \t3\n","\n","[=================================================================] 100.00% 108 0s 8ms\n","Time for merging to resultDB_secondary.m8: 0h 0m 0s 0ms\n","Time for processing: 0h 0m 0s 91ms\n","found 95 genes\n","\n","\n","Creating weighted directed acyclic graph of all ORFs and subORFs on contig 2...\n","\n"],"name":"stdout"},{"output_type":"display_data","data":{"application/vnd.jupyter.widget-view+json":{"model_id":"6289935e919349d9b32246b8693b0705","version_minor":0,"version_major":2},"text/plain":["HBox(children=(FloatProgress(value=0.0, max=241.0), HTML(value='')))"]},"metadata":{"tags":[]}},{"output_type":"stream","text":["\n","Maximizing geneiness...\n","\n","Filtering predictions with mmseqs2...\n","createdb /content/mmseqs2/tmp/42/candidate_genes_35.fasta /content/mmseqs2/tmp/42/candidateDB_35 \n","\n","MMseqs Version:       \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Database type         \t0\n","Shuffle input database\ttrue\n","Createdb mode         \t0\n","Write lookup file     \t1\n","Offset of numeric ids \t0\n","Compressed            \t0\n","Verbosity             \t3\n","\n","Converting sequences\n","\n","Time for merging to candidateDB_35_h: 0h 0m 0s 1ms\n","Time for merging to candidateDB_35: 0h 0m 0s 1ms\n","Database type: Aminoacid\n","Time for processing: 0h 0m 0s 7ms\n","createdb /content/mmseqs2/tmp/42/candidate_genes_53.fasta /content/mmseqs2/tmp/42/candidateDB_53 \n","\n","MMseqs Version:       \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Database type         \t0\n","Shuffle input database\ttrue\n","Createdb mode         \t0\n","Write lookup file     \t1\n","Offset of numeric ids \t0\n","Compressed            \t0\n","Verbosity             \t3\n","\n","Converting sequences\n","\n","Time for merging to candidateDB_53_h: 0h 0m 0s 1ms\n","Time for merging to candidateDB_53: 0h 0m 0s 1ms\n","Database type: Aminoacid\n","Time for processing: 0h 0m 0s 7ms\n","search -s 7.0 /content/mmseqs2/tmp/42/candidateDB_35 /content/mmseqs2/genexa/genexaDB /content/mmseqs2/tmp/42/resultsDB_35 /content/mmseqs2/tmp/42 \n","\n","MMseqs Version:                     \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Substitution matrix                 \tnucl:nucleotide.out,aa:blosum62.out\n","Add backtrace                       \tfalse\n","Alignment mode                      \t2\n","Allow wrapped scoring               \tfalse\n","E-value threshold                   \t0.001\n","Seq. id. threshold                  \t0\n","Min alignment length                \t0\n","Seq. id. mode                       \t0\n","Alternative alignments              \t0\n","Coverage threshold                  \t0\n","Coverage mode                       \t0\n","Max sequence length                 \t65535\n","Compositional bias                  \t1\n","Realign hits                        \tfalse\n","Max reject                          \t2147483647\n","Max accept                          \t2147483647\n","Include identical seq. id.          \tfalse\n","Preload mode                        \t0\n","Pseudo count a                      \t1\n","Pseudo count b                      \t1.5\n","Score bias                          \t0\n","Gap open cost                       \tnucl:5,aa:11\n","Gap extension cost                  \tnucl:2,aa:1\n","Zdrop                               \t40\n","Threads                             \t4\n","Compressed                          \t0\n","Verbosity                           \t3\n","Seed substitution matrix            \tnucl:nucleotide.out,aa:VTML80.out\n","Sensitivity                         \t7\n","k-mer length                        \t0\n","k-score                             \t2147483647\n","Alphabet size                       \tnucl:5,aa:21\n","Max results per query               \t300\n","Split database                      \t0\n","Split mode                          \t2\n","Split memory limit                  \t0\n","Diagonal scoring                    \ttrue\n","Exact k-mer matching                \t0\n","Mask residues                       \t1\n","Mask lower case residues            \t0\n","Minimum diagonal score              \t15\n","Spaced k-mers                       \t1\n","Spaced k-mer pattern                \t\n","Local temporary path                \t\n","Rescore mode                        \t0\n","Remove hits by seq. id. and coverage\tfalse\n","Sort results                        \t0\n","Mask profile                        \t1\n","Profile e-value threshold           \t0.1\n","Global sequence weighting           \tfalse\n","Allow deletions                     \tfalse\n","Filter MSA                          \t1\n","Maximum seq. id. threshold          \t0.9\n","Minimum seq. id.                    \t0\n","Minimum score per column            \t-20\n","Minimum coverage                    \t0\n","Select N most diverse seqs          \t1000\n","Omit consensus                      \tfalse\n","Min codons in orf                   \t30\n","Max codons in length                \t32734\n","Max orf gaps                        \t2147483647\n","Contig start mode                   \t2\n","Contig end mode                     \t2\n","Orf start mode                      \t1\n","Forward frames                      \t1,2,3\n","Reverse frames                      \t1,2,3\n","Translation table                   \t1\n","Translate orf                       \t0\n","Use all table starts                \tfalse\n","Offset of numeric ids               \t0\n","Create lookup                       \t0\n","Add orf stop                        \tfalse\n","Chain overlapping alignments        \t0\n","Merge query                         \t1\n","Search type                         \t0\n","Search iterations                   \t1\n","Start sensitivity                   \t4\n","Search steps                        \t1\n","Slice search mode                   \tfalse\n","Strand selection                    \t1\n","Disk space limit                    \t0\n","MPI runner                          \t\n","Force restart with latest tmp       \tfalse\n","Remove temporary files              \tfalse\n","\n","prefilter /content/mmseqs2/tmp/42/candidateDB_35 /content/mmseqs2/genexa/genexaDB.idx /content/mmseqs2/tmp/42/5595201572742321251/pref_0 --sub-mat nucl:nucleotide.out,aa:blosum62.out --seed-sub-mat nucl:nucleotide.out,aa:VTML80.out -k 0 --k-score 2147483647 --alph-size nucl:5,aa:21 --max-seq-len 65535 --max-seqs 300 --split 0 --split-mode 2 --split-memory-limit 0 -c 0 --cov-mode 0 --comp-bias-corr 1 --diag-score 1 --exact-kmer-matching 0 --mask 1 --mask-lower-case 0 --min-ungapped-score 15 --add-self-matches 0 --spaced-kmer-mode 1 --db-load-mode 0 --pca 1 --pcb 1.5 --threads 4 --compressed 0 -v 3 -s 7.0 \n","\n","Index version: 16\n","Generated by:  16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","ScoreMatrix:  VTML80.out\n","Query database size: 63 type: Aminoacid\n","Estimated memory consumption: 2G\n","Target database size: 468351 type: Aminoacid\n","Process prefiltering step 1 of 1\n","\n","k-mer similarity threshold: 100\n","Starting prefiltering scores calculation (step 1 of 1)\n","Query db start 1 to 63\n","Target db start 1 to 468351\n","[=================================================================] 100.00% 63 0s 908ms\n","\n","983.576479 k-mers per position\n","1154228 DB matches per sequence\n","13 overflows\n","0 queries produce too many hits (truncated result)\n","295 sequences passed prefiltering per query sequence\n","300 median result list length\n","0 sequences with 0 size result lists\n","Time for merging to pref_0: 0h 0m 0s 0ms\n","Time for processing: 0h 0m 2s 399ms\n","align /content/mmseqs2/tmp/42/candidateDB_35 /content/mmseqs2/genexa/genexaDB.idx /content/mmseqs2/tmp/42/5595201572742321251/pref_0 /content/mmseqs2/tmp/42/resultsDB_35 --sub-mat nucl:nucleotide.out,aa:blosum62.out -a 0 --alignment-mode 2 --wrapped-scoring 0 -e 0.001 --min-seq-id 0 --min-aln-len 0 --seq-id-mode 0 --alt-ali 0 -c 0 --cov-mode 0 --max-seq-len 65535 --comp-bias-corr 1 --realign 0 --max-rejected 2147483647 --max-accept 2147483647 --add-self-matches 0 --db-load-mode 0 --pca 1 --pcb 1.5 --score-bias 0 --gap-open nucl:5,aa:11 --gap-extend nucl:2,aa:1 --zdrop 40 --threads 4 --compressed 0 -v 3 \n","\n","Index version: 16\n","Generated by:  16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","ScoreMatrix:  VTML80.out\n","Compute score and coverage\n","Query database size: 63 type: Aminoacid\n","Target database size: 468351 type: Aminoacid\n","Calculation of alignments\n","[=================================================================] 100.00% 63 1s 268ms\n","Time for merging to resultsDB_35: 0h 0m 0s 0ms\n","\n","18628 alignments calculated.\n","7291 sequence pairs passed the thresholds (0.391400 of overall calculated).\n","115.730156 hits per query sequence.\n","Time for processing: 0h 0m 1s 324ms\n","convertalis /content/mmseqs2/tmp/42/candidateDB_35 /content/mmseqs2/genexa/genexaDB /content/mmseqs2/tmp/42/resultsDB_35 /content/mmseqs2/tmp/42/resultDB_genexa.m8 --format-output query,target,evalue,raw \n","\n","MMseqs Version:        \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Substitution matrix    \tnucl:nucleotide.out,aa:blosum62.out\n","Alignment format       \t0\n","Format alignment output\tquery,target,evalue,raw\n","Translation table      \t1\n","Gap open cost          \tnucl:5,aa:11\n","Gap extension cost     \tnucl:2,aa:1\n","Database output        \tfalse\n","Preload mode           \t0\n","Search type            \t0\n","Threads                \t4\n","Compressed             \t0\n","Verbosity              \t3\n","\n","[=================================================================] 100.00% 63 0s 8ms\n","Time for merging to resultDB_genexa.m8: 0h 0m 0s 0ms\n","Time for processing: 0h 0m 0s 70ms\n","search -s 7.0 /content/mmseqs2/tmp/42/candidateDB_53 /content/mmseqs2/swissprot/swissprotDB /content/mmseqs2/tmp/42/resultsDB_53 /content/mmseqs2/tmp/42 \n","\n","MMseqs Version:                     \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Substitution matrix                 \tnucl:nucleotide.out,aa:blosum62.out\n","Add backtrace                       \tfalse\n","Alignment mode                      \t2\n","Allow wrapped scoring               \tfalse\n","E-value threshold                   \t0.001\n","Seq. id. threshold                  \t0\n","Min alignment length                \t0\n","Seq. id. mode                       \t0\n","Alternative alignments              \t0\n","Coverage threshold                  \t0\n","Coverage mode                       \t0\n","Max sequence length                 \t65535\n","Compositional bias                  \t1\n","Realign hits                        \tfalse\n","Max reject                          \t2147483647\n","Max accept                          \t2147483647\n","Include identical seq. id.          \tfalse\n","Preload mode                        \t0\n","Pseudo count a                      \t1\n","Pseudo count b                      \t1.5\n","Score bias                          \t0\n","Gap open cost                       \tnucl:5,aa:11\n","Gap extension cost                  \tnucl:2,aa:1\n","Zdrop                               \t40\n","Threads                             \t4\n","Compressed                          \t0\n","Verbosity                           \t3\n","Seed substitution matrix            \tnucl:nucleotide.out,aa:VTML80.out\n","Sensitivity                         \t7\n","k-mer length                        \t0\n","k-score                             \t2147483647\n","Alphabet size                       \tnucl:5,aa:21\n","Max results per query               \t300\n","Split database                      \t0\n","Split mode                          \t2\n","Split memory limit                  \t0\n","Diagonal scoring                    \ttrue\n","Exact k-mer matching                \t0\n","Mask residues                       \t1\n","Mask lower case residues            \t0\n","Minimum diagonal score              \t15\n","Spaced k-mers                       \t1\n","Spaced k-mer pattern                \t\n","Local temporary path                \t\n","Rescore mode                        \t0\n","Remove hits by seq. id. and coverage\tfalse\n","Sort results                        \t0\n","Mask profile                        \t1\n","Profile e-value threshold           \t0.1\n","Global sequence weighting           \tfalse\n","Allow deletions                     \tfalse\n","Filter MSA                          \t1\n","Maximum seq. id. threshold          \t0.9\n","Minimum seq. id.                    \t0\n","Minimum score per column            \t-20\n","Minimum coverage                    \t0\n","Select N most diverse seqs          \t1000\n","Omit consensus                      \tfalse\n","Min codons in orf                   \t30\n","Max codons in length                \t32734\n","Max orf gaps                        \t2147483647\n","Contig start mode                   \t2\n","Contig end mode                     \t2\n","Orf start mode                      \t1\n","Forward frames                      \t1,2,3\n","Reverse frames                      \t1,2,3\n","Translation table                   \t1\n","Translate orf                       \t0\n","Use all table starts                \tfalse\n","Offset of numeric ids               \t0\n","Create lookup                       \t0\n","Add orf stop                        \tfalse\n","Chain overlapping alignments        \t0\n","Merge query                         \t1\n","Search type                         \t0\n","Search iterations                   \t1\n","Start sensitivity                   \t4\n","Search steps                        \t1\n","Slice search mode                   \tfalse\n","Strand selection                    \t1\n","Disk space limit                    \t0\n","MPI runner                          \t\n","Force restart with latest tmp       \tfalse\n","Remove temporary files              \tfalse\n","\n","prefilter /content/mmseqs2/tmp/42/candidateDB_53 /content/mmseqs2/swissprot/swissprotDB /content/mmseqs2/tmp/42/13841897065884210369/pref_0 --sub-mat nucl:nucleotide.out,aa:blosum62.out --seed-sub-mat nucl:nucleotide.out,aa:VTML80.out -k 0 --k-score 2147483647 --alph-size nucl:5,aa:21 --max-seq-len 65535 --max-seqs 300 --split 0 --split-mode 2 --split-memory-limit 0 -c 0 --cov-mode 0 --comp-bias-corr 1 --diag-score 1 --exact-kmer-matching 0 --mask 1 --mask-lower-case 0 --min-ungapped-score 15 --add-self-matches 0 --spaced-kmer-mode 1 --db-load-mode 0 --pca 1 --pcb 1.5 --threads 4 --compressed 0 -v 3 -s 7.0 \n","\n","Query database size: 63 type: Aminoacid\n","Estimated memory consumption: 2G\n","Target database size: 562755 type: Aminoacid\n","Index table k-mer threshold: 100 at k-mer size 6 \n","Index table: counting k-mers\n","[=================================================================] 100.00% 562.76K 24s 51ms\n","Index table: Masked residues: 3164874\n","Index table: fill\n","[=================================================================] 100.00% 562.76K 17s 757ms\n","Index statistics\n","Entries:          192757622\n","DB size:          1591 MB\n","Avg k-mer size:   3.011838\n","Top 10 k-mers\n","    GPGGTL\t1835\n","    GQSWTV\t1706\n","    WGMFAT\t1638\n","    PGVFEV\t1634\n","    VLWQFW\t1623\n","    AYIRPN\t1587\n","    RSPKGV\t1585\n","    TPHKWY\t1560\n","    HIRVGW\t1548\n","    ITLSPY\t1541\n","Time for index table init: 0h 0m 43s 842ms\n","Process prefiltering step 1 of 1\n","\n","k-mer similarity threshold: 100\n","Starting prefiltering scores calculation (step 1 of 1)\n","Query db start 1 to 63\n","Target db start 1 to 562755\n","[=================================================================] 100.00% 63 1s 55ms\n","\n","977.489835 k-mers per position\n","1435375 DB matches per sequence\n","18 overflows\n","0 queries produce too many hits (truncated result)\n","298 sequences passed prefiltering per query sequence\n","300 median result list length\n","0 sequences with 0 size result lists\n","Time for merging to pref_0: 0h 0m 0s 0ms\n","Time for processing: 0h 0m 46s 83ms\n","align /content/mmseqs2/tmp/42/candidateDB_53 /content/mmseqs2/swissprot/swissprotDB /content/mmseqs2/tmp/42/13841897065884210369/pref_0 /content/mmseqs2/tmp/42/resultsDB_53 --sub-mat nucl:nucleotide.out,aa:blosum62.out -a 0 --alignment-mode 2 --wrapped-scoring 0 -e 0.001 --min-seq-id 0 --min-aln-len 0 --seq-id-mode 0 --alt-ali 0 -c 0 --cov-mode 0 --max-seq-len 65535 --comp-bias-corr 1 --realign 0 --max-rejected 2147483647 --max-accept 2147483647 --add-self-matches 0 --db-load-mode 0 --pca 1 --pcb 1.5 --score-bias 0 --gap-open nucl:5,aa:11 --gap-extend nucl:2,aa:1 --zdrop 40 --threads 4 --compressed 0 -v 3 \n","\n","Compute score and coverage\n","Query database size: 63 type: Aminoacid\n","Target database size: 562755 type: Aminoacid\n","Calculation of alignments\n","[=================================================================] 100.00% 63 0s 664ms\n","Time for merging to resultsDB_53: 0h 0m 0s 0ms\n","\n","18784 alignments calculated.\n","2749 sequence pairs passed the thresholds (0.146348 of overall calculated).\n","43.634922 hits per query sequence.\n","Time for processing: 0h 0m 0s 771ms\n","convertalis /content/mmseqs2/tmp/42/candidateDB_53 /content/mmseqs2/swissprot/swissprotDB /content/mmseqs2/tmp/42/resultsDB_53 /content/mmseqs2/tmp/42/resultDB_secondary.m8 --format-output query,target,evalue,raw \n","\n","MMseqs Version:        \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Substitution matrix    \tnucl:nucleotide.out,aa:blosum62.out\n","Alignment format       \t0\n","Format alignment output\tquery,target,evalue,raw\n","Translation table      \t1\n","Gap open cost          \tnucl:5,aa:11\n","Gap extension cost     \tnucl:2,aa:1\n","Database output        \tfalse\n","Preload mode           \t0\n","Search type            \t0\n","Threads                \t4\n","Compressed             \t0\n","Verbosity              \t3\n","\n","[=================================================================] 100.00% 63 0s 3ms\n","Time for merging to resultDB_secondary.m8: 0h 0m 0s 0ms\n","Time for processing: 0h 0m 0s 87ms\n","found 56 genes\n","\n","\n","Reading fasta file GCF_000009905.1_ASM990v1_genomic.fna.gz...\n","\n","Finding and translating open reading frames...\n","\n","Encoding amino acids...\n","\n","Applying kmer prefilter...\n","\n","Scoring ORFs with temporal convolutional network...\n","\n"],"name":"stdout"},{"output_type":"display_data","data":{"application/vnd.jupyter.widget-view+json":{"model_id":"e20337dbc1894121baf459979ae468ce","version_minor":0,"version_major":2},"text/plain":["HBox(children=(FloatProgress(value=0.0, max=66.0), HTML(value='')))"]},"metadata":{"tags":[]}},{"output_type":"stream","text":["\n","Scoring translation initiation sites...\n","\n"],"name":"stdout"},{"output_type":"display_data","data":{"application/vnd.jupyter.widget-view+json":{"model_id":"eb7d892bcef947a0aeeb415db7478c83","version_minor":0,"version_major":2},"text/plain":["HBox(children=(FloatProgress(value=0.0, max=20.0), HTML(value='')))"]},"metadata":{"tags":[]}},{"output_type":"stream","text":["\n","Creating weighted directed acyclic graph of all ORFs and subORFs on contig 0...\n","\n"],"name":"stdout"},{"output_type":"display_data","data":{"application/vnd.jupyter.widget-view+json":{"model_id":"24c0025637e240a889d0edbb570a461c","version_minor":0,"version_major":2},"text/plain":["HBox(children=(FloatProgress(value=0.0, max=195625.0), HTML(value='')))"]},"metadata":{"tags":[]}},{"output_type":"stream","text":["\n","Maximizing geneiness...\n","\n","Filtering predictions with mmseqs2...\n","createdb /content/mmseqs2/tmp/43/candidate_genes_35.fasta /content/mmseqs2/tmp/43/candidateDB_35 \n","\n","MMseqs Version:       \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Database type         \t0\n","Shuffle input database\ttrue\n","Createdb mode         \t0\n","Write lookup file     \t1\n","Offset of numeric ids \t0\n","Compressed            \t0\n","Verbosity             \t3\n","\n","Converting sequences\n","[5455] 0s 8ms\n","Time for merging to candidateDB_35_h: 0h 0m 0s 3ms\n","Time for merging to candidateDB_35: 0h 0m 0s 4ms\n","Database type: Aminoacid\n","Time for processing: 0h 0m 0s 24ms\n","createdb /content/mmseqs2/tmp/43/candidate_genes_53.fasta /content/mmseqs2/tmp/43/candidateDB_53 \n","\n","MMseqs Version:       \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Database type         \t0\n","Shuffle input database\ttrue\n","Createdb mode         \t0\n","Write lookup file     \t1\n","Offset of numeric ids \t0\n","Compressed            \t0\n","Verbosity             \t3\n","\n","Converting sequences\n","[5455] 0s 8ms\n","Time for merging to candidateDB_53_h: 0h 0m 0s 3ms\n","Time for merging to candidateDB_53: 0h 0m 0s 5ms\n","Database type: Aminoacid\n","Time for processing: 0h 0m 0s 24ms\n","search -s 7.0 /content/mmseqs2/tmp/43/candidateDB_35 /content/mmseqs2/genexa/genexaDB /content/mmseqs2/tmp/43/resultsDB_35 /content/mmseqs2/tmp/43 \n","\n","MMseqs Version:                     \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Substitution matrix                 \tnucl:nucleotide.out,aa:blosum62.out\n","Add backtrace                       \tfalse\n","Alignment mode                      \t2\n","Allow wrapped scoring               \tfalse\n","E-value threshold                   \t0.001\n","Seq. id. threshold                  \t0\n","Min alignment length                \t0\n","Seq. id. mode                       \t0\n","Alternative alignments              \t0\n","Coverage threshold                  \t0\n","Coverage mode                       \t0\n","Max sequence length                 \t65535\n","Compositional bias                  \t1\n","Realign hits                        \tfalse\n","Max reject                          \t2147483647\n","Max accept                          \t2147483647\n","Include identical seq. id.          \tfalse\n","Preload mode                        \t0\n","Pseudo count a                      \t1\n","Pseudo count b                      \t1.5\n","Score bias                          \t0\n","Gap open cost                       \tnucl:5,aa:11\n","Gap extension cost                  \tnucl:2,aa:1\n","Zdrop                               \t40\n","Threads                             \t4\n","Compressed                          \t0\n","Verbosity                           \t3\n","Seed substitution matrix            \tnucl:nucleotide.out,aa:VTML80.out\n","Sensitivity                         \t7\n","k-mer length                        \t0\n","k-score                             \t2147483647\n","Alphabet size                       \tnucl:5,aa:21\n","Max results per query               \t300\n","Split database                      \t0\n","Split mode                          \t2\n","Split memory limit                  \t0\n","Diagonal scoring                    \ttrue\n","Exact k-mer matching                \t0\n","Mask residues                       \t1\n","Mask lower case residues            \t0\n","Minimum diagonal score              \t15\n","Spaced k-mers                       \t1\n","Spaced k-mer pattern                \t\n","Local temporary path                \t\n","Rescore mode                        \t0\n","Remove hits by seq. id. and coverage\tfalse\n","Sort results                        \t0\n","Mask profile                        \t1\n","Profile e-value threshold           \t0.1\n","Global sequence weighting           \tfalse\n","Allow deletions                     \tfalse\n","Filter MSA                          \t1\n","Maximum seq. id. threshold          \t0.9\n","Minimum seq. id.                    \t0\n","Minimum score per column            \t-20\n","Minimum coverage                    \t0\n","Select N most diverse seqs          \t1000\n","Omit consensus                      \tfalse\n","Min codons in orf                   \t30\n","Max codons in length                \t32734\n","Max orf gaps                        \t2147483647\n","Contig start mode                   \t2\n","Contig end mode                     \t2\n","Orf start mode                      \t1\n","Forward frames                      \t1,2,3\n","Reverse frames                      \t1,2,3\n","Translation table                   \t1\n","Translate orf                       \t0\n","Use all table starts                \tfalse\n","Offset of numeric ids               \t0\n","Create lookup                       \t0\n","Add orf stop                        \tfalse\n","Chain overlapping alignments        \t0\n","Merge query                         \t1\n","Search type                         \t0\n","Search iterations                   \t1\n","Start sensitivity                   \t4\n","Search steps                        \t1\n","Slice search mode                   \tfalse\n","Strand selection                    \t1\n","Disk space limit                    \t0\n","MPI runner                          \t\n","Force restart with latest tmp       \tfalse\n","Remove temporary files              \tfalse\n","\n","prefilter /content/mmseqs2/tmp/43/candidateDB_35 /content/mmseqs2/genexa/genexaDB.idx /content/mmseqs2/tmp/43/334915469632625762/pref_0 --sub-mat nucl:nucleotide.out,aa:blosum62.out --seed-sub-mat nucl:nucleotide.out,aa:VTML80.out -k 0 --k-score 2147483647 --alph-size nucl:5,aa:21 --max-seq-len 65535 --max-seqs 300 --split 0 --split-mode 2 --split-memory-limit 0 -c 0 --cov-mode 0 --comp-bias-corr 1 --diag-score 1 --exact-kmer-matching 0 --mask 1 --mask-lower-case 0 --min-ungapped-score 15 --add-self-matches 0 --spaced-kmer-mode 1 --db-load-mode 0 --pca 1 --pcb 1.5 --threads 4 --compressed 0 -v 3 -s 7.0 \n","\n","Index version: 16\n","Generated by:  16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","ScoreMatrix:  VTML80.out\n","Query database size: 5495 type: Aminoacid\n","Estimated memory consumption: 2G\n","Target database size: 468351 type: Aminoacid\n","Process prefiltering step 1 of 1\n","\n","k-mer similarity threshold: 100\n","Starting prefiltering scores calculation (step 1 of 1)\n","Query db start 1 to 5495\n","Target db start 1 to 468351\n","[=================================================================] 100.00% 5.49K 38s 243ms\n","\n","837.529040 k-mers per position\n","576251 DB matches per sequence\n","190 overflows\n","0 queries produce too many hits (truncated result)\n","273 sequences passed prefiltering per query sequence\n","300 median result list length\n","5 sequences with 0 size result lists\n","Time for merging to pref_0: 0h 0m 0s 1ms\n","Time for processing: 0h 0m 39s 682ms\n","align /content/mmseqs2/tmp/43/candidateDB_35 /content/mmseqs2/genexa/genexaDB.idx /content/mmseqs2/tmp/43/334915469632625762/pref_0 /content/mmseqs2/tmp/43/resultsDB_35 --sub-mat nucl:nucleotide.out,aa:blosum62.out -a 0 --alignment-mode 2 --wrapped-scoring 0 -e 0.001 --min-seq-id 0 --min-aln-len 0 --seq-id-mode 0 --alt-ali 0 -c 0 --cov-mode 0 --max-seq-len 65535 --comp-bias-corr 1 --realign 0 --max-rejected 2147483647 --max-accept 2147483647 --add-self-matches 0 --db-load-mode 0 --pca 1 --pcb 1.5 --score-bias 0 --gap-open nucl:5,aa:11 --gap-extend nucl:2,aa:1 --zdrop 40 --threads 4 --compressed 0 -v 3 \n","\n","Index version: 16\n","Generated by:  16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","ScoreMatrix:  VTML80.out\n","Compute score and coverage\n","Query database size: 5495 type: Aminoacid\n","Target database size: 468351 type: Aminoacid\n","Calculation of alignments\n","[=================================================================] 100.00% 5.49K 34s 142ms\n","Time for merging to resultsDB_35: 0h 0m 0s 1ms\n","\n","1504591 alignments calculated.\n","396395 sequence pairs passed the thresholds (0.263457 of overall calculated).\n","72.137398 hits per query sequence.\n","Time for processing: 0h 0m 34s 242ms\n","convertalis /content/mmseqs2/tmp/43/candidateDB_35 /content/mmseqs2/genexa/genexaDB /content/mmseqs2/tmp/43/resultsDB_35 /content/mmseqs2/tmp/43/resultDB_genexa.m8 --format-output query,target,evalue,raw \n","\n","MMseqs Version:        \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Substitution matrix    \tnucl:nucleotide.out,aa:blosum62.out\n","Alignment format       \t0\n","Format alignment output\tquery,target,evalue,raw\n","Translation table      \t1\n","Gap open cost          \tnucl:5,aa:11\n","Gap extension cost     \tnucl:2,aa:1\n","Database output        \tfalse\n","Preload mode           \t0\n","Search type            \t0\n","Threads                \t4\n","Compressed             \t0\n","Verbosity              \t3\n","\n","[=================================================================] 100.00% 5.49K 0s 288ms\n","Time for merging to resultDB_genexa.m8: 0h 0m 0s 10ms\n","Time for processing: 0h 0m 0s 368ms\n","search -s 7.0 /content/mmseqs2/tmp/43/candidateDB_53 /content/mmseqs2/swissprot/swissprotDB /content/mmseqs2/tmp/43/resultsDB_53 /content/mmseqs2/tmp/43 \n","\n","MMseqs Version:                     \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Substitution matrix                 \tnucl:nucleotide.out,aa:blosum62.out\n","Add backtrace                       \tfalse\n","Alignment mode                      \t2\n","Allow wrapped scoring               \tfalse\n","E-value threshold                   \t0.001\n","Seq. id. threshold                  \t0\n","Min alignment length                \t0\n","Seq. id. mode                       \t0\n","Alternative alignments              \t0\n","Coverage threshold                  \t0\n","Coverage mode                       \t0\n","Max sequence length                 \t65535\n","Compositional bias                  \t1\n","Realign hits                        \tfalse\n","Max reject                          \t2147483647\n","Max accept                          \t2147483647\n","Include identical seq. id.          \tfalse\n","Preload mode                        \t0\n","Pseudo count a                      \t1\n","Pseudo count b                      \t1.5\n","Score bias                          \t0\n","Gap open cost                       \tnucl:5,aa:11\n","Gap extension cost                  \tnucl:2,aa:1\n","Zdrop                               \t40\n","Threads                             \t4\n","Compressed                          \t0\n","Verbosity                           \t3\n","Seed substitution matrix            \tnucl:nucleotide.out,aa:VTML80.out\n","Sensitivity                         \t7\n","k-mer length                        \t0\n","k-score                             \t2147483647\n","Alphabet size                       \tnucl:5,aa:21\n","Max results per query               \t300\n","Split database                      \t0\n","Split mode                          \t2\n","Split memory limit                  \t0\n","Diagonal scoring                    \ttrue\n","Exact k-mer matching                \t0\n","Mask residues                       \t1\n","Mask lower case residues            \t0\n","Minimum diagonal score              \t15\n","Spaced k-mers                       \t1\n","Spaced k-mer pattern                \t\n","Local temporary path                \t\n","Rescore mode                        \t0\n","Remove hits by seq. id. and coverage\tfalse\n","Sort results                        \t0\n","Mask profile                        \t1\n","Profile e-value threshold           \t0.1\n","Global sequence weighting           \tfalse\n","Allow deletions                     \tfalse\n","Filter MSA                          \t1\n","Maximum seq. id. threshold          \t0.9\n","Minimum seq. id.                    \t0\n","Minimum score per column            \t-20\n","Minimum coverage                    \t0\n","Select N most diverse seqs          \t1000\n","Omit consensus                      \tfalse\n","Min codons in orf                   \t30\n","Max codons in length                \t32734\n","Max orf gaps                        \t2147483647\n","Contig start mode                   \t2\n","Contig end mode                     \t2\n","Orf start mode                      \t1\n","Forward frames                      \t1,2,3\n","Reverse frames                      \t1,2,3\n","Translation table                   \t1\n","Translate orf                       \t0\n","Use all table starts                \tfalse\n","Offset of numeric ids               \t0\n","Create lookup                       \t0\n","Add orf stop                        \tfalse\n","Chain overlapping alignments        \t0\n","Merge query                         \t1\n","Search type                         \t0\n","Search iterations                   \t1\n","Start sensitivity                   \t4\n","Search steps                        \t1\n","Slice search mode                   \tfalse\n","Strand selection                    \t1\n","Disk space limit                    \t0\n","MPI runner                          \t\n","Force restart with latest tmp       \tfalse\n","Remove temporary files              \tfalse\n","\n","prefilter /content/mmseqs2/tmp/43/candidateDB_53 /content/mmseqs2/swissprot/swissprotDB /content/mmseqs2/tmp/43/13943262053929556352/pref_0 --sub-mat nucl:nucleotide.out,aa:blosum62.out --seed-sub-mat nucl:nucleotide.out,aa:VTML80.out -k 0 --k-score 2147483647 --alph-size nucl:5,aa:21 --max-seq-len 65535 --max-seqs 300 --split 0 --split-mode 2 --split-memory-limit 0 -c 0 --cov-mode 0 --comp-bias-corr 1 --diag-score 1 --exact-kmer-matching 0 --mask 1 --mask-lower-case 0 --min-ungapped-score 15 --add-self-matches 0 --spaced-kmer-mode 1 --db-load-mode 0 --pca 1 --pcb 1.5 --threads 4 --compressed 0 -v 3 -s 7.0 \n","\n","Query database size: 5495 type: Aminoacid\n","Estimated memory consumption: 2G\n","Target database size: 562755 type: Aminoacid\n","Index table k-mer threshold: 100 at k-mer size 6 \n","Index table: counting k-mers\n","[=================================================================] 100.00% 562.76K 24s 132ms\n","Index table: Masked residues: 3164874\n","Index table: fill\n","[=================================================================] 100.00% 562.76K 17s 663ms\n","Index statistics\n","Entries:          192757622\n","DB size:          1591 MB\n","Avg k-mer size:   3.011838\n","Top 10 k-mers\n","    GPGGTL\t1835\n","    GQSWTV\t1706\n","    WGMFAT\t1638\n","    PGVFEV\t1634\n","    VLWQFW\t1623\n","    AYIRPN\t1587\n","    RSPKGV\t1585\n","    TPHKWY\t1560\n","    HIRVGW\t1548\n","    ITLSPY\t1541\n","Time for index table init: 0h 0m 43s 889ms\n","Process prefiltering step 1 of 1\n","\n","k-mer similarity threshold: 100\n","Starting prefiltering scores calculation (step 1 of 1)\n","Query db start 1 to 5495\n","Target db start 1 to 562755\n","[=================================================================] 100.00% 5.49K 41s 335ms\n","\n","837.402086 k-mers per position\n","650139 DB matches per sequence\n","286 overflows\n","0 queries produce too many hits (truncated result)\n","279 sequences passed prefiltering per query sequence\n","300 median result list length\n","4 sequences with 0 size result lists\n","Time for merging to pref_0: 0h 0m 0s 1ms\n","Time for processing: 0h 1m 26s 357ms\n","align /content/mmseqs2/tmp/43/candidateDB_53 /content/mmseqs2/swissprot/swissprotDB /content/mmseqs2/tmp/43/13943262053929556352/pref_0 /content/mmseqs2/tmp/43/resultsDB_53 --sub-mat nucl:nucleotide.out,aa:blosum62.out -a 0 --alignment-mode 2 --wrapped-scoring 0 -e 0.001 --min-seq-id 0 --min-aln-len 0 --seq-id-mode 0 --alt-ali 0 -c 0 --cov-mode 0 --max-seq-len 65535 --comp-bias-corr 1 --realign 0 --max-rejected 2147483647 --max-accept 2147483647 --add-self-matches 0 --db-load-mode 0 --pca 1 --pcb 1.5 --score-bias 0 --gap-open nucl:5,aa:11 --gap-extend nucl:2,aa:1 --zdrop 40 --threads 4 --compressed 0 -v 3 \n","\n","Compute score and coverage\n","Query database size: 5495 type: Aminoacid\n","Target database size: 562755 type: Aminoacid\n","Calculation of alignments\n","[=================================================================] 100.00% 5.49K 31s 839ms\n","Time for merging to resultsDB_53: 0h 0m 0s 1ms\n","\n","1536002 alignments calculated.\n","284917 sequence pairs passed the thresholds (0.185493 of overall calculated).\n","51.850227 hits per query sequence.\n","Time for processing: 0h 0m 31s 901ms\n","convertalis /content/mmseqs2/tmp/43/candidateDB_53 /content/mmseqs2/swissprot/swissprotDB /content/mmseqs2/tmp/43/resultsDB_53 /content/mmseqs2/tmp/43/resultDB_secondary.m8 --format-output query,target,evalue,raw \n","\n","MMseqs Version:        \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Substitution matrix    \tnucl:nucleotide.out,aa:blosum62.out\n","Alignment format       \t0\n","Format alignment output\tquery,target,evalue,raw\n","Translation table      \t1\n","Gap open cost          \tnucl:5,aa:11\n","Gap extension cost     \tnucl:2,aa:1\n","Database output        \tfalse\n","Preload mode           \t0\n","Search type            \t0\n","Threads                \t4\n","Compressed             \t0\n","Verbosity              \t3\n","\n","[=================================================================] 100.00% 5.49K 0s 201ms\n","Time for merging to resultDB_secondary.m8: 0h 0m 0s 9ms\n","Time for processing: 0h 0m 0s 303ms\n","found 3233 genes\n","\n","\n","Reading fasta file GCF_000008625.1_ASM862v1_genomic.fna.gz...\n","\n","Finding and translating open reading frames...\n","\n","Encoding amino acids...\n","\n","Applying kmer prefilter...\n","\n","Scoring ORFs with temporal convolutional network...\n","\n"],"name":"stdout"},{"output_type":"display_data","data":{"application/vnd.jupyter.widget-view+json":{"model_id":"1be10f9fd0ef47a98590193f26431ac6","version_minor":0,"version_major":2},"text/plain":["HBox(children=(FloatProgress(value=0.0, max=17.0), HTML(value='')))"]},"metadata":{"tags":[]}},{"output_type":"stream","text":["\n","Scoring translation initiation sites...\n","\n"],"name":"stdout"},{"output_type":"display_data","data":{"application/vnd.jupyter.widget-view+json":{"model_id":"fb61f186dda04a64a658bea713b75d80","version_minor":0,"version_major":2},"text/plain":["HBox(children=(FloatProgress(value=0.0, max=5.0), HTML(value='')))"]},"metadata":{"tags":[]}},{"output_type":"stream","text":["\n","Creating weighted directed acyclic graph of all ORFs and subORFs on contig 0...\n","\n"],"name":"stdout"},{"output_type":"display_data","data":{"application/vnd.jupyter.widget-view+json":{"model_id":"ab6e48c0a5c84a4d9ca9bdc68d258b26","version_minor":0,"version_major":2},"text/plain":["HBox(children=(FloatProgress(value=0.0, max=48762.0), HTML(value='')))"]},"metadata":{"tags":[]}},{"output_type":"stream","text":["\n","Maximizing geneiness...\n","\n","Filtering predictions with mmseqs2...\n","createdb /content/mmseqs2/tmp/44/candidate_genes_35.fasta /content/mmseqs2/tmp/44/candidateDB_35 \n","\n","MMseqs Version:       \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Database type         \t0\n","Shuffle input database\ttrue\n","Createdb mode         \t0\n","Write lookup file     \t1\n","Offset of numeric ids \t0\n","Compressed            \t0\n","Verbosity             \t3\n","\n","Converting sequences\n","[2021] 0s 4ms\n","Time for merging to candidateDB_35_h: 0h 0m 0s 4ms\n","Time for merging to candidateDB_35: 0h 0m 0s 3ms\n","Database type: Aminoacid\n","Time for processing: 0h 0m 0s 16ms\n","createdb /content/mmseqs2/tmp/44/candidate_genes_53.fasta /content/mmseqs2/tmp/44/candidateDB_53 \n","\n","MMseqs Version:       \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Database type         \t0\n","Shuffle input database\ttrue\n","Createdb mode         \t0\n","Write lookup file     \t1\n","Offset of numeric ids \t0\n","Compressed            \t0\n","Verbosity             \t3\n","\n","Converting sequences\n","[2021] 0s 4ms\n","Time for merging to candidateDB_53_h: 0h 0m 0s 3ms\n","Time for merging to candidateDB_53: 0h 0m 0s 3ms\n","Database type: Aminoacid\n","Time for processing: 0h 0m 0s 14ms\n","search -s 7.0 /content/mmseqs2/tmp/44/candidateDB_35 /content/mmseqs2/genexa/genexaDB /content/mmseqs2/tmp/44/resultsDB_35 /content/mmseqs2/tmp/44 \n","\n","MMseqs Version:                     \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Substitution matrix                 \tnucl:nucleotide.out,aa:blosum62.out\n","Add backtrace                       \tfalse\n","Alignment mode                      \t2\n","Allow wrapped scoring               \tfalse\n","E-value threshold                   \t0.001\n","Seq. id. threshold                  \t0\n","Min alignment length                \t0\n","Seq. id. mode                       \t0\n","Alternative alignments              \t0\n","Coverage threshold                  \t0\n","Coverage mode                       \t0\n","Max sequence length                 \t65535\n","Compositional bias                  \t1\n","Realign hits                        \tfalse\n","Max reject                          \t2147483647\n","Max accept                          \t2147483647\n","Include identical seq. id.          \tfalse\n","Preload mode                        \t0\n","Pseudo count a                      \t1\n","Pseudo count b                      \t1.5\n","Score bias                          \t0\n","Gap open cost                       \tnucl:5,aa:11\n","Gap extension cost                  \tnucl:2,aa:1\n","Zdrop                               \t40\n","Threads                             \t4\n","Compressed                          \t0\n","Verbosity                           \t3\n","Seed substitution matrix            \tnucl:nucleotide.out,aa:VTML80.out\n","Sensitivity                         \t7\n","k-mer length                        \t0\n","k-score                             \t2147483647\n","Alphabet size                       \tnucl:5,aa:21\n","Max results per query               \t300\n","Split database                      \t0\n","Split mode                          \t2\n","Split memory limit                  \t0\n","Diagonal scoring                    \ttrue\n","Exact k-mer matching                \t0\n","Mask residues                       \t1\n","Mask lower case residues            \t0\n","Minimum diagonal score              \t15\n","Spaced k-mers                       \t1\n","Spaced k-mer pattern                \t\n","Local temporary path                \t\n","Rescore mode                        \t0\n","Remove hits by seq. id. and coverage\tfalse\n","Sort results                        \t0\n","Mask profile                        \t1\n","Profile e-value threshold           \t0.1\n","Global sequence weighting           \tfalse\n","Allow deletions                     \tfalse\n","Filter MSA                          \t1\n","Maximum seq. id. threshold          \t0.9\n","Minimum seq. id.                    \t0\n","Minimum score per column            \t-20\n","Minimum coverage                    \t0\n","Select N most diverse seqs          \t1000\n","Omit consensus                      \tfalse\n","Min codons in orf                   \t30\n","Max codons in length                \t32734\n","Max orf gaps                        \t2147483647\n","Contig start mode                   \t2\n","Contig end mode                     \t2\n","Orf start mode                      \t1\n","Forward frames                      \t1,2,3\n","Reverse frames                      \t1,2,3\n","Translation table                   \t1\n","Translate orf                       \t0\n","Use all table starts                \tfalse\n","Offset of numeric ids               \t0\n","Create lookup                       \t0\n","Add orf stop                        \tfalse\n","Chain overlapping alignments        \t0\n","Merge query                         \t1\n","Search type                         \t0\n","Search iterations                   \t1\n","Start sensitivity                   \t4\n","Search steps                        \t1\n","Slice search mode                   \tfalse\n","Strand selection                    \t1\n","Disk space limit                    \t0\n","MPI runner                          \t\n","Force restart with latest tmp       \tfalse\n","Remove temporary files              \tfalse\n","\n","prefilter /content/mmseqs2/tmp/44/candidateDB_35 /content/mmseqs2/genexa/genexaDB.idx /content/mmseqs2/tmp/44/13521373440232481889/pref_0 --sub-mat nucl:nucleotide.out,aa:blosum62.out --seed-sub-mat nucl:nucleotide.out,aa:VTML80.out -k 0 --k-score 2147483647 --alph-size nucl:5,aa:21 --max-seq-len 65535 --max-seqs 300 --split 0 --split-mode 2 --split-memory-limit 0 -c 0 --cov-mode 0 --comp-bias-corr 1 --diag-score 1 --exact-kmer-matching 0 --mask 1 --mask-lower-case 0 --min-ungapped-score 15 --add-self-matches 0 --spaced-kmer-mode 1 --db-load-mode 0 --pca 1 --pcb 1.5 --threads 4 --compressed 0 -v 3 -s 7.0 \n","\n","Index version: 16\n","Generated by:  16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","ScoreMatrix:  VTML80.out\n","Query database size: 2052 type: Aminoacid\n","Estimated memory consumption: 2G\n","Target database size: 468351 type: Aminoacid\n","Process prefiltering step 1 of 1\n","\n","k-mer similarity threshold: 100\n","Starting prefiltering scores calculation (step 1 of 1)\n","Query db start 1 to 2052\n","Target db start 1 to 468351\n","[=================================================================] 100.00% 2.05K 18s 13ms\n","\n","944.671752 k-mers per position\n","695952 DB matches per sequence\n","76 overflows\n","0 queries produce too many hits (truncated result)\n","286 sequences passed prefiltering per query sequence\n","300 median result list length\n","1 sequences with 0 size result lists\n","Time for merging to pref_0: 0h 0m 0s 0ms\n","Time for processing: 0h 0m 19s 416ms\n","align /content/mmseqs2/tmp/44/candidateDB_35 /content/mmseqs2/genexa/genexaDB.idx /content/mmseqs2/tmp/44/13521373440232481889/pref_0 /content/mmseqs2/tmp/44/resultsDB_35 --sub-mat nucl:nucleotide.out,aa:blosum62.out -a 0 --alignment-mode 2 --wrapped-scoring 0 -e 0.001 --min-seq-id 0 --min-aln-len 0 --seq-id-mode 0 --alt-ali 0 -c 0 --cov-mode 0 --max-seq-len 65535 --comp-bias-corr 1 --realign 0 --max-rejected 2147483647 --max-accept 2147483647 --add-self-matches 0 --db-load-mode 0 --pca 1 --pcb 1.5 --score-bias 0 --gap-open nucl:5,aa:11 --gap-extend nucl:2,aa:1 --zdrop 40 --threads 4 --compressed 0 -v 3 \n","\n","Index version: 16\n","Generated by:  16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","ScoreMatrix:  VTML80.out\n","Compute score and coverage\n","Query database size: 2052 type: Aminoacid\n","Target database size: 468351 type: Aminoacid\n","Calculation of alignments\n","[=================================================================] 100.00% 2.05K 16s 327ms\n","Time for merging to resultsDB_35: 0h 0m 0s 0ms\n","\n","587134 alignments calculated.\n","193159 sequence pairs passed the thresholds (0.328986 of overall calculated).\n","94.132065 hits per query sequence.\n","Time for processing: 0h 0m 16s 449ms\n","convertalis /content/mmseqs2/tmp/44/candidateDB_35 /content/mmseqs2/genexa/genexaDB /content/mmseqs2/tmp/44/resultsDB_35 /content/mmseqs2/tmp/44/resultDB_genexa.m8 --format-output query,target,evalue,raw \n","\n","MMseqs Version:        \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Substitution matrix    \tnucl:nucleotide.out,aa:blosum62.out\n","Alignment format       \t0\n","Format alignment output\tquery,target,evalue,raw\n","Translation table      \t1\n","Gap open cost          \tnucl:5,aa:11\n","Gap extension cost     \tnucl:2,aa:1\n","Database output        \tfalse\n","Preload mode           \t0\n","Search type            \t0\n","Threads                \t4\n","Compressed             \t0\n","Verbosity              \t3\n","\n","[=================================================================] 100.00% 2.05K 0s 139ms\n","Time for merging to resultDB_genexa.m8: 0h 0m 0s 5ms\n","Time for processing: 0h 0m 0s 211ms\n","search -s 7.0 /content/mmseqs2/tmp/44/candidateDB_53 /content/mmseqs2/swissprot/swissprotDB /content/mmseqs2/tmp/44/resultsDB_53 /content/mmseqs2/tmp/44 \n","\n","MMseqs Version:                     \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Substitution matrix                 \tnucl:nucleotide.out,aa:blosum62.out\n","Add backtrace                       \tfalse\n","Alignment mode                      \t2\n","Allow wrapped scoring               \tfalse\n","E-value threshold                   \t0.001\n","Seq. id. threshold                  \t0\n","Min alignment length                \t0\n","Seq. id. mode                       \t0\n","Alternative alignments              \t0\n","Coverage threshold                  \t0\n","Coverage mode                       \t0\n","Max sequence length                 \t65535\n","Compositional bias                  \t1\n","Realign hits                        \tfalse\n","Max reject                          \t2147483647\n","Max accept                          \t2147483647\n","Include identical seq. id.          \tfalse\n","Preload mode                        \t0\n","Pseudo count a                      \t1\n","Pseudo count b                      \t1.5\n","Score bias                          \t0\n","Gap open cost                       \tnucl:5,aa:11\n","Gap extension cost                  \tnucl:2,aa:1\n","Zdrop                               \t40\n","Threads                             \t4\n","Compressed                          \t0\n","Verbosity                           \t3\n","Seed substitution matrix            \tnucl:nucleotide.out,aa:VTML80.out\n","Sensitivity                         \t7\n","k-mer length                        \t0\n","k-score                             \t2147483647\n","Alphabet size                       \tnucl:5,aa:21\n","Max results per query               \t300\n","Split database                      \t0\n","Split mode                          \t2\n","Split memory limit                  \t0\n","Diagonal scoring                    \ttrue\n","Exact k-mer matching                \t0\n","Mask residues                       \t1\n","Mask lower case residues            \t0\n","Minimum diagonal score              \t15\n","Spaced k-mers                       \t1\n","Spaced k-mer pattern                \t\n","Local temporary path                \t\n","Rescore mode                        \t0\n","Remove hits by seq. id. and coverage\tfalse\n","Sort results                        \t0\n","Mask profile                        \t1\n","Profile e-value threshold           \t0.1\n","Global sequence weighting           \tfalse\n","Allow deletions                     \tfalse\n","Filter MSA                          \t1\n","Maximum seq. id. threshold          \t0.9\n","Minimum seq. id.                    \t0\n","Minimum score per column            \t-20\n","Minimum coverage                    \t0\n","Select N most diverse seqs          \t1000\n","Omit consensus                      \tfalse\n","Min codons in orf                   \t30\n","Max codons in length                \t32734\n","Max orf gaps                        \t2147483647\n","Contig start mode                   \t2\n","Contig end mode                     \t2\n","Orf start mode                      \t1\n","Forward frames                      \t1,2,3\n","Reverse frames                      \t1,2,3\n","Translation table                   \t1\n","Translate orf                       \t0\n","Use all table starts                \tfalse\n","Offset of numeric ids               \t0\n","Create lookup                       \t0\n","Add orf stop                        \tfalse\n","Chain overlapping alignments        \t0\n","Merge query                         \t1\n","Search type                         \t0\n","Search iterations                   \t1\n","Start sensitivity                   \t4\n","Search steps                        \t1\n","Slice search mode                   \tfalse\n","Strand selection                    \t1\n","Disk space limit                    \t0\n","MPI runner                          \t\n","Force restart with latest tmp       \tfalse\n","Remove temporary files              \tfalse\n","\n","prefilter /content/mmseqs2/tmp/44/candidateDB_53 /content/mmseqs2/swissprot/swissprotDB /content/mmseqs2/tmp/44/14044627041974902335/pref_0 --sub-mat nucl:nucleotide.out,aa:blosum62.out --seed-sub-mat nucl:nucleotide.out,aa:VTML80.out -k 0 --k-score 2147483647 --alph-size nucl:5,aa:21 --max-seq-len 65535 --max-seqs 300 --split 0 --split-mode 2 --split-memory-limit 0 -c 0 --cov-mode 0 --comp-bias-corr 1 --diag-score 1 --exact-kmer-matching 0 --mask 1 --mask-lower-case 0 --min-ungapped-score 15 --add-self-matches 0 --spaced-kmer-mode 1 --db-load-mode 0 --pca 1 --pcb 1.5 --threads 4 --compressed 0 -v 3 -s 7.0 \n","\n","Query database size: 2052 type: Aminoacid\n","Estimated memory consumption: 2G\n","Target database size: 562755 type: Aminoacid\n","Index table k-mer threshold: 100 at k-mer size 6 \n","Index table: counting k-mers\n","[=================================================================] 100.00% 562.76K 23s 973ms\n","Index table: Masked residues: 3164874\n","Index table: fill\n","[=================================================================] 100.00% 562.76K 17s 744ms\n","Index statistics\n","Entries:          192757622\n","DB size:          1591 MB\n","Avg k-mer size:   3.011838\n","Top 10 k-mers\n","    GPGGTL\t1835\n","    GQSWTV\t1706\n","    WGMFAT\t1638\n","    PGVFEV\t1634\n","    VLWQFW\t1623\n","    AYIRPN\t1587\n","    RSPKGV\t1585\n","    TPHKWY\t1560\n","    HIRVGW\t1548\n","    ITLSPY\t1541\n","Time for index table init: 0h 0m 43s 819ms\n","Process prefiltering step 1 of 1\n","\n","k-mer similarity threshold: 100\n","Starting prefiltering scores calculation (step 1 of 1)\n","Query db start 1 to 2052\n","Target db start 1 to 562755\n","[=================================================================] 100.00% 2.05K 21s 196ms\n","\n","945.675850 k-mers per position\n","876999 DB matches per sequence\n","165 overflows\n","0 queries produce too many hits (truncated result)\n","290 sequences passed prefiltering per query sequence\n","300 median result list length\n","1 sequences with 0 size result lists\n","Time for merging to pref_0: 0h 0m 0s 0ms\n","Time for processing: 0h 1m 6s 123ms\n","align /content/mmseqs2/tmp/44/candidateDB_53 /content/mmseqs2/swissprot/swissprotDB /content/mmseqs2/tmp/44/14044627041974902335/pref_0 /content/mmseqs2/tmp/44/resultsDB_53 --sub-mat nucl:nucleotide.out,aa:blosum62.out -a 0 --alignment-mode 2 --wrapped-scoring 0 -e 0.001 --min-seq-id 0 --min-aln-len 0 --seq-id-mode 0 --alt-ali 0 -c 0 --cov-mode 0 --max-seq-len 65535 --comp-bias-corr 1 --realign 0 --max-rejected 2147483647 --max-accept 2147483647 --add-self-matches 0 --db-load-mode 0 --pca 1 --pcb 1.5 --score-bias 0 --gap-open nucl:5,aa:11 --gap-extend nucl:2,aa:1 --zdrop 40 --threads 4 --compressed 0 -v 3 \n","\n","Compute score and coverage\n","Query database size: 2052 type: Aminoacid\n","Target database size: 562755 type: Aminoacid\n","Calculation of alignments\n","[=================================================================] 100.00% 2.05K 16s 972ms\n","Time for merging to resultsDB_53: 0h 0m 0s 0ms\n","\n","595468 alignments calculated.\n","177067 sequence pairs passed the thresholds (0.297358 of overall calculated).\n","86.289963 hits per query sequence.\n","Time for processing: 0h 0m 17s 37ms\n","convertalis /content/mmseqs2/tmp/44/candidateDB_53 /content/mmseqs2/swissprot/swissprotDB /content/mmseqs2/tmp/44/resultsDB_53 /content/mmseqs2/tmp/44/resultDB_secondary.m8 --format-output query,target,evalue,raw \n","\n","MMseqs Version:        \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Substitution matrix    \tnucl:nucleotide.out,aa:blosum62.out\n","Alignment format       \t0\n","Format alignment output\tquery,target,evalue,raw\n","Translation table      \t1\n","Gap open cost          \tnucl:5,aa:11\n","Gap extension cost     \tnucl:2,aa:1\n","Database output        \tfalse\n","Preload mode           \t0\n","Search type            \t0\n","Threads                \t4\n","Compressed             \t0\n","Verbosity              \t3\n","\n","[=================================================================] 100.00% 2.05K 0s 119ms\n","Time for merging to resultDB_secondary.m8: 0h 0m 0s 5ms\n","Time for processing: 0h 0m 0s 211ms\n","found 1671 genes\n","\n","\n","Creating weighted directed acyclic graph of all ORFs and subORFs on contig 1...\n","\n"],"name":"stdout"},{"output_type":"display_data","data":{"application/vnd.jupyter.widget-view+json":{"model_id":"e50537a8c3bc44abb9b5a523f75c918a","version_minor":0,"version_major":2},"text/plain":["HBox(children=(FloatProgress(value=0.0, max=1077.0), HTML(value='')))"]},"metadata":{"tags":[]}},{"output_type":"stream","text":["\n","Maximizing geneiness...\n","\n","Filtering predictions with mmseqs2...\n","createdb /content/mmseqs2/tmp/45/candidate_genes_35.fasta /content/mmseqs2/tmp/45/candidateDB_35 \n","\n","MMseqs Version:       \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Database type         \t0\n","Shuffle input database\ttrue\n","Createdb mode         \t0\n","Write lookup file     \t1\n","Offset of numeric ids \t0\n","Compressed            \t0\n","Verbosity             \t3\n","\n","Converting sequences\n","[203] 0s 2ms\n","Time for merging to candidateDB_35_h: 0h 0m 0s 1ms\n","Time for merging to candidateDB_35: 0h 0m 0s 1ms\n","Database type: Aminoacid\n","Time for processing: 0h 0m 0s 7ms\n","createdb /content/mmseqs2/tmp/45/candidate_genes_53.fasta /content/mmseqs2/tmp/45/candidateDB_53 \n","\n","MMseqs Version:       \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Database type         \t0\n","Shuffle input database\ttrue\n","Createdb mode         \t0\n","Write lookup file     \t1\n","Offset of numeric ids \t0\n","Compressed            \t0\n","Verbosity             \t3\n","\n","Converting sequences\n","[203] 0s 3ms\n","Time for merging to candidateDB_53_h: 0h 0m 0s 2ms\n","Time for merging to candidateDB_53: 0h 0m 0s 1ms\n","Database type: Aminoacid\n","Time for processing: 0h 0m 0s 8ms\n","search -s 7.0 /content/mmseqs2/tmp/45/candidateDB_35 /content/mmseqs2/genexa/genexaDB /content/mmseqs2/tmp/45/resultsDB_35 /content/mmseqs2/tmp/45 \n","\n","MMseqs Version:                     \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Substitution matrix                 \tnucl:nucleotide.out,aa:blosum62.out\n","Add backtrace                       \tfalse\n","Alignment mode                      \t2\n","Allow wrapped scoring               \tfalse\n","E-value threshold                   \t0.001\n","Seq. id. threshold                  \t0\n","Min alignment length                \t0\n","Seq. id. mode                       \t0\n","Alternative alignments              \t0\n","Coverage threshold                  \t0\n","Coverage mode                       \t0\n","Max sequence length                 \t65535\n","Compositional bias                  \t1\n","Realign hits                        \tfalse\n","Max reject                          \t2147483647\n","Max accept                          \t2147483647\n","Include identical seq. id.          \tfalse\n","Preload mode                        \t0\n","Pseudo count a                      \t1\n","Pseudo count b                      \t1.5\n","Score bias                          \t0\n","Gap open cost                       \tnucl:5,aa:11\n","Gap extension cost                  \tnucl:2,aa:1\n","Zdrop                               \t40\n","Threads                             \t4\n","Compressed                          \t0\n","Verbosity                           \t3\n","Seed substitution matrix            \tnucl:nucleotide.out,aa:VTML80.out\n","Sensitivity                         \t7\n","k-mer length                        \t0\n","k-score                             \t2147483647\n","Alphabet size                       \tnucl:5,aa:21\n","Max results per query               \t300\n","Split database                      \t0\n","Split mode                          \t2\n","Split memory limit                  \t0\n","Diagonal scoring                    \ttrue\n","Exact k-mer matching                \t0\n","Mask residues                       \t1\n","Mask lower case residues            \t0\n","Minimum diagonal score              \t15\n","Spaced k-mers                       \t1\n","Spaced k-mer pattern                \t\n","Local temporary path                \t\n","Rescore mode                        \t0\n","Remove hits by seq. id. and coverage\tfalse\n","Sort results                        \t0\n","Mask profile                        \t1\n","Profile e-value threshold           \t0.1\n","Global sequence weighting           \tfalse\n","Allow deletions                     \tfalse\n","Filter MSA                          \t1\n","Maximum seq. id. threshold          \t0.9\n","Minimum seq. id.                    \t0\n","Minimum score per column            \t-20\n","Minimum coverage                    \t0\n","Select N most diverse seqs          \t1000\n","Omit consensus                      \tfalse\n","Min codons in orf                   \t30\n","Max codons in length                \t32734\n","Max orf gaps                        \t2147483647\n","Contig start mode                   \t2\n","Contig end mode                     \t2\n","Orf start mode                      \t1\n","Forward frames                      \t1,2,3\n","Reverse frames                      \t1,2,3\n","Translation table                   \t1\n","Translate orf                       \t0\n","Use all table starts                \tfalse\n","Offset of numeric ids               \t0\n","Create lookup                       \t0\n","Add orf stop                        \tfalse\n","Chain overlapping alignments        \t0\n","Merge query                         \t1\n","Search type                         \t0\n","Search iterations                   \t1\n","Start sensitivity                   \t4\n","Search steps                        \t1\n","Slice search mode                   \tfalse\n","Strand selection                    \t1\n","Disk space limit                    \t0\n","MPI runner                          \t\n","Force restart with latest tmp       \tfalse\n","Remove temporary files              \tfalse\n","\n","prefilter /content/mmseqs2/tmp/45/candidateDB_35 /content/mmseqs2/genexa/genexaDB.idx /content/mmseqs2/tmp/45/8261087337122786400/pref_0 --sub-mat nucl:nucleotide.out,aa:blosum62.out --seed-sub-mat nucl:nucleotide.out,aa:VTML80.out -k 0 --k-score 2147483647 --alph-size nucl:5,aa:21 --max-seq-len 65535 --max-seqs 300 --split 0 --split-mode 2 --split-memory-limit 0 -c 0 --cov-mode 0 --comp-bias-corr 1 --diag-score 1 --exact-kmer-matching 0 --mask 1 --mask-lower-case 0 --min-ungapped-score 15 --add-self-matches 0 --spaced-kmer-mode 1 --db-load-mode 0 --pca 1 --pcb 1.5 --threads 4 --compressed 0 -v 3 -s 7.0 \n","\n","Index version: 16\n","Generated by:  16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","ScoreMatrix:  VTML80.out\n","Query database size: 213 type: Aminoacid\n","Estimated memory consumption: 2G\n","Target database size: 468351 type: Aminoacid\n","Process prefiltering step 1 of 1\n","\n","k-mer similarity threshold: 100\n","Starting prefiltering scores calculation (step 1 of 1)\n","Query db start 1 to 213\n","Target db start 1 to 468351\n","[=================================================================] 100.00% 213 1s 229ms\n","\n","885.005195 k-mers per position\n","460939 DB matches per sequence\n","5 overflows\n","0 queries produce too many hits (truncated result)\n","274 sequences passed prefiltering per query sequence\n","300 median result list length\n","0 sequences with 0 size result lists\n","Time for merging to pref_0: 0h 0m 0s 0ms\n","Time for processing: 0h 0m 2s 624ms\n","align /content/mmseqs2/tmp/45/candidateDB_35 /content/mmseqs2/genexa/genexaDB.idx /content/mmseqs2/tmp/45/8261087337122786400/pref_0 /content/mmseqs2/tmp/45/resultsDB_35 --sub-mat nucl:nucleotide.out,aa:blosum62.out -a 0 --alignment-mode 2 --wrapped-scoring 0 -e 0.001 --min-seq-id 0 --min-aln-len 0 --seq-id-mode 0 --alt-ali 0 -c 0 --cov-mode 0 --max-seq-len 65535 --comp-bias-corr 1 --realign 0 --max-rejected 2147483647 --max-accept 2147483647 --add-self-matches 0 --db-load-mode 0 --pca 1 --pcb 1.5 --score-bias 0 --gap-open nucl:5,aa:11 --gap-extend nucl:2,aa:1 --zdrop 40 --threads 4 --compressed 0 -v 3 \n","\n","Index version: 16\n","Generated by:  16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","ScoreMatrix:  VTML80.out\n","Compute score and coverage\n","Query database size: 213 type: Aminoacid\n","Target database size: 468351 type: Aminoacid\n","Calculation of alignments\n","[=================================================================] 100.00% 213 1s 423ms\n","Time for merging to resultsDB_35: 0h 0m 0s 0ms\n","\n","58369 alignments calculated.\n","20185 sequence pairs passed the thresholds (0.345817 of overall calculated).\n","94.765259 hits per query sequence.\n","Time for processing: 0h 0m 1s 439ms\n","convertalis /content/mmseqs2/tmp/45/candidateDB_35 /content/mmseqs2/genexa/genexaDB /content/mmseqs2/tmp/45/resultsDB_35 /content/mmseqs2/tmp/45/resultDB_genexa.m8 --format-output query,target,evalue,raw \n","\n","MMseqs Version:        \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Substitution matrix    \tnucl:nucleotide.out,aa:blosum62.out\n","Alignment format       \t0\n","Format alignment output\tquery,target,evalue,raw\n","Translation table      \t1\n","Gap open cost          \tnucl:5,aa:11\n","Gap extension cost     \tnucl:2,aa:1\n","Database output        \tfalse\n","Preload mode           \t0\n","Search type            \t0\n","Threads                \t4\n","Compressed             \t0\n","Verbosity              \t3\n","\n","[=================================================================] 100.00% 213 0s 15ms\n","Time for merging to resultDB_genexa.m8: 0h 0m 0s 0ms\n","Time for processing: 0h 0m 0s 79ms\n","search -s 7.0 /content/mmseqs2/tmp/45/candidateDB_53 /content/mmseqs2/swissprot/swissprotDB /content/mmseqs2/tmp/45/resultsDB_53 /content/mmseqs2/tmp/45 \n","\n","MMseqs Version:                     \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Substitution matrix                 \tnucl:nucleotide.out,aa:blosum62.out\n","Add backtrace                       \tfalse\n","Alignment mode                      \t2\n","Allow wrapped scoring               \tfalse\n","E-value threshold                   \t0.001\n","Seq. id. threshold                  \t0\n","Min alignment length                \t0\n","Seq. id. mode                       \t0\n","Alternative alignments              \t0\n","Coverage threshold                  \t0\n","Coverage mode                       \t0\n","Max sequence length                 \t65535\n","Compositional bias                  \t1\n","Realign hits                        \tfalse\n","Max reject                          \t2147483647\n","Max accept                          \t2147483647\n","Include identical seq. id.          \tfalse\n","Preload mode                        \t0\n","Pseudo count a                      \t1\n","Pseudo count b                      \t1.5\n","Score bias                          \t0\n","Gap open cost                       \tnucl:5,aa:11\n","Gap extension cost                  \tnucl:2,aa:1\n","Zdrop                               \t40\n","Threads                             \t4\n","Compressed                          \t0\n","Verbosity                           \t3\n","Seed substitution matrix            \tnucl:nucleotide.out,aa:VTML80.out\n","Sensitivity                         \t7\n","k-mer length                        \t0\n","k-score                             \t2147483647\n","Alphabet size                       \tnucl:5,aa:21\n","Max results per query               \t300\n","Split database                      \t0\n","Split mode                          \t2\n","Split memory limit                  \t0\n","Diagonal scoring                    \ttrue\n","Exact k-mer matching                \t0\n","Mask residues                       \t1\n","Mask lower case residues            \t0\n","Minimum diagonal score              \t15\n","Spaced k-mers                       \t1\n","Spaced k-mer pattern                \t\n","Local temporary path                \t\n","Rescore mode                        \t0\n","Remove hits by seq. id. and coverage\tfalse\n","Sort results                        \t0\n","Mask profile                        \t1\n","Profile e-value threshold           \t0.1\n","Global sequence weighting           \tfalse\n","Allow deletions                     \tfalse\n","Filter MSA                          \t1\n","Maximum seq. id. threshold          \t0.9\n","Minimum seq. id.                    \t0\n","Minimum score per column            \t-20\n","Minimum coverage                    \t0\n","Select N most diverse seqs          \t1000\n","Omit consensus                      \tfalse\n","Min codons in orf                   \t30\n","Max codons in length                \t32734\n","Max orf gaps                        \t2147483647\n","Contig start mode                   \t2\n","Contig end mode                     \t2\n","Orf start mode                      \t1\n","Forward frames                      \t1,2,3\n","Reverse frames                      \t1,2,3\n","Translation table                   \t1\n","Translate orf                       \t0\n","Use all table starts                \tfalse\n","Offset of numeric ids               \t0\n","Create lookup                       \t0\n","Add orf stop                        \tfalse\n","Chain overlapping alignments        \t0\n","Merge query                         \t1\n","Search type                         \t0\n","Search iterations                   \t1\n","Start sensitivity                   \t4\n","Search steps                        \t1\n","Slice search mode                   \tfalse\n","Strand selection                    \t1\n","Disk space limit                    \t0\n","MPI runner                          \t\n","Force restart with latest tmp       \tfalse\n","Remove temporary files              \tfalse\n","\n","prefilter /content/mmseqs2/tmp/45/candidateDB_53 /content/mmseqs2/swissprot/swissprotDB /content/mmseqs2/tmp/45/14145992030020248318/pref_0 --sub-mat nucl:nucleotide.out,aa:blosum62.out --seed-sub-mat nucl:nucleotide.out,aa:VTML80.out -k 0 --k-score 2147483647 --alph-size nucl:5,aa:21 --max-seq-len 65535 --max-seqs 300 --split 0 --split-mode 2 --split-memory-limit 0 -c 0 --cov-mode 0 --comp-bias-corr 1 --diag-score 1 --exact-kmer-matching 0 --mask 1 --mask-lower-case 0 --min-ungapped-score 15 --add-self-matches 0 --spaced-kmer-mode 1 --db-load-mode 0 --pca 1 --pcb 1.5 --threads 4 --compressed 0 -v 3 -s 7.0 \n","\n","Query database size: 213 type: Aminoacid\n","Estimated memory consumption: 2G\n","Target database size: 562755 type: Aminoacid\n","Index table k-mer threshold: 100 at k-mer size 6 \n","Index table: counting k-mers\n","[=================================================================] 100.00% 562.76K 24s 58ms\n","Index table: Masked residues: 3164874\n","Index table: fill\n","[=================================================================] 100.00% 562.76K 17s 695ms\n","Index statistics\n","Entries:          192757622\n","DB size:          1591 MB\n","Avg k-mer size:   3.011838\n","Top 10 k-mers\n","    GPGGTL\t1835\n","    GQSWTV\t1706\n","    WGMFAT\t1638\n","    PGVFEV\t1634\n","    VLWQFW\t1623\n","    AYIRPN\t1587\n","    RSPKGV\t1585\n","    TPHKWY\t1560\n","    HIRVGW\t1548\n","    ITLSPY\t1541\n","Time for index table init: 0h 0m 43s 843ms\n","Process prefiltering step 1 of 1\n","\n","k-mer similarity threshold: 100\n","Starting prefiltering scores calculation (step 1 of 1)\n","Query db start 1 to 213\n","Target db start 1 to 562755\n","[=================================================================] 100.00% 213 1s 453ms\n","\n","883.929020 k-mers per position\n","576669 DB matches per sequence\n","9 overflows\n","0 queries produce too many hits (truncated result)\n","282 sequences passed prefiltering per query sequence\n","300 median result list length\n","0 sequences with 0 size result lists\n","Time for merging to pref_0: 0h 0m 0s 0ms\n","Time for processing: 0h 0m 46s 409ms\n","align /content/mmseqs2/tmp/45/candidateDB_53 /content/mmseqs2/swissprot/swissprotDB /content/mmseqs2/tmp/45/14145992030020248318/pref_0 /content/mmseqs2/tmp/45/resultsDB_53 --sub-mat nucl:nucleotide.out,aa:blosum62.out -a 0 --alignment-mode 2 --wrapped-scoring 0 -e 0.001 --min-seq-id 0 --min-aln-len 0 --seq-id-mode 0 --alt-ali 0 -c 0 --cov-mode 0 --max-seq-len 65535 --comp-bias-corr 1 --realign 0 --max-rejected 2147483647 --max-accept 2147483647 --add-self-matches 0 --db-load-mode 0 --pca 1 --pcb 1.5 --score-bias 0 --gap-open nucl:5,aa:11 --gap-extend nucl:2,aa:1 --zdrop 40 --threads 4 --compressed 0 -v 3 \n","\n","Compute score and coverage\n","Query database size: 213 type: Aminoacid\n","Target database size: 562755 type: Aminoacid\n","Calculation of alignments\n","[=================================================================] 100.00% 213 1s 873ms\n","Time for merging to resultsDB_53: 0h 0m 0s 0ms\n","\n","60232 alignments calculated.\n","22255 sequence pairs passed the thresholds (0.369488 of overall calculated).\n","104.483566 hits per query sequence.\n","Time for processing: 0h 0m 2s 61ms\n","convertalis /content/mmseqs2/tmp/45/candidateDB_53 /content/mmseqs2/swissprot/swissprotDB /content/mmseqs2/tmp/45/resultsDB_53 /content/mmseqs2/tmp/45/resultDB_secondary.m8 --format-output query,target,evalue,raw \n","\n","MMseqs Version:        \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Substitution matrix    \tnucl:nucleotide.out,aa:blosum62.out\n","Alignment format       \t0\n","Format alignment output\tquery,target,evalue,raw\n","Translation table      \t1\n","Gap open cost          \tnucl:5,aa:11\n","Gap extension cost     \tnucl:2,aa:1\n","Database output        \tfalse\n","Preload mode           \t0\n","Search type            \t0\n","Threads                \t4\n","Compressed             \t0\n","Verbosity              \t3\n","\n","[=================================================================] 100.00% 213 0s 15ms\n","Time for merging to resultDB_secondary.m8: 0h 0m 0s 1ms\n","Time for processing: 0h 0m 0s 102ms\n","found 149 genes\n","\n","\n","Reading fasta file GCF_000005845.2_ASM584v2_genomic.fna.gz...\n","\n","Finding and translating open reading frames...\n","\n","Encoding amino acids...\n","\n","Applying kmer prefilter...\n","\n","Scoring ORFs with temporal convolutional network...\n","\n"],"name":"stdout"},{"output_type":"display_data","data":{"application/vnd.jupyter.widget-view+json":{"model_id":"722d734d9e97467c8ad0f1576da77e5b","version_minor":0,"version_major":2},"text/plain":["HBox(children=(FloatProgress(value=0.0, max=81.0), HTML(value='')))"]},"metadata":{"tags":[]}},{"output_type":"stream","text":["\n","Scoring translation initiation sites...\n","\n"],"name":"stdout"},{"output_type":"display_data","data":{"application/vnd.jupyter.widget-view+json":{"model_id":"e18dcbc034b0447fa67a49cabfdd6dbf","version_minor":0,"version_major":2},"text/plain":["HBox(children=(FloatProgress(value=0.0, max=25.0), HTML(value='')))"]},"metadata":{"tags":[]}},{"output_type":"stream","text":["\n","Creating weighted directed acyclic graph of all ORFs and subORFs on contig 0...\n","\n"],"name":"stdout"},{"output_type":"display_data","data":{"application/vnd.jupyter.widget-view+json":{"model_id":"4d7ca9761f994e1a92d8e6e58670b6a3","version_minor":0,"version_major":2},"text/plain":["HBox(children=(FloatProgress(value=0.0, max=242033.0), HTML(value='')))"]},"metadata":{"tags":[]}},{"output_type":"stream","text":["\n","Maximizing geneiness...\n","\n","Filtering predictions with mmseqs2...\n","createdb /content/mmseqs2/tmp/46/candidate_genes_35.fasta /content/mmseqs2/tmp/46/candidateDB_35 \n","\n","MMseqs Version:       \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Database type         \t0\n","Shuffle input database\ttrue\n","Createdb mode         \t0\n","Write lookup file     \t1\n","Offset of numeric ids \t0\n","Compressed            \t0\n","Verbosity             \t3\n","\n","Converting sequences\n","[7778] 0s 11ms\n","Time for merging to candidateDB_35_h: 0h 0m 0s 5ms\n","Time for merging to candidateDB_35: 0h 0m 0s 5ms\n","Database type: Aminoacid\n","Time for processing: 0h 0m 0s 31ms\n","createdb /content/mmseqs2/tmp/46/candidate_genes_53.fasta /content/mmseqs2/tmp/46/candidateDB_53 \n","\n","MMseqs Version:       \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Database type         \t0\n","Shuffle input database\ttrue\n","Createdb mode         \t0\n","Write lookup file     \t1\n","Offset of numeric ids \t0\n","Compressed            \t0\n","Verbosity             \t3\n","\n","Converting sequences\n","[7778] 0s 11ms\n","Time for merging to candidateDB_53_h: 0h 0m 0s 4ms\n","Time for merging to candidateDB_53: 0h 0m 0s 5ms\n","Database type: Aminoacid\n","Time for processing: 0h 0m 0s 30ms\n","search -s 7.0 /content/mmseqs2/tmp/46/candidateDB_35 /content/mmseqs2/genexa/genexaDB /content/mmseqs2/tmp/46/resultsDB_35 /content/mmseqs2/tmp/46 \n","\n","MMseqs Version:                     \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Substitution matrix                 \tnucl:nucleotide.out,aa:blosum62.out\n","Add backtrace                       \tfalse\n","Alignment mode                      \t2\n","Allow wrapped scoring               \tfalse\n","E-value threshold                   \t0.001\n","Seq. id. threshold                  \t0\n","Min alignment length                \t0\n","Seq. id. mode                       \t0\n","Alternative alignments              \t0\n","Coverage threshold                  \t0\n","Coverage mode                       \t0\n","Max sequence length                 \t65535\n","Compositional bias                  \t1\n","Realign hits                        \tfalse\n","Max reject                          \t2147483647\n","Max accept                          \t2147483647\n","Include identical seq. id.          \tfalse\n","Preload mode                        \t0\n","Pseudo count a                      \t1\n","Pseudo count b                      \t1.5\n","Score bias                          \t0\n","Gap open cost                       \tnucl:5,aa:11\n","Gap extension cost                  \tnucl:2,aa:1\n","Zdrop                               \t40\n","Threads                             \t4\n","Compressed                          \t0\n","Verbosity                           \t3\n","Seed substitution matrix            \tnucl:nucleotide.out,aa:VTML80.out\n","Sensitivity                         \t7\n","k-mer length                        \t0\n","k-score                             \t2147483647\n","Alphabet size                       \tnucl:5,aa:21\n","Max results per query               \t300\n","Split database                      \t0\n","Split mode                          \t2\n","Split memory limit                  \t0\n","Diagonal scoring                    \ttrue\n","Exact k-mer matching                \t0\n","Mask residues                       \t1\n","Mask lower case residues            \t0\n","Minimum diagonal score              \t15\n","Spaced k-mers                       \t1\n","Spaced k-mer pattern                \t\n","Local temporary path                \t\n","Rescore mode                        \t0\n","Remove hits by seq. id. and coverage\tfalse\n","Sort results                        \t0\n","Mask profile                        \t1\n","Profile e-value threshold           \t0.1\n","Global sequence weighting           \tfalse\n","Allow deletions                     \tfalse\n","Filter MSA                          \t1\n","Maximum seq. id. threshold          \t0.9\n","Minimum seq. id.                    \t0\n","Minimum score per column            \t-20\n","Minimum coverage                    \t0\n","Select N most diverse seqs          \t1000\n","Omit consensus                      \tfalse\n","Min codons in orf                   \t30\n","Max codons in length                \t32734\n","Max orf gaps                        \t2147483647\n","Contig start mode                   \t2\n","Contig end mode                     \t2\n","Orf start mode                      \t1\n","Forward frames                      \t1,2,3\n","Reverse frames                      \t1,2,3\n","Translation table                   \t1\n","Translate orf                       \t0\n","Use all table starts                \tfalse\n","Offset of numeric ids               \t0\n","Create lookup                       \t0\n","Add orf stop                        \tfalse\n","Chain overlapping alignments        \t0\n","Merge query                         \t1\n","Search type                         \t0\n","Search iterations                   \t1\n","Start sensitivity                   \t4\n","Search steps                        \t1\n","Slice search mode                   \tfalse\n","Strand selection                    \t1\n","Disk space limit                    \t0\n","MPI runner                          \t\n","Force restart with latest tmp       \tfalse\n","Remove temporary files              \tfalse\n","\n","prefilter /content/mmseqs2/tmp/46/candidateDB_35 /content/mmseqs2/genexa/genexaDB.idx /content/mmseqs2/tmp/46/3000801234013090911/pref_0 --sub-mat nucl:nucleotide.out,aa:blosum62.out --seed-sub-mat nucl:nucleotide.out,aa:VTML80.out -k 0 --k-score 2147483647 --alph-size nucl:5,aa:21 --max-seq-len 65535 --max-seqs 300 --split 0 --split-mode 2 --split-memory-limit 0 -c 0 --cov-mode 0 --comp-bias-corr 1 --diag-score 1 --exact-kmer-matching 0 --mask 1 --mask-lower-case 0 --min-ungapped-score 15 --add-self-matches 0 --spaced-kmer-mode 1 --db-load-mode 0 --pca 1 --pcb 1.5 --threads 4 --compressed 0 -v 3 -s 7.0 \n","\n","Index version: 16\n","Generated by:  16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","ScoreMatrix:  VTML80.out\n","Query database size: 7788 type: Aminoacid\n","Estimated memory consumption: 2G\n","Target database size: 468351 type: Aminoacid\n","Process prefiltering step 1 of 1\n","\n","k-mer similarity threshold: 100\n","Starting prefiltering scores calculation (step 1 of 1)\n","Query db start 1 to 7788\n","Target db start 1 to 468351\n","[=================================================================] 100.00% 7.79K 50s 881ms\n","\n","973.833468 k-mers per position\n","505262 DB matches per sequence\n","242 overflows\n","0 queries produce too many hits (truncated result)\n","270 sequences passed prefiltering per query sequence\n","300 median result list length\n","5 sequences with 0 size result lists\n","Time for merging to pref_0: 0h 0m 0s 2ms\n","Time for processing: 0h 0m 52s 395ms\n","align /content/mmseqs2/tmp/46/candidateDB_35 /content/mmseqs2/genexa/genexaDB.idx /content/mmseqs2/tmp/46/3000801234013090911/pref_0 /content/mmseqs2/tmp/46/resultsDB_35 --sub-mat nucl:nucleotide.out,aa:blosum62.out -a 0 --alignment-mode 2 --wrapped-scoring 0 -e 0.001 --min-seq-id 0 --min-aln-len 0 --seq-id-mode 0 --alt-ali 0 -c 0 --cov-mode 0 --max-seq-len 65535 --comp-bias-corr 1 --realign 0 --max-rejected 2147483647 --max-accept 2147483647 --add-self-matches 0 --db-load-mode 0 --pca 1 --pcb 1.5 --score-bias 0 --gap-open nucl:5,aa:11 --gap-extend nucl:2,aa:1 --zdrop 40 --threads 4 --compressed 0 -v 3 \n","\n","Index version: 16\n","Generated by:  16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","ScoreMatrix:  VTML80.out\n","Compute score and coverage\n","Query database size: 7788 type: Aminoacid\n","Target database size: 468351 type: Aminoacid\n","Calculation of alignments\n","[=================================================================] 100.00% 7.79K 47s 231ms\n","Time for merging to resultsDB_35: 0h 0m 0s 2ms\n","\n","2107713 alignments calculated.\n","526577 sequence pairs passed the thresholds (0.249833 of overall calculated).\n","67.613892 hits per query sequence.\n","Time for processing: 0h 0m 47s 270ms\n","convertalis /content/mmseqs2/tmp/46/candidateDB_35 /content/mmseqs2/genexa/genexaDB /content/mmseqs2/tmp/46/resultsDB_35 /content/mmseqs2/tmp/46/resultDB_genexa.m8 --format-output query,target,evalue,raw \n","\n","MMseqs Version:        \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Substitution matrix    \tnucl:nucleotide.out,aa:blosum62.out\n","Alignment format       \t0\n","Format alignment output\tquery,target,evalue,raw\n","Translation table      \t1\n","Gap open cost          \tnucl:5,aa:11\n","Gap extension cost     \tnucl:2,aa:1\n","Database output        \tfalse\n","Preload mode           \t0\n","Search type            \t0\n","Threads                \t4\n","Compressed             \t0\n","Verbosity              \t3\n","\n","[=================================================================] 100.00% 7.79K 0s 377ms\n","Time for merging to resultDB_genexa.m8: 0h 0m 0s 14ms\n","Time for processing: 0h 0m 0s 467ms\n","search -s 7.0 /content/mmseqs2/tmp/46/candidateDB_53 /content/mmseqs2/swissprot/swissprotDB /content/mmseqs2/tmp/46/resultsDB_53 /content/mmseqs2/tmp/46 \n","\n","MMseqs Version:                     \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Substitution matrix                 \tnucl:nucleotide.out,aa:blosum62.out\n","Add backtrace                       \tfalse\n","Alignment mode                      \t2\n","Allow wrapped scoring               \tfalse\n","E-value threshold                   \t0.001\n","Seq. id. threshold                  \t0\n","Min alignment length                \t0\n","Seq. id. mode                       \t0\n","Alternative alignments              \t0\n","Coverage threshold                  \t0\n","Coverage mode                       \t0\n","Max sequence length                 \t65535\n","Compositional bias                  \t1\n","Realign hits                        \tfalse\n","Max reject                          \t2147483647\n","Max accept                          \t2147483647\n","Include identical seq. id.          \tfalse\n","Preload mode                        \t0\n","Pseudo count a                      \t1\n","Pseudo count b                      \t1.5\n","Score bias                          \t0\n","Gap open cost                       \tnucl:5,aa:11\n","Gap extension cost                  \tnucl:2,aa:1\n","Zdrop                               \t40\n","Threads                             \t4\n","Compressed                          \t0\n","Verbosity                           \t3\n","Seed substitution matrix            \tnucl:nucleotide.out,aa:VTML80.out\n","Sensitivity                         \t7\n","k-mer length                        \t0\n","k-score                             \t2147483647\n","Alphabet size                       \tnucl:5,aa:21\n","Max results per query               \t300\n","Split database                      \t0\n","Split mode                          \t2\n","Split memory limit                  \t0\n","Diagonal scoring                    \ttrue\n","Exact k-mer matching                \t0\n","Mask residues                       \t1\n","Mask lower case residues            \t0\n","Minimum diagonal score              \t15\n","Spaced k-mers                       \t1\n","Spaced k-mer pattern                \t\n","Local temporary path                \t\n","Rescore mode                        \t0\n","Remove hits by seq. id. and coverage\tfalse\n","Sort results                        \t0\n","Mask profile                        \t1\n","Profile e-value threshold           \t0.1\n","Global sequence weighting           \tfalse\n","Allow deletions                     \tfalse\n","Filter MSA                          \t1\n","Maximum seq. id. threshold          \t0.9\n","Minimum seq. id.                    \t0\n","Minimum score per column            \t-20\n","Minimum coverage                    \t0\n","Select N most diverse seqs          \t1000\n","Omit consensus                      \tfalse\n","Min codons in orf                   \t30\n","Max codons in length                \t32734\n","Max orf gaps                        \t2147483647\n","Contig start mode                   \t2\n","Contig end mode                     \t2\n","Orf start mode                      \t1\n","Forward frames                      \t1,2,3\n","Reverse frames                      \t1,2,3\n","Translation table                   \t1\n","Translate orf                       \t0\n","Use all table starts                \tfalse\n","Offset of numeric ids               \t0\n","Create lookup                       \t0\n","Add orf stop                        \tfalse\n","Chain overlapping alignments        \t0\n","Merge query                         \t1\n","Search type                         \t0\n","Search iterations                   \t1\n","Start sensitivity                   \t4\n","Search steps                        \t1\n","Slice search mode                   \tfalse\n","Strand selection                    \t1\n","Disk space limit                    \t0\n","MPI runner                          \t\n","Force restart with latest tmp       \tfalse\n","Remove temporary files              \tfalse\n","\n","prefilter /content/mmseqs2/tmp/46/candidateDB_53 /content/mmseqs2/swissprot/swissprotDB /content/mmseqs2/tmp/46/14247357018065594301/pref_0 --sub-mat nucl:nucleotide.out,aa:blosum62.out --seed-sub-mat nucl:nucleotide.out,aa:VTML80.out -k 0 --k-score 2147483647 --alph-size nucl:5,aa:21 --max-seq-len 65535 --max-seqs 300 --split 0 --split-mode 2 --split-memory-limit 0 -c 0 --cov-mode 0 --comp-bias-corr 1 --diag-score 1 --exact-kmer-matching 0 --mask 1 --mask-lower-case 0 --min-ungapped-score 15 --add-self-matches 0 --spaced-kmer-mode 1 --db-load-mode 0 --pca 1 --pcb 1.5 --threads 4 --compressed 0 -v 3 -s 7.0 \n","\n","Query database size: 7788 type: Aminoacid\n","Estimated memory consumption: 2G\n","Target database size: 562755 type: Aminoacid\n","Index table k-mer threshold: 100 at k-mer size 6 \n","Index table: counting k-mers\n","[=================================================================] 100.00% 562.76K 23s 989ms\n","Index table: Masked residues: 3164874\n","Index table: fill\n","[=================================================================] 100.00% 562.76K 17s 504ms\n","Index statistics\n","Entries:          192757622\n","DB size:          1591 MB\n","Avg k-mer size:   3.011838\n","Top 10 k-mers\n","    GPGGTL\t1835\n","    GQSWTV\t1706\n","    WGMFAT\t1638\n","    PGVFEV\t1634\n","    VLWQFW\t1623\n","    AYIRPN\t1587\n","    RSPKGV\t1585\n","    TPHKWY\t1560\n","    HIRVGW\t1548\n","    ITLSPY\t1541\n","Time for index table init: 0h 0m 43s 554ms\n","Process prefiltering step 1 of 1\n","\n","k-mer similarity threshold: 100\n","Starting prefiltering scores calculation (step 1 of 1)\n","Query db start 1 to 7788\n","Target db start 1 to 562755\n","[=================================================================] 100.00% 7.79K 57s 193ms\n","\n","973.067670 k-mers per position\n","604936 DB matches per sequence\n","414 overflows\n","0 queries produce too many hits (truncated result)\n","280 sequences passed prefiltering per query sequence\n","300 median result list length\n","1 sequences with 0 size result lists\n","Time for merging to pref_0: 0h 0m 0s 2ms\n","Time for processing: 0h 1m 41s 978ms\n","align /content/mmseqs2/tmp/46/candidateDB_53 /content/mmseqs2/swissprot/swissprotDB /content/mmseqs2/tmp/46/14247357018065594301/pref_0 /content/mmseqs2/tmp/46/resultsDB_53 --sub-mat nucl:nucleotide.out,aa:blosum62.out -a 0 --alignment-mode 2 --wrapped-scoring 0 -e 0.001 --min-seq-id 0 --min-aln-len 0 --seq-id-mode 0 --alt-ali 0 -c 0 --cov-mode 0 --max-seq-len 65535 --comp-bias-corr 1 --realign 0 --max-rejected 2147483647 --max-accept 2147483647 --add-self-matches 0 --db-load-mode 0 --pca 1 --pcb 1.5 --score-bias 0 --gap-open nucl:5,aa:11 --gap-extend nucl:2,aa:1 --zdrop 40 --threads 4 --compressed 0 -v 3 \n","\n","Compute score and coverage\n","Query database size: 7788 type: Aminoacid\n","Target database size: 562755 type: Aminoacid\n","Calculation of alignments\n","[=================================================================] 100.00% 7.79K 45s 193ms\n","Time for merging to resultsDB_53: 0h 0m 0s 2ms\n","\n","2185078 alignments calculated.\n","383099 sequence pairs passed the thresholds (0.175325 of overall calculated).\n","49.190933 hits per query sequence.\n","Time for processing: 0h 0m 45s 384ms\n","convertalis /content/mmseqs2/tmp/46/candidateDB_53 /content/mmseqs2/swissprot/swissprotDB /content/mmseqs2/tmp/46/resultsDB_53 /content/mmseqs2/tmp/46/resultDB_secondary.m8 --format-output query,target,evalue,raw \n","\n","MMseqs Version:        \t16e79a2a85eb895fce1a68a1ec37277f42cd376c\n","Substitution matrix    \tnucl:nucleotide.out,aa:blosum62.out\n","Alignment format       \t0\n","Format alignment output\tquery,target,evalue,raw\n","Translation table      \t1\n","Gap open cost          \tnucl:5,aa:11\n","Gap extension cost     \tnucl:2,aa:1\n","Database output        \tfalse\n","Preload mode           \t0\n","Search type            \t0\n","Threads                \t4\n","Compressed             \t0\n","Verbosity              \t3\n","\n","[=================================================================] 100.00% 7.79K 0s 273ms\n","Time for merging to resultDB_secondary.m8: 0h 0m 0s 11ms\n","Time for processing: 0h 0m 0s 378ms\n","found 4392 genes\n","\n","\n","Done\n"],"name":"stdout"}]},{"cell_type":"code","metadata":{"id":"M3_sRPUzDfkX","colab_type":"code","cellView":"form","colab":{"base_uri":"https://localhost:8080/","height":17},"executionInfo":{"status":"ok","timestamp":1593453435637,"user_tz":240,"elapsed":10153913,"user":{"displayName":"Markus Sommer","photoUrl":"https://lh3.googleusercontent.com/a-/AOh14GjXujbpCzD6oVvcRIT5EP-oi3Bif3Cfcu7npJUnce8=s64","userId":"08083804319459268544"}},"outputId":"f7a9c379-86bd-4c14-f2f5-cc132b2b3f34"},"source":["# @title Download genome annotation (you may need to rerun this cell and/or allow multiple downloads in browser)\n","# TODO support different output formats\n","\n","def write_GFF(start, end, contig, strand, contig_name_all, contig_length_all, contig_seq_all, GFF_path_out):\n","    # writes to same format as Prokka GFF\n","    bases_per_line = 60\n","    with open(GFF_path_out, \"wt\") as f:\n","        # header\n","        f.writelines([\"##gff-version 3\\n\"])\n","\n","        # contig names and lengths\n","        datalines = [\" \".join([\"##sequence-region\", \n","                               str(contig_name_all[i]), \n","                               \"1\", \n","                               str(contig_length_all[i]), \n","                               \"\\n\"]) for i in range(len(contig_name_all))]\n","        f.writelines(datalines)\n","\n","        # CDS features\n","        datalines = [\"\\t\".join([contig[i],\n","                                \"Balrog\",\n","                                \"CDS\",\n","                                str(int(start[i]) + 1),\n","                                end[i],\n","                                \".\", # TODO: replace . with actual gene score\n","                                strand[i],\n","                                \"0\",\n","                                \"inference=ab initio prediction:Balrog;product=hypothetical protein\"\n","                                \"\\n\"]) for i in range(len(start))]\n","        f.writelines(datalines)\n","\n","        # contig sequences\n","        f.writelines([\"##FASTA\", \"\\n\"])\n","        for i, name in enumerate(contig_name_all):\n","            f.writelines([\">\", str(name), \"\\n\"])\n","            bases_per_line = 60\n","            seq = contig_seq_all[i]\n","            datalines = [str(seq[j:j+bases_per_line])+\"\\n\" for j in range(0, len(seq), bases_per_line)]\n","            f.writelines(datalines)\n","\n","fasta_names = list(genome_dict.keys())\n","gff_names = [str(x) + \"__.gff\" for x in fasta_names] # simpler than removing all combinations of fasta and gz from the end\n","\n","for genome_idx, gff in enumerate(gff_names):\n","    # combine all info for gene predictions\n","    contig_gene_start_flat = []\n","    contig_gene_end_flat = []\n","    contig_gene_strand_flat = []\n","    contig_gene_contig_flat = []\n","    try:\n","        for i, contig_gene_coord in enumerate(contig_gene_coord_list[genome_idx]): # TODO get rid of jenky nested lists\n","            for k, coord in enumerate(contig_gene_coord):\n","                start = str(coord[0] - 3)\n","                end = str(coord[1] + 3)\n","                strandnum = contig_gene_strand_list[genome_idx][i][k]\n","                if strandnum == 1:\n","                    strand = \"+\"\n","                else:\n","                    strand = \"-\"\n","                contig = str(contig_name_list[genome_idx][i])\n","\n","                contig_gene_start_flat.append(start)\n","                contig_gene_end_flat.append(end)\n","                contig_gene_strand_flat.append(strand)\n","                contig_gene_contig_flat.append(contig)\n","\n","        write_GFF(contig_gene_start_flat, contig_gene_end_flat, contig_gene_contig_flat, contig_gene_strand_flat, \n","                contig_name_list[genome_idx], contig_length_list[genome_idx], contig_seq_list[genome_idx], gff)\n","    except:\n","        print(\"Could not generate \", gff)\n","\n","for gff in gff_names:\n","    try:\n","        files.download(gff)\n","    except:\n","        print(\"Could not download \", gff)"],"execution_count":18,"outputs":[{"output_type":"display_data","data":{"application/javascript":["\n","    async function download(id, filename, size) {\n","      if (!google.colab.kernel.accessAllowed) {\n","        return;\n","      }\n","      const div = document.createElement('div');\n","      const label = document.createElement('label');\n","      label.textContent = `Downloading \"${filename}\": `;\n","      div.appendChild(label);\n","      const progress = document.createElement('progress');\n","      progress.max = size;\n","      div.appendChild(progress);\n","      document.body.appendChild(div);\n","\n","      const buffers = [];\n","      let downloaded = 0;\n","\n","      const channel = await google.colab.kernel.comms.open(id);\n","      // Send a message to notify the kernel that we're ready.\n","      channel.send({})\n","\n","      for await (const message of channel.messages) {\n","        // Send a message to notify the kernel that we're ready.\n","        channel.send({})\n","        if (message.buffers) {\n","          for (const buffer of message.buffers) {\n","            buffers.push(buffer);\n","            downloaded += buffer.byteLength;\n","            progress.value = downloaded;\n","          }\n","        }\n","      }\n","      const blob = new Blob(buffers, {type: 'application/binary'});\n","      const a = document.createElement('a');\n","      a.href = window.URL.createObjectURL(blob);\n","      a.download = filename;\n","      div.appendChild(a);\n","      a.click();\n","      div.remove();\n","    }\n","  "],"text/plain":["<IPython.core.display.Javascript object>"]},"metadata":{"tags":[]}},{"output_type":"display_data","data":{"application/javascript":["download(\"download_a98b430d-f0f1-48d7-86d6-26014babcd72\", \"GCF_900465355.1_Ran1-1_genomic.fna.gz__.gff\", 1485893)"],"text/plain":["<IPython.core.display.Javascript object>"]},"metadata":{"tags":[]}},{"output_type":"display_data","data":{"application/javascript":["\n","    async function download(id, filename, size) {\n","      if (!google.colab.kernel.accessAllowed) {\n","        return;\n","      }\n","      const div = document.createElement('div');\n","      const label = document.createElement('label');\n","      label.textContent = `Downloading \"${filename}\": `;\n","      div.appendChild(label);\n","      const progress = document.createElement('progress');\n","      progress.max = size;\n","      div.appendChild(progress);\n","      document.body.appendChild(div);\n","\n","      const buffers = [];\n","      let downloaded = 0;\n","\n","      const channel = await google.colab.kernel.comms.open(id);\n","      // Send a message to notify the kernel that we're ready.\n","      channel.send({})\n","\n","      for await (const message of channel.messages) {\n","        // Send a message to notify the kernel that we're ready.\n","        channel.send({})\n","        if (message.buffers) {\n","          for (const buffer of message.buffers) {\n","            buffers.push(buffer);\n","            downloaded += buffer.byteLength;\n","            progress.value = downloaded;\n","          }\n","        }\n","      }\n","      const blob = new Blob(buffers, {type: 'application/binary'});\n","      const a = document.createElement('a');\n","      a.href = window.URL.createObjectURL(blob);\n","      a.download = filename;\n","      div.appendChild(a);\n","      a.click();\n","      div.remove();\n","    }\n","  "],"text/plain":["<IPython.core.display.Javascript object>"]},"metadata":{"tags":[]}},{"output_type":"display_data","data":{"application/javascript":["download(\"download_fbd73655-3937-4d23-8a45-98c47dcfc045\", \"GCF_003047205.1_ASM304720v1_genomic.fna.gz__.gff\", 3021136)"],"text/plain":["<IPython.core.display.Javascript object>"]},"metadata":{"tags":[]}},{"output_type":"display_data","data":{"application/javascript":["\n","    async function download(id, filename, size) {\n","      if (!google.colab.kernel.accessAllowed) {\n","        return;\n","      }\n","      const div = document.createElement('div');\n","      const label = document.createElement('label');\n","      label.textContent = `Downloading \"${filename}\": `;\n","      div.appendChild(label);\n","      const progress = document.createElement('progress');\n","      progress.max = size;\n","      div.appendChild(progress);\n","      document.body.appendChild(div);\n","\n","      const buffers = [];\n","      let downloaded = 0;\n","\n","      const channel = await google.colab.kernel.comms.open(id);\n","      // Send a message to notify the kernel that we're ready.\n","      channel.send({})\n","\n","      for await (const message of channel.messages) {\n","        // Send a message to notify the kernel that we're ready.\n","        channel.send({})\n","        if (message.buffers) {\n","          for (const buffer of message.buffers) {\n","            buffers.push(buffer);\n","            downloaded += buffer.byteLength;\n","            progress.value = downloaded;\n","          }\n","        }\n","      }\n","      const blob = new Blob(buffers, {type: 'application/binary'});\n","      const a = document.createElement('a');\n","      a.href = window.URL.createObjectURL(blob);\n","      a.download = filename;\n","      div.appendChild(a);\n","      a.click();\n","      div.remove();\n","    }\n","  "],"text/plain":["<IPython.core.display.Javascript object>"]},"metadata":{"tags":[]}},{"output_type":"display_data","data":{"application/javascript":["download(\"download_01dee9ed-0d4c-460c-a615-0bc779b52edf\", \"GCF_002142475.1_ASM214247v1_genomic.fna.gz__.gff\", 7738107)"],"text/plain":["<IPython.core.display.Javascript object>"]},"metadata":{"tags":[]}},{"output_type":"display_data","data":{"application/javascript":["\n","    async function download(id, filename, size) {\n","      if (!google.colab.kernel.accessAllowed) {\n","        return;\n","      }\n","      const div = document.createElement('div');\n","      const label = document.createElement('label');\n","      label.textContent = `Downloading \"${filename}\": `;\n","      div.appendChild(label);\n","      const progress = document.createElement('progress');\n","      progress.max = size;\n","      div.appendChild(progress);\n","      document.body.appendChild(div);\n","\n","      const buffers = [];\n","      let downloaded = 0;\n","\n","      const channel = await google.colab.kernel.comms.open(id);\n","      // Send a message to notify the kernel that we're ready.\n","      channel.send({})\n","\n","      for await (const message of channel.messages) {\n","        // Send a message to notify the kernel that we're ready.\n","        channel.send({})\n","        if (message.buffers) {\n","          for (const buffer of message.buffers) {\n","            buffers.push(buffer);\n","            downloaded += buffer.byteLength;\n","            progress.value = downloaded;\n","          }\n","        }\n","      }\n","      const blob = new Blob(buffers, {type: 'application/binary'});\n","      const a = document.createElement('a');\n","      a.href = window.URL.createObjectURL(blob);\n","      a.download = filename;\n","      div.appendChild(a);\n","      a.click();\n","      div.remove();\n","    }\n","  "],"text/plain":["<IPython.core.display.Javascript object>"]},"metadata":{"tags":[]}},{"output_type":"display_data","data":{"application/javascript":["download(\"download_e357cd22-a504-4f58-9a4a-bfa066c3f318\", \"GCF_002113825.1_ASM211382v1_genomic.fna.gz__.gff\", 2718758)"],"text/plain":["<IPython.core.display.Javascript object>"]},"metadata":{"tags":[]}},{"output_type":"display_data","data":{"application/javascript":["\n","    async function download(id, filename, size) {\n","      if (!google.colab.kernel.accessAllowed) {\n","        return;\n","      }\n","      const div = document.createElement('div');\n","      const label = document.createElement('label');\n","      label.textContent = `Downloading \"${filename}\": `;\n","      div.appendChild(label);\n","      const progress = document.createElement('progress');\n","      progress.max = size;\n","      div.appendChild(progress);\n","      document.body.appendChild(div);\n","\n","      const buffers = [];\n","      let downloaded = 0;\n","\n","      const channel = await google.colab.kernel.comms.open(id);\n","      // Send a message to notify the kernel that we're ready.\n","      channel.send({})\n","\n","      for await (const message of channel.messages) {\n","        // Send a message to notify the kernel that we're ready.\n","        channel.send({})\n","        if (message.buffers) {\n","          for (const buffer of message.buffers) {\n","            buffers.push(buffer);\n","            downloaded += buffer.byteLength;\n","            progress.value = downloaded;\n","          }\n","        }\n","      }\n","      const blob = new Blob(buffers, {type: 'application/binary'});\n","      const a = document.createElement('a');\n","      a.href = window.URL.createObjectURL(blob);\n","      a.download = filename;\n","      div.appendChild(a);\n","      a.click();\n","      div.remove();\n","    }\n","  "],"text/plain":["<IPython.core.display.Javascript object>"]},"metadata":{"tags":[]}},{"output_type":"display_data","data":{"application/javascript":["download(\"download_b76bd64d-7ed2-49dc-89db-aca54e133c19\", \"GCF_002009335.2_ASM200933v2_genomic.fna.gz__.gff\", 4958473)"],"text/plain":["<IPython.core.display.Javascript object>"]},"metadata":{"tags":[]}},{"output_type":"display_data","data":{"application/javascript":["\n","    async function download(id, filename, size) {\n","      if (!google.colab.kernel.accessAllowed) {\n","        return;\n","      }\n","      const div = document.createElement('div');\n","      const label = document.createElement('label');\n","      label.textContent = `Downloading \"${filename}\": `;\n","      div.appendChild(label);\n","      const progress = document.createElement('progress');\n","      progress.max = size;\n","      div.appendChild(progress);\n","      document.body.appendChild(div);\n","\n","      const buffers = [];\n","      let downloaded = 0;\n","\n","      const channel = await google.colab.kernel.comms.open(id);\n","      // Send a message to notify the kernel that we're ready.\n","      channel.send({})\n","\n","      for await (const message of channel.messages) {\n","        // Send a message to notify the kernel that we're ready.\n","        channel.send({})\n","        if (message.buffers) {\n","          for (const buffer of message.buffers) {\n","            buffers.push(buffer);\n","            downloaded += buffer.byteLength;\n","            progress.value = downloaded;\n","          }\n","        }\n","      }\n","      const blob = new Blob(buffers, {type: 'application/binary'});\n","      const a = document.createElement('a');\n","      a.href = window.URL.createObjectURL(blob);\n","      a.download = filename;\n","      div.appendChild(a);\n","      a.click();\n","      div.remove();\n","    }\n","  "],"text/plain":["<IPython.core.display.Javascript object>"]},"metadata":{"tags":[]}},{"output_type":"display_data","data":{"application/javascript":["download(\"download_f114977c-0a1b-4995-9cce-94f71aee7d43\", \"GCF_002005165.1_ASM200516v1_genomic.fna.gz__.gff\", 4093329)"],"text/plain":["<IPython.core.display.Javascript object>"]},"metadata":{"tags":[]}},{"output_type":"display_data","data":{"application/javascript":["\n","    async function download(id, filename, size) {\n","      if (!google.colab.kernel.accessAllowed) {\n","        return;\n","      }\n","      const div = document.createElement('div');\n","      const label = document.createElement('label');\n","      label.textContent = `Downloading \"${filename}\": `;\n","      div.appendChild(label);\n","      const progress = document.createElement('progress');\n","      progress.max = size;\n","      div.appendChild(progress);\n","      document.body.appendChild(div);\n","\n","      const buffers = [];\n","      let downloaded = 0;\n","\n","      const channel = await google.colab.kernel.comms.open(id);\n","      // Send a message to notify the kernel that we're ready.\n","      channel.send({})\n","\n","      for await (const message of channel.messages) {\n","        // Send a message to notify the kernel that we're ready.\n","        channel.send({})\n","        if (message.buffers) {\n","          for (const buffer of message.buffers) {\n","            buffers.push(buffer);\n","            downloaded += buffer.byteLength;\n","            progress.value = downloaded;\n","          }\n","        }\n","      }\n","      const blob = new Blob(buffers, {type: 'application/binary'});\n","      const a = document.createElement('a');\n","      a.href = window.URL.createObjectURL(blob);\n","      a.download = filename;\n","      div.appendChild(a);\n","      a.click();\n","      div.remove();\n","    }\n","  "],"text/plain":["<IPython.core.display.Javascript object>"]},"metadata":{"tags":[]}},{"output_type":"display_data","data":{"application/javascript":["download(\"download_375e0fc2-21b1-4bcc-bdac-3dd65a9f1349\", \"GCF_001729945.1_ASM172994v1_genomic.fna.gz__.gff\", 3356467)"],"text/plain":["<IPython.core.display.Javascript object>"]},"metadata":{"tags":[]}},{"output_type":"display_data","data":{"application/javascript":["\n","    async function download(id, filename, size) {\n","      if (!google.colab.kernel.accessAllowed) {\n","        return;\n","      }\n","      const div = document.createElement('div');\n","      const label = document.createElement('label');\n","      label.textContent = `Downloading \"${filename}\": `;\n","      div.appendChild(label);\n","      const progress = document.createElement('progress');\n","      progress.max = size;\n","      div.appendChild(progress);\n","      document.body.appendChild(div);\n","\n","      const buffers = [];\n","      let downloaded = 0;\n","\n","      const channel = await google.colab.kernel.comms.open(id);\n","      // Send a message to notify the kernel that we're ready.\n","      channel.send({})\n","\n","      for await (const message of channel.messages) {\n","        // Send a message to notify the kernel that we're ready.\n","        channel.send({})\n","        if (message.buffers) {\n","          for (const buffer of message.buffers) {\n","            buffers.push(buffer);\n","            downloaded += buffer.byteLength;\n","            progress.value = downloaded;\n","          }\n","        }\n","      }\n","      const blob = new Blob(buffers, {type: 'application/binary'});\n","      const a = document.createElement('a');\n","      a.href = window.URL.createObjectURL(blob);\n","      a.download = filename;\n","      div.appendChild(a);\n","      a.click();\n","      div.remove();\n","    }\n","  "],"text/plain":["<IPython.core.display.Javascript object>"]},"metadata":{"tags":[]}},{"output_type":"display_data","data":{"application/javascript":["download(\"download_fa613347-dfd0-4db1-bbd4-0a6b1e3a6b98\", \"GCF_001547735.1_ASM154773v1_genomic.fna.gz__.gff\", 2052090)"],"text/plain":["<IPython.core.display.Javascript object>"]},"metadata":{"tags":[]}},{"output_type":"display_data","data":{"application/javascript":["\n","    async function download(id, filename, size) {\n","      if (!google.colab.kernel.accessAllowed) {\n","        return;\n","      }\n","      const div = document.createElement('div');\n","      const label = document.createElement('label');\n","      label.textContent = `Downloading \"${filename}\": `;\n","      div.appendChild(label);\n","      const progress = document.createElement('progress');\n","      progress.max = size;\n","      div.appendChild(progress);\n","      document.body.appendChild(div);\n","\n","      const buffers = [];\n","      let downloaded = 0;\n","\n","      const channel = await google.colab.kernel.comms.open(id);\n","      // Send a message to notify the kernel that we're ready.\n","      channel.send({})\n","\n","      for await (const message of channel.messages) {\n","        // Send a message to notify the kernel that we're ready.\n","        channel.send({})\n","        if (message.buffers) {\n","          for (const buffer of message.buffers) {\n","            buffers.push(buffer);\n","            downloaded += buffer.byteLength;\n","            progress.value = downloaded;\n","          }\n","        }\n","      }\n","      const blob = new Blob(buffers, {type: 'application/binary'});\n","      const a = document.createElement('a');\n","      a.href = window.URL.createObjectURL(blob);\n","      a.download = filename;\n","      div.appendChild(a);\n","      a.click();\n","      div.remove();\n","    }\n","  "],"text/plain":["<IPython.core.display.Javascript object>"]},"metadata":{"tags":[]}},{"output_type":"display_data","data":{"application/javascript":["download(\"download_25ba3351-8359-4944-b9f9-7bec85f4bb6b\", \"GCF_001263175.1_ASM126317v1_genomic.fna.gz__.gff\", 4832063)"],"text/plain":["<IPython.core.display.Javascript object>"]},"metadata":{"tags":[]}},{"output_type":"display_data","data":{"application/javascript":["\n","    async function download(id, filename, size) {\n","      if (!google.colab.kernel.accessAllowed) {\n","        return;\n","      }\n","      const div = document.createElement('div');\n","      const label = document.createElement('label');\n","      label.textContent = `Downloading \"${filename}\": `;\n","      div.appendChild(label);\n","      const progress = document.createElement('progress');\n","      progress.max = size;\n","      div.appendChild(progress);\n","      document.body.appendChild(div);\n","\n","      const buffers = [];\n","      let downloaded = 0;\n","\n","      const channel = await google.colab.kernel.comms.open(id);\n","      // Send a message to notify the kernel that we're ready.\n","      channel.send({})\n","\n","      for await (const message of channel.messages) {\n","        // Send a message to notify the kernel that we're ready.\n","        channel.send({})\n","        if (message.buffers) {\n","          for (const buffer of message.buffers) {\n","            buffers.push(buffer);\n","            downloaded += buffer.byteLength;\n","            progress.value = downloaded;\n","          }\n","        }\n","      }\n","      const blob = new Blob(buffers, {type: 'application/binary'});\n","      const a = document.createElement('a');\n","      a.href = window.URL.createObjectURL(blob);\n","      a.download = filename;\n","      div.appendChild(a);\n","      a.click();\n","      div.remove();\n","    }\n","  "],"text/plain":["<IPython.core.display.Javascript object>"]},"metadata":{"tags":[]}},{"output_type":"display_data","data":{"application/javascript":["download(\"download_a1f92501-bcdb-47e7-a148-cc4d7f850fc8\", \"GCF_001027025.1_ASM102702v1_genomic.fna.gz__.gff\", 2432910)"],"text/plain":["<IPython.core.display.Javascript object>"]},"metadata":{"tags":[]}},{"output_type":"display_data","data":{"application/javascript":["\n","    async function download(id, filename, size) {\n","      if (!google.colab.kernel.accessAllowed) {\n","        return;\n","      }\n","      const div = document.createElement('div');\n","      const label = document.createElement('label');\n","      label.textContent = `Downloading \"${filename}\": `;\n","      div.appendChild(label);\n","      const progress = document.createElement('progress');\n","      progress.max = size;\n","      div.appendChild(progress);\n","      document.body.appendChild(div);\n","\n","      const buffers = [];\n","      let downloaded = 0;\n","\n","      const channel = await google.colab.kernel.comms.open(id);\n","      // Send a message to notify the kernel that we're ready.\n","      channel.send({})\n","\n","      for await (const message of channel.messages) {\n","        // Send a message to notify the kernel that we're ready.\n","        channel.send({})\n","        if (message.buffers) {\n","          for (const buffer of message.buffers) {\n","            buffers.push(buffer);\n","            downloaded += buffer.byteLength;\n","            progress.value = downloaded;\n","          }\n","        }\n","      }\n","      const blob = new Blob(buffers, {type: 'application/binary'});\n","      const a = document.createElement('a');\n","      a.href = window.URL.createObjectURL(blob);\n","      a.download = filename;\n","      div.appendChild(a);\n","      a.click();\n","      div.remove();\n","    }\n","  "],"text/plain":["<IPython.core.display.Javascript object>"]},"metadata":{"tags":[]}},{"output_type":"display_data","data":{"application/javascript":["download(\"download_228101f9-7d71-4898-8e51-9d8c0dce4e4a\", \"GCF_000814265.1_ASM81426v1_genomic.fna.gz__.gff\", 1996946)"],"text/plain":["<IPython.core.display.Javascript object>"]},"metadata":{"tags":[]}},{"output_type":"display_data","data":{"application/javascript":["\n","    async function download(id, filename, size) {\n","      if (!google.colab.kernel.accessAllowed) {\n","        return;\n","      }\n","      const div = document.createElement('div');\n","      const label = document.createElement('label');\n","      label.textContent = `Downloading \"${filename}\": `;\n","      div.appendChild(label);\n","      const progress = document.createElement('progress');\n","      progress.max = size;\n","      div.appendChild(progress);\n","      document.body.appendChild(div);\n","\n","      const buffers = [];\n","      let downloaded = 0;\n","\n","      const channel = await google.colab.kernel.comms.open(id);\n","      // Send a message to notify the kernel that we're ready.\n","      channel.send({})\n","\n","      for await (const message of channel.messages) {\n","        // Send a message to notify the kernel that we're ready.\n","        channel.send({})\n","        if (message.buffers) {\n","          for (const buffer of message.buffers) {\n","            buffers.push(buffer);\n","            downloaded += buffer.byteLength;\n","            progress.value = downloaded;\n","          }\n","        }\n","      }\n","      const blob = new Blob(buffers, {type: 'application/binary'});\n","      const a = document.createElement('a');\n","      a.href = window.URL.createObjectURL(blob);\n","      a.download = filename;\n","      div.appendChild(a);\n","      a.click();\n","      div.remove();\n","    }\n","  "],"text/plain":["<IPython.core.display.Javascript object>"]},"metadata":{"tags":[]}},{"output_type":"display_data","data":{"application/javascript":["download(\"download_46487d4e-d2c3-4057-9edf-eeea85ef9bd6\", \"GCF_000737325.1_ASM73732v2_genomic.fna.gz__.gff\", 11455943)"],"text/plain":["<IPython.core.display.Javascript object>"]},"metadata":{"tags":[]}},{"output_type":"display_data","data":{"application/javascript":["\n","    async function download(id, filename, size) {\n","      if (!google.colab.kernel.accessAllowed) {\n","        return;\n","      }\n","      const div = document.createElement('div');\n","      const label = document.createElement('label');\n","      label.textContent = `Downloading \"${filename}\": `;\n","      div.appendChild(label);\n","      const progress = document.createElement('progress');\n","      progress.max = size;\n","      div.appendChild(progress);\n","      document.body.appendChild(div);\n","\n","      const buffers = [];\n","      let downloaded = 0;\n","\n","      const channel = await google.colab.kernel.comms.open(id);\n","      // Send a message to notify the kernel that we're ready.\n","      channel.send({})\n","\n","      for await (const message of channel.messages) {\n","        // Send a message to notify the kernel that we're ready.\n","        channel.send({})\n","        if (message.buffers) {\n","          for (const buffer of message.buffers) {\n","            buffers.push(buffer);\n","            downloaded += buffer.byteLength;\n","            progress.value = downloaded;\n","          }\n","        }\n","      }\n","      const blob = new Blob(buffers, {type: 'application/binary'});\n","      const a = document.createElement('a');\n","      a.href = window.URL.createObjectURL(blob);\n","      a.download = filename;\n","      div.appendChild(a);\n","      a.click();\n","      div.remove();\n","    }\n","  "],"text/plain":["<IPython.core.display.Javascript object>"]},"metadata":{"tags":[]}},{"output_type":"display_data","data":{"application/javascript":["download(\"download_0f49956d-e050-469b-83ac-4eb6b5a25bb6\", \"GCF_000725345.1_ASM72534v1_genomic.fna.gz__.gff\", 5720553)"],"text/plain":["<IPython.core.display.Javascript object>"]},"metadata":{"tags":[]}},{"output_type":"display_data","data":{"application/javascript":["\n","    async function download(id, filename, size) {\n","      if (!google.colab.kernel.accessAllowed) {\n","        return;\n","      }\n","      const div = document.createElement('div');\n","      const label = document.createElement('label');\n","      label.textContent = `Downloading \"${filename}\": `;\n","      div.appendChild(label);\n","      const progress = document.createElement('progress');\n","      progress.max = size;\n","      div.appendChild(progress);\n","      document.body.appendChild(div);\n","\n","      const buffers = [];\n","      let downloaded = 0;\n","\n","      const channel = await google.colab.kernel.comms.open(id);\n","      // Send a message to notify the kernel that we're ready.\n","      channel.send({})\n","\n","      for await (const message of channel.messages) {\n","        // Send a message to notify the kernel that we're ready.\n","        channel.send({})\n","        if (message.buffers) {\n","          for (const buffer of message.buffers) {\n","            buffers.push(buffer);\n","            downloaded += buffer.byteLength;\n","            progress.value = downloaded;\n","          }\n","        }\n","      }\n","      const blob = new Blob(buffers, {type: 'application/binary'});\n","      const a = document.createElement('a');\n","      a.href = window.URL.createObjectURL(blob);\n","      a.download = filename;\n","      div.appendChild(a);\n","      a.click();\n","      div.remove();\n","    }\n","  "],"text/plain":["<IPython.core.display.Javascript object>"]},"metadata":{"tags":[]}},{"output_type":"display_data","data":{"application/javascript":["download(\"download_b8e3e400-f0fe-4a1a-9c61-0fd35c473002\", \"GCF_000661895.1_ASM66189v1_genomic.fna.gz__.gff\", 3746363)"],"text/plain":["<IPython.core.display.Javascript object>"]},"metadata":{"tags":[]}},{"output_type":"display_data","data":{"application/javascript":["\n","    async function download(id, filename, size) {\n","      if (!google.colab.kernel.accessAllowed) {\n","        return;\n","      }\n","      const div = document.createElement('div');\n","      const label = document.createElement('label');\n","      label.textContent = `Downloading \"${filename}\": `;\n","      div.appendChild(label);\n","      const progress = document.createElement('progress');\n","      progress.max = size;\n","      div.appendChild(progress);\n","      document.body.appendChild(div);\n","\n","      const buffers = [];\n","      let downloaded = 0;\n","\n","      const channel = await google.colab.kernel.comms.open(id);\n","      // Send a message to notify the kernel that we're ready.\n","      channel.send({})\n","\n","      for await (const message of channel.messages) {\n","        // Send a message to notify the kernel that we're ready.\n","        channel.send({})\n","        if (message.buffers) {\n","          for (const buffer of message.buffers) {\n","            buffers.push(buffer);\n","            downloaded += buffer.byteLength;\n","            progress.value = downloaded;\n","          }\n","        }\n","      }\n","      const blob = new Blob(buffers, {type: 'application/binary'});\n","      const a = document.createElement('a');\n","      a.href = window.URL.createObjectURL(blob);\n","      a.download = filename;\n","      div.appendChild(a);\n","      a.click();\n","      div.remove();\n","    }\n","  "],"text/plain":["<IPython.core.display.Javascript object>"]},"metadata":{"tags":[]}},{"output_type":"display_data","data":{"application/javascript":["download(\"download_1f9baa48-a21c-44e9-a7ca-e554434b880c\", \"GCF_000525675.1_ASM52567v1_genomic.fna.gz__.gff\", 3367301)"],"text/plain":["<IPython.core.display.Javascript object>"]},"metadata":{"tags":[]}},{"output_type":"display_data","data":{"application/javascript":["\n","    async function download(id, filename, size) {\n","      if (!google.colab.kernel.accessAllowed) {\n","        return;\n","      }\n","      const div = document.createElement('div');\n","      const label = document.createElement('label');\n","      label.textContent = `Downloading \"${filename}\": `;\n","      div.appendChild(label);\n","      const progress = document.createElement('progress');\n","      progress.max = size;\n","      div.appendChild(progress);\n","      document.body.appendChild(div);\n","\n","      const buffers = [];\n","      let downloaded = 0;\n","\n","      const channel = await google.colab.kernel.comms.open(id);\n","      // Send a message to notify the kernel that we're ready.\n","      channel.send({})\n","\n","      for await (const message of channel.messages) {\n","        // Send a message to notify the kernel that we're ready.\n","        channel.send({})\n","        if (message.buffers) {\n","          for (const buffer of message.buffers) {\n","            buffers.push(buffer);\n","            downloaded += buffer.byteLength;\n","            progress.value = downloaded;\n","          }\n","        }\n","      }\n","      const blob = new Blob(buffers, {type: 'application/binary'});\n","      const a = document.createElement('a');\n","      a.href = window.URL.createObjectURL(blob);\n","      a.download = filename;\n","      div.appendChild(a);\n","      a.click();\n","      div.remove();\n","    }\n","  "],"text/plain":["<IPython.core.display.Javascript object>"]},"metadata":{"tags":[]}},{"output_type":"display_data","data":{"application/javascript":["download(\"download_9bdf13f6-bb20-42ad-a900-812d2b692f84\", \"GCF_000219875.1_ASM21987v1_genomic.fna.gz__.gff\", 3513023)"],"text/plain":["<IPython.core.display.Javascript object>"]},"metadata":{"tags":[]}},{"output_type":"display_data","data":{"application/javascript":["\n","    async function download(id, filename, size) {\n","      if (!google.colab.kernel.accessAllowed) {\n","        return;\n","      }\n","      const div = document.createElement('div');\n","      const label = document.createElement('label');\n","      label.textContent = `Downloading \"${filename}\": `;\n","      div.appendChild(label);\n","      const progress = document.createElement('progress');\n","      progress.max = size;\n","      div.appendChild(progress);\n","      document.body.appendChild(div);\n","\n","      const buffers = [];\n","      let downloaded = 0;\n","\n","      const channel = await google.colab.kernel.comms.open(id);\n","      // Send a message to notify the kernel that we're ready.\n","      channel.send({})\n","\n","      for await (const message of channel.messages) {\n","        // Send a message to notify the kernel that we're ready.\n","        channel.send({})\n","        if (message.buffers) {\n","          for (const buffer of message.buffers) {\n","            buffers.push(buffer);\n","            downloaded += buffer.byteLength;\n","            progress.value = downloaded;\n","          }\n","        }\n","      }\n","      const blob = new Blob(buffers, {type: 'application/binary'});\n","      const a = document.createElement('a');\n","      a.href = window.URL.createObjectURL(blob);\n","      a.download = filename;\n","      div.appendChild(a);\n","      a.click();\n","      div.remove();\n","    }\n","  "],"text/plain":["<IPython.core.display.Javascript object>"]},"metadata":{"tags":[]}},{"output_type":"display_data","data":{"application/javascript":["download(\"download_ea976a48-a799-4af7-a678-989c692dce94\", \"GCF_000212395.1_ASM21239v1_genomic.fna.gz__.gff\", 2133829)"],"text/plain":["<IPython.core.display.Javascript object>"]},"metadata":{"tags":[]}},{"output_type":"display_data","data":{"application/javascript":["\n","    async function download(id, filename, size) {\n","      if (!google.colab.kernel.accessAllowed) {\n","        return;\n","      }\n","      const div = document.createElement('div');\n","      const label = document.createElement('label');\n","      label.textContent = `Downloading \"${filename}\": `;\n","      div.appendChild(label);\n","      const progress = document.createElement('progress');\n","      progress.max = size;\n","      div.appendChild(progress);\n","      document.body.appendChild(div);\n","\n","      const buffers = [];\n","      let downloaded = 0;\n","\n","      const channel = await google.colab.kernel.comms.open(id);\n","      // Send a message to notify the kernel that we're ready.\n","      channel.send({})\n","\n","      for await (const message of channel.messages) {\n","        // Send a message to notify the kernel that we're ready.\n","        channel.send({})\n","        if (message.buffers) {\n","          for (const buffer of message.buffers) {\n","            buffers.push(buffer);\n","            downloaded += buffer.byteLength;\n","            progress.value = downloaded;\n","          }\n","        }\n","      }\n","      const blob = new Blob(buffers, {type: 'application/binary'});\n","      const a = document.createElement('a');\n","      a.href = window.URL.createObjectURL(blob);\n","      a.download = filename;\n","      div.appendChild(a);\n","      a.click();\n","      div.remove();\n","    }\n","  "],"text/plain":["<IPython.core.display.Javascript object>"]},"metadata":{"tags":[]}},{"output_type":"display_data","data":{"application/javascript":["download(\"download_dc19c6d0-1ab8-41b5-bb44-505704919b94\", \"GCF_000195295.1_ASM19529v1_genomic.fna.gz__.gff\", 3655679)"],"text/plain":["<IPython.core.display.Javascript object>"]},"metadata":{"tags":[]}},{"output_type":"display_data","data":{"application/javascript":["\n","    async function download(id, filename, size) {\n","      if (!google.colab.kernel.accessAllowed) {\n","        return;\n","      }\n","      const div = document.createElement('div');\n","      const label = document.createElement('label');\n","      label.textContent = `Downloading \"${filename}\": `;\n","      div.appendChild(label);\n","      const progress = document.createElement('progress');\n","      progress.max = size;\n","      div.appendChild(progress);\n","      document.body.appendChild(div);\n","\n","      const buffers = [];\n","      let downloaded = 0;\n","\n","      const channel = await google.colab.kernel.comms.open(id);\n","      // Send a message to notify the kernel that we're ready.\n","      channel.send({})\n","\n","      for await (const message of channel.messages) {\n","        // Send a message to notify the kernel that we're ready.\n","        channel.send({})\n","        if (message.buffers) {\n","          for (const buffer of message.buffers) {\n","            buffers.push(buffer);\n","            downloaded += buffer.byteLength;\n","            progress.value = downloaded;\n","          }\n","        }\n","      }\n","      const blob = new Blob(buffers, {type: 'application/binary'});\n","      const a = document.createElement('a');\n","      a.href = window.URL.createObjectURL(blob);\n","      a.download = filename;\n","      div.appendChild(a);\n","      a.click();\n","      div.remove();\n","    }\n","  "],"text/plain":["<IPython.core.display.Javascript object>"]},"metadata":{"tags":[]}},{"output_type":"display_data","data":{"application/javascript":["download(\"download_8e959d01-99d1-464e-a1f9-0e2f7dce1a6f\", \"GCF_000191045.1_ASM19104v1_genomic.fna.gz__.gff\", 1737429)"],"text/plain":["<IPython.core.display.Javascript object>"]},"metadata":{"tags":[]}},{"output_type":"display_data","data":{"application/javascript":["\n","    async function download(id, filename, size) {\n","      if (!google.colab.kernel.accessAllowed) {\n","        return;\n","      }\n","      const div = document.createElement('div');\n","      const label = document.createElement('label');\n","      label.textContent = `Downloading \"${filename}\": `;\n","      div.appendChild(label);\n","      const progress = document.createElement('progress');\n","      progress.max = size;\n","      div.appendChild(progress);\n","      document.body.appendChild(div);\n","\n","      const buffers = [];\n","      let downloaded = 0;\n","\n","      const channel = await google.colab.kernel.comms.open(id);\n","      // Send a message to notify the kernel that we're ready.\n","      channel.send({})\n","\n","      for await (const message of channel.messages) {\n","        // Send a message to notify the kernel that we're ready.\n","        channel.send({})\n","        if (message.buffers) {\n","          for (const buffer of message.buffers) {\n","            buffers.push(buffer);\n","            downloaded += buffer.byteLength;\n","            progress.value = downloaded;\n","          }\n","        }\n","      }\n","      const blob = new Blob(buffers, {type: 'application/binary'});\n","      const a = document.createElement('a');\n","      a.href = window.URL.createObjectURL(blob);\n","      a.download = filename;\n","      div.appendChild(a);\n","      a.click();\n","      div.remove();\n","    }\n","  "],"text/plain":["<IPython.core.display.Javascript object>"]},"metadata":{"tags":[]}},{"output_type":"display_data","data":{"application/javascript":["download(\"download_d2841510-8c8e-4727-8240-011c0f291ba9\", \"GCF_000189295.2_ASM18929v2_genomic.fna.gz__.gff\", 4307101)"],"text/plain":["<IPython.core.display.Javascript object>"]},"metadata":{"tags":[]}},{"output_type":"display_data","data":{"application/javascript":["\n","    async function download(id, filename, size) {\n","      if (!google.colab.kernel.accessAllowed) {\n","        return;\n","      }\n","      const div = document.createElement('div');\n","      const label = document.createElement('label');\n","      label.textContent = `Downloading \"${filename}\": `;\n","      div.appendChild(label);\n","      const progress = document.createElement('progress');\n","      progress.max = size;\n","      div.appendChild(progress);\n","      document.body.appendChild(div);\n","\n","      const buffers = [];\n","      let downloaded = 0;\n","\n","      const channel = await google.colab.kernel.comms.open(id);\n","      // Send a message to notify the kernel that we're ready.\n","      channel.send({})\n","\n","      for await (const message of channel.messages) {\n","        // Send a message to notify the kernel that we're ready.\n","        channel.send({})\n","        if (message.buffers) {\n","          for (const buffer of message.buffers) {\n","            buffers.push(buffer);\n","            downloaded += buffer.byteLength;\n","            progress.value = downloaded;\n","          }\n","        }\n","      }\n","      const blob = new Blob(buffers, {type: 'application/binary'});\n","      const a = document.createElement('a');\n","      a.href = window.URL.createObjectURL(blob);\n","      a.download = filename;\n","      div.appendChild(a);\n","      a.click();\n","      div.remove();\n","    }\n","  "],"text/plain":["<IPython.core.display.Javascript object>"]},"metadata":{"tags":[]}},{"output_type":"display_data","data":{"application/javascript":["download(\"download_1a824eb8-c8d1-48a1-93bb-66b33fc28b44\", \"GCF_000185805.1_ASM18580v1_genomic.fna.gz__.gff\", 1999844)"],"text/plain":["<IPython.core.display.Javascript object>"]},"metadata":{"tags":[]}},{"output_type":"display_data","data":{"application/javascript":["\n","    async function download(id, filename, size) {\n","      if (!google.colab.kernel.accessAllowed) {\n","        return;\n","      }\n","      const div = document.createElement('div');\n","      const label = document.createElement('label');\n","      label.textContent = `Downloading \"${filename}\": `;\n","      div.appendChild(label);\n","      const progress = document.createElement('progress');\n","      progress.max = size;\n","      div.appendChild(progress);\n","      document.body.appendChild(div);\n","\n","      const buffers = [];\n","      let downloaded = 0;\n","\n","      const channel = await google.colab.kernel.comms.open(id);\n","      // Send a message to notify the kernel that we're ready.\n","      channel.send({})\n","\n","      for await (const message of channel.messages) {\n","        // Send a message to notify the kernel that we're ready.\n","        channel.send({})\n","        if (message.buffers) {\n","          for (const buffer of message.buffers) {\n","            buffers.push(buffer);\n","            downloaded += buffer.byteLength;\n","            progress.value = downloaded;\n","          }\n","        }\n","      }\n","      const blob = new Blob(buffers, {type: 'application/binary'});\n","      const a = document.createElement('a');\n","      a.href = window.URL.createObjectURL(blob);\n","      a.download = filename;\n","      div.appendChild(a);\n","      a.click();\n","      div.remove();\n","    }\n","  "],"text/plain":["<IPython.core.display.Javascript object>"]},"metadata":{"tags":[]}},{"output_type":"display_data","data":{"application/javascript":["download(\"download_c3df86c6-202f-47f7-b8a2-2b796aa3fbc4\", \"GCF_000184705.1_ASM18470v1_genomic.fna.gz__.gff\", 3156996)"],"text/plain":["<IPython.core.display.Javascript object>"]},"metadata":{"tags":[]}},{"output_type":"display_data","data":{"application/javascript":["\n","    async function download(id, filename, size) {\n","      if (!google.colab.kernel.accessAllowed) {\n","        return;\n","      }\n","      const div = document.createElement('div');\n","      const label = document.createElement('label');\n","      label.textContent = `Downloading \"${filename}\": `;\n","      div.appendChild(label);\n","      const progress = document.createElement('progress');\n","      progress.max = size;\n","      div.appendChild(progress);\n","      document.body.appendChild(div);\n","\n","      const buffers = [];\n","      let downloaded = 0;\n","\n","      const channel = await google.colab.kernel.comms.open(id);\n","      // Send a message to notify the kernel that we're ready.\n","      channel.send({})\n","\n","      for await (const message of channel.messages) {\n","        // Send a message to notify the kernel that we're ready.\n","        channel.send({})\n","        if (message.buffers) {\n","          for (const buffer of message.buffers) {\n","            buffers.push(buffer);\n","            downloaded += buffer.byteLength;\n","            progress.value = downloaded;\n","          }\n","        }\n","      }\n","      const blob = new Blob(buffers, {type: 'application/binary'});\n","      const a = document.createElement('a');\n","      a.href = window.URL.createObjectURL(blob);\n","      a.download = filename;\n","      div.appendChild(a);\n","      a.click();\n","      div.remove();\n","    }\n","  "],"text/plain":["<IPython.core.display.Javascript object>"]},"metadata":{"tags":[]}},{"output_type":"display_data","data":{"application/javascript":["download(\"download_6fc12d60-72e3-4a58-a48e-8071214a2bcd\", \"GCF_000177635.2_ASM17763v2_genomic.fna.gz__.gff\", 3270218)"],"text/plain":["<IPython.core.display.Javascript object>"]},"metadata":{"tags":[]}},{"output_type":"display_data","data":{"application/javascript":["\n","    async function download(id, filename, size) {\n","      if (!google.colab.kernel.accessAllowed) {\n","        return;\n","      }\n","      const div = document.createElement('div');\n","      const label = document.createElement('label');\n","      label.textContent = `Downloading \"${filename}\": `;\n","      div.appendChild(label);\n","      const progress = document.createElement('progress');\n","      progress.max = size;\n","      div.appendChild(progress);\n","      document.body.appendChild(div);\n","\n","      const buffers = [];\n","      let downloaded = 0;\n","\n","      const channel = await google.colab.kernel.comms.open(id);\n","      // Send a message to notify the kernel that we're ready.\n","      channel.send({})\n","\n","      for await (const message of channel.messages) {\n","        // Send a message to notify the kernel that we're ready.\n","        channel.send({})\n","        if (message.buffers) {\n","          for (const buffer of message.buffers) {\n","            buffers.push(buffer);\n","            downloaded += buffer.byteLength;\n","            progress.value = downloaded;\n","          }\n","        }\n","      }\n","      const blob = new Blob(buffers, {type: 'application/binary'});\n","      const a = document.createElement('a');\n","      a.href = window.URL.createObjectURL(blob);\n","      a.download = filename;\n","      div.appendChild(a);\n","      a.click();\n","      div.remove();\n","    }\n","  "],"text/plain":["<IPython.core.display.Javascript object>"]},"metadata":{"tags":[]}},{"output_type":"display_data","data":{"application/javascript":["download(\"download_ab90823e-a8a7-478d-9184-f822734597ec\", \"GCF_000147695.2_ASM14769v3_genomic.fna.gz__.gff\", 3140551)"],"text/plain":["<IPython.core.display.Javascript object>"]},"metadata":{"tags":[]}},{"output_type":"display_data","data":{"application/javascript":["\n","    async function download(id, filename, size) {\n","      if (!google.colab.kernel.accessAllowed) {\n","        return;\n","      }\n","      const div = document.createElement('div');\n","      const label = document.createElement('label');\n","      label.textContent = `Downloading \"${filename}\": `;\n","      div.appendChild(label);\n","      const progress = document.createElement('progress');\n","      progress.max = size;\n","      div.appendChild(progress);\n","      document.body.appendChild(div);\n","\n","      const buffers = [];\n","      let downloaded = 0;\n","\n","      const channel = await google.colab.kernel.comms.open(id);\n","      // Send a message to notify the kernel that we're ready.\n","      channel.send({})\n","\n","      for await (const message of channel.messages) {\n","        // Send a message to notify the kernel that we're ready.\n","        channel.send({})\n","        if (message.buffers) {\n","          for (const buffer of message.buffers) {\n","            buffers.push(buffer);\n","            downloaded += buffer.byteLength;\n","            progress.value = downloaded;\n","          }\n","        }\n","      }\n","      const blob = new Blob(buffers, {type: 'application/binary'});\n","      const a = document.createElement('a');\n","      a.href = window.URL.createObjectURL(blob);\n","      a.download = filename;\n","      div.appendChild(a);\n","      a.click();\n","      div.remove();\n","    }\n","  "],"text/plain":["<IPython.core.display.Javascript object>"]},"metadata":{"tags":[]}},{"output_type":"display_data","data":{"application/javascript":["download(\"download_2a5b287a-7cc2-49f1-b467-16bd1b939a85\", \"GCF_000144645.1_ASM14464v1_genomic.fna.gz__.gff\", 2567790)"],"text/plain":["<IPython.core.display.Javascript object>"]},"metadata":{"tags":[]}},{"output_type":"display_data","data":{"application/javascript":["\n","    async function download(id, filename, size) {\n","      if (!google.colab.kernel.accessAllowed) {\n","        return;\n","      }\n","      const div = document.createElement('div');\n","      const label = document.createElement('label');\n","      label.textContent = `Downloading \"${filename}\": `;\n","      div.appendChild(label);\n","      const progress = document.createElement('progress');\n","      progress.max = size;\n","      div.appendChild(progress);\n","      document.body.appendChild(div);\n","\n","      const buffers = [];\n","      let downloaded = 0;\n","\n","      const channel = await google.colab.kernel.comms.open(id);\n","      // Send a message to notify the kernel that we're ready.\n","      channel.send({})\n","\n","      for await (const message of channel.messages) {\n","        // Send a message to notify the kernel that we're ready.\n","        channel.send({})\n","        if (message.buffers) {\n","          for (const buffer of message.buffers) {\n","            buffers.push(buffer);\n","            downloaded += buffer.byteLength;\n","            progress.value = downloaded;\n","          }\n","        }\n","      }\n","      const blob = new Blob(buffers, {type: 'application/binary'});\n","      const a = document.createElement('a');\n","      a.href = window.URL.createObjectURL(blob);\n","      a.download = filename;\n","      div.appendChild(a);\n","      a.click();\n","      div.remove();\n","    }\n","  "],"text/plain":["<IPython.core.display.Javascript object>"]},"metadata":{"tags":[]}},{"output_type":"display_data","data":{"application/javascript":["download(\"download_f6393d7e-5292-4a08-8ff0-8f8cee57b682\", \"GCF_000020965.1_ASM2096v1_genomic.fna.gz__.gff\", 2198727)"],"text/plain":["<IPython.core.display.Javascript object>"]},"metadata":{"tags":[]}},{"output_type":"display_data","data":{"application/javascript":["\n","    async function download(id, filename, size) {\n","      if (!google.colab.kernel.accessAllowed) {\n","        return;\n","      }\n","      const div = document.createElement('div');\n","      const label = document.createElement('label');\n","      label.textContent = `Downloading \"${filename}\": `;\n","      div.appendChild(label);\n","      const progress = document.createElement('progress');\n","      progress.max = size;\n","      div.appendChild(progress);\n","      document.body.appendChild(div);\n","\n","      const buffers = [];\n","      let downloaded = 0;\n","\n","      const channel = await google.colab.kernel.comms.open(id);\n","      // Send a message to notify the kernel that we're ready.\n","      channel.send({})\n","\n","      for await (const message of channel.messages) {\n","        // Send a message to notify the kernel that we're ready.\n","        channel.send({})\n","        if (message.buffers) {\n","          for (const buffer of message.buffers) {\n","            buffers.push(buffer);\n","            downloaded += buffer.byteLength;\n","            progress.value = downloaded;\n","          }\n","        }\n","      }\n","      const blob = new Blob(buffers, {type: 'application/binary'});\n","      const a = document.createElement('a');\n","      a.href = window.URL.createObjectURL(blob);\n","      a.download = filename;\n","      div.appendChild(a);\n","      a.click();\n","      div.remove();\n","    }\n","  "],"text/plain":["<IPython.core.display.Javascript object>"]},"metadata":{"tags":[]}},{"output_type":"display_data","data":{"application/javascript":["download(\"download_ec1cfbff-3d8a-490d-96cf-4dc54101505b\", \"GCF_000020505.1_ASM2050v1_genomic.fna.gz__.gff\", 2556212)"],"text/plain":["<IPython.core.display.Javascript object>"]},"metadata":{"tags":[]}},{"output_type":"display_data","data":{"application/javascript":["\n","    async function download(id, filename, size) {\n","      if (!google.colab.kernel.accessAllowed) {\n","        return;\n","      }\n","      const div = document.createElement('div');\n","      const label = document.createElement('label');\n","      label.textContent = `Downloading \"${filename}\": `;\n","      div.appendChild(label);\n","      const progress = document.createElement('progress');\n","      progress.max = size;\n","      div.appendChild(progress);\n","      document.body.appendChild(div);\n","\n","      const buffers = [];\n","      let downloaded = 0;\n","\n","      const channel = await google.colab.kernel.comms.open(id);\n","      // Send a message to notify the kernel that we're ready.\n","      channel.send({})\n","\n","      for await (const message of channel.messages) {\n","        // Send a message to notify the kernel that we're ready.\n","        channel.send({})\n","        if (message.buffers) {\n","          for (const buffer of message.buffers) {\n","            buffers.push(buffer);\n","            downloaded += buffer.byteLength;\n","            progress.value = downloaded;\n","          }\n","        }\n","      }\n","      const blob = new Blob(buffers, {type: 'application/binary'});\n","      const a = document.createElement('a');\n","      a.href = window.URL.createObjectURL(blob);\n","      a.download = filename;\n","      div.appendChild(a);\n","      a.click();\n","      div.remove();\n","    }\n","  "],"text/plain":["<IPython.core.display.Javascript object>"]},"metadata":{"tags":[]}},{"output_type":"display_data","data":{"application/javascript":["download(\"download_4298c025-ab44-4396-83ca-0b9d05e370c3\", \"GCF_000020005.1_ASM2000v1_genomic.fna.gz__.gff\", 3581621)"],"text/plain":["<IPython.core.display.Javascript object>"]},"metadata":{"tags":[]}},{"output_type":"display_data","data":{"application/javascript":["\n","    async function download(id, filename, size) {\n","      if (!google.colab.kernel.accessAllowed) {\n","        return;\n","      }\n","      const div = document.createElement('div');\n","      const label = document.createElement('label');\n","      label.textContent = `Downloading \"${filename}\": `;\n","      div.appendChild(label);\n","      const progress = document.createElement('progress');\n","      progress.max = size;\n","      div.appendChild(progress);\n","      document.body.appendChild(div);\n","\n","      const buffers = [];\n","      let downloaded = 0;\n","\n","      const channel = await google.colab.kernel.comms.open(id);\n","      // Send a message to notify the kernel that we're ready.\n","      channel.send({})\n","\n","      for await (const message of channel.messages) {\n","        // Send a message to notify the kernel that we're ready.\n","        channel.send({})\n","        if (message.buffers) {\n","          for (const buffer of message.buffers) {\n","            buffers.push(buffer);\n","            downloaded += buffer.byteLength;\n","            progress.value = downloaded;\n","          }\n","        }\n","      }\n","      const blob = new Blob(buffers, {type: 'application/binary'});\n","      const a = document.createElement('a');\n","      a.href = window.URL.createObjectURL(blob);\n","      a.download = filename;\n","      div.appendChild(a);\n","      a.click();\n","      div.remove();\n","    }\n","  "],"text/plain":["<IPython.core.display.Javascript object>"]},"metadata":{"tags":[]}},{"output_type":"display_data","data":{"application/javascript":["download(\"download_badceb0b-d76e-4672-a18f-f4c583d93440\", \"GCF_000009905.1_ASM990v1_genomic.fna.gz__.gff\", 3985869)"],"text/plain":["<IPython.core.display.Javascript object>"]},"metadata":{"tags":[]}},{"output_type":"display_data","data":{"application/javascript":["\n","    async function download(id, filename, size) {\n","      if (!google.colab.kernel.accessAllowed) {\n","        return;\n","      }\n","      const div = document.createElement('div');\n","      const label = document.createElement('label');\n","      label.textContent = `Downloading \"${filename}\": `;\n","      div.appendChild(label);\n","      const progress = document.createElement('progress');\n","      progress.max = size;\n","      div.appendChild(progress);\n","      document.body.appendChild(div);\n","\n","      const buffers = [];\n","      let downloaded = 0;\n","\n","      const channel = await google.colab.kernel.comms.open(id);\n","      // Send a message to notify the kernel that we're ready.\n","      channel.send({})\n","\n","      for await (const message of channel.messages) {\n","        // Send a message to notify the kernel that we're ready.\n","        channel.send({})\n","        if (message.buffers) {\n","          for (const buffer of message.buffers) {\n","            buffers.push(buffer);\n","            downloaded += buffer.byteLength;\n","            progress.value = downloaded;\n","          }\n","        }\n","      }\n","      const blob = new Blob(buffers, {type: 'application/binary'});\n","      const a = document.createElement('a');\n","      a.href = window.URL.createObjectURL(blob);\n","      a.download = filename;\n","      div.appendChild(a);\n","      a.click();\n","      div.remove();\n","    }\n","  "],"text/plain":["<IPython.core.display.Javascript object>"]},"metadata":{"tags":[]}},{"output_type":"display_data","data":{"application/javascript":["download(\"download_6410bfbe-8e8d-49d5-b27b-acbf11f2f260\", \"GCF_000008625.1_ASM862v1_genomic.fna.gz__.gff\", 1818104)"],"text/plain":["<IPython.core.display.Javascript object>"]},"metadata":{"tags":[]}},{"output_type":"display_data","data":{"application/javascript":["\n","    async function download(id, filename, size) {\n","      if (!google.colab.kernel.accessAllowed) {\n","        return;\n","      }\n","      const div = document.createElement('div');\n","      const label = document.createElement('label');\n","      label.textContent = `Downloading \"${filename}\": `;\n","      div.appendChild(label);\n","      const progress = document.createElement('progress');\n","      progress.max = size;\n","      div.appendChild(progress);\n","      document.body.appendChild(div);\n","\n","      const buffers = [];\n","      let downloaded = 0;\n","\n","      const channel = await google.colab.kernel.comms.open(id);\n","      // Send a message to notify the kernel that we're ready.\n","      channel.send({})\n","\n","      for await (const message of channel.messages) {\n","        // Send a message to notify the kernel that we're ready.\n","        channel.send({})\n","        if (message.buffers) {\n","          for (const buffer of message.buffers) {\n","            buffers.push(buffer);\n","            downloaded += buffer.byteLength;\n","            progress.value = downloaded;\n","          }\n","        }\n","      }\n","      const blob = new Blob(buffers, {type: 'application/binary'});\n","      const a = document.createElement('a');\n","      a.href = window.URL.createObjectURL(blob);\n","      a.download = filename;\n","      div.appendChild(a);\n","      a.click();\n","      div.remove();\n","    }\n","  "],"text/plain":["<IPython.core.display.Javascript object>"]},"metadata":{"tags":[]}},{"output_type":"display_data","data":{"application/javascript":["download(\"download_d99c4bdb-c66e-4b57-808f-910c3b576b78\", \"GCF_000005845.2_ASM584v2_genomic.fna.gz__.gff\", 5208939)"],"text/plain":["<IPython.core.display.Javascript object>"]},"metadata":{"tags":[]}}]},{"cell_type":"markdown","metadata":{"id":"4D-TsN2GZTFj","colab_type":"text"},"source":["# MIT License\n"," Copyright (c) 2020 Markus J. Sommer\n","\n","Permission is hereby granted, free of charge, to any person obtaining a copy\n","of this software and associated documentation files (the \"Software\"), to deal\n","in the Software without restriction, including without limitation the rights\n","to use, copy, modify, merge, publish, distribute, sublicense, and/or sell\n","copies of the Software, and to permit persons to whom the Software is\n","furnished to do so, subject to the following conditions:\n","\n","The above copyright notice and this permission notice shall be included in all\n","copies or substantial portions of the Software.\n","\n","THE SOFTWARE IS PROVIDED \"AS IS\", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR\n","IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,\n","FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE\n","AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER\n","LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,\n","OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE\n","SOFTWARE."]}]}