{"nbformat":4,"nbformat_minor":0,"metadata":{"colab":{"name":"Balrog_0.3.1.ipynb","provenance":[],"collapsed_sections":[],"toc_visible":true,"mount_file_id":"1rAQd-UsKhliEHw_tDMGv2a_UMS4iPA3V","authorship_tag":"ABX9TyN0bA+hEGlEVnPYU+ciCkt7"},"kernelspec":{"name":"python3","display_name":"Python 3"},"accelerator":"GPU","widgets":{"application/vnd.jupyter.widget-state+json":{"7977b776a3a74915b48aaabcb8ad0991":{"model_module":"@jupyter-widgets/controls","model_name":"HBoxModel","state":{"_view_name":"HBoxView","_dom_classes":[],"_model_name":"HBoxModel","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.5.0","box_style":"","layout":"IPY_MODEL_3b37b21389cf4cc285293764a9208733","_model_module":"@jupyter-widgets/controls","children":["IPY_MODEL_7c00d07501114a4989d889e4c2dc9b61","IPY_MODEL_13de506a4ec6456982547694e0c7c845"]}},"3b37b21389cf4cc285293764a9208733":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"7c00d07501114a4989d889e4c2dc9b61":{"model_module":"@jupyter-widgets/controls","model_name":"FloatProgressModel","state":{"_view_name":"ProgressView","style":"IPY_MODEL_145ae87c4b9643e6a418e83ea685a20a","_dom_classes":[],"description":"100%","_model_name":"FloatProgressModel","bar_style":"success","max":315,"_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":315,"_view_count":null,"_view_module_version":"1.5.0","orientation":"horizontal","min":0,"description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_9278878f5b774b0d9def450906d388b3"}},"13de506a4ec6456982547694e0c7c845":{"model_module":"@jupyter-widgets/controls","model_name":"HTMLModel","state":{"_view_name":"HTMLView","style":"IPY_MODEL_b56880c809494da4b84e7596550a6b64","_dom_classes":[],"description":"","_model_name":"HTMLModel","placeholder":"​","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":" 315/315 [00:10&lt;00:00, 28.95 batch/s]","_view_count":null,"_view_module_version":"1.5.0","description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_f1a2fc4f62cd4e71be7be689c5be3299"}},"145ae87c4b9643e6a418e83ea685a20a":{"model_module":"@jupyter-widgets/controls","model_name":"ProgressStyleModel","state":{"_view_name":"StyleView","_model_name":"ProgressStyleModel","description_width":"initial","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","bar_color":null,"_model_module":"@jupyter-widgets/controls"}},"9278878f5b774b0d9def450906d388b3":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"b56880c809494da4b84e7596550a6b64":{"model_module":"@jupyter-widgets/controls","model_name":"DescriptionStyleModel","state":{"_view_name":"StyleView","_model_name":"DescriptionStyleModel","description_width":"","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","_model_module":"@jupyter-widgets/controls"}},"f1a2fc4f62cd4e71be7be689c5be3299":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"5d15e0c51e7d406c8b642fa8548c0c5e":{"model_module":"@jupyter-widgets/controls","model_name":"HBoxModel","state":{"_view_name":"HBoxView","_dom_classes":[],"_model_name":"HBoxModel","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.5.0","box_style":"","layout":"IPY_MODEL_16806a86cb7c444fa7cec1c5ad7e302c","_model_module":"@jupyter-widgets/controls","children":["IPY_MODEL_d2e1117978704746bb721116210c9480","IPY_MODEL_931580a5c8ad4855ad7cd16248786120"]}},"16806a86cb7c444fa7cec1c5ad7e302c":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"d2e1117978704746bb721116210c9480":{"model_module":"@jupyter-widgets/controls","model_name":"FloatProgressModel","state":{"_view_name":"ProgressView","style":"IPY_MODEL_afeff62a0f6f4c41ad635a0936f238f7","_dom_classes":[],"description":"100%","_model_name":"FloatProgressModel","bar_style":"success","max":63,"_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":63,"_view_count":null,"_view_module_version":"1.5.0","orientation":"horizontal","min":0,"description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_4c9ff1da3b62444e96293e5eb2148510"}},"931580a5c8ad4855ad7cd16248786120":{"model_module":"@jupyter-widgets/controls","model_name":"HTMLModel","state":{"_view_name":"HTMLView","style":"IPY_MODEL_989bdb2d67e94962bbacae7f2a3d5b5d","_dom_classes":[],"description":"","_model_name":"HTMLModel","placeholder":"​","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":" 63/63 [04:12&lt;00:00,  4.01s/ batch]","_view_count":null,"_view_module_version":"1.5.0","description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_4f0a51cb0b894c2d96894a9b2ae2e6fd"}},"afeff62a0f6f4c41ad635a0936f238f7":{"model_module":"@jupyter-widgets/controls","model_name":"ProgressStyleModel","state":{"_view_name":"StyleView","_model_name":"ProgressStyleModel","description_width":"initial","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","bar_color":null,"_model_module":"@jupyter-widgets/controls"}},"4c9ff1da3b62444e96293e5eb2148510":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"989bdb2d67e94962bbacae7f2a3d5b5d":{"model_module":"@jupyter-widgets/controls","model_name":"DescriptionStyleModel","state":{"_view_name":"StyleView","_model_name":"DescriptionStyleModel","description_width":"","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","_model_module":"@jupyter-widgets/controls"}},"4f0a51cb0b894c2d96894a9b2ae2e6fd":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"4a9040116613475b9c7a81fb1c0cd7e9":{"model_module":"@jupyter-widgets/controls","model_name":"HBoxModel","state":{"_view_name":"HBoxView","_dom_classes":[],"_model_name":"HBoxModel","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.5.0","box_style":"","layout":"IPY_MODEL_ffc19f5ad6c74c5b9062540883ae70b0","_model_module":"@jupyter-widgets/controls","children":["IPY_MODEL_fa8dc13e01ac4cef849aa73b86722dfb","IPY_MODEL_d6d821d3b15a4a6b8a6c1bccdacd0c39"]}},"ffc19f5ad6c74c5b9062540883ae70b0":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"fa8dc13e01ac4cef849aa73b86722dfb":{"model_module":"@jupyter-widgets/controls","model_name":"FloatProgressModel","state":{"_view_name":"ProgressView","style":"IPY_MODEL_cee1b7d6d6414966b62009c12eced574","_dom_classes":[],"description":"100%","_model_name":"FloatProgressModel","bar_style":"success","max":62860,"_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":62860,"_view_count":null,"_view_module_version":"1.5.0","orientation":"horizontal","min":0,"description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_ebe91a4f4d294dd7a4d5a82fb95918d4"}},"d6d821d3b15a4a6b8a6c1bccdacd0c39":{"model_module":"@jupyter-widgets/controls","model_name":"HTMLModel","state":{"_view_name":"HTMLView","style":"IPY_MODEL_6b84f65d546b4ae79739f0489a33f8a7","_dom_classes":[],"description":"","_model_name":"HTMLModel","placeholder":"​","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":" 62860/62860 [03:34&lt;00:00, 292.85 ORF/s]","_view_count":null,"_view_module_version":"1.5.0","description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_e08f1af1d60a4a08a27d73f9c0db2bf3"}},"cee1b7d6d6414966b62009c12eced574":{"model_module":"@jupyter-widgets/controls","model_name":"ProgressStyleModel","state":{"_view_name":"StyleView","_model_name":"ProgressStyleModel","description_width":"initial","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","bar_color":null,"_model_module":"@jupyter-widgets/controls"}},"ebe91a4f4d294dd7a4d5a82fb95918d4":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"6b84f65d546b4ae79739f0489a33f8a7":{"model_module":"@jupyter-widgets/controls","model_name":"DescriptionStyleModel","state":{"_view_name":"StyleView","_model_name":"DescriptionStyleModel","description_width":"","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","_model_module":"@jupyter-widgets/controls"}},"e08f1af1d60a4a08a27d73f9c0db2bf3":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"aa21151cc1c24f24aa33fb91c9b294ab":{"model_module":"@jupyter-widgets/controls","model_name":"HBoxModel","state":{"_view_name":"HBoxView","_dom_classes":[],"_model_name":"HBoxModel","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.5.0","box_style":"","layout":"IPY_MODEL_ff5147ab95eb4b73b85887ec22e2ef8d","_model_module":"@jupyter-widgets/controls","children":["IPY_MODEL_421b1e265c3246e2a73c6c03a3522a7e","IPY_MODEL_f8827866f95a40dfa75683d2ad18ad6f"]}},"ff5147ab95eb4b73b85887ec22e2ef8d":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"421b1e265c3246e2a73c6c03a3522a7e":{"model_module":"@jupyter-widgets/controls","model_name":"FloatProgressModel","state":{"_view_name":"ProgressView","style":"IPY_MODEL_46b6215f398f40ea8199b2fdfa708b8a","_dom_classes":[],"description":"100%","_model_name":"FloatProgressModel","bar_style":"success","max":447,"_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":447,"_view_count":null,"_view_module_version":"1.5.0","orientation":"horizontal","min":0,"description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_cc773c135af04acd98529e37a31abac8"}},"f8827866f95a40dfa75683d2ad18ad6f":{"model_module":"@jupyter-widgets/controls","model_name":"HTMLModel","state":{"_view_name":"HTMLView","style":"IPY_MODEL_b635ddfae7e54c04b3c9cec5c4fa17c2","_dom_classes":[],"description":"","_model_name":"HTMLModel","placeholder":"​","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":" 447/447 [00:12&lt;00:00, 34.90 batch/s]","_view_count":null,"_view_module_version":"1.5.0","description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_e2b67b7421db42ae83372d45b7e8529a"}},"46b6215f398f40ea8199b2fdfa708b8a":{"model_module":"@jupyter-widgets/controls","model_name":"ProgressStyleModel","state":{"_view_name":"StyleView","_model_name":"ProgressStyleModel","description_width":"initial","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","bar_color":null,"_model_module":"@jupyter-widgets/controls"}},"cc773c135af04acd98529e37a31abac8":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"b635ddfae7e54c04b3c9cec5c4fa17c2":{"model_module":"@jupyter-widgets/controls","model_name":"DescriptionStyleModel","state":{"_view_name":"StyleView","_model_name":"DescriptionStyleModel","description_width":"","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","_model_module":"@jupyter-widgets/controls"}},"e2b67b7421db42ae83372d45b7e8529a":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"9f043bf115c648d48fbf6361c6ed7974":{"model_module":"@jupyter-widgets/controls","model_name":"HBoxModel","state":{"_view_name":"HBoxView","_dom_classes":[],"_model_name":"HBoxModel","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.5.0","box_style":"","layout":"IPY_MODEL_0f9fb553cee84dd5bf66add251a0c8a0","_model_module":"@jupyter-widgets/controls","children":["IPY_MODEL_f1c3b5413f014680ae99dbe54b905b2c","IPY_MODEL_538e89d21c0040dcacf1ac1b328f1da7"]}},"0f9fb553cee84dd5bf66add251a0c8a0":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"f1c3b5413f014680ae99dbe54b905b2c":{"model_module":"@jupyter-widgets/controls","model_name":"FloatProgressModel","state":{"_view_name":"ProgressView","style":"IPY_MODEL_31cd5630cf4c4518bf80a2092ec9edc4","_dom_classes":[],"description":"100%","_model_name":"FloatProgressModel","bar_style":"success","max":90,"_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":90,"_view_count":null,"_view_module_version":"1.5.0","orientation":"horizontal","min":0,"description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_b614c9fa3c9c499caf19103ea7533ac0"}},"538e89d21c0040dcacf1ac1b328f1da7":{"model_module":"@jupyter-widgets/controls","model_name":"HTMLModel","state":{"_view_name":"HTMLView","style":"IPY_MODEL_13132cddf38c464d82cd2e2017f9bfc2","_dom_classes":[],"description":"","_model_name":"HTMLModel","placeholder":"​","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":" 90/90 [05:06&lt;00:00,  3.40s/ batch]","_view_count":null,"_view_module_version":"1.5.0","description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_14d7c7e0b36f4967b9211d81221ca644"}},"31cd5630cf4c4518bf80a2092ec9edc4":{"model_module":"@jupyter-widgets/controls","model_name":"ProgressStyleModel","state":{"_view_name":"StyleView","_model_name":"ProgressStyleModel","description_width":"initial","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","bar_color":null,"_model_module":"@jupyter-widgets/controls"}},"b614c9fa3c9c499caf19103ea7533ac0":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"13132cddf38c464d82cd2e2017f9bfc2":{"model_module":"@jupyter-widgets/controls","model_name":"DescriptionStyleModel","state":{"_view_name":"StyleView","_model_name":"DescriptionStyleModel","description_width":"","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","_model_module":"@jupyter-widgets/controls"}},"14d7c7e0b36f4967b9211d81221ca644":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"ce980cd159934f818f76c85a31b190a7":{"model_module":"@jupyter-widgets/controls","model_name":"HBoxModel","state":{"_view_name":"HBoxView","_dom_classes":[],"_model_name":"HBoxModel","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.5.0","box_style":"","layout":"IPY_MODEL_50fd16a576844022b905c98eda4f773c","_model_module":"@jupyter-widgets/controls","children":["IPY_MODEL_e7d2f806e53a4bb78d493ae315be8847","IPY_MODEL_da6b20a369b94e4f9e1a3ed08afb8c9b"]}},"50fd16a576844022b905c98eda4f773c":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"e7d2f806e53a4bb78d493ae315be8847":{"model_module":"@jupyter-widgets/controls","model_name":"FloatProgressModel","state":{"_view_name":"ProgressView","style":"IPY_MODEL_ceadc3e1fc3f4781860415bc040f46e2","_dom_classes":[],"description":"100%","_model_name":"FloatProgressModel","bar_style":"success","max":89307,"_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":89307,"_view_count":null,"_view_module_version":"1.5.0","orientation":"horizontal","min":0,"description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_ea2f44df4e6e44659e4c9dfbba6752ef"}},"da6b20a369b94e4f9e1a3ed08afb8c9b":{"model_module":"@jupyter-widgets/controls","model_name":"HTMLModel","state":{"_view_name":"HTMLView","style":"IPY_MODEL_26c2bfc43ffb4bd7b77ef4428809cbba","_dom_classes":[],"description":"","_model_name":"HTMLModel","placeholder":"​","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":" 89307/89307 [04:56&lt;00:00, 301.23 ORF/s]","_view_count":null,"_view_module_version":"1.5.0","description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_bd487d2a0f264b22824fb217d6c60717"}},"ceadc3e1fc3f4781860415bc040f46e2":{"model_module":"@jupyter-widgets/controls","model_name":"ProgressStyleModel","state":{"_view_name":"StyleView","_model_name":"ProgressStyleModel","description_width":"initial","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","bar_color":null,"_model_module":"@jupyter-widgets/controls"}},"ea2f44df4e6e44659e4c9dfbba6752ef":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"26c2bfc43ffb4bd7b77ef4428809cbba":{"model_module":"@jupyter-widgets/controls","model_name":"DescriptionStyleModel","state":{"_view_name":"StyleView","_model_name":"DescriptionStyleModel","description_width":"","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","_model_module":"@jupyter-widgets/controls"}},"bd487d2a0f264b22824fb217d6c60717":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"8a4dc35a4b6e43a4b06d087bf11c0b57":{"model_module":"@jupyter-widgets/controls","model_name":"HBoxModel","state":{"_view_name":"HBoxView","_dom_classes":[],"_model_name":"HBoxModel","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.5.0","box_style":"","layout":"IPY_MODEL_8f98a6f44da24d338a0d2ffc1f827560","_model_module":"@jupyter-widgets/controls","children":["IPY_MODEL_8ceae92967054fa4aeb30c044d8c91ea","IPY_MODEL_91b2ca25d8ee4c02acc3a20af9505243"]}},"8f98a6f44da24d338a0d2ffc1f827560":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"8ceae92967054fa4aeb30c044d8c91ea":{"model_module":"@jupyter-widgets/controls","model_name":"FloatProgressModel","state":{"_view_name":"ProgressView","style":"IPY_MODEL_41441da3be8444b8857683e930d26d01","_dom_classes":[],"description":"100%","_model_name":"FloatProgressModel","bar_style":"success","max":801,"_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":801,"_view_count":null,"_view_module_version":"1.5.0","orientation":"horizontal","min":0,"description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_8151f7e33c26426e9dd77670967043bd"}},"91b2ca25d8ee4c02acc3a20af9505243":{"model_module":"@jupyter-widgets/controls","model_name":"HTMLModel","state":{"_view_name":"HTMLView","style":"IPY_MODEL_52dbe6705de346a7aa60710a453f923c","_dom_classes":[],"description":"","_model_name":"HTMLModel","placeholder":"​","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":" 801/801 [00:24&lt;00:00, 33.04 batch/s]","_view_count":null,"_view_module_version":"1.5.0","description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_18ec8a2356ed4f368224e14ac307ee08"}},"41441da3be8444b8857683e930d26d01":{"model_module":"@jupyter-widgets/controls","model_name":"ProgressStyleModel","state":{"_view_name":"StyleView","_model_name":"ProgressStyleModel","description_width":"initial","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","bar_color":null,"_model_module":"@jupyter-widgets/controls"}},"8151f7e33c26426e9dd77670967043bd":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"52dbe6705de346a7aa60710a453f923c":{"model_module":"@jupyter-widgets/controls","model_name":"DescriptionStyleModel","state":{"_view_name":"StyleView","_model_name":"DescriptionStyleModel","description_width":"","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","_model_module":"@jupyter-widgets/controls"}},"18ec8a2356ed4f368224e14ac307ee08":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"a5b54807a89945ae9767d8677a4a76f7":{"model_module":"@jupyter-widgets/controls","model_name":"HBoxModel","state":{"_view_name":"HBoxView","_dom_classes":[],"_model_name":"HBoxModel","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.5.0","box_style":"","layout":"IPY_MODEL_d8edb7bbba0a4117adc661954ed909f8","_model_module":"@jupyter-widgets/controls","children":["IPY_MODEL_7711e91fae58418bab84a2efa6c5fcbe","IPY_MODEL_3e8f56a839414257a2ee45a157953887"]}},"d8edb7bbba0a4117adc661954ed909f8":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"7711e91fae58418bab84a2efa6c5fcbe":{"model_module":"@jupyter-widgets/controls","model_name":"FloatProgressModel","state":{"_view_name":"ProgressView","style":"IPY_MODEL_9e9bd36888c34ee98489660fde13fa3b","_dom_classes":[],"description":"100%","_model_name":"FloatProgressModel","bar_style":"success","max":161,"_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":161,"_view_count":null,"_view_module_version":"1.5.0","orientation":"horizontal","min":0,"description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_38256f5f339f4fdfa2242c020b34aa3d"}},"3e8f56a839414257a2ee45a157953887":{"model_module":"@jupyter-widgets/controls","model_name":"HTMLModel","state":{"_view_name":"HTMLView","style":"IPY_MODEL_bc215acc4f5b4d6aa5bcdded31a6edbb","_dom_classes":[],"description":"","_model_name":"HTMLModel","placeholder":"​","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":" 161/161 [12:53&lt;00:00,  4.81s/ batch]","_view_count":null,"_view_module_version":"1.5.0","description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_f75b905f5e654846b42a40112b177c4a"}},"9e9bd36888c34ee98489660fde13fa3b":{"model_module":"@jupyter-widgets/controls","model_name":"ProgressStyleModel","state":{"_view_name":"StyleView","_model_name":"ProgressStyleModel","description_width":"initial","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","bar_color":null,"_model_module":"@jupyter-widgets/controls"}},"38256f5f339f4fdfa2242c020b34aa3d":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"bc215acc4f5b4d6aa5bcdded31a6edbb":{"model_module":"@jupyter-widgets/controls","model_name":"DescriptionStyleModel","state":{"_view_name":"StyleView","_model_name":"DescriptionStyleModel","description_width":"","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","_model_module":"@jupyter-widgets/controls"}},"f75b905f5e654846b42a40112b177c4a":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"f038606a5e8a42d7b4585ea88c1cdf6b":{"model_module":"@jupyter-widgets/controls","model_name":"HBoxModel","state":{"_view_name":"HBoxView","_dom_classes":[],"_model_name":"HBoxModel","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.5.0","box_style":"","layout":"IPY_MODEL_58e10a92f50943b482bf06458b63344d","_model_module":"@jupyter-widgets/controls","children":["IPY_MODEL_a755e7f2f03b46d6b0926d268f7acfa5","IPY_MODEL_66e1bf6b0cee4dad8993e9bc48e19429"]}},"58e10a92f50943b482bf06458b63344d":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"a755e7f2f03b46d6b0926d268f7acfa5":{"model_module":"@jupyter-widgets/controls","model_name":"FloatProgressModel","state":{"_view_name":"ProgressView","style":"IPY_MODEL_54916cb120e74bd2965e1b008296022f","_dom_classes":[],"description":"100%","_model_name":"FloatProgressModel","bar_style":"success","max":113736,"_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":113736,"_view_count":null,"_view_module_version":"1.5.0","orientation":"horizontal","min":0,"description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_734443a68cbe4e9790ed40f2c200ec0b"}},"66e1bf6b0cee4dad8993e9bc48e19429":{"model_module":"@jupyter-widgets/controls","model_name":"HTMLModel","state":{"_view_name":"HTMLView","style":"IPY_MODEL_fd66aee63683423a9f4ebd08d7d6a3c7","_dom_classes":[],"description":"","_model_name":"HTMLModel","placeholder":"​","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":" 113736/113736 [11:49&lt;00:00, 160.23 ORF/s]","_view_count":null,"_view_module_version":"1.5.0","description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_503f02cf791e4239a487070c37252309"}},"54916cb120e74bd2965e1b008296022f":{"model_module":"@jupyter-widgets/controls","model_name":"ProgressStyleModel","state":{"_view_name":"StyleView","_model_name":"ProgressStyleModel","description_width":"initial","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","bar_color":null,"_model_module":"@jupyter-widgets/controls"}},"734443a68cbe4e9790ed40f2c200ec0b":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"fd66aee63683423a9f4ebd08d7d6a3c7":{"model_module":"@jupyter-widgets/controls","model_name":"DescriptionStyleModel","state":{"_view_name":"StyleView","_model_name":"DescriptionStyleModel","description_width":"","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","_model_module":"@jupyter-widgets/controls"}},"503f02cf791e4239a487070c37252309":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"58b858f22c204471a4cf71d7c2458d4d":{"model_module":"@jupyter-widgets/controls","model_name":"HBoxModel","state":{"_view_name":"HBoxView","_dom_classes":[],"_model_name":"HBoxModel","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.5.0","box_style":"","layout":"IPY_MODEL_50de702199cd4a2abbb4228af66a3925","_model_module":"@jupyter-widgets/controls","children":["IPY_MODEL_4a3246a4014b4d5584a543a59d29536c","IPY_MODEL_478a81f1317643709b884c690db8eead"]}},"50de702199cd4a2abbb4228af66a3925":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"4a3246a4014b4d5584a543a59d29536c":{"model_module":"@jupyter-widgets/controls","model_name":"FloatProgressModel","state":{"_view_name":"ProgressView","style":"IPY_MODEL_899713e6fee94abc9567e00b71bc56c1","_dom_classes":[],"description":"100%","_model_name":"FloatProgressModel","bar_style":"success","max":25767,"_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":25767,"_view_count":null,"_view_module_version":"1.5.0","orientation":"horizontal","min":0,"description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_1d557593e4d844d98702316d42333271"}},"478a81f1317643709b884c690db8eead":{"model_module":"@jupyter-widgets/controls","model_name":"HTMLModel","state":{"_view_name":"HTMLView","style":"IPY_MODEL_3be46630a17543c2b30b4de721b7493e","_dom_classes":[],"description":"","_model_name":"HTMLModel","placeholder":"​","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":" 25767/25767 [06:05&lt;00:00, 70.53 ORF/s]","_view_count":null,"_view_module_version":"1.5.0","description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_a3e057a75335419c877989b80716c612"}},"899713e6fee94abc9567e00b71bc56c1":{"model_module":"@jupyter-widgets/controls","model_name":"ProgressStyleModel","state":{"_view_name":"StyleView","_model_name":"ProgressStyleModel","description_width":"initial","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","bar_color":null,"_model_module":"@jupyter-widgets/controls"}},"1d557593e4d844d98702316d42333271":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"3be46630a17543c2b30b4de721b7493e":{"model_module":"@jupyter-widgets/controls","model_name":"DescriptionStyleModel","state":{"_view_name":"StyleView","_model_name":"DescriptionStyleModel","description_width":"","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","_model_module":"@jupyter-widgets/controls"}},"a3e057a75335419c877989b80716c612":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"32cd0fe732b54002b4664c28dc5e074b":{"model_module":"@jupyter-widgets/controls","model_name":"HBoxModel","state":{"_view_name":"HBoxView","_dom_classes":[],"_model_name":"HBoxModel","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.5.0","box_style":"","layout":"IPY_MODEL_fe4238f572fd4131aee053c6845d40c3","_model_module":"@jupyter-widgets/controls","children":["IPY_MODEL_6add7163926a4f92b0113d7f28ff9fdd","IPY_MODEL_3e4c8d22929f4eaab967b51d08000dde"]}},"fe4238f572fd4131aee053c6845d40c3":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"6add7163926a4f92b0113d7f28ff9fdd":{"model_module":"@jupyter-widgets/controls","model_name":"FloatProgressModel","state":{"_view_name":"ProgressView","style":"IPY_MODEL_c51a8eaa4b4b43188ad81f66a0d6b44e","_dom_classes":[],"description":"100%","_model_name":"FloatProgressModel","bar_style":"success","max":20499,"_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":20499,"_view_count":null,"_view_module_version":"1.5.0","orientation":"horizontal","min":0,"description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_d385bdce97454bccba72d91dc4357539"}},"3e4c8d22929f4eaab967b51d08000dde":{"model_module":"@jupyter-widgets/controls","model_name":"HTMLModel","state":{"_view_name":"HTMLView","style":"IPY_MODEL_3aac636b96f64775b96fb405bbf3501f","_dom_classes":[],"description":"","_model_name":"HTMLModel","placeholder":"​","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":" 20499/20499 [02:56&lt;00:00, 116.38 ORF/s]","_view_count":null,"_view_module_version":"1.5.0","description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_0ed1714d75e64e8a823b6b4ec19ec7db"}},"c51a8eaa4b4b43188ad81f66a0d6b44e":{"model_module":"@jupyter-widgets/controls","model_name":"ProgressStyleModel","state":{"_view_name":"StyleView","_model_name":"ProgressStyleModel","description_width":"initial","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","bar_color":null,"_model_module":"@jupyter-widgets/controls"}},"d385bdce97454bccba72d91dc4357539":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"3aac636b96f64775b96fb405bbf3501f":{"model_module":"@jupyter-widgets/controls","model_name":"DescriptionStyleModel","state":{"_view_name":"StyleView","_model_name":"DescriptionStyleModel","description_width":"","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","_model_module":"@jupyter-widgets/controls"}},"0ed1714d75e64e8a823b6b4ec19ec7db":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"83252378b2d443de95e55481e3788ffa":{"model_module":"@jupyter-widgets/controls","model_name":"HBoxModel","state":{"_view_name":"HBoxView","_dom_classes":[],"_model_name":"HBoxModel","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.5.0","box_style":"","layout":"IPY_MODEL_c128e610689c4c70bda2a581fb244537","_model_module":"@jupyter-widgets/controls","children":["IPY_MODEL_d1cc6df4231b4569ac3443e9b68083f3","IPY_MODEL_999de9276e984e47b01c05a90d2d077a"]}},"c128e610689c4c70bda2a581fb244537":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"d1cc6df4231b4569ac3443e9b68083f3":{"model_module":"@jupyter-widgets/controls","model_name":"FloatProgressModel","state":{"_view_name":"ProgressView","style":"IPY_MODEL_9caa4fa3605647db96043437153d85fd","_dom_classes":[],"description":"100%","_model_name":"FloatProgressModel","bar_style":"success","max":506,"_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":506,"_view_count":null,"_view_module_version":"1.5.0","orientation":"horizontal","min":0,"description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_8e7302e3c0434dc6a8619e582bb9ceda"}},"999de9276e984e47b01c05a90d2d077a":{"model_module":"@jupyter-widgets/controls","model_name":"HTMLModel","state":{"_view_name":"HTMLView","style":"IPY_MODEL_b7ae3b2c79854a3897b8bee8408ce218","_dom_classes":[],"description":"","_model_name":"HTMLModel","placeholder":"​","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":" 506/506 [00:17&lt;00:00, 28.11 batch/s]","_view_count":null,"_view_module_version":"1.5.0","description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_cddd3b1c0f1f4d85afd3d14cec9e5451"}},"9caa4fa3605647db96043437153d85fd":{"model_module":"@jupyter-widgets/controls","model_name":"ProgressStyleModel","state":{"_view_name":"StyleView","_model_name":"ProgressStyleModel","description_width":"initial","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","bar_color":null,"_model_module":"@jupyter-widgets/controls"}},"8e7302e3c0434dc6a8619e582bb9ceda":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"b7ae3b2c79854a3897b8bee8408ce218":{"model_module":"@jupyter-widgets/controls","model_name":"DescriptionStyleModel","state":{"_view_name":"StyleView","_model_name":"DescriptionStyleModel","description_width":"","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","_model_module":"@jupyter-widgets/controls"}},"cddd3b1c0f1f4d85afd3d14cec9e5451":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"784f9f6d4f91404d9e6060a04e01b7de":{"model_module":"@jupyter-widgets/controls","model_name":"HBoxModel","state":{"_view_name":"HBoxView","_dom_classes":[],"_model_name":"HBoxModel","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.5.0","box_style":"","layout":"IPY_MODEL_3b20bfe327f84e20be79dc4f605d56d3","_model_module":"@jupyter-widgets/controls","children":["IPY_MODEL_a7f6170e69ab442888025f2f481d26cb","IPY_MODEL_13e646e6783348fba1cbd7dff6024970"]}},"3b20bfe327f84e20be79dc4f605d56d3":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"a7f6170e69ab442888025f2f481d26cb":{"model_module":"@jupyter-widgets/controls","model_name":"FloatProgressModel","state":{"_view_name":"ProgressView","style":"IPY_MODEL_11253536107d4a959d340a2ea0681ca9","_dom_classes":[],"description":"100%","_model_name":"FloatProgressModel","bar_style":"success","max":102,"_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":102,"_view_count":null,"_view_module_version":"1.5.0","orientation":"horizontal","min":0,"description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_907a7da9fbd04fb38a8276ad51288d5e"}},"13e646e6783348fba1cbd7dff6024970":{"model_module":"@jupyter-widgets/controls","model_name":"HTMLModel","state":{"_view_name":"HTMLView","style":"IPY_MODEL_a501c387432643f2bbee44b737e13545","_dom_classes":[],"description":"","_model_name":"HTMLModel","placeholder":"​","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":" 102/102 [05:38&lt;00:00,  3.32s/ batch]","_view_count":null,"_view_module_version":"1.5.0","description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_ab1c440c08f94f479a5e74395b8340ca"}},"11253536107d4a959d340a2ea0681ca9":{"model_module":"@jupyter-widgets/controls","model_name":"ProgressStyleModel","state":{"_view_name":"StyleView","_model_name":"ProgressStyleModel","description_width":"initial","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","bar_color":null,"_model_module":"@jupyter-widgets/controls"}},"907a7da9fbd04fb38a8276ad51288d5e":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"a501c387432643f2bbee44b737e13545":{"model_module":"@jupyter-widgets/controls","model_name":"DescriptionStyleModel","state":{"_view_name":"StyleView","_model_name":"DescriptionStyleModel","description_width":"","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","_model_module":"@jupyter-widgets/controls"}},"ab1c440c08f94f479a5e74395b8340ca":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"7f6ca2b1839f4a639b5847198413d50b":{"model_module":"@jupyter-widgets/controls","model_name":"HBoxModel","state":{"_view_name":"HBoxView","_dom_classes":[],"_model_name":"HBoxModel","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.5.0","box_style":"","layout":"IPY_MODEL_31b2737a16f44e7ab8268b269182c4c0","_model_module":"@jupyter-widgets/controls","children":["IPY_MODEL_5044715b24e449ffa6df6ad7ad53fc60","IPY_MODEL_93da4928074145aab18d1f46ad5f42b1"]}},"31b2737a16f44e7ab8268b269182c4c0":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"5044715b24e449ffa6df6ad7ad53fc60":{"model_module":"@jupyter-widgets/controls","model_name":"FloatProgressModel","state":{"_view_name":"ProgressView","style":"IPY_MODEL_211e29291f3a4ad492a486d4f72be181","_dom_classes":[],"description":"100%","_model_name":"FloatProgressModel","bar_style":"success","max":101058,"_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":101058,"_view_count":null,"_view_module_version":"1.5.0","orientation":"horizontal","min":0,"description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_255b58f0dabd46aaab9192559102e80b"}},"93da4928074145aab18d1f46ad5f42b1":{"model_module":"@jupyter-widgets/controls","model_name":"HTMLModel","state":{"_view_name":"HTMLView","style":"IPY_MODEL_65ab6117be40401a898539588d3c74bc","_dom_classes":[],"description":"","_model_name":"HTMLModel","placeholder":"​","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":" 101058/101058 [05:27&lt;00:00, 308.75 ORF/s]","_view_count":null,"_view_module_version":"1.5.0","description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_aa3766b99b4c4b5dbaf8b73b2cf21337"}},"211e29291f3a4ad492a486d4f72be181":{"model_module":"@jupyter-widgets/controls","model_name":"ProgressStyleModel","state":{"_view_name":"StyleView","_model_name":"ProgressStyleModel","description_width":"initial","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","bar_color":null,"_model_module":"@jupyter-widgets/controls"}},"255b58f0dabd46aaab9192559102e80b":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"65ab6117be40401a898539588d3c74bc":{"model_module":"@jupyter-widgets/controls","model_name":"DescriptionStyleModel","state":{"_view_name":"StyleView","_model_name":"DescriptionStyleModel","description_width":"","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","_model_module":"@jupyter-widgets/controls"}},"aa3766b99b4c4b5dbaf8b73b2cf21337":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"0b6a02b6506b4f38ac38a893c54ad922":{"model_module":"@jupyter-widgets/controls","model_name":"HBoxModel","state":{"_view_name":"HBoxView","_dom_classes":[],"_model_name":"HBoxModel","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.5.0","box_style":"","layout":"IPY_MODEL_47c6425636264c56bf7582690dad39e1","_model_module":"@jupyter-widgets/controls","children":["IPY_MODEL_8614af9bb457441f8ee5b4193d166798","IPY_MODEL_8d95a75fa276402198a536d81f94e95f"]}},"47c6425636264c56bf7582690dad39e1":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"8614af9bb457441f8ee5b4193d166798":{"model_module":"@jupyter-widgets/controls","model_name":"FloatProgressModel","state":{"_view_name":"ProgressView","style":"IPY_MODEL_49f8fa5710de4498be8c4633960e63c3","_dom_classes":[],"description":"100%","_model_name":"FloatProgressModel","bar_style":"success","max":786,"_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":786,"_view_count":null,"_view_module_version":"1.5.0","orientation":"horizontal","min":0,"description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_f1b221f15f0b4f12be664836a43c50a3"}},"8d95a75fa276402198a536d81f94e95f":{"model_module":"@jupyter-widgets/controls","model_name":"HTMLModel","state":{"_view_name":"HTMLView","style":"IPY_MODEL_4663fdcdd9134b66bb679c6f2e0a66c5","_dom_classes":[],"description":"","_model_name":"HTMLModel","placeholder":"​","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":" 786/786 [00:21&lt;00:00, 35.77 batch/s]","_view_count":null,"_view_module_version":"1.5.0","description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_e292d22e4c2a42288a75b049b6179266"}},"49f8fa5710de4498be8c4633960e63c3":{"model_module":"@jupyter-widgets/controls","model_name":"ProgressStyleModel","state":{"_view_name":"StyleView","_model_name":"ProgressStyleModel","description_width":"initial","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","bar_color":null,"_model_module":"@jupyter-widgets/controls"}},"f1b221f15f0b4f12be664836a43c50a3":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"4663fdcdd9134b66bb679c6f2e0a66c5":{"model_module":"@jupyter-widgets/controls","model_name":"DescriptionStyleModel","state":{"_view_name":"StyleView","_model_name":"DescriptionStyleModel","description_width":"","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","_model_module":"@jupyter-widgets/controls"}},"e292d22e4c2a42288a75b049b6179266":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"f0dd7a8b691842ab9dfadd32461bfa58":{"model_module":"@jupyter-widgets/controls","model_name":"HBoxModel","state":{"_view_name":"HBoxView","_dom_classes":[],"_model_name":"HBoxModel","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.5.0","box_style":"","layout":"IPY_MODEL_f7cd396d41e14a0fbcbc5f92684d0157","_model_module":"@jupyter-widgets/controls","children":["IPY_MODEL_522063d62fb54c7fb9d4751b639014e1","IPY_MODEL_ca3713042d6d4fb188c3bc1de7e8563d"]}},"f7cd396d41e14a0fbcbc5f92684d0157":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"522063d62fb54c7fb9d4751b639014e1":{"model_module":"@jupyter-widgets/controls","model_name":"FloatProgressModel","state":{"_view_name":"ProgressView","style":"IPY_MODEL_18d3bcfcd2654d14aae6bd456107a9ef","_dom_classes":[],"description":"100%","_model_name":"FloatProgressModel","bar_style":"success","max":158,"_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":158,"_view_count":null,"_view_module_version":"1.5.0","orientation":"horizontal","min":0,"description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_c1f7955c23074f408d78a65a3294d97b"}},"ca3713042d6d4fb188c3bc1de7e8563d":{"model_module":"@jupyter-widgets/controls","model_name":"HTMLModel","state":{"_view_name":"HTMLView","style":"IPY_MODEL_3ea0dc9ff9044ab39a7911f80acb7e1b","_dom_classes":[],"description":"","_model_name":"HTMLModel","placeholder":"​","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":" 158/158 [00:01&lt;00:00, 136.30 batch/s]","_view_count":null,"_view_module_version":"1.5.0","description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_f82e488be9354d2895a1fc0c142ee10f"}},"18d3bcfcd2654d14aae6bd456107a9ef":{"model_module":"@jupyter-widgets/controls","model_name":"ProgressStyleModel","state":{"_view_name":"StyleView","_model_name":"ProgressStyleModel","description_width":"initial","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","bar_color":null,"_model_module":"@jupyter-widgets/controls"}},"c1f7955c23074f408d78a65a3294d97b":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"3ea0dc9ff9044ab39a7911f80acb7e1b":{"model_module":"@jupyter-widgets/controls","model_name":"DescriptionStyleModel","state":{"_view_name":"StyleView","_model_name":"DescriptionStyleModel","description_width":"","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","_model_module":"@jupyter-widgets/controls"}},"f82e488be9354d2895a1fc0c142ee10f":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"eddc3f39ed034abbaccb8d81076303d3":{"model_module":"@jupyter-widgets/controls","model_name":"HBoxModel","state":{"_view_name":"HBoxView","_dom_classes":[],"_model_name":"HBoxModel","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.5.0","box_style":"","layout":"IPY_MODEL_362580f0dfaf40ae839b0c60fecde399","_model_module":"@jupyter-widgets/controls","children":["IPY_MODEL_a0ad90dc837146019312c6afae7c4d9d","IPY_MODEL_6ca23d2c8fca4da584b4cfa0592dc6f0"]}},"362580f0dfaf40ae839b0c60fecde399":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"a0ad90dc837146019312c6afae7c4d9d":{"model_module":"@jupyter-widgets/controls","model_name":"FloatProgressModel","state":{"_view_name":"ProgressView","style":"IPY_MODEL_6f90fce0d629439f9f597239d8eb9b89","_dom_classes":[],"description":"100%","_model_name":"FloatProgressModel","bar_style":"success","max":157113,"_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":157113,"_view_count":null,"_view_module_version":"1.5.0","orientation":"horizontal","min":0,"description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_20e63d3b0e394aae80cca1d4818fa778"}},"6ca23d2c8fca4da584b4cfa0592dc6f0":{"model_module":"@jupyter-widgets/controls","model_name":"HTMLModel","state":{"_view_name":"HTMLView","style":"IPY_MODEL_eed2003e72974af79e7d652de685aaeb","_dom_classes":[],"description":"","_model_name":"HTMLModel","placeholder":"​","_view_module":"@jupyter-widgets/controls","_model_module_version":"1.5.0","value":" 157113/157113 [07:36&lt;00:00, 344.42 ORF/s]","_view_count":null,"_view_module_version":"1.5.0","description_tooltip":null,"_model_module":"@jupyter-widgets/controls","layout":"IPY_MODEL_42c9235aa9b347a0b5846273a39a3d03"}},"6f90fce0d629439f9f597239d8eb9b89":{"model_module":"@jupyter-widgets/controls","model_name":"ProgressStyleModel","state":{"_view_name":"StyleView","_model_name":"ProgressStyleModel","description_width":"initial","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","bar_color":null,"_model_module":"@jupyter-widgets/controls"}},"20e63d3b0e394aae80cca1d4818fa778":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}},"eed2003e72974af79e7d652de685aaeb":{"model_module":"@jupyter-widgets/controls","model_name":"DescriptionStyleModel","state":{"_view_name":"StyleView","_model_name":"DescriptionStyleModel","description_width":"","_view_module":"@jupyter-widgets/base","_model_module_version":"1.5.0","_view_count":null,"_view_module_version":"1.2.0","_model_module":"@jupyter-widgets/controls"}},"42c9235aa9b347a0b5846273a39a3d03":{"model_module":"@jupyter-widgets/base","model_name":"LayoutModel","state":{"_view_name":"LayoutView","grid_template_rows":null,"right":null,"justify_content":null,"_view_module":"@jupyter-widgets/base","overflow":null,"_model_module_version":"1.2.0","_view_count":null,"flex_flow":null,"width":null,"min_width":null,"border":null,"align_items":null,"bottom":null,"_model_module":"@jupyter-widgets/base","top":null,"grid_column":null,"overflow_y":null,"overflow_x":null,"grid_auto_flow":null,"grid_area":null,"grid_template_columns":null,"flex":null,"_model_name":"LayoutModel","justify_items":null,"grid_row":null,"max_height":null,"align_content":null,"visibility":null,"align_self":null,"height":null,"min_height":null,"padding":null,"grid_auto_rows":null,"grid_gap":null,"max_width":null,"order":null,"_view_module_version":"1.2.0","grid_template_areas":null,"object_position":null,"object_fit":null,"grid_auto_columns":null,"margin":null,"display":null,"left":null}}}}},"cells":[{"cell_type":"markdown","metadata":{"id":"6SYiIHRhZM-m","colab_type":"text"},"source":["# Google Colab Intro\n","#### This notebook downloads data and performs computations. These computations are made significantly faster with the use of a GPU, provided free by Google as part of Colab.\n","\n","#### Press the play button on the left side of each cell to run it. Alternatively, hold shift or ctrl and press enter to run cells.\n","#### Double click the top of a cell to inspect the code inside and change things. Double click the right side of the cell to hide the code.\n","#### Have fun!\n","\n","#### (bug me on github if a significantly faster c++ version would be useful to you)"]},{"cell_type":"markdown","metadata":{"id":"aEszqvakZN9C","colab_type":"text"},"source":["# Setup"]},{"cell_type":"code","metadata":{"id":"VMJQa6JAXDf_","colab_type":"code","cellView":"form","colab":{"base_uri":"https://localhost:8080/","height":204},"executionInfo":{"status":"ok","timestamp":1599437733921,"user_tz":240,"elapsed":7961,"user":{"displayName":"Markus Sommer","photoUrl":"https://lh3.googleusercontent.com/a-/AOh14GjXujbpCzD6oVvcRIT5EP-oi3Bif3Cfcu7npJUnce8=s64","userId":"08083804319459268544"}},"outputId":"87779f37-4da2-4ab1-a8e5-58f630298cd2"},"source":["# @title Install dependencies\n","!pip install biopython\n","!pip install torch\n","print(\"\\nDone\")"],"execution_count":1,"outputs":[{"output_type":"stream","text":["Collecting biopython\n","\u001b[?25l  Downloading https://files.pythonhosted.org/packages/76/02/8b606c4aa92ff61b5eda71d23b499ab1de57d5e818be33f77b01a6f435a8/biopython-1.78-cp36-cp36m-manylinux1_x86_64.whl (2.3MB)\n","\u001b[K     |████████████████████████████████| 2.3MB 7.6MB/s \n","\u001b[?25hRequirement already satisfied: numpy in /usr/local/lib/python3.6/dist-packages (from biopython) (1.18.5)\n","Installing collected packages: biopython\n","Successfully installed biopython-1.78\n","Requirement already satisfied: torch in /usr/local/lib/python3.6/dist-packages (1.6.0+cu101)\n","Requirement already satisfied: numpy in /usr/local/lib/python3.6/dist-packages (from torch) (1.18.5)\n","Requirement already satisfied: future in /usr/local/lib/python3.6/dist-packages (from torch) (0.16.0)\n","\n","Done\n"],"name":"stdout"}]},{"cell_type":"code","metadata":{"id":"pe1PHlcjeHS8","colab_type":"code","cellView":"both","colab":{"base_uri":"https://localhost:8080/","height":34},"executionInfo":{"status":"ok","timestamp":1599437737828,"user_tz":240,"elapsed":11864,"user":{"displayName":"Markus Sommer","photoUrl":"https://lh3.googleusercontent.com/a-/AOh14GjXujbpCzD6oVvcRIT5EP-oi3Bif3Cfcu7npJUnce8=s64","userId":"08083804319459268544"}},"outputId":"8c927ef5-361e-4fa5-9827-1629c8dcc0e5"},"source":["# @title Import python packages\n","import os\n","import sys\n","import gzip\n","import copy\n","import time\n","import pandas\n","import pickle\n","import numpy as np\n","from tqdm.auto import tqdm\n","from Bio import SeqIO\n","from scipy.special import expit\n","from scipy.special import logit\n","from multiprocessing import Pool\n","\n","import torch\n","import torch.nn as nn\n","import torch.nn.functional as F\n","from torch.nn.utils import weight_norm\n","\n","# # import networkx as nx\n","# from scipy.sparse import coo_matrix\n","# from scipy.sparse import csr_matrix\n","# from scipy.sparse.csgraph import bellman_ford\n","\n","print(\"Done\")"],"execution_count":2,"outputs":[{"output_type":"stream","text":["Done\n"],"name":"stdout"}]},{"cell_type":"code","metadata":{"id":"usWmfqJNhM2X","colab_type":"code","cellView":"both","colab":{"base_uri":"https://localhost:8080/","height":34},"executionInfo":{"status":"ok","timestamp":1599437737828,"user_tz":240,"elapsed":7757,"user":{"displayName":"Markus Sommer","photoUrl":"https://lh3.googleusercontent.com/a-/AOh14GjXujbpCzD6oVvcRIT5EP-oi3Bif3Cfcu7npJUnce8=s64","userId":"08083804319459268544"}},"outputId":"bc260f86-a663-4ccd-bd3e-eafc0324e70d"},"source":["# @title Set parameters (double click top of cell to change defaults)\n","\"\"\" Print what the program is doing.\"\"\"\n","verbose = True\n","\n","\"\"\" Maximum ORF (open reading frame) overlap length in nucleotides.\"\"\"\n","max_gene_overlap = 60\n","\n","\"\"\" Minimum ORF length in nucleotides.\"\"\"\n","min_orf_length = 60\n","\n","\"\"\" Use kmer prefilter to increase gene sensitivity. \n","May not play nice with very high GC genomes.\"\"\"\n","# protein_kmer_filter = True\n","protein_kmer_filter = False\n","\n","\"\"\" Use mmseqs2 and a gene score cutoff to remove most false positive predictions.\"\"\"\n","# mmseqs2_gene_filter = True\n","mmseqs2_gene_filter = False\n","\n","\n","\"\"\" Nucleotide to amino acid translation table. 11 for most bacteria/archaea.\n","4 for Mycoplasma/Spiroplasma.\"\"\"\n","translation_table = 11\n","# translation_table = 4\n","\n","\"\"\" Maximum number of forward connections in the directed acyclic graph used to\n","find a set of coherent genes in each genome.\n","Higher values will slow execution time and increase memory usage,\n","but may slightly increase performace.\n","Recommended range ~30-50\"\"\"\n","max_forward_connections = 50\n","\n","\"\"\" Batch size for the temporal convolutional network used to score genes.\n","Small batches and big batches slow down the model. Very big batches may crash the \n","GPU. \"\"\"\n","gene_batch_size = 200\n","TIS_batch_size = 1000\n","\n","\"\"\" Where the pre-trained gene model should be saved.\"\"\"\n","model_dir = \"/home\"\n","\n","\"\"\" All following are internal parameters. Change at your own risk.\"\"\"\n","weight_gene_prob = 0.9746869839852076 \n","weight_TIS_prob = 0.25380288790532707 \n","score_threshold = 0.47256101519707244\n","weight_ATG = 0.84249804151264 \n","weight_GTG = 0.7083689705744909\n","weight_TTG = 0.7512400826652517 \n","unidirectional_penalty_per_base = 3.895921717182765 # 3' 5' overlap\n","convergent_penalty_per_base = 4.603432608883688 # 3' 3' overlap\n","divergent_penalty_per_base = 3.3830814940689975 # 5' 5' overlap\n","k_seengene = 10\n","multimer_threshold = 2\n","\n","print(\"Done\")"],"execution_count":3,"outputs":[{"output_type":"stream","text":["Done\n"],"name":"stdout"}]},{"cell_type":"code","metadata":{"id":"idOvBwhe0NtP","colab_type":"code","cellView":"form","colab":{"base_uri":"https://localhost:8080/","height":102},"executionInfo":{"status":"ok","timestamp":1599437758891,"user_tz":240,"elapsed":21059,"user":{"displayName":"Markus Sommer","photoUrl":"https://lh3.googleusercontent.com/a-/AOh14GjXujbpCzD6oVvcRIT5EP-oi3Bif3Cfcu7npJUnce8=s64","userId":"08083804319459268544"}},"outputId":"6172e5af-120c-45ff-87aa-f59f5d4d8e64"},"source":["# @title Load pre-trained gene and translation initiation site models\n","\n","\"\"\" if you're interested in the inner workings of the \n","temporal convolutional network, see hubconf.py in the Github repo below.\"\"\"\n","\n","repo = \"salzberg-lab/Balrog\"\n","# repo = \"salzberg-lab/Balrog:develop\"\n","\n","torch.hub.set_dir(model_dir)\n","if torch.cuda.device_count() > 0:\n","    print(\"GPU detected...\")\n","    model = torch.hub.load(repo, \"geneTCN\", force_reload=True).cuda()\n","    model_tis = torch.hub.load(repo, \"tisTCN\", force_reload=False).cuda()\n","    time.sleep(0.5)\n","    print(\"\\nDone\")\n","else:\n","    print(\"No GPU detected, using CPU...\")\n","    model = torch.hub.load(repo, \"geneTCN\", force_reload=True)\n","    model_tis = torch.hub.load(repo, \"tisTCN\", force_reload=False)\n","    time.sleep(0.5)\n","    print(\"\\nDone\")"],"execution_count":4,"outputs":[{"output_type":"stream","text":["GPU detected...\n"],"name":"stdout"},{"output_type":"stream","text":["Downloading: \"https://github.com/salzberg-lab/Balrog/archive/master.zip\" to /home/master.zip\n","Using cache found in /home/salzberg-lab_Balrog_master\n"],"name":"stderr"},{"output_type":"stream","text":["\n","Done\n"],"name":"stdout"}]},{"cell_type":"code","metadata":{"id":"wQMPFNbdk8vv","colab_type":"code","cellView":"form","colab":{"base_uri":"https://localhost:8080/","height":51},"executionInfo":{"status":"ok","timestamp":1599437758892,"user_tz":240,"elapsed":21057,"user":{"displayName":"Markus Sommer","photoUrl":"https://lh3.googleusercontent.com/a-/AOh14GjXujbpCzD6oVvcRIT5EP-oi3Bif3Cfcu7npJUnce8=s64","userId":"08083804319459268544"}},"outputId":"5da81910-4926-4283-c04a-3d7f899a684e"},"source":["# @title Prepare protein kmer filter\n","if protein_kmer_filter:\n","    # decompress kmer filter\n","    seengene_dir = \"/content/kmerfilter/\"\n","    !mkdir {seengene_dir}\n","    %cd {seengene_dir}\n","    !tar -xvzf /home/salzberg-lab_Balrog_master/kmer_filter/genexa_10mer_thresh2_minusARF_all.tar.gz\n","    # !tar -xvzf /home/salzberg-lab_Balrog_develop/kmer_filter/genexa_10mer_thresh2_minusARF_all.tar.gz\n","    genexa_kmer_path = os.path.join(seengene_dir, \"10mer_thresh2_minusARF_all.pkl\")\n","\n","    # load kmer filter\n","    with open(genexa_kmer_path, \"rb\") as f:\n","        aa_kmer_set = pickle.load(f)\n","\n","print(\"\\nDone\")"],"execution_count":5,"outputs":[{"output_type":"stream","text":["\n","Done\n"],"name":"stdout"}]},{"cell_type":"code","metadata":{"id":"p3jzwdcGp7XN","colab_type":"code","cellView":"form","colab":{"base_uri":"https://localhost:8080/","height":51},"executionInfo":{"status":"ok","timestamp":1599437758892,"user_tz":240,"elapsed":21054,"user":{"displayName":"Markus Sommer","photoUrl":"https://lh3.googleusercontent.com/a-/AOh14GjXujbpCzD6oVvcRIT5EP-oi3Bif3Cfcu7npJUnce8=s64","userId":"08083804319459268544"}},"outputId":"07635862-772e-42f8-cfa4-5964244d81b0"},"source":["# @title Prepare mmseqs2\n","\n","if mmseqs2_gene_filter:\n","    ![ $(uname -m) = \"x86_64\" ] && echo \"64bit: Yes\" || echo \"64bit: No\"\n","    !grep -q sse4_1 /proc/cpuinfo && echo \"SSE4.1: Yes\" || echo \"SSE4.1: No\"\n","    !grep -q avx2 /proc/cpuinfo && echo \"AVX2: Yes\" || echo \"AVX2: No\"\n","\n","    # install\n","    !mkdir /content/mmseqs2\n","    %cd /content/mmseqs2\n","    !wget https://mmseqs.com/latest/mmseqs-linux-avx2.tar.gz\n","    !tar xvzf mmseqs-linux-avx2.tar.gz\n","\n","    # decompress fasta\n","    !mkdir /content/mmseqs2/genexa\n","    %cd /content/mmseqs2/genexa\n","    !!tar -xvzf /home/salzberg-lab_Balrog_master/protein_filter/genexa_genes.tar.gz\n","    # !!tar -xvzf /home/salzberg-lab_Balrog_develop/protein_filter/genexa_genes.tar.gz\n","    genexa_fasta_path = \"/content/mmseqs2/genexa/genexa_genes.fasta\"\n","\n","    # create DB\n","    genexa_DB_path = \"/content/mmseqs2/genexa/genexaDB\"\n","    !/content/mmseqs2/mmseqs/bin/mmseqs createdb {genexa_fasta_path} {genexa_DB_path}\n","\n","    # build mmseqs index\n","    !mkdir /content/mmseqs2/tmp\n","    !/content/mmseqs2/mmseqs/bin/mmseqs createindex {genexa_DB_path} /content/mmseqs2/tmp\n","\n","    # download swissprot DB\n","    !mkdir /content/mmseqs2/swissprot\n","    !/content/mmseqs2/mmseqs/bin/mmseqs databases UniProtKB/Swiss-Prot /content/mmseqs2/swissprot/swissprotDB /content/mmseqs2/tmp\n","    swissprot_DB_path = \"/content/mmseqs2/swissprot/swissprotDB\"\n","    \n","print(\"\\nDone\")"],"execution_count":6,"outputs":[{"output_type":"stream","text":["\n","Done\n"],"name":"stdout"}]},{"cell_type":"markdown","metadata":{"id":"1Ea6jrhpfF4y","colab_type":"text"},"source":["# Gene Prediction"]},{"cell_type":"code","metadata":{"id":"uh274cXnWlva","colab_type":"code","cellView":"form","colab":{"resources":{"http://localhost:8080/nbextensions/google.colab/files.js":{"data":"Ly8gQ29weXJpZ2h0IDIwMTcgR29vZ2xlIExMQwovLwovLyBMaWNlbnNlZCB1bmRlciB0aGUgQXBhY2hlIExpY2Vuc2UsIFZlcnNpb24gMi4wICh0aGUgIkxpY2Vuc2UiKTsKLy8geW91IG1heSBub3QgdXNlIHRoaXMgZmlsZSBleGNlcHQgaW4gY29tcGxpYW5jZSB3aXRoIHRoZSBMaWNlbnNlLgovLyBZb3UgbWF5IG9idGFpbiBhIGNvcHkgb2YgdGhlIExpY2Vuc2UgYXQKLy8KLy8gICAgICBodHRwOi8vd3d3LmFwYWNoZS5vcmcvbGljZW5zZXMvTElDRU5TRS0yLjAKLy8KLy8gVW5sZXNzIHJlcXVpcmVkIGJ5IGFwcGxpY2FibGUgbGF3IG9yIGFncmVlZCB0byBpbiB3cml0aW5nLCBzb2Z0d2FyZQovLyBkaXN0cmlidXRlZCB1bmRlciB0aGUgTGljZW5zZSBpcyBkaXN0cmlidXRlZCBvbiBhbiAiQVMgSVMiIEJBU0lTLAovLyBXSVRIT1VUIFdBUlJBTlRJRVMgT1IgQ09ORElUSU9OUyBPRiBBTlkgS0lORCwgZWl0aGVyIGV4cHJlc3Mgb3IgaW1wbGllZC4KLy8gU2VlIHRoZSBMaWNlbnNlIGZvciB0aGUgc3BlY2lmaWMgbGFuZ3VhZ2UgZ292ZXJuaW5nIHBlcm1pc3Npb25zIGFuZAovLyBsaW1pdGF0aW9ucyB1bmRlciB0aGUgTGljZW5zZS4KCi8qKgogKiBAZmlsZW92ZXJ2aWV3IEhlbHBlcnMgZm9yIGdvb2dsZS5jb2xhYiBQeXRob24gbW9kdWxlLgogKi8KKGZ1bmN0aW9uKHNjb3BlKSB7CmZ1bmN0aW9uIHNwYW4odGV4dCwgc3R5bGVBdHRyaWJ1dGVzID0ge30pIHsKICBjb25zdCBlbGVtZW50ID0gZG9jdW1lbnQuY3JlYXRlRWxlbWVudCgnc3BhbicpOwogIGVsZW1lbnQudGV4dENvbnRlbnQgPSB0ZXh0OwogIGZvciAoY29uc3Qga2V5IG9mIE9iamVjdC5rZXlzKHN0eWxlQXR0cmlidXRlcykpIHsKICAgIGVsZW1lbnQuc3R5bGVba2V5XSA9IHN0eWxlQXR0cmlidXRlc1trZXldOwogIH0KICByZXR1cm4gZWxlbWVudDsKfQoKLy8gTWF4IG51bWJlciBvZiBieXRlcyB3aGljaCB3aWxsIGJlIHVwbG9hZGVkIGF0IGEgdGltZS4KY29uc3QgTUFYX1BBWUxPQURfU0laRSA9IDEwMCAqIDEwMjQ7CgpmdW5jdGlvbiBfdXBsb2FkRmlsZXMoaW5wdXRJZCwgb3V0cHV0SWQpIHsKICBjb25zdCBzdGVwcyA9IHVwbG9hZEZpbGVzU3RlcChpbnB1dElkLCBvdXRwdXRJZCk7CiAgY29uc3Qgb3V0cHV0RWxlbWVudCA9IGRvY3VtZW50LmdldEVsZW1lbnRCeUlkKG91dHB1dElkKTsKICAvLyBDYWNoZSBzdGVwcyBvbiB0aGUgb3V0cHV0RWxlbWVudCB0byBtYWtlIGl0IGF2YWlsYWJsZSBmb3IgdGhlIG5leHQgY2FsbAogIC8vIHRvIHVwbG9hZEZpbGVzQ29udGludWUgZnJvbSBQeXRob24uCiAgb3V0cHV0RWxlbWVudC5zdGVwcyA9IHN0ZXBzOwoKICByZXR1cm4gX3VwbG9hZEZpbGVzQ29udGludWUob3V0cHV0SWQpOwp9CgovLyBUaGlzIGlzIHJvdWdobHkgYW4gYXN5bmMgZ2VuZXJhdG9yIChub3Qgc3VwcG9ydGVkIGluIHRoZSBicm93c2VyIHlldCksCi8vIHdoZXJlIHRoZXJlIGFyZSBtdWx0aXBsZSBhc3luY2hyb25vdXMgc3RlcHMgYW5kIHRoZSBQeXRob24gc2lkZSBpcyBnb2luZwovLyB0byBwb2xsIGZvciBjb21wbGV0aW9uIG9mIGVhY2ggc3RlcC4KLy8gVGhpcyB1c2VzIGEgUHJvbWlzZSB0byBibG9jayB0aGUgcHl0aG9uIHNpZGUgb24gY29tcGxldGlvbiBvZiBlYWNoIHN0ZXAsCi8vIHRoZW4gcGFzc2VzIHRoZSByZXN1bHQgb2YgdGhlIHByZXZpb3VzIHN0ZXAgYXMgdGhlIGlucHV0IHRvIHRoZSBuZXh0IHN0ZXAuCmZ1bmN0aW9uIF91cGxvYWRGaWxlc0NvbnRpbnVlKG91dHB1dElkKSB7CiAgY29uc3Qgb3V0cHV0RWxlbWVudCA9IGRvY3VtZW50LmdldEVsZW1lbnRCeUlkKG91dHB1dElkKTsKICBjb25zdCBzdGVwcyA9IG91dHB1dEVsZW1lbnQuc3RlcHM7CgogIGNvbnN0IG5leHQgPSBzdGVwcy5uZXh0KG91dHB1dEVsZW1lbnQubGFzdFByb21pc2VWYWx1ZSk7CiAgcmV0dXJuIFByb21pc2UucmVzb2x2ZShuZXh0LnZhbHVlLnByb21pc2UpLnRoZW4oKHZhbHVlKSA9PiB7CiAgICAvLyBDYWNoZSB0aGUgbGFzdCBwcm9taXNlIHZhbHVlIHRvIG1ha2UgaXQgYXZhaWxhYmxlIHRvIHRoZSBuZXh0CiAgICAvLyBzdGVwIG9mIHRoZSBnZW5lcmF0b3IuCiAgICBvdXRwdXRFbGVtZW50Lmxhc3RQcm9taXNlVmFsdWUgPSB2YWx1ZTsKICAgIHJldHVybiBuZXh0LnZhbHVlLnJlc3BvbnNlOwogIH0pOwp9CgovKioKICogR2VuZXJhdG9yIGZ1bmN0aW9uIHdoaWNoIGlzIGNhbGxlZCBiZXR3ZWVuIGVhY2ggYXN5bmMgc3RlcCBvZiB0aGUgdXBsb2FkCiAqIHByb2Nlc3MuCiAqIEBwYXJhbSB7c3RyaW5nfSBpbnB1dElkIEVsZW1lbnQgSUQgb2YgdGhlIGlucHV0IGZpbGUgcGlja2VyIGVsZW1lbnQuCiAqIEBwYXJhbSB7c3RyaW5nfSBvdXRwdXRJZCBFbGVtZW50IElEIG9mIHRoZSBvdXRwdXQgZGlzcGxheS4KICogQHJldHVybiB7IUl0ZXJhYmxlPCFPYmplY3Q+fSBJdGVyYWJsZSBvZiBuZXh0IHN0ZXBzLgogKi8KZnVuY3Rpb24qIHVwbG9hZEZpbGVzU3RlcChpbnB1dElkLCBvdXRwdXRJZCkgewogIGNvbnN0IGlucHV0RWxlbWVudCA9IGRvY3VtZW50LmdldEVsZW1lbnRCeUlkKGlucHV0SWQpOwogIGlucHV0RWxlbWVudC5kaXNhYmxlZCA9IGZhbHNlOwoKICBjb25zdCBvdXRwdXRFbGVtZW50ID0gZG9jdW1lbnQuZ2V0RWxlbWVudEJ5SWQob3V0cHV0SWQpOwogIG91dHB1dEVsZW1lbnQuaW5uZXJIVE1MID0gJyc7CgogIGNvbnN0IHBpY2tlZFByb21pc2UgPSBuZXcgUHJvbWlzZSgocmVzb2x2ZSkgPT4gewogICAgaW5wdXRFbGVtZW50LmFkZEV2ZW50TGlzdGVuZXIoJ2NoYW5nZScsIChlKSA9PiB7CiAgICAgIHJlc29sdmUoZS50YXJnZXQuZmlsZXMpOwogICAgfSk7CiAgfSk7CgogIGNvbnN0IGNhbmNlbCA9IGRvY3VtZW50LmNyZWF0ZUVsZW1lbnQoJ2J1dHRvbicpOwogIGlucHV0RWxlbWVudC5wYXJlbnRFbGVtZW50LmFwcGVuZENoaWxkKGNhbmNlbCk7CiAgY2FuY2VsLnRleHRDb250ZW50ID0gJ0NhbmNlbCB1cGxvYWQnOwogIGNvbnN0IGNhbmNlbFByb21pc2UgPSBuZXcgUHJvbWlzZSgocmVzb2x2ZSkgPT4gewogICAgY2FuY2VsLm9uY2xpY2sgPSAoKSA9PiB7CiAgICAgIHJlc29sdmUobnVsbCk7CiAgICB9OwogIH0pOwoKICAvLyBXYWl0IGZvciB0aGUgdXNlciB0byBwaWNrIHRoZSBmaWxlcy4KICBjb25zdCBmaWxlcyA9IHlpZWxkIHsKICAgIHByb21pc2U6IFByb21pc2UucmFjZShbcGlja2VkUHJvbWlzZSwgY2FuY2VsUHJvbWlzZV0pLAogICAgcmVzcG9uc2U6IHsKICAgICAgYWN0aW9uOiAnc3RhcnRpbmcnLAogICAgfQogIH07CgogIGNhbmNlbC5yZW1vdmUoKTsKCiAgLy8gRGlzYWJsZSB0aGUgaW5wdXQgZWxlbWVudCBzaW5jZSBmdXJ0aGVyIHBpY2tzIGFyZSBub3QgYWxsb3dlZC4KICBpbnB1dEVsZW1lbnQuZGlzYWJsZWQgPSB0cnVlOwoKICBpZiAoIWZpbGVzKSB7CiAgICByZXR1cm4gewogICAgICByZXNwb25zZTogewogICAgICAgIGFjdGlvbjogJ2NvbXBsZXRlJywKICAgICAgfQogICAgfTsKICB9CgogIGZvciAoY29uc3QgZmlsZSBvZiBmaWxlcykgewogICAgY29uc3QgbGkgPSBkb2N1bWVudC5jcmVhdGVFbGVtZW50KCdsaScpOwogICAgbGkuYXBwZW5kKHNwYW4oZmlsZS5uYW1lLCB7Zm9udFdlaWdodDogJ2JvbGQnfSkpOwogICAgbGkuYXBwZW5kKHNwYW4oCiAgICAgICAgYCgke2ZpbGUudHlwZSB8fCAnbi9hJ30pIC0gJHtmaWxlLnNpemV9IGJ5dGVzLCBgICsKICAgICAgICBgbGFzdCBtb2RpZmllZDogJHsKICAgICAgICAgICAgZmlsZS5sYXN0TW9kaWZpZWREYXRlID8gZmlsZS5sYXN0TW9kaWZpZWREYXRlLnRvTG9jYWxlRGF0ZVN0cmluZygpIDoKICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgJ24vYSd9IC0gYCkpOwogICAgY29uc3QgcGVyY2VudCA9IHNwYW4oJzAlIGRvbmUnKTsKICAgIGxpLmFwcGVuZENoaWxkKHBlcmNlbnQpOwoKICAgIG91dHB1dEVsZW1lbnQuYXBwZW5kQ2hpbGQobGkpOwoKICAgIGNvbnN0IGZpbGVEYXRhUHJvbWlzZSA9IG5ldyBQcm9taXNlKChyZXNvbHZlKSA9PiB7CiAgICAgIGNvbnN0IHJlYWRlciA9IG5ldyBGaWxlUmVhZGVyKCk7CiAgICAgIHJlYWRlci5vbmxvYWQgPSAoZSkgPT4gewogICAgICAgIHJlc29sdmUoZS50YXJnZXQucmVzdWx0KTsKICAgICAgfTsKICAgICAgcmVhZGVyLnJlYWRBc0FycmF5QnVmZmVyKGZpbGUpOwogICAgfSk7CiAgICAvLyBXYWl0IGZvciB0aGUgZGF0YSB0byBiZSByZWFkeS4KICAgIGxldCBmaWxlRGF0YSA9IHlpZWxkIHsKICAgICAgcHJvbWlzZTogZmlsZURhdGFQcm9taXNlLAogICAgICByZXNwb25zZTogewogICAgICAgIGFjdGlvbjogJ2NvbnRpbnVlJywKICAgICAgfQogICAgfTsKCiAgICAvLyBVc2UgYSBjaHVua2VkIHNlbmRpbmcgdG8gYXZvaWQgbWVzc2FnZSBzaXplIGxpbWl0cy4gU2VlIGIvNjIxMTU2NjAuCiAgICBsZXQgcG9zaXRpb24gPSAwOwogICAgd2hpbGUgKHBvc2l0aW9uIDwgZmlsZURhdGEuYnl0ZUxlbmd0aCkgewogICAgICBjb25zdCBsZW5ndGggPSBNYXRoLm1pbihmaWxlRGF0YS5ieXRlTGVuZ3RoIC0gcG9zaXRpb24sIE1BWF9QQVlMT0FEX1NJWkUpOwogICAgICBjb25zdCBjaHVuayA9IG5ldyBVaW50OEFycmF5KGZpbGVEYXRhLCBwb3NpdGlvbiwgbGVuZ3RoKTsKICAgICAgcG9zaXRpb24gKz0gbGVuZ3RoOwoKICAgICAgY29uc3QgYmFzZTY0ID0gYnRvYShTdHJpbmcuZnJvbUNoYXJDb2RlLmFwcGx5KG51bGwsIGNodW5rKSk7CiAgICAgIHlpZWxkIHsKICAgICAgICByZXNwb25zZTogewogICAgICAgICAgYWN0aW9uOiAnYXBwZW5kJywKICAgICAgICAgIGZpbGU6IGZpbGUubmFtZSwKICAgICAgICAgIGRhdGE6IGJhc2U2NCwKICAgICAgICB9LAogICAgICB9OwogICAgICBwZXJjZW50LnRleHRDb250ZW50ID0KICAgICAgICAgIGAke01hdGgucm91bmQoKHBvc2l0aW9uIC8gZmlsZURhdGEuYnl0ZUxlbmd0aCkgKiAxMDApfSUgZG9uZWA7CiAgICB9CiAgfQoKICAvLyBBbGwgZG9uZS4KICB5aWVsZCB7CiAgICByZXNwb25zZTogewogICAgICBhY3Rpb246ICdjb21wbGV0ZScsCiAgICB9CiAgfTsKfQoKc2NvcGUuZ29vZ2xlID0gc2NvcGUuZ29vZ2xlIHx8IHt9OwpzY29wZS5nb29nbGUuY29sYWIgPSBzY29wZS5nb29nbGUuY29sYWIgfHwge307CnNjb3BlLmdvb2dsZS5jb2xhYi5fZmlsZXMgPSB7CiAgX3VwbG9hZEZpbGVzLAogIF91cGxvYWRGaWxlc0NvbnRpbnVlLAp9Owp9KShzZWxmKTsK","ok":true,"headers":[["content-type","application/javascript"]],"status":200,"status_text":""}},"base_uri":"https://localhost:8080/","height":242},"executionInfo":{"status":"ok","timestamp":1599427233720,"user_tz":240,"elapsed":38468,"user":{"displayName":"Markus Sommer","photoUrl":"https://lh3.googleusercontent.com/a-/AOh14GjXujbpCzD6oVvcRIT5EP-oi3Bif3Cfcu7npJUnce8=s64","userId":"08083804319459268544"}},"outputId":"f449d80c-6a3d-453b-acb2-80407b820733"},"source":["# @title Upload prokaryotic genomes as FASTA or gzipped FASTA.\n","from google.colab import files\n","genome_dict = files.upload()\n","\n","print(\"\\nDone\")"],"execution_count":26,"outputs":[{"output_type":"display_data","data":{"text/html":["\n","     <input type=\"file\" id=\"files-941c7208-fcd4-46cb-a0bd-e82e11f05719\" name=\"files[]\" multiple disabled\n","        style=\"border:none\" />\n","     <output id=\"result-941c7208-fcd4-46cb-a0bd-e82e11f05719\">\n","      Upload widget is only available when the cell has been executed in the\n","      current browser session. Please rerun this cell to enable.\n","      </output>\n","      <script src=\"/nbextensions/google.colab/files.js\"></script> "],"text/plain":["<IPython.core.display.HTML object>"]},"metadata":{"tags":[]}},{"output_type":"stream","text":["Saving GCA_000830275.1_ASM83027v1_genomic.fna.gz to GCA_000830275.1_ASM83027v1_genomic.fna.gz\n","Saving GCF_000015765.1_ASM1576v1_genomic.fna.gz to GCF_000015765.1_ASM1576v1_genomic.fna.gz\n","Saving GCF_000022205.1_ASM2220v1_genomic.fna.gz to GCF_000022205.1_ASM2220v1_genomic.fna.gz\n","Saving GCF_000024185.1_ASM2418v1_genomic.fna.gz to GCF_000024185.1_ASM2418v1_genomic.fna.gz\n","Saving GCF_000969965.1_ASM96996v1_genomic.fna.gz to GCF_000969965.1_ASM96996v1_genomic.fna.gz\n","\n","Done\n"],"name":"stdout"}]},{"cell_type":"code","metadata":{"id":"NC4llyrGWIuK","colab_type":"code","cellView":"both","colab":{"base_uri":"https://localhost:8080/","height":1000,"referenced_widgets":["7977b776a3a74915b48aaabcb8ad0991","3b37b21389cf4cc285293764a9208733","7c00d07501114a4989d889e4c2dc9b61","13de506a4ec6456982547694e0c7c845","145ae87c4b9643e6a418e83ea685a20a","9278878f5b774b0d9def450906d388b3","b56880c809494da4b84e7596550a6b64","f1a2fc4f62cd4e71be7be689c5be3299","5d15e0c51e7d406c8b642fa8548c0c5e","16806a86cb7c444fa7cec1c5ad7e302c","d2e1117978704746bb721116210c9480","931580a5c8ad4855ad7cd16248786120","afeff62a0f6f4c41ad635a0936f238f7","4c9ff1da3b62444e96293e5eb2148510","989bdb2d67e94962bbacae7f2a3d5b5d","4f0a51cb0b894c2d96894a9b2ae2e6fd","4a9040116613475b9c7a81fb1c0cd7e9","ffc19f5ad6c74c5b9062540883ae70b0","fa8dc13e01ac4cef849aa73b86722dfb","d6d821d3b15a4a6b8a6c1bccdacd0c39","cee1b7d6d6414966b62009c12eced574","ebe91a4f4d294dd7a4d5a82fb95918d4","6b84f65d546b4ae79739f0489a33f8a7","e08f1af1d60a4a08a27d73f9c0db2bf3","aa21151cc1c24f24aa33fb91c9b294ab","ff5147ab95eb4b73b85887ec22e2ef8d","421b1e265c3246e2a73c6c03a3522a7e","f8827866f95a40dfa75683d2ad18ad6f","46b6215f398f40ea8199b2fdfa708b8a","cc773c135af04acd98529e37a31abac8","b635ddfae7e54c04b3c9cec5c4fa17c2","e2b67b7421db42ae83372d45b7e8529a","9f043bf115c648d48fbf6361c6ed7974","0f9fb553cee84dd5bf66add251a0c8a0","f1c3b5413f014680ae99dbe54b905b2c","538e89d21c0040dcacf1ac1b328f1da7","31cd5630cf4c4518bf80a2092ec9edc4","b614c9fa3c9c499caf19103ea7533ac0","13132cddf38c464d82cd2e2017f9bfc2","14d7c7e0b36f4967b9211d81221ca644","ce980cd159934f818f76c85a31b190a7","50fd16a576844022b905c98eda4f773c","e7d2f806e53a4bb78d493ae315be8847","da6b20a369b94e4f9e1a3ed08afb8c9b","ceadc3e1fc3f4781860415bc040f46e2","ea2f44df4e6e44659e4c9dfbba6752ef","26c2bfc43ffb4bd7b77ef4428809cbba","bd487d2a0f264b22824fb217d6c60717","8a4dc35a4b6e43a4b06d087bf11c0b57","8f98a6f44da24d338a0d2ffc1f827560","8ceae92967054fa4aeb30c044d8c91ea","91b2ca25d8ee4c02acc3a20af9505243","41441da3be8444b8857683e930d26d01","8151f7e33c26426e9dd77670967043bd","52dbe6705de346a7aa60710a453f923c","18ec8a2356ed4f368224e14ac307ee08","a5b54807a89945ae9767d8677a4a76f7","d8edb7bbba0a4117adc661954ed909f8","7711e91fae58418bab84a2efa6c5fcbe","3e8f56a839414257a2ee45a157953887","9e9bd36888c34ee98489660fde13fa3b","38256f5f339f4fdfa2242c020b34aa3d","bc215acc4f5b4d6aa5bcdded31a6edbb","f75b905f5e654846b42a40112b177c4a","f038606a5e8a42d7b4585ea88c1cdf6b","58e10a92f50943b482bf06458b63344d","a755e7f2f03b46d6b0926d268f7acfa5","66e1bf6b0cee4dad8993e9bc48e19429","54916cb120e74bd2965e1b008296022f","734443a68cbe4e9790ed40f2c200ec0b","fd66aee63683423a9f4ebd08d7d6a3c7","503f02cf791e4239a487070c37252309","58b858f22c204471a4cf71d7c2458d4d","50de702199cd4a2abbb4228af66a3925","4a3246a4014b4d5584a543a59d29536c","478a81f1317643709b884c690db8eead","899713e6fee94abc9567e00b71bc56c1","1d557593e4d844d98702316d42333271","3be46630a17543c2b30b4de721b7493e","a3e057a75335419c877989b80716c612","32cd0fe732b54002b4664c28dc5e074b","fe4238f572fd4131aee053c6845d40c3","6add7163926a4f92b0113d7f28ff9fdd","3e4c8d22929f4eaab967b51d08000dde","c51a8eaa4b4b43188ad81f66a0d6b44e","d385bdce97454bccba72d91dc4357539","3aac636b96f64775b96fb405bbf3501f","0ed1714d75e64e8a823b6b4ec19ec7db","83252378b2d443de95e55481e3788ffa","c128e610689c4c70bda2a581fb244537","d1cc6df4231b4569ac3443e9b68083f3","999de9276e984e47b01c05a90d2d077a","9caa4fa3605647db96043437153d85fd","8e7302e3c0434dc6a8619e582bb9ceda","b7ae3b2c79854a3897b8bee8408ce218","cddd3b1c0f1f4d85afd3d14cec9e5451","784f9f6d4f91404d9e6060a04e01b7de","3b20bfe327f84e20be79dc4f605d56d3","a7f6170e69ab442888025f2f481d26cb","13e646e6783348fba1cbd7dff6024970","11253536107d4a959d340a2ea0681ca9","907a7da9fbd04fb38a8276ad51288d5e","a501c387432643f2bbee44b737e13545","ab1c440c08f94f479a5e74395b8340ca","7f6ca2b1839f4a639b5847198413d50b","31b2737a16f44e7ab8268b269182c4c0","5044715b24e449ffa6df6ad7ad53fc60","93da4928074145aab18d1f46ad5f42b1","211e29291f3a4ad492a486d4f72be181","255b58f0dabd46aaab9192559102e80b","65ab6117be40401a898539588d3c74bc","aa3766b99b4c4b5dbaf8b73b2cf21337","0b6a02b6506b4f38ac38a893c54ad922","47c6425636264c56bf7582690dad39e1","8614af9bb457441f8ee5b4193d166798","8d95a75fa276402198a536d81f94e95f","49f8fa5710de4498be8c4633960e63c3","f1b221f15f0b4f12be664836a43c50a3","4663fdcdd9134b66bb679c6f2e0a66c5","e292d22e4c2a42288a75b049b6179266","f0dd7a8b691842ab9dfadd32461bfa58","f7cd396d41e14a0fbcbc5f92684d0157","522063d62fb54c7fb9d4751b639014e1","ca3713042d6d4fb188c3bc1de7e8563d","18d3bcfcd2654d14aae6bd456107a9ef","c1f7955c23074f408d78a65a3294d97b","3ea0dc9ff9044ab39a7911f80acb7e1b","f82e488be9354d2895a1fc0c142ee10f","eddc3f39ed034abbaccb8d81076303d3","362580f0dfaf40ae839b0c60fecde399","a0ad90dc837146019312c6afae7c4d9d","6ca23d2c8fca4da584b4cfa0592dc6f0","6f90fce0d629439f9f597239d8eb9b89","20e63d3b0e394aae80cca1d4818fa778","eed2003e72974af79e7d652de685aaeb","42c9235aa9b347a0b5846273a39a3d03"]},"executionInfo":{"status":"ok","timestamp":1599429543311,"user_tz":240,"elapsed":2338393,"user":{"displayName":"Markus Sommer","photoUrl":"https://lh3.googleusercontent.com/a-/AOh14GjXujbpCzD6oVvcRIT5EP-oi3Bif3Cfcu7npJUnce8=s64","userId":"08083804319459268544"}},"outputId":"2cc89a70-d5a2-4615-c4bb-000ec0c9af4c"},"source":["# @title Find genes\n","\n","def tokenize_aa_seq(aa_seq):\n","    \"\"\" Convert amino acid letters to integers.\"\"\"\n","    table = {\"L\": 1,\n","             \"V\": 2,\n","             \"I\": 3,\n","             \"M\": 4,\n","             \"C\": 5,\n","             \"A\": 6,\n","             \"G\": 7,\n","             \"S\": 8,\n","             \"T\": 9,\n","             \"P\": 10,\n","             \"F\": 11,\n","             \"Y\": 12,\n","             \"W\": 13,\n","             \"E\": 14,\n","             \"D\": 15,\n","             \"N\": 16,\n","             \"Q\": 17,\n","             \"K\": 18,\n","             \"R\": 19,\n","             \"H\": 20,\n","             \"*\": 0,\n","             \"X\": 0}\n","    tokenized = torch.tensor([table[aa] for aa in aa_seq])\n","    return tokenized\n","\n","\n","def get_start_codon(seq, orfcoords, strand):\n","    if strand == 1:\n","        # forward strand\n","        startcoord = orfcoords[0]\n","        return seq[startcoord-3:startcoord]\n","    else:\n","        # reverse strand\n","        startcoord = orfcoords[1]\n","        return seq[startcoord:startcoord+3].reverse_complement()\n","\n","\n","def find_ORFs(nuc_seq, minimum_length):\n","    \"\"\"find positions of all open reading frames in given reading frame\"\"\"\n","    if translation_table == 11:\n","        starts = set([\"ATG\", \"GTG\", \"TTG\"])\n","        stops = set([\"TAA\", \"TAG\", \"TGA\"])\n","    elif translation_table == 4:\n","        starts = set([\"ATG\", \"GTG\", \"TTG\"])\n","        stops = set([\"TAA\", \"TAG\"])\n","    else:\n","        print(\"Translation table \", translation_table, \" not implemented. Please open a GitHub issue if this is a problem.\")\n","        sys.exit()\n","\n","    ORF_startstop = []\n","    temp_starts = []\n","    l = len(nuc_seq)\n","    for i in range(0, l, 3): \n","        if i==0 or nuc_seq[i:i+3] in starts:\n","            temp_starts.append(i)\n","            continue\n","        if ((nuc_seq[i:i+3] in stops) or (i+3==l)) and len(temp_starts) != 0:\n","            for start in temp_starts:\n","                if (i-start >= minimum_length):\n","                    ORF_startstop.append((start, i))\n","            temp_starts = []\n","    return ORF_startstop\n","\n","# def find_ORFs(nuc_seq, minimum_length):\n","#     MAX_STARTS = 20\n","\n","#     \"\"\"find positions of all open reading frames in given reading frame\"\"\"\n","#     if translation_table == 11:\n","#         starts = set([\"ATG\", \"GTG\", \"TTG\"])\n","#         stops = set([\"TAA\", \"TAG\", \"TGA\"])\n","#     elif translation_table == 4:\n","#         starts = set([\"ATG\", \"GTG\", \"TTG\"])\n","#         stops = set([\"TAA\", \"TAG\"])\n","#     else:\n","#         print(\"Translation table \", translation_table, \" not implemented. Please open a GitHub issue if this is a problem.\")\n","#         sys.exit()\n","\n","#     ORF_startstop = []\n","#     temp_starts = []\n","#     l = len(nuc_seq)\n","#     for i in range(0, l, 3):\n","#         if (i==0 or nuc_seq[i:i+3] in starts) and (len(temp_starts) < MAX_STARTS):\n","#             temp_starts.append(i)\n","#             continue\n","#         if ((nuc_seq[i:i+3] in stops) or (i+3==l)) and len(temp_starts) != 0:\n","#             for start in temp_starts:\n","#                 if (i-start >= minimum_length):\n","#                     ORF_startstop.append((start, i))\n","#             temp_starts = []\n","#     return ORF_startstop\n","\n","\n","def get_ORF_info(seq_list):\n","    ORF_seq = []\n","    ORF_coord = []\n","    ORF_nucseq = []\n","    for i, seq in enumerate(seq_list[:]):\n","        # frame 0: starts at 0\n","        # frame 1: starts at 1\n","        # frame 2: starts at 2\n","        # frame r0: ends at 0, MAY NOT START AT THE LAST COORD DUE TO MULTIPLE OF 3 DIFFERENCES\n","        # frame r1: ends at 1\n","        # frame r2: ends at 2\n","\n","        seqstr = str(seq)\n","        seq_c = seq.complement()\n","        seqstr_c = str(seq_c)\n","        l = len(seqstr)\n","        frame_0_end = (l-0)-(l-0)%3+0\n","        frame_1_end = (l-1)-(l-1)%3+1\n","        frame_2_end = (l-2)-(l-2)%3+2\n","\n","        frame_0 = find_ORFs(seqstr[0:frame_0_end], min_orf_length)\n","        frame_1 = find_ORFs(seqstr[1:frame_1_end], min_orf_length)\n","        frame_2 = find_ORFs(seqstr[2:frame_2_end], min_orf_length)\n","\n","        frame_r0 = find_ORFs(seqstr_c[0:frame_0_end][::-1], min_orf_length)\n","        frame_r1 = find_ORFs(seqstr_c[1:frame_1_end][::-1], min_orf_length)\n","        frame_r2 = find_ORFs(seqstr_c[2:frame_2_end][::-1], min_orf_length)\n","\n","        # standardize coords\n","        ORF_0f_standard_nuccoord = [(x[0]+3, x[1]) for x in frame_0]\n","        ORF_1f_standard_nuccoord = [(x[0]+4, x[1]+1) for x in frame_1]\n","        ORF_2f_standard_nuccoord = [(x[0]+5, x[1]+2) for x in frame_2]\n","\n","        ORF_0r_standard_nuccoord = [(frame_0_end-x[1], frame_0_end-x[0]-3) for x in frame_r0]\n","        ORF_1r_standard_nuccoord = [(frame_1_end-x[1], frame_1_end-x[0]-3) for x in frame_r1]\n","        ORF_2r_standard_nuccoord = [(frame_2_end-x[1], frame_2_end-x[0]-3) for x in frame_r2]\n","\n","        # translate once per frame, then slice\n","        aa_0 = str(seq[0:frame_0_end].translate(table=translation_table, to_stop=False))\n","        aa_1 = str(seq[1:frame_1_end].translate(table=translation_table, to_stop=False))\n","        aa_2 = str(seq[2:frame_2_end].translate(table=translation_table, to_stop=False))\n","        aa_r0 = str(seq_c[0:frame_0_end][::-1].translate(table=translation_table, to_stop=False))\n","        aa_r1 = str(seq_c[1:frame_1_end][::-1].translate(table=translation_table, to_stop=False))\n","        aa_r2 = str(seq_c[2:frame_2_end][::-1].translate(table=translation_table, to_stop=False))\n","\n","        ORF_0f_aa = [aa_0[slice(*tuple(int(idx/3) for idx in x))][::-1] for x in frame_0] # reversed because model is trained with first amino acid directly upstream of stop codon\n","        ORF_1f_aa = [aa_1[slice(*tuple(int(idx/3) for idx in x))][::-1] for x in frame_1] \n","        ORF_2f_aa = [aa_2[slice(*tuple(int(idx/3) for idx in x))][::-1] for x in frame_2]\n","        ORF_0r_aa = [aa_r0[slice(*tuple(int(idx/3) for idx in x))][::-1] for x in frame_r0]\n","        ORF_1r_aa = [aa_r1[slice(*tuple(int(idx/3) for idx in x))][::-1] for x in frame_r1]\n","        ORF_2r_aa = [aa_r2[slice(*tuple(int(idx/3) for idx in x))][::-1] for x in frame_r2]\n","\n","        ORF_seq.append([ORF_0f_aa, ORF_1f_aa, ORF_2f_aa, \n","                        ORF_0r_aa, ORF_1r_aa, ORF_2r_aa])\n","        ORF_coord.append([ORF_0f_standard_nuccoord, ORF_1f_standard_nuccoord, ORF_2f_standard_nuccoord, \n","                          ORF_0r_standard_nuccoord, ORF_1r_standard_nuccoord, ORF_2r_standard_nuccoord])\n","        \n","        ORF_nucseq.append([str(seq[0:frame_0_end]), # all 5' to 3'\n","                           str(seq[1:frame_1_end]),\n","                           str(seq[2:frame_2_end]),\n","                           str(seq_c[0:frame_0_end][::-1]),\n","                           str(seq_c[1:frame_1_end][::-1]),\n","                           str(seq_c[2:frame_2_end][::-1])])\n","    return ORF_seq, ORF_nucseq, ORF_coord\n","\n","\n","def analyze_overlap(coords0, coords1, strand0, strand1,\n","                    unidirectional_penalty_per_base,\n","                    convergent_penalty_per_base,\n","                    divergent_penalty_per_base):\n","    overlap = coords0[1] - coords1[0] # TODO account for fully overlapped gene\n","\n","    if overlap <= 0:\n","        compatible, penalty = True, 0\n","        return compatible, penalty\n","    \n","    if overlap > max_gene_overlap:\n","        compatible, penalty = False, 0\n","        return compatible, penalty\n","\n","    # get prime locations\n","    if strand0 == 1:\n","        threeprime0 = coords0[1]\n","        fiveprime0 = coords0[0]\n","    else:\n","        threeprime0 = coords0[0]\n","        fiveprime0 = coords0[1]\n","    if strand1 == 1:\n","        threeprime1 = coords1[1]\n","        fiveprime1 = coords1[0]\n","    else:\n","        threeprime1 = coords1[0]\n","        fiveprime1 = coords1[1]\n","    \n","    # exclude ORFs in same frame sharing same stop codon\n","    if strand0 == strand1 and threeprime0 == threeprime1:\n","        compatible, penalty = False, 0\n","        return compatible, penalty\n","\n","    # unidirectional overlap\n","    if (threeprime0 < fiveprime0) == (threeprime1 < fiveprime1):\n","        compatible, penalty = True, overlap * unidirectional_penalty_per_base\n","        return compatible, penalty\n","\n","    # convergent overlap\n","    if (fiveprime0 < threeprime1 <= threeprime0) or (fiveprime1 < threeprime0 <= threeprime1):\n","        compatible, penalty = True, overlap * convergent_penalty_per_base\n","        return compatible, penalty\n","    \n","    # divergent overlap\n","    if (threeprime0 < fiveprime1 <= fiveprime0) or (threeprime1 < fiveprime0 <= fiveprime1):\n","        compatible, penalty = True, overlap * divergent_penalty_per_base\n","        return compatible, penalty\n","\n","    return True, 0 # edge case of exactly 1 ORF\n","\n","def predict(X):\n","    model.eval()\n","    with torch.no_grad():\n","        if torch.cuda.device_count() > 0:\n","            X_enc = F.one_hot(X, 21).permute(0,2,1).float().cuda()\n","            probs = expit(model(X_enc).cpu())\n","            del X_enc\n","            torch.cuda.empty_cache()\n","        else:\n","            X_enc = F.one_hot(X, 21).permute(0,2,1).float()\n","            probs = expit(model(X_enc).cpu())\n","\n","    return probs\n","\n","def predict_tis(X):\n","    model_tis.eval()\n","    with torch.no_grad():\n","        if torch.cuda.device_count() > 0:\n","            X_enc = F.one_hot(X, 4).permute(0,2,1).float().cuda()\n","        else:\n","            X_enc = F.one_hot(X, 4).permute(0,2,1).float()\n","        probs = expit(model_tis(X_enc).cpu())\n","    return probs\n","\n","nuc_encode = {\"A\":0,\n","              \"T\":1,\n","              \"G\":2,\n","              \"C\":3,\n","              \"N\":0,\n","              \"M\":0,\n","              \"R\":0,\n","              \"Y\":0,\n","              \"W\":0,\n","              \"K\":0}\n","              \n","start_enc = {\"ATG\":0,\n","             \"GTG\":1,\n","             \"TTG\":2}\n","\n","def tensor_to_seq(tensor):\n","    table = {0: \"X\",\n","             1: \"L\",\n","             2: \"V\",\n","             3: \"I\",\n","             4: \"M\",\n","             5: \"C\",\n","             6: \"A\",\n","             7: \"G\",\n","             8: \"S\",\n","             9: \"T\",\n","             10: \"P\",\n","             11: \"F\",\n","             12: \"Y\",\n","             13: \"W\",\n","             14: \"E\",\n","             15: \"D\",\n","             16: \"N\",\n","             17: \"Q\",\n","             18: \"K\",\n","             19: \"R\",\n","             20: \"H\"}\n","    return \"\".join([table[x] for x in tensor])\n","\n","def kmerfilter(seq):\n","    kmerset = kmerize(seq, k_seengene)\n","    s = [x in aa_kmer_set for x in kmerset]\n","    seen = np.sum(s) >= multimer_threshold\n","    return seen\n","\n","def kmerize(seq, k):\n","    kmerset = set()\n","    for i in range(len(seq) - k + 1):\n","        kmer = tuple(seq[i: i + k].tolist())\n","        kmerset.add(kmer)\n","    return kmerset\n","\n","# find genes for each uploaded genome\n","GCF_list = []\n","contig_name_list = []\n","contig_length_list = []\n","contig_seq_list = []\n","contig_gene_coord_list = []\n","contig_gene_strand_list = []\n","\n","for genome_name in genome_dict.keys():\n","    if verbose:\n","        print(\"Reading fasta file\", str(genome_name) + \"...\\n\")\n","\n","    # read genome sequence\n","    seq_list = []\n","    contig_name_sublist = []\n","    contig_length_sublist = []\n","    if os.path.splitext(genome_name)[1].lower() == \".gz\":\n","        with gzip.open(genome_name, \"rt\") as f:\n","            for record in SeqIO.parse(f, \"fasta\"):\n","                seq_list.append(record.seq)\n","                contig_name_sublist.append(record.id)\n","                contig_length_sublist.append(len(record.seq))\n","    else:\n","        with open(genome_name, \"rt\") as f:\n","            for record in SeqIO.parse(f, \"fasta\"):\n","                seq_list.append(record.seq)\n","                contig_name_sublist.append(record.id)\n","                contig_length_sublist.append(len(record.seq))\n","    contig_name_list.append(contig_name_sublist)\n","    contig_length_list.append(contig_length_sublist)\n","    contig_seq_list.append(seq_list)\n","\n","    # get sequences and coordinates of ORFs\n","    if verbose:\n","        print(\"Finding and translating open reading frames...\\n\")\n","\n","    ORF_seq_list, ORF_nucseq_list, ORF_coord_list = get_ORF_info(seq_list)\n","\n","    # combine ORFs to submit to GPU in batches\n","    ORF_seq_combine = []\n","    for i, contig in enumerate(ORF_seq_list):\n","        for j, frame in enumerate(contig):\n","            for k, coord in enumerate(frame):\n","                ORF_seq_combine.append(coord)\n","\n","    # encode amino acids as integers\n","    if verbose:\n","        print(\"Encoding amino acids...\\n\")\n","    ORF_seq_enc = [tokenize_aa_seq(x) for x in ORF_seq_combine]\n","\n","    # seengene check\n","    if protein_kmer_filter:\n","        if verbose:\n","            print(\"Applying protein kmer filter...\\n\")\n","        seengene = []\n","        for s in ORF_seq_enc:\n","            kmerset = kmerize(s, k_seengene)\n","            s = [x in aa_kmer_set for x in kmerset]\n","            seen = np.sum(s) >= multimer_threshold\n","\n","            seengene.append(seen)\n","\n","    # score\n","    if verbose:\n","        print(\"Scoring ORFs with temporal convolutional network...\\n\")\n","\n","    # sort by length to minimize impact of batch padding \n","    ORF_lengths = np.asarray([len(x) for x in ORF_seq_enc])\n","    length_idx = np.argsort(ORF_lengths)\n","    ORF_seq_sorted = [ORF_seq_enc[i] for i in length_idx]\n","\n","    # pad to allow creation of batch matrix\n","    prob_list = []\n","    for i in tqdm(range(0, len(ORF_seq_sorted), gene_batch_size), unit=\" batch\"):\n","        batch = ORF_seq_sorted[i:i+gene_batch_size]\n","        seq_lengths = torch.LongTensor(list(map(len, batch)))\n","        seq_tensor = torch.zeros((len(batch), seq_lengths.max())).long()\n","\n","        for idx, (seq, seqlen) in enumerate(zip(batch, seq_lengths)):\n","            seq_tensor[idx, :seqlen] = torch.LongTensor(seq)\n","\n","        pred_all = predict(seq_tensor)\n","\n","        pred = []\n","        for j, length in enumerate(seq_lengths):\n","            subseq = pred_all[j, 0, 0:int(length)]\n","            predprob = float(expit(torch.mean(logit(subseq))))\n","            pred.append(predprob)\n","        \n","        prob_list.extend(pred)\n","    prob_arr = np.asarray(prob_list, dtype=float)\n","\n","    # unsort\n","    unsort_idx = np.argsort(length_idx)\n","    ORF_prob = prob_arr[unsort_idx]\n","\n","    # recombine ORFs\n","    idx = 0\n","    ORF_gene_score = copy.deepcopy(ORF_coord_list) # fill coord matrix with scores\n","    for i, contig in enumerate(ORF_gene_score):\n","        for j, frame in enumerate(contig):\n","            for k, coord in enumerate(frame):\n","                ORF_gene_score[i][j][k] = float(ORF_prob[idx])\n","                idx += 1\n","\n","    # create strand information\n","    ORF_strand_flat = []\n","    for i, seq in enumerate(ORF_seq_list):\n","        if not any(seq):\n","            ORF_strand_flat.append([])\n","            continue\n","        n_forward = len(seq[0]) + len(seq[1]) + len(seq[2])\n","        n_reverse = len(seq[3]) + len(seq[4]) + len(seq[5])\n","        ORF_allframe_strand = [1]*n_forward + [-1]*n_reverse\n","        ORF_strand_flat.append(ORF_allframe_strand)\n","\n","    # flatten coords within contigs\n","    ORF_coord_flat = [[item for sublist in x for item in sublist] for x in ORF_coord_list]\n","\n","    # get ORF lengths\n","    ORF_length_flat = [[coords[1]-coords[0] for coords in x] for x in ORF_coord_flat]\n","    \n","    if verbose:\n","        print(\"Scoring translation initiation sites...\\n\")\n","\n","    # extract nucleotide sequence surrounding potential start codons\n","    ORF_TIS_seq = copy.deepcopy(ORF_coord_list)\n","    ORF_start_codon = copy.deepcopy(ORF_coord_list)\n","\n","    for i, contig in enumerate(ORF_TIS_seq):\n","        n = 0 # count to index into flat structure # TODO make sure this works as expected\n","\n","        nucseq = ORF_nucseq_list[i][0] # easier to use coords relative to single nucseq\n","        nucseq_c = ORF_nucseq_list[i][3][::-1]\n","        contig_nuclength = len(nucseq)\n","\n","\n","        for j, frame in enumerate(contig):\n","            for k, temp in enumerate(frame):\n","                if any(temp):\n","                    coords = ORF_coord_list[i][j][k]\n","                    strand = ORF_strand_flat[i][n]\n","                    n += 1\n","                    if strand == 1:\n","                        fiveprime = coords[0]\n","                        if fiveprime >= 16 + 3: # NOTE 16 HARD CODED HERE\n","                            downstream = nucseq[fiveprime: fiveprime + 16]\n","                            upstream = nucseq[fiveprime - 16 - 3: fiveprime - 3]\n","                            start_codon = start_enc[nucseq[fiveprime-3: fiveprime]]\n","                            TIS_seq = torch.tensor([nuc_encode[c] for c in (upstream + downstream)[::-1]], dtype=int) # model scores 3' to 5' direction\n","                        else:\n","                            TIS_seq = -1 # deal with gene fragments later\n","                            start_codon = 2\n","\n","                        ORF_TIS_seq[i][j][k] = TIS_seq\n","                        ORF_start_codon[i][j][k] = start_codon\n","                        \n","                    else: # reverse strand\n","                        fiveprime = coords[1]\n","                        if contig_nuclength - fiveprime + 3 >= 16 + 3: # NOTE 16 HARD CODED HERE\n","                            downstream = nucseq_c[fiveprime - 16: fiveprime][::-1]\n","                            upstream = nucseq_c[fiveprime + 3: fiveprime + 3 + 16][::-1]\n","                            start_codon = start_enc[nucseq_c[fiveprime: fiveprime + 3][::-1]]\n","                            TIS_seq = torch.tensor([nuc_encode[c] for c in (upstream + downstream)[::-1]], dtype=int) # model scores 3' to 5' direction\n","                        else:\n","                            TIS_seq = -1 # deal with gene fragments later\n","                            start_codon = 2\n","                            \n","                        ORF_TIS_seq[i][j][k] = TIS_seq\n","                        ORF_start_codon[i][j][k] = start_codon\n","\n","    # flatten TIS for batching\n","    ORF_TIS_prob = copy.deepcopy(ORF_TIS_seq)\n","\n","    ORF_TIS_seq_flat = []\n","    ORF_TIS_seq_idx = []\n","    for i, contig in enumerate(ORF_TIS_seq):\n","        for j, frame in enumerate(contig):\n","            for k, seq in enumerate(frame):\n","                if type(seq) == int: # fragment\n","                    ORF_TIS_prob[i][j][k] = 0.5 # HOW BEST TO DEAL WITH FRAGMENT TIS?\n","                elif len(seq) != 32:\n","                    ORF_TIS_prob[i][j][k] = 0.5 \n","                else:\n","                    ORF_TIS_seq_flat.append(seq)\n","                    ORF_TIS_seq_idx.append((i, j, k))\n","\n","    # batch score TIS\n","    TIS_prob_list = []\n","    for i in tqdm(range(0, len(ORF_TIS_seq_flat), TIS_batch_size), unit=\" batch\"):\n","        batch = ORF_TIS_seq_flat[i:i+TIS_batch_size]\n","        TIS_stacked = torch.stack(batch)\n","        pred = predict_tis(TIS_stacked)\n","\n","        TIS_prob_list.extend(pred)\n","    y_pred_TIS = np.asarray(TIS_prob_list, dtype=float)\n","\n","    # reindex batched scores\n","    for i, prob in enumerate(y_pred_TIS):\n","        idx = ORF_TIS_seq_idx[i]\n","        ORF_TIS_prob[idx[0]][idx[1]][idx[2]] = float(prob)\n","\n","    # combine all info into single score for each ORF\n","    if protein_kmer_filter:\n","        ORF_score_flat = []\n","        for i, contig in enumerate(ORF_gene_score):\n","            if not any(contig):\n","                ORF_score_flat.append([])\n","                continue\n","            temp = []\n","            seengene_idx = 0\n","            for j, frame in enumerate(contig):\n","                for k, geneprob in enumerate(frame):\n","                    length = ORF_coord_list[i][j][k][1] - ORF_coord_list[i][j][k][0] + 1 \n","                    TIS_prob = ORF_TIS_prob[i][j][k]\n","                    start_codon = ORF_start_codon[i][j][k]\n","                    ATG = start_codon == 0\n","                    GTG = start_codon == 1\n","                    TTG = start_codon == 2\n","\n","                    combprob =   geneprob * weight_gene_prob \\\n","                            + TIS_prob * weight_TIS_prob \\\n","                            + ATG * weight_ATG \\\n","                            + GTG * weight_TTG \\\n","                            + TTG * weight_GTG\n","                    maxprob = weight_gene_prob + weight_TIS_prob + max(weight_ATG, weight_TTG, weight_GTG)\n","                    probthresh = score_threshold * maxprob\n","                    score = (combprob - probthresh) * length  + 1e6*seengene[seengene_idx]\n","                    seengene_idx += 1\n","\n","                    temp.append(score)\n","            ORF_score_flat.append(temp)\n","\n","    else:\n","        ORF_score_flat = []\n","        for i, contig in enumerate(ORF_gene_score):\n","            if not any(contig):\n","                ORF_score_flat.append([])\n","                continue\n","            temp = []\n","            for j, frame in enumerate(contig):\n","                for k, geneprob in enumerate(frame):\n","                    length = ORF_coord_list[i][j][k][1] - ORF_coord_list[i][j][k][0] + 1 \n","                    TIS_prob = ORF_TIS_prob[i][j][k]\n","                    start_codon = ORF_start_codon[i][j][k]\n","                    ATG = start_codon == 0\n","                    GTG = start_codon == 1\n","                    TTG = start_codon == 2\n","\n","                    combprob =   geneprob * weight_gene_prob \\\n","                            + TIS_prob * weight_TIS_prob \\\n","                            + ATG * weight_ATG \\\n","                            + GTG * weight_TTG \\\n","                            + TTG * weight_GTG\n","                    maxprob = weight_gene_prob + weight_TIS_prob + max(weight_ATG, weight_TTG, weight_GTG)\n","                    probthresh = score_threshold * maxprob\n","                    score = (combprob - probthresh) * length\n","\n","                    temp.append(score)\n","            ORF_score_flat.append(temp)\n","\n","    # DAGs to maximize geneiness on each contig\n","    contig_gene_coord = []\n","    contig_gene_strand = []\n","\n","    for i, coords in enumerate(ORF_coord_flat):\n","        if verbose:\n","            print(\"Creating graph of contig \" + str(i) + \"...\\n\")\n","\n","        # sort coords, lengths, strands, and scores\n","        startpos = np.array([x[0] for x in coords])\n","        sortidx = list(np.argsort(startpos))\n","\n","        coords_sorted = [coords[j] for j in sortidx]\n","\n","        lengths = ORF_length_flat[i]\n","        lengths_sorted = [lengths[j] for j in sortidx]\n","\n","        scores = ORF_score_flat[i]\n","        scores_sorted = [scores[j] for j in sortidx]\n","\n","        strands = ORF_strand_flat[i]\n","        strands_sorted = [strands[j] for j in sortidx]\n","\n","        # create DAG\n","        # keep track of graph path and score\n","        predecessor = np.zeros(len(scores_sorted), dtype=np.int64)\n","        max_path_score = np.zeros(len(scores_sorted), dtype=np.int64)\n","\n","        # add null starting node\n","        n_connections = 0\n","        idx_offset = 1\n","        while n_connections < max_forward_connections:\n","            k = idx_offset\n","            idx_offset += 1\n","            if k > len(scores_sorted)-1: # dont try to add edge past last ORF\n","                n_connections += 1\n","                continue\n","            edge_weight = scores_sorted[k-1]\n","\n","            # initial scores from null node\n","            max_path_score[k] = edge_weight\n","            predecessor[k] = 0\n","            \n","            n_connections += 1\n","\n","        # add edges between compatible ORFs\n","        for j in tqdm(range(1, len(scores_sorted)-1), unit=\" ORF\"):\n","            n_connections = 0\n","            idx_offset = 1\n","\n","            while n_connections < max_forward_connections:\n","                k = j + idx_offset\n","                idx_offset += 1\n","\n","                if k > len(scores_sorted)-1: # dont try to add edge past end of contigs\n","                    n_connections += 1\n","                    continue \n","\n","                coords0 = coords_sorted[j-1]\n","                coords1 = coords_sorted[k-1]\n","\n","                strand0 = strands_sorted[j-1]\n","                strand1 = strands_sorted[k-1]\n","\n","                compat, penalty = analyze_overlap(coords0, coords1, \n","                                                  strand0, strand1,\n","                                                  unidirectional_penalty_per_base,\n","                                                  convergent_penalty_per_base,\n","                                                  divergent_penalty_per_base)\n","\n","                if compat:\n","                    score = scores_sorted[k-1] - penalty\n","\n","                    path_score = max_path_score[j] + score\n","                    if path_score > max_path_score[k]:\n","                        max_path_score[k] = path_score\n","                        predecessor[k] = j\n","\n","\n","                    n_connections += 1\n","\n","\n","        # solve for geneiest path through contig\n","        if verbose:\n","            print(\"Maximizing geneiness...\")\n","\n","        pred_idx = np.argmax(max_path_score)\n","        pred_path = []\n","        while pred_idx > 0:\n","            pred_path.append(pred_idx)\n","            pred_idx = predecessor[pred_idx]\n","\n","        # max_ORF_PATH = [x-1 for x in max_ORF_PATH_withnull[1:]]\n","        max_ORF_PATH = [x-1 for x in pred_path[:]] # 0 isnt added\n","\n","        gene_predict_coords = [coords_sorted[j] for j in max_ORF_PATH]\n","        gene_predict_strand = [strands_sorted[j] for j in max_ORF_PATH]\n","\n","        # mmseqs filter\n","        if mmseqs2_gene_filter:\n","            if verbose:\n","                print(\"\\nFiltering predictions with mmseqs2...\")\n","\n","            # get amino acid sequence from coherent ORFs\n","            # 3' TO 5'\n","            aa_sorted = [ORF_seq_enc[j] for j in sortidx]\n","            aa_tensor = [aa_sorted[j] for j in max_ORF_PATH]\n","            aa_seq = [tensor_to_seq([int(y) for y in x]) for x in aa_tensor]\n","\n","            # make temp dir to store mmseqs stuff\n","            finding_empty_dir = True\n","            dir_idx = 0\n","            while finding_empty_dir:\n","                dirpath = os.path.join(\"/content/mmseqs2/tmp\", str(dir_idx))\n","                if os.path.isdir(dirpath):\n","                    dir_idx += 1\n","                    continue\n","                else:\n","                    !mkdir {dirpath}\n","                    finding_empty_dir = False\n","            \n","            # mmseqs create query DB     3' to 5'\n","            query_fasta_path_35 = os.path.join(dirpath, \"candidate_genes_35.fasta\")\n","            with open(query_fasta_path_35, \"w\") as f:\n","                for i, s in enumerate(aa_seq):\n","                    f.writelines(\">\" + str(i) + \"\\n\")\n","                    f.writelines(str(s) + \"\\n\")\n","\n","            # mmseqs create query DB     5' to 3'\n","            query_fasta_path_53 = os.path.join(dirpath, \"candidate_genes_53.fasta\")\n","            with open(query_fasta_path_53, \"w\") as f:\n","                for i, s in enumerate(aa_seq):\n","                    f.writelines(\">\" + str(i) + \"\\n\")\n","                    f.writelines(str(s)[::-1] + \"\\n\")\n","\n","            query_DB_path_35 = os.path.join(dirpath, \"candidateDB_35\")\n","            !/content/mmseqs2/mmseqs/bin/mmseqs createdb {query_fasta_path_35} {query_DB_path_35}\n","            \n","            query_DB_path_53 = os.path.join(dirpath, \"candidateDB_53\")\n","            !/content/mmseqs2/mmseqs/bin/mmseqs createdb {query_fasta_path_53} {query_DB_path_53}\n","            \n","            # mmseqs search\n","            results_DB_path_35 = os.path.join(dirpath, \"resultsDB_35\")\n","            !/content/mmseqs2/mmseqs/bin/mmseqs search -s 7.0 {query_DB_path_35} {genexa_DB_path} {results_DB_path_35} {dirpath}\n","\n","            # convert to readable format\n","            m8_path_genexa = os.path.join(dirpath, \"resultDB_genexa.m8\")\n","            !/content/mmseqs2/mmseqs/bin/mmseqs convertalis {query_DB_path_35} {genexa_DB_path} {results_DB_path_35} {m8_path_genexa} --format-output \"query,target,evalue,raw\"\n","\n","            # load search results\n","            mmseqs_results_genexa = pandas.read_table(m8_path_genexa, header=None, names=[\"query\", \"target\", \"evalue\", \"raw\"]).to_numpy()\n","\n","            # get hits\n","            hit_idx_genexa = np.unique(mmseqs_results_genexa[:, 0]).astype(int)\n","\n","            # mmseqs search\n","            results_DB_path_53 = os.path.join(dirpath, \"resultsDB_53\")\n","            !/content/mmseqs2/mmseqs/bin/mmseqs search -s 7.0 {query_DB_path_53} {swissprot_DB_path} {results_DB_path_53} {dirpath}\n","\n","            # convert to readable format\n","            m8_path_secondary = os.path.join(dirpath, \"resultDB_secondary.m8\")\n","            !/content/mmseqs2/mmseqs/bin/mmseqs convertalis {query_DB_path_53} {swissprot_DB_path} {results_DB_path_53} {m8_path_secondary} --format-output \"query,target,evalue,raw\"\n","\n","            # load search results\n","            mmseqs_results_secondary = pandas.read_table(m8_path_secondary, header=None, names=[\"query\", \"target\", \"evalue\", \"raw\"]).to_numpy()\n","\n","            # get hits\n","            hit_idx_secondary = np.unique(mmseqs_results_secondary[:, 0]).astype(int)\n","\n","            # filter gene predictions, keep if mmseqs hit or high enough gene score\n","            cutoff = 200\n","\n","            cutoffpath = [x for i, x in enumerate(max_ORF_PATH) if (scores_sorted[x] > cutoff or (i in hit_idx_genexa or i in hit_idx_secondary))]\n","            gene_predict_coords = [coords_sorted[j] for j in cutoffpath]\n","            gene_predict_strand = [strands_sorted[j] for j in cutoffpath]\n","\n","            graph_score_cutoff = [scores_sorted[j] for j in cutoffpath]\n","            contig_gene_coord.append(gene_predict_coords)\n","            contig_gene_strand.append(gene_predict_strand)\n","\n","            n_genes = len(gene_predict_coords)\n","            if verbose:\n","                print(\"found\", n_genes, \"genes\\n\\n\")\n","\n","\n","        else:\n","            cutoff = 100\n","            cutoffpath = [x for x in max_ORF_PATH if scores_sorted[x] > cutoff]\n","            gene_predict_coords = [coords_sorted[j] for j in cutoffpath]\n","            gene_predict_strand = [strands_sorted[j] for j in cutoffpath]\n","\n","            graph_score_cutoff = [scores_sorted[j] for j in cutoffpath]\n","            contig_gene_coord.append(gene_predict_coords)\n","            contig_gene_strand.append(gene_predict_strand)\n","\n","            n_genes = len(gene_predict_coords)\n","            if verbose:\n","                print(\"found\", n_genes, \"genes\\n\\n\")\n","    contig_gene_coord_list.append(contig_gene_coord)\n","    contig_gene_strand_list.append(contig_gene_strand)\n","\n","print(\"Done\")"],"execution_count":27,"outputs":[{"output_type":"stream","text":["Reading fasta file GCA_000830275.1_ASM83027v1_genomic.fna.gz...\n","\n","Finding and translating open reading frames...\n","\n","Encoding amino acids...\n","\n","Applying protein kmer filter...\n","\n","Scoring ORFs with temporal convolutional network...\n","\n"],"name":"stdout"},{"output_type":"display_data","data":{"application/vnd.jupyter.widget-view+json":{"model_id":"7977b776a3a74915b48aaabcb8ad0991","version_minor":0,"version_major":2},"text/plain":["HBox(children=(FloatProgress(value=0.0, max=315.0), HTML(value='')))"]},"metadata":{"tags":[]}},{"output_type":"stream","text":["\n","Scoring translation initiation sites...\n","\n"],"name":"stdout"},{"output_type":"display_data","data":{"application/vnd.jupyter.widget-view+json":{"model_id":"5d15e0c51e7d406c8b642fa8548c0c5e","version_minor":0,"version_major":2},"text/plain":["HBox(children=(FloatProgress(value=0.0, max=63.0), HTML(value='')))"]},"metadata":{"tags":[]}},{"output_type":"stream","text":["\n","Creating graph of contig 0...\n","\n"],"name":"stdout"},{"output_type":"display_data","data":{"application/vnd.jupyter.widget-view+json":{"model_id":"4a9040116613475b9c7a81fb1c0cd7e9","version_minor":0,"version_major":2},"text/plain":["HBox(children=(FloatProgress(value=0.0, max=62860.0), HTML(value='')))"]},"metadata":{"tags":[]}},{"output_type":"stream","text":["\n","Maximizing geneiness...\n","\n","Filtering predictions with mmseqs2...\n","createdb /content/mmseqs2/tmp/26/candidate_genes_35.fasta /content/mmseqs2/tmp/26/candidateDB_35 \n","\n","MMseqs Version:       \tdf69c26e1c9aaeaa3f5d72fd6e782d02742b2b0c\n","Database type         \t0\n","Shuffle input database\ttrue\n","Createdb mode         \t0\n","Write lookup file     \t1\n","Offset of numeric ids \t0\n","Compressed            \t0\n","Verbosity             \t3\n","\n","Converting sequences\n","[1920] 0s 4ms\n","Time for merging to candidateDB_35_h: 0h 0m 0s 2ms\n","Time for merging to candidateDB_35: 0h 0m 0s 2ms\n","Database type: Aminoacid\n","Time for processing: 0h 0m 0s 14ms\n","createdb /content/mmseqs2/tmp/26/candidate_genes_53.fasta /content/mmseqs2/tmp/26/candidateDB_53 \n","\n","MMseqs Version:       \tdf69c26e1c9aaeaa3f5d72fd6e782d02742b2b0c\n","Database type         \t0\n","Shuffle input database\ttrue\n","Createdb mode         \t0\n","Write lookup file     \t1\n","Offset of numeric ids \t0\n","Compressed            \t0\n","Verbosity             \t3\n","\n","Converting sequences\n","[1920] 0s 5ms\n","Time for merging to candidateDB_53_h: 0h 0m 0s 2ms\n","Time for merging to candidateDB_53: 0h 0m 0s 2ms\n","Database type: Aminoacid\n","Time for processing: 0h 0m 0s 14ms\n","search -s 7.0 /content/mmseqs2/tmp/26/candidateDB_35 /content/mmseqs2/genexa/genexaDB /content/mmseqs2/tmp/26/resultsDB_35 /content/mmseqs2/tmp/26 \n","\n","MMseqs Version:                     \tdf69c26e1c9aaeaa3f5d72fd6e782d02742b2b0c\n","Substitution matrix                 \tnucl:nucleotide.out,aa:blosum62.out\n","Add backtrace                       \tfalse\n","Alignment mode                      \t2\n","Allow wrapped scoring               \tfalse\n","E-value threshold                   \t0.001\n","Seq. id. threshold                  \t0\n","Min alignment length                \t0\n","Seq. id. mode                       \t0\n","Alternative alignments              \t0\n","Coverage threshold                  \t0\n","Coverage mode                       \t0\n","Max sequence length                 \t65535\n","Compositional bias                  \t1\n","Realign hits                        \tfalse\n","Max reject                          \t2147483647\n","Max accept                          \t2147483647\n","Include identical seq. id.          \tfalse\n","Preload mode                        \t0\n","Pseudo count a                      \t1\n","Pseudo count b                      \t1.5\n","Score bias                          \t0\n","Gap open cost                       \tnucl:5,aa:11\n","Gap extension cost                  \tnucl:2,aa:1\n","Zdrop                               \t40\n","Threads                             \t2\n","Compressed                          \t0\n","Verbosity                           \t3\n","Seed substitution matrix            \tnucl:nucleotide.out,aa:VTML80.out\n","Sensitivity                         \t7\n","k-mer length                        \t0\n","k-score                             \t2147483647\n","Alphabet size                       \tnucl:5,aa:21\n","Max results per query               \t300\n","Split database                      \t0\n","Split mode                          \t2\n","Split memory limit                  \t0\n","Diagonal scoring                    \ttrue\n","Exact k-mer matching                \t0\n","Mask residues                       \t1\n","Mask lower case residues            \t0\n","Minimum diagonal score              \t15\n","Spaced k-mers                       \t1\n","Spaced k-mer pattern                \t\n","Local temporary path                \t\n","Rescore mode                        \t0\n","Remove hits by seq. id. and coverage\tfalse\n","Sort results                        \t0\n","Mask profile                        \t1\n","Profile E-value threshold           \t0.1\n","Global sequence weighting           \tfalse\n","Allow deletions                     \tfalse\n","Filter MSA                          \t1\n","Maximum seq. id. threshold          \t0.9\n","Minimum seq. id.                    \t0\n","Minimum score per column            \t-20\n","Minimum coverage                    \t0\n","Select N most diverse seqs          \t1000\n","Omit consensus                      \tfalse\n","Min codons in orf                   \t30\n","Max codons in length                \t32734\n","Max orf gaps                        \t2147483647\n","Contig start mode                   \t2\n","Contig end mode                     \t2\n","Orf start mode                      \t1\n","Forward frames                      \t1,2,3\n","Reverse frames                      \t1,2,3\n","Translation table                   \t1\n","Translate orf                       \t0\n","Use all table starts                \tfalse\n","Offset of numeric ids               \t0\n","Create lookup                       \t0\n","Add orf stop                        \tfalse\n","Chain overlapping alignments        \t0\n","Merge query                         \t1\n","Search type                         \t0\n","Search iterations                   \t1\n","Start sensitivity                   \t4\n","Search steps                        \t1\n","Slice search mode                   \tfalse\n","Strand selection                    \t1\n","Disk space limit                    \t0\n","MPI runner                          \t\n","Force restart with latest tmp       \tfalse\n","Remove temporary files              \tfalse\n","\n","prefilter /content/mmseqs2/tmp/26/candidateDB_35 /content/mmseqs2/genexa/genexaDB.idx /content/mmseqs2/tmp/26/14477264947103173251/pref_0 --sub-mat nucl:nucleotide.out,aa:blosum62.out --seed-sub-mat nucl:nucleotide.out,aa:VTML80.out -k 0 --k-score 2147483647 --alph-size nucl:5,aa:21 --max-seq-len 65535 --max-seqs 300 --split 0 --split-mode 2 --split-memory-limit 0 -c 0 --cov-mode 0 --comp-bias-corr 1 --diag-score 1 --exact-kmer-matching 0 --mask 1 --mask-lower-case 0 --min-ungapped-score 15 --add-self-matches 0 --spaced-kmer-mode 1 --db-load-mode 0 --pca 1 --pcb 1.5 --threads 2 --compressed 0 -v 3 -s 7.0 \n","\n","Index version: 16\n","Generated by:  df69c26e1c9aaeaa3f5d72fd6e782d02742b2b0c\n","ScoreMatrix:  VTML80.out\n","Query database size: 1948 type: Aminoacid\n","Estimated memory consumption: 2G\n","Target database size: 468351 type: Aminoacid\n","Process prefiltering step 1 of 1\n","\n","k-mer similarity threshold: 100\n","Starting prefiltering scores calculation (step 1 of 1)\n","Query db start 1 to 1948\n","Target db start 1 to 468351\n","[=================================================================] 100.00% 1.95K 27s 451ms\n","\n","808.361333 k-mers per position\n","474526 DB matches per sequence\n","42 overflows\n","0 queries produce too many hits (truncated result)\n","274 sequences passed prefiltering per query sequence\n","300 median result list length\n","2 sequences with 0 size result lists\n","Time for merging to pref_0: 0h 0m 0s 0ms\n","Time for processing: 0h 0m 29s 74ms\n","align /content/mmseqs2/tmp/26/candidateDB_35 /content/mmseqs2/genexa/genexaDB.idx /content/mmseqs2/tmp/26/14477264947103173251/pref_0 /content/mmseqs2/tmp/26/resultsDB_35 --sub-mat nucl:nucleotide.out,aa:blosum62.out -a 0 --alignment-mode 2 --wrapped-scoring 0 -e 0.001 --min-seq-id 0 --min-aln-len 0 --seq-id-mode 0 --alt-ali 0 -c 0 --cov-mode 0 --max-seq-len 65535 --comp-bias-corr 1 --realign 0 --max-rejected 2147483647 --max-accept 2147483647 --add-self-matches 0 --db-load-mode 0 --pca 1 --pcb 1.5 --score-bias 0 --gap-open nucl:5,aa:11 --gap-extend nucl:2,aa:1 --zdrop 40 --threads 2 --compressed 0 -v 3 \n","\n","Index version: 16\n","Generated by:  df69c26e1c9aaeaa3f5d72fd6e782d02742b2b0c\n","ScoreMatrix:  VTML80.out\n","Compute score and coverage\n","Query database size: 1948 type: Aminoacid\n","Target database size: 468351 type: Aminoacid\n","Calculation of alignments\n","[=================================================================] 100.00% 1.95K 16s 959ms\n","Time for merging to resultsDB_35: 0h 0m 0s 0ms\n","\n","535679 alignments calculated.\n","77451 sequence pairs passed the thresholds (0.144585 of overall calculated).\n","39.759239 hits per query sequence.\n","Time for processing: 0h 0m 16s 996ms\n","convertalis /content/mmseqs2/tmp/26/candidateDB_35 /content/mmseqs2/genexa/genexaDB /content/mmseqs2/tmp/26/resultsDB_35 /content/mmseqs2/tmp/26/resultDB_genexa.m8 --format-output query,target,evalue,raw \n","\n","MMseqs Version:        \tdf69c26e1c9aaeaa3f5d72fd6e782d02742b2b0c\n","Substitution matrix    \tnucl:nucleotide.out,aa:blosum62.out\n","Alignment format       \t0\n","Format alignment output\tquery,target,evalue,raw\n","Translation table      \t1\n","Gap open cost          \tnucl:5,aa:11\n","Gap extension cost     \tnucl:2,aa:1\n","Database output        \tfalse\n","Preload mode           \t0\n","Search type            \t0\n","Threads                \t2\n","Compressed             \t0\n","Verbosity              \t3\n","\n","[=================================================================] 100.00% 1.95K 0s 123ms\n","Time for merging to resultDB_genexa.m8: 0h 0m 0s 3ms\n","Time for processing: 0h 0m 5s 73ms\n","search -s 7.0 /content/mmseqs2/tmp/26/candidateDB_53 /content/mmseqs2/swissprot/swissprotDB /content/mmseqs2/tmp/26/resultsDB_53 /content/mmseqs2/tmp/26 \n","\n","MMseqs Version:                     \tdf69c26e1c9aaeaa3f5d72fd6e782d02742b2b0c\n","Substitution matrix                 \tnucl:nucleotide.out,aa:blosum62.out\n","Add backtrace                       \tfalse\n","Alignment mode                      \t2\n","Allow wrapped scoring               \tfalse\n","E-value threshold                   \t0.001\n","Seq. id. threshold                  \t0\n","Min alignment length                \t0\n","Seq. id. mode                       \t0\n","Alternative alignments              \t0\n","Coverage threshold                  \t0\n","Coverage mode                       \t0\n","Max sequence length                 \t65535\n","Compositional bias                  \t1\n","Realign hits                        \tfalse\n","Max reject                          \t2147483647\n","Max accept                          \t2147483647\n","Include identical seq. id.          \tfalse\n","Preload mode                        \t0\n","Pseudo count a                      \t1\n","Pseudo count b                      \t1.5\n","Score bias                          \t0\n","Gap open cost                       \tnucl:5,aa:11\n","Gap extension cost                  \tnucl:2,aa:1\n","Zdrop                               \t40\n","Threads                             \t2\n","Compressed                          \t0\n","Verbosity                           \t3\n","Seed substitution matrix            \tnucl:nucleotide.out,aa:VTML80.out\n","Sensitivity                         \t7\n","k-mer length                        \t0\n","k-score                             \t2147483647\n","Alphabet size                       \tnucl:5,aa:21\n","Max results per query               \t300\n","Split database                      \t0\n","Split mode                          \t2\n","Split memory limit                  \t0\n","Diagonal scoring                    \ttrue\n","Exact k-mer matching                \t0\n","Mask residues                       \t1\n","Mask lower case residues            \t0\n","Minimum diagonal score              \t15\n","Spaced k-mers                       \t1\n","Spaced k-mer pattern                \t\n","Local temporary path                \t\n","Rescore mode                        \t0\n","Remove hits by seq. id. and coverage\tfalse\n","Sort results                        \t0\n","Mask profile                        \t1\n","Profile E-value threshold           \t0.1\n","Global sequence weighting           \tfalse\n","Allow deletions                     \tfalse\n","Filter MSA                          \t1\n","Maximum seq. id. threshold          \t0.9\n","Minimum seq. id.                    \t0\n","Minimum score per column            \t-20\n","Minimum coverage                    \t0\n","Select N most diverse seqs          \t1000\n","Omit consensus                      \tfalse\n","Min codons in orf                   \t30\n","Max codons in length                \t32734\n","Max orf gaps                        \t2147483647\n","Contig start mode                   \t2\n","Contig end mode                     \t2\n","Orf start mode                      \t1\n","Forward frames                      \t1,2,3\n","Reverse frames                      \t1,2,3\n","Translation table                   \t1\n","Translate orf                       \t0\n","Use all table starts                \tfalse\n","Offset of numeric ids               \t0\n","Create lookup                       \t0\n","Add orf stop                        \tfalse\n","Chain overlapping alignments        \t0\n","Merge query                         \t1\n","Search type                         \t0\n","Search iterations                   \t1\n","Start sensitivity                   \t4\n","Search steps                        \t1\n","Slice search mode                   \tfalse\n","Strand selection                    \t1\n","Disk space limit                    \t0\n","MPI runner                          \t\n","Force restart with latest tmp       \tfalse\n","Remove temporary files              \tfalse\n","\n","prefilter /content/mmseqs2/tmp/26/candidateDB_53 /content/mmseqs2/swissprot/swissprotDB /content/mmseqs2/tmp/26/10865995062372120037/pref_0 --sub-mat nucl:nucleotide.out,aa:blosum62.out --seed-sub-mat nucl:nucleotide.out,aa:VTML80.out -k 0 --k-score 2147483647 --alph-size nucl:5,aa:21 --max-seq-len 65535 --max-seqs 300 --split 0 --split-mode 2 --split-memory-limit 0 -c 0 --cov-mode 0 --comp-bias-corr 1 --diag-score 1 --exact-kmer-matching 0 --mask 1 --mask-lower-case 0 --min-ungapped-score 15 --add-self-matches 0 --spaced-kmer-mode 1 --db-load-mode 0 --pca 1 --pcb 1.5 --threads 2 --compressed 0 -v 3 -s 7.0 \n","\n","Query database size: 1948 type: Aminoacid\n","Estimated memory consumption: 2G\n","Target database size: 563082 type: Aminoacid\n","Index table k-mer threshold: 100 at k-mer size 6 \n","Index table: counting k-mers\n","[=================================================================] 100.00% 563.08K 40s 907ms\n","Index table: Masked residues: 3167729\n","Index table: fill\n","[=================================================================] 100.00% 563.08K 45s 481ms\n","Index statistics\n","Entries:          192947007\n","DB size:          1592 MB\n","Avg k-mer size:   3.014797\n","Top 10 k-mers\n","    GPGGTL\t1837\n","    GQSWTV\t1706\n","    WGMFAT\t1638\n","    PGVFEV\t1634\n","    VLWQFW\t1623\n","    AYIRPN\t1587\n","    RSPKGV\t1585\n","    TPHKWY\t1560\n","    HIRVGW\t1548\n","    ITLSPY\t1541\n","Time for index table init: 0h 1m 29s 918ms\n","Process prefiltering step 1 of 1\n","\n","k-mer similarity threshold: 100\n","Starting prefiltering scores calculation (step 1 of 1)\n","Query db start 1 to 1948\n","Target db start 1 to 563082\n","[=================================================================] 100.00% 1.95K 31s 864ms\n","\n","809.419210 k-mers per position\n","596454 DB matches per sequence\n","76 overflows\n","0 queries produce too many hits (truncated result)\n","284 sequences passed prefiltering per query sequence\n","300 median result list length\n","1 sequences with 0 size result lists\n","Time for merging to pref_0: 0h 0m 0s 0ms\n","Time for processing: 0h 2m 4s 445ms\n","align /content/mmseqs2/tmp/26/candidateDB_53 /content/mmseqs2/swissprot/swissprotDB /content/mmseqs2/tmp/26/10865995062372120037/pref_0 /content/mmseqs2/tmp/26/resultsDB_53 --sub-mat nucl:nucleotide.out,aa:blosum62.out -a 0 --alignment-mode 2 --wrapped-scoring 0 -e 0.001 --min-seq-id 0 --min-aln-len 0 --seq-id-mode 0 --alt-ali 0 -c 0 --cov-mode 0 --max-seq-len 65535 --comp-bias-corr 1 --realign 0 --max-rejected 2147483647 --max-accept 2147483647 --add-self-matches 0 --db-load-mode 0 --pca 1 --pcb 1.5 --score-bias 0 --gap-open nucl:5,aa:11 --gap-extend nucl:2,aa:1 --zdrop 40 --threads 2 --compressed 0 -v 3 \n","\n","Compute score and coverage\n","Query database size: 1948 type: Aminoacid\n","Target database size: 563082 type: Aminoacid\n","Calculation of alignments\n","[=================================================================] 100.00% 1.95K 19s 182ms\n","Time for merging to resultsDB_53: 0h 0m 0s 0ms\n","\n","553419 alignments calculated.\n","76957 sequence pairs passed the thresholds (0.139057 of overall calculated).\n","39.505646 hits per query sequence.\n","Time for processing: 0h 0m 19s 253ms\n","convertalis /content/mmseqs2/tmp/26/candidateDB_53 /content/mmseqs2/swissprot/swissprotDB /content/mmseqs2/tmp/26/resultsDB_53 /content/mmseqs2/tmp/26/resultDB_secondary.m8 --format-output query,target,evalue,raw \n","\n","MMseqs Version:        \tdf69c26e1c9aaeaa3f5d72fd6e782d02742b2b0c\n","Substitution matrix    \tnucl:nucleotide.out,aa:blosum62.out\n","Alignment format       \t0\n","Format alignment output\tquery,target,evalue,raw\n","Translation table      \t1\n","Gap open cost          \tnucl:5,aa:11\n","Gap extension cost     \tnucl:2,aa:1\n","Database output        \tfalse\n","Preload mode           \t0\n","Search type            \t0\n","Threads                \t2\n","Compressed             \t0\n","Verbosity              \t3\n","\n","[=================================================================] 100.00% 1.95K 0s 111ms\n","Time for merging to resultDB_secondary.m8: 0h 0m 0s 3ms\n","Time for processing: 0h 0m 1s 85ms\n","found 1224 genes\n","\n","\n","Reading fasta file GCF_000015765.1_ASM1576v1_genomic.fna.gz...\n","\n","Finding and translating open reading frames...\n","\n","Encoding amino acids...\n","\n","Applying protein kmer filter...\n","\n","Scoring ORFs with temporal convolutional network...\n","\n"],"name":"stdout"},{"output_type":"display_data","data":{"application/vnd.jupyter.widget-view+json":{"model_id":"aa21151cc1c24f24aa33fb91c9b294ab","version_minor":0,"version_major":2},"text/plain":["HBox(children=(FloatProgress(value=0.0, max=447.0), HTML(value='')))"]},"metadata":{"tags":[]}},{"output_type":"stream","text":["\n","Scoring translation initiation sites...\n","\n"],"name":"stdout"},{"output_type":"display_data","data":{"application/vnd.jupyter.widget-view+json":{"model_id":"9f043bf115c648d48fbf6361c6ed7974","version_minor":0,"version_major":2},"text/plain":["HBox(children=(FloatProgress(value=0.0, max=90.0), HTML(value='')))"]},"metadata":{"tags":[]}},{"output_type":"stream","text":["\n","Creating graph of contig 0...\n","\n"],"name":"stdout"},{"output_type":"display_data","data":{"application/vnd.jupyter.widget-view+json":{"model_id":"ce980cd159934f818f76c85a31b190a7","version_minor":0,"version_major":2},"text/plain":["HBox(children=(FloatProgress(value=0.0, max=89307.0), HTML(value='')))"]},"metadata":{"tags":[]}},{"output_type":"stream","text":["\n","Maximizing geneiness...\n","\n","Filtering predictions with mmseqs2...\n","createdb /content/mmseqs2/tmp/27/candidate_genes_35.fasta /content/mmseqs2/tmp/27/candidateDB_35 \n","\n","MMseqs Version:       \tdf69c26e1c9aaeaa3f5d72fd6e782d02742b2b0c\n","Database type         \t0\n","Shuffle input database\ttrue\n","Createdb mode         \t0\n","Write lookup file     \t1\n","Offset of numeric ids \t0\n","Compressed            \t0\n","Verbosity             \t3\n","\n","Converting sequences\n","[2930] 0s 6ms\n","Time for merging to candidateDB_35_h: 0h 0m 0s 2ms\n","Time for merging to candidateDB_35: 0h 0m 0s 2ms\n","Database type: Aminoacid\n","Time for processing: 0h 0m 0s 17ms\n","createdb /content/mmseqs2/tmp/27/candidate_genes_53.fasta /content/mmseqs2/tmp/27/candidateDB_53 \n","\n","MMseqs Version:       \tdf69c26e1c9aaeaa3f5d72fd6e782d02742b2b0c\n","Database type         \t0\n","Shuffle input database\ttrue\n","Createdb mode         \t0\n","Write lookup file     \t1\n","Offset of numeric ids \t0\n","Compressed            \t0\n","Verbosity             \t3\n","\n","Converting sequences\n","[2930] 0s 6ms\n","Time for merging to candidateDB_53_h: 0h 0m 0s 2ms\n","Time for merging to candidateDB_53: 0h 0m 0s 3ms\n","Database type: Aminoacid\n","Time for processing: 0h 0m 0s 18ms\n","search -s 7.0 /content/mmseqs2/tmp/27/candidateDB_35 /content/mmseqs2/genexa/genexaDB /content/mmseqs2/tmp/27/resultsDB_35 /content/mmseqs2/tmp/27 \n","\n","MMseqs Version:                     \tdf69c26e1c9aaeaa3f5d72fd6e782d02742b2b0c\n","Substitution matrix                 \tnucl:nucleotide.out,aa:blosum62.out\n","Add backtrace                       \tfalse\n","Alignment mode                      \t2\n","Allow wrapped scoring               \tfalse\n","E-value threshold                   \t0.001\n","Seq. id. threshold                  \t0\n","Min alignment length                \t0\n","Seq. id. mode                       \t0\n","Alternative alignments              \t0\n","Coverage threshold                  \t0\n","Coverage mode                       \t0\n","Max sequence length                 \t65535\n","Compositional bias                  \t1\n","Realign hits                        \tfalse\n","Max reject                          \t2147483647\n","Max accept                          \t2147483647\n","Include identical seq. id.          \tfalse\n","Preload mode                        \t0\n","Pseudo count a                      \t1\n","Pseudo count b                      \t1.5\n","Score bias                          \t0\n","Gap open cost                       \tnucl:5,aa:11\n","Gap extension cost                  \tnucl:2,aa:1\n","Zdrop                               \t40\n","Threads                             \t2\n","Compressed                          \t0\n","Verbosity                           \t3\n","Seed substitution matrix            \tnucl:nucleotide.out,aa:VTML80.out\n","Sensitivity                         \t7\n","k-mer length                        \t0\n","k-score                             \t2147483647\n","Alphabet size                       \tnucl:5,aa:21\n","Max results per query               \t300\n","Split database                      \t0\n","Split mode                          \t2\n","Split memory limit                  \t0\n","Diagonal scoring                    \ttrue\n","Exact k-mer matching                \t0\n","Mask residues                       \t1\n","Mask lower case residues            \t0\n","Minimum diagonal score              \t15\n","Spaced k-mers                       \t1\n","Spaced k-mer pattern                \t\n","Local temporary path                \t\n","Rescore mode                        \t0\n","Remove hits by seq. id. and coverage\tfalse\n","Sort results                        \t0\n","Mask profile                        \t1\n","Profile E-value threshold           \t0.1\n","Global sequence weighting           \tfalse\n","Allow deletions                     \tfalse\n","Filter MSA                          \t1\n","Maximum seq. id. threshold          \t0.9\n","Minimum seq. id.                    \t0\n","Minimum score per column            \t-20\n","Minimum coverage                    \t0\n","Select N most diverse seqs          \t1000\n","Omit consensus                      \tfalse\n","Min codons in orf                   \t30\n","Max codons in length                \t32734\n","Max orf gaps                        \t2147483647\n","Contig start mode                   \t2\n","Contig end mode                     \t2\n","Orf start mode                      \t1\n","Forward frames                      \t1,2,3\n","Reverse frames                      \t1,2,3\n","Translation table                   \t1\n","Translate orf                       \t0\n","Use all table starts                \tfalse\n","Offset of numeric ids               \t0\n","Create lookup                       \t0\n","Add orf stop                        \tfalse\n","Chain overlapping alignments        \t0\n","Merge query                         \t1\n","Search type                         \t0\n","Search iterations                   \t1\n","Start sensitivity                   \t4\n","Search steps                        \t1\n","Slice search mode                   \tfalse\n","Strand selection                    \t1\n","Disk space limit                    \t0\n","MPI runner                          \t\n","Force restart with latest tmp       \tfalse\n","Remove temporary files              \tfalse\n","\n","prefilter /content/mmseqs2/tmp/27/candidateDB_35 /content/mmseqs2/genexa/genexaDB.idx /content/mmseqs2/tmp/27/13765853412066370724/pref_0 --sub-mat nucl:nucleotide.out,aa:blosum62.out --seed-sub-mat nucl:nucleotide.out,aa:VTML80.out -k 0 --k-score 2147483647 --alph-size nucl:5,aa:21 --max-seq-len 65535 --max-seqs 300 --split 0 --split-mode 2 --split-memory-limit 0 -c 0 --cov-mode 0 --comp-bias-corr 1 --diag-score 1 --exact-kmer-matching 0 --mask 1 --mask-lower-case 0 --min-ungapped-score 15 --add-self-matches 0 --spaced-kmer-mode 1 --db-load-mode 0 --pca 1 --pcb 1.5 --threads 2 --compressed 0 -v 3 -s 7.0 \n","\n","Index version: 16\n","Generated by:  df69c26e1c9aaeaa3f5d72fd6e782d02742b2b0c\n","ScoreMatrix:  VTML80.out\n","Query database size: 2992 type: Aminoacid\n","Estimated memory consumption: 2G\n","Target database size: 468351 type: Aminoacid\n","Process prefiltering step 1 of 1\n","\n","k-mer similarity threshold: 100\n","Starting prefiltering scores calculation (step 1 of 1)\n","Query db start 1 to 2992\n","Target db start 1 to 468351\n","[=================================================================] 100.00% 2.99K 49s 576ms\n","\n","1018.327690 k-mers per position\n","528864 DB matches per sequence\n","74 overflows\n","0 queries produce too many hits (truncated result)\n","281 sequences passed prefiltering per query sequence\n","300 median result list length\n","0 sequences with 0 size result lists\n","Time for merging to pref_0: 0h 0m 0s 1ms\n","Time for processing: 0h 0m 55s 677ms\n","align /content/mmseqs2/tmp/27/candidateDB_35 /content/mmseqs2/genexa/genexaDB.idx /content/mmseqs2/tmp/27/13765853412066370724/pref_0 /content/mmseqs2/tmp/27/resultsDB_35 --sub-mat nucl:nucleotide.out,aa:blosum62.out -a 0 --alignment-mode 2 --wrapped-scoring 0 -e 0.001 --min-seq-id 0 --min-aln-len 0 --seq-id-mode 0 --alt-ali 0 -c 0 --cov-mode 0 --max-seq-len 65535 --comp-bias-corr 1 --realign 0 --max-rejected 2147483647 --max-accept 2147483647 --add-self-matches 0 --db-load-mode 0 --pca 1 --pcb 1.5 --score-bias 0 --gap-open nucl:5,aa:11 --gap-extend nucl:2,aa:1 --zdrop 40 --threads 2 --compressed 0 -v 3 \n","\n","Index version: 16\n","Generated by:  df69c26e1c9aaeaa3f5d72fd6e782d02742b2b0c\n","ScoreMatrix:  VTML80.out\n","Compute score and coverage\n","Query database size: 2992 type: Aminoacid\n","Target database size: 468351 type: Aminoacid\n","Calculation of alignments\n","[=================================================================] 100.00% 2.99K 29s 112ms\n","Time for merging to resultsDB_35: 0h 0m 0s 1ms\n","\n","840880 alignments calculated.\n","134197 sequence pairs passed the thresholds (0.159591 of overall calculated).\n","44.851940 hits per query sequence.\n","Time for processing: 0h 0m 33s 633ms\n","convertalis /content/mmseqs2/tmp/27/candidateDB_35 /content/mmseqs2/genexa/genexaDB /content/mmseqs2/tmp/27/resultsDB_35 /content/mmseqs2/tmp/27/resultDB_genexa.m8 --format-output query,target,evalue,raw \n","\n","MMseqs Version:        \tdf69c26e1c9aaeaa3f5d72fd6e782d02742b2b0c\n","Substitution matrix    \tnucl:nucleotide.out,aa:blosum62.out\n","Alignment format       \t0\n","Format alignment output\tquery,target,evalue,raw\n","Translation table      \t1\n","Gap open cost          \tnucl:5,aa:11\n","Gap extension cost     \tnucl:2,aa:1\n","Database output        \tfalse\n","Preload mode           \t0\n","Search type            \t0\n","Threads                \t2\n","Compressed             \t0\n","Verbosity              \t3\n","\n","[=================================================================] 100.00% 2.99K 0s 222ms\n","Time for merging to resultDB_genexa.m8: 0h 0m 0s 6ms\n","Time for processing: 0h 0m 0s 311ms\n","search -s 7.0 /content/mmseqs2/tmp/27/candidateDB_53 /content/mmseqs2/swissprot/swissprotDB /content/mmseqs2/tmp/27/resultsDB_53 /content/mmseqs2/tmp/27 \n","\n","MMseqs Version:                     \tdf69c26e1c9aaeaa3f5d72fd6e782d02742b2b0c\n","Substitution matrix                 \tnucl:nucleotide.out,aa:blosum62.out\n","Add backtrace                       \tfalse\n","Alignment mode                      \t2\n","Allow wrapped scoring               \tfalse\n","E-value threshold                   \t0.001\n","Seq. id. threshold                  \t0\n","Min alignment length                \t0\n","Seq. id. mode                       \t0\n","Alternative alignments              \t0\n","Coverage threshold                  \t0\n","Coverage mode                       \t0\n","Max sequence length                 \t65535\n","Compositional bias                  \t1\n","Realign hits                        \tfalse\n","Max reject                          \t2147483647\n","Max accept                          \t2147483647\n","Include identical seq. id.          \tfalse\n","Preload mode                        \t0\n","Pseudo count a                      \t1\n","Pseudo count b                      \t1.5\n","Score bias                          \t0\n","Gap open cost                       \tnucl:5,aa:11\n","Gap extension cost                  \tnucl:2,aa:1\n","Zdrop                               \t40\n","Threads                             \t2\n","Compressed                          \t0\n","Verbosity                           \t3\n","Seed substitution matrix            \tnucl:nucleotide.out,aa:VTML80.out\n","Sensitivity                         \t7\n","k-mer length                        \t0\n","k-score                             \t2147483647\n","Alphabet size                       \tnucl:5,aa:21\n","Max results per query               \t300\n","Split database                      \t0\n","Split mode                          \t2\n","Split memory limit                  \t0\n","Diagonal scoring                    \ttrue\n","Exact k-mer matching                \t0\n","Mask residues                       \t1\n","Mask lower case residues            \t0\n","Minimum diagonal score              \t15\n","Spaced k-mers                       \t1\n","Spaced k-mer pattern                \t\n","Local temporary path                \t\n","Rescore mode                        \t0\n","Remove hits by seq. id. and coverage\tfalse\n","Sort results                        \t0\n","Mask profile                        \t1\n","Profile E-value threshold           \t0.1\n","Global sequence weighting           \tfalse\n","Allow deletions                     \tfalse\n","Filter MSA                          \t1\n","Maximum seq. id. threshold          \t0.9\n","Minimum seq. id.                    \t0\n","Minimum score per column            \t-20\n","Minimum coverage                    \t0\n","Select N most diverse seqs          \t1000\n","Omit consensus                      \tfalse\n","Min codons in orf                   \t30\n","Max codons in length                \t32734\n","Max orf gaps                        \t2147483647\n","Contig start mode                   \t2\n","Contig end mode                     \t2\n","Orf start mode                      \t1\n","Forward frames                      \t1,2,3\n","Reverse frames                      \t1,2,3\n","Translation table                   \t1\n","Translate orf                       \t0\n","Use all table starts                \tfalse\n","Offset of numeric ids               \t0\n","Create lookup                       \t0\n","Add orf stop                        \tfalse\n","Chain overlapping alignments        \t0\n","Merge query                         \t1\n","Search type                         \t0\n","Search iterations                   \t1\n","Start sensitivity                   \t4\n","Search steps                        \t1\n","Slice search mode                   \tfalse\n","Strand selection                    \t1\n","Disk space limit                    \t0\n","MPI runner                          \t\n","Force restart with latest tmp       \tfalse\n","Remove temporary files              \tfalse\n","\n","prefilter /content/mmseqs2/tmp/27/candidateDB_53 /content/mmseqs2/swissprot/swissprotDB /content/mmseqs2/tmp/27/10125815752975973190/pref_0 --sub-mat nucl:nucleotide.out,aa:blosum62.out --seed-sub-mat nucl:nucleotide.out,aa:VTML80.out -k 0 --k-score 2147483647 --alph-size nucl:5,aa:21 --max-seq-len 65535 --max-seqs 300 --split 0 --split-mode 2 --split-memory-limit 0 -c 0 --cov-mode 0 --comp-bias-corr 1 --diag-score 1 --exact-kmer-matching 0 --mask 1 --mask-lower-case 0 --min-ungapped-score 15 --add-self-matches 0 --spaced-kmer-mode 1 --db-load-mode 0 --pca 1 --pcb 1.5 --threads 2 --compressed 0 -v 3 -s 7.0 \n","\n","Query database size: 2992 type: Aminoacid\n","Estimated memory consumption: 2G\n","Target database size: 563082 type: Aminoacid\n","Index table k-mer threshold: 100 at k-mer size 6 \n","Index table: counting k-mers\n","[=================================================================] 100.00% 563.08K 41s 513ms\n","Index table: Masked residues: 3167729\n","Index table: fill\n","[=================================================================] 100.00% 563.08K 46s 260ms\n","Index statistics\n","Entries:          192947007\n","DB size:          1592 MB\n","Avg k-mer size:   3.014797\n","Top 10 k-mers\n","    GPGGTL\t1837\n","    GQSWTV\t1706\n","    WGMFAT\t1638\n","    PGVFEV\t1634\n","    VLWQFW\t1623\n","    AYIRPN\t1587\n","    RSPKGV\t1585\n","    TPHKWY\t1560\n","    HIRVGW\t1548\n","    ITLSPY\t1541\n","Time for index table init: 0h 1m 31s 392ms\n","Process prefiltering step 1 of 1\n","\n","k-mer similarity threshold: 100\n","Starting prefiltering scores calculation (step 1 of 1)\n","Query db start 1 to 2992\n","Target db start 1 to 563082\n","[=================================================================] 100.00% 2.99K 55s 133ms\n","\n","1018.828207 k-mers per position\n","644887 DB matches per sequence\n","145 overflows\n","0 queries produce too many hits (truncated result)\n","288 sequences passed prefiltering per query sequence\n","300 median result list length\n","0 sequences with 0 size result lists\n","Time for merging to pref_0: 0h 0m 0s 1ms\n","Time for processing: 0h 2m 29s 66ms\n","align /content/mmseqs2/tmp/27/candidateDB_53 /content/mmseqs2/swissprot/swissprotDB /content/mmseqs2/tmp/27/10125815752975973190/pref_0 /content/mmseqs2/tmp/27/resultsDB_53 --sub-mat nucl:nucleotide.out,aa:blosum62.out -a 0 --alignment-mode 2 --wrapped-scoring 0 -e 0.001 --min-seq-id 0 --min-aln-len 0 --seq-id-mode 0 --alt-ali 0 -c 0 --cov-mode 0 --max-seq-len 65535 --comp-bias-corr 1 --realign 0 --max-rejected 2147483647 --max-accept 2147483647 --add-self-matches 0 --db-load-mode 0 --pca 1 --pcb 1.5 --score-bias 0 --gap-open nucl:5,aa:11 --gap-extend nucl:2,aa:1 --zdrop 40 --threads 2 --compressed 0 -v 3 \n","\n","Compute score and coverage\n","Query database size: 2992 type: Aminoacid\n","Target database size: 563082 type: Aminoacid\n","Calculation of alignments\n","[=================================================================] 100.00% 2.99K 31s 377ms\n","Time for merging to resultsDB_53: 0h 0m 0s 1ms\n","\n","863507 alignments calculated.\n","127370 sequence pairs passed the thresholds (0.147503 of overall calculated).\n","42.570187 hits per query sequence.\n","Time for processing: 0h 0m 31s 467ms\n","convertalis /content/mmseqs2/tmp/27/candidateDB_53 /content/mmseqs2/swissprot/swissprotDB /content/mmseqs2/tmp/27/resultsDB_53 /content/mmseqs2/tmp/27/resultDB_secondary.m8 --format-output query,target,evalue,raw \n","\n","MMseqs Version:        \tdf69c26e1c9aaeaa3f5d72fd6e782d02742b2b0c\n","Substitution matrix    \tnucl:nucleotide.out,aa:blosum62.out\n","Alignment format       \t0\n","Format alignment output\tquery,target,evalue,raw\n","Translation table      \t1\n","Gap open cost          \tnucl:5,aa:11\n","Gap extension cost     \tnucl:2,aa:1\n","Database output        \tfalse\n","Preload mode           \t0\n","Search type            \t0\n","Threads                \t2\n","Compressed             \t0\n","Verbosity              \t3\n","\n","[=================================================================] 100.00% 2.99K 0s 187ms\n","Time for merging to resultDB_secondary.m8: 0h 0m 0s 4ms\n","Time for processing: 0h 0m 0s 307ms\n","found 1751 genes\n","\n","\n","Reading fasta file GCF_000022205.1_ASM2220v1_genomic.fna.gz...\n","\n","Finding and translating open reading frames...\n","\n","Encoding amino acids...\n","\n","Applying protein kmer filter...\n","\n","Scoring ORFs with temporal convolutional network...\n","\n"],"name":"stdout"},{"output_type":"display_data","data":{"application/vnd.jupyter.widget-view+json":{"model_id":"8a4dc35a4b6e43a4b06d087bf11c0b57","version_minor":0,"version_major":2},"text/plain":["HBox(children=(FloatProgress(value=0.0, max=801.0), HTML(value='')))"]},"metadata":{"tags":[]}},{"output_type":"stream","text":["\n","Scoring translation initiation sites...\n","\n"],"name":"stdout"},{"output_type":"display_data","data":{"application/vnd.jupyter.widget-view+json":{"model_id":"a5b54807a89945ae9767d8677a4a76f7","version_minor":0,"version_major":2},"text/plain":["HBox(children=(FloatProgress(value=0.0, max=161.0), HTML(value='')))"]},"metadata":{"tags":[]}},{"output_type":"stream","text":["\n","Creating graph of contig 0...\n","\n"],"name":"stdout"},{"output_type":"display_data","data":{"application/vnd.jupyter.widget-view+json":{"model_id":"f038606a5e8a42d7b4585ea88c1cdf6b","version_minor":0,"version_major":2},"text/plain":["HBox(children=(FloatProgress(value=0.0, max=113736.0), HTML(value='')))"]},"metadata":{"tags":[]}},{"output_type":"stream","text":["\n","Maximizing geneiness...\n","\n","Filtering predictions with mmseqs2...\n","createdb /content/mmseqs2/tmp/28/candidate_genes_35.fasta /content/mmseqs2/tmp/28/candidateDB_35 \n","\n","MMseqs Version:       \tdf69c26e1c9aaeaa3f5d72fd6e782d02742b2b0c\n","Database type         \t0\n","Shuffle input database\ttrue\n","Createdb mode         \t0\n","Write lookup file     \t1\n","Offset of numeric ids \t0\n","Compressed            \t0\n","Verbosity             \t3\n","\n","Converting sequences\n","[4142] 0s 8ms\n","Time for merging to candidateDB_35_h: 0h 0m 0s 2ms\n","Time for merging to candidateDB_35: 0h 0m 0s 3ms\n","Database type: Aminoacid\n","Time for processing: 0h 0m 0s 22ms\n","createdb /content/mmseqs2/tmp/28/candidate_genes_53.fasta /content/mmseqs2/tmp/28/candidateDB_53 \n","\n","MMseqs Version:       \tdf69c26e1c9aaeaa3f5d72fd6e782d02742b2b0c\n","Database type         \t0\n","Shuffle input database\ttrue\n","Createdb mode         \t0\n","Write lookup file     \t1\n","Offset of numeric ids \t0\n","Compressed            \t0\n","Verbosity             \t3\n","\n","Converting sequences\n","[4142] 0s 7ms\n","Time for merging to candidateDB_53_h: 0h 0m 0s 2ms\n","Time for merging to candidateDB_53: 0h 0m 0s 3ms\n","Database type: Aminoacid\n","Time for processing: 0h 0m 0s 20ms\n","search -s 7.0 /content/mmseqs2/tmp/28/candidateDB_35 /content/mmseqs2/genexa/genexaDB /content/mmseqs2/tmp/28/resultsDB_35 /content/mmseqs2/tmp/28 \n","\n","MMseqs Version:                     \tdf69c26e1c9aaeaa3f5d72fd6e782d02742b2b0c\n","Substitution matrix                 \tnucl:nucleotide.out,aa:blosum62.out\n","Add backtrace                       \tfalse\n","Alignment mode                      \t2\n","Allow wrapped scoring               \tfalse\n","E-value threshold                   \t0.001\n","Seq. id. threshold                  \t0\n","Min alignment length                \t0\n","Seq. id. mode                       \t0\n","Alternative alignments              \t0\n","Coverage threshold                  \t0\n","Coverage mode                       \t0\n","Max sequence length                 \t65535\n","Compositional bias                  \t1\n","Realign hits                        \tfalse\n","Max reject                          \t2147483647\n","Max accept                          \t2147483647\n","Include identical seq. id.          \tfalse\n","Preload mode                        \t0\n","Pseudo count a                      \t1\n","Pseudo count b                      \t1.5\n","Score bias                          \t0\n","Gap open cost                       \tnucl:5,aa:11\n","Gap extension cost                  \tnucl:2,aa:1\n","Zdrop                               \t40\n","Threads                             \t2\n","Compressed                          \t0\n","Verbosity                           \t3\n","Seed substitution matrix            \tnucl:nucleotide.out,aa:VTML80.out\n","Sensitivity                         \t7\n","k-mer length                        \t0\n","k-score                             \t2147483647\n","Alphabet size                       \tnucl:5,aa:21\n","Max results per query               \t300\n","Split database                      \t0\n","Split mode                          \t2\n","Split memory limit                  \t0\n","Diagonal scoring                    \ttrue\n","Exact k-mer matching                \t0\n","Mask residues                       \t1\n","Mask lower case residues            \t0\n","Minimum diagonal score              \t15\n","Spaced k-mers                       \t1\n","Spaced k-mer pattern                \t\n","Local temporary path                \t\n","Rescore mode                        \t0\n","Remove hits by seq. id. and coverage\tfalse\n","Sort results                        \t0\n","Mask profile                        \t1\n","Profile E-value threshold           \t0.1\n","Global sequence weighting           \tfalse\n","Allow deletions                     \tfalse\n","Filter MSA                          \t1\n","Maximum seq. id. threshold          \t0.9\n","Minimum seq. id.                    \t0\n","Minimum score per column            \t-20\n","Minimum coverage                    \t0\n","Select N most diverse seqs          \t1000\n","Omit consensus                      \tfalse\n","Min codons in orf                   \t30\n","Max codons in length                \t32734\n","Max orf gaps                        \t2147483647\n","Contig start mode                   \t2\n","Contig end mode                     \t2\n","Orf start mode                      \t1\n","Forward frames                      \t1,2,3\n","Reverse frames                      \t1,2,3\n","Translation table                   \t1\n","Translate orf                       \t0\n","Use all table starts                \tfalse\n","Offset of numeric ids               \t0\n","Create lookup                       \t0\n","Add orf stop                        \tfalse\n","Chain overlapping alignments        \t0\n","Merge query                         \t1\n","Search type                         \t0\n","Search iterations                   \t1\n","Start sensitivity                   \t4\n","Search steps                        \t1\n","Slice search mode                   \tfalse\n","Strand selection                    \t1\n","Disk space limit                    \t0\n","MPI runner                          \t\n","Force restart with latest tmp       \tfalse\n","Remove temporary files              \tfalse\n","\n","prefilter /content/mmseqs2/tmp/28/candidateDB_35 /content/mmseqs2/genexa/genexaDB.idx /content/mmseqs2/tmp/28/13054441877029568197/pref_0 --sub-mat nucl:nucleotide.out,aa:blosum62.out --seed-sub-mat nucl:nucleotide.out,aa:VTML80.out -k 0 --k-score 2147483647 --alph-size nucl:5,aa:21 --max-seq-len 65535 --max-seqs 300 --split 0 --split-mode 2 --split-memory-limit 0 -c 0 --cov-mode 0 --comp-bias-corr 1 --diag-score 1 --exact-kmer-matching 0 --mask 1 --mask-lower-case 0 --min-ungapped-score 15 --add-self-matches 0 --spaced-kmer-mode 1 --db-load-mode 0 --pca 1 --pcb 1.5 --threads 2 --compressed 0 -v 3 -s 7.0 \n","\n","Index version: 16\n","Generated by:  df69c26e1c9aaeaa3f5d72fd6e782d02742b2b0c\n","ScoreMatrix:  VTML80.out\n","Query database size: 4149 type: Aminoacid\n","Estimated memory consumption: 2G\n","Target database size: 468351 type: Aminoacid\n","Process prefiltering step 1 of 1\n","\n","k-mer similarity threshold: 100\n","Starting prefiltering scores calculation (step 1 of 1)\n","Query db start 1 to 4149\n","Target db start 1 to 468351\n","[=================================================================] 100.00% 4.15K 1m 1s 960ms\n","\n","720.572253 k-mers per position\n","520550 DB matches per sequence\n","90 overflows\n","0 queries produce too many hits (truncated result)\n","273 sequences passed prefiltering per query sequence\n","300 median result list length\n","3 sequences with 0 size result lists\n","Time for merging to pref_0: 0h 0m 0s 1ms\n","Time for processing: 0h 1m 3s 575ms\n","align /content/mmseqs2/tmp/28/candidateDB_35 /content/mmseqs2/genexa/genexaDB.idx /content/mmseqs2/tmp/28/13054441877029568197/pref_0 /content/mmseqs2/tmp/28/resultsDB_35 --sub-mat nucl:nucleotide.out,aa:blosum62.out -a 0 --alignment-mode 2 --wrapped-scoring 0 -e 0.001 --min-seq-id 0 --min-aln-len 0 --seq-id-mode 0 --alt-ali 0 -c 0 --cov-mode 0 --max-seq-len 65535 --comp-bias-corr 1 --realign 0 --max-rejected 2147483647 --max-accept 2147483647 --add-self-matches 0 --db-load-mode 0 --pca 1 --pcb 1.5 --score-bias 0 --gap-open nucl:5,aa:11 --gap-extend nucl:2,aa:1 --zdrop 40 --threads 2 --compressed 0 -v 3 \n","\n","Index version: 16\n","Generated by:  df69c26e1c9aaeaa3f5d72fd6e782d02742b2b0c\n","ScoreMatrix:  VTML80.out\n","Compute score and coverage\n","Query database size: 4149 type: Aminoacid\n","Target database size: 468351 type: Aminoacid\n","Calculation of alignments\n","[=================================================================] 100.00% 4.15K 43s 553ms\n","Time for merging to resultsDB_35: 0h 0m 0s 1ms\n","\n","1132998 alignments calculated.\n","207938 sequence pairs passed the thresholds (0.183529 of overall calculated).\n","50.117619 hits per query sequence.\n","Time for processing: 0h 0m 43s 631ms\n","convertalis /content/mmseqs2/tmp/28/candidateDB_35 /content/mmseqs2/genexa/genexaDB /content/mmseqs2/tmp/28/resultsDB_35 /content/mmseqs2/tmp/28/resultDB_genexa.m8 --format-output query,target,evalue,raw \n","\n","MMseqs Version:        \tdf69c26e1c9aaeaa3f5d72fd6e782d02742b2b0c\n","Substitution matrix    \tnucl:nucleotide.out,aa:blosum62.out\n","Alignment format       \t0\n","Format alignment output\tquery,target,evalue,raw\n","Translation table      \t1\n","Gap open cost          \tnucl:5,aa:11\n","Gap extension cost     \tnucl:2,aa:1\n","Database output        \tfalse\n","Preload mode           \t0\n","Search type            \t0\n","Threads                \t2\n","Compressed             \t0\n","Verbosity              \t3\n","\n","[=================================================================] 100.00% 4.15K 0s 358ms\n","Time for merging to resultDB_genexa.m8: 0h 0m 0s 6ms\n","Time for processing: 0h 0m 0s 451ms\n","search -s 7.0 /content/mmseqs2/tmp/28/candidateDB_53 /content/mmseqs2/swissprot/swissprotDB /content/mmseqs2/tmp/28/resultsDB_53 /content/mmseqs2/tmp/28 \n","\n","MMseqs Version:                     \tdf69c26e1c9aaeaa3f5d72fd6e782d02742b2b0c\n","Substitution matrix                 \tnucl:nucleotide.out,aa:blosum62.out\n","Add backtrace                       \tfalse\n","Alignment mode                      \t2\n","Allow wrapped scoring               \tfalse\n","E-value threshold                   \t0.001\n","Seq. id. threshold                  \t0\n","Min alignment length                \t0\n","Seq. id. mode                       \t0\n","Alternative alignments              \t0\n","Coverage threshold                  \t0\n","Coverage mode                       \t0\n","Max sequence length                 \t65535\n","Compositional bias                  \t1\n","Realign hits                        \tfalse\n","Max reject                          \t2147483647\n","Max accept                          \t2147483647\n","Include identical seq. id.          \tfalse\n","Preload mode                        \t0\n","Pseudo count a                      \t1\n","Pseudo count b                      \t1.5\n","Score bias                          \t0\n","Gap open cost                       \tnucl:5,aa:11\n","Gap extension cost                  \tnucl:2,aa:1\n","Zdrop                               \t40\n","Threads                             \t2\n","Compressed                          \t0\n","Verbosity                           \t3\n","Seed substitution matrix            \tnucl:nucleotide.out,aa:VTML80.out\n","Sensitivity                         \t7\n","k-mer length                        \t0\n","k-score                             \t2147483647\n","Alphabet size                       \tnucl:5,aa:21\n","Max results per query               \t300\n","Split database                      \t0\n","Split mode                          \t2\n","Split memory limit                  \t0\n","Diagonal scoring                    \ttrue\n","Exact k-mer matching                \t0\n","Mask residues                       \t1\n","Mask lower case residues            \t0\n","Minimum diagonal score              \t15\n","Spaced k-mers                       \t1\n","Spaced k-mer pattern                \t\n","Local temporary path                \t\n","Rescore mode                        \t0\n","Remove hits by seq. id. and coverage\tfalse\n","Sort results                        \t0\n","Mask profile                        \t1\n","Profile E-value threshold           \t0.1\n","Global sequence weighting           \tfalse\n","Allow deletions                     \tfalse\n","Filter MSA                          \t1\n","Maximum seq. id. threshold          \t0.9\n","Minimum seq. id.                    \t0\n","Minimum score per column            \t-20\n","Minimum coverage                    \t0\n","Select N most diverse seqs          \t1000\n","Omit consensus                      \tfalse\n","Min codons in orf                   \t30\n","Max codons in length                \t32734\n","Max orf gaps                        \t2147483647\n","Contig start mode                   \t2\n","Contig end mode                     \t2\n","Orf start mode                      \t1\n","Forward frames                      \t1,2,3\n","Reverse frames                      \t1,2,3\n","Translation table                   \t1\n","Translate orf                       \t0\n","Use all table starts                \tfalse\n","Offset of numeric ids               \t0\n","Create lookup                       \t0\n","Add orf stop                        \tfalse\n","Chain overlapping alignments        \t0\n","Merge query                         \t1\n","Search type                         \t0\n","Search iterations                   \t1\n","Start sensitivity                   \t4\n","Search steps                        \t1\n","Slice search mode                   \tfalse\n","Strand selection                    \t1\n","Disk space limit                    \t0\n","MPI runner                          \t\n","Force restart with latest tmp       \tfalse\n","Remove temporary files              \tfalse\n","\n","prefilter /content/mmseqs2/tmp/28/candidateDB_53 /content/mmseqs2/swissprot/swissprotDB /content/mmseqs2/tmp/28/9385636443579826343/pref_0 --sub-mat nucl:nucleotide.out,aa:blosum62.out --seed-sub-mat nucl:nucleotide.out,aa:VTML80.out -k 0 --k-score 2147483647 --alph-size nucl:5,aa:21 --max-seq-len 65535 --max-seqs 300 --split 0 --split-mode 2 --split-memory-limit 0 -c 0 --cov-mode 0 --comp-bias-corr 1 --diag-score 1 --exact-kmer-matching 0 --mask 1 --mask-lower-case 0 --min-ungapped-score 15 --add-self-matches 0 --spaced-kmer-mode 1 --db-load-mode 0 --pca 1 --pcb 1.5 --threads 2 --compressed 0 -v 3 -s 7.0 \n","\n","Query database size: 4149 type: Aminoacid\n","Estimated memory consumption: 2G\n","Target database size: 563082 type: Aminoacid\n","Index table k-mer threshold: 100 at k-mer size 6 \n","Index table: counting k-mers\n","[=================================================================] 100.00% 563.08K 41s 864ms\n","Index table: Masked residues: 3167729\n","Index table: fill\n","[=================================================================] 100.00% 563.08K 46s 402ms\n","Index statistics\n","Entries:          192947007\n","DB size:          1592 MB\n","Avg k-mer size:   3.014797\n","Top 10 k-mers\n","    GPGGTL\t1837\n","    GQSWTV\t1706\n","    WGMFAT\t1638\n","    PGVFEV\t1634\n","    VLWQFW\t1623\n","    AYIRPN\t1587\n","    RSPKGV\t1585\n","    TPHKWY\t1560\n","    HIRVGW\t1548\n","    ITLSPY\t1541\n","Time for index table init: 0h 1m 31s 887ms\n","Process prefiltering step 1 of 1\n","\n","k-mer similarity threshold: 100\n","Starting prefiltering scores calculation (step 1 of 1)\n","Query db start 1 to 4149\n","Target db start 1 to 563082\n","[=================================================================] 100.00% 4.15K 1m 5s 995ms\n","\n","720.567177 k-mers per position\n","589613 DB matches per sequence\n","144 overflows\n","0 queries produce too many hits (truncated result)\n","278 sequences passed prefiltering per query sequence\n","300 median result list length\n","1 sequences with 0 size result lists\n","Time for merging to pref_0: 0h 0m 0s 1ms\n","Time for processing: 0h 2m 40s 314ms\n","align /content/mmseqs2/tmp/28/candidateDB_53 /content/mmseqs2/swissprot/swissprotDB /content/mmseqs2/tmp/28/9385636443579826343/pref_0 /content/mmseqs2/tmp/28/resultsDB_53 --sub-mat nucl:nucleotide.out,aa:blosum62.out -a 0 --alignment-mode 2 --wrapped-scoring 0 -e 0.001 --min-seq-id 0 --min-aln-len 0 --seq-id-mode 0 --alt-ali 0 -c 0 --cov-mode 0 --max-seq-len 65535 --comp-bias-corr 1 --realign 0 --max-rejected 2147483647 --max-accept 2147483647 --add-self-matches 0 --db-load-mode 0 --pca 1 --pcb 1.5 --score-bias 0 --gap-open nucl:5,aa:11 --gap-extend nucl:2,aa:1 --zdrop 40 --threads 2 --compressed 0 -v 3 \n","\n","Compute score and coverage\n","Query database size: 4149 type: Aminoacid\n","Target database size: 563082 type: Aminoacid\n","Calculation of alignments\n","[=================================================================] 100.00% 4.15K 43s 439ms\n","Time for merging to resultsDB_53: 0h 0m 0s 1ms\n","\n","1155020 alignments calculated.\n","168268 sequence pairs passed the thresholds (0.145684 of overall calculated).\n","40.556278 hits per query sequence.\n","Time for processing: 0h 0m 43s 565ms\n","convertalis /content/mmseqs2/tmp/28/candidateDB_53 /content/mmseqs2/swissprot/swissprotDB /content/mmseqs2/tmp/28/resultsDB_53 /content/mmseqs2/tmp/28/resultDB_secondary.m8 --format-output query,target,evalue,raw \n","\n","MMseqs Version:        \tdf69c26e1c9aaeaa3f5d72fd6e782d02742b2b0c\n","Substitution matrix    \tnucl:nucleotide.out,aa:blosum62.out\n","Alignment format       \t0\n","Format alignment output\tquery,target,evalue,raw\n","Translation table      \t1\n","Gap open cost          \tnucl:5,aa:11\n","Gap extension cost     \tnucl:2,aa:1\n","Database output        \tfalse\n","Preload mode           \t0\n","Search type            \t0\n","Threads                \t2\n","Compressed             \t0\n","Verbosity              \t3\n","\n","[=================================================================] 100.00% 4.15K 0s 263ms\n","Time for merging to resultDB_secondary.m8: 0h 0m 0s 6ms\n","Time for processing: 0h 0m 0s 386ms\n","found 2614 genes\n","\n","\n","Creating graph of contig 1...\n","\n"],"name":"stdout"},{"output_type":"display_data","data":{"application/vnd.jupyter.widget-view+json":{"model_id":"58b858f22c204471a4cf71d7c2458d4d","version_minor":0,"version_major":2},"text/plain":["HBox(children=(FloatProgress(value=0.0, max=25767.0), HTML(value='')))"]},"metadata":{"tags":[]}},{"output_type":"stream","text":["\n","Maximizing geneiness...\n","\n","Filtering predictions with mmseqs2...\n","createdb /content/mmseqs2/tmp/29/candidate_genes_35.fasta /content/mmseqs2/tmp/29/candidateDB_35 \n","\n","MMseqs Version:       \tdf69c26e1c9aaeaa3f5d72fd6e782d02742b2b0c\n","Database type         \t0\n","Shuffle input database\ttrue\n","Createdb mode         \t0\n","Write lookup file     \t1\n","Offset of numeric ids \t0\n","Compressed            \t0\n","Verbosity             \t3\n","\n","Converting sequences\n","[1718] 0s 4ms\n","Time for merging to candidateDB_35_h: 0h 0m 0s 2ms\n","Time for merging to candidateDB_35: 0h 0m 0s 2ms\n","Database type: Aminoacid\n","Time for processing: 0h 0m 0s 13ms\n","createdb /content/mmseqs2/tmp/29/candidate_genes_53.fasta /content/mmseqs2/tmp/29/candidateDB_53 \n","\n","MMseqs Version:       \tdf69c26e1c9aaeaa3f5d72fd6e782d02742b2b0c\n","Database type         \t0\n","Shuffle input database\ttrue\n","Createdb mode         \t0\n","Write lookup file     \t1\n","Offset of numeric ids \t0\n","Compressed            \t0\n","Verbosity             \t3\n","\n","Converting sequences\n","[1718] 0s 4ms\n","Time for merging to candidateDB_53_h: 0h 0m 0s 2ms\n","Time for merging to candidateDB_53: 0h 0m 0s 2ms\n","Database type: Aminoacid\n","Time for processing: 0h 0m 0s 13ms\n","search -s 7.0 /content/mmseqs2/tmp/29/candidateDB_35 /content/mmseqs2/genexa/genexaDB /content/mmseqs2/tmp/29/resultsDB_35 /content/mmseqs2/tmp/29 \n","\n","MMseqs Version:                     \tdf69c26e1c9aaeaa3f5d72fd6e782d02742b2b0c\n","Substitution matrix                 \tnucl:nucleotide.out,aa:blosum62.out\n","Add backtrace                       \tfalse\n","Alignment mode                      \t2\n","Allow wrapped scoring               \tfalse\n","E-value threshold                   \t0.001\n","Seq. id. threshold                  \t0\n","Min alignment length                \t0\n","Seq. id. mode                       \t0\n","Alternative alignments              \t0\n","Coverage threshold                  \t0\n","Coverage mode                       \t0\n","Max sequence length                 \t65535\n","Compositional bias                  \t1\n","Realign hits                        \tfalse\n","Max reject                          \t2147483647\n","Max accept                          \t2147483647\n","Include identical seq. id.          \tfalse\n","Preload mode                        \t0\n","Pseudo count a                      \t1\n","Pseudo count b                      \t1.5\n","Score bias                          \t0\n","Gap open cost                       \tnucl:5,aa:11\n","Gap extension cost                  \tnucl:2,aa:1\n","Zdrop                               \t40\n","Threads                             \t2\n","Compressed                          \t0\n","Verbosity                           \t3\n","Seed substitution matrix            \tnucl:nucleotide.out,aa:VTML80.out\n","Sensitivity                         \t7\n","k-mer length                        \t0\n","k-score                             \t2147483647\n","Alphabet size                       \tnucl:5,aa:21\n","Max results per query               \t300\n","Split database                      \t0\n","Split mode                          \t2\n","Split memory limit                  \t0\n","Diagonal scoring                    \ttrue\n","Exact k-mer matching                \t0\n","Mask residues                       \t1\n","Mask lower case residues            \t0\n","Minimum diagonal score              \t15\n","Spaced k-mers                       \t1\n","Spaced k-mer pattern                \t\n","Local temporary path                \t\n","Rescore mode                        \t0\n","Remove hits by seq. id. and coverage\tfalse\n","Sort results                        \t0\n","Mask profile                        \t1\n","Profile E-value threshold           \t0.1\n","Global sequence weighting           \tfalse\n","Allow deletions                     \tfalse\n","Filter MSA                          \t1\n","Maximum seq. id. threshold          \t0.9\n","Minimum seq. id.                    \t0\n","Minimum score per column            \t-20\n","Minimum coverage                    \t0\n","Select N most diverse seqs          \t1000\n","Omit consensus                      \tfalse\n","Min codons in orf                   \t30\n","Max codons in length                \t32734\n","Max orf gaps                        \t2147483647\n","Contig start mode                   \t2\n","Contig end mode                     \t2\n","Orf start mode                      \t1\n","Forward frames                      \t1,2,3\n","Reverse frames                      \t1,2,3\n","Translation table                   \t1\n","Translate orf                       \t0\n","Use all table starts                \tfalse\n","Offset of numeric ids               \t0\n","Create lookup                       \t0\n","Add orf stop                        \tfalse\n","Chain overlapping alignments        \t0\n","Merge query                         \t1\n","Search type                         \t0\n","Search iterations                   \t1\n","Start sensitivity                   \t4\n","Search steps                        \t1\n","Slice search mode                   \tfalse\n","Strand selection                    \t1\n","Disk space limit                    \t0\n","MPI runner                          \t\n","Force restart with latest tmp       \tfalse\n","Remove temporary files              \tfalse\n","\n","prefilter /content/mmseqs2/tmp/29/candidateDB_35 /content/mmseqs2/genexa/genexaDB.idx /content/mmseqs2/tmp/29/12343030341992765670/pref_0 --sub-mat nucl:nucleotide.out,aa:blosum62.out --seed-sub-mat nucl:nucleotide.out,aa:VTML80.out -k 0 --k-score 2147483647 --alph-size nucl:5,aa:21 --max-seq-len 65535 --max-seqs 300 --split 0 --split-mode 2 --split-memory-limit 0 -c 0 --cov-mode 0 --comp-bias-corr 1 --diag-score 1 --exact-kmer-matching 0 --mask 1 --mask-lower-case 0 --min-ungapped-score 15 --add-self-matches 0 --spaced-kmer-mode 1 --db-load-mode 0 --pca 1 --pcb 1.5 --threads 2 --compressed 0 -v 3 -s 7.0 \n","\n","Index version: 16\n","Generated by:  df69c26e1c9aaeaa3f5d72fd6e782d02742b2b0c\n","ScoreMatrix:  VTML80.out\n","Query database size: 1727 type: Aminoacid\n","Estimated memory consumption: 2G\n","Target database size: 468351 type: Aminoacid\n","Process prefiltering step 1 of 1\n","\n","k-mer similarity threshold: 100\n","Starting prefiltering scores calculation (step 1 of 1)\n","Query db start 1 to 1727\n","Target db start 1 to 468351\n","[=================================================================] 100.00% 1.73K 21s 186ms\n","\n","565.526391 k-mers per position\n","423658 DB matches per sequence\n","39 overflows\n","0 queries produce too many hits (truncated result)\n","250 sequences passed prefiltering per query sequence\n","300 median result list length\n","5 sequences with 0 size result lists\n","Time for merging to pref_0: 0h 0m 0s 0ms\n","Time for processing: 0h 0m 22s 835ms\n","align /content/mmseqs2/tmp/29/candidateDB_35 /content/mmseqs2/genexa/genexaDB.idx /content/mmseqs2/tmp/29/12343030341992765670/pref_0 /content/mmseqs2/tmp/29/resultsDB_35 --sub-mat nucl:nucleotide.out,aa:blosum62.out -a 0 --alignment-mode 2 --wrapped-scoring 0 -e 0.001 --min-seq-id 0 --min-aln-len 0 --seq-id-mode 0 --alt-ali 0 -c 0 --cov-mode 0 --max-seq-len 65535 --comp-bias-corr 1 --realign 0 --max-rejected 2147483647 --max-accept 2147483647 --add-self-matches 0 --db-load-mode 0 --pca 1 --pcb 1.5 --score-bias 0 --gap-open nucl:5,aa:11 --gap-extend nucl:2,aa:1 --zdrop 40 --threads 2 --compressed 0 -v 3 \n","\n","Index version: 16\n","Generated by:  df69c26e1c9aaeaa3f5d72fd6e782d02742b2b0c\n","ScoreMatrix:  VTML80.out\n","Compute score and coverage\n","Query database size: 1727 type: Aminoacid\n","Target database size: 468351 type: Aminoacid\n","Calculation of alignments\n","[=================================================================] 100.00% 1.73K 20s 132ms\n","Time for merging to resultsDB_35: 0h 0m 0s 0ms\n","\n","432231 alignments calculated.\n","108919 sequence pairs passed the thresholds (0.251993 of overall calculated).\n","63.068325 hits per query sequence.\n","Time for processing: 0h 0m 20s 162ms\n","convertalis /content/mmseqs2/tmp/29/candidateDB_35 /content/mmseqs2/genexa/genexaDB /content/mmseqs2/tmp/29/resultsDB_35 /content/mmseqs2/tmp/29/resultDB_genexa.m8 --format-output query,target,evalue,raw \n","\n","MMseqs Version:        \tdf69c26e1c9aaeaa3f5d72fd6e782d02742b2b0c\n","Substitution matrix    \tnucl:nucleotide.out,aa:blosum62.out\n","Alignment format       \t0\n","Format alignment output\tquery,target,evalue,raw\n","Translation table      \t1\n","Gap open cost          \tnucl:5,aa:11\n","Gap extension cost     \tnucl:2,aa:1\n","Database output        \tfalse\n","Preload mode           \t0\n","Search type            \t0\n","Threads                \t2\n","Compressed             \t0\n","Verbosity              \t3\n","\n","[=================================================================] 100.00% 1.73K 0s 184ms\n","Time for merging to resultDB_genexa.m8: 0h 0m 0s 4ms\n","Time for processing: 0h 0m 0s 272ms\n","search -s 7.0 /content/mmseqs2/tmp/29/candidateDB_53 /content/mmseqs2/swissprot/swissprotDB /content/mmseqs2/tmp/29/resultsDB_53 /content/mmseqs2/tmp/29 \n","\n","MMseqs Version:                     \tdf69c26e1c9aaeaa3f5d72fd6e782d02742b2b0c\n","Substitution matrix                 \tnucl:nucleotide.out,aa:blosum62.out\n","Add backtrace                       \tfalse\n","Alignment mode                      \t2\n","Allow wrapped scoring               \tfalse\n","E-value threshold                   \t0.001\n","Seq. id. threshold                  \t0\n","Min alignment length                \t0\n","Seq. id. mode                       \t0\n","Alternative alignments              \t0\n","Coverage threshold                  \t0\n","Coverage mode                       \t0\n","Max sequence length                 \t65535\n","Compositional bias                  \t1\n","Realign hits                        \tfalse\n","Max reject                          \t2147483647\n","Max accept                          \t2147483647\n","Include identical seq. id.          \tfalse\n","Preload mode                        \t0\n","Pseudo count a                      \t1\n","Pseudo count b                      \t1.5\n","Score bias                          \t0\n","Gap open cost                       \tnucl:5,aa:11\n","Gap extension cost                  \tnucl:2,aa:1\n","Zdrop                               \t40\n","Threads                             \t2\n","Compressed                          \t0\n","Verbosity                           \t3\n","Seed substitution matrix            \tnucl:nucleotide.out,aa:VTML80.out\n","Sensitivity                         \t7\n","k-mer length                        \t0\n","k-score                             \t2147483647\n","Alphabet size                       \tnucl:5,aa:21\n","Max results per query               \t300\n","Split database                      \t0\n","Split mode                          \t2\n","Split memory limit                  \t0\n","Diagonal scoring                    \ttrue\n","Exact k-mer matching                \t0\n","Mask residues                       \t1\n","Mask lower case residues            \t0\n","Minimum diagonal score              \t15\n","Spaced k-mers                       \t1\n","Spaced k-mer pattern                \t\n","Local temporary path                \t\n","Rescore mode                        \t0\n","Remove hits by seq. id. and coverage\tfalse\n","Sort results                        \t0\n","Mask profile                        \t1\n","Profile E-value threshold           \t0.1\n","Global sequence weighting           \tfalse\n","Allow deletions                     \tfalse\n","Filter MSA                          \t1\n","Maximum seq. id. threshold          \t0.9\n","Minimum seq. id.                    \t0\n","Minimum score per column            \t-20\n","Minimum coverage                    \t0\n","Select N most diverse seqs          \t1000\n","Omit consensus                      \tfalse\n","Min codons in orf                   \t30\n","Max codons in length                \t32734\n","Max orf gaps                        \t2147483647\n","Contig start mode                   \t2\n","Contig end mode                     \t2\n","Orf start mode                      \t1\n","Forward frames                      \t1,2,3\n","Reverse frames                      \t1,2,3\n","Translation table                   \t1\n","Translate orf                       \t0\n","Use all table starts                \tfalse\n","Offset of numeric ids               \t0\n","Create lookup                       \t0\n","Add orf stop                        \tfalse\n","Chain overlapping alignments        \t0\n","Merge query                         \t1\n","Search type                         \t0\n","Search iterations                   \t1\n","Start sensitivity                   \t4\n","Search steps                        \t1\n","Slice search mode                   \tfalse\n","Strand selection                    \t1\n","Disk space limit                    \t0\n","MPI runner                          \t\n","Force restart with latest tmp       \tfalse\n","Remove temporary files              \tfalse\n","\n","prefilter /content/mmseqs2/tmp/29/candidateDB_53 /content/mmseqs2/swissprot/swissprotDB /content/mmseqs2/tmp/29/8645457134183679496/pref_0 --sub-mat nucl:nucleotide.out,aa:blosum62.out --seed-sub-mat nucl:nucleotide.out,aa:VTML80.out -k 0 --k-score 2147483647 --alph-size nucl:5,aa:21 --max-seq-len 65535 --max-seqs 300 --split 0 --split-mode 2 --split-memory-limit 0 -c 0 --cov-mode 0 --comp-bias-corr 1 --diag-score 1 --exact-kmer-matching 0 --mask 1 --mask-lower-case 0 --min-ungapped-score 15 --add-self-matches 0 --spaced-kmer-mode 1 --db-load-mode 0 --pca 1 --pcb 1.5 --threads 2 --compressed 0 -v 3 -s 7.0 \n","\n","Query database size: 1727 type: Aminoacid\n","Estimated memory consumption: 2G\n","Target database size: 563082 type: Aminoacid\n","Index table k-mer threshold: 100 at k-mer size 6 \n","Index table: counting k-mers\n","[=================================================================] 100.00% 563.08K 42s 351ms\n","Index table: Masked residues: 3167729\n","Index table: fill\n","[=================================================================] 100.00% 563.08K 47s 8ms\n","Index statistics\n","Entries:          192947007\n","DB size:          1592 MB\n","Avg k-mer size:   3.014797\n","Top 10 k-mers\n","    GPGGTL\t1837\n","    GQSWTV\t1706\n","    WGMFAT\t1638\n","    PGVFEV\t1634\n","    VLWQFW\t1623\n","    AYIRPN\t1587\n","    RSPKGV\t1585\n","    TPHKWY\t1560\n","    HIRVGW\t1548\n","    ITLSPY\t1541\n","Time for index table init: 0h 1m 32s 999ms\n","Process prefiltering step 1 of 1\n","\n","k-mer similarity threshold: 100\n","Starting prefiltering scores calculation (step 1 of 1)\n","Query db start 1 to 1727\n","Target db start 1 to 563082\n","[=================================================================] 100.00% 1.73K 22s 483ms\n","\n","567.477526 k-mers per position\n","479514 DB matches per sequence\n","56 overflows\n","0 queries produce too many hits (truncated result)\n","256 sequences passed prefiltering per query sequence\n","300 median result list length\n","7 sequences with 0 size result lists\n","Time for merging to pref_0: 0h 0m 0s 0ms\n","Time for processing: 0h 1m 57s 969ms\n","align /content/mmseqs2/tmp/29/candidateDB_53 /content/mmseqs2/swissprot/swissprotDB /content/mmseqs2/tmp/29/8645457134183679496/pref_0 /content/mmseqs2/tmp/29/resultsDB_53 --sub-mat nucl:nucleotide.out,aa:blosum62.out -a 0 --alignment-mode 2 --wrapped-scoring 0 -e 0.001 --min-seq-id 0 --min-aln-len 0 --seq-id-mode 0 --alt-ali 0 -c 0 --cov-mode 0 --max-seq-len 65535 --comp-bias-corr 1 --realign 0 --max-rejected 2147483647 --max-accept 2147483647 --add-self-matches 0 --db-load-mode 0 --pca 1 --pcb 1.5 --score-bias 0 --gap-open nucl:5,aa:11 --gap-extend nucl:2,aa:1 --zdrop 40 --threads 2 --compressed 0 -v 3 \n","\n","Compute score and coverage\n","Query database size: 1727 type: Aminoacid\n","Target database size: 563082 type: Aminoacid\n","Calculation of alignments\n","[=================================================================] 100.00% 1.73K 18s 324ms\n","Time for merging to resultsDB_53: 0h 0m 0s 0ms\n","\n","442415 alignments calculated.\n","91351 sequence pairs passed the thresholds (0.206483 of overall calculated).\n","52.895775 hits per query sequence.\n","Time for processing: 0h 0m 18s 399ms\n","convertalis /content/mmseqs2/tmp/29/candidateDB_53 /content/mmseqs2/swissprot/swissprotDB /content/mmseqs2/tmp/29/resultsDB_53 /content/mmseqs2/tmp/29/resultDB_secondary.m8 --format-output query,target,evalue,raw \n","\n","MMseqs Version:        \tdf69c26e1c9aaeaa3f5d72fd6e782d02742b2b0c\n","Substitution matrix    \tnucl:nucleotide.out,aa:blosum62.out\n","Alignment format       \t0\n","Format alignment output\tquery,target,evalue,raw\n","Translation table      \t1\n","Gap open cost          \tnucl:5,aa:11\n","Gap extension cost     \tnucl:2,aa:1\n","Database output        \tfalse\n","Preload mode           \t0\n","Search type            \t0\n","Threads                \t2\n","Compressed             \t0\n","Verbosity              \t3\n","\n","[=================================================================] 100.00% 1.73K 0s 137ms\n","Time for merging to resultDB_secondary.m8: 0h 0m 0s 3ms\n","Time for processing: 0h 0m 0s 246ms\n","found 976 genes\n","\n","\n","Creating graph of contig 2...\n","\n"],"name":"stdout"},{"output_type":"display_data","data":{"application/vnd.jupyter.widget-view+json":{"model_id":"32cd0fe732b54002b4664c28dc5e074b","version_minor":0,"version_major":2},"text/plain":["HBox(children=(FloatProgress(value=0.0, max=20499.0), HTML(value='')))"]},"metadata":{"tags":[]}},{"output_type":"stream","text":["\n","Maximizing geneiness...\n","\n","Filtering predictions with mmseqs2...\n","createdb /content/mmseqs2/tmp/30/candidate_genes_35.fasta /content/mmseqs2/tmp/30/candidateDB_35 \n","\n","MMseqs Version:       \tdf69c26e1c9aaeaa3f5d72fd6e782d02742b2b0c\n","Database type         \t0\n","Shuffle input database\ttrue\n","Createdb mode         \t0\n","Write lookup file     \t1\n","Offset of numeric ids \t0\n","Compressed            \t0\n","Verbosity             \t3\n","\n","Converting sequences\n","[1415] 0s 4ms\n","Time for merging to candidateDB_35_h: 0h 0m 0s 2ms\n","Time for merging to candidateDB_35: 0h 0m 0s 2ms\n","Database type: Aminoacid\n","Time for processing: 0h 0m 0s 12ms\n","createdb /content/mmseqs2/tmp/30/candidate_genes_53.fasta /content/mmseqs2/tmp/30/candidateDB_53 \n","\n","MMseqs Version:       \tdf69c26e1c9aaeaa3f5d72fd6e782d02742b2b0c\n","Database type         \t0\n","Shuffle input database\ttrue\n","Createdb mode         \t0\n","Write lookup file     \t1\n","Offset of numeric ids \t0\n","Compressed            \t0\n","Verbosity             \t3\n","\n","Converting sequences\n","[1415] 0s 4ms\n","Time for merging to candidateDB_53_h: 0h 0m 0s 2ms\n","Time for merging to candidateDB_53: 0h 0m 0s 2ms\n","Database type: Aminoacid\n","Time for processing: 0h 0m 0s 12ms\n","search -s 7.0 /content/mmseqs2/tmp/30/candidateDB_35 /content/mmseqs2/genexa/genexaDB /content/mmseqs2/tmp/30/resultsDB_35 /content/mmseqs2/tmp/30 \n","\n","MMseqs Version:                     \tdf69c26e1c9aaeaa3f5d72fd6e782d02742b2b0c\n","Substitution matrix                 \tnucl:nucleotide.out,aa:blosum62.out\n","Add backtrace                       \tfalse\n","Alignment mode                      \t2\n","Allow wrapped scoring               \tfalse\n","E-value threshold                   \t0.001\n","Seq. id. threshold                  \t0\n","Min alignment length                \t0\n","Seq. id. mode                       \t0\n","Alternative alignments              \t0\n","Coverage threshold                  \t0\n","Coverage mode                       \t0\n","Max sequence length                 \t65535\n","Compositional bias                  \t1\n","Realign hits                        \tfalse\n","Max reject                          \t2147483647\n","Max accept                          \t2147483647\n","Include identical seq. id.          \tfalse\n","Preload mode                        \t0\n","Pseudo count a                      \t1\n","Pseudo count b                      \t1.5\n","Score bias                          \t0\n","Gap open cost                       \tnucl:5,aa:11\n","Gap extension cost                  \tnucl:2,aa:1\n","Zdrop                               \t40\n","Threads                             \t2\n","Compressed                          \t0\n","Verbosity                           \t3\n","Seed substitution matrix            \tnucl:nucleotide.out,aa:VTML80.out\n","Sensitivity                         \t7\n","k-mer length                        \t0\n","k-score                             \t2147483647\n","Alphabet size                       \tnucl:5,aa:21\n","Max results per query               \t300\n","Split database                      \t0\n","Split mode                          \t2\n","Split memory limit                  \t0\n","Diagonal scoring                    \ttrue\n","Exact k-mer matching                \t0\n","Mask residues                       \t1\n","Mask lower case residues            \t0\n","Minimum diagonal score              \t15\n","Spaced k-mers                       \t1\n","Spaced k-mer pattern                \t\n","Local temporary path                \t\n","Rescore mode                        \t0\n","Remove hits by seq. id. and coverage\tfalse\n","Sort results                        \t0\n","Mask profile                        \t1\n","Profile E-value threshold           \t0.1\n","Global sequence weighting           \tfalse\n","Allow deletions                     \tfalse\n","Filter MSA                          \t1\n","Maximum seq. id. threshold          \t0.9\n","Minimum seq. id.                    \t0\n","Minimum score per column            \t-20\n","Minimum coverage                    \t0\n","Select N most diverse seqs          \t1000\n","Omit consensus                      \tfalse\n","Min codons in orf                   \t30\n","Max codons in length                \t32734\n","Max orf gaps                        \t2147483647\n","Contig start mode                   \t2\n","Contig end mode                     \t2\n","Orf start mode                      \t1\n","Forward frames                      \t1,2,3\n","Reverse frames                      \t1,2,3\n","Translation table                   \t1\n","Translate orf                       \t0\n","Use all table starts                \tfalse\n","Offset of numeric ids               \t0\n","Create lookup                       \t0\n","Add orf stop                        \tfalse\n","Chain overlapping alignments        \t0\n","Merge query                         \t1\n","Search type                         \t0\n","Search iterations                   \t1\n","Start sensitivity                   \t4\n","Search steps                        \t1\n","Slice search mode                   \tfalse\n","Strand selection                    \t1\n","Disk space limit                    \t0\n","MPI runner                          \t\n","Force restart with latest tmp       \tfalse\n","Remove temporary files              \tfalse\n","\n","prefilter /content/mmseqs2/tmp/30/candidateDB_35 /content/mmseqs2/genexa/genexaDB.idx /content/mmseqs2/tmp/30/15138720644892661692/pref_0 --sub-mat nucl:nucleotide.out,aa:blosum62.out --seed-sub-mat nucl:nucleotide.out,aa:VTML80.out -k 0 --k-score 2147483647 --alph-size nucl:5,aa:21 --max-seq-len 65535 --max-seqs 300 --split 0 --split-mode 2 --split-memory-limit 0 -c 0 --cov-mode 0 --comp-bias-corr 1 --diag-score 1 --exact-kmer-matching 0 --mask 1 --mask-lower-case 0 --min-ungapped-score 15 --add-self-matches 0 --spaced-kmer-mode 1 --db-load-mode 0 --pca 1 --pcb 1.5 --threads 2 --compressed 0 -v 3 -s 7.0 \n","\n","Index version: 16\n","Generated by:  df69c26e1c9aaeaa3f5d72fd6e782d02742b2b0c\n","ScoreMatrix:  VTML80.out\n","Query database size: 1425 type: Aminoacid\n","Estimated memory consumption: 2G\n","Target database size: 468351 type: Aminoacid\n","Process prefiltering step 1 of 1\n","\n","k-mer similarity threshold: 100\n","Starting prefiltering scores calculation (step 1 of 1)\n","Query db start 1 to 1425\n","Target db start 1 to 468351\n","[=================================================================] 100.00% 1.42K 17s 531ms\n","\n","570.076824 k-mers per position\n","428752 DB matches per sequence\n","27 overflows\n","0 queries produce too many hits (truncated result)\n","254 sequences passed prefiltering per query sequence\n","300 median result list length\n","8 sequences with 0 size result lists\n","Time for merging to pref_0: 0h 0m 0s 0ms\n","Time for processing: 0h 0m 19s 236ms\n","align /content/mmseqs2/tmp/30/candidateDB_35 /content/mmseqs2/genexa/genexaDB.idx /content/mmseqs2/tmp/30/15138720644892661692/pref_0 /content/mmseqs2/tmp/30/resultsDB_35 --sub-mat nucl:nucleotide.out,aa:blosum62.out -a 0 --alignment-mode 2 --wrapped-scoring 0 -e 0.001 --min-seq-id 0 --min-aln-len 0 --seq-id-mode 0 --alt-ali 0 -c 0 --cov-mode 0 --max-seq-len 65535 --comp-bias-corr 1 --realign 0 --max-rejected 2147483647 --max-accept 2147483647 --add-self-matches 0 --db-load-mode 0 --pca 1 --pcb 1.5 --score-bias 0 --gap-open nucl:5,aa:11 --gap-extend nucl:2,aa:1 --zdrop 40 --threads 2 --compressed 0 -v 3 \n","\n","Index version: 16\n","Generated by:  df69c26e1c9aaeaa3f5d72fd6e782d02742b2b0c\n","ScoreMatrix:  VTML80.out\n","Compute score and coverage\n","Query database size: 1425 type: Aminoacid\n","Target database size: 468351 type: Aminoacid\n","Calculation of alignments\n","[=================================================================] 100.00% 1.42K 18s 64ms\n","Time for merging to resultsDB_35: 0h 0m 0s 0ms\n","\n","362015 alignments calculated.\n","86235 sequence pairs passed the thresholds (0.238208 of overall calculated).\n","60.515789 hits per query sequence.\n","Time for processing: 0h 0m 18s 94ms\n","convertalis /content/mmseqs2/tmp/30/candidateDB_35 /content/mmseqs2/genexa/genexaDB /content/mmseqs2/tmp/30/resultsDB_35 /content/mmseqs2/tmp/30/resultDB_genexa.m8 --format-output query,target,evalue,raw \n","\n","MMseqs Version:        \tdf69c26e1c9aaeaa3f5d72fd6e782d02742b2b0c\n","Substitution matrix    \tnucl:nucleotide.out,aa:blosum62.out\n","Alignment format       \t0\n","Format alignment output\tquery,target,evalue,raw\n","Translation table      \t1\n","Gap open cost          \tnucl:5,aa:11\n","Gap extension cost     \tnucl:2,aa:1\n","Database output        \tfalse\n","Preload mode           \t0\n","Search type            \t0\n","Threads                \t2\n","Compressed             \t0\n","Verbosity              \t3\n","\n","[=================================================================] 100.00% 1.42K 0s 151ms\n","Time for merging to resultDB_genexa.m8: 0h 0m 0s 3ms\n","Time for processing: 0h 0m 0s 237ms\n","search -s 7.0 /content/mmseqs2/tmp/30/candidateDB_53 /content/mmseqs2/swissprot/swissprotDB /content/mmseqs2/tmp/30/resultsDB_53 /content/mmseqs2/tmp/30 \n","\n","MMseqs Version:                     \tdf69c26e1c9aaeaa3f5d72fd6e782d02742b2b0c\n","Substitution matrix                 \tnucl:nucleotide.out,aa:blosum62.out\n","Add backtrace                       \tfalse\n","Alignment mode                      \t2\n","Allow wrapped scoring               \tfalse\n","E-value threshold                   \t0.001\n","Seq. id. threshold                  \t0\n","Min alignment length                \t0\n","Seq. id. mode                       \t0\n","Alternative alignments              \t0\n","Coverage threshold                  \t0\n","Coverage mode                       \t0\n","Max sequence length                 \t65535\n","Compositional bias                  \t1\n","Realign hits                        \tfalse\n","Max reject                          \t2147483647\n","Max accept                          \t2147483647\n","Include identical seq. id.          \tfalse\n","Preload mode                        \t0\n","Pseudo count a                      \t1\n","Pseudo count b                      \t1.5\n","Score bias                          \t0\n","Gap open cost                       \tnucl:5,aa:11\n","Gap extension cost                  \tnucl:2,aa:1\n","Zdrop                               \t40\n","Threads                             \t2\n","Compressed                          \t0\n","Verbosity                           \t3\n","Seed substitution matrix            \tnucl:nucleotide.out,aa:VTML80.out\n","Sensitivity                         \t7\n","k-mer length                        \t0\n","k-score                             \t2147483647\n","Alphabet size                       \tnucl:5,aa:21\n","Max results per query               \t300\n","Split database                      \t0\n","Split mode                          \t2\n","Split memory limit                  \t0\n","Diagonal scoring                    \ttrue\n","Exact k-mer matching                \t0\n","Mask residues                       \t1\n","Mask lower case residues            \t0\n","Minimum diagonal score              \t15\n","Spaced k-mers                       \t1\n","Spaced k-mer pattern                \t\n","Local temporary path                \t\n","Rescore mode                        \t0\n","Remove hits by seq. id. and coverage\tfalse\n","Sort results                        \t0\n","Mask profile                        \t1\n","Profile E-value threshold           \t0.1\n","Global sequence weighting           \tfalse\n","Allow deletions                     \tfalse\n","Filter MSA                          \t1\n","Maximum seq. id. threshold          \t0.9\n","Minimum seq. id.                    \t0\n","Minimum score per column            \t-20\n","Minimum coverage                    \t0\n","Select N most diverse seqs          \t1000\n","Omit consensus                      \tfalse\n","Min codons in orf                   \t30\n","Max codons in length                \t32734\n","Max orf gaps                        \t2147483647\n","Contig start mode                   \t2\n","Contig end mode                     \t2\n","Orf start mode                      \t1\n","Forward frames                      \t1,2,3\n","Reverse frames                      \t1,2,3\n","Translation table                   \t1\n","Translate orf                       \t0\n","Use all table starts                \tfalse\n","Offset of numeric ids               \t0\n","Create lookup                       \t0\n","Add orf stop                        \tfalse\n","Chain overlapping alignments        \t0\n","Merge query                         \t1\n","Search type                         \t0\n","Search iterations                   \t1\n","Start sensitivity                   \t4\n","Search steps                        \t1\n","Slice search mode                   \tfalse\n","Strand selection                    \t1\n","Disk space limit                    \t0\n","MPI runner                          \t\n","Force restart with latest tmp       \tfalse\n","Remove temporary files              \tfalse\n","\n","prefilter /content/mmseqs2/tmp/30/candidateDB_53 /content/mmseqs2/swissprot/swissprotDB /content/mmseqs2/tmp/30/10808256401178000478/pref_0 --sub-mat nucl:nucleotide.out,aa:blosum62.out --seed-sub-mat nucl:nucleotide.out,aa:VTML80.out -k 0 --k-score 2147483647 --alph-size nucl:5,aa:21 --max-seq-len 65535 --max-seqs 300 --split 0 --split-mode 2 --split-memory-limit 0 -c 0 --cov-mode 0 --comp-bias-corr 1 --diag-score 1 --exact-kmer-matching 0 --mask 1 --mask-lower-case 0 --min-ungapped-score 15 --add-self-matches 0 --spaced-kmer-mode 1 --db-load-mode 0 --pca 1 --pcb 1.5 --threads 2 --compressed 0 -v 3 -s 7.0 \n","\n","Query database size: 1425 type: Aminoacid\n","Estimated memory consumption: 2G\n","Target database size: 563082 type: Aminoacid\n","Index table k-mer threshold: 100 at k-mer size 6 \n","Index table: counting k-mers\n","[=================================================================] 100.00% 563.08K 42s 768ms\n","Index table: Masked residues: 3167729\n","Index table: fill\n","[=================================================================] 100.00% 563.08K 47s 14ms\n","Index statistics\n","Entries:          192947007\n","DB size:          1592 MB\n","Avg k-mer size:   3.014797\n","Top 10 k-mers\n","    GPGGTL\t1837\n","    GQSWTV\t1706\n","    WGMFAT\t1638\n","    PGVFEV\t1634\n","    VLWQFW\t1623\n","    AYIRPN\t1587\n","    RSPKGV\t1585\n","    TPHKWY\t1560\n","    HIRVGW\t1548\n","    ITLSPY\t1541\n","Time for index table init: 0h 1m 33s 451ms\n","Process prefiltering step 1 of 1\n","\n","k-mer similarity threshold: 100\n","Starting prefiltering scores calculation (step 1 of 1)\n","Query db start 1 to 1425\n","Target db start 1 to 563082\n","[=================================================================] 100.00% 1.42K 18s 745ms\n","\n","568.524867 k-mers per position\n","485576 DB matches per sequence\n","49 overflows\n","0 queries produce too many hits (truncated result)\n","260 sequences passed prefiltering per query sequence\n","300 median result list length\n","5 sequences with 0 size result lists\n","Time for merging to pref_0: 0h 0m 0s 0ms\n","Time for processing: 0h 1m 54s 737ms\n","align /content/mmseqs2/tmp/30/candidateDB_53 /content/mmseqs2/swissprot/swissprotDB /content/mmseqs2/tmp/30/10808256401178000478/pref_0 /content/mmseqs2/tmp/30/resultsDB_53 --sub-mat nucl:nucleotide.out,aa:blosum62.out -a 0 --alignment-mode 2 --wrapped-scoring 0 -e 0.001 --min-seq-id 0 --min-aln-len 0 --seq-id-mode 0 --alt-ali 0 -c 0 --cov-mode 0 --max-seq-len 65535 --comp-bias-corr 1 --realign 0 --max-rejected 2147483647 --max-accept 2147483647 --add-self-matches 0 --db-load-mode 0 --pca 1 --pcb 1.5 --score-bias 0 --gap-open nucl:5,aa:11 --gap-extend nucl:2,aa:1 --zdrop 40 --threads 2 --compressed 0 -v 3 \n","\n","Compute score and coverage\n","Query database size: 1425 type: Aminoacid\n","Target database size: 563082 type: Aminoacid\n","Calculation of alignments\n","[=================================================================] 100.00% 1.42K 15s 531ms\n","Time for merging to resultsDB_53: 0h 0m 0s 0ms\n","\n","370950 alignments calculated.\n","68565 sequence pairs passed the thresholds (0.184836 of overall calculated).\n","48.115791 hits per query sequence.\n","Time for processing: 0h 0m 15s 600ms\n","convertalis /content/mmseqs2/tmp/30/candidateDB_53 /content/mmseqs2/swissprot/swissprotDB /content/mmseqs2/tmp/30/resultsDB_53 /content/mmseqs2/tmp/30/resultDB_secondary.m8 --format-output query,target,evalue,raw \n","\n","MMseqs Version:        \tdf69c26e1c9aaeaa3f5d72fd6e782d02742b2b0c\n","Substitution matrix    \tnucl:nucleotide.out,aa:blosum62.out\n","Alignment format       \t0\n","Format alignment output\tquery,target,evalue,raw\n","Translation table      \t1\n","Gap open cost          \tnucl:5,aa:11\n","Gap extension cost     \tnucl:2,aa:1\n","Database output        \tfalse\n","Preload mode           \t0\n","Search type            \t0\n","Threads                \t2\n","Compressed             \t0\n","Verbosity              \t3\n","\n","[=================================================================] 100.00% 1.42K 0s 103ms\n","Time for merging to resultDB_secondary.m8: 0h 0m 0s 2ms\n","Time for processing: 0h 0m 0s 214ms\n","found 776 genes\n","\n","\n","Reading fasta file GCF_000024185.1_ASM2418v1_genomic.fna.gz...\n","\n","Finding and translating open reading frames...\n","\n","Encoding amino acids...\n","\n","Applying protein kmer filter...\n","\n","Scoring ORFs with temporal convolutional network...\n","\n"],"name":"stdout"},{"output_type":"display_data","data":{"application/vnd.jupyter.widget-view+json":{"model_id":"83252378b2d443de95e55481e3788ffa","version_minor":0,"version_major":2},"text/plain":["HBox(children=(FloatProgress(value=0.0, max=506.0), HTML(value='')))"]},"metadata":{"tags":[]}},{"output_type":"stream","text":["\n","Scoring translation initiation sites...\n","\n"],"name":"stdout"},{"output_type":"display_data","data":{"application/vnd.jupyter.widget-view+json":{"model_id":"784f9f6d4f91404d9e6060a04e01b7de","version_minor":0,"version_major":2},"text/plain":["HBox(children=(FloatProgress(value=0.0, max=102.0), HTML(value='')))"]},"metadata":{"tags":[]}},{"output_type":"stream","text":["\n","Creating graph of contig 0...\n","\n"],"name":"stdout"},{"output_type":"display_data","data":{"application/vnd.jupyter.widget-view+json":{"model_id":"7f6ca2b1839f4a639b5847198413d50b","version_minor":0,"version_major":2},"text/plain":["HBox(children=(FloatProgress(value=0.0, max=101058.0), HTML(value='')))"]},"metadata":{"tags":[]}},{"output_type":"stream","text":["\n","Maximizing geneiness...\n","\n","Filtering predictions with mmseqs2...\n","createdb /content/mmseqs2/tmp/31/candidate_genes_35.fasta /content/mmseqs2/tmp/31/candidateDB_35 \n","\n","MMseqs Version:       \tdf69c26e1c9aaeaa3f5d72fd6e782d02742b2b0c\n","Database type         \t0\n","Shuffle input database\ttrue\n","Createdb mode         \t0\n","Write lookup file     \t1\n","Offset of numeric ids \t0\n","Compressed            \t0\n","Verbosity             \t3\n","\n","Converting sequences\n","[4849] 0s 8ms\n","Time for merging to candidateDB_35_h: 0h 0m 0s 2ms\n","Time for merging to candidateDB_35: 0h 0m 0s 3ms\n","Database type: Aminoacid\n","Time for processing: 0h 0m 0s 21ms\n","createdb /content/mmseqs2/tmp/31/candidate_genes_53.fasta /content/mmseqs2/tmp/31/candidateDB_53 \n","\n","MMseqs Version:       \tdf69c26e1c9aaeaa3f5d72fd6e782d02742b2b0c\n","Database type         \t0\n","Shuffle input database\ttrue\n","Createdb mode         \t0\n","Write lookup file     \t1\n","Offset of numeric ids \t0\n","Compressed            \t0\n","Verbosity             \t3\n","\n","Converting sequences\n","[4849] 0s 8ms\n","Time for merging to candidateDB_53_h: 0h 0m 0s 2ms\n","Time for merging to candidateDB_53: 0h 0m 0s 3ms\n","Database type: Aminoacid\n","Time for processing: 0h 0m 0s 27ms\n","search -s 7.0 /content/mmseqs2/tmp/31/candidateDB_35 /content/mmseqs2/genexa/genexaDB /content/mmseqs2/tmp/31/resultsDB_35 /content/mmseqs2/tmp/31 \n","\n","MMseqs Version:                     \tdf69c26e1c9aaeaa3f5d72fd6e782d02742b2b0c\n","Substitution matrix                 \tnucl:nucleotide.out,aa:blosum62.out\n","Add backtrace                       \tfalse\n","Alignment mode                      \t2\n","Allow wrapped scoring               \tfalse\n","E-value threshold                   \t0.001\n","Seq. id. threshold                  \t0\n","Min alignment length                \t0\n","Seq. id. mode                       \t0\n","Alternative alignments              \t0\n","Coverage threshold                  \t0\n","Coverage mode                       \t0\n","Max sequence length                 \t65535\n","Compositional bias                  \t1\n","Realign hits                        \tfalse\n","Max reject                          \t2147483647\n","Max accept                          \t2147483647\n","Include identical seq. id.          \tfalse\n","Preload mode                        \t0\n","Pseudo count a                      \t1\n","Pseudo count b                      \t1.5\n","Score bias                          \t0\n","Gap open cost                       \tnucl:5,aa:11\n","Gap extension cost                  \tnucl:2,aa:1\n","Zdrop                               \t40\n","Threads                             \t2\n","Compressed                          \t0\n","Verbosity                           \t3\n","Seed substitution matrix            \tnucl:nucleotide.out,aa:VTML80.out\n","Sensitivity                         \t7\n","k-mer length                        \t0\n","k-score                             \t2147483647\n","Alphabet size                       \tnucl:5,aa:21\n","Max results per query               \t300\n","Split database                      \t0\n","Split mode                          \t2\n","Split memory limit                  \t0\n","Diagonal scoring                    \ttrue\n","Exact k-mer matching                \t0\n","Mask residues                       \t1\n","Mask lower case residues            \t0\n","Minimum diagonal score              \t15\n","Spaced k-mers                       \t1\n","Spaced k-mer pattern                \t\n","Local temporary path                \t\n","Rescore mode                        \t0\n","Remove hits by seq. id. and coverage\tfalse\n","Sort results                        \t0\n","Mask profile                        \t1\n","Profile E-value threshold           \t0.1\n","Global sequence weighting           \tfalse\n","Allow deletions                     \tfalse\n","Filter MSA                          \t1\n","Maximum seq. id. threshold          \t0.9\n","Minimum seq. id.                    \t0\n","Minimum score per column            \t-20\n","Minimum coverage                    \t0\n","Select N most diverse seqs          \t1000\n","Omit consensus                      \tfalse\n","Min codons in orf                   \t30\n","Max codons in length                \t32734\n","Max orf gaps                        \t2147483647\n","Contig start mode                   \t2\n","Contig end mode                     \t2\n","Orf start mode                      \t1\n","Forward frames                      \t1,2,3\n","Reverse frames                      \t1,2,3\n","Translation table                   \t1\n","Translate orf                       \t0\n","Use all table starts                \tfalse\n","Offset of numeric ids               \t0\n","Create lookup                       \t0\n","Add orf stop                        \tfalse\n","Chain overlapping alignments        \t0\n","Merge query                         \t1\n","Search type                         \t0\n","Search iterations                   \t1\n","Start sensitivity                   \t4\n","Search steps                        \t1\n","Slice search mode                   \tfalse\n","Strand selection                    \t1\n","Disk space limit                    \t0\n","MPI runner                          \t\n","Force restart with latest tmp       \tfalse\n","Remove temporary files              \tfalse\n","\n","prefilter /content/mmseqs2/tmp/31/candidateDB_35 /content/mmseqs2/genexa/genexaDB.idx /content/mmseqs2/tmp/31/14427309109855859165/pref_0 --sub-mat nucl:nucleotide.out,aa:blosum62.out --seed-sub-mat nucl:nucleotide.out,aa:VTML80.out -k 0 --k-score 2147483647 --alph-size nucl:5,aa:21 --max-seq-len 65535 --max-seqs 300 --split 0 --split-mode 2 --split-memory-limit 0 -c 0 --cov-mode 0 --comp-bias-corr 1 --diag-score 1 --exact-kmer-matching 0 --mask 1 --mask-lower-case 0 --min-ungapped-score 15 --add-self-matches 0 --spaced-kmer-mode 1 --db-load-mode 0 --pca 1 --pcb 1.5 --threads 2 --compressed 0 -v 3 -s 7.0 \n","\n","Index version: 16\n","Generated by:  df69c26e1c9aaeaa3f5d72fd6e782d02742b2b0c\n","ScoreMatrix:  VTML80.out\n","Query database size: 4906 type: Aminoacid\n","Estimated memory consumption: 2G\n","Target database size: 468351 type: Aminoacid\n","Process prefiltering step 1 of 1\n","\n","k-mer similarity threshold: 100\n","Starting prefiltering scores calculation (step 1 of 1)\n","Query db start 1 to 4906\n","Target db start 1 to 468351\n","[=================================================================] 100.00% 4.91K 1m 1s 349ms\n","\n","667.875863 k-mers per position\n","390149 DB matches per sequence\n","110 overflows\n","0 queries produce too many hits (truncated result)\n","198 sequences passed prefiltering per query sequence\n","300 median result list length\n","161 sequences with 0 size result lists\n","Time for merging to pref_0: 0h 0m 0s 1ms\n","Time for processing: 0h 1m 2s 963ms\n","align /content/mmseqs2/tmp/31/candidateDB_35 /content/mmseqs2/genexa/genexaDB.idx /content/mmseqs2/tmp/31/14427309109855859165/pref_0 /content/mmseqs2/tmp/31/resultsDB_35 --sub-mat nucl:nucleotide.out,aa:blosum62.out -a 0 --alignment-mode 2 --wrapped-scoring 0 -e 0.001 --min-seq-id 0 --min-aln-len 0 --seq-id-mode 0 --alt-ali 0 -c 0 --cov-mode 0 --max-seq-len 65535 --comp-bias-corr 1 --realign 0 --max-rejected 2147483647 --max-accept 2147483647 --add-self-matches 0 --db-load-mode 0 --pca 1 --pcb 1.5 --score-bias 0 --gap-open nucl:5,aa:11 --gap-extend nucl:2,aa:1 --zdrop 40 --threads 2 --compressed 0 -v 3 \n","\n","Index version: 16\n","Generated by:  df69c26e1c9aaeaa3f5d72fd6e782d02742b2b0c\n","ScoreMatrix:  VTML80.out\n","Compute score and coverage\n","Query database size: 4906 type: Aminoacid\n","Target database size: 468351 type: Aminoacid\n","Calculation of alignments\n","[=================================================================] 100.00% 4.91K 34s 10ms\n","Time for merging to resultsDB_35: 0h 0m 0s 1ms\n","\n","975674 alignments calculated.\n","141012 sequence pairs passed the thresholds (0.144528 of overall calculated).\n","28.742764 hits per query sequence.\n","Time for processing: 0h 0m 34s 500ms\n","convertalis /content/mmseqs2/tmp/31/candidateDB_35 /content/mmseqs2/genexa/genexaDB /content/mmseqs2/tmp/31/resultsDB_35 /content/mmseqs2/tmp/31/resultDB_genexa.m8 --format-output query,target,evalue,raw \n","\n","MMseqs Version:        \tdf69c26e1c9aaeaa3f5d72fd6e782d02742b2b0c\n","Substitution matrix    \tnucl:nucleotide.out,aa:blosum62.out\n","Alignment format       \t0\n","Format alignment output\tquery,target,evalue,raw\n","Translation table      \t1\n","Gap open cost          \tnucl:5,aa:11\n","Gap extension cost     \tnucl:2,aa:1\n","Database output        \tfalse\n","Preload mode           \t0\n","Search type            \t0\n","Threads                \t2\n","Compressed             \t0\n","Verbosity              \t3\n","\n","[=================================================================] 100.00% 4.91K 0s 239ms\n","Time for merging to resultDB_genexa.m8: 0h 0m 0s 6ms\n","Time for processing: 0h 0m 0s 331ms\n","search -s 7.0 /content/mmseqs2/tmp/31/candidateDB_53 /content/mmseqs2/swissprot/swissprotDB /content/mmseqs2/tmp/31/resultsDB_53 /content/mmseqs2/tmp/31 \n","\n","MMseqs Version:                     \tdf69c26e1c9aaeaa3f5d72fd6e782d02742b2b0c\n","Substitution matrix                 \tnucl:nucleotide.out,aa:blosum62.out\n","Add backtrace                       \tfalse\n","Alignment mode                      \t2\n","Allow wrapped scoring               \tfalse\n","E-value threshold                   \t0.001\n","Seq. id. threshold                  \t0\n","Min alignment length                \t0\n","Seq. id. mode                       \t0\n","Alternative alignments              \t0\n","Coverage threshold                  \t0\n","Coverage mode                       \t0\n","Max sequence length                 \t65535\n","Compositional bias                  \t1\n","Realign hits                        \tfalse\n","Max reject                          \t2147483647\n","Max accept                          \t2147483647\n","Include identical seq. id.          \tfalse\n","Preload mode                        \t0\n","Pseudo count a                      \t1\n","Pseudo count b                      \t1.5\n","Score bias                          \t0\n","Gap open cost                       \tnucl:5,aa:11\n","Gap extension cost                  \tnucl:2,aa:1\n","Zdrop                               \t40\n","Threads                             \t2\n","Compressed                          \t0\n","Verbosity                           \t3\n","Seed substitution matrix            \tnucl:nucleotide.out,aa:VTML80.out\n","Sensitivity                         \t7\n","k-mer length                        \t0\n","k-score                             \t2147483647\n","Alphabet size                       \tnucl:5,aa:21\n","Max results per query               \t300\n","Split database                      \t0\n","Split mode                          \t2\n","Split memory limit                  \t0\n","Diagonal scoring                    \ttrue\n","Exact k-mer matching                \t0\n","Mask residues                       \t1\n","Mask lower case residues            \t0\n","Minimum diagonal score              \t15\n","Spaced k-mers                       \t1\n","Spaced k-mer pattern                \t\n","Local temporary path                \t\n","Rescore mode                        \t0\n","Remove hits by seq. id. and coverage\tfalse\n","Sort results                        \t0\n","Mask profile                        \t1\n","Profile E-value threshold           \t0.1\n","Global sequence weighting           \tfalse\n","Allow deletions                     \tfalse\n","Filter MSA                          \t1\n","Maximum seq. id. threshold          \t0.9\n","Minimum seq. id.                    \t0\n","Minimum score per column            \t-20\n","Minimum coverage                    \t0\n","Select N most diverse seqs          \t1000\n","Omit consensus                      \tfalse\n","Min codons in orf                   \t30\n","Max codons in length                \t32734\n","Max orf gaps                        \t2147483647\n","Contig start mode                   \t2\n","Contig end mode                     \t2\n","Orf start mode                      \t1\n","Forward frames                      \t1,2,3\n","Reverse frames                      \t1,2,3\n","Translation table                   \t1\n","Translate orf                       \t0\n","Use all table starts                \tfalse\n","Offset of numeric ids               \t0\n","Create lookup                       \t0\n","Add orf stop                        \tfalse\n","Chain overlapping alignments        \t0\n","Merge query                         \t1\n","Search type                         \t0\n","Search iterations                   \t1\n","Start sensitivity                   \t4\n","Search steps                        \t1\n","Slice search mode                   \tfalse\n","Strand selection                    \t1\n","Disk space limit                    \t0\n","MPI runner                          \t\n","Force restart with latest tmp       \tfalse\n","Remove temporary files              \tfalse\n","\n","prefilter /content/mmseqs2/tmp/31/candidateDB_53 /content/mmseqs2/swissprot/swissprotDB /content/mmseqs2/tmp/31/10068077091781853631/pref_0 --sub-mat nucl:nucleotide.out,aa:blosum62.out --seed-sub-mat nucl:nucleotide.out,aa:VTML80.out -k 0 --k-score 2147483647 --alph-size nucl:5,aa:21 --max-seq-len 65535 --max-seqs 300 --split 0 --split-mode 2 --split-memory-limit 0 -c 0 --cov-mode 0 --comp-bias-corr 1 --diag-score 1 --exact-kmer-matching 0 --mask 1 --mask-lower-case 0 --min-ungapped-score 15 --add-self-matches 0 --spaced-kmer-mode 1 --db-load-mode 0 --pca 1 --pcb 1.5 --threads 2 --compressed 0 -v 3 -s 7.0 \n","\n","Query database size: 4906 type: Aminoacid\n","Estimated memory consumption: 2G\n","Target database size: 563082 type: Aminoacid\n","Index table k-mer threshold: 100 at k-mer size 6 \n","Index table: counting k-mers\n","[=================================================================] 100.00% 563.08K 41s 683ms\n","Index table: Masked residues: 3167729\n","Index table: fill\n","[=================================================================] 100.00% 563.08K 42s 843ms\n","Index statistics\n","Entries:          192947007\n","DB size:          1592 MB\n","Avg k-mer size:   3.014797\n","Top 10 k-mers\n","    GPGGTL\t1837\n","    GQSWTV\t1706\n","    WGMFAT\t1638\n","    PGVFEV\t1634\n","    VLWQFW\t1623\n","    AYIRPN\t1587\n","    RSPKGV\t1585\n","    TPHKWY\t1560\n","    HIRVGW\t1548\n","    ITLSPY\t1541\n","Time for index table init: 0h 1m 28s 154ms\n","Process prefiltering step 1 of 1\n","\n","k-mer similarity threshold: 100\n","Starting prefiltering scores calculation (step 1 of 1)\n","Query db start 1 to 4906\n","Target db start 1 to 563082\n","[=================================================================] 100.00% 4.91K 1m 8s 679ms\n","\n","666.963095 k-mers per position\n","497960 DB matches per sequence\n","209 overflows\n","0 queries produce too many hits (truncated result)\n","217 sequences passed prefiltering per query sequence\n","300 median result list length\n","145 sequences with 0 size result lists\n","Time for merging to pref_0: 0h 0m 0s 1ms\n","Time for processing: 0h 2m 39s 290ms\n","align /content/mmseqs2/tmp/31/candidateDB_53 /content/mmseqs2/swissprot/swissprotDB /content/mmseqs2/tmp/31/10068077091781853631/pref_0 /content/mmseqs2/tmp/31/resultsDB_53 --sub-mat nucl:nucleotide.out,aa:blosum62.out -a 0 --alignment-mode 2 --wrapped-scoring 0 -e 0.001 --min-seq-id 0 --min-aln-len 0 --seq-id-mode 0 --alt-ali 0 -c 0 --cov-mode 0 --max-seq-len 65535 --comp-bias-corr 1 --realign 0 --max-rejected 2147483647 --max-accept 2147483647 --add-self-matches 0 --db-load-mode 0 --pca 1 --pcb 1.5 --score-bias 0 --gap-open nucl:5,aa:11 --gap-extend nucl:2,aa:1 --zdrop 40 --threads 2 --compressed 0 -v 3 \n","\n","Compute score and coverage\n","Query database size: 4906 type: Aminoacid\n","Target database size: 563082 type: Aminoacid\n","Calculation of alignments\n","[=================================================================] 100.00% 4.91K 37s 390ms\n","Time for merging to resultsDB_53: 0h 0m 0s 1ms\n","\n","1067818 alignments calculated.\n","130359 sequence pairs passed the thresholds (0.122080 of overall calculated).\n","26.571341 hits per query sequence.\n","Time for processing: 0h 0m 37s 463ms\n","convertalis /content/mmseqs2/tmp/31/candidateDB_53 /content/mmseqs2/swissprot/swissprotDB /content/mmseqs2/tmp/31/resultsDB_53 /content/mmseqs2/tmp/31/resultDB_secondary.m8 --format-output query,target,evalue,raw \n","\n","MMseqs Version:        \tdf69c26e1c9aaeaa3f5d72fd6e782d02742b2b0c\n","Substitution matrix    \tnucl:nucleotide.out,aa:blosum62.out\n","Alignment format       \t0\n","Format alignment output\tquery,target,evalue,raw\n","Translation table      \t1\n","Gap open cost          \tnucl:5,aa:11\n","Gap extension cost     \tnucl:2,aa:1\n","Database output        \tfalse\n","Preload mode           \t0\n","Search type            \t0\n","Threads                \t2\n","Compressed             \t0\n","Verbosity              \t3\n","\n","[=================================================================] 100.00% 4.91K 0s 193ms\n","Time for merging to resultDB_secondary.m8: 0h 0m 0s 8ms\n","Time for processing: 0h 0m 0s 313ms\n","found 2132 genes\n","\n","\n","Reading fasta file GCF_000969965.1_ASM96996v1_genomic.fna.gz...\n","\n","Finding and translating open reading frames...\n","\n","Encoding amino acids...\n","\n","Applying protein kmer filter...\n","\n","Scoring ORFs with temporal convolutional network...\n","\n"],"name":"stdout"},{"output_type":"display_data","data":{"application/vnd.jupyter.widget-view+json":{"model_id":"0b6a02b6506b4f38ac38a893c54ad922","version_minor":0,"version_major":2},"text/plain":["HBox(children=(FloatProgress(value=0.0, max=786.0), HTML(value='')))"]},"metadata":{"tags":[]}},{"output_type":"stream","text":["\n","Scoring translation initiation sites...\n","\n"],"name":"stdout"},{"output_type":"display_data","data":{"application/vnd.jupyter.widget-view+json":{"model_id":"f0dd7a8b691842ab9dfadd32461bfa58","version_minor":0,"version_major":2},"text/plain":["HBox(children=(FloatProgress(value=0.0, max=158.0), HTML(value='')))"]},"metadata":{"tags":[]}},{"output_type":"stream","text":["\n","Creating graph of contig 0...\n","\n"],"name":"stdout"},{"output_type":"display_data","data":{"application/vnd.jupyter.widget-view+json":{"model_id":"eddc3f39ed034abbaccb8d81076303d3","version_minor":0,"version_major":2},"text/plain":["HBox(children=(FloatProgress(value=0.0, max=157113.0), HTML(value='')))"]},"metadata":{"tags":[]}},{"output_type":"stream","text":["\n","Maximizing geneiness...\n","\n","Filtering predictions with mmseqs2...\n","createdb /content/mmseqs2/tmp/32/candidate_genes_35.fasta /content/mmseqs2/tmp/32/candidateDB_35 \n","\n","MMseqs Version:       \tdf69c26e1c9aaeaa3f5d72fd6e782d02742b2b0c\n","Database type         \t0\n","Shuffle input database\ttrue\n","Createdb mode         \t0\n","Write lookup file     \t1\n","Offset of numeric ids \t0\n","Compressed            \t0\n","Verbosity             \t3\n","\n","Converting sequences\n","[9697] 0s 16ms\n","Time for merging to candidateDB_35_h: 0h 0m 0s 2ms\n","Time for merging to candidateDB_35: 0h 0m 0s 4ms\n","Database type: Aminoacid\n","Time for processing: 0h 0m 0s 35ms\n","createdb /content/mmseqs2/tmp/32/candidate_genes_53.fasta /content/mmseqs2/tmp/32/candidateDB_53 \n","\n","MMseqs Version:       \tdf69c26e1c9aaeaa3f5d72fd6e782d02742b2b0c\n","Database type         \t0\n","Shuffle input database\ttrue\n","Createdb mode         \t0\n","Write lookup file     \t1\n","Offset of numeric ids \t0\n","Compressed            \t0\n","Verbosity             \t3\n","\n","Converting sequences\n","[9697] 0s 14ms\n","Time for merging to candidateDB_53_h: 0h 0m 0s 3ms\n","Time for merging to candidateDB_53: 0h 0m 0s 3ms\n","Database type: Aminoacid\n","Time for processing: 0h 0m 0s 33ms\n","search -s 7.0 /content/mmseqs2/tmp/32/candidateDB_35 /content/mmseqs2/genexa/genexaDB /content/mmseqs2/tmp/32/resultsDB_35 /content/mmseqs2/tmp/32 \n","\n","MMseqs Version:                     \tdf69c26e1c9aaeaa3f5d72fd6e782d02742b2b0c\n","Substitution matrix                 \tnucl:nucleotide.out,aa:blosum62.out\n","Add backtrace                       \tfalse\n","Alignment mode                      \t2\n","Allow wrapped scoring               \tfalse\n","E-value threshold                   \t0.001\n","Seq. id. threshold                  \t0\n","Min alignment length                \t0\n","Seq. id. mode                       \t0\n","Alternative alignments              \t0\n","Coverage threshold                  \t0\n","Coverage mode                       \t0\n","Max sequence length                 \t65535\n","Compositional bias                  \t1\n","Realign hits                        \tfalse\n","Max reject                          \t2147483647\n","Max accept                          \t2147483647\n","Include identical seq. id.          \tfalse\n","Preload mode                        \t0\n","Pseudo count a                      \t1\n","Pseudo count b                      \t1.5\n","Score bias                          \t0\n","Gap open cost                       \tnucl:5,aa:11\n","Gap extension cost                  \tnucl:2,aa:1\n","Zdrop                               \t40\n","Threads                             \t2\n","Compressed                          \t0\n","Verbosity                           \t3\n","Seed substitution matrix            \tnucl:nucleotide.out,aa:VTML80.out\n","Sensitivity                         \t7\n","k-mer length                        \t0\n","k-score                             \t2147483647\n","Alphabet size                       \tnucl:5,aa:21\n","Max results per query               \t300\n","Split database                      \t0\n","Split mode                          \t2\n","Split memory limit                  \t0\n","Diagonal scoring                    \ttrue\n","Exact k-mer matching                \t0\n","Mask residues                       \t1\n","Mask lower case residues            \t0\n","Minimum diagonal score              \t15\n","Spaced k-mers                       \t1\n","Spaced k-mer pattern                \t\n","Local temporary path                \t\n","Rescore mode                        \t0\n","Remove hits by seq. id. and coverage\tfalse\n","Sort results                        \t0\n","Mask profile                        \t1\n","Profile E-value threshold           \t0.1\n","Global sequence weighting           \tfalse\n","Allow deletions                     \tfalse\n","Filter MSA                          \t1\n","Maximum seq. id. threshold          \t0.9\n","Minimum seq. id.                    \t0\n","Minimum score per column            \t-20\n","Minimum coverage                    \t0\n","Select N most diverse seqs          \t1000\n","Omit consensus                      \tfalse\n","Min codons in orf                   \t30\n","Max codons in length                \t32734\n","Max orf gaps                        \t2147483647\n","Contig start mode                   \t2\n","Contig end mode                     \t2\n","Orf start mode                      \t1\n","Forward frames                      \t1,2,3\n","Reverse frames                      \t1,2,3\n","Translation table                   \t1\n","Translate orf                       \t0\n","Use all table starts                \tfalse\n","Offset of numeric ids               \t0\n","Create lookup                       \t0\n","Add orf stop                        \tfalse\n","Chain overlapping alignments        \t0\n","Merge query                         \t1\n","Search type                         \t0\n","Search iterations                   \t1\n","Start sensitivity                   \t4\n","Search steps                        \t1\n","Slice search mode                   \tfalse\n","Strand selection                    \t1\n","Disk space limit                    \t0\n","MPI runner                          \t\n","Force restart with latest tmp       \tfalse\n","Remove temporary files              \tfalse\n","\n","prefilter /content/mmseqs2/tmp/32/candidateDB_35 /content/mmseqs2/genexa/genexaDB.idx /content/mmseqs2/tmp/32/13715897574819056638/pref_0 --sub-mat nucl:nucleotide.out,aa:blosum62.out --seed-sub-mat nucl:nucleotide.out,aa:VTML80.out -k 0 --k-score 2147483647 --alph-size nucl:5,aa:21 --max-seq-len 65535 --max-seqs 300 --split 0 --split-mode 2 --split-memory-limit 0 -c 0 --cov-mode 0 --comp-bias-corr 1 --diag-score 1 --exact-kmer-matching 0 --mask 1 --mask-lower-case 0 --min-ungapped-score 15 --add-self-matches 0 --spaced-kmer-mode 1 --db-load-mode 0 --pca 1 --pcb 1.5 --threads 2 --compressed 0 -v 3 -s 7.0 \n","\n","Index version: 16\n","Generated by:  df69c26e1c9aaeaa3f5d72fd6e782d02742b2b0c\n","ScoreMatrix:  VTML80.out\n","Query database size: 9730 type: Aminoacid\n","Estimated memory consumption: 2G\n","Target database size: 468351 type: Aminoacid\n","Process prefiltering step 1 of 1\n","\n","k-mer similarity threshold: 100\n","Starting prefiltering scores calculation (step 1 of 1)\n","Query db start 1 to 9730\n","Target db start 1 to 468351\n","[=================================================================] 100.00% 9.73K 1m 34s 621ms\n","\n","859.892555 k-mers per position\n","313883 DB matches per sequence\n","157 overflows\n","0 queries produce too many hits (truncated result)\n","248 sequences passed prefiltering per query sequence\n","300 median result list length\n","52 sequences with 0 size result lists\n","Time for merging to pref_0: 0h 0m 0s 2ms\n","Time for processing: 0h 1m 36s 254ms\n","align /content/mmseqs2/tmp/32/candidateDB_35 /content/mmseqs2/genexa/genexaDB.idx /content/mmseqs2/tmp/32/13715897574819056638/pref_0 /content/mmseqs2/tmp/32/resultsDB_35 --sub-mat nucl:nucleotide.out,aa:blosum62.out -a 0 --alignment-mode 2 --wrapped-scoring 0 -e 0.001 --min-seq-id 0 --min-aln-len 0 --seq-id-mode 0 --alt-ali 0 -c 0 --cov-mode 0 --max-seq-len 65535 --comp-bias-corr 1 --realign 0 --max-rejected 2147483647 --max-accept 2147483647 --add-self-matches 0 --db-load-mode 0 --pca 1 --pcb 1.5 --score-bias 0 --gap-open nucl:5,aa:11 --gap-extend nucl:2,aa:1 --zdrop 40 --threads 2 --compressed 0 -v 3 \n","\n","Index version: 16\n","Generated by:  df69c26e1c9aaeaa3f5d72fd6e782d02742b2b0c\n","ScoreMatrix:  VTML80.out\n","Compute score and coverage\n","Query database size: 9730 type: Aminoacid\n","Target database size: 468351 type: Aminoacid\n","Calculation of alignments\n","[=================================================================] 100.00% 9.73K 58s 637ms\n","Time for merging to resultsDB_35: 0h 0m 0s 2ms\n","\n","2416375 alignments calculated.\n","239179 sequence pairs passed the thresholds (0.098983 of overall calculated).\n","24.581604 hits per query sequence.\n","Time for processing: 0h 0m 58s 670ms\n","convertalis /content/mmseqs2/tmp/32/candidateDB_35 /content/mmseqs2/genexa/genexaDB /content/mmseqs2/tmp/32/resultsDB_35 /content/mmseqs2/tmp/32/resultDB_genexa.m8 --format-output query,target,evalue,raw \n","\n","MMseqs Version:        \tdf69c26e1c9aaeaa3f5d72fd6e782d02742b2b0c\n","Substitution matrix    \tnucl:nucleotide.out,aa:blosum62.out\n","Alignment format       \t0\n","Format alignment output\tquery,target,evalue,raw\n","Translation table      \t1\n","Gap open cost          \tnucl:5,aa:11\n","Gap extension cost     \tnucl:2,aa:1\n","Database output        \tfalse\n","Preload mode           \t0\n","Search type            \t0\n","Threads                \t2\n","Compressed             \t0\n","Verbosity              \t3\n","\n","[=================================================================] 100.00% 9.73K 0s 390ms\n","Time for merging to resultDB_genexa.m8: 0h 0m 0s 9ms\n","Time for processing: 0h 0m 0s 491ms\n","search -s 7.0 /content/mmseqs2/tmp/32/candidateDB_53 /content/mmseqs2/swissprot/swissprotDB /content/mmseqs2/tmp/32/resultsDB_53 /content/mmseqs2/tmp/32 \n","\n","MMseqs Version:                     \tdf69c26e1c9aaeaa3f5d72fd6e782d02742b2b0c\n","Substitution matrix                 \tnucl:nucleotide.out,aa:blosum62.out\n","Add backtrace                       \tfalse\n","Alignment mode                      \t2\n","Allow wrapped scoring               \tfalse\n","E-value threshold                   \t0.001\n","Seq. id. threshold                  \t0\n","Min alignment length                \t0\n","Seq. id. mode                       \t0\n","Alternative alignments              \t0\n","Coverage threshold                  \t0\n","Coverage mode                       \t0\n","Max sequence length                 \t65535\n","Compositional bias                  \t1\n","Realign hits                        \tfalse\n","Max reject                          \t2147483647\n","Max accept                          \t2147483647\n","Include identical seq. id.          \tfalse\n","Preload mode                        \t0\n","Pseudo count a                      \t1\n","Pseudo count b                      \t1.5\n","Score bias                          \t0\n","Gap open cost                       \tnucl:5,aa:11\n","Gap extension cost                  \tnucl:2,aa:1\n","Zdrop                               \t40\n","Threads                             \t2\n","Compressed                          \t0\n","Verbosity                           \t3\n","Seed substitution matrix            \tnucl:nucleotide.out,aa:VTML80.out\n","Sensitivity                         \t7\n","k-mer length                        \t0\n","k-score                             \t2147483647\n","Alphabet size                       \tnucl:5,aa:21\n","Max results per query               \t300\n","Split database                      \t0\n","Split mode                          \t2\n","Split memory limit                  \t0\n","Diagonal scoring                    \ttrue\n","Exact k-mer matching                \t0\n","Mask residues                       \t1\n","Mask lower case residues            \t0\n","Minimum diagonal score              \t15\n","Spaced k-mers                       \t1\n","Spaced k-mer pattern                \t\n","Local temporary path                \t\n","Rescore mode                        \t0\n","Remove hits by seq. id. and coverage\tfalse\n","Sort results                        \t0\n","Mask profile                        \t1\n","Profile E-value threshold           \t0.1\n","Global sequence weighting           \tfalse\n","Allow deletions                     \tfalse\n","Filter MSA                          \t1\n","Maximum seq. id. threshold          \t0.9\n","Minimum seq. id.                    \t0\n","Minimum score per column            \t-20\n","Minimum coverage                    \t0\n","Select N most diverse seqs          \t1000\n","Omit consensus                      \tfalse\n","Min codons in orf                   \t30\n","Max codons in length                \t32734\n","Max orf gaps                        \t2147483647\n","Contig start mode                   \t2\n","Contig end mode                     \t2\n","Orf start mode                      \t1\n","Forward frames                      \t1,2,3\n","Reverse frames                      \t1,2,3\n","Translation table                   \t1\n","Translate orf                       \t0\n","Use all table starts                \tfalse\n","Offset of numeric ids               \t0\n","Create lookup                       \t0\n","Add orf stop                        \tfalse\n","Chain overlapping alignments        \t0\n","Merge query                         \t1\n","Search type                         \t0\n","Search iterations                   \t1\n","Start sensitivity                   \t4\n","Search steps                        \t1\n","Slice search mode                   \tfalse\n","Strand selection                    \t1\n","Disk space limit                    \t0\n","MPI runner                          \t\n","Force restart with latest tmp       \tfalse\n","Remove temporary files              \tfalse\n","\n","prefilter /content/mmseqs2/tmp/32/candidateDB_53 /content/mmseqs2/swissprot/swissprotDB /content/mmseqs2/tmp/32/9327897782385706784/pref_0 --sub-mat nucl:nucleotide.out,aa:blosum62.out --seed-sub-mat nucl:nucleotide.out,aa:VTML80.out -k 0 --k-score 2147483647 --alph-size nucl:5,aa:21 --max-seq-len 65535 --max-seqs 300 --split 0 --split-mode 2 --split-memory-limit 0 -c 0 --cov-mode 0 --comp-bias-corr 1 --diag-score 1 --exact-kmer-matching 0 --mask 1 --mask-lower-case 0 --min-ungapped-score 15 --add-self-matches 0 --spaced-kmer-mode 1 --db-load-mode 0 --pca 1 --pcb 1.5 --threads 2 --compressed 0 -v 3 -s 7.0 \n","\n","Query database size: 9730 type: Aminoacid\n","Estimated memory consumption: 2G\n","Target database size: 563082 type: Aminoacid\n","Index table k-mer threshold: 100 at k-mer size 6 \n","Index table: counting k-mers\n","[=================================================================] 100.00% 563.08K 40s 757ms\n","Index table: Masked residues: 3167729\n","Index table: fill\n","[=================================================================] 100.00% 563.08K 43s 892ms\n","Index statistics\n","Entries:          192947007\n","DB size:          1592 MB\n","Avg k-mer size:   3.014797\n","Top 10 k-mers\n","    GPGGTL\t1837\n","    GQSWTV\t1706\n","    WGMFAT\t1638\n","    PGVFEV\t1634\n","    VLWQFW\t1623\n","    AYIRPN\t1587\n","    RSPKGV\t1585\n","    TPHKWY\t1560\n","    HIRVGW\t1548\n","    ITLSPY\t1541\n","Time for index table init: 0h 1m 28s 129ms\n","Process prefiltering step 1 of 1\n","\n","k-mer similarity threshold: 100\n","Starting prefiltering scores calculation (step 1 of 1)\n","Query db start 1 to 9730\n","Target db start 1 to 563082\n","[=================================================================] 100.00% 9.73K 1m 44s 455ms\n","\n","859.037860 k-mers per position\n","393564 DB matches per sequence\n","305 overflows\n","0 queries produce too many hits (truncated result)\n","264 sequences passed prefiltering per query sequence\n","300 median result list length\n","45 sequences with 0 size result lists\n","Time for merging to pref_0: 0h 0m 0s 8ms\n","Time for processing: 0h 3m 14s 974ms\n","align /content/mmseqs2/tmp/32/candidateDB_53 /content/mmseqs2/swissprot/swissprotDB /content/mmseqs2/tmp/32/9327897782385706784/pref_0 /content/mmseqs2/tmp/32/resultsDB_53 --sub-mat nucl:nucleotide.out,aa:blosum62.out -a 0 --alignment-mode 2 --wrapped-scoring 0 -e 0.001 --min-seq-id 0 --min-aln-len 0 --seq-id-mode 0 --alt-ali 0 -c 0 --cov-mode 0 --max-seq-len 65535 --comp-bias-corr 1 --realign 0 --max-rejected 2147483647 --max-accept 2147483647 --add-self-matches 0 --db-load-mode 0 --pca 1 --pcb 1.5 --score-bias 0 --gap-open nucl:5,aa:11 --gap-extend nucl:2,aa:1 --zdrop 40 --threads 2 --compressed 0 -v 3 \n","\n","Compute score and coverage\n","Query database size: 9730 type: Aminoacid\n","Target database size: 563082 type: Aminoacid\n","Calculation of alignments\n","[=================================================================] 100.00% 9.73K 59s 372ms\n","Time for merging to resultsDB_53: 0h 0m 0s 2ms\n","\n","2576559 alignments calculated.\n","184493 sequence pairs passed the thresholds (0.071604 of overall calculated).\n","18.961254 hits per query sequence.\n","Time for processing: 0h 0m 59s 764ms\n","convertalis /content/mmseqs2/tmp/32/candidateDB_53 /content/mmseqs2/swissprot/swissprotDB /content/mmseqs2/tmp/32/resultsDB_53 /content/mmseqs2/tmp/32/resultDB_secondary.m8 --format-output query,target,evalue,raw \n","\n","MMseqs Version:        \tdf69c26e1c9aaeaa3f5d72fd6e782d02742b2b0c\n","Substitution matrix    \tnucl:nucleotide.out,aa:blosum62.out\n","Alignment format       \t0\n","Format alignment output\tquery,target,evalue,raw\n","Translation table      \t1\n","Gap open cost          \tnucl:5,aa:11\n","Gap extension cost     \tnucl:2,aa:1\n","Database output        \tfalse\n","Preload mode           \t0\n","Search type            \t0\n","Threads                \t2\n","Compressed             \t0\n","Verbosity              \t3\n","\n","[=================================================================] 100.00% 9.73K 0s 270ms\n","Time for merging to resultDB_secondary.m8: 0h 0m 0s 9ms\n","Time for processing: 0h 0m 0s 601ms\n","found 3407 genes\n","\n","\n","Done\n"],"name":"stdout"}]},{"cell_type":"code","metadata":{"id":"M3_sRPUzDfkX","colab_type":"code","cellView":"form","colab":{"base_uri":"https://localhost:8080/","height":17},"executionInfo":{"status":"ok","timestamp":1599429543605,"user_tz":240,"elapsed":2338307,"user":{"displayName":"Markus Sommer","photoUrl":"https://lh3.googleusercontent.com/a-/AOh14GjXujbpCzD6oVvcRIT5EP-oi3Bif3Cfcu7npJUnce8=s64","userId":"08083804319459268544"}},"outputId":"9738e27a-b8d8-4e5b-f5a2-5fc30251f63f"},"source":["# @title Download genome annotation (you may need to rerun this cell and/or allow multiple downloads in browser)\n","# TODO support different output formats\n","\n","def write_GFF(start, end, contig, strand, contig_name_all, contig_length_all, contig_seq_all, GFF_path_out):\n","    # writes to same format as Prokka GFF\n","    bases_per_line = 60\n","    with open(GFF_path_out, \"wt\") as f:\n","        # header\n","        f.writelines([\"##gff-version 3\\n\"])\n","\n","        # contig names and lengths\n","        datalines = [\" \".join([\"##sequence-region\", \n","                               str(contig_name_all[i]), \n","                               \"1\", \n","                               str(contig_length_all[i]), \n","                               \"\\n\"]) for i in range(len(contig_name_all))]\n","        f.writelines(datalines)\n","\n","        # CDS features\n","        datalines = [\"\\t\".join([contig[i],\n","                                \"Balrog\",\n","                                \"CDS\",\n","                                str(int(start[i]) + 1),\n","                                end[i],\n","                                \".\", # TODO: replace . with actual gene score\n","                                strand[i],\n","                                \"0\",\n","                                \"inference=ab initio prediction:Balrog;product=hypothetical protein\"\n","                                \"\\n\"]) for i in range(len(start))]\n","        f.writelines(datalines)\n","\n","        # # contig sequences\n","        # f.writelines([\"##FASTA\", \"\\n\"])\n","        # for i, name in enumerate(contig_name_all):\n","        #     f.writelines([\">\", str(name), \"\\n\"])\n","        #     bases_per_line = 60\n","        #     seq = contig_seq_all[i]\n","        #     datalines = [str(seq[j:j+bases_per_line])+\"\\n\" for j in range(0, len(seq), bases_per_line)]\n","        #     f.writelines(datalines)\n","\n","fasta_names = list(genome_dict.keys())\n","gff_names = [str(x) + \"__.gff\" for x in fasta_names] # simpler than removing all combinations of fasta and gz from the end\n","\n","for genome_idx, gff in enumerate(gff_names):\n","    # combine all info for gene predictions\n","    contig_gene_start_flat = []\n","    contig_gene_end_flat = []\n","    contig_gene_strand_flat = []\n","    contig_gene_contig_flat = []\n","    try:\n","        for i, contig_gene_coord in enumerate(contig_gene_coord_list[genome_idx]): # TODO get rid of jenky nested lists\n","            for k, coord in enumerate(contig_gene_coord):\n","                start = str(coord[0] - 3)\n","                end = str(coord[1] + 3)\n","                strandnum = contig_gene_strand_list[genome_idx][i][k]\n","                if strandnum == 1:\n","                    strand = \"+\"\n","                else:\n","                    strand = \"-\"\n","                contig = str(contig_name_list[genome_idx][i])\n","\n","                contig_gene_start_flat.append(start)\n","                contig_gene_end_flat.append(end)\n","                contig_gene_strand_flat.append(strand)\n","                contig_gene_contig_flat.append(contig)\n","\n","        write_GFF(contig_gene_start_flat, contig_gene_end_flat, contig_gene_contig_flat, contig_gene_strand_flat, \n","                contig_name_list[genome_idx], contig_length_list[genome_idx], contig_seq_list[genome_idx], gff)\n","    except:\n","        print(\"Could not generate \", gff)\n","\n","for gff in gff_names:\n","    try:\n","        files.download(gff)\n","    except:\n","        print(\"Could not download \", gff)"],"execution_count":28,"outputs":[{"output_type":"display_data","data":{"application/javascript":["\n","    async function download(id, filename, size) {\n","      if (!google.colab.kernel.accessAllowed) {\n","        return;\n","      }\n","      const div = document.createElement('div');\n","      const label = document.createElement('label');\n","      label.textContent = `Downloading \"${filename}\": `;\n","      div.appendChild(label);\n","      const progress = document.createElement('progress');\n","      progress.max = size;\n","      div.appendChild(progress);\n","      document.body.appendChild(div);\n","\n","      const buffers = [];\n","      let downloaded = 0;\n","\n","      const channel = await google.colab.kernel.comms.open(id);\n","      // Send a message to notify the kernel that we're ready.\n","      channel.send({})\n","\n","      for await (const message of channel.messages) {\n","        // Send a message to notify the kernel that we're ready.\n","        channel.send({})\n","        if (message.buffers) {\n","          for (const buffer of message.buffers) {\n","            buffers.push(buffer);\n","            downloaded += buffer.byteLength;\n","            progress.value = downloaded;\n","          }\n","        }\n","      }\n","      const blob = new Blob(buffers, {type: 'application/binary'});\n","      const a = document.createElement('a');\n","      a.href = window.URL.createObjectURL(blob);\n","      a.download = filename;\n","      div.appendChild(a);\n","      a.click();\n","      div.remove();\n","    }\n","  "],"text/plain":["<IPython.core.display.Javascript object>"]},"metadata":{"tags":[]}},{"output_type":"display_data","data":{"application/javascript":["download(\"download_365f2a40-cfe3-437e-bdac-814f764e2851\", \"GCA_000830275.1_ASM83027v1_genomic.fna.gz__.gff\", 133730)"],"text/plain":["<IPython.core.display.Javascript object>"]},"metadata":{"tags":[]}},{"output_type":"display_data","data":{"application/javascript":["\n","    async function download(id, filename, size) {\n","      if (!google.colab.kernel.accessAllowed) {\n","        return;\n","      }\n","      const div = document.createElement('div');\n","      const label = document.createElement('label');\n","      label.textContent = `Downloading \"${filename}\": `;\n","      div.appendChild(label);\n","      const progress = document.createElement('progress');\n","      progress.max = size;\n","      div.appendChild(progress);\n","      document.body.appendChild(div);\n","\n","      const buffers = [];\n","      let downloaded = 0;\n","\n","      const channel = await google.colab.kernel.comms.open(id);\n","      // Send a message to notify the kernel that we're ready.\n","      channel.send({})\n","\n","      for await (const message of channel.messages) {\n","        // Send a message to notify the kernel that we're ready.\n","        channel.send({})\n","        if (message.buffers) {\n","          for (const buffer of message.buffers) {\n","            buffers.push(buffer);\n","            downloaded += buffer.byteLength;\n","            progress.value = downloaded;\n","          }\n","        }\n","      }\n","      const blob = new Blob(buffers, {type: 'application/binary'});\n","      const a = document.createElement('a');\n","      a.href = window.URL.createObjectURL(blob);\n","      a.download = filename;\n","      div.appendChild(a);\n","      a.click();\n","      div.remove();\n","    }\n","  "],"text/plain":["<IPython.core.display.Javascript object>"]},"metadata":{"tags":[]}},{"output_type":"display_data","data":{"application/javascript":["download(\"download_924e51e0-fbdd-411a-a7c7-91d5a7c69a17\", \"GCF_000015765.1_ASM1576v1_genomic.fna.gz__.gff\", 193939)"],"text/plain":["<IPython.core.display.Javascript object>"]},"metadata":{"tags":[]}},{"output_type":"display_data","data":{"application/javascript":["\n","    async function download(id, filename, size) {\n","      if (!google.colab.kernel.accessAllowed) {\n","        return;\n","      }\n","      const div = document.createElement('div');\n","      const label = document.createElement('label');\n","      label.textContent = `Downloading \"${filename}\": `;\n","      div.appendChild(label);\n","      const progress = document.createElement('progress');\n","      progress.max = size;\n","      div.appendChild(progress);\n","      document.body.appendChild(div);\n","\n","      const buffers = [];\n","      let downloaded = 0;\n","\n","      const channel = await google.colab.kernel.comms.open(id);\n","      // Send a message to notify the kernel that we're ready.\n","      channel.send({})\n","\n","      for await (const message of channel.messages) {\n","        // Send a message to notify the kernel that we're ready.\n","        channel.send({})\n","        if (message.buffers) {\n","          for (const buffer of message.buffers) {\n","            buffers.push(buffer);\n","            downloaded += buffer.byteLength;\n","            progress.value = downloaded;\n","          }\n","        }\n","      }\n","      const blob = new Blob(buffers, {type: 'application/binary'});\n","      const a = document.createElement('a');\n","      a.href = window.URL.createObjectURL(blob);\n","      a.download = filename;\n","      div.appendChild(a);\n","      a.click();\n","      div.remove();\n","    }\n","  "],"text/plain":["<IPython.core.display.Javascript object>"]},"metadata":{"tags":[]}},{"output_type":"display_data","data":{"application/javascript":["download(\"download_fb6f476e-8742-4a20-a1f8-10cab162f609\", \"GCF_000022205.1_ASM2220v1_genomic.fna.gz__.gff\", 482712)"],"text/plain":["<IPython.core.display.Javascript object>"]},"metadata":{"tags":[]}},{"output_type":"display_data","data":{"application/javascript":["\n","    async function download(id, filename, size) {\n","      if (!google.colab.kernel.accessAllowed) {\n","        return;\n","      }\n","      const div = document.createElement('div');\n","      const label = document.createElement('label');\n","      label.textContent = `Downloading \"${filename}\": `;\n","      div.appendChild(label);\n","      const progress = document.createElement('progress');\n","      progress.max = size;\n","      div.appendChild(progress);\n","      document.body.appendChild(div);\n","\n","      const buffers = [];\n","      let downloaded = 0;\n","\n","      const channel = await google.colab.kernel.comms.open(id);\n","      // Send a message to notify the kernel that we're ready.\n","      channel.send({})\n","\n","      for await (const message of channel.messages) {\n","        // Send a message to notify the kernel that we're ready.\n","        channel.send({})\n","        if (message.buffers) {\n","          for (const buffer of message.buffers) {\n","            buffers.push(buffer);\n","            downloaded += buffer.byteLength;\n","            progress.value = downloaded;\n","          }\n","        }\n","      }\n","      const blob = new Blob(buffers, {type: 'application/binary'});\n","      const a = document.createElement('a');\n","      a.href = window.URL.createObjectURL(blob);\n","      a.download = filename;\n","      div.appendChild(a);\n","      a.click();\n","      div.remove();\n","    }\n","  "],"text/plain":["<IPython.core.display.Javascript object>"]},"metadata":{"tags":[]}},{"output_type":"display_data","data":{"application/javascript":["download(\"download_0e5752d1-e630-461a-8006-c21c443ed4f7\", \"GCF_000024185.1_ASM2418v1_genomic.fna.gz__.gff\", 237235)"],"text/plain":["<IPython.core.display.Javascript object>"]},"metadata":{"tags":[]}},{"output_type":"display_data","data":{"application/javascript":["\n","    async function download(id, filename, size) {\n","      if (!google.colab.kernel.accessAllowed) {\n","        return;\n","      }\n","      const div = document.createElement('div');\n","      const label = document.createElement('label');\n","      label.textContent = `Downloading \"${filename}\": `;\n","      div.appendChild(label);\n","      const progress = document.createElement('progress');\n","      progress.max = size;\n","      div.appendChild(progress);\n","      document.body.appendChild(div);\n","\n","      const buffers = [];\n","      let downloaded = 0;\n","\n","      const channel = await google.colab.kernel.comms.open(id);\n","      // Send a message to notify the kernel that we're ready.\n","      channel.send({})\n","\n","      for await (const message of channel.messages) {\n","        // Send a message to notify the kernel that we're ready.\n","        channel.send({})\n","        if (message.buffers) {\n","          for (const buffer of message.buffers) {\n","            buffers.push(buffer);\n","            downloaded += buffer.byteLength;\n","            progress.value = downloaded;\n","          }\n","        }\n","      }\n","      const blob = new Blob(buffers, {type: 'application/binary'});\n","      const a = document.createElement('a');\n","      a.href = window.URL.createObjectURL(blob);\n","      a.download = filename;\n","      div.appendChild(a);\n","      a.click();\n","      div.remove();\n","    }\n","  "],"text/plain":["<IPython.core.display.Javascript object>"]},"metadata":{"tags":[]}},{"output_type":"display_data","data":{"application/javascript":["download(\"download_e52fe9c9-69ed-4fc1-9491-9c3ed7a472ef\", \"GCF_000969965.1_ASM96996v1_genomic.fna.gz__.gff\", 386608)"],"text/plain":["<IPython.core.display.Javascript object>"]},"metadata":{"tags":[]}}]},{"cell_type":"markdown","metadata":{"id":"4D-TsN2GZTFj","colab_type":"text"},"source":["# MIT License\n"," Copyright (c) 2020 Markus J. Sommer & Steven L. Salzberg\n","\n","Permission is hereby granted, free of charge, to any person obtaining a copy\n","of this software and associated documentation files (the \"Software\"), to deal\n","in the Software without restriction, including without limitation the rights\n","to use, copy, modify, merge, publish, distribute, sublicense, and/or sell\n","copies of the Software, and to permit persons to whom the Software is\n","furnished to do so, subject to the following conditions:\n","\n","The above copyright notice and this permission notice shall be included in all\n","copies or substantial portions of the Software.\n","\n","THE SOFTWARE IS PROVIDED \"AS IS\", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR\n","IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,\n","FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE\n","AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER\n","LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,\n","OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE\n","SOFTWARE."]}]}